{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "gpuType": "T4"
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "OApzwC5ZqbXo",
        "outputId": "9f4e28fd-9eb5-4acb-cd69-78a69f1b4d6d"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['E_01_001\\t28652246\\t--\\thg19\\tchr6\\t86160197\\t86176777\\tHuman\\tMelanoma\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\tA ChIP-qPCR approach revealed prominent binding of c-Jun at the AP-1 sites #1-2 and #5-6 (Fig. 3E) coinciding with enhancer regions (H3K27Ac) identified by the ENCODE ChIP-seq project (Fig. 3D).\\tEnhancer\\tNT5E\\tCRISPR/Cas9\\t\"qPCR,ChIP-qPCR\"\\t\"Next, we devised a CRISPR/Cas9 approach to clarify the functional relevance of the individual c-Jun/AP-1 binding sites.c-Jun/AP-1 induces CD73 expression via binding to an intronic enhancer.Strong enrichment of AP-1 site mutagenesis frequencies in the CD73 low subfractions (52% versus 4% in CD73 lowest 10% versus highest 10%) was observed only in the cell population transfected with the sgRNA against the c-Jun/AP-1 site #5 (Fig. 3G-H) being consistent with reduced CD73 surface expression (Fig. S4A).\"\\t\"CALJA,CD73,E5NT,NT,NT5,NTE,eN,eNT\"\\tMelanoma\\tDOID:1909\\tD008545\\tThe intronic Enhancer of the CD73 gene controls its c-Jun/AP-1-dependent transcriptional activation downstream of mitogenic MAPK and inflammatory cytokine signaling. \\t\"ChIP-qPCR,CRISPR/Cas9\"\\t\"Taken together, we identified a critical c-Jun/AP-1 binding site in an intronic enhancer of the CD73 gene that controls its c-Jun/AP-1-dependent transcriptional activation downstream of mitogenic MAPK and inflammatory cytokine signaling.\"\\tJUN\\t\"AP-1,AP1,c-Jun,cJUN,p39\"\\tChIP-qPCR\\tWe looked for potential c-Jun/AP-1Cbinding sites in the CD73 gene using ENCODE ChIP-Seq data.A ChIP-qPCR approach revealed prominent binding of c-Jun at the AP-1 sites #1C2 and #5C6 coinciding with Enhancer regions (H3K27Ac)identified by the ENCODE ChIP-seq project.\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t9186', 'E_01_002\\t26751173\\tERenh588\\thg19\\tchr11\\t69329684\\t69330954\\tHuman\\t\"MCF-7,T-47D\"\\tLow+High throughput\\t\"ChIP-seq,CRISPR/Cas9,Luciferase Reporter Assay,Western blot,Competitive Proliferation Assay\"\\t\"To demonstrate the generalizability of our screening approach, we designed a dropout screen to identify novel ER-bound enhancers.', 'E_01_003\\t26252164\\tNEC1 Enhancer\\thg19\\tchr2\\t223164458\\t223164688\\tHuman\\tMelanoma\\tLow throughput\\t\"Phylogenetic Footprinting,Luciferase Reporter Assay\"\\t\"Through phylogenic footprinting and in silico promoter analys islands of homology within the proximal 1.6 Kb PAX3 promo referred to as either NCE1 and NCE2 ([Milewski et al., 2004], shown schematically and by sequence in Fig. 4A,B) or sites I and II [Natoli et al., 1997].', 'E_01_004\\t26252164\\tNEC2 Enhancer\\thg19\\tchr2\\t223164846\\t223165065\\tHuman\\tMelanoma\\tLow throughput\\t\"Phylogenetic Footprinting,Luciferase Reporter Assay\"\\t\"Through phylogenic footprinting and in silico promoter analys islands of homology within the proximal 1.6 Kb PAX3 promo referred to as either NCE1 and NCE2 ([Milewski et al., 2004], shown schematically and by sequence in Fig. 4A,B) or sites I and II [Natoli et al., 1997].', 'E_01_005\\t26751173\\tp53enh3507\\thg19\\tchr6\\t36631690\\t36640853\\tHuman\\tBJ\\tLow+High throughput\\t\"ChIP-seq,CRISPR/Cas9,Luciferase Reporter Assay,siRNA,GRO-seq\"\\t\"We identified several functional enhancer elements and characterized the role of two of them in mediating p53 (TP53) and ERa (ESR1) gene regulation.', 'E_01_006\\t26751173\\tp53enh3508\\thg19\\tchr6\\t36640690\\t36643070\\tHuman\\tBJ\\tLow+High throughput\\t\"ChIP-seq,CRISPR/Cas9,Luciferase Reporter Assay,siRNA\"\\t\"We present two distinct CRISPR-Cas9 genetic screens to identify and characterize functional enhancers in their native context. Our strategy is to target Cas9 to transcription factor binding sites in enhancer regions. We identified several functional enhancer elements and characterized the role of two of them in mediating p53 (TP53) and ERa (ESR1) gene regulation.', 'E_01_007\\t26374622\\t--\\thg19\\tchr21\\t36180372\\t36181392\\tHuman\\t\"Leukemia Cell Lines,K-562\"\\tLow throughput\\t\"Luciferase Reporter Assay,DNaseI hypersensitivity\"\\t\"We searched 510 bp P1 promoter, 218bp P2 promoter, 237bp +23 enhancer, and 392bp intron 5.2 enhancer of RUNX1, 109bp upstream regulatory element (URE) and 296bp URE distal of PU.1, 443bp promoter of CEBPA, and 324bp promoter of CEBPE for single nucleotide polymorphisms (SNPs) in UCSC genome browser.510bp RUNX1 intron 5.2 enhancer contains H3K27Ac mark which is often found near active regulatory elements, and DNaseI hypersensitivity clusters which are another mark for transcription active regions.\"\\tEnhancer\\tRUNX1\\t--\\tLuciferase Reporter Assay\\t\"510bp RUNX1 intron 5.2 enhancer contains H3K27Ac mark which is often found near active regulatory elements, and DNaseI hypersensitivity clusters which are another mark for transcription active regions.We next cloned 510bp RUNX1 intron 5.2 enhancer with either rs2249650 minor allele (A) and rs2268276 major allele (G) or rs2249650 major allele (G)and rs2268276 minor allele (A) into pGL3-promoter vector. Then luciferase reporter plasmids were transfected into leukemia cell lines K562 to measure the reporter activities. 510bp reporter with rs2249650 A allele and rs2268276 G allele (L1A2G) showed significantly higher promoter activities than the reporter with rs2249650 G allele and rs2268276 A allele (Figure 2b, left).\"\\t\"AML1,AML1-EVI-1,AMLCR1,CBF2alpha,CBFA2,EVI-1,PEBP2aB,PEBP2alpha\"\\tAcute Myeloid Leukemia\\tDOID:9119\\tD015470\\tRs2249650 and rs2268276 at RUNX1 intron 5.2 enhancer were found to be associated with acute myeloid leukemia susceptibility.\\t\"Luciferase Reporter Assay,EMSA\"\\t\"We next narrowed 510bp enhancer down into 69 bp (20bp+rs2249650(A/G)-rs2268276(G/A)+20bp) DNA and cloned these nucleotides into pGL3-promoter to construct 1A2G, 1G2A, 1A2A, and 1G2G reporters and measured their activities. Both 1A2A and 1G2G reporters had medium luciferase activity. The activity of 1A2A reporter was not significantly different from that of 1A2G reporter whereas the activity of 1G2G was significantly different from that of 1A2G. Thus, rs2249650 contributed to reporter activities more than rs2268276.\"\\tSPI1\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A\"\\tEMSA\\t\"We checked the 69bp DNA and found that a classical consensus PU.1 binding sequence (GAGGAA) is located between rs2249650 and rs2268276 (Figure 1b). We adopted supershift to test if the differentially bound proteins in EMSA is PU.1. Indeed, PU.1 antibodies resulted in the lost of interested protein (Figure 3b).\"\\t\"rs2249650,rs2268276\"\\t\"34808689, 34808717 \"\\t\"Luciferase Reporter Assay,EMSA\"\\tIntron\\t\\t\\t\\t20785', 'E_01_008\\t26766440\\t--\\thg19\\tchr8\\t23448386\\t23454886\\tHuman\\tHuman CD34+ Hematopoietic Stem/Progenitor Cell\\tLow+High throughput\\t\"CRISPR/Cas9,ChIP-seq\"\\t\"The enhancer cluster upstream of the SLC25A37 gene, consisting of three distinct constituent enhancers as measured by H3K4me1 and H3K27ac ChIP-seq, is defined', 'E_01_009\\t26766440\\t--\\thg19\\tchr8\\t23433011\\t23439511\\tHuman\\tHuman CD34+ Hematopoietic Stem/Progenitor Cell\\tLow+High throughput\\t\"CRISPR/Cas9,ChIP-seq\"\\t\"The enhancer cluster upstream of the SLC25A37 gene, consisting of three distinct constituent enhancers as measured by H3K4me1 and H3K27ac ChIP-seq, is defined', 'E_01_010\\t26766440\\t--\\thg19\\tchr8\\t23422136\\t23432636\\tHuman\\tHuman CD34+ Hematopoietic Stem/Progenitor Cell\\tLow+High throughput\\t\"CRISPR/Cas9,ChIP-seq\"\\t\"The enhancer cluster upstream of the SLC25A37 gene, consisting of three distinct constituent enhancers as measured by H3K4me1 and H3K27ac ChIP-seq, is defined', 'E_01_011\\t26656844\\tMYC-LASE\\thg19\\tchr8\\t129166547\\t129190290\\tHuman\\t\"A549,NCI-H358,NCI-H2009\"\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,3C-qPCR,CRISPR/Cas9,siRNA\"\\t\"We performed H3K27ac ChIPseq in two additional lung adenocarcinoma cell lines, NCI-H2009 and NCI-H358, the lung squamous cell carcinoma cell line HCC95 and the small cell lung carcinoma cell line NCI-H2171 and validated that MYC-LASE is part of a lung adenocarcinomaCspecific super-enhancer.\"\\tSuper-Enhancer\\tMYC\\t3C\\t--\\t\"We performed chromosome conformation capture (3C) assays in A549 and Ishikawa cells and found that MYC-LASE physically interacted with the MYC promoter only in A549 cells and, reciprocally, that MYC-ECSE physically interacted with the MYC promoter only in Ishikawa cells\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\tLung Adenocarcinoma\\tDOID:3910\\tD000077192\\tTwo distinct focal amplifications of super-enhancers 3 to MYC in lung adenocarcinoma (MYC-LASE) and endometrial carcinoma (MYC-ECSE) are physically associated with the MYC promoter and correlate with MYC overexpression.\\t\"3C,ChIP-seq\"\\t\"We performed chromosome conformation capture (3C) assays in A549 and Ishikawa cells and found that MYC-LASE physically interacted with the MYC promoter only in A549 cells and, reciprocally,that MYC-ECSE physically interacted with the MYC promoter only in Ishikawa cells (Fig. 2e). In addition, tumors with amplification of MYC alone or MYC-LASE or MYC-ECSE alone had higher MYC expression than tumors lacking either amplification (Fig. 2f). These results suggest that both MYC-LASE and MYC-ECSE drive MYC expression through lineage-specific chromatin loops.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t430105', 'E_01_012\\t24413736\\t--\\thg19\\tchr5\\t51773619\\t51857504\\tHuman\\t\"HEK-293,-Cell\"\\tLow+High throughput\\t\"RNA-seq,ChIP-seq,ChIP,EMSA,PCR,4C,Luciferase Reporter Assay\"\\t\"To define enhancer clusters, we first created 1,000 iterations of randomized C3 sites in the mappable genome of individual chromosomes. We then calculated for each chromosome the 25th percentile of inter-site distances of randomized C3 sites (Supplementary Fig. 6a). Next,we defined clusters of islet C3 sites as any group of 3 C3 sites in which all adjacent C3 sites were separated by less than the abovementioned 25th percentile distance for randomized sites in the same chromosome. The distribution of islet C3 clusters differed from that of clusters generated with randomized C3 sites (Supplementary Fig. 6b)\"\\tEnhancer\\tISL1\\t4C\\t--\\tinteractions between the ISL1 promoter and the C3-3 enhancer tested in zebrafish transgenics and located >1 Mb away (green star)\\t\"ISLET1,ist-1\"\\tType 2 Diabetes Mellitus\\tDOID:9352\\tD003924\\tDysregulation of islet enhancers is relevant to the mechanisms underlying type 2 diabetes.\\tLuciferase Reporter Assay\\tWe next attempted to define functional enhancer variants within established GWAs loci that might be causal for T2D susceptibility. These results illustrate how islet accessible chromatin maps can pinpoint functional cis-regulatory variants that are strong candidates for a causal role in driving T2D-association signals.\\t\"FOXA2,MAFB,NKX2-2\"\\t\"HNF3B,TCF3B,DURS3,KRML,MCTO,NKX2.2,NKX2B\"\\tChIP-seq\\t\"To determine the genomic binding sites of these islet transcription factors, we used chromatin immunoprecipitation and sequencing(ChIP-seq) in duplicate human islet samples and identified 3,911-32,747 high-confidence sites per transcription factor. \"\\t--\\t--\\t--\\t>2KB\\t1136605', 'E_01_013\\t26656844\\t--\\thg19\\tchr13\\t73880690\\t73990596\\tHuman\\tBICR-31\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,3C-qPCR,CRISPR/Cas9,siRNA\"\\t\"The focal amplification on chr13q identified in head and neck squamous cell carcinoma (HNSC) (~110 kb, chr13:73880690-73990596) and esophageal carcinoma (ESCA) (~162 kb, chr13:73880413-74042621)\"\\tSuper-Enhancer\\tKLF5\\t--\\tChIP-seq\"The expression of KLF5, but not KLF12, is significantly higher in HNSC tumors with KLF5-HNSE amplification, suggesting that KLF5 is the target gene.\"\\t\"BTEB2,CKLF,IKLF\"\\tEsophageal Carcinoma\\tDOID:1107\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t302714', 'E_01_014\\t24413736\\t--\\thg19\\tchr6\\t170855462\\t170856787\\tHuman\\t\"HEK-293,-Cell\"\\tLow+High throughput\\t\"RNA-seq,ChIP-seq,ChIP,EMSA,PCR,4C,Luciferase Reporter Assay\"\\t\"To define enhancer clusters, we first created 1,000 iterations of randomized C3 sites in the', 'E_01_015\\t26656844\\t--\\thg19\\tchr13\\t73880413\\t74042621\\tHuman\\tEsophageal Carcinoma\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,3C-qPCR,CRISPR/Cas9,siRNA\"\\t\"The focal amplification on chr13q identified in head and neck squamous cell carcinoma (HNSC) (~110 kb, chr13:73880690-73990596) and esophageal carcinoma (ESCA) (~162 kb, chr13:73880413-74042621)\"\\tSuper-Enhancer\\tKLF5\\t--\\tChIP-seq\\t\"Similarly, the ESCA amplicon also harbors a super-enhancer based on the H3K27ac ChIP-seq profile of esophageal cells and ESCA tumors with this amplicon exhibited a trend towards increased KLF5 expression\"\\t\"BTEB2,CKLF,IKLF\"\\tLung Adenocarcinoma\\tDOID:391D000077192\\t\"Copy number gain of the e3 Enhancer region drives MYC overexpression, which contributes to tumorigenic phenotype\"\\t\"ChIP-seq,CRISPR/Cas9,PCR\"\\t\"Finally, repression of the e3 Enhancer led to a significant decrease in both anchorage-independent and clonogenic growth, suggesting that activity of the e3 Enhancer is critical for the tumorigenicity of lung adenocarcinoma cells. These results suggest that copy number gain of the e3 Enhancer region drives MYC overexpression, which contributes to tumorigenic phenotype.\"\\t\"NFE2L2,CEBPB\"\\t\"HEBP1,IMDDHH,NRF2,C/EBP-beta,IL6DBP,NF-IL6,TCF5\"\\t\"ChIP-seq,siRNA,Luciferase Reporter Assay\"\\t\"Short interfering RNA (siRNA)-mediated knockdown of NFE2L2 and CEBPB in A549 cells led to a significant reduction in e3-driven luciferase reporter activity as compared to exposure to control siRNAs. Binding of NFE2L2 and CEBP to the super-Enhancer was subsequently confirmed by ChIP-seq, with the greatest enrichment at the e3 constituent Enhancer.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t54786797', 'E_01_016\\t26656844\\t--\\thg19\\tchr13\\t27523026\\t27544353\\tHuman\\tColon Crypt Cell\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,3C-qPCR,CRISPR/Cas9,siRNA\"\\t\"Additional focal amplification peaks on chromosome 13q in colorectal carcinoma (CRC) (~21 kb, chr. 13: 27,523,026C27,544,353) and chromosome 20q in liver hepatocellular carcinoma (~22 kb,chr. 20: 48,997,377C49,019,434) were identified. ChIP-seq profiling of H3K27ac in colon crypt cells31 and in the hepatocellular carcinoma cell line HepG2 (ref. 20) demonstrated that these amplicons contain', 'E_01_017\\t26656844\\t--\\thg19\\tchr20\\t48997377\\t49019434\\tHuman\\tHep G2\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,3C-qPCR,CRISPR/Cas9,siRNA\"\\t\"Additional focal amplification peaks on chromosome 13q in colorectal carcinoma (CRC) (~21 kb, chr. 13: 27,523,026C27,544,353) and chromosome 20q in liver hepatocellular carcinoma (~22 kb,chr. 20: 48,997,377C49,019,434) were identified. ChIP-seq profiling of H3K27ac in colon crypt cells31 and in the hepatocellular carcinoma cell line HepG2 (ref. 20) demonstrated that these amplicons contain', 'E_01_018\\t26673704\\t--\\thg19\\tchr7\\t117038917\\t117040917\\tHuman\\tCaco-2\\tLow throughput\\t\"RT-qPCR,4C,ChIP,CRISPR/Cas9\"\\t\"Two pairs of gRNAs ?anking the ?20.9 kb CTCF site', 'E_01_019\\t26673704\\t--\\thg19\\tchr7\\t117099048\\t117099450\\tHuman\\tCaco-2\\tLow throughput\\t\"RT-qPCR,4C,ChIP,CRISPR/Cas9\"\\t\"Two pairs of gRNAs ?anking the ?20.9 kb CTCF site', 'E_01_020\\t26673704\\t--\\thg19\\tchr7\\t117228051\\t117229440\\tHuman\\tCaco-2\\tLow throughput\\t\"RT-qPCR,4C,ChIP,CRISPR/Cas9\"\\t\"Two pairs of gRNAs ?anking the ?20.9 kb CTCF site', 'E_01_021\\t26656844\\tMYC-ECSE (MYC endometrial carcinoma super-Enhancer)\\thg19\\tchr8\\t129543949\\t129554294\\tHuman\\tEndometrial Carcinoma Cell Lines Ishikawa\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,3C-qPCR,CRISPR/Cas9,siRNA\"\\t\"The endometrial carcinoma peak (chr. 8: 129,543,949C129,554,294) encompasses a 10-kb noncoding region that harbors a super-enhancer,as defined by the H3K27ac ChIP-seq profile of the endometrial carcinoma cell line Ishikawa, which we refer to as MYC-ECSE(MYC endometrial carcinoma super-enhancer).\"Super-Enhancer\\tMYC\\t3C\\t--\\t\"We performed chromosome conformation capture (3C) assays in A549 and Ishikawa cells and found that MYC-LASE physically interacted with the MYC promoter only in A549 cells and, reciprocally, that MYC-ECSE physically interacted with the MYC promoter only in Ishikawa cells\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\tEndometrial Carcinoma\\tDOID:2871\\tD016889\\tTwo distinct focal amplifications of super-enhancers 3 to MYC in lung adenocarcinoma (MYC-LASE) and endometrial carcinoma (MYC-ECSE) are physically associated with the MYC promoter and correlate with MYC overexpression.\\t\"3C,ChIP-seq\"\\t\"We performed chromosome conformation capture (3C) assays in A549 and Ishikawa cells and found that MYC-LASE physically interacted with the MYC promoter only in A549 cells and, reciprocally,that MYC-ECSE physically interacted with the MYC promoter only in Ishikawa cells. In addition, tumors with amplification of MYC alone or MYC-LASE or MYC-ECSE alone had higher MYC expression than tumors lacking either amplification. These results suggest that both MYC-LASE and MYC-ECSE drive MYC expression through lineage-specific chromatin loops.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t800808', 'E_01_022\\t29045844\\t--\\thg19\\tchr8\\t98253530\\t98257713\\tHuman\\tMantle Cell Lymphoma (MCL)\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR,4C-seq\"\\t\"These candidate enhancers (E1 and E2, Figure 2A) are the most highly acetylated elements', 'E_01_023\\t29045844\\t--\\thg19\\tchr8\\t98310330\\t98314427\\tHuman\\tMantle Cell Lymphoma (MCL)\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR,4C-seq\"\\t\"These candidate enhancers (E1', 'E_01_024\\t26751080\\t--\\thg19\\tchr17\\t26077792\\t26078792\\tHuman\\tA549\\tLow throughput\\t3C\\t\"Lastly, our 3C assay was able to detect a long DNA loop between the hiNOS enhancer and core promoter site, and ChIP loop assay confirmed that p300 binds to AP--1 and RNA pol II proteins. \"\\tEnhancer\\tNOS2\\t3C\\t--\\t\"Lastly, our 3C assay was able to detect a long DNA loop between the hiNOS enhancer and core promoter site.\"\\t\"HEP-NOS,INOS,NOS,NOS2A\"\\t--\\t--\\t--\\tThe coactivator p300 mediates cytokine-induced hiNOS transactivation by forming a distant DNA loop between its enhancer and core promoter region.\\tLuciferase Reporter Assay\\tThese findings indicate that the cytokine-induced Enhancer region at -5 kb to -6 kb is required for p300-mediated increase in CM-induced hiNOS promoter activity.\\tJUND\\tAP-1\\tChIP\\tThese findings are consistent with p300 mediated hiNOS transactivation at the hiNOS Enhancer mainly through the upstream AP-1 site at -5.3 kb by forming a DNA-Protein complex.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5499', 'E_01_025\\t19386363\\tCD200 distal Enhancer\\thg19\\tchr3\\t112045517\\t112046117\\tHuman\\tHUVEC\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,PCR,DNase I Hypersensitivity Assay\"\\t\"Using a DNase I hypersensitivity assay, we identified a distal enhancer region responsible for inducible expression', 'E_01_026\\t24413736\\t--\\thg19\\tchr6\\t37772587\\t37778315\\tHuman\\t\"HEK-293,-cell\"\\tLow+High throughput\\t\"RNA-seq,ChIP-seq,ChIP,EMSA,PCR,4C,Luciferase Reporter Assay\"\\t\"To define enhancer clusters, we first created 1,000 iterations of randomized C3 sites in the', 'E_01_027\\t18202151\\t hRLD2 Enhancer\\thg19\\tchr13\\t43111372\\t43112072\\tHuman\\tMG-63\\tLow throughput\\t\"EMSA,ChIP,RT-PCR\"\\t\"Previous work in our laboratory using', 'E_01_028\\t19467240\\t--\\thg19\\tchr17\\t26072592\\t26074592\\tHuman\\tHep G2\\tLow throughput\\t\"Luciferase Reporter Assay,PCR,EMSA,ChIP\"\\t\"Taken together, this work identifies a far-upstream functional Oct-1 enhancer motif at -10.2 kb in the hiNOS promoter that regulates cytokine-induced hiNOS gene transcription, and further underscores the tight control mechanisms regulating expression of the human iNOS gene.\"\\tEnhancer\\tNOS2\\t--\\t\"EMSA,ChIP\"\\t\"Taken together, this work identifies a far-upstream functional Oct-1 enhancer motif at -10.2 kb in the hiNOS promoter that regulates cytokine-induced hiNOS gene transcription, and further underscores the tight control mechanisms regulating expression of the human iNOS gene.\"\\t\"HEP-NOS,INOS,NOS,NOS2A\"\\t--\\t--\\t--\\t--\\t--\\t--\\tPOU2F1\\t\"OCT1,OTF1,oct-1B\"\\t\"EMSA,ChIP\"\\t\"To examine whether the Oct-1 protein can interact with the potential Oct-1 binding site at ?10.2 kb of hiNOS in vitro, we carried out electrophoretic mobility shift assays (EMSAs) with nuclear extracts from HepG2 cells.Chromatin immunoprecipitation assay confirms that Oct-1 specifically binds to the motif at ?10.2 kb in the hiNOS promoter in vivo.\"\\t--\\t--\\t--\\t>2KB\\t\\t10199', 'E_01_029\\t19708858\\tHREs Enhancer\\thg19\\tchr3\\t48598405\\t48598558\\tHuman\\tTE-671\\tLow throughput\\t\"Luciferase Reporter Assay,RT-PCR\"\\t\"This activation was conserved in constructs with the 3\\' FLR fragment placed upstream of the luciferase gene, indicating an enhancer function for HRE1.To test their role as enhancers, we cloned several fragments of the 3 \\'FLR of the hUcn2 gene downstream of the firefly luciferase gene in (a) the pGL3 ?746 bp vector containing the functional hUcn2 promoter, and (b) the pGL3control vector containing an internal SV40 promoter (Figure 2A).', 'E_01_030\\t19773398\\tSF-1 Enhancer\\thg19\\tchrx\\t30317939\\t30318739\\tHuman\\tHEK-293\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\tLuciferase reporter gene activity in transiently transfected HEK293 cells. Relative luciferase activity refers to the activity of the firefly reporter gene normalized to Renilla luciferase internal control for transfection efficiency. Firefly reporter constructs contained the DAX1 conserved noncoding enhancer elements upstream of a minimal thymidine kinase promoter driving the firefly reporter.\\tEnhancer\\tNR0B1\\t--\\t\"Luciferase Reporter Assay,PCR\"\\tOur results therefore strongly suggest the existence of an SF-1 binding site 4kb upstream of NR0B1 that acts as an enhancer of DAX1 expression. \\t\"AHC,AHCH,AHX,DAX-1,DAX1,DSS,GTD,HHG,NROB1,SRXY2\"\\tX-Linked Congenital Adrenal Hypoplasia\\tDOID:0080156\\tD000075262\\tA reporter construct lacking this Enhancer element upstream of NR0B1 was unresponsive to SF-1 transcriptional activation.\\tLuciferase Reporter Assay\\t\"SF-1 acted as an activator on the wild-type construct, whereas the mutant construct,lacking the SF-1 consensus binding site, showed no response to SF-1.', 'E_01_031\\t20061391\\tMnSOD Enhancer\\thg19\\tchr6\\t160101620\\t160102231\\tHuman\\tPC-3\\tLow throughput\\t\"RT-PCR,Luciferase Reporter Assay,EMSA,ChIP\"\\t\"For reporter gene activity, a luciferase assay was performed by co-transfecting the cells with the p53 expression vector and vector containing the enhancer (I2E) and the promoter (?3400/+24, ?555/+24, or ?210/+24) of the human MnSOD gene in the pGL3 reporter vector.\"\\tEnhancer\\tSOD2\\t--\\t\"Luciferase Reporter Assay,EMSA,PCR,ChIP\"\\tThese results suggest that the positive effect of p53 on MnSOD expression is dependent on the presence of the intronic enhancer element of the MnSOD gene.FIGURE 3\\t\"IPO-B,IPOB,MNSOD,MVCD6,Mn-SOD\"\\t--\\t--\\t--\\tThe positive effect of p53 on MnSOD expression is dependent on the presence of the intronic Enhancer element of the MnSOD gene\\tLuciferase Reporter Assay\\tAddition of the Enhancer element to the MnSOD promoter construct led to the bi-directional effect of p53 as observed with the endogenous MnSOD gene.These results suggest that the positive effect of p53 on MnSOD expression is dependent on the presence of the intronic Enhancer element of the MnSOD gene.\\tTP53\\t\"BCC7,BMFS5,LFS1,P53,TRP53\"\\t\"PCR,EMSA,ChIP\"\\t\"The promoter fragment amplified after p53 overexpression was decreased when the ChIP was preformed with the Sp1 antibody.These results are consistent with the DNA-binding data obtained by EMSA, suggesting that p65 binding to the MnSOD enhancer may contribute to the induction of MnSOD gene transcription.\"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t1831', 'E_01_032\\t20061391\\tMnSOD Enhancer\\thg19\\tchr6\\t160101890\\t160102231\\tHuman\\tPC-3\\tLow throughput\\t\"Luciferase Reporter Assay,EMSA,PCR,ChIP\"\\t\"For reporter gene activity, a luciferase assay was performed by co-transfecting the cells with the p53 expression vector and vector containing the enhancer (I2E) and the promoter (?3400/+24, ?555/+24, or ?210/+24) of the human MnSOD gene in the pGL3 reporter vector.\"\\tEnhancer\\tSOD2\\t--\\t\"Luciferase Reporter Assay,EMSA,PCR,ChIP\"\\tThese results suggest that the positive effect of p53 on MnSOD expression is dependent on the presence of the intronic enhancer element of the MnSOD gene.FIGURE 3\\t\"IPO-B,IPOB,MNSOD,MVCD6,Mn-SOD\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSP1\\tSP1\\t\"PCR,EMSA,ChIP\"\\t\"The promoter fragment amplified after p53 overexpression was decreased when the ChIP was preformed with the Sp1 antibody.These results are consistent with the DNA-binding data obtained by EMSA, suggesting that p65 binding to the MnSOD enhancer may contribute to the induction of MnSOD gene transcription.\"\\t--\\t--\\t--\\tIntron\\t1966', 'E_01_033\\t19285986\\tDi\\thg19\\tchr16\\t67519201\\t67519833\\tHuman\\t\"H295R,N38\"\\tLow throughput\\t\"Transgenic mice,Luciferase Reporter Assay\"\\t\"Schematic representation of the genomic region encompassing the AgRP and the ATPase locus. Expression constructs for regions A, B, C, and D were directionally cloned into pGL3-basic luciferase vector.Regions C and D were evaluated in the same fashion. Region D (but not region C) contained', 'E_01_034\\t19285986\\tDic\\thg19\\tchr16\\t67519609\\t67519833\\tHuman\\t\"H295R,N38\"\\tLow throughput\\t\"Transgenic mice,Luciferase Reporter Assay\"\\t\"Di and Diii were further engineered into three overlapping sub-regions each and the', 'E_01_035\\t19285986\\tDiii\\thg19\\tchr16\\t67520417\\t67521049\\tHuman\\t\"H295R,N38\"\\tLow throughput\\t\"Transgenic mice,Luciferase Reporter Assay\"\\t\"Schematic representation of the genomic region encompassing the AgRP and the ATPase locus. Expression constructs for regions A, B, C, and D were directionally cloned into pGL3-basic luciferase vector.Regions C and D were evaluated in the same fashion. Region D (but not region C) contained enhancer elements in both neuronal and somatic cell types (Fig. 1a). Region D was therefore', 'E_01_036\\t19285986\\tDiiia\\thg19\\tchr16\\t67520417\\t67520638\\tHuman\\t\"AtT-20,N38\"\\tLow throughput\\t\"Transgenic mice,Luciferase Reporter Assay,Transfection\"\\t\"Di and Diii were further engineered into three overlapping sub-regions each and the', 'E_01_037\\t19285986\\tDiiib\\thg19\\tchr16\\t67520620\\t67520851\\tHuman\\tH295R\\tLow throughput\\t\"Luciferase Reporter Assay,Transfection\"\\tDi and Diii were further engineered into three overlapping sub-regions each and the experiments were performed in the N38 and h295R mouse hypothalamic cells. \\tEnhancer\\tAgRP-ATP6V0D2\\t--\\t\"Luciferase Reporter Assay,Transgenic mice\"\\tSchematic of transgenes and bioluminescence of offspring representing the two enhancers. (Fig. 3)\\t\"AGRT,ART,ASIP2,ATP6D2,VMA6\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t4262', 'E_01_038\\t20093418\\tFKBP51 Enhancer\\thg19\\tchr6\\t35574362\\t35576362\\tHuman\\tA549\\tLow throughput\\t\"RT-qPCR,ChIP\"\\t\"According to our quantitative ChIP and enhancer assays, there are at least three major distal gene regions that significantly bind GR', 'E_01_039\\t20093418\\tFKBP51 Enhancer\\thg19\\tchr6\\t35505138\\t35511492\\tHuman\\t\"A549,VCaP\"\\tLow throughput\\t\"RT-qPCR,ChIP\"\\t\"According to our quantitative ChIP and enhancer assays, there are at least three major distal gene regions that significantly bind GR', 'E_01_040\\t20093418\\tFKBP51 Enhancer\\thg19\\tchr6\\t35616283\\t35622638\\tHuman\\t\"A549,VCaP\"\\tLow throughput\\t\"RT-qPCR,ChIP\"\\t\"According to our quantitative ChIP and enhancer assays, there are at least three major distal gene regions that significantly bind GR', 'E_01_041\\t20093418\\tFKBP51 Enhancer\\thg19\\tchr6\\t35636622\\t35642872\\tHuman\\t\"A549,VCaP\"\\tLow throughput\\t\"RT-qPCR,ChIP\"\\t\"According to our quantitative ChIP and enhancer assays, there are at least three major distal gene regions that significantly bind GR', 'E_01_042\\t20952403\\tRET Enhancer\\thg19\\tchr10\\t43569296\\t43572684\\tHuman\\tSK-N-BE\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\tWe then performed chromatin immunoprecipitation (ChIP) experiments to verify the presence of RAR\\tEnhancer\\tRET\\t--\\t\"ChIP,qRT-PCR\"\\t\"In order to study possible chromatin changes occurring during RET gene activation, we determined by a series of ChIP assays coupled with qRTCPCR, the histone modification state at the RET regulatory regions after different times of RA treatment.\"\\t\"CDHF12,CDHR16,HSCR1,MEN2A,MEN2B,MTC1,PTC,RET-ELE1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"EZH2,DNMT1\"\\t\"ENX-1,ENX1b,KMT6,KMT6A,WVS,WVS2,EZH2,ADCADN,AIM,CXXC9,DNMT,HSN1E,MCMT,m.HsaI\"\\t\"ChIP,RT-PCR\"\\tEZH2 and DNMT1 binding to RET regulatory regions.SK-N-BE cells were treated for the indicated times with 10 mM RA.ChIP experiments were performed using anti-EZH2 and anti-DNMT1 antibodies and bound DNA was quantitated by real time PCR.\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1526', 'E_01_043\\t20952659\\tCCL2 Enhancer\\thg19\\tchr17\\t32579684\\t32579693\\tHuman\\t\"HEK-293,HEK-MOR Cells\"\\tLow throughput\\t\"ChIP,PCR\"\\tChIP analysis shows that DAMGO administration induces binding of p65 to the enhancer region of the CCL2 promoter.\\tEnhancerCCL2\\t--\\t\"ChIP,PCR\"\\t ChIP analysis shows that DAMGO administration induces binding of p65 to the enhancer region of the CCL2 promoter. \\t\"GDCF-2,HC11,HSMCR30,MCAF,MCP-1,MCP1,SCYA2,SMC-CF\"\\t--\\t--\\t--\\t--\\t--\\t--\\tNFKB1\\t\"CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP,PCR\"\\t\"Based on these studies and our pre_x0002_liminary experiments, we designed PCR primers to span both of the NF-KB-binding elements in the CCL2 Enhancer region for PCR analysis following ChIP.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2606', 'E_01_044\\t21051589\\tandrogen Enhancer\\thg19\\tchr19\\t50850114\\t50850982\\tHuman\\tLNCaP\\tLow throughput\\t\"Luciferase Reporter Assay,RT-PCR\"\\t\"The luciferase activity of the pPSABHE reporter vector, which contains the promoter (2589 to 241) and androgen enhancer regions (24801 to 23933) of the PSA gene, increased by 8--fold after IL--6 treatment. \"\\tEnhancer\\tKLK3\\t--\\tLuciferase Reporter Assay\\t\"The luciferase activity of the pPSABHE reporter vector, which contains the promoter (2589 to 241) and androgen enhancer regions (24801 to 23933) of the PSA gene, increased by 8--fold after IL--6 treatment.\"\\t\"APS,KLK2A1,PSA,hK3\"\\tProstate Cancer\\tDOID:10283\\tD011471\\t\"Results from 5\\'-deletion reporter assays revealed that the effects of IL-6 appear to be mediated via an androgen Enhancer region (24801 to 23933), which is dependent on the signal transducer and activator of the transcription 3 (STAT3) pathway.\"\\tLuciferase Reporter Assay\\t\"Moreover, we also found that IL-6 treatment did not further enhance the stimulatory effect of transient overexpression of STAT3 on luciferase activity of the pPSABHE reporter vector, suggesting that the STAT3 signaling pathway, which is dependent on the Enhancer region of PSA gene, is the major pathway involved in the IL-6Cregulated expression of the PSA gene in LNCaP cells.\"\\tSTAT3\\t\"ADMIO,ADMIO1,APRF,HIES\"\\tLuciferase Reporter Assay\\tTransient gene expression assays of cells cotransfected with the pPSABHE reporter vector and the STAT3 expression vector revealed that transient overexpression of STAT3 significantly upregulated the luciferase activity of the pPSABHE reporter vector in an androgen-independent manner.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t507622', 'E_01_045\\t21056086\\t--\\thg19\\tchr1\\t23881789\\t23881796\\tHuman\\t\"Immortalized Murine Gonadotrope LT2 Cell,Ovarian Cancer Cell\"\\tLow throughput\\t\"qRT-PCR,Luciferase Reporter Assay,ChIP\"\\t\"To determine if these base pairs also play a role in BMP2induction of the human ID3 promoter, we introduced mutations comparable to Mut4 and Mut5 in the murine Id3 promoter into the minimal hID3 promoter--reporter construct (?653/+402 ID3--luc) (Fig. 6(B)). \"\\tEnhancer\\tID3\\t--\\tChIP\\tWe identified the same sequence in the murine gene (?3283/?3276) (Fig. S8) and observed through ChIP analysis that BMP2 stimulated recruitment of SMAD1 to this region of the endogenous Id3 gene in L?T2 cells\\t\"HEIR-1,bHLHb25\"\\t--\\t--\\t--\\tA more distal Enhancer was shown to mediate BMP4-induction of the human ID3 gene in ovarian cancer cells.\\tChIP\\t\"We identified the same sequence in the murine gene (?3283/?3276) and observed through ChIP analysis that BMP2 stimulated recruitment of SMAD1 to this region of the endogenous Id3 gene in LT2 cells. Mutation of the distal BRE in the longer murine Id3 reporter greatly reduced, but did not abolish, BMP2 induction.\"\\tBMP2\\t\"BDA2A, SSFSC, BMP2\"\\tChIP\\t\"We identified the same sequence in the murine gene (?3283/?3276) (Fig. S8) and observed through ChIP analysis that BMP2 stimulated recruitment of SMAD1 to this region of the endogenous Id3 gene in LT2 cells (Fig. S7A). Muta_x0002_tion of the distal BRE in the longer murine Id3 reporter greatly reduced, but did not abolish, BMP2 induction.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2627', 'E_01_046\\t21087445\\t--\\thg19\\tchr22\\t24236392\\t24237411\\tHuman\\t\"CEMC7A,THP-1,293T,SW982\"\\tLow throughput\\t\"Luciferase Reporter Assay,PCR,EMSA,DNaseI-seq\"\\t\" DNase I hypersensitivity was used to identify potential hypersensitive sites (HS) across the MIF gene locus. Reporter gene assays were performed in different human cell lines with constructs containing the native or mutated HS element. Following phylogenetic and transcription factor binding profiling, electrophoretic mobility shift assay (EMSA) and RNA interference were performed and the effects of incubation with mithramycin, an antibiotic that binds GC boxes, were also studied. An HS centred on the first intron of MIF was identified. The HS acted as an enhancer in human T lymphoblasts (CEMC7A), human embryonic kidney cells (HEK293T) and human monocytic cells (THP-1), but not in a fibroblast-like synoviocyte (FLS) cell line (SW982) or cultured FLS derived from rheumatoid arthritis (RA) patients.\"\\tEnhancerMIF\\t--\\t\"Luciferase Reporter Assay,PCR,EMSA,DNaseI-seq\"\\t\" DNase I hypersensitivity was used to identify potential hypersensitive sites (HS) across the MIF gene locus. Reporter gene assays were performed in different human cell lines with constructs containing the native or mutated HS element. Following phylogenetic and transcription factor binding profiling, electrophoretic mobility shift assay (EMSA) and RNA interference were performed and the effects of incubation with mithramycin, an antibiotic that binds GC boxes, were also studied. An HS centred on the first intron of MIF was identified. The HS acted as an enhancer in human T lymphoblasts (CEMC7A), human embryonic kidney cells (HEK293T) and human monocytic cells (THP-1), but not in a fibroblast-like synoviocyte (FLS) cell line (SW982) or cultured FLS derived from rheumatoid arthritis (RA) patients.\"\\t\"GIF,GLIF,MMIF\"\\tRheumatoid Arthritis\\tDOID:7148\\tD001172\\t--\\t--\\t--\\tSP1\\tSP1\\t\"Luciferase Reporter Assay,EMSA\"\"In order to determine whether Sp1 is recruited to the MIF intron 1 Enhancer, EMSA was performed using CEMC7A nuclear extract and a radiolabelled DNA probe of the 5region of the intronic sequence.We were able to demonstrate that Sp1 bound this sequence, as competition with excess cold Sp1 consensus sequence and addition of Sp1-specific antibody resulted in the abrogation of a shifted complex.\"\\t--\\t--\\t--\\t<2KB\\t338', 'E_01_047\\t21310710\\t--\\thg19\\tchr18\\t60786011\\t60788011\\tHuman\\tJURKAT\\tLow throughput\\t\"3C,ChIP,RT-PCR\"\\t\"With chromosome conformation capture (3C) and chromatin immunoprecipitation (ChIP) assays we demonstrated that the mbr could physically interact with BCL2 promoter through SATB1-mediated chromatin looping, which was required for epigenetic modifications of the promoter, CREB accessibility and high expression of the BCL2 gene.\"\\tEnhancer\\tBCL2\\t3C\\t\"ChIP,RT-PCR\"\\t\"With chromosome conformation capture (3C) and chromatin immunoprecipitation (ChIP) assays we demonstrated that the mbr could physically interact with BCL2 promoter through SATB1-mediated chromatin looping, which was required for epigenetic modifications of the promoter, CREB accessibility and high expression of the BCL2 gene.\"\\t\"Bcl-2,PPP1R50\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"SATB1,CEBPB,EP300\"\\t\"SATB1,C/EBP-beta,IL6DBP,NF-IL6,TCF5,KAT3B,MKHK2,RSTS2,p300\"\\t\"Western blot,3C,RT-PCR\"\\t\"Our data showed that a reduction of the interaction between SBS1 and mbr induced by knockdown of SATB1 was significantly correlated with a decrease in the BCL2 mRNA level , suggesting that SATB1-mediated mbr-promoter interaction was required for transcriptional activity of the gene.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3567', 'E_01_048\\t21327791\\tLCT Enhancer\\thg19\\tchr2\\t136531405\\t136531505\\tHuman\\tCaco-2\\tLow throughput\\t\"Transfection,Luciferase Reporter Assay\"\\t\"The LCT enhancer activity has previously been characterized by transfection experiments using a 450 bp fragment. These fragments were cloned at a', 'E_01_049\\t21348942\\t--\\thg19\\tchr12\\t100693355\\t100695355\\tHuman\\tMCF-7\\tLow throughput\\t\"ChIP,Western blot,siRNA,qPCR\"\\tThe reaction was performed at a combined annealing/extension temperature of 60?C for 40 cycles using primers designed to specifically amplify the sequence containing the +571 ERE half\\\\site and two regions covering ER\\\\binding sites named enhancer?1 and enhancer?2 located at ?168 and ?206?kb in the PR regulatory regions.\\tEnhancer\\tRB1\\t--\\tChIP\\t ChIP analysis of PR regulatory regions in MCF--7 cells.Distal ER--binding sites and a proximal half--ERE located 571 bp downstream of the PRB tsp are highlighted with black diamonds.\\t\"OSRC,PPP1R130,RB,p105-Rb,pRb,pp110\"\\t--\\t--\\t--\\t--\\t--\\t--\\tKDM5A\\t\"RBBP-2,RBBP2,RBP2\"\\tWestern blot\\tOverexpression of the wild-type and mutated JARID1A forms after transfection of the corresponding plasmids in MCF-7 cells was confirmed at the protein level by western blot analysis.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t51816473', 'E_01_050\\t21348942\\t--\\thg19\\tchr12\\t100731355\\t100733355\\tHuman\\tMCF-7\\tLow throughput\\t\"ChIP,Western blot,siRNA,qPCR\"\\tThe reaction was performed at a combined annealing/extension temperature of 60?C for 40 cycles using primers designed to specifically amplify the sequence containing the +571 ERE half\\\\site and two regions covering ER\\\\binding sites named enhancer?1 and enhancer?2 located at ?168 and ?206?kb in the PR regulatory regions.\\tEnhancer\\tRB1\\t--\\tChIP\\t ChIP analysis of PR regulatory regions in MCF--7 cells.Distal ER--binding sites and a proximal half--ERE located 571 bp downstream of the PRB tsp are highlighted with black diamonds.\\t\"OSRC,PPP1R130,RB,p105-Rb,pRb,pp110\"\\t--\\t--\\t--\\t--\\t--\\t--\\tKDM5A\\t\"RBBP-2,RBBP2,RBP2\"\\tWestern blot\\tOverexpression of the wild-type and mutated JARID1A forms after transfection of the corresponding plasmids in MCF-7 cells was confirmed at the protein level by western blot analysis.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t51854473', 'E_01_051\\t21355081\\t--\\thg19\\tchr10\\t131265598\\t131265656\\tHuman\\t\"Human Bronchial Epithelial Cell,Lung cancer-derived Cell Lines,Glioblastoma Cell Lines\"\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\tThis study provides strong evidence that the A allele of a MGMT promoter-enhancer SNP (rs16906252) is a key determinant in the acquisition of MGMT methylation in lung carcinogenesis.\\tEnhancer\\tMGMT\\t--\\t\"Luciferase Reporter Assay,PCR\"\\tThis study provides strong evidence that the A allele of a MGMT promoter-enhancer SNP (rs16906252) is a key determinant in the acquisition of MGMT methylation in lung carcinogenesis.\\tMGMT\\tLung Adenocarcinoma\\tDOID:3910\\tD000077192\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\trs16906252\\t131265545\\t\"Luciferase Reporter Assay,PCR\"\\t<2KB\\t\\t180', 'E_01_052\\t21378152\\tWT1 Enhancer\\thg19\\tchr11\\t32403489\\t32405489\\tHuman\\tHuman Podocyte Cell Line AB8/13\\tLow throughput\\t\"qRT-PCR,Luciferase Reporter Assay\"\\tWe tested the effect of TGF-beta1 on a human WT1 enhancer located ?4.3 kb upstream of the transcription start site using a reporter vector WTA.\\tEnhancer\\tWT1\\t--\\t\"qRT-PCR,Luciferase Reporter Assay\"\\t\" TGF-beta1 did not alter luciferase activity of the reporter construct for a human WT1 promoter but reduced that for a human WT1 5 enhancer construct, suggesting that TGF-beta1 may regulate WT1 expression by altering the 5 enhancer activity.\"\\t\"AWT1,GUD,NPHS4,WAGR,WIT-2,WT33\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSMAD4\\t\"DPC4,JIP,MADH4,MYHRS\"\\tLuciferase Reporter Assay\\t\"The luciferase activity of p3TP-lux, a reporter construct for Smadbinding element, was determined after incubation with 5 ng/mL TGFbeta1 for 24 h. TGF-beta1 increased the luciferase activity by ~12-fold in the presence of the NTC shRNA, while this increase was largely prevented by Sh-Smad4. The luciferase activity of p3TP-lux was normalized to beta-galactosidase activity of pCMV-beta-gal and shown as relative values to that of pGL2 basic.\"\\t--\\t--\\t--\\t>2KB\\t\\t4831', 'E_01_053\\t21385855\\t--\\thg19\\tchr6\\t135459326\\t135461326\\tHuman\\tK-562\\tLow throughput\\t\"Luciferase Reporter Assay,PCR,ChIP\"\\t\" Based on transient expression of a luciferase reporter plasmid, the DNA fragment encompassing the 3-bp deletion polymorphism has enhancer-like activity that is further augmented by the introduction of the 3-bp deletion.\"\\tEnhancer\\tHBG2\\t--\\t\"Luciferase Reporter Assay,PCR,ChIP\"\\t\"In an initial attempt to demonstrate functionality of the DNA fragment surrounding the 3-bp deletion polymorphism, DNA fragment either without or with the 3-bp deletion (Figure 3A) was ligated to an expression vector consisting of 1.4 kb HBG2 proximal promoter and luciferase reporter gene. \"\\t\"HBG-T1,TNCY\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"RUNX1,GATA\"\\t\"AML1,AML1-EVI-1,AMLCR1,CBF2alpha,CBFA2,EVI-1,PEBP2aB,PEBP2alpha,GATA\"\\tLuciferase Reporter Assay\\t\"Effect of mutation of transcription factor binding sites in the 58-bp DNA fragment with the 3-bp deletion on its enhancer-like activity.The luciferase activity of the plasmid with the 58-bp DNA fragment was normalized to be 1.0. Mutation of the E-box, RUNX1, or 3_x0004_ GATA binding site down-regulates the enhancer-like activity to approximately 50%. Mutation of the 5_x0004_ GATA binding site does not decrease the enhancer-like activity of the DNA fragment. \"\\trs9399137\\t135419018\\t\"Luciferase Reporter Assay,PCR,ChIP\"\\t>2KB\\t\\t\\t\\t130185906', 'E_01_054\\t21402921\\t--\\thg19\\tchr12\\t8217947\\t8219702\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"\\t\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\tEnhancer\\tC3AR1\\t--\\t\"ChIP-chip,RT-qPCR,ChIP-qPCR\"\\t\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\t\"AZ3B,C3AR,HNFAG09\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\\t\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p74 occupancy and active chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t9386', 'E_01_055\\t21402921\\t--\\thg19\\tchr12\\t54757502\\t54758950\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\tEnhancer\\tGPR84\\t--\\t\"ChIP-chip,RT-qPCR,ChIP-qPCR\"\\t\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\t\"EX33,GPCR4\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\\t\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1998', 'E_01_056\\t21402921\\t--\\thg19\\tchr17\\t34411749\\t34415000\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\tEnhancer\\tCCL3\\t--\\t\"ChIP-chip,RT-qPCR,ChIP-qPCR\"\\t\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\t\"G0S19-1,LD78ALPHA,MIP-1-alpha,MIP1A,SCYA3\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\\t\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2226', 'E_01_057\\t21402921\\t--\\thg19\\tchr17\\t34416207\\t34419359\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\tEnhancer\\tCCL3\\t--\\t\"ChIP-chip,RT-qPCR,ChIP-qPCR\"\\t\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\t\"G0S19-1,LD78ALPHA,MIP-1-alpha,MIP1A,SCYA3\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\\t\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2182', 'E_01_058\\t21402921\\t--\\thg19\\tchr17\\t34429460\\t34432613\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\tEnhancer\\tCCL3\\t--\\t\"ChIP-chip,RT-qPCR,ChIP-qPCR\"\\t\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\t\"G0S19-1,LD78ALPHA,MIP-1-alpha,MIP1A,SCYA3\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\\t\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t15436', 'E_01_059\\t21402921\\t--\\thg19\\tchr2\\t113595983\\t113598617\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"Premarked enhancers and promoters predict cell-specific p65 binding.(Fig.2)\\tEnhancer\\tIL1B\\t--\\t\"ChIP-qPCR,ChIP\"\\tPremarked enhancers and promoters predict cell-specific p65 binding.(Fig.2)\\t\"IL-1,IL1-BETA,IL1F2,IL1beta\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t9964', 'E_01_060\\t21402921\\t--\\thg19\\tchr2\\t113597789\\t113600339\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"Premarked enhancers and promoters predict cell-specific p65 binding.(Fig.2)\\tEnhancer\\tIL1B\\t--\\t\"ChIP-qPCR,ChIP\"\\tPremarked enhancers and promoters predict cell-specific p65 binding.(Fig.2)\\t\"IL-1,IL1-BETA,IL1F2,IL1beta\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t11728', 'E_01_061\\t21402921\\t--\\thg19\\tchr5\\t158755252\\t158761634\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\tEnhancer\\tIL12B\\t--\\t\"ChIP-chip,RT-qPCR,ChIP-qPCR\"\\t\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\t\"CLMF,CLMF2,IL-12B,IMD28,IMD29,NKSF,NKSF2\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\\t\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t16653', 'E_01_062\\t21402921\\t--\\thg19\\tchr5\\t158783552\\t158790360\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\tEnhancer\\tIL12B\\t--\\t\"ChIP-chip,RT-qPCR,ChIP-qPCR\"\\t\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\t\"CLMF,CLMF2,IL-12B,IMD28,IMD29,NKSF,NKSF2\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\\t\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t45166', 'E_01_063\\t21402921\\t--\\thg19\\tchr5\\t158787331\\t158794352\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\tEnhancer\\tIL12B\\t--\\t\"ChIP-chip,RT-qPCR,ChIP-qPCR\"\\t\" To test this hypothesis, we chose the top 10 genes specifically induced by TNF- in THP-1 cells, but not HeLa cells, from microarray data (Fig. 1B), and verified five of them (CCL3, CCL4, IL12B, C3AR1, and GPR84) had PU.1 peaks as well as THP-1Cspecific p65 binding sites within 30 kb of their promoters (Fig. 4A).\"\\t\"CLMF,CLMF2,IL-12B,IMD28,IMD29,NKSF,NKSF2\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\\t\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t49052', 'E_01_064\\t21402921\\t--\\thg19\\tchr1\\t203273730\\t203275395\\tHuman\\t\"HeLa,THP-1\"\\tLow throughput\\t\"ChIP-qPCR,ChIP\"Premarked enhancers and promoters predict cell-specific p65 binding.(Fig.2)\\tEnhancer\\tBTG2\\t--\\t\"ChIP-qPCR,ChIP\"\\tPremarked enhancers and promoters predict cell-specific p65 binding.(Fig.2)\\t\"APRO1,PC3,TIS21\"\\t--\\t--\\t--\\t\"The diversity of transcriptional programs in mammalian cells arises, at least in part,is from preexisting enhancers that are established by cell-specific transcription factors.\"\\tChIP\\tThese results strongly support the model that combinatorial binding of THP-1Cspecific TFs PU.1 and C/EBP synergistically create a distinct group of enhancers and mediate selection of a subset of monocyte/macrophage-specific NF-B target genes for specific induction by allowing the binding of p65 to these locations for productive gene expression.\\t\"SPI1,CEBPB,NFKB1\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A,C/EBP-beta,IL6DBP,NF-IL6,TCF5,CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\t\"ChIP-qPCR,ChIP\"\\t\"Strikingly, we found that all these genes gained responsiveness to TNF- stimulation only when both PU.1 and C/EBP were coexpressed in HeLa cells (Fig. 4B). To test the chromatin events near these genes, we further identified eight THP-1Cspecific p65 locations (P1CP8; Fig. 4A) around these five genes and performed ChIP/quantitative PCR (qPCR) to detect p75 occupancy and active chromatin.\"\\t--\\t--\\t--\\t<2KB\\t\\t100', 'E_01_065\\t21419113\\t--\\thg19\\tchr1\\t226065482\\t226067482\\tHuman\\tPerinodal Cells\\tLow throughput\\t\"PCR,Transgenic mice,Immunostaining\"\\tWe have now identified a transcriptional enhancer (ANE) in the human LEFTY1 gene that exhibits marked L>R asymmetric activity in perinodal cells of the mouse embryos.the enhancer activity of each fragment was examined in transgenic mouse embryos.\\tEnhancer\\tLEFTY1\\t--\\t\"PCR,Transgenic mice,Immunostaining\"\\tWe have now identified a transcriptional enhancer (ANE) in the human LEFTY1 gene that exhibits marked L>R asymmetric activity in perinodal cells of the mouse embryos.\\t\"LEFTB,LEFTYB\"\\t--\\t--\\t--\\t--\\t--\\t--\\tFOXH1\\t\"FAST-1,FAST1\"\\tX-Gal Staining Assay \\t\"The approximate location of FoxH1 binding sites is indicated by red ovals. Restriction sites: E, EcoRI; Bam, BamHI; BgI, BglI; BX, BstXI; BgII, BglII; Sma, SmaI; Xb, XbaI; Sac, SacI. (BCE) Ventral views of X-galCstained embryos harboring the indicated lacZ transgenes. (FCI) Magnified views of the node region of X-galCstained embryos harboring hNPE7.5-lacZ.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t7499', 'E_01_066\\t21424707\\t--\\thg19\\tchr1\\t204123719\\t204123725\\tHuman\\tCalu-6\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"To test the functional relevance of enhCRE, we fused it to RENMin (construct enhCRE--RENMin). Both cAMP agonists increased the transcriptional activity of enhCRE--RENMin.construct enhCRE--RENMin contains human renin enhancer CRE (enhCRE) fused to RENMin. RLA, relative luciferase activity\"\\tEnhancer\\tREN\\t--\\tLuciferase Reporter Assay\\t\"To test the functional relevance of enhCRE, we fused it to RENMin (construct enhCRE--RENMin). Both cAMP agonists increased the transcriptional activity of enhCRE--RENMin.construct enhCRE--RENMin contains human renin enhancer CRE (enhCRE) fused to RENMin. RLA, relative luciferase activity\"\\tHNFJ2\\t--\\t--\\t--\\t--\\t--\\t--\\tCAMP\\t\"CAP-18,CAP18,CRAMP,FALL-39,FALL39,HSD26,LL37\"\\t\"Luciferase Reporter Assay,ChIP,qRT-PCR\"\\t\"Construct RENMin represents the minimal human renin promoter (-199 to +23) which contains the cAMP target sequence CNRE at ?135 to ?107. The PPAR agonist rosiglitazone (Rosi, 500 nM for 20 h) was applied as positive control;RLA, relative luciferase activity.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t221', 'E_01_067\\t21445863\\t--\\thg19\\tchr21\\t35982223\\t35983306\\tHuman\\tAcute T-Cell Leukaemia Cell\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"To test the putative enhancer activity of RE1 and RE2, we made a number of genetic constructs with a pGL3--Basic vector, which contains the firefly Luciferase gene.\"\\tEnhancer\\tRUNX1\\t--\\tLuciferase Reporter Assay\\tThe promoters were cloned upstream from the Luciferase gene and candidate enhancer elements-downstream of the gene in direct genomic orientation and reverse genomic orientation relative to the corresponding promoter.\\t\"AML1,AML1-EVI-1,AMLCR1,CBF2alpha,CBFA2,EVI-1,PEBP2aB,PEBP2alpha\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t177332', 'E_01_068\\t21445863\\t--\\thg19\\tchr21\\t36141738\\t36142821\\tHuman\\tAcute T-Cell Leukaemia Cell\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"To test the putative enhancer activity of RE1 and RE2, we made a number of genetic constructs with a pGL4--Basic vector, which contains the firefly Luciferase gene.\"\\tEnhancer\\tRUNX1\\t--\\tLuciferase Reporter Assay\\tThe promoters were cloned upstream from the Luciferase gene and candidate enhancer elements-downstream of the gene in direct genomic orientation and reverse genomic orientation relative to the corresponding promoter.\\t\"AML1,AML1-EVI-1,AMLCR1,CBF2alpha,CBFA2,EVI-1,PEBP2aB,PEBP2alpha\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t17817', 'E_01_069\\t21445905\\t--\\thg19\\tchr2\\t136582836\\t136583741\\tHuman\\t\"Caco-2,Cheek Cell\"\\tLow throughput\\t\"Luciferase Reporter Assay,PCR,EMSA\"\\t\"In transient transfection experiments using Caco\\\\2 cells (a human colorectal adenocarcinoma cell line that mimics intestinal epithelial cells) and lactase promoter/luciferase gene constructs, small DNA fragments containing either the C?13,910 or T?13,910 variants of the enhancer region were shown to significantly increase the expression of the reporter gene when included in the constructs. \"\\tEnhancer\\tMCM6\\t--\\t\"Luciferase Reporter Assay,PCR,\"\\t Map of the region of human chromosome 2q21 containing the human lactase (LCT) and minichromosome maintenance \\\\6 (MCM6) genes (panel a). \\t\"MCG40308,Mis5,P105MCM\"\\t--\\t--\\t--\\t--\\t--\\t--\\tPOU2F1\\t\"OCT1,OTF1,oct-1B\"\\tEMSA\\t\"In addition, the transcription factor Oct-1 was shown to bind more strongly to DNA fragments containing the T213,910 variant than the C213,910 variant in electrophoretic mobility shift assays.\"\\trs4988235\\t136608646\\tPCR\\t>2KB\\t\\t13906', 'E_01_070\\t21464207\\t--\\thg19\\tchr1\\t101402363\\t101412724\\tHuman\\tHUVEC\\tLow+High throughput\\t\"Luciferase Reporter Assay,PCR,ChIP,ChIP-qPCR,ChIP-seq\"\\t\"ChIP-seq revealed three distinct STAT6 binding regions in the VCAM-1 locus (kb ?16, ?11, and +15, relative to the TSS) (Fig. 7A).\"\\tEnhancer\\tVCAM1\\t--\\t\"Luciferase Reporter Assay,PCR,ChIP,ChIP-qPCR,ChIP-seq\"\\t\"Moreover, our strategy led to the identification of a novel functionally important STAT6 binding site within 16 kb upstream of the VCAM-1 gene.\"\\t\"CD106,INCAM-100Atherosclerosis\\tDOID:1936\\tD050197\\t--\\t--\\t--\\tSTAT6\\t\"D12S1644,IL-4-STAT,STAT6,STAT6C\"\\t\"ChIP-seq,PCR,Luciferase Reporter Assay\"\\tGenome-wide survey of IL-4-mediated STAT6 binding from sequential chromatin_x0002_immunoprecipitation with deep sequencing (chromatin immunoprecipitation sequencing [ChIP-seq]) in endothelial cells revealed regions of transient and sustained transcription factor binding. \\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t222349', 'E_01_071\\t21494683\\t--\\thg19\\tchr1\\t64083756\\t64083861\\tHuman\\tSK-N-MC\\tLow throughput\\t\"ChIP,PCR\"\\t\"To investigate histone modification and transcription factor binding, we performed chromatin immunoprecipitation (ChIP) assays using a ChIP assay kit (Millipore, Billerica, MA, USA) according to the manufacturer\\'s protocol. \"\\tEnhancer\\tROR1\\t--\\tChIP-seq\\t\"The ROR1 SNPs flanked a strong peak of H3K4me1 (Figure 4A) and the PLCB1 SNPs resided near an H3K4me1 peak and two CTCF binding sites (Figure 5A). H3K4me1, often with H3K27ac, is a chromatin marker for enhancers [29]. CTCF is an insulator-binding element that can interfere with nearby enhancers.\"\\t\"NTRKR1,dJ537F10.1\"\\tInsomnia\\t--\\tD007319\\t--\\t--\\t--\\tPAX6\\t\"AN,AN1,AN2,ASGD5,D11S812E,FVH1,MGDA,WAGR\"\\t\"ChIP,PCR\"\\t\"Cross ChIP PCR between PAX6 and CTCF. PAX6 ChIP DNA was detected in the two CTCF sites, and CTCF ChIP DNA was detected in the PAX6 site.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t155880', 'E_01_072\\t21494683\\t--\\thg19\\tchr1\\t64087305\\t64087660\\tHuman\\tSK-N-MC\\tLow throughput\\t\"ChIP,PCR\"\\t\"To investigate histone modification and transcription factor binding, we performed chromatin immunoprecipitation (ChIP) assays using a ChIP assay kit (Millipore, Billerica, MA, USA) according to the manufacturer\\'s protocol. \"\\tEnhancer\\tROR1\\t--\\tChIP-seq\\t\"The ROR1 SNPs flanked a strong peak of H3K4me1 (Figure 4A) and the PLCB1 SNPs resided near an H3K4me1 peak and two CTCF binding sites (Figure 5A). H3K4me1, often with H3K27ac, is a chromatin marker for enhancers [29]. CTCF is an insulator-binding element that can interfere with nearby enhancers.\"\\t\"NTRKR1,dJ537F10.1\"\\tInsomnia\\t--\\tD007319\\t--\\t--\\t--\\tPAX6\\t\"AN,AN1,AN2,ASGD5,D11S812E,FVH1,MGDA,WAGR\"\\t\"ChIP,PCR\"\\t\"Cross ChIP PCR between PAX6 and CTCF. PAX6 ChIP DNA was detected in the two CTCF sites, and CTCF ChIP DNA was detected in the PAX6 site.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t152206', 'E_01_073\\t21494683\\t--\\thg19\\tchr20\\t8684437\\t8686556\\tHuman\\tSK-N-MC\\tLow throughput\\t\"ChIP,PCR\"\\t\"To investigate histone modification and transcription factor binding, we performed chromatin immunoprecipitation (ChIP) assays using a ChIP assay kit (Millipore, Billerica, MA, USA) according to the manufacturer\\'s protocol. \"\\tEnhancer\\tPLCB1\\t--\\tChIP-seq\\t\"The ROR1 SNPs flanked a strong peak of H3K4me1 (Figure 4A) and the PLCB1 SNPs resided near an H3K4me1 peak and two CTCF binding sites (Figure 5A). H3K4me1, often with H3K27ac, is a chromatin marker for enhancers [29]. CTCF is an insulator-binding elementhat can interfere with nearby enhancers.\"\\t\"EIEE12,PI-PLC,PLC-154,PLC-I,PLC-beta-1,PLC154,PLCB1A,PLCB1B\"\\tInsomnia\\t--\\tD007319\\t--\\t--\\t--\\tPAX6\\t\"AN,AN1,AN2,ASGD5,D11S812E,FVH1,MGDA,WAGR\"\\t\"ChIP,PCR\"\\t\"Cross ChIP PCR between PAX6 and CTCF. PAX6 ChIP DNA was detected in the two CTCF sites, and CTCF ChIP DNA was detected in the PAX6 site.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t572586', 'E_01_074\\t21494683\\t--\\thg19\\tchr20\\t8687417\\t8688741\\tHuman\\tSK-N-MC\\tLow throughput\\t\"ChIP,PCR\"\\t\"To investigate histone modification and transcription factor binding, we performed chromatin immunoprecipitation (ChIP) assays using a ChIP assay kit (Millipore, Billerica, MA, USA) according to the manufacturer\\'s protocol. \"\\tEnhancer\\tPLCB1\\t--\\tChIP-seq\\t\"The ROR1 SNPs flanked a strong peak of H3K4me1 (Figure 4A) and the PLCB1 SNPs resided near an H3K4me1 peak and two CTCF binding sites (Figure 5A). H3K4me1, often with H3K27ac, is a chromatin marker for enhancers [29]. CTCF is an insulator-binding elementt can interfere with nearby enhancers.\"\\t\"EIEE12,PI-PLC,PLC-154,PLC-I,PLC-beta-1,PLC154,PLCB1A,PLCB1B\"\\tInsomnia\\t--\\tD007319\\t--\\t--\\t--\\tPAX6\\t\"AN,AN1,AN2,ASGD5,D11S812E,FVH1,MGDA,WAGR\"\\t\"ChIP,PCR\"\\t\"Cross ChIP PCR between PAX6 and CTCF. PAX6 ChIP DNA was detected in the two CTCF sites, and CTCF ChIP DNA was detected in the PAX6 site.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t575168', 'E_01_075\\t21494683\\t--\\thg19\\tchr20\\t8718344\\t8719933\\tHuman\\tSK-N-MC\\tLow throughput\\t\"ChIP,PCR\"\\t\"To investigate histone modification and transcription factor binding, we performed chromatin immunoprecipitation (ChIP) assays using a ChIP assay kit (Millipore, Billerica, MA, USA) according to the manufacturer\\'s protocol. \"\\tEnhancer\\tPLCB1\\t--\\tChIP-seq\\t\"The ROR1 SNPs flanked a strong peak of H3K4me1 (Figure 4A) and the PLCB1 SNPs resided near an H3K4me1 peak and two CTCF binding sites (Figure 5A). H3K4me1, often with H3K27ac, is a chromatin marker for enhancers [29]. CTCF is an insulator-binding elementcan interfere with nearby enhancers.\"\\t\"EIEE12,PI-PLC,PLC-154,PLC-I,PLC-beta-1,PLC154,PLCB1A,PLCB1B\"\\tInsomnia\\t--\\tD007319\\t--\\t--\\t--\\tPAX6\\t\"AN,AN1,AN2,ASGD5,D11S812E,FVH1,MGDA,WAGR\"\\t\"ChIP,PCR\"\\t\"Cross ChIP PCR between PAX6 and CTCF. PAX6 ChIP DNA was detected in the two CTCF sites, and CTCF ChIP DNA was detected in the PAX6 site.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t606228', 'E_01_076\\t21527503\\t--\\thg19\\tchr1\\t33353796\\t33354196\\tHuman\\tHC11 Mammary Epithelium\\tLow throughput\\t\"Luciferase Reporter Assay,PCR,ChIP\"\\t\"As detected by chromatin immunoprecipitation assays, treatment of cells with the progestin agonist R5020 induced a rapid recruitment (5 min) of PR to the proximal promoter (?235 bp) and distal enhancer (?6 kb upstream of transcription start site) of -casein\"\\tEnhancer\\t\"TMEM54,-casein gene\"\\t--\\tChIP\\t\"In this study, we used HC-11 cells and ChIP assays to examine the influence of progesterone and PR on cooperative interactions and dynamics of the various transcription factors and coregulatory proteins that interact with the promoter and enhancer of the endogenous -casein gene.\"\\t\"BCLP,CAC-1,CAC1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"STAT5B,CEBPB,YY1\"\\t\"STAT5,C/EBP-beta,IL6DBP,NF-IL6,TCF5,DELTA,GADEVS,INO80S,NF-E1,UCRBP,YIN-YANG-1\"\\t\"PCR,ChIP\"\\t\"Primers for PCR amplification of the proximal promoter span a 170-bp region (-190 to -20 from the transcription start site) containing binding sites for Stat5a,C/EBP,YY1,and GRE/progestin response element (PRE) half-sites, whereas a 400-bp region (-6400 to -6000) of the distal Enhancer was amplified that contains binding sites for Stat5,C/EBP,and other factors.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t6197', 'E_01_077\\t21533051\\t--\\thg19\\tchr8\\t128747315\\t128748315\\tHuman\\tHCT 116\\tLow+High throughput\\t\"ChIP,PCR,ChIP-seq\"\\tA diagram of novel ?-catenin binding regions (labeled A through F) and characterized Wnt responsive enhancers (labeled 5 WRE and 3 WRE) that were identified in the screen.Real-time PCR analysis of DNA elements precipitated using ?-catenin specific antibodies in ChIP assays conducted in HCT116 (black bars) or HEK293 (gray bars).\\tEnhancer\\tMYC\\tq3C\\t--\\t\"Using quantitative chromatin conformation capture, I show that the distal WREs are aligned with the MYC promoter through large chromatin loops.\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\tColorectal CanceDOID:9256\\tD015179\\tThe architecture of the MYC promoter is comprised of distal elements.\\t3C\\tThese findings suggest that the architecture of the MYC promoter is comprised of distal elements that are juxtaposed through large chromatin loops and that -catenin/TCF4 complexes utilize this conformation to activate MYCexpression in colon cancer cells.\\tTCF4\\t\"E2-2,FECD3,ITF-2,ITF2,PTHS,SEF-2,SEF2,SEF2-1,SEF2-1A,SEF2-1B,SEF2-1D,TCF-4,bHLHb19\"\\tChIP-seq\\t\"In the present study, I characterize five novel -catenin/TCF4 binding sites that were identified using a ChIP-Seq screen for -catenin binding regions in the human HCT116 colon cancer cell line.\"\\trs6983267\\t128413305\\t\"qRT-PCR,3C\"\\t<2KB\\t\\t\\t\\t499', 'E_01_078\\t21533051\\t--\\thg19\\tchr8\\t128749415\\t128750415\\tHuman\\tHCT 116\\tLow+High throughput\\t\"ChIP,PCR,ChIP-seq\"\\tA diagram of novel ?-catenin binding regions (labeled A through F) and characterized Wnt responsive enhancers (labeled 5 WRE and 3 WRE) that were identified in the screen.Real-time PCR analysis of DNA elements precipitated using ?-catenin specific antibodies in ChIP assays conducted in HCT116 (black bars) or HEK293 (gray bars).\\tEnhancer\\tMYC\\tq3C\\t--\\t\"Using quantitative chromatin conformation capture, I show that the distal WREs are aligned with the MYC promoter through large chromatin loops.\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\tColorectal CanceDOID:9256\\tD015179\\tThe architecture of the MYC promoter is comprised of distal elements.\\t3C\\tThese findings suggest that the architecture of the MYC promoter is comprised of distal elements that are juxtaposed through large chromatin loops and that ?-catenin/TCF4 complexes utilize this conformation to activate MYC expression in colon cancer cells.\\tTCF4\\t\"E2-2,FECD3,ITF-2,ITF2,PTHS,SEF-2,SEF2,SEF2-1,SEF2-1A,SEF2-1B,SEF2-1D,TCF-4,bHLHb19\"\\tChIP-seq\\t\"In the present study, I characterize five novel -catenin/TCF4 binding sites that were identified using a ChIP-Seq screen for -catenin binding regions in the human HCT116 colon cancer cell line.\"\\trs6983267\\t128413305\\t\"qRT-PCR,3C\"\\t<2KB\\t\\t\\t\\t1601', 'E_01_079\\t21533051\\t--\\thg19\\tchr8\\t128346390\\t128347994\\tHuman\\tHEK-293\\tLow+High throughput\\t\"ChIP,PCR,ChIP-seq\"\\tA diagram of novel ?-catenin binding regions (labeled A through F) and characterized Wnt responsive enhancers (labeled 5 WRE and 3 WRE) that were identified in the screen.Real-time PCR analysis of DNA elements precipitated using ?-catenin specific antibies in ChIP assays conducted in HCT116 (black bars) or HEK293 (gray bars).\\tEnhancer\\tMYC\\tq3C\\t--\\t\"Using quantitative chromatin conformation capture, I show that the distal WREs are aligned with the MYC promoter through large chromatin loops.\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t401122', 'E_01_080\\t21533051\\t--\\thg19\\tchr8\\t128327941\\t128330615\\tHuman\\tHEK-293\\tLow+High throughput\\t\"ChIP,PCR,ChIP-seq\"\\tA diagram of novel ?-catenin binding regions (labeled A through F) and characterized Wnt responsive enhancers (labeled 5 WRE and 3 WRE) that were identified in the screen.Real-time PCR analysis of DNA elements precipitated using ?-catenin specific antibs in ChIP assays conducted in HCT116 (black bars) or HEK293 (gray bars).\\tEnhancer\\tMYC\\tq3C\\t--\\t\"Using quantitative chromatin conformation capture, I show that the distal WREs are aligned with the MYC promoter through large chromatin loops.\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t419036', 'E_01_081\\t21533051\\t--\\thg19\\tchr8\\t128299198\\t128301336\\tHuman\\tHEK-293\\tLow+High throughput\\t\"ChIP,PCR,ChIP-seq\"\\tA diagram of novel ?-catenin binding regions (labeled A through F) and characterized Wnt responsive enhancers (labeled 5 WRE and 3 WRE) that were identified in the screen.Real-time PCR analysis of DNA elements precipitated using ?-catenin specific antibin ChIP assays conducted in HCT116 (black bars) or HEK293 (gray bars).\\tEnhancer\\tMYC\\tq3C\\t--\\t\"Using quantitative chromatin conformation capture, I show that the distal WREs are aligned with the MYC promoter through large chromatin loops.\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t448047', 'E_01_082\\t21533051\\t--\\thg19\\tchr8\\t128296257\\t128297326\\tHuman\\tHEK-293\\tLow+High throughput\\t\"ChIP,PCR,ChIP-seq\"\\tA diagram of novel ?-catenin binding regions (labeled A through F) and characterized Wnt responsive enhancers (labeled 5 WRE and 3 WRE) that were identified in the screen.Real-time PCR analysis of DNA elements precipitated using ?-catenin specific antib ChIP assays conducted in HCT116 (black bars) or HEK293 (gray bars).\\tEnhancer\\tMYC\\tq3C\\t--\\t\"Using quantitative chromatin conformation capture, I show that the distal WREs are aligned with the MYC promoter through large chromatin loops.\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t451522', 'E_01_083\\t21536859\\t--\\thg19\\tchr21\\t39823183\\t39825183\\tHuman\\t\"T-acute lymphoblastic leukemia (T-ALL),Molt4,REH,416B\"\\tLow+High throughput\\t\"Luciferase Reporter Assay,PCR,ChIP,ChIP-PCR,ChIP-seq\"\\tThe ERG stem cell enhancer is active in human T-cell progenitors and in T-ALL.Graphs at right show these peaks of enrichment confirmed by ChIP-PCR.\\tEnhancer\\t\"ERG', 'E_01_084\\t21551367\\t--\\thg19\\tchr17\\t32565596\\t32566796\\tHuman\\tU-87MG\\tLow throughput\\t\"Luciferase Reporter Assay,PCR,ChIP,EMSA,3C\"\\tSchematic of CCL2 locus showing regions that were tested in the reporter assays.\\tEnhancer\\tCCL2\\t3C\\t\"PCR,qPCR\"\\t\"The C16.5 ECR physically interacts with CCL2 proximal promoter.To detect such physical interactions we used the 3C assay. In the 3C assay, interaction between widely separated regulatory elements is detected. \"\\t\"GDCF-2,HC11,HSMCR30,MCAF,MCP-1,MCP1,SCYA2,SMC-CF\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"NFKB1,SP1,CEBPB\"\\t\"CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50,SP1,C/EBP-beta,IL6DBP,NF-IL6,TCF5\"\\t\"ChIP,EMSA\"\\tThe EMSA and the pulldown experiments demonstrated in vitro interaction of the C16.5 ECR with NF-kB and c/EBPb. We used ChIP assay to determine whether the C16.5 ECR can bind NF-kB and c/EBPb in vivo. Sheared chromatin that was obtained from U87MG cells (treated or not treated with TNF-a) was immmu_x0002_noprecipitated with Abs directed against p65 and p50 subunits of NF-kB and c/EBPb or with control Abs.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t16099', 'E_01_085\\t21556051\\tUBE2C Enhancer\\thg19\\tchr20\\t44430820\\t44433428\\tHuman\\t\"LNCaP,PC-3\"\\tLow throughput\\t\"RT-PCR,Luciferase Reporter Assay,ChIP,3C\"\\t\"we performed quantitative 3C assays for the UBE2C locus in LNCaP and PC-3 cells, in order to identify potential PC-3-specific UBE2C enhancers.\"\\tEnhancer\\tUBE2C\\t3C\\t\"RT-PCR,Western blot\"\\t3C assays were performed at the UBE2C locus to measure crosslinking frequencies between PC-3-specific UBE2C enhancers and the UBE2C promoter\\t\"UBCH10,dJ447F3.2\"\\tProstate Cancer\\tDOID:10283\\tD011471\\t--\\t--\\t--\\t\"FOXA1,GATA2,POU2F1,ETS1\"\\t\"HNF3A,TCF3A,DCML,IMD21,MONOMAC,NFE1B,ETS-1,EWSR2,c-ets-1,p54,OCT1,OTF1,oct-1B\"\\tChIP\\t\"To examine whether these transcription factors were differentially recruited to E1,E2 and E3 on chromatin, chromatin immunoprecipitation (ChIP) assays were performed using specific antibodies against FoxA1, GATA2, Oct1 and ETS1 in LNCaP and PC-3 cells treated with or without DHT.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t9090', 'E_01_086\\t21556051\\tUBE2C Enhancer\\thg19\\tchr20\\t44441255\\t44444472\\tHuman\\t\"LNCaP,PC-3\"\\tLow throughput\\t\"RT-PCR,Luciferase Reporter Assay,ChIP,3C\"\\t\"we performed quantitative 3C assays for the UBE2C locus in LNCaP and PC-3 cells, in order to identify potential PC-3-specific UBE2C enhancers.\"\\tEnhancer\\tUBE2C\\t3C\\t\"RT-PCR,Western blot\"\\t3C assays were performed at the UBE2C locus to measure crosslinking frequencies between PC-3-specific UBE2C enhancers and the UBE2C promoter\\t\"UBCH10,dJ447F3.2\"\\tProstate Cancer\\tDOID:10283\\tD011471\\t--\\t--\\t--\\t\"FOXA1,GATA2,POU2F1,ETS1\"\\t\"HNF3A,TCF3A,DCML,IMD21,MONOMAC,NFE1B,ETS-1,EWSR2,c-ets-1,p54,OCT1,OTF1,oct-1B\"\\tChIP\\t\"To examine whether these transcription factors were differentially recruited to E1,E2 and E3 on chromatin, chromatin immunoprecipitation (ChIP) assays were performed using specific antibodies against FoxA1, GATA2, Oct1 and ETS1 in LNCaP and PC-3 cells treated with or without DHT.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1650', 'E_01_087\\t21556051\\tUBE2C Enhancer\\thg19\\tchr20\\t44478646\\t44484559\\tHuman\\t\"LNCaP,PC-3\"\\tLow throughput\\t\"RT-PCR,Luciferase Reporter Assay,ChIP,3C\"\\t\"we performed quantitative 3C assays for the UBE2C locus in LNCaP and PC-3 cells, in order to identify potential PC-3-specific UBE2C enhancers.\"\\tEnhancer\\tUBE2C\\t3C\\t\"RT-PCR,Western blot\"\\t3C assays were performed at the UBE2C locus to measure crosslinking frequencies between PC-3-specific UBE2C enhancers and the UBE2C promoter\\t\"UBCH10,dJ447F3.2\"\\tProstate Cancer\\tDOID:10283\\tD011471\\t--\\t--\\t--\\t\"FOXA1,GATA2,POU2F1,ETS1\"\\t\"HNF3A,TCF3A,DCML,IMD21,MONOMAC,NFE1B,ETS-1,EWSR2,c-ets-1,p54,OCT1,OTF1,oct-1B\"\\tChIP\\t\"To examine whether these transcription factors were differentially recruited to E1,E2 and E3 on chromatin, chromatin immunoprecipitation (ChIP) assays were performed using specific antibodies against FoxA1, GATA2, Oct1 and ETS1 in LNCaP and PC-3 cells treated with or without DHT.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t40389', 'E_01_088\\t21556051\\tUBE2C Enhancer\\thg19\\tchr20\\t44413081\\t44426646\\tHuman\\t\"PC-3,DU-145\"\\tLow throughput\\t\"RT-PCR,Luciferase Reporter Assay,ChIP,3C\"\\t\"we performed quantitative 3C assays for the UBE2C locus in LNCaP and PC-3 cells, in order to identify potential PC-3-specific UBE2C enhancers.\"\\tEnhancer\\tUBE2C\\t3C\\t\"RT-PCR,Western blot\"\\t3C assays were performed at the UBE2C locus to measure crosslinking frequencies between PC-3-specific UBE2C enhancers and the UBE2C promoter\\t\"UBCH10,dJ447F3.2\"\\tProstate Cancer\\tDOID:10283\\tD011471\\t--\\t--\\t--\\t\"FOXA1,GATA2,POU2F1,ETS1\"\\t\"HNF3A,TCF3A,DCML,IMD21,MONOMAC,NFE1B,ETS-1,EWSR2,c-ets-1,p54,OCT1,OTF1,oct-1B\"\\tChIP\\t\"To examine whether these transcription factors were differentially recruited to E1,E2 and E3 on chromatin, chromatin immunoprecipitation (ChIP) assays were performed using specific antibodies against FoxA1, GATA2, Oct1 and ETS1 in LNCaP and PC-3 cells treated with or without DHT.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t21350', 'E_01_089\\t21665992\\t--\\thg19\\tchr20\\t62593247\\t62595247\\tHuman\\t\"SK-N-AS,Hep G2\"\\tLow throughput\\t\"Luciferase Reporter Assay,EMSA,PCR\"\\t\"To gain insight into the biological significance of rs2275294,luciferase reporter plasmids corresponding to a genomic DNA fragment containing rs2275294 were constructed and a luciferase assay using the human neuroblastoma cell line SK--N--Be(2)C was performed. Constructs containing the ALS--susceptibility allele (C allele) of rs2275294 showed lower enhancer activity than those containing the nonsusceptibility allele, indicating that the SNP affects the ZNF512B transcription level (Fig. 2A). We then examined the allelic difference in the binding of genomic DNA containing rs2275294 to nuclear proteins by the electrophoretic mobility shift assay (EMSA).\"\\tEnhancer\\tZNF512B\\t--\\t\"Luciferase Reporter Assay,EMSA\"\\t\"To gain insight into the biological significance of rs2275294,luciferase reporter plasmids corresponding to a genomic DNA fragment containing rs2275294 were constructed and a luciferase assay using the human neuroblastoma cell line SK--N--Be(2)C was performed. Constructs containing the ALS--susceptibility allele (C allele) of rs2275294 showed lower enhancer activity than those containing the nonsusceptibility allele, indicating that the SNP affects the ZNF512B transcription level (Fig. 2A). We then examined the allelic difference in the binding of genomic DNA containing rs2275294 to nuclear proteins by the electrophoretic mobility shift assay (EMSA).\"\\tGM632\\tAmyotrophic Lateral Sclerosis\\tDOID:332\\tD000690\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\trs2275294\\t62594247\\t\"Luciferase Reporter Assay,EMSA,PCR\"\\t>2KB\\t\\t\\t\\t6191', 'E_01_090\\t21666600\\tGATA Enhancer\\thg19\\tchr4\\t101576220\\t101578659\\tHuman\\tEndothelial Cells (ECs)\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"Subsequently, we searched the GATA2 binding within the locus. From the duplicate ChIP-seqs, GATA2 associations were commonly detected at ?139 kbp and at a weaker level at ?132 kbp (Figure 5A). ChIPCqPCR validation resulted in significant bindings of GATA2 to ?132 and ?139 kbp regions (Figure 5B). To verify which region from the ?132 or ?139 kbp is epigenetically designed as an enhancer, we performed ChIPCqPCR analysis with antibodies against active PolII, H3Ac, H4Ac, and p300. As shown in Figure 5B, compared with the negative control region at ?84 kbp, only the ?139 kbp region showed strong merged binding by p300, H3Ac, H4Ac, and active PolII. \"\\tEnhancer\\tEMCN\\t--\\t\"ChIP-seq,ChIP-qPCR\"\\t\"To verify which region from the ?132 or ?139 kbp is epigenetically designed as an enhancer, we performed ChIPCqPCR analysis with antibodies against active PolII, H3Ac, H4Ac, and p300. As shown in Figure 5B, compared with the negative control region at ?84 kbp, only the ?139 kbp region showed strong merged binding by p300, H3Ac, H4Ac, and active PolII. Moreover, the comparative genome analysis among mammalian species resulted in sharp homologous signals at ?139 kbp (Figure 5A, bottom). Collectively, these findings suggest that the ?139 kbp region is the main GATA2-binding enhancer within the endomucin locus.\"\\t\"EMCN2,MUC14\"\\t--\\t--\\t--\\tEndothelial-speci?c GATA2-mediated endomucin gene expression was regulated by the endothelial-speci?c chromatin loop with a GATA2-associated distal enhancer and core promoter. \\t\"ChIP-seq,3C\"\\t\"By using the ChIP-seq with epigenetic histone modications and chromatin conformation capture assays; we elucidated the mechanistic regulation of endothelial-specic GATA2-mediated endomucin gene expression, that was regulated by the endothelial-specic chromatin loop with a GATA2-associated distal enhancer and core promoter. \"\\tGATA2\\t\"DCML,IMD21,MONOMAC,NFE1B\"\\t\"ChIP,ChIP-seq,3C,PCR,siRNA\"\\tChIP analysis with GATA2 antibody. Immunoprecipitated DNA were quantified with primer pairs described in Supplementary Table SIV.\\t--\\t--\\t--\\t>2KB\\t260943', 'E_01_091\\t21681857\\tNAGS Enhancer\\thg19\\tchr17\\t42078932\\t42079032\\tHuman\\tUreagenesis\\tLow throughput\\tLuciferase Reporter Assay\\t\"The remaining change, a homozygous CCA transversion 3,064--bp upstream of the NAGS translation start codon (c.--3064C > A) (Supp. Fig. S1), was determined to be in the recently identified hepatic nuclear factor 1 (HNF--1)bindingsiteofaconservedNAGSregulatoryregion(Fig.4).Transcription factor binding sites in the NAGS enhancer.Transcription factors HNF--1 and NF--Y bind with in the human NAGS enhancer.\"\\tEnhancer\\tNAGS\\t--\\tLuciferase Reporter Assay\\t\"The functional effect of the mutation on NAGS transcription was examined using a luciferase reporter assay, driven by the NAGS enhancer with either the wild--type, mutated, or consensus HNF--1 binding sites.Cells were cotransfected with vector pGL4.74, containing Renilla luciferase hRluc, as a transfection efficiency control.\"\\t\"AGAS,ARGA\"\\t--\\t--\\t--\\t--\\t--\\t--\\tHNF1A\\t\"HNF-1A,HNF1,HNF4A,IDDM20,LFB1,MODY3,TCF-1,TCF1\"\\tLuciferase Reporter Assay\\t\"The functional effect of the mutation on NAGS transcription was examined using aLuciferase assay,driven by the NAGS Enhancer with either the wild-type,mutated,or consensus HNF-1 binding sites. Reporter plasmids included the NAGS wild-type promoter with the wild-type Enhancer,4.10WT,the Enhancer containing the patients mutation -3064C>A,4.10Pat,or the Enhancer containing the consensus HNF-1 binding site,4.10Con (Fig. 5A).\"\\t--\\t--\\t--\\t>2KB\\t\\t3049', 'E_01_092\\t21689639\\t--\\thg19\\tchr12\\t7940709\\t7941994\\tHuman\\tHEK-293\\tLow throughput\\tLuciferase Reporter Assay\\t\"Genome sequence analysis of the enhancer region of the human NANOG gene showed the presence of two conserved Tcf/Lef--binding motifs: CAAAGA at', 'E_01_093\\t21711161\\t--\\thg19\\tchr17\\t62028378\\t62028924\\tHuman\\t\"HEK-293,HeLa\"\\tLow throughput\\tLuciferase Reporter Assay\\t\"Upstream HS III sequences contained in a 574 bp fragment were previously shown to act as effective enhancers of minimal TKp activity in transfected human HEK293 and HeLa cells.To assess for enhancer--blocking activity, the same HS V fragments were inserted between the HS III enhancer and the downstream TKp and firefly luciferase (Luc) reporter gene. Significant decreases in enhancer activity were observed in both the forward and reverse orientations in both cell lines (p<0.05, n=5C6)(Fig. 2)\"\\tEnhancer\\tGH1\\t--\\t\"Luciferase Reporter Assay,Transfection\"\\t\"Upstream HS III sequences contained in a 574 bp fragment were previously shown to act as effective enhancers of minimal TKp activity in transfected human HEK293 and HeLa cells.To assess for enhancer--blocking activity, the same HS V fragments were inserted between the HS III enhancer and the downstream TKp and firefly luciferase (Luc) reporter gene. Significant decreases in enhancer activity were observed in both the forward and reverse orientations in both cell lines (p<0.05, n=5C6)(Fig. 2)\"\\t\"GH,GH-N,GHB5,GHN,IGHD1A,IGHD1B,IGHD2,hGH-N\"\\t--\\t--\\t--\\tHS V sequences contain efficient Enhancer-blocking activity\\tLuciferase Reporter Assay\\t\"To assess for enhancerblocking activity, the same HS V fragments were inserted between the HS III enhancer and the downstream TKp and firefly luciferase (Luc) reporter gene\"\\t\"YY1,CTCF\"\\t\"DELTA,GADEVS,INO80S,NF-E1,UCRBP,YIN-YANG-1,\"\\tChIP\\t\"In terms of enhancer-blocking activity, the decrease in either CTCF or Yy1 was associated with a significant *50% increase or rescue of enhancer activity in the case of HSVR, and*80% when siRNA for both CTCF and Yy1 were used.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t34099', 'E_01_094\\t21798992\\t--\\thg19\\tchr8\\t117978845\\t117979466\\tHuman\\t\"TC-3,TC-6\"\\tLow throughput\\t\"Luciferase Reporter Assay,PCR,Transfection\"\\t\"we have identified a conserved islet beta cell-specific enhancer in SLC30A8 intron', 'E_01_095\\t21868451\\t--\\thg19\\tchr5\\t134350424\\t134352424\\tHuman\\tMCF-7\\tLow throughput\\t\"ChIP,Luciferase Reporter Assay,3C\"\\t\"Genetic locus from the UCSC', 'E_01_096\\t22160855\\t--\\thg19\\tchr6\\t31130679\\t31130761\\tHuman\\tNT2\\tLow throughput\\t\"ChIP,qPCR\"\\t\"Chromatin immunoprecipitation analysis of the four regions in the proximal promoter (PP), proximal enhancer (PE1 and PE2),and distal enhancer (DE) unraveled consistent histone marks associated with transcription.Oct4 gene expression is dependent on at least three upstream cis-regulatory regions including proximal pro_x0002_moter (PP) located within the first 250 bp of the tran_x0002_scription initiation site, proximal enhancer (PE) located 1.3-1.6 kb upstream, and distal enhancer (DE) located 2.5 kb upstream.\"\\tEnhancer\\tPOU5F1\\t--\\tRT-PCR\\t\"To verify induction of differentiation, mRNA level of the stemness marker gene, Oct4, was evaluated in both kinds of cell cultures, using RT-PCR (Fig. 1b) as well as quantitative real-time.As expected,expression of Oct4 gene was dominant in undifferentiated embryonal cells,and rapidly declined following RA-induced differentiation.\"\\t\"OCT3, OCT4, OTF-3, OTF3, OTF4, Oct-3, Oct-4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1394', 'E_01_097\\t22160855\\t--\\thg19\\tchr6\\t31130445\\t31130606\\tHuman\\tNT2\\tLow throughput\\t\"ChIP,qPCR\"\\t\"Chromatin immunoprecipitation analysis of the four regions in the proximal promoter (PP), proximal enhancer (PE1 and PE2),and distal enhancer (DE) unraveled consistent histone marks associated with transcription.Oct4 gene expression is dependent on at least three upstream cis-regulatory regions including proximal pro_x0002_moter (PP) located within the first 250 bp of the tran_x0002_scription initiation site, proximal enhancer (PE) located 1.3-1.6 kb upstream, and distal enhancer (DE) located 2.5 kb upstream.\"\\tEnhancer\\tPOU5F1\\t--\\tRT-PCR\\t\"To verify induction of differentiation, mRNA level of the stemness marker gene, Oct4, was evaluated in both kinds of cell cultures, using RT-PCR (Fig. 1b) as well as quantitative real-time.As expected,expression of Oct4 gene was dominant in undifferentiated embryonal cells,and rapidly declined following RA-induced differentiation.\"\\t\"OCT3, OCT4, OTF-3, OTF3, OTF4, Oct-3, Oct-4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1588', 'E_01_098\\t22160855\\t--\\thg19\\tchr6\\t31129568\\t31129735\\tHuman\\tNT2\\tLow throughput\\t\"ChIP,qPCR\"\\t\"Chromatin immunoprecipitation analysis of the four regions in the proximal promoter (PP), proximal enhancer (PE1 and PE2),and distal enhancer (DE) unraveled consistent histone marks associated with transcription.Oct4 gene expression is dependent on at least three upstream cis-regulatory regions including proximal pro_x0002_moter (PP) located within the first 250 bp of the tran_x0002_scription initiation site, proximal enhancer (PE) located 1.3-1.6 kb upstream, and distal enhancer (DE) located 2.5 kb upstream.\"\\tEnhancer\\tPOU5F1\\t--\\tRT-PCR\\t\"To verify induction of differentiation, mRNA level of the stemness marker gene, Oct4, was evaluated in both kinds of cell cultures, using RT-PCR (Fig. 1b) as well as quantitative real-time.As expected,expression of Oct4 gene was dominant in undifferentiated embryonal cells,and rapidly declined following RA-induced differentiation.\"\\t\"OCT3, OCT4, OTF-3, OTF3, OTF4, Oct-3, Oct-4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2462', 'E_01_099\\t22212979\\tPSA Enhancer\\thg19\\tchr19\\t51354804\\t51355203\\tHuman\\tLNCaP\\tLow throughput\\t\"qRT-PCR,ChIP\"\\t\"Chromatin immunoprecipitation (ChIP) on the promoters and enhancers of prostate specific antigen (PSA) and transmembrane protease, serine(TMPRSS2) genes was performed.For PSA, in addition to the AREs located 131 and 150 bp from the tran_x0002_scription start site (TSS), an enhancer region 4 kb upstream of the TSS has been documented.For the TMPRSS2 gene, the AREs in the regulatory region are located 13.5 kb [27,38] and 700 bp up_x0002_stream of the TSS.\"\\tEnhancer\\tKLK3\\t--\\t\"ChIP-qPCR,ChIP\"\\t\"Chromatin immunoprecipitation(ChIP) on the promoters and enhancers of prostate specific antigen (PSA) and transmembrane protease, serine 2(TMPRSS2) genes was performed. qRT-PCR was used to measure the levels of PSA and TMPRSS2 transcripts.We performed ChIP-qPCR analyses on these selected genomic sites of PSA and TMPRSS2.\"\\t\"APS,KLK2A1,PSA,hK3\"\\t--\\t--\\t--\\t--\\t--\\t--\\tAR\\t\"AIS8,DHTR,HUMARA,HYSP1,KD,NR3C4,SBMA,SMAX1,TFM\"\\tChIP-qPCR\\t\"At the 1 hr time point, AR loading onto the TMPRSS2 enhancer (Fig. 3A) was already significantly (P < 0.001, t-test) higher in LNCaP-ARhi cells than in the control cells. However, the TMPRSS2 promoter did not significantly recruit AR, at least in the 4 hr time (Fig. 3B).ChIP-qPCR was performed to assess the AR loading.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3166', 'E_01_100\\t22212979\\tTMPRSS2 Enhancer\\thg19\\tchr21\\t42822120\\t42822800\\tHuman\\tLNCaP\\tLow throughput\\t\"qRT-PCR,ChIP\"\\t\"Chromatin immunoprecipitation (ChIP) on the promoters and enhancers of prostate specific antigen (PSA) and transmembrane protease, serine(TMPRSS2) genes was performed.For PSA, in addition to the AREs located 131 and 150 bp from the tran_x0002_scription start site (TSS), an enhancer region 4 kb upstream of the TSS has been documented.For the TMPRSS2 gene, the AREs in the regulatory region are located 13.5 kb [27,38] and 700 bp up_x0002_stream of the TSS.\"\\tEnhancer\\tTMPRSS2\\t--\\t\"ChIP-qPCR,ChIP\"\\t\"Chromatin immunoprecipitation(ChIP) on the promoters and enhancers of prostate specific antigen (PSA) and transmembrane protease, serine 2(TMPRSS2) genes was performed. qRT-PCR was used to measure the levels of PSA and TMPRSS2 transcripts.We performed ChIP-qPCR analyses on these selected genomic sites of PSA and TMPRSS2.\"\\t\"PP9284,PRSS10\"\\t--\\t--\\t--\\t--\\t--\\t--\\tAR\\t\"AIS8,DHTR,HUMARA,HYSP1,KD,NR3C4,SBMA,SMAX1,TFM\"\\tChIP-qPCR\\t\"At the 1 hr time point, AR loading onto the TMPRSS2 enhancer (Fig. 3A) was already significantly (P < 0.001, t-test) higher in LNCaP-ARhi cells than in the control cells. However, the TMPRSS2 promoter did not significantly recruit AR, at least in the 4 hr time (Fig. 3B).ChIP-qPCR was performed to assess the AR loading.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t14017', 'E_01_101\\t22253448\\t--\\thg19\\tchr20\\t55417234\\t55417304\\tHuman\\t\"NK-92,U-2 OS\"\\tLow throughput\\t\"Luciferase Reporter Assay,EMSA,RT-PCR,ChIP\"\\t\"We won_x0002_dered whether the essential and enhancer elements of the NCR1 promoter were tissue-specific. Thus, we ran luciferase assays using the 5_x0004_ truncation constructs in the U2OS cell line.Thus, the essential region acts primarily as a pan-tissue promoter. On the other hand, the -200 to -270 region acts as an enhancer or suppressor, depending on the cellular context.\"\\tEnhancer\\tNCR1\\t--\\t\"Luciferase Reporter Assay,EMSA,RT-PCR,ChIP\"\\t\"We wondered whether either RUNX1 or RUNX3 can indeed bind the predicted motifs in the NCR1 promoter. To test this possibility, electrophoretic mobility shift assays (EMSA) were performed. Here we focus on the proximal upstream region of the human NCR1 gene. We identify two cis-regulatory elements in this region and describe a role of runt related transcription factor (RUNX) proteins, especially RUNX3, in regulating NCR1 expression.\"\\t\"CD335,LY94,NK-p46,NKP46\"\\t--\\t--\\t--\\tThis latter regulatory element contains a runt related-transcription factor (RUNX) recognition motif that preferentially binds RUNX3. Interfering with RUNX proteins using a dominant negative form results in decreased Ncr1 expression. \\t\"qPCR,ChIP\"\\t\"We found two key cis-regulatory elements in the immediate vicinity upstream of the gene. One element acts as an essential promoter, whereas the other acts as a tissue-dependent enhancer/repressor. This latter regulatory element contains a runt related-transcription factor (RUNX) recognition motif that preferentially binds RUNX3. Interfering with RUNX proteins using a dominant negative form results in decreased Ncr1 expression. \"\\t\"RUNX1,RUNX3\"\\t\"AML1,AML1-EVI-1,AMLCR1,CBF2alpha,CBFA2,EVI-1,PEBP2aB,PEBP2alpha,AML2,CBFA3,PEBP2aC\"\\t\"RT-PCR,EMSA,ChIP\"\\t\"To test this possibility, electrophoretic mobility shift assays (EMSA) were performed. We tested the two RUNX sites separately. Probes spanning the putative RUNX binding site in the Enhancer were shifted by NK92 nuclear extract.To determine whether RUNX binding to the NCR1 promoter is NK-specific, ChIP was performed in other cell lines.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t238', 'E_01_102\\t22260630\\t--\\thg19\\tchr9\\t132491985\\t132492534\\tHuman\\tHFL-1\\tLow throughput\\t\"DNaseI-seq,ChIP,RT-PCR\"\"We identified two DNase I-hypersensitive sites within the proximal promoter near the Egr-1 element and a novel distal site near ? 8.6 kb. Functional analysis of the distal', 'E_01_103\\t22265404\\t--\\thg19\\tchr14\\t90847327\\t90849327\\tHuman\\t\"MCF-7,K-562,HeLa,HCT 116,Nb4\"\\tLow+High throughput\\t\"3C-qPCR,Luciferase Reporter Assay,ChIP-seq\"\\t\"To examine potential enhancer activity of promoters, we per_x0002_formed luciferase reporter gene assays, a commonly used method for promoter and enhancer characterization.\"\\tEnhancer\\tCALM1 \\t3C-qPCR\\tLuciferase Reporter Assa\"In another example (Figure 5F), the promoter of CALM1 interacts with an enhancer element 15 kb upstream and connects to the promoter of C14orf102 further upstream in 65kb. Both RNA-Seq data and the H3K4me3/me1 log ratio indi_x0002_cated that the CALM1 promoter was strong, whereas the C14orf102 promoter was weak and enhancer-like. The luciferase reporter gene assay showed marginal enhancement to the CALM1 promoter reporter gene activity by the native CALM1 enhancer and the C14orf102 promoter individually.\"\\t\"CALML2,CAM2,CAM3,CAMB,CAMC,CAMI,CAMIII,CPVT4,DD132,LQT14,PHKD,caM\"\\tCongenital Limb Deformity\\t--\\tD017880\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t15000', 'E_01_104\\t22303449\\t--\\thg19\\tchr5\\t44385749\\t44386845\\tHuman\\tHuman Embryos\\tLow throughput\\t\"ChIP,EMSA\"\\t\"In conjunction, we identified a novel enhancer containing a highly conserved ISL1 consensus binding site within the FGF10 first intron. ChIP and EMSA demonstrated its direct occupation by ISL1. Transcription mediated by ISL1 from this FGF10 intronic element was enhanced by the presence of GATA4 and TBX20 cardiac transcription factors. Finally, transgenic mice confirmed that endogenous factors bound the human FGF10 intronic enhancer to drive reporter expression in the developing cardiac outflow tract.\"\\tEnhancer\\tFGF10\\t--\\t\"ChIP,EMSA\"\\t\"Using chromatin immunoprecipitation (ChIP) of microdissected embryonic human hearts, we demonstrated that at Carnegie stages 14C15 (33C36 dpf), ISL1 bound to and enriched a 327 bp FGF10-Int1 fragment (Fig. 2A). In contrast, ISL1 did not occupy FGF10-Pr1 or FGF10-Pr2. Acetylated histone H4 did bind both the ISL1 and FGF10 promoters at CS14-15, confirming that the chromatin around these two promoters is transcriptionally active in the human heart at these stages.\"\\tFGF10\\t--\\t--\\t--\\tEndogenous factors bound the human FGF10 intronic enhancer to drive reporter expression in the developing cardiac outflow tract.\\t\"ChIP,EMSA,Transgenic mice\"\\t\"Finally, transgenic mice confirmed that endogenous factors bound the human FGF10 intronic enhancer to drive reporter expression in the developing cardiac outflow tract.\"\\t\"ISL1,GATA4,TBX20\"\\t\"ISLET1,Isl-1,ASD2,TACHD,TOF,VSD1,ASD4\"\\t\"ChIP,EMSA\"\\tChIP and EMSA demonstrated its direct occupation by ISL1. Transcription mediated by ISL1 from this FGF10 intronic element was enhanced by the presence of GATA4 and TBX20 cardiac transcription factors. \\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t81201', 'E_01_105\\t22337891\\t--\\thg19\\tchr11\\t101981958\\t101982137\\tHuman\\t\"HCT 116,SW620,SW480,RKO\"\\tLow throughput\\t\"ChIP,RT-PCR,Luciferase Reporter Assay\"\\t\"To test whether -catenin/TCF4 complexes were responsible for enhancer activity, we mutated the consensus TCF motif to CGCTGAT.In transfected cells, levels of luciferase', 'E_01_106\\t22370642\\tARE Enhancer\\thg19\\tchr22\\t35766960\\t35768960\\tHuman\\tHeLa\\tLow throughput\\t\"qPCR,ChIP\"\\t\"Further_x0002_more, chromatin immunoprecipitation showed that RAC3 bound tightly to the ARE enhancer region of the HO-1 promoter via Nrf2 binding. These data suggest that Nrf2', 'E_01_107\\t22383952\\t--\\thg19\\tchr17\\t42078865\\t42079198\\tHuman\\tHep G2\\tLow throughput\\t\"PCR,ChIP,Luciferase Reporter Assay\"\\t\"Reporter assays to compare the effect of the enhancer in liver,intestine and lung cells, included data that were normalized to the reporter expression driven by the NAGS promoter. While the NAGS enhancer (4.10PromEnh) increased expression of the reporter gene by 50% in liver derived cells (Figure 2A), expression of the luciferase gene did not increase in the intestine or lung derived cells (Figure 8) suggesting that the enhancer may determine tissue specificity of NAGS expression.\"\\tEnhancer\\tNAGS\\t--\\t\"ChIP,Luciferase Reporter Assay\"\\t\"We subsequently confirmed that these regions function as promoter and enhancer and that the enhancer is most effective in liver cells. Avidin-agarose protein_x0002_DNA pull-down assays have been used to confirm binding of Sp1 and CREB within the NAGS promoter and Hepatic Nuclear Factor 1 (HNF-1) and NF-Y within the enhancer regions. Chromatin immunoprecipitation (ChIP) and quantitative real_x0002_time PCR have been used to independently verify that Sp1 and CREB bind to the promoter region, and HNF-1 and NF-Y bind to', 'E_01_108\\t22408256\\t--\\thg19\\tchr9\\t136141944\\t136147321\\tHuman\\t\"K-562,KATO III\"\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"To verify the enhancer potential of these sites using luciferase reporter assays,PCR-amplified fragments or genomic DNAs from the HG-1 clone encompassing them were subcloned upstream of the ABO proximal promoter sequence in luciferase reporter plasmids. Transient transfections into K562 cells were carried out using these reporter vectors. DNA regions used in the reporter plasmids are denoted as regions -8.8,-5.5,-3.8,+2.0,+5.8, and +11.5.\"\\tEnhancer\\tABO\\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"On the basis of DNase IChypersensitive sites in and up_x0002_stream of ABO in K562 cells, in the pres_x0002_ent study, we prepared reporter plasmid constructs including these sites. Subsequent luciferase assays indicated a novel positive regulatory element in intron 1.This element was shown to enhance ABO promoter activity in an erythroid cellCspecific manner.\"\\t\"A3GALNT,A3GALT1,GTB,NAGAT\"\\t--\\t--\\t--\\t--\\t--\\t--\\tGATA1\\t\"ERYF1,GATA-1,GF-1,GF1,NF-E1,NFE1,XLANP,XLTDA,XLTT\"\\t\"Luciferase Reporter Assay,EMSA\"\\tThis element was shown to enhance ABO promoter activity in an erythroid cellCspecific manner. Electrophoretic mobilityCshift assays demonstrated that it bound to the tissue-restricted transcription fac_x0002_tor GATA-1.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t14071', 'E_01_109\\t22645302\\t--\\thg19\\tchr12\\t8036824\\t8048824\\tHuman\\t\"HEK-293,HeLa\"\\tLow+High throughput\\t\"ChIP-PCR,RT-PCR,3C-ChIP,ChIP-seq\"\\t\"To examine whether histone enhancers mark changes under hypoxia, we compared the HIF1 binding sites and H3K27ac under normoxia and hypoxia on a genome-wide scale. The number of total reads and uniquely mapped sequences of the ChIP-seq for histone marks are shown in Table S4 in the supplemental material. H3K27ac sites overlapped at 12,498 sites between the state of normoxia (14,679 sites) and hypoxia (16,366 sites) on a genome-wide scale.The results of the HIF1 ChIP-seq analysis provided data of hypoxic enhancer 2 (kbp -24 from the TSS) at the SLC2A3 loci.\"Enhancer\\tSLC2A3\\t3C\\t\"ChIP-PCR,ChIP\"\\tWe elucidated that both the chromatin conformational structure and histone modification change under hy_x0002_poxic conditions and enhance the expression of SLC2A3 based on the combined results of chromatin conformation capture (3C) and ChIP assays. KDM3A is recruited to the SLC2A3 locus in an HIF1-dependent manner and demethylates H3K9me2 so as to upregulate its expression.These findings provide novel insights into the interaction between HIF1 and KDM3A and also the epigenetic regulation of HIF1.\\tGLUT3\\t--\\t--\\t--\\tThe enhancer 2 was functionally active under hypoxic condition\\t\"ChIP-PCR,Luciferase Reporter Assay\"\\t\"The experimental data described above showed that distal enhancer 2 was functionally active under hypoxic conditions, suggesting that enhancer 2 has an especially close proximity to the promoter of SLC2A3 under hypoxic conditions. \"\\tHIF1A\\t\"HIF-1-alpha,HIF-1A,HIF-1alpha,HIF1,HIF1-ALPHA,MOP1,PASD8,bHLHe78\"\\tChIP\\tGenome-wide analysis of HIF1 binding sites (chromatin immunoprecipitation[ChIP] with deep sequencing) of endothelial cells clarified that HIF1 mainly binds to the intergenic regions distal from tran_x0002_scriptional starting sites under both normoxia and hypoxia.\\t--\\t--\\t--\\t>2KB\\t28999', 'E_01_110\\t22645302\\t--\\thg19\\tchr12\\t8035824\\t8037824\\tHuman\\t\"HEK-293,HeLa\"\\tLow+High throughput\\t\"ChIP-PCR,RT-PCR,3C-ChIP,ChIP-seq\"\\t\"To examine whether histone enhancers mark changes under hypoxia, we compared the HIF1 binding sites and H3K27ac under normoxia and hypoxia on a genome-wide scale. The number of total reads and uniquely mapped sequences of the ChIP-seq for histone marks are shown in Table S4 in the supplemental material. H3K27ac sites overlapped at 12,498 sites between the state of normoxia (14,679 sites) and hypoxia (16,366 sites) on a genome-wide scale.The results of the HIF1 ChIP-seq analysis provided data of hypoxic enhancer 2 (kbp -24 from the TSS) at the SLC2A3 loci.\"Enhancer\\tSLC2A3\\t3C\\t\"ChIP-PCR,ChIP\"\\tWe elucidated that both the chromatin conformational structure and histone modification change under hy_x0002_poxic conditions and enhance the expression of SLC2A3 based on the combined results of chromatin conformation capture (3C) and ChIP assays. KDM3A is recruited to the SLC2A3 locus in an HIF1-dependent manner and demethylates H3K9me2 so as to upregulate its expression.These findings provide novel insights into the interaction between HIF1 and KDM3A and also the epigenetic regulation of HIF1.\\tGLUT3\\t--\\t--\\t--\\tHIF1 is necessary to keep the close proximity of chromatin conformation at the kbp 35 regions of SLC2A3 loci\\t\"ChIP-seq,3C-ChIP,ChIP-PCR\"\\t\"This result indicated that HIF1 is necessary to keep the close proximity of chromatin conformation at the kbp 35 regions of SLC2A3 loci. Taking all of the ndings together, under normoxia, the kbp 35 region functions as an enhancer to induce the expression of SLC2A3 at a relatively low level and retains chromatin conformation also under normoxia.\"\\tHIF1A\\t\"HIF-1-alpha,HIF-1A,HIF-1alpha,HIF1,HIF1-ALPHA,MOP1,PASD8,bHLHe78\"\\tChIP\\tGenome-wide analysis of HIF1 binding sites (chromatin immunoprecipitation[ChIP] with deep sequencing) of endothelial cells clarified that HIF1 mainly binds to the intergenic regions distal from tran_x0002_scriptional starting sites under both normoxia and hypoxia.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t34999', 'E_01_111\\t22665440\\t--\\thg19\\tchr17\\t69107686\\t69108045\\tHuman\\tLNCaP\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,3C-qPCR,ChIP-qPCR\"\\t\"To determine if the rs1859962 PCa risk LD block harbors enhancers, we examined the H3K4me1 ge_x0002_nome-wide distribution defined by ChIP-seq assays in LNCaP and VCaP PCa cells.We show that there are five enhancers within the target genomic segment, hereafter referred to as E1CE5.\"\\tEnhancer\\tSOX9\\t--\\tqPCR\\t\"This chromatin loop was confirmed through TaqMan probeCbased qPCR assays with a probe targeting the SOX9 gene (Fig. 2C). The chromatin interaction was further confirmed by sequencing the PCR product (Supplemental Fig. S7). Further_x0002_more, this chromatin loop appears to be specific to SOX9, because no other loop could be detected between E1 and any other genes(MAP2K6, KCNJ16, KCNJ2) found within this ;3-Mb window.\"\\t\"CMD1,CMPD1,SRA1,SRXX2,SRXY10\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"AR,FOXA1\"\\t\"AIS8, DHTR, HUMARA, HYSP1, KD, NR3C4, SBMA, SMAX1,TFM,HNF3A,TCF3A\"\\t\"ChIP,Luciferase Reporter Assay\"\\t\"In agreement, the luciferase assay in LNCaP cells depletedor not of AR using siRNA reveals that AR is required for the increased transcriptional response associated with the rs8072254 variant A allele.The FKH PWM, defined based on thousands of FOXA1 binding sites identified by ChIP-seq assays, reveals that the tenth position is invariably an A residue.\"\\trs8072254\\t69107816\\t\"Luciferase Reporter Assay,ChIP\"\\t>2KB\\t\\t1009294', 'E_01_112\\t22665440\\t--\\thg19\\tchr17\\t69108466\\t69108765\\tHuman\\tLNCaP\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,3C-qPCR,ChIP-qPCR\"\\t\"To determine if the rs1859962 PCa risk LD block harbors enhancers, we examined the H3K4me1 ge_x0002_nome-wide distribution defined by ChIP-seq assays in LNCaP and VCaP PCa cells.We show that there are five enhancers within the target genomic segment, hereafter referred to as E1CE5.\"\\tEnhancer\\tSOX9\\t--\\tqPCR\\t\"This chromatin loop was confirmed through TaqMan probeCbased qPCR assays with a probe targeting the SOX9 gene (Fig. 2C). The chromatin interaction was further confirmed by sequencing the PCR product (Supplemental Fig. S7). Further_x0002_more, this chromatin loop appears to be specific to SOX9, because no other loop could be detected between E1 and any other genes(MAP2K6, KCNJ16, KCNJ2) found within this ;3-Mb window.\"\\t\"CMD1,CMPD1,SRA1,SRXX2,SRXY10\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"AR,FOXA1\"\\t\"AIS8, DHTR, HUMARA, HYSP1, KD, NR3C4, SBMA, SMAX1,TFM,HNF3A,TCF4A\"\\t\"ChIP,Luciferase Reporter Assay\"\\t\"In agreement, the luciferase assay in LNCaP cells depletedor not of AR using siRNA reveals that AR is required for the increased transcriptional response associated with the rs8072254 variant A allele.The FKH PWM, defined based on thousands of FOXA1 binding sites identified by ChIP-seq assays, reveals that the tenth position is invariably an A residue.\"\\trs1859961\\t69108655\\t\"Luciferase Reporter Assay,ChIP\"\\t>2KB\\t\\t1008544', 'E_01_113\\t22771493\\t--\\thg19\\tchr8\\t124323890\\t124325890\\tHuman\\tLNCaP\\tLow throughput\\t\"qPCR,Luciferase Reporter Assay,ChIP,EMSA,3C\"\\t\"Taken together, these data demonstrate that the poten_x0002_tial ARBS identified within the distal enhancer region of ATAD2 gene indeed interacts strongly and specifically with AR in vitro and suggest that it could be a good candidate as a functional ARBS in vivo.To ascertain whether the ARBS identified within the ATAD2 enhancer region is functional in cells in vivo, we performed ChIP using formaldehyde cross-linked LNCaP cells. \"\\tEnhancer\\tATAD2\\t--\\t\"ChIP,EMSA,RT-qPCR\"\\t\"In addition, we analyzed chromatin immunoprecipita_x0002_tion (ChIP)-seq data from the literature to identify genes that might share the same regulation mechanism as ATAD2.With this in mind, we aimed first at checking whether ATAD2 might be a direct target gene of AR. To test this hypothesis, we analyzed ATAD2 gene response to androgens in LNCaP cells in the presence of cycloheximide (CHX), a protein synthesis inhibitor.\"\\t\"ANCCA,CT137,PRO2000\"\\tProstate Cancer\\tDOID:10283\\tD011471\\t--\\t--\\t--\\t\"AR,E2F1\"\\t\"AIS8, DHTR, HUMARA, HYSP1, KD, NR3C4, SBMA, SMAX1, TFM,E2F-1,RBAP1,RBBP3,RBP3\"\\tEMSA\\t\"To check the binding of AR to this putative ARBS, we first performed EMSA experiments with bacterially produced AR-DNA-binding domain (DBD) purified to homogeneity,siE2F1 and siAR transfection showed the very efficient and specific knockdown of E2F1 and AR, respectively, at both mRNA and protein levels. --\\t--\\t--\\t>2KB\\t\\t\\t\\t7199', 'E_01_114\\t22771493\\t--\\thg19\\tchr8\\t124321890\\t124323890\\tHuman\\tLNCaP\\tLow throughput\\t\"qPCR,Luciferase Reporter Assay,ChIP,EMSA,3C\"\\t\"Taken together, these data demonstrate that the poten_x0002_tial ARBS identified within the distal enhancer region of ATAD2 gene indeed interacts strongly and specifically with AR in vitro and suggest that it could be a good candidate as a functional ARBS in vivo.To ascertain whether the ARBS identified within the ATAD2 enhancer region is functional in cells in vivo, we performed ChIP using formaldehyde cross-linked LNCaP cells. \"\\tEnhancer\\tATAD2\\t--\\t\"ChIP,EMSA,RT-qPCR\"\\t\"In addition, we analyzed chromatin immunoprecipita_x0002_tion (ChIP)-seq data from the literature to identify genes that might share the same regulation mechanism as ATAD2.With this in mind, we aimed first at checking whether ATAD2 might be a direct target gene of AR. To test this hypothesis, we analyzed ATAD3 gene response to androgens in LNCaP cells in the presence of cycloheximide (CHX), a protein synthesis inhibitor.\"\\t\"ANCCA,CT137,PRO2000\"\\tProstate Cancer\\tDOID:10283\\tD011471\\tE2F1 interacts with AR on the promoter of ATAD2 through a chromatin loop to activate transcription when stimulated by androgens.\\t\"ChIP-seq,qRT-PCR,Re-ChIP\"\\t\"Moreover, Re-ChIP experiments demonstrated that AR and E2F1 interact in their promoter region. This mechanism of regulation by the RB/ E2F1/AR axis might, thus, be a general mechanism for several androgen-regulated genes in prostate cells.\"\\t\"AR,E2F1\"\\t\"AIS8, DHTR, HUMARA, HYSP1, KD, NR3C4, SBMA, SMAX1, TFM,E2F-1,RBAP1,RBBP3,RBP3\"\\tEMSA\\t\"To check the binding of AR to this putative ARBS, we first performed EMSA experiments with bacterially produced AR-DNA-binding domain (DBD) purified to homogeneity,siE2F1 and siAR transfection showed the very efficient and specific knockdown of E2F0 and AR, respectively, at both mRNA and protein levels. \"\\t--\\t--\\t--\\t>2KB\\t\\t9199', 'E_01_115\\t22771493\\t--\\thg19\\tchr8\\t124319890\\t124321890\\tHuman\\tLNCaP\\tLow throughput\\t\"qPCR,Luciferase Reporter Assay,ChIP,EMSA,3C\"\\t\"Taken together, these data demonstrate that the poten_x0002_tial ARBS identified within the distal enhancer region of ATAD2 gene indeed interacts strongly and specifically with AR in vitro and suggest that it could be a good candidate as a functional ARBS in vivo.To ascertain whether the ARBS identified within the ATAD2 enhancer region is functional in cells in vivo, we performed ChIP using formaldehyde cross-linked LNCaP cells. \"\\tEnhancer\\tATAD2\\t--\\t\"ChIP,EMSA,RT-qPCR\"\\t\"In addition, we analyzed chromatin immunoprecipita_x0002_tion (ChIP)-seq data from the literature to identify genes that might share the same regulation mechanism as ATAD2.With this in mind, we aimed first at checking whether ATAD2 might be a direct target gene of AR. To test this hypothesis, we analyzed ATAD4 gene response to androgens in LNCaP cells in the presence of cycloheximide (CHX), a protein synthesis inhibitor.\"\\t\"ANCCA,CT137,PRO2000\"\\tProstate Cancer\\tDOID:10283\\tD011471\\t--\\t--\\t--\\t\"AR,E2F1\"\\t\"AIS8, DHTR, HUMARA, HYSP1, KD, NR3C4, SBMA, SMAX1, TFM,E2F-1,RBAP1,RBBP3,RBP3\"\\tEMSA\\t\"To check the binding of AR to this putative ARBS, we first performed EMSA experiments with bacterially produced AR-DNA-binding domain (DBD) purified to homogeneity,siE2F1 and siAR transfection showed the very efficient and specific knockdown of E2F1 and AR, respectively, at both mRNA and protein levels. --\\t--\\t--\\t>2KB\\t\\t\\t\\t11199', 'E_01_116\\t23159876\\tbone Enhancer\\thg19\\tchr17\\t48258757\\t48260757\\tHuman\\tC2C12\\tLow throughput\\tLuciferase Reporter Assay\\t\"The Col1a1 promoter analysis showed the existence of two evolutionarily conserved regions containing pairs of Sp1 sites.These regions are located one at 1700 pb from the transcription start, known as bone enhancer(BE) and where the regulation of Col1a1 expression by Osx has been described,and another region located in the proximal promoter', 'E_01_117\\t27980063\\t\\thg19\\tchr1\\t24601612\\t24626812\\tHuman\\tRaji lymphoblastic B cells\\tLow+High throughput\\t\"ChIP-seq,Western blot\"\\t\"ChIP analyses of BRD4 and RNAPII occupancy revealed that decreased BRD4 occupanc on enhancers in response to JQ1 or OTX015 treatment leads to a signiicant concomitant reduction in RNAPII occupancy on these regions (Figure 4H, I and K, L). While BRD4 occupancy was not significantly affected around the Np63 TSS, the potential enhancers displayed a substantial reduction in the occupancy of both BRD4 and RNAPII (Figure 4HandI).', 'E_01_118\\t23184392\\t--\\thg19\\tchr15\\t90288698\\t90290698\\tHuman\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,qRT-PCR,Luciferase Reporter Assay\"\\t\"In addition, a third conserved region is evident from these analyses located -4 to -5.4 kb upstream of the ATG, which had been described as an enhancer for MesP1 expression in nascent mesoderm, primitive streak and cranial mesoderm .We therefore speculated that the cardiovascular-specific region of the human MesP1 promoter resides within the', 'E_01_119\\t23273978\\tP53BER4 Enhancer\\thg19\\tchr1\\t181104024\\t181104549\\tHuman\\tMCF-7\\tLow+High throughput\\t\"qRT-PCR,4C,ChIP-seq,Luciferase Reporter Assay,ChIP\"\\t\"Interestingly, many of these regions possess conserved p53-binding sites and all known hallmarks of enhancer regions. We demonstrate that these p53-bound enhancer regions (p53BERs) indeed contain enhancer activity and interact intra_x0002_chromosomally with multiple neighboring genes to convey long-distance p53-dependent transcription regulation.\"\\tEnhancer\\tMDM2\\t--\\tChIP\\t\"To identify possible target genes of p53-dependent enhancer domains, we applied chromosome conformation capture (4C) technology in combination with next-generation sequencing.). To confirm that these genes are indeed regulated by p53, we treated cells with the MDM2-inhibitor Nut_x0002_lin-3 and analyzed mRNA levels of one prominent neighboring gene for each p53BER.\"\\t\"ACTFS,HDMX,hdm2\"\\t--\\t--\\t--\\tp53BERs produce enhancer RNAs (eRNAs) that are required for efficient transcriptional enhancement of interacting target genes and induction of a p53-dependent cell-cycle arres\\t\"qRT-PCR,4C,ChIP\"\\t\"We show that eRNAs are functional ncRNAs involved in transcription enhancement of genes interacting with the Enhancer they are produced from. Thus, our results imply that p53 can regulate transcription of multiple distantly located genes through binding to Enhancers and that eRNAs produced from these Enhancers are involved in transcrip_x0002_tion enhancement.\"\\tTP53\\t\"BCC7,BMFS5,LFS1,P53,TRP53\"\\tChIP\\t\"Importantly, the enhancer activity of all p53BERs studied here was p53 dependent, as cotransfection of a p53 knockdown vector (p53KD) significantly inhibited enhancer activity.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t111902336', 'E_01_120\\t23273978\\tP53BER2 Enhancer\\thg19\\tchr9 \\t118706130\\t118707280\\tHuman\\tMCF-7\\tLow+High throughput\\t\"qRT-PCR,4C,ChIP-seq,Luciferase Reporter Assay,ChIP\"\\t\"Interestingly, many of these regions possess conserved p53-binding sites and all known hallmarks of enhancer regions. We demonstrate that these p53-bound enhancer regions (p53BERs) indeed contain enhancer activity and interact intra_x0002_chromosomally with multiple neighboring genes to convey long-distance p53-dependent transcription regulation.\"\\tEnhancer\\tMDM2\\t--\\tChIP\\t\"To identify possible target genes of p53-dependent enhancer domains, we applied chromosome conformation capture (4C) technology in combination with next-generation sequencing.). To confirm that these genes are indeed regulated by p53, we treated cells with the MDM2-inhibitor Nut_x0002_lin-3 and analyzed mRNA levels of one prominent neighboring gene for each p53BER.\"\\t\"ACTFS,HDMX,hdm2\"\\t--\\t--\\t--\\tp53BERs produce enhancer RNAs (eRNAs) that are required for efficient transcriptional enhancement of interacting target genes and induction of a p53-dependent cell-cycle arres\\t\"qRT-PCR,4C,ChIP\"\\t\"We show that eRNAs are functional ncRNAs involved in transcription enhancement of genes inter_x0002_acting with the Enhancer they are produced from. Thus, our results imply that p53 can regulate transcription of multiple distantly located genes through binding to Enhancers and that eRNAs produced from these Enhancers are involved in transcrip_x0002_tion enhancement.\"\\tTP53\\t\"BCC7,BMFS5,LFS1,P53,TRP53\"\\tChIP\\t\"Importantly, the enhancer activity of all p53BERs studied here was p53 dependent, as cotransfection of a p53 knockdown vector (p53KD) significantly inhibited enhancer activity.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t49504754', 'E_01_121\\t23302769\\t--\\thg19\\tchr11\\t8301851\\t8303851\\tHuman\\tT-acute lymphoblastic leukemia (T-ALL)\\tLow throughput\\t\"ChIP,RT-PCR,Luciferase Reporter Assay\"\\t\"The LMO1 promoters display bivalent histone marks in multiple blood lineages including T-cells, and a 3 flanking region at LMO1 ? 57 contains a transcriptional enhancer that is active in developing blood cells in transgenic mouse embryos. The LMO1 promoters become activated in T-ALL together with the 3 enhancer, which is bound in primary T-ALL cells by SCL/TAL1 and GATA3.\"\\tEnhancer\\tLMO1\\t--\\t\"ChIP,RT-PCR,Luciferase Reporter Assay\"\\t\"To investigate whether lack of expres_x0002_sion in the haematopoietic system may be due to active repression of the two LMO1 promoters, we analysed ChIP-Sequencing data released to the public domain by the NIH Roadmap Epigeno_x0002_mics Mapping Project.\"\\t\"RBTN1,RHOM1,TTG1\"\\tT-Cell leukemia\\tDOID:715\\tD015458\\t--\\t--\\t--\\t\"TAL1,GATA3\"\\t\"SCL,TCL5,bHLHa17,tal-1,HDR,HDRS\"\\t\"ChIP,qPCR\"\\t\"ChIP assays on T-ALL cells from the same primagraft (X31) using antibodies against SCL/TAL1and GATA3. Analysis by quantitative PCR demonstrates significant binding of both SCL/TAL1 and GATA3 to the LMO1 +57 enhancer, with no binding to either of the two LMO1 promoters.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t57001', 'E_01_122\\t23443045\\t--\\thg19\\tchr9\\t21816751\\t21818751\\tHuman\\tHuman 2BS Diploid Fibroblast Cells\\tLow throughput\\t\"ChIP-qPCR,3C,ChIP,Luciferase Reporter Assay,EMSA\"\\t\"Finally, a systematic survey of putative FOXA1 binding sites in the p16INK4a genomic region revealed an B150 kb distal element that could loop back to the promoter and potenti ate p16INK4a expression.To further investigate the possibility that this 150 kb element can function as authentic enhancer, we inserted a stretch of genomic sequence centred on this site into the luciferase vector containing the p16INK4a promoter and co-transfected these constructs with FOXA1 expression plas_x0002_mids into HeLa cells.\"Enhancer\\tCDKN2A\\t--\\t\"qPCR,ChIP,Luciferase Reporter Assay\"\\t\"To determine whether FOXA1 was required for p16INK4a activation in replicative exhaustion and OIS, we knocked down FOXA1 using lentiviral shRNA and examined the resultant expression change of p16INK4a. As expected, induc_x0002_tion of p16INK4a protein at late passage was impaired in 2BS cells which integrated with lentiviral shRNA for FOXA1 in', 'E_01_123\\t23598529\\t--\\thg19\\tchr2\\t191834162\\t191834795\\tHuman\\tMm96\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,PCR\"\\t\"Our previous analysis of the STAT1 gene recognized a region downstream of the promoter and enhancer regions strongly repressing transcription of luciferase gene repor_x0002_ters (20). In the present study, we define an RE-1 element within this repressor region, to which REST bound and repressed transcription. In the absence of REST, transcriptional repression of the STAT1 gene was dramatically reduced.\"\\tEnhancerSTAT1\\t--\\t\"Luciferase Reporter Assay,ChIP,PCR\"\\t\"Analysis of REST and STAT1 expression levels in human melanoma cells showed that REST is commonly expressed in melanomas and, together with transcription_x0002_al activation and chromatin immunoprecipitation (ChIP) assay, shows that REST negatively regulates STAT1 expression, contributing to the nonresponsiveness of mel_x0002_anomas to IFN.\"\\t\"CANDF7,IMD31A,IMD31B,IMD31C,ISGF-3,STAT91\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t718', 'E_01_124\\t23631855\\t--\\thg19\\tchr12\\t48369059\\t48369213\\tHuman\\tSW 1353\\tLow throughput\\t\"PCR,ChIP,Luciferase Reporter Assay\"\\t\"In addition, CTS increased the association between Smad2/3 and the COL2A1 enhancer on chromatin in SW1353 cells (Fig. 3C). In stretched cells, DNA fragment that contained the COL2A1 enhancer was coimmunoprecipitated with Smad2/3, probably via complex for_x0002_mation among Smad2/3-SOX9-the SOX9-binding site, and was amplified by PCR using specific primers for the COL2A1 enhancer.\"\\tEnhancer\\tCOL2A1\\t--\\t\"PCR,ChIP,Luciferase Reporter Assay\"\\tChromatin immunoprecipitation revealed that CTS increased Smad2/3 interaction with the CCN2 promoter and the COL2A1 enhancer. Our results suggest that CTS epigenetically stimulates CCN2 transcription via TGF-1 release associated with Smad2/3 activation and enhances COL2A1 expression through the complex formation between SOX9 and Smad2/3.\\t\"ANFH,AOM,COL11A3,SEDC,STL1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSOX9\\t\"CMD1,CMPD1,SRA1,SRXX2,SRXY10\"\\t\"Western blot,ChIP\"\\t\"In Western blot(WB)analysis,CTS treatment(2h)did not influence total a mounts of endogenous Smad2/3 and SOX9 in each whole cell lysate(B,20 g/lane). However,Smad2/3 detected in the nuclear fraction was increased by CTS(C,nucleus,20 g/lane). In IP analys is using an anti-SOX9 antibody,CTS increased the association between phosphorylated Smad2/3 and SOX9 in the nuclear fraction derived from stretched SW1353 cells.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2389', 'E_01_125\\t23636943\\t--\\thg19\\tchr11\\t72897942\\t72912333\\tHuman\\tMCF-7\\tLow+High throughput\\t\"3C,ChIP-seq,ChIP-qPCR\"\\t\"We integrated the data from our GRO-seq assays with data from a variety of other genomic assays (e.g., ChIP-seq, DNase-seq, ChIA-PET) using a novel computational pipeline to provide a comprehensive and global view of ESR1 enhancers and their regulation by E2 in MCF-7 cells. Together, our studies have shed new light on the activity of ESR1 at its enhancer sites and provide new insights about en_x0002_hancer function in general, including the potential roles of en_x0002_hancer transcription.\"\\tEnhancer\\tP2RY2\\t--\\t\"ChIP-seq,ChIP-qPCR\"\\t\"As shown in the GRO-seq browser tracks in Figure 1A, transcription of the P2RY2 gene and a region around ERBS1 is up-regulated rapidly in a short time course of treatment with 17b-estradiol.The transcripts from ERBS1(Fig. 1A), as well as ERBSs 2C5 (Supplemental Fig. 2), are produced bidirectionally from both strands of DNA,reminiscent of the enhancer RNAs (eRNAs) described previously (Kim et al. 2010), and the transcribed regions are associated with RNA Pol II and pre_x0002_viously identified transcription start sites (TSSs).\"\\t\"HP2U,P2RU1,P2U,P2U1,P2UR,P2Y2,P2Y2R\"\\t--\\t--\\t--\\t--\\t--\\t--\\tFOXA1\\t\"HNF3A,TCF3A\"\\t3C-PCR\\t\"3C-PCR assay showing E2-induced looping between ERBS1 and the P2RY2 gene. The lowercase letters correspond to the primers denoted by orange arrows shown in panel A. The assays were conducted in the presence (experimental) or absence (control) of DNA ligase, as indicated. Digested and ligated bacterial artificial chromosome (BAC) DNA spanning the entire P2YR2 locus was used as a PCR control.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t24204', 'E_01_126\\t23644027\\t--\\thg19\\tchr5\\t172648612\\t172649177\\tHuman\\tLeukocytes\\tLow throughput\\t\"PCR,Luciferase Reporter Assay\"\\t\"The wild type and variant upstream enhancers (566 bp) were generated with PCR, and then subcloned into BamHI and SalI sites of a luciferase gene expression vector with human NKX2-5 gene promoter, which has been previously reportedll expression constructs were confirmed by direct sequencing.HEK-293 cells were cultured in DMEM with 10% fetal bovine serum,1% glutamine and 1% penicillin/streptomycin under 5% CO2 at 37 C.\"\\tEnhancer\\tNKX2-5\\t--\\t\"PCR,Luciferase Reporter Assay\"\\t\"To determine the effects of the DSVs on transcriptional activity of the enhancer, wild type and variant enhancers were subcloned upstream to the NKX2-5 gene promoter in the luciferase gene expression vector, which has been previously constructed and reported,to generate expression constructs pGL3-WT, pGL3-17483557Ins,pGL3-17483564T and pGL3-17483576G. These constructs were then transiently transfected into HEK-293 cells and luciferase activities were measured. The results were shown in Fig. 2. Expression levels of pGL3-17483564T and pGL3-17483576G were significantly decreased compared to pGL3-WT.\"\\t\"CHNG5,CSX,CSX1,HLHS2,NKX2.5,NKX2E,NKX4-1,VSD3\"\\tCongenital Heart Disease\\tDOID:1682\\tD006330\\tThe DSVs within the upstream Enhancer of the NKX2-5 gene may contribute to a small number of VSD. \\tLuciferase Reporter Assay\\t\"To determine the effects of the DSVs on transcriptional activity of the Enhancer,wild type and variant Enhancers were subcloned upstream to the NKX2-5 gene promoter in the Luciferase gene expression vector,which has been previously constructed and reported. Collectively, these results indicated that the activity of the upstream enhancer of the NKX2-5 gene was decreased by the DSVs g.17483564C>T and g.17483576C>G.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t10211', 'E_01_127\\t23659698\\t--\\thg19\\tchr12\\t21439733\\t21439973\\tHuman\\tHeLa\\tLow throughput\\t\"DNaseI-seq,PCR\"\\t\"To find the enhancers of the IFN gene,different 240-bp fragments of the gene were inserted between the GFP gene and the Alu tandem repeats in the C1-Alu14 plasmid.GFP expression in transfected cells was assessed using fluores_x0002_cence microscopy.We found that five different 240-bp fragments displayed strong enhancer activity.One enhancer was located in an upstream sequence of the IFN gene (?480~?720 bp,FUIFN3F3R);two enhancers (IFN4F4R and IFN6F6R) were located in intron 1;another two enhancers (IFN21F21R and IFN22F22R) were located in a noncoding region downstream of the IFN gene. The percentages of the GFP-positive cells for the five enhancers were 6.9%, 2.6%, 6.3%, 7.7%, and 4.9%,respectively.The ratio of the percentage of the GFP-positive cells induced by the inserting fragments compared with that of C1-Alu14 reflects enhancer activity.\"\\tEnhancerIFNG\\t--\\tDNaseI-seq\\t\"To determine which fragments of the IFN gene activate the GFP reporter gene, the IFN gene (4972 bp) and its upstream (720 bp) and downstream (308 bp) sequences were divided into 25 fragments, in which each fragment was 240 bp long.\"\\t\"IFG,IFI\"\\t--\\t--\\t--\\tThese enhancers may be targets of IFN gene expression regulation.\\t\"PCR,Fuorescence Microscopy\"\\t\"We found that five different 240-bp fragments (FUIFN3F3R, IFN4F4R, IFN6F6R, IFN21F21R, and IFN22F22R) and two 60-bp core sequences (IFN6-2F2R and IFN21-3-4F3-4R) derived from the IFN_x0001_gene contain Enhancers that can activate the GFP reporter gene. These enhancers may be targets of IFN gene expression regulation.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t47108696', 'E_01_128\\t23659698\\t--\\thg19\\tchr12\\t68549287\\t68549545\\tHuman\\tHeLa\\tLow throughput\\t\"DNaseI-seq,PCR\"\\t\"To find the enhancers of the IFN gene,different 240-bp fragments of the gene were inserted between the GFP gene and the Alu tandem repeats in the C1-Alu14 plasmid.GFP expression in transfected cells was assessed using fluores_x0002_cence microscopy.We found that five different 240-bp fragments displayed strong enhancer activity.One enhancer was located in an upstream sequence of the IFN gene (?480~?720 bp,FUIFN3F3R);two enhancers (IFN4F4R and IFN6F6R) were located in intron 1;another two enhancers (IFN21F21R and IFN22F22R) were located in a noncoding region downstream of the IFN gene. The percentages of the GFP-positive cells for the five enhancers were 6.9%, 2.6%, 6.3%, 7.7%, and 4.9%,respectively.The ratio of the percentage of the GFP-positive cells induced by the inserting fragments compared with that of C1-Alu14 reflects enhancer activity.\"\\tEnhancerIFNG\\t--\\tDNaseI-seq\\t\"To determine which fragments of the IFN gene activate the GFP reporter gene, the IFN gene (4972 bp) and its upstream (720 bp) and downstream (308 bp) sequences were divided into 25 fragments, in which each fragment was 240 bp long.\"\\t\"IFG,IFI\"\\t--\\t--\\t--\\tThese enhancers may be targets of IFN gene expression regulation.\\t\"PCR,Fuorescence Microscopy\"\\t\"We found that five different 240-bp fragments (FUIFN3F3R, IFN4F4R, IFN6F6R, IFN21F21R, and IFN22F22R) and two 60-bp core sequences (IFN6-2F2R and IFN21-3-4F3-4R) derived from the IFN_x0001_gene contain Enhancers that can activate the GFP reporter gene. These enhancers may be targets of IFN gene expression regulation.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t867', 'E_01_129\\t23659698\\t--\\thg19\\tchr12\\t68549796\\t68550055\\tHuman\\tHeLa\\tLow throughput\\t\"DNaseI-seq,PCR\"\\t\"To find the enhancers of the IFN gene,different 240-bp fragments of the gene were inserted between the GFP gene and the Alu tandem repeats in the C1-Alu14 plasmid.GFP expression in transfected cells was assessed using fluores_x0002_cence microscopy.We found that five different 240-bp fragments displayed strong enhancer activity.One enhancer was located in an upstream sequence of the IFN gene (?480~?720 bp,FUIFN3F3R);two enhancers (IFN4F4R and IFN6F6R) were located in intron 1;another two enhancers (IFN21F21R and IFN22F22R) were located in a noncoding region downstream of the IFN gene. The percentages of the GFP-positive cells for the five enhancers were 6.9%, 2.6%, 6.3%, 7.7%, and 4.9%,respectively.The ratio of the percentage of the GFP-positive cells induced by the inserting fragments compared with that of C1-Alu14 reflects enhancer activity.\"\\tEnhancerIFNG\\t--\\tDNaseI-seq\\t\"To determine which fragments of the IFN gene activate the GFP reporter gene, the IFN gene (4972 bp) and its upstream (720 bp) and downstream (308 bp) sequences were divided into 25 fragments, in which each fragment was 240 bp long.\"\\t\"IFG,IFI\"\\t--\\t--\\t--\\tThese enhancers may be targets of IFN gene expression regulation.\\t\"PCR,Fuorescence Microscopy\"\\t\"We found that five different 240-bp fragments (FUIFN3F3R, IFN4F4R, IFN6F6R, IFN21F21R, and IFN22F22R) and two 60-bp core sequences (IFN6-2F2R and IFN21-3-4F3-4R) derived from the IFN_x0001_gene contain Enhancers that can activate the GFP reporter gene. These enhancers may be targets of IFN gene expression regulation.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1377', 'E_01_130\\t23659698\\t--\\thg19\\tchr12\\t68553503\\t68553754\\tHuman\\tHeLa\\tLow throughput\\t\"DNaseI-seq,PCR\"\\t\"To find the enhancers of the IFN gene,different 240-bp fragments of the gene were inserted between the GFP gene and the Alu tandem repeats in the C1-Alu14 plasmid.GFP expression in transfected cells was assessed using fluores_x0002_cence microscopy.We found that five different 240-bp fragments displayed strong enhancer activity.One enhancer was located in an upstream sequence of the IFN gene (?480~?720 bp,FUIFN3F3R);two enhancers (IFN4F4R and IFN6F6R) were located in intron 1;another two enhancers (IFN21F21R and IFN22F22R) were located in a noncoding region downstream of the IFN gene. The percentages of the GFP-positive cells for the five enhancers were 6.9%, 2.6%, 6.3%, 7.7%, and 4.9%,respectively.The ratio of the percentage of the GFP-positive cells induced by the inserting fragments compared with that of C1-Alu14 reflects enhancer activity.\"\\tEnhancerIFNG\\t--\\tDNaseI-seq\\t\"To determine which fragments of the IFN gene activate the GFP reporter gene, the IFN gene (4972 bp) and its upstream (720 bp) and downstream (308 bp) sequences were divided into 25 fragments, in which each fragment was 240 bp long.\"\\t\"IFG,IFI\"\\t--\\t--\\t--\\tThese enhancers may be targets of IFN gene expression regulation.\\t\"PCR,Fuorescence Microscopy\"\\t\"We found that five different 240-bp fragments (FUIFN3F3R, IFN4F4R, IFN6F6R, IFN21F21R, and IFN22F22R) and two 60-bp core sequences (IFN6-2F2R and IFN21-3-4F3-4R) derived from the IFN_x0001_gene contain Enhancers that can activate the GFP reporter gene. These enhancers may be targets of IFN gene expression regulation.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5080', 'E_01_131\\t23659698\\t--\\thg19\\tchr12\\t68553762\\t68554012\\tHuman\\tHeLa\\tLow throughput\\t\"DNaseI-seq,PCR\"\\t\"To find the enhancers of the IFN gene,different 240-bp fragments of the gene were inserted between the GFP gene and the Alu tandem repeats in the C1-Alu14 plasmid.GFP expression in transfected cells was assessed using fluores_x0002_cence microscopy.We found that five different 240-bp fragments displayed strong enhancer activity.One enhancer was located in an upstream sequence of the IFN gene (?480~?720 bp,FUIFN3F3R);two enhancers (IFN4F4R and IFN6F6R) were located in intron 1;another two enhancers (IFN21F21R and IFN22F22R) were located in a noncoding region downstream of the IFN gene. The percentages of the GFP-positive cells for the five enhancers were 6.9%, 2.6%, 6.3%, 7.7%, and 4.9%,respectively.The ratio of the percentage of the GFP-positive cells induced by the inserting fragments compared with that of C1-Alu14 reflects enhancer activity.\"\\tEnhancerIFNG\\t--\\tDNaseI-seq\\t\"To determine which fragments of the IFN gene activate the GFP reporter gene, the IFN gene (4972 bp) and its upstream (720 bp) and downstream (308 bp) sequences were divided into 25 fragments, in which each fragment was 240 bp long.\"\\t\"IFG,IFI\"\\t--\\t--\\t--\\tThese enhancers may be targets of IFN gene expression regulation.\\t\"PCR,Fuorescence Microscopy\"\\t\"We found that five different 240-bp fragments (FUIFN3F3R, IFN4F4R, IFN6F6R, IFN21F21R, and IFN22F22R) and two 60-bp core sequences (IFN6-2F2R and IFN21-3-4F3-4R) derived from the IFN_x0001_gene contain Enhancers that can activate the GFP reporter gene. These enhancers may be targets of IFN gene expression regulation.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5338', 'E_01_132\\t23675462\\t--\\thg19\\tchr21\\t43770915\\t43772915\\tHuman\\tMCF-7\\tLow throughput\\t\"RT-PCR,ChIP\"\\t\"To determine whether MSK1 co-occupies the enhancer and UPE with H3S10ph, we performed sequential ChIP assays (re_x0002_ChIP) with mononucleosomes prepared from TPA- treated and', 'E_01_133\\t23872150\\tAP3 Enhancer\\thg19\\tchr6\\t161119825\\t161121825\\tHuman\\t\"Hep G2,MCF-7\"\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,EMSA,PCR\"\\t\"An AP3-like (activator protein 3) enhancer element, located 2.4 kb upstream of PLG-TSS, is involved in PLG transcriptional regulationIn summary we demonstrated that estrogen regulates plasminogen promoter activity. The 5-flanking region of the PLG gene con_x0002_tains several cis-acting elements that are responsive to estrogen.An enhancer located at -11.5 kb confers a dramatic estrogen re_x0002_sponse in reporter assays.\"\\tEnhancer\\tPLG\\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"In our study we could show that a construct comprising the entire upstream region, containing the AP3 re_x0002_sponse element (2.4 kb-PLG) was able to further elevate basal luciferase activity 1.7-fold above that of PLG minimal promoters.The observed increase corresponds well to previously published results [20] and the 2.4 kb-PLG construct represents the most active contiguous promoter fragment analysed in this study. Surprisingly, with E2 stimulation this effect decreased by 54% down to the activity level of the PLG core promoter,although no ERE could be identified within this region.\"\\tPLG\\tHypoplasminogenemia\\t--\\t--\\t\"An AP3-like (activator protein 3) enhancer element, located 2.4 kb upstream of PLG-TSS, is involved in PLG transcriptional regulation.\"\\tLuciferase Reporter Assay\\t\"An AP3-like (activator protein 3) enhancer element, located 2.4 kb upstream of PLG-TSS, is involved in PLG transcriptional regulation [20]. In our study we could show that a construct comprising the entire upstream region, containing the AP3 response element (2.4 kb-PLG) was able to further elevate basal luciferase activity 1.7-fold above that of PLG minimal promoters.\"\\t\"DHII,ERE\"\\t\"anon-EST:Posey114,anon-WO02059370.45,CG13094,DH31,DH-II,dh31,DH31,DH[31],DH[[31],Dh[[31]],DH[[31]],Dmel\\\\CG13094,Drm-DH[[31]],drome-DH31,Drome-DH[[31]],ERE\"\\tLuciferase Reporter Assay\\tTranscriptional effect of the DHII and/or ERE (_x0003_11.5 kb) enhancers on the heterologous crystallin promoter (cryst) driven luciferase activity in HepG2.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2399', 'E_01_134\\t23872150\\tERE Enhancer\\thg19\\tchr6\\t161110725\\t161112725\\tHuman\\t\"Hep G2,MCF-7\"\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,EMSA,PCR\"\\t\"An AP3-like (activator protein 3) enhancer element, located 2.4 kb upstream of PLG-TSS, is involved in PLG transcriptional regulationIn summary we demonstrated that estrogen regulates plasminogen promoter activity. The 5-flanking region of the PLG gene con_x0002_tains several cis-acting elements that are responsive to estrogen.An enhancer located at -11.5 kb confers a dramatic estrogen re_x0002_sponse in reporter assays.\"\\tEnhancer\\tPLG\\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"In our study we could show that a construct comprising the entire upstream region, containing the AP3 re_x0002_sponse element (2.4 kb-PLG) was able to further elevate basal luciferase activity 1.7-fold above that of PLG minimal promoters.The observed increase corresponds well to previously published results [20] and the 2.4 kb-PLG construct represents the most active contiguous promoter fragment analysed in this study. Surprisingly, with E2 stimulation this effect decreased by 54% down to the activity level of the PLG core promoter,although no ERE could be identified within this region.\"\\tPLG\\tHypoplasminogenemia\\t--\\t--\\tThe intergenic region between both genes comprises several transcription-regulatory regions with Enhancer sequences that increase the basal activity of the PLG core promoter.\\tLuciferase Reporter Assay\\tAn estrogen response element located 11.5 kb upstream of the PLG transcription start site is able to convey a dramatic estrogen-dependent elevation of PLG-minimal promoter driven reporter gene expression.\\t\"DHII,ERE\"\\t\"anon-EST:Posey114,anon-WO02059370.45,CG13094,DH31,DH-II,dh31,DH31,DH[31],DH[[31],Dh[[31]],DH[[31]],Dmel\\\\CG13094,Drm-DH[[31]],drome-DH31,Drome-DH[[31]],ERE\"\\tLuciferase Reporter Assay\\tTranscriptional effect of the DHII and/or ERE (_x0003_11.5 kb) enhancers on the heterologous crystallin promoter (cryst) driven luciferase activity in HepG2.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t11499', 'E_01_135\\t23873758\\t--\\thg19\\tchr12\\t48368959\\t48369159\\tHuman\\t\"HeLa,HEK-293\"\\tLow throughput\\t\"PCR,ChIP\"\\t\"Real\\\\time PCR (qPCR) reactions for ChIP and ChIP reChIP were carried out using primers flanking the COL2A1 promoter start site(C167) to (C65), see illustration in Figure 6A. Additional analyses were performed on the Sox9\\\\binding enhancer region of COL2A1(?2259) to (?2430). As a negative control, a distal set of primers flanking the intron 38 (?24148) to (?24368) of the COL2A1 gene was used, based on the protocol by Furlan\\\\Magarill and colleagues.\"\\tEnhancer\\tCOL2A1\\t--\\t\"PCR,ChIP\"\\t\"To understand the regulation of the active COL2A1 promoter, we performed ChIP assays using specific primers flanking the promoter start site and enhancer site within the first intron (illustrated in Fig. 6A). We found enhanced occupancy of the DNA\\\\binding transcription factor SP1 on the promoter start site of 3D cultured cells, whereas passaged cells (P5) showed a slight but insignificant reduction of SP1 (Fig. 6B). Additionally, passaging significantly decreased Sox9 occupancy on the enhancer site.\"\\t\"ANFH,AOM,COL11A3,SEDC,STL1\"\\t--\\t--\\t--\\t\"Although Sox9 (SRY\\\\related high mobility group\\\\Box gene 9) has been shown to regulate the expression of the COL2A1 gene by binding the Enhancer within the first intron,numerous additional coactivators (eg, LSox5, Sox6, PGC1a, Notch) are involved in activa\"\\t\"PCR,ChIP\"\\t\"3D cultures also exhibited augmented mRNA and protein levels of the transcription factors SP1 and Sox9 (Fig. 5A. B), which are known to regulate COL2A1 expression by binding the DNA-motifs of the promoter and Enhancer regions, respectively.\"\\t\"SP1,SOX9\"\\t\"SP1,CMD1,CMPD1,SRA1,SRXX2,SRXY10\"\\tChIP\\t\"Chromatin immunoprecipitation (ChIP) analyses of 3D cultures showed augmented levels of the DNA\\\\binding transcription factor SP1, and the histone methyltransferase Set7/9, on the COL2A1 promoter site.\"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t2312', 'E_01_136\\t23884959\\tPSA Enhancer\\thg19\\tchr8\\t57069398\\t57069482\\tHuman\\tLNCaP\\tLow throughput\\t\"ChIP,qPCR,qRT-PCR\"\\t\"The PSA gene contains HREs at its enhancer region in addition to the androgen-responsive elements. ChIP analyses using HIF-1 antibody identified the HIF-1 occu_x0002_pancy on the enhancer region of the PSA gene in response to hypoxia but not on the promoter region. DHT treatment further increased HIF-1 occupancy on the enhancer under hypoxia (Figure?1D). ChIP anal_x0002_yses using AR antibody showed that more AR binds to the enhancer than to the promoter, and that hypoxia augments DHT-induced AR occupancy.\"\\tEnhancer\\tKLK3\\t--\\t\"ChIP,qRT-PCR\"\\t\"Knockdown of JMJD1A and JMJD2C increased the occurrence of H3K9me2 and me3 on the PSA gene, respectively, thus, blocking AR-induced tran_x0002_scriptional activity.These findings indicated that androgen_x0002_induced PSA gene is a target regulated by JMJD1A and JMJD2C. \"\\t\"APS,KLK2A1,PSA,hK3\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5711270', 'E_01_137\\t24040411\\tECR18 Enhancer\\thg19\\tchr19\\t57566145\\t57571006\\tHuman\\t\"HeLa,HEK-293\"\\tLow throughput\\t\"Luciferase Reporter Assay,PCR,3C,ChIP\"\\t\"The observed interaction is most prominent in brain, but was also detected in testis. Histone modification and DNA methylation on ECR18 show no allele bias, implying that this region is likely functional on both alleles. In vitro assays also reveal ECR18 as a potential enhancer or repressor for the promoter of Peg3. Overall, these results indicate that the promoters of several imprinted genes in the Peg3 domain interact with one evolutionarily conserved region, ECR18, and further suggest that ECR18 may play key roles in the transcription and imprinting control of the Peg3 domain as a distant regulatory element.\"\\tEnhancer\\tPEG3\\t3C\\t\"Luciferase Reporter Assay,PCR\"\\t\"In the current study, the chromatin conformation capture (3C) method was utilized to detect potential interactions of these ECRs with the imprinted genes. According to the results, one region, ECR18, located 200-kb upstream of Peg3 interacts with the two promoter regions of Peg3 and Zim2. The observed interaction is most prominent in brain, but was also detected in testis. \"\\t\"PW1,ZKSCAN22,ZNF904,ZSCAN24\"\\t--\\t--\\t--\\tECR18 may play key roles in the transcription and imprinting control of the Peg3 domain as a distant regulatory element.\\t\"3C,ChIP\"\\t\"In the current study, we performed a series of Chromatin Conformation Capture (3C) and Chromatin Immunoprecipitation (ChIP) analyses to further confirm this prediction. Overall, the current study identifies one region, ECR18, as a key regulatory region for the transcription and imprinting of the Peg3 domain.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t247132', 'E_01_138\\t24086551\\t--\\thg19\\tchr1\\t1083900\\t1085900\\tHuman\\tHMLE\\tLow+High throughput\\t\"qRT-PCR,Luciferase Reporter Assay,ChIP-seq,ChIP-qPCR\"\\t\"We have analyzed the miR-200b~200a~429 locus for epigenetic modifications in breast epithelial and mesenchymal cell lines using chromatin immunoprecipitation assays and DNA methylation analysis. We discovered a novel enhancer located approximately 5.1kb upstream of the miR-200b~200a~429 transcriptional start site. This region was associated with', 'E_01_139\\t24212882\\t--\\thg19\\tchr10\\t23502416\\t23510031\\tHuman\\tEmbryonic Stem Cell\\tLow throughput\\t\"ChIP,PCR,3C,EMSA\"\\t\"In the tenth family a 7.6kb deletion was identified by long range PCR, and sequence analysis showed that the deleted region (chr10:23502416-23510031) included the entire putative enhancer.\"\\tEnhancer\\tPTF1A\\t3C\\tChIP-seq\\t\"To assess if this enhancer truly targets PTF1A, we performed chromatin conformation capture (3C) experiments. This demonstrated that the enhancer region establishes direct interactions with the PTF1A promoter in human pancreatic progenitor cells.\"\\t\"PACA,PAGEN2,PTF1-p48,bHLHa29,p48\"\\tPancreatic Agenesis\\tDOID:0050877\\t--\\t--\\t--\\t--\\t\"FOXA2,PDX1\"\\t\"HNF3B,TCF3B,GSF,IDX-1,IPF1,IUF1,MODY4,PAGEN1,PDX-1,STF-1\"\\tEMSA\\t\"Electrophoretic mobility shift assay showing high affinity, sequence-specific interaction of FOXA2 with a double stranded oligonucleotide containing the wild type (WT) sequence, but not the 363 A>G mutation (MUT).Electrophoretic mobility shift assay showing sequence-specific interaction of PDX1 with oligonucleotide containing the wild type (WT) sequence, but not the 446 A>C mutation (MUT). The retardation signal was suppressed by unlabelled consensus high-affinity binding site for PDX1, and supershifted with antibodies recognizing PDX1.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t24765', 'E_01_140\\t24256810\\t--\\thg19\\tchr11\\t111150916\\t111155431\\tHuman\\t\"HCT 116,SW480,Colorectal Cancer\"\\tLow+High throughput\\t\"PCR,ChIP-seq,Luciferase Reporter Assay,Transfection,EMSA\"\\t\"One potentially active promoter region and several candidate enhancer regions were identified using high-resolution genome wide ChIP-seq profiles for histone modifications including H3K4me1, H3K4me3 and H3K9/14ac (Figure 1A and Supple_x0002_mentary Material, Fig. S1), which we generated for the SW480 and HCT-116 CRC cell lines.\"\\tEnhancer\\tC11orf53\\t--\\tChIP-seq\\t\"This is manifest in normal human colon tissues as eQTL for three genes, C11orf53, C11orf92 and C11orf93. Using bicolor EMSA, we further showed that protein binding is markedly dif_x0002_ferent for the sequences containing the two alleles of rs7130173,with binding affinity of the C allele roughly four times greater than the A allele.\"\\tC11orf53\\tColorectal Cancer\\tDOID:9256\\tD015179\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\trs7130173\\t111154072\\t\"ChIP-seq,Luciferase Reporter Assay,EMSA\"\\t>2KB\\t26468', 'E_01_141\\t24285714\\t--\\thg19\\tchr8\\t127047315\\t127049315\\tHuman\\tAcute Myeloid Leukemia Cell\\tLow+High throughput\\t\"ChIP,ChIP-seq,Luciferase Reporter Assay,4C,ChIP-qPCR,3C,qPCR\"\\t\"To account for this context-specific function, we identify a cluster of lineage-specific enhancers located 1.7 Mb downstream from Myc that are occupied by SWI/SNF as well as the BET protein Brd4. Brg1 is required at these distal elements to maintain transcription factor occupancy and for long-range chromatin looping interactions with the Myc promoter. Notably,these distal Myc enhancers coincide with a region that is focally amplified in ~3% of acute myeloid leukemias.\"\\tEnhancer\\tMYC\\t--\\t\"ChIP,ChIP-seq,ChIP-qPCR\"\\t\"To explain this observation, we hypothesized that Brg1 regulates Myc transcription in leukemia cells by occupying cell type-specific regulatory elements. We evaluated this possibility by performing chromatin immunoprecipitation (ChIP) followed by next_x0002_generation sequencing (ChIP-seq) analysis of Brg1, his_x0002_tone H3 Lys 4 trimethylation (H3K4me3), and histone H3 Lys 27 acetylation (H3K27ac) in RN2 leukemia cells.\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\tAcute Leukemia\\tDOID:12603\\t--\\t--\\t--\\t--\\t\"SMARCA4,CEBPA,CEBPB,ERG,LMO2\"\\t\"BAF190,BAF190A,BRG1,CSS4,MRD16,RTPS2,SNF2,SNF2L4,SNF2LB,SWI2,hSNF2b,C/EBP-alpha,CEBP,C/EBP-beta,IL6DBP,NF-IL6,TCF5,erg-3,p55,LMO-2,RBTN2,RBTNL1,RHOM2,TTG2\"\\t\"ChIP-qPCR,shRNA\"\\t\"Since all of these TFs are expressed in RN2 cells (Supplemental Fig. 13), we anticipated that these factors would occupy E1CE5 in leukemia as well. Indeed, we detected occupancy of Lmo2, PU.1, and Erg at various subsets of the E1CE5 enhancers in RN2 using ChIP-qPCR (Fig. 5ICK). Additionally, we found that the hematopoietic TFs Cebp and Cebp, both of which are highly expressed in RN2 cells, also occupied the E1CE5 elements (Fig. 5L,M; Supplemental Fig. 13).While expression of several regulators was perturbed upon Brg1 knockdown, Myc was among the most down-regulated genes identified (P < 0.01) (Fig. 3A). Hoxa9, which is a direct target of the MLL-AF9 oncoprotein, was also down-regulated (P < 0.01), albeit to a lesser extent than Myc (Fig. 3A). Gene signatures linked to Myc and Hoxa9 function were globally suppressed following Brg1 knockdown, further confirming an effect on these two pathways (Fig. 3B,C; Supplemental Fig. 5).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1699999', 'E_01_142\\t24288367\\t--\\thg19\\tchr21\\t43781591\\t43782591\\tHuman\\t\"MCF-7,MDA-MB-231,293T\"\\tLow throughput\\t\"qRT-PCR,ChIP\"\\t\"TFF1 has three EREs, and ERE3 at -9.9 kb upstream of transcriptional start site (TSS) is the major ER binding site and considered to be the major enhancer element. Our chromatin immunoprecipitation (ChIP) assay showed that the enhancer region of the TFF1 gene (ERE3,-9.9 kb) was highly occupied by MLL1 and that the recruitment of MLL1 to this region was further enhanced after E2 treatment in MCF-7 cells (Figure 2A).We validated these findings by a ChIP scanning assay,which surveyed from _x0002_10 kb upstream through 5 kb downstream of the TSS of the TFF1 gene. The distal enhancer region of TFF1 was found to be highly occupied by MLL1.\"\\tEnhancer\\tTFF1\\t--\\t\"qRT-PCR,ChIP\"\\t\"When FLAG-tagged MLL1-CX or MLL1-CXPB was over_x0002_expressed in MCF-7 cells by transient transfection, quan_x0002_titative ChIP assays performed with anti-FLAG antibody demonstrated the occupancy by both MLL1-CX and MLL1-CXPB on ERE3 (but not ERE2 or ERE1) of the TFF1 gene, indicating that the 40 kDa MLL1-CX fragment is sufficient for recruitment to the enhancer', 'E_01_143\\t24288367\\t--\\thg19\\tchr21\\t43772291\\t43774291\\tHuman\\t\"MCF-7,MDA-MB-231,293T\"\\tLow throughput\\t\"qRT-PCR,ChIP\"\\t\"TFF1 has three EREs, and ERE3 at -9.9 kb upstream of transcriptional start site (TSS) is the major ER binding site and considered to be the major enhancer element. Our chromatin immunoprecipitation (ChIP) assay showed that the enhancer region of the TFF1 gene (ERE3,-9.9 kb) was highly occupied by MLL1 and that the recruitment of MLL1 to this region was further enhanced after E2 treatment in MCF-7 cells (Figure 2A).We validated these findings by a ChIP scanning assay,which surveyed from _x0002_10 kb upstream through 5 kb downstream of the TSS of the TFF1 gene. The distal enhancer region of TFF1 was found to be highly occupied by MLL1.\"\\tEnhancer\\tTFF1\\t--\\t\"qRT-PCR,ChIP\"\\t\"When FLAG-tagged MLL1-CX or MLL1-CXPB was over_x0002_expressed in MCF-7 cells by transient transfection, quan_x0002_titative ChIP assays performed with anti-FLAG antibody demonstrated the occupancy by both MLL1-CX and MLL1-CXPB on ERE3 (but not ERE2 or ERE1) of the TFF1 gene, indicating that the 40 kDa MLL1-CX fragment is sufficient for recruitment to the enhancer', 'E_01_144\\t24288367\\t--\\thg19\\tchr21\\t43771491\\t43773491\\tHuman\\t\"MCF-7,MDA-MB-231,293T\"\\tLow throughput\\t\"qRT-PCR,ChIP\"\\t\"TFF1 has three EREs, and ERE3 at -9.9 kb upstream of transcriptional start site (TSS) is the major ER binding site and considered to be the major enhancer element. Our chromatin immunoprecipitation (ChIP) assay showed that the enhancer region of the TFF1 gene (ERE3,-9.9 kb) was highly occupied by MLL1 and that the recruitment of MLL1 to this region was further enhanced after E2 treatment in MCF-7 cells (Figure 2A).We validated these findings by a ChIP scanning assay,which surveyed from _x0002_10 kb upstream through 5 kb downstream of the TSS of the TFF1 gene. The distal enhancer region of TFF1 was found to be highly occupied by MLL1.\"\\tEnhancer\\tTFF1\\t--\\t\"qRT-PCR,ChIP\"\\t\"When FLAG-tagged MLL1-CX or MLL1-CXPB was over_x0002_expressed in MCF-7 cells by transient transfection, quan_x0002_titative ChIP assays performed with anti-FLAG antibody demonstrated the occupancy by both MLL1-CX and MLL1-CXPB on ERE3 (but not ERE2 or ERE1) of the TFF1 gene, indicating that the 40 kDa MLL1-CX fragment is sufficient for recruitment to the enhancer', 'E_01_145\\t24321386\\t--\\thg19\\tchr17\\t72860066\\t72860183\\tHuman\\t\"HeLa,H295R,KGN\"\\tLow throughput\\t\"ChIP,Luciferase Reporter Assay,PCR,EMSA\"\\t\"Next, histone modifi- cation in the SF-1-induced cells was examined via ChIP assays, using antibodies against two active enhancer markers, histone H3K27ac and H3K4me2, to determine alterations of chromatin state within the region. The results clearly showed that the signals of these active enhancer markers around the SF-1 binding region increased along with the transduction of SF-1 (Fig. 1D and E), suggesting that the transduction of SF-1 led to the active state of chromatin within the FDXR intronic region.\"\\tEnhancer\\tFDXR\\t--\\t\"ChIP,Luciferase Reporter Assay,PCR\"\\t\"Genome-wide analysis of differentiated cells, using a combination of DNA microarray and promoter tiling array analyses, showed that FDXR is a novel SF-1 target gene. In this study,the transcriptional regulatory mechanism of FDXR was examined in steroidogenic cells.\"\\t\"ADR,ADXR,ANOA\"\\t--\\t--\\t--\\t--\\t--\\t--\\tNR5A1\\t\"AD4BP,ELP,FTZ1,FTZF1,POF7,SF-1,SF1,SPGF8,SRXX4,SRXY3,hSF-1\"\\t\"ChIP,qPCR\"\\t\"Closed and open squares represent Flag-tagged SF-1 (FlagSF-1)- and GFP (Control)-transduced MSCs, respectively; real-time PCR analysis of immunoprecipitated chromatin performed using primers for indicated genomic region; and expressed as a percentage of input.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1507', 'E_01_146\\t24332044\\t--\\thg19\\tchr11\\t110716521\\t110832306\\tHuman\\tB-Cell Lymphoma Cell Lines\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"As predicted, BRD4 load is asymmetrically distributed throughout the genome at enhancer sites. Completely unexpected is the magnitude by which BRD4 load varies among active', 'E_01_147\\t24332044\\t--\\thg19\\tchr3\\t188892648\\t189232665\\tHuman\\tB-Cell Lymphoma Cell Lines\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"As predicted, BRD4 load is asymmetrically distributed throughout the genome at enhancer sites. Completely unexpected is the magnitude by which BRD4 load varies among active', 'E_01_148\\t24332857\\t--\\thg19\\tchr1\\t226085596\\t226087967\\tHuman\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,siRNA\"\\t\"Importantly, the SMAD DNA binding partner FOXH1, a major specifier of ME, is found near TSO elements, and upon fate specificationwe show that TSO is disrupted with subsequent', 'E_01_149\\t24362753\\t--\\thg19\\tchr14\\t77264349\\t77265491\\tHuman\\tHEK-293\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,PCR\"\\t\"Using reporter constructs we identified potential promoter/enhancer elements between the transcription start site and 1142 bp upstream. Using 184 post mortem temporal lobe samples we replicated the reported negative effect of age, and after genotyping tagging SNPs we found one (rs981471) showing a significant correlation with the genes expression and another(rs8014408) showing an interaction with age, the rare C allele being correlated with higher expression and faster decline.\"\\tEnhancer\\tNGB\\t--\\t\"ChIP-seq,Luciferase Reporter Assay,PCR\"\\t\"Real time PCR quantification of the NGB transcript gave consistent amplification from cDNA with little variation within triplicates. It was clear however that the NGB transcript is at low abundance in the adult human temporal lobe, as the detection threshold (Ct) in our qPCR analysis was reached at cycle 26.5 compared to cycle 19 for the ACTB gene at the', 'E_01_150\\t24374176\\t--\\thg19\\tchr15\\t37116298\\t37122452\\tHuman\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,ChIP\"\\t\"Based on these results, we identified sequence d as a MB Meis2 enhancer, and hereafter designate it as MBE. We went on to look for accumulation of histone H3K27 acetylation (H3K27ac) and H3K4 monomethylation (H3K4me1), which demarcate active and potential enhancers, respectively, at MBE by ChIP-seq.\"\\tEnhancer\\tMEIS2\\t--\\t\"ChIP-qPCR,ChIP-seq\"\\t\"In wild-type mice, Meis2 showed distinctive expres_x0002_sion patterns at 11.5 dpc in FB, MB, pharyngeal arches, spinal cord, neural crest, and somites. In Ring1 mutants, however,we found atypical derepression of Meis2 in facial and cephalic mesenchyme, heart primordium, LM, and somatic mesoderm,coupled with its downregulation in FB and MB (Figure 1C). These', 'E_01_151\\t24510239\\t--\\thg19\\tchr13\\t95865319\\t95866199\\tHuman\\t\"Umbilical Vein Endothelial Cell,ESCC\"\\tLow throughput\\tLuciferase Reporter Assay\\t\"Luciferase reporter gene assays verified that the ABCC4 intron 4 sequence, which was annotated as an enhancer element by ENCODE(chr13: 94,663,320-94,664,200), also exhibited enhancer activity in multiple cancer cell types.However,the resulting intensities were much higher in two ESCC cells, KYSE140 and EC0156which suggests that the enhancer activity within this region is rela_x0002_tively specific to ESCC. \"\\tEnhancer\\tABCC4\\t--\\tLuciferase Reporter Assay\\t\"GRAIL analysis highlighted potential cancer-associated func_x0002_tions of the genes from the four CNV regions with frequencies >5%.ABCC4 had strong associations with transport regulation, immunol_x0002_ogy and multiple cancer-related signaling pathways.Furthermore, conditional logistic regression analysis showed that variations in ABCC4copy number were associated with ESCC after adjusting for age, sex and lifestyle.\"\\t\"MOAT-B,MOATB,MRP4\"\\tEsophageal Squamous Cell Carcinoma\\t--\\tD000077277\\t--\\t--\\t--\\tCRE\\tCRE\\tWestern blot\\tWestern blot analysis showed that ABCC4 was effectively depleted following siRNA transfection.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t193677', 'E_01_152\\t24512546\\t--\\thg19\\tchr8\\t128709135\\t128745673\\tHuman\\t\"AN3 CA,RL95-2,HEC-1-B\"\\tLow+High throughput\\t\"ChIP,ChIP-PCR,ChIP-seq,qRT-PCR\"\\t\"Interestingly, we showed that FOXA1 and AR more evidently bound to the MYC enhancer regions as compared to MYC promoter regions. These results could be attributed to other co-regulators involved in this binding process. Since TCF7L2, a protein mediating DNA looping for long-distance interactions of distal en_x0002_hancers and proximal promoters, physically interacts with FOXA1 and AR and mediates the transcription of MYC in breast cancer [19], future investigation will be needed to clarify which co-regulators are involved in', 'E_01_153\\t24512546\\t--\\thg19\\tchr8\\t128392788\\t128433173\\tHuman\\t\"AN3 CA,RL95-2,HEC-1-B\"\\tLow+High throughput\\t\"ChIP,ChIP-PCR,ChIP-seq,qRT-PCR\"\\t\"Interestingly, we showed that FOXA1 and AR more evidently bound to the MYC enhancer regions as compared to MYC promoter regions. These results could be attributed to other co-regulators involved in this binding process. Since TCF7L2, a protein mediating DNA looping for long-distance interactions of distal en_x0002_hancers and proximal promoters, physically interacts with FOXA1 and AR and mediates the transcription of MYC in breast cancer [19], future investigation will be needed to clarify which co-regulators are involved in', 'E_01_154\\t24564208\\t--\\thg19\\tchr4\\t74586223\\t74591223\\tHuman\\tAC16\\tLow+High throughput\\t\"RT-qPCR,ChIP-seq\"\\t\"We have recently developed a computational approach for identifying functional enhancers based on these pat_x0002_terns of transcription in GRO-seq data.Using this approach, we identified 1,146 sites of paired intergenic eRNA production in AC16 cells.Metagene analyses of ChIP-seq data from adult human heart, fetal human heart, and human skeletal muscle myotubes(HSMM) for the 1,146 putative enhancers showed ex_x0002_pected patterns of enrichment for well characterized en_x0002_hancer features, such as p300, H3K4me1,and H3K27ac.Remarkably, the putative enhancers identi_x0002_fied in AC16 cells by GRO-seq match well with enhan_x0002_cer features in the ChIP-seq data from related,but distinct, cell types.\"\\tEnhancer\\tIL8\\t--\\tChIP-seq\\t\"For example, at the proinflammatory gene IL8, we observed two sites of TNF-induced eRNA pro_x0002_duction about 15C20 kb upstream of the TSS (Figure 6A).Both sites showed TNF-dependent accumulation of GRO-seq and Pol II ChIP-seq signals, while only the more proximal of the two sites was bound by NF-B.These enhancers and their associated', 'E_01_155\\t24594601\\t--\\thg19\\tchr8\\t128230172\\t128243000\\tHuman\\t\"HCT 116,RKO\"\\tLow throughput\\t\"qRT-PCR,3C\"\\t\"The enhancer region showed long-range in_x005f_x0002_teraction with MYC (?335 kb from the region),suggesting that the enhancer region could regulate CARLo-5 (?180 kb from', 'E_01_156\\t24662484\\t--\\thg19\\tchr8\\t119677099\\t119681977\\tHuman\\t\"HT-29,HCT 116\"\\tLow throughput\\t\"qRT-PCR,3C,ChIP\"The human 8q24 gene desert contains multiple enhancers that form tissue-specific long-range chromatin loops with the MYC oncogene, but how chromatin looping at the MYC locus is regulated remains poorly understood. Here we demonstrate that a long noncoding RNA (lncRNA), CCAT1-L, is transcribed specifically in human colorectal cancers from a locus 515 kb upstream of MYC. This lncRNA plays a role in MYC transcriptional regulation and promotes long-range chromatin looping. \"\\tSuper-Enhancer\\tMYC\\t3C\\t\"qRT-PCR,ChIP\"\\t\"Importantly, we found that knock_x0002_down of CCAT1-L greatly reduced the transcription of nascent MYC RNA (Figure 2C), further suggesting that CCAT1-L regulates MYC expression at the transcriptional level. Nevertheless, while the reduction on the steady_x0002_state MYC mRNA that we observed was modest, it is possible that this at least partly reflects the known com_x0002_plexity of MYC regulation.\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\tColorectal Cancer\\tDOID:9256\\tD015179\\tThe CCAT1-L locus is located within a strong super-enhancer and is spatially close to MYC\\t\"3C,RT-PCR\"\\t\"Here we demonstrate that a long noncoding RNA (lncRNA), CCAT1-L, is transcribed specifically in human colorectal cancers from a locus 515 kb upstream of MYC. This lncRNA plays a role in MYC transcriptional regulation and promotes long-range chromatin looping. Importantly, the CCAT1-L locus is located within a strong super-enhancer and is spatially close to MYC. Knockdown of CCAT1-L reduced long-range interactions between the MYC promoter and its enhancers.\"\\t\"TCF4,CTCF\"\\t\"E2-2,FECD3,ITF-2,ITF2,PTHS,SEF-2,SEF2,SEF2-1,SEF2-1A,MRD21SEF2-1B,SEF2-1D,TCF-4,bHLHb19\"\\tChIP-seq\\tAnalyses of the available ChIP-seq datasets [39] re_x0002_vealed that TCF4 and CTCF are highly enriched in this 8q24 region in HCT116 cells (Figure 6A). Extensive TCF4 binding in the chromatin region between MYC-515 and MYC-335 was consistent with the notion that this chromatin region is a super-Enhancer that can recruit transcriptional factors.\\trs6983267\\t128413305\\t\"qRT-PCR,3C\"\\t>2KB\\t\\t\\t\\t9068776', 'E_01_157\\t24667089\\t--\\thg19\\tchr8\\t141161900\\t141175436\\tHuman\\tLymphoblastoid Cell Lines\\tLow throughput\\t\"RT-PCR,ChIP,Luciferase Reporter Assay,3C,DNaseI-seq\"\\t\"Using 3C, we identify mutually exclusive approximately 58 and 500 kb chromatin loops in adult frontal cortex between a novel brain-specific enhancer, marked by H3K4me1 and H3K27ac, with the KCNK9 and PEG13 promoters which we propose regulates brain-specific expression.\"\\tEnhancer\\tKCNK9\\t3C\\t\"RT-PCR,ChIP\"\\t\"Utilizing allele-specific RT-PCR, bisulphite sequencing, chromatin immunoprecipitation and chromosome conformation capture (3C) we show the reciprocal expression of the novel, paternally expressed, PEG13 non-coding RNA and maternally expressed KCNK9 genes in brain, and the biallelic expression of flanking transcripts in a range of tissues.\"\\t\"K2p9.1,KT3.2,TASK-3,TASK3\"\\tIntellectual Disability\\tDOID:1059\\tD008607\\t--\\t--\\t--\\tCTCF\\tMRD21\\t3C-qPCR\\t3C-qPCR assays were performed on cerebellar samples and interaction frequencies were determined between a constant HindIII site located within the unmethylated CTCF-cohesin binding site within the KCNK9 promoter and other HindIII sites throughout the locus. We identified strong interactions between the KCNK9 promoter constant fragment with the PEG13-DMR and the CTCF-cohesin site in the enhancer region located within intron 17 of TRAPPC9. \\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t555588', 'E_01_158\\t24667606\\t--\\thg19\\tchr17\\t60983452\\t60984775\\tHuman\\t\"HEK293FT,FLP-IN T-REX 293 CELLS\"\\tLow throughput\\t\"qRT-PCR,ChIP,Luciferase Reporter Assay\"\\t\"Here we tested whether dTALEs can be used to target genes whose expression is controlled by Wnt/-catenin signaling.TALE DNA binding domains were engineered to recognize sequences adjacent to Wnt responsive enhancer elements (WREs) that control expression of axis inhibition protein 2 (AXIN2) and c-MYC (MYC).The TALE-SIDs repressed luciferase reporter activity, bound their genomic target sites, and repressed AXIN2 and MYC expression in HEK293 cells. We generated a novel HEK293 cell line to determine whether the TALE-SIDs could function downstream of oncogenic Wnt/-catenin signaling. \"\\tEnhancer\\tAXIN2\\t--\\t\"qRT-PCR,ChIP,Luciferase Reporter Assay\"\"The AXIN2 luciferase reporter plasmid was generated by PCR-amplifying a 787-bp fragment of the AXIN2 gene from genomic HCT116 DNA that includes the TALE binding sites. The', 'E_01_159\\t24667606\\t--\\thg19\\tchr17\\t60986269\\t60989232\\tHuman\\t\"HEK293FT,FLP-IN T-REX 293 CELLS\"\\tLow throughput\\t\"qRT-PCR,ChIP,Luciferase Reporter Assay\"\\t\"Here we tested whether dTALEs can be used to target genes whose expression is controlled by Wnt/-catenin signaling.TALE DNA binding domains were engineered to recognize sequences adjacent to Wnt responsive enhancer elements (WREs) that control expression of axis inhibition protein 2 (AXIN2) and c-MYC (MYC).The TALE-SIDs repressed luciferase reporter activity, bound their genomic target sites, and repressed AXIN2 and MYC expression in HEK293 cells. We generated a novel HEK293 cell line to determine whether the TALE-SIDs could function downstream of oncogenic Wnt/-catenin signaling. \"\\tEnhancer\\tAXIN2\\t--\\t\"qRT-PCR,ChIP,Luciferase Reporter Assay\"\"The AXIN2 luciferase reporter plasmid was generated by PCR-amplifying a 787-bp fragment of the AXIN2 gene from genomic HCT116 DNA that includes the TALE binding sites. The', 'E_01_160\\t24667606\\t--\\thg19\\tchr8\\t128816178\\t128817335\\tHuman\\t\"HEK293FT,FLP-IN T-REX 293 CELLS\"\\tLow throughput\\t\"qRT-PCR,ChIP,Luciferase Reporter Assay\"\\t\"Here we tested whether dTALEs can be used to target genes whose expression is controlled by Wnt/-catenin signaling.TALE DNA binding domains were engineered to recognize sequences adjacent to Wnt responsive enhancer elements (WREs) that control expression of axis inhibition protein 2 (AXIN2) and c-MYC (MYC).The TALE-SIDs repressed luciferase reporter activity, bound their genomic target sites, and repressed AXIN2 and MYC expression in HEK293 cells. We generated a novel HEK293 cell line to determine whether the TALE-SIDs could function downstream of oncogenic Wnt/-catenin signaling. \"\\tEnhancer\\tMYC\\t--\\t\"qRT-PCR,ChIP,Luciferase Reporter Assay\"\"The MYC luciferase transgene, which contains approximately 8.6-kb of genomic DNA that encompasses MYC, was generated stepwise with two PCR fragments amplified from a bacterial artificial chromosome (BAC) that harbors human MYC. The firefly luciferase gene was inserted in-frame into the second exon of MYC. \"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\tColorectal Cancer\\tDOID:9256\\tD015179\\t--\\t--\\t--\\tSID\\tSID\\t\"ChIP,qRT-PCR\"\\t\"using ChIP assays and qRT-PCR analysis, we found that TALE3-SID bound its target sequence and repressed MYC transcription in transiently transfected HEK293 cells.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t68443', 'E_01_161\\t24671955\\t--\\thg19\\tchr5\\t139970046\\t139972237\\tHuman\\tPeripheral Blood Monocytes\\tLow+High throughput\\t\"RT-PCR,ChIP,Luciferase Reporter Assay,ChIP-seq\"\\t\"To extend the analysis of regulatory sites to putative enhancers, we also carried out ChIP sequencing for 2 histone marks, namely H3K4me1 and H3K27ac,that were previously associated with enhancers.30-34 H3K27 is a major substrate for the coactivators p300 and CBP and its acetylation marks active enhancers, whereas H3K4me1 is generally associated with distal regulatory elements, including poised enhancers.\"\\tEnhancer\\tCD14\\t--\\t\"RT-PCR,Luciferase Reporter Assay\"\\t\"On this basis, we sought an explanation for the differential ex_x0002_pression of CD14 in classical monocytes. Studies in transgenic mice by Zhang and coworkers established that a region of 80 kb surrounding the human CD14 gene is sufficient to direct its monocyte specific expression,42 whereas smaller constructs recapitulated human liver expression but failed to direct monocyte-specific expression.The genomic interval downstream of the CD14 gene contained a number of H3K27ac-marked sites specific for classical,CD14-expressing monocytes.These sites frequently overlapped with bidirectional enhancers identified using the FANTOM5 expression atlas44 as well as binding sites for PU.1 or C/EBPb in total monocytes,18 which are key factors in establishing distal regulatory sites in these cells.\"\\tCD14\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t40170', 'E_01_162\\t24671955\\t--\\thg19\\tchr5\\t139974521\\t139975981\\tHuman\\tPeripheral Blood Monocytes\\tLow+High throughput\\t\"RT-PCR,ChIP,Luciferase Reporter Assay,ChIP-seq\"\\t\"To extend the analysis of regulatory sites to putative enhancers, we also carried out ChIP sequencing for 2 histone marks, namely H3K4me1 and H3K27ac,that were previously associated with enhancers.30-34 H3K27 is a major substrate for the coactivators p300 and CBP and its acetylation marks active enhancers, whereas H3K4me1 is generally associated with distal regulatory elements, including poised enhancers.\"\\tEnhancer\\tCD14\\t--\\t\"RT-PCR,Luciferase Reporter Assay\"\\t\"On this basis, we sought an explanation for the differential ex_x0002_pression of CD14 in classical monocytes. Studies in transgenic mice by Zhang and coworkers established that a region of 80 kb surrounding the human CD14 gene is sufficient to direct its monocyte specific expression,42 whereas smaller constructs recapitulated human liver expression but failed to direct monocyte-specific expression.The genomic interval downstream of the CD14 gene contained a number of H3K27ac-marked sites specific for classical,CD14-expressing monocytes.These sites frequently overlapped with bidirectional enhancers identified using the FANTOM5 expression atlas44 as well as binding sites for PU.1 or C/EBPb in total monocytes,18 which are key factors in establishing distal regulatory sites in these cells.\"\\tCD14\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t36061', 'E_01_163\\t24671955\\t--\\thg19\\tchr5\\t139985479\\t139987671\\tHuman\\tPeripheral Blood Monocytes\\tLow+High throughput\\t\"RT-PCR,ChIP,Luciferase Reporter Assay,ChIP-seq\"\\t\"To extend the analysis of regulatory sites to putative enhancers, we also carried out ChIP sequencing for 2 histone marks, namely H3K4me1 and H3K27ac,that were previously associated with enhancers.30-34 H3K27 is a major substrate for the coactivators p300 and CBP and its acetylation marks active enhancers, whereas H3K4me1 is generally associated with distal regulatory elements, including poised enhancers.\"\\tEnhancer\\tCD14\\t--\\t\"RT-PCR,Luciferase Reporter Assay\"\\t\"On this basis, we sought an explanation for the differential ex_x0002_pression of CD14 in classical monocytes. Studies in transgenic mice by Zhang and coworkers established that a region of 80 kb surrounding the human CD14 gene is sufficient to direct its monocyte specific expression,42 whereas smaller constructs recapitulated human liver expression but failed to direct monocyte-specific expression.The genomic interval downstream of the CD14 gene contained a number of H3K27ac-marked sites specific for classical,CD14-expressing monocytes.These sites frequently overlapped with bidirectional enhancers identified using the FANTOM5 expression atlas44 as well as binding sites for PU.1 or C/EBPb in total monocytes,18 which are key factors in establishing distal regulatory sites in these cells.\"\\tCD14\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t24737', 'E_01_164\\t24671955\\t--\\thg19\\tchr5\\t139987854\\t139989680\\tHuman\\tPeripheral Blood Monocytes\\tLow+High throughput\\t\"RT-PCR,ChIP,Luciferase Reporter Assay,ChIP-seq\"\\t\"To extend the analysis of regulatory sites to putative enhancers, we also carried out ChIP sequencing for 2 histone marks, namely H3K4me1 and H3K27ac,that were previously associated with enhancers.30-34 H3K27 is a major substrate for the coactivators p300 and CBP and its acetylation marks active enhancers, whereas H3K4me1 is generally associated with distal regulatory elements, including poised enhancers.\"\\tEnhancer\\tCD14\\t--\\t\"RT-PCR,Luciferase Reporter Assay\"\\t\"On this basis, we sought an explanation for the differential ex_x0002_pression of CD14 in classical monocytes. Studies in transgenic mice by Zhang and coworkers established that a region of 80 kb surrounding the human CD14 gene is sufficient to direct its monocyte specific expression,42 whereas smaller constructs recapitulated human liver expression but failed to direct monocyte-specific expression.The genomic interval downstream of the CD14 gene contained a number of H3K27ac-marked sites specific for classical,CD14-expressing monocytes.These sites frequently overlapped with bidirectional enhancers identified using the FANTOM5 expression atlas44 as well as binding sites for PU.1 or C/EBPb in total monocytes,18 which are key factors in establishing distal regulatory sites in these cells.\"\\tCD14\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t22545', 'E_01_165\\t24671955\\t--\\thg19\\tchr5\\t139990822\\t139992831\\tHuman\\tPeripheral Blood Monocytes\\tLow+High throughput\\t\"RT-PCR,ChIP,Luciferase Reporter Assay,ChIP-seq\"\\t\"To extend the analysis of regulatory sites to putative enhancers, we also carried out ChIP sequencing for 2 histone marks, namely H3K4me1 and H3K27ac,that were previously associated with enhancers.30-34 H3K27 is a major substrate for the coactivators p300 and CBP and its acetylation marks active enhancers, whereas H3K4me1 is generally associated with distal regulatory elements, including poised enhancers.\"\\tEnhancer\\tCD14\\t--\\t\"RT-PCR,Luciferase Reporter Assay\"\\t\"On this basis, we sought an explanation for the differential ex_x0002_pression of CD14 in classical monocytes. Studies in transgenic mice by Zhang and coworkers established that a region of 80 kb surrounding the human CD14 gene is sufficient to direct its monocyte specific expression,42 whereas smaller constructs recapitulated human liver expression but failed to direct monocyte-specific expression.The genomic interval downstream of the CD14 gene contained a number of H3K27ac-marked sites specific for classical,CD14-expressing monocytes.These sites frequently overlapped with bidirectional enhancers identified using the FANTOM5 expression atlas44 as well as binding sites for PU.1 or C/EBPb in total monocytes,18 which are key factors in establishing distal regulatory sites in these cells.\"\\tCD14\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t19485', 'E_01_166\\t24671955\\t--\\thg19\\tchr5\\t140009726\\t140011735\\tHuman\\tPeripheral Blood Monocytes\\tLow+High throughput\\t\"RT-PCR,ChIP,Luciferase Reporter Assay,ChIP-seq\"\\t\"To extend the analysis of regulatory sites to putative enhancers, we also carried out ChIP sequencing for 2 histone marks, namely H3K4me1 and H3K27ac,that were previously associated with enhancers.30-34 H3K27 is a major substrate for the coactivators p300 and CBP and its acetylation marks active enhancers, whereas H3K4me1 is generally associated with distal regulatory elements, including poised enhancers.\"\\tEnhancer\\tCD14\\t--\\t\"RT-PCR,Luciferase Reporter Assay\"\\t\"On this basis, we sought an explanation for the differential ex_x0002_pression of CD14 in classical monocytes. Studies in transgenic mice by Zhang and coworkers established that a region of 80 kb surrounding the human CD14 gene is sufficient to direct its monocyte specific expression,42 whereas smaller constructs recapitulated human liver expression but failed to direct monocyte-specific expression.The genomic interval downstream of the CD14 gene contained a number of H3K27ac-marked sites specific for classical,CD14-expressing monocytes.These sites frequently overlapped with bidirectional enhancers identified using the FANTOM5 expression atlas44 as well as binding sites for PU.1 or C/EBPb in total monocytes,18 which are key factors in establishing distal regulatory sites in these cells.\"\\tCD14\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t581', 'E_01_167\\t24703844\\t--\\thg19\\tchr1\\t9996966\\t9997243\\tHuman\\t\"GM12878,CD4+ T Cell\"\\tLow+High throughput\\t\"Hi-C,4C,Luciferase Reporter Assay,ChIP-seq\"\\t\"By hierarchical clustering of the histone modification profiles,Alu elements, but not the other three transposons, cluster together with RefSeq genes, putative enhancers,and DNase I hypersensitive sites (DHSs).Although we removed TSS -10 and +5 kb regions from DHS,the DHS regions may still contain not only enhancers but also some promoters,as shown by the high level of H3K4me3,a modification exclusively localized to promoters. H3K4me3 is excluded on all four transposons,consistent with their underrepresentation at promoter regions.\"\\tEnhancer\\tLZIC\\t--\\tLuciferase Reporter Assay\\t\"Two examples of Alus with specific P300, H3K27ac, and H3K4me1 peaks in one tissue but not in other tissues are shown in Figures 2D and 2E. Because P300, H3K27ac, and H3K4me1 peaks do not coexist on other Alus and control re_x0002_peats in the same region, the high active modification levels are unlikely to be conferred by genomic background.\"\\tLZIC\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t14935', 'E_01_168\\t24703844\\t--\\thg19\\tchr3\\t11249688\\t11250280\\tHuman\\t\"GM12878,CD4+ T Cell\"\\tLow+High throughput\\t\"Hi-C,4C,Luciferase Reporter Assay,ChIP-seq\"\\t\"By hierarchical clustering of the histone modification profiles,Alu elements, but not the other three transposons, cluster together with RefSeq genes, putative enhancers,and DNase I hypersensitive sites (DHSs).Although we removed TSS -10 and +5 kb regions from DHS,the DHS regions may still contain not only enhancers but also some promoters,as shown by the high level of H3K4me3,a modification exclusively localized to promoters. H3K4me3 is excluded on all four transposons,consistent with their underrepresentation at promoter regions.\"\\tEnhancer\\tHRH1\\t--\\tLuciferase Reporter Assay\\t\"Two examples of Alus with specific P300, H3K27ac, and H3K4me1 peaks in one tissue but not in other tissues are shown in Figures 2D and 2E. Because P300, H3K27ac, and H3K4me1 peaks do not coexist on other Alus and control re_x0002_peats in the same region, the high active modification levels are unlikely to be conferred by genomic background.\"\\t\"H1-R,H1R,HH1R,hisH1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t71206', 'E_01_169\\t24703906\\t--\\thg19\\tchr3\\t184276287\\t184278287\\tHuman\\t\"Crc Cell,HT-29\"\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,qRT-PCR\"\\t\"Importantly, the stimulatory capacity of the ?2.3-kb ECR differed widely among the CRC cell lines and was most pronounced in LS174T cells with highest levels of endogenous EPHB3. Thus, the EPHB3 ?2.3-kb ECR functions as a cell type-specific transcriptional enhancer. Consistent with its evolutionary conservation, also the corresponding region from the mouse EphB3 gene has enhancer properties and exhibits a remarkably similar cell-type specificity in CRC cell lines.We probed these features by chromatin immunoprecipitation (ChIP) and formaldehyde_x0002_assisted isolation of regulatory elements (FAIRE).\"\\tEnhancer\\tEPHB3\\t--\\t\"ChIP,qRT-PCR\"\\t\"Next, we performed ChIP experiments to investigate the distri_x0002_bution of the histone modifications H3K4me1 and H3K27ac,the acetyltransferase p300, and the Wnt pathway effector TCF7L2 at the EPHB3 locus.Significantly,its cell type-specific activity matches the expression of the endogenous EPHB3 gene in CRC cell lines, suggesting that differences in enhancer function underlie differential EPHB3 expression in CRC.\"\\t\"EK2,ETK2,HEK2,TYRO6\"\\t--\\t--\\t--\\t--\\t--\\t--\\tASCL2\\t\"ASH2,HASH2,MASH2,bHLHa45\"\\tLuciferase Reporter Assay\\twe generated luciferase-reporter constructs with point mutations in the TBE and the binding sites for RBPJ and ETS factors.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2299', 'E_01_170\\t24778216\\t--\\thg19\\tchr19\\t51358995\\t51369041\\tHuman\\t\"LNCaP,VCaP\"\\tLow throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,3C\"\\t\"Because the induction of eRNA is strongly correlated with the enhancer looping to target gene promotersit is possible that the KLK3 enhancer element is able to interact with the KLK2 promoter. To address this, we used the chromosome conformation capture (3C) assay to study the spatial interaction of the KLK3/2 loci. It should be noted that the KLK2 promoter contains an ARE I that shares over 80% sequence homology with the ARE I in the KLK3 promoter.\"\\tEnhancer\\tKLK3\\t3C\\tRT-PCR\\t\"In fact, the transcription of KLK3 eRNA (KLK3e) was demon_x0002_strated by GRO-seq data (10) and confirmed by our reverse transcriptionCquantitative PCR (RT-qPCR) analyses in andro_x0002_gen-dependent LNCaP and VCaP, and androgen-independent LNCaP-abl cells (21).\"\\t\"APS,KLK2A1,PSA,hK3\"\\tProstate Cancer\\tDOID:10283\\tD011471\\tKLK3e carries the core enhancer element derived from the androgen response element III (ARE III).KLK3e facilitates the spatial interaction of the KLK3 Enhancer and the KLK2 promoter and enhances long-distance KLK2 transcriptional activation.\\t\"RT-PCR,3C,RIP,3C-qPCR\"\\t\"Kallikrein-related peptidase 3 (KLK3), which codes for prostate-specific antigen (PSA), is a well-known AR-regulated gene and its upstream enhancers produce bi-', 'E_01_171\\t24788237\\t--\\thg19\\tchr12\\t54357811\\t54358445\\tHuman\\t\"KYSE-30,KESY-150,ESCC\"\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"Because the ESCC susceptibility SNP rs920778 is located within the HOTAIR intron 2 and previous H3K4me1 and H3K4me3 modi_x0002_fication data suggest that there might be a potential enhancer in this intron,we therefore examined the enhancer activity of this region by a set of luciferase reporter gene constructs in human ESCC KYSE30 and KYSE150 cells These results indicated that there may be a negative regulatory element between +1463bp and +1718bpfrom the transcriptional start site and the core region of this intronic enhancer might exist between +1719bp and +2353bp from the tran_x0002_scriptional start?site.\"\\tEnhancer\\tHOTAIR \\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"We hypothesized that the functional single nucleotide polymorphisms (SNP) in HOTAIR may affect HOTAIR expression and/or its function and, thus, ESCC risk. Therefore, we examined the association between three haplotype_x0002_tagging SNPs (htSNP) across the whole HOTAIR locus and ESCC risk as well as the functional relevance of an ESCC susceptibility', 'E_01_172\\t24797517\\t--\\thg19\\tchr6\\t36632237\\t36634237\\tHuman\\tHuman Diploid BJ Fibroblast Cell Lines\\tLow+High throughput\\t\"ChIP-qPCR,ChIP,ChIP-seq\"\\t\"On the other hand,only 4% are positive for H3K4me3, indicating that these sites are likely to represent regulatory enhancer elements.Furthermore, we validated by ChIP-qPCR that p53 and JMJD3 display an IR responsive recruitment to several putative enhancer elements.We confirmed that these sites possess features of active enhancers; they are DNase I positiveshow binding of p300 and have enrichment of H3K4me1 and H3K27ac but not H3K4me3 and H3K27me3.We also observed that exposure to IR correlated with an increased binding of p300 and subsequently acetylation of H3K27,which is in agreement with data demonstrating that p53 interacts with p300.\"\\tEnhancer\\tCDKN1A\\t--\\tChIP-qPCR\\t\"Examples of p53 (before and after IR treatment) and JMJD3 (before and after IR treatment), H3K4me3 and DNase I-seq tracks at two putative enhancer elements located 11 kb upstream of CDKN1A (upper panel) or 19 kb upstream of GML (lower panel).Corresponding ChIP-qPCR validations of the binding of p53, JMJD3 and p300 as well as the levels of histone modifications H3K4me1, H3K4me3, H3K27ac and H3K27me3 at the two distal binding sites listed above.\"\\t\"CAP20,CDKN1,CIP1,MDA-6,P21,SDI1,WAF1,p21CIP1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tTP53\\t\"BCC7,BMFS5,LFS1,P53,TRP53\"\\tChIP-seq\\t\"To understand if JMJD3 and p53 also display genomic co_x0002_localization, we performed global mapping of JMJD3 and p53 binding sites in telomerase-immortalized human BJ diploid fibroblasts by chromatin immunoprecipitation followed by se_x0002_quencing (ChIP-seq).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t10999', 'E_01_173\\t24797517\\t--\\thg19\\tchr8\\t143896217\\t143898217\\tHuman\\tHuman Diploid BJ Fibroblast Cell Lines\\tLow+High throughput\\t\"ChIP-qPCR,ChIP,ChIP-seq\"\\t\"On the other hand,only 4% are positive for H3K4me3, indicating that these sites are likely to represent regulatory enhancer elements.Furthermore, we validated by ChIP-qPCR that p53 and JMJD3 display an IR responsive recruitment to several putative enhancer elements.We confirmed that these sites possess features of active enhancers; they are DNase I positiveshow binding of p300 and have enrichment of H3K4me1 and H3K27ac but not H3K4me3 and H3K27me3.We also observed that exposure to IR correlated with an increased binding of p300 and subsequently acetylation of H3K27,which is in agreement with data demonstrating that p53 interacts with p300.\"\\tEnhancer\\tGML\\t--\\tChIP-qPCR\\t\"Examples of p53 (before and after IR treatment) and JMJD3 (before and after IR treatment), H3K4me3 and DNase I-seq tracks at two putative enhancer elements located 11 kb upstream of CDKN1A (upper panel) or 19 kb upstream of GML (lower panel).Corresponding ChIP-qPCR validations of the binding of p53, JMJD3 and p300 as well as the levels of histone modifications H3K4me1, H3K4me3, H3K27ac and H3K27me3 at the two distal binding sites listed above.\"\\tLY6D\\t--\\t--\\t--\\t--\\t--\\t--\\tTP53\\t\"BCC7,BMFS5,LFS1,P53,TRP53\"\\tChIP-seq\"To understand if JMJD3 and p53 also display genomic co_x0002_localization, we performed global mapping of JMJD3 and p53 binding sites in telomerase-immortalized human BJ diploid fibroblasts by chromatin immunoprecipitation followed by se_x0002_quencing (ChIP-seq).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t18999', 'E_01_174\\t24842713\\t--\\thg19\\tchr16\\t86212040\\t86271919\\tHuman\\tLung\\tLow+High throughput\\t\"ChIP-seq,qRT-PCR,ChIP-qPCR\"\\t\"Using chromatin immunoprecipitation with high-throughput sequencing (ChIP-seq) of histone H3K27 acetylation (H3K27ac), a mark of active enhancers,we first identified enhancer regions in MYCN-amplified Kelly and nonamplified SH-SY5Y cells.\"\\tSuper-Enhancer\\tFOXF1\\t--\\tshRNA\\t\"Here we investigate whether inhibition of transcriptional CDKs can be exploited to disrupt aberrant MYC-driven transcription, using the deregulated expression of MYCN as a model.', 'E_01_175\\t24916375\\t--\\thg19\\tchr9\\t124638991\\t124641160\\tHuman\\tLP89\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"We profiled the chromatin structure of the BNC2 region to identify potential regulatory elements using chromatin immune-precipitation (ChIP) of H3K27Ac (an active-enhancer mark) in combination with next generation sequencing (ChIP-seq) and ENCODE data. We provide evidence that a region upstream from the canonical BNC2 promoter acts as an enhancer regulating the expression of BNC2, and this transcriptional regulation is dependent on the allelic status of an LD partner of the pigmentation-associated SNP rs10756819.\"\\tEnhancer\\tBNC2\\t--\\t\"ChIP-seq,RT-qPCR\"\\t\"We confirmed this finding by RTCqPCR analysis ofBNC2-expression levels atthree different loca_x0002_tions within the gene, using one primer set targeting mRNA upstream of the alternative promoter, and two primer sets targeting the mRNA downstream of the alternative promoter. \"\\tBSN2\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t108230576', 'E_01_176\\t24997565\\t--\\thg19\\tchr3\\t32993199\\t32996133\\tHuman\\tT Cell\\tLow+High throughput\\t\"ChIP-seq,PCR\"\\t\"We performed 120 high-throughput whole-genome ChIP-seq assays, which generated ~2.8  109 reads covering~1  1011 bases across three cell types (Supplementary Table 1), and computationally analyzed this data set to identify enhancers associated with CD4+ memory T cell differentiation as well as asthma.\"\\tEnhancer\\tCCR4\\t--\\tChIP-seq\\t\"We found clear evidence of cell typeCspecific cis_x0002_regulatory DNA elements in noncoding sequences: for instance, compared to naive T cells,the TH2 cell subset displayed extensive enrichment of H3K4me2 at promoters and cis_x0002_regulatory regions in the extended CCR4 and CCR6 loci.\"\\t\"CC-CKR-4,CD194,CKR4,CMKBR4,ChemR13,HGCN:14099,K5-5\"\\tAsthma\\tDOID:2841\\tD001249\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1601', 'E_01_177\\t24997565\\t--\\thg19\\tchr3\\t33000399\\t33003599\\tHuman\\tT Cell\\tLow+High throughput\\t\"ChIP-seq,PCR\"\\t\"We performed 120 high-throughput whole-genome ChIP-seq assays, which generated ~2.8  109 reads covering~1  1011 bases across three cell types (Supplementary Table 1), and computationally analyzed this data set to identify enhancers associated with CD4+ memory T cell differentiation as well as asthma.\"\\tEnhancer\\tCCR4\\t--\\tChIP-seq\\t\"We found clear evidence of cell typeCspecific cis_x0002_regulatory DNA elements in noncoding sequences: for instance, compared to naive T cells,the TH2 cell subset displayed extensive enrichment of H3K4me2 at promoters and cis_x0002_regulatory regions in the extended CCR4 and CCR6 loci.\"\\t\"CC-CKR-4,CD194,CKR4,CMKBR4,ChemR13,HGCN:14099,K5-5\"\\tAsthma\\tDOID:2841\\tD001249\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t8934', 'E_01_178\\t24997565\\t--\\thg19\\tchr3\\t34181595\\t34184395\\tHuman\\tT Cell\\tLow+High throughput\\t\"ChIP-seq,PCR\"\\t\"We performed 120 high-throughput whole-genome ChIP-seq assays, which generated ~2.8  109 reads covering~1  1011 bases across three cell types (Supplementary Table 1), and computationally analyzed this data set to identify enhancers associated with CD4+ memory T cell differentiation as well as asthma.\"\\tEnhancer\\tCCR4\\t--\\tChIP-seq\\t\"We found clear evidence of cell typeCspecific cis_x0002_regulatory DNA elements in noncoding sequences: for instance, compared to naive T cells,the TH2 cell subset displayed extensive enrichment of H3K4me2 at promoters and cis_x0002_regulatory regions in the extended CCR4 and CCR6 loci.\"\\t\"CC-CKR-4,CD194,CKR4,CMKBR4,ChemR13,HGCN:14099,K5-5\"\\tAsthma\\tDOID:2841\\tD001249\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1189930', 'E_01_179\\t24997565\\t--\\thg19\\tchr3\\t34201595\\t34204662\\tHuman\\tT Cell\\tLow+High throughput\\t\"ChIP-seq,PCR\"\\t\"We performed 120 high-throughput whole-genome ChIP-seq assays, which generated ~2.8  109 reads covering~1  1011 bases across three cell types (Supplementary Table 1), and computationally analyzed this data set to identify enhancers associated with CD4+ memory T cell differentiation as well as asthma.\"\\tEnhancer\\tCCR4\\t--\\tChIP-seq\\t\"We found clear evidence of cell typeCspecific cis_x0002_regulatory DNA elements in noncoding sequences: for instance, compared to naive T cells,the TH2 cell subset displayed extensive enrichment of H3K4me2 at promoters and cis_x0002_regulatory regions in the extended CCR4 and CCR6 loci.\"\\t\"CC-CKR-4,CD194,CKR4,CMKBR4,ChemR13,HGCN:14099,K5-5\"\\tAsthma\\tDOID:2841\\tD001249\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1210064', 'E_01_180\\t25081415\\t--\\thg19\\tchr12\\t48368875\\t48369590\\tHuman\\t\"10T1/2,MC615 Cells,MCF-7\"\\tLow throughput\\t\"RT-PCR,ChIP,Luciferase Reporter Assay,EMSA\"\\t\"Thus, several binding sites of the intronic enhancer have been shown to interact with transcription factors, such as Sox-9,Sox-6, and L-Sox-5, acting cooperatively and consequently activating expression of COL2A1 in 10T1/2 and MC615 cells.Therefore, in order to validate our in vitro hypoth_x0002_esis and to determine whether these transcription factors were effectively able to bind in vivo to Col2a1-specific enhancer,we performed ChIP assays in primary chondrocytes using primers mapping the +2,383/+2,611-bp region of COL2A1 first intron. This region includes a Sox-9 cis-element (+2,384/+2,404 bp), one HMG binding site (+2,388/+2,394 bp), and one Sp1/Sp3 binding sequence (+2,440/+2,485 bp) but does not harbor ERE cis-sequence.\"\\tEnhancer\\tCOL2A1\\t--\\t\"ChIP,EMSA\"\\t\"Altogether, these EMSA assays indicate that 17-E2 and hER66 enhance transcription of COL2A1 by increasing the binding activity of Sp1 and to a lesser extent of Sp3, not only', 'E_01_181\\t25194570\\t--\\thg19\\tchr8\\t130179643\\t130181490\\tHuman\\t\"HPB-ALL,JURKAT,T-acute lymphoblastic leukemia (T-ALL)\"\\tLow+High throughput\\t\"ChIP-seq,ChIP,Luciferase Reporter Assay,3C,PCR\"\\t\"This highly conserved regulatory element, hereby named N-Me for NOTCH MYC enhancer, is located within a broad super-enhancer region +1.47 Mb from the MYC transcription initiating site,interacts with the MYC proximal promoter and induces orientation-independent MYC expression in reporter assays.Moreover, luciferase reporter assays showed strong, orientationindependent activation of reporter constructs containing this enhancer in association with a ?2.5 kb MYC proximal promoter21 in JURKAT T-ALL cells,which express high levels of constitutively active NOTCH1 protein 22, but not in Daudi and Raji B-cell lineage cells.\"\\tEnhancerMYC\\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"Consistent with this hypothesis, chromatin configuration 3C analysis of the MYC locus demonstrated the association of this enhancer with proximal regulatory sequences in the MYC promoter.Moreover, histological analysis of multiple tissues including those in which NOTCH signaling plays important developmental roles such as breast epithelium, skin, and intestine showed no alterations (Supplementary Fig. 7) and RT-PCR analysis showed no effects of N_x0002_Me deletion in Myc expression.\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\tT-Cell Acute Lymphoblastic Leukemia\\tDOID:5602\\tD054218\\t--\\t--\\t--\\tNOTCH1\\t\"AOS5,AOVD1,TAN1,hN1\"\\t\"ChIP,ChIP-seq\"\\t\"Interestingly, chromatin immunoprecipitation (ChIP) followed by next generation sequencing (ChIP-seq) analysis of NOTCH1 chromatin binding sites in HPB-ALL T-ALL cells revealed a prominent 1 kb NOTCH1 peak in chromosome 8q24 located within the common 40 Kb segment duplicated in all these eight leukemia cases. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1432253', 'E_01_182\\t25214635\\t--\\thg19\\tchr10\\t6064657\\t6065397\\tHuman\\tT Cell\\tLow throughput\\t\"EMSA,Luciferase Reporter Assay\"\\t\"We first tested for enhancer activity by introducing ~200 bp fragments centered around each SNP (Fig. 6A) into a luciferase reporter vector, driven by SV40 or IL2RA minimal promoters, and transfecting them into Jurkat T cells.The higher enhancer function of the T allele was dependent on activation of the Jurkat cells (Fig. 6C), suggesting that enhancer activity is boosted upon cell activation by interactions affected by the single base change at rs12251836.\"\\tEnhancer\\tIL2RA\\t--\\tGWAS\\t\"Together, these results point to functional effects of at least two independent associations in the IL2RA locus, which may have balancing effects. We surmise that the associated variant rs12251836 is mainly active in acutely triggered T effector cells, but not in Treg cells, since IL2RA mRNA in unstimulated CD4+ T cells would predominantly come from CD25hi Treg cells, and this association was not apparent in unstimulated cells. \"\\t\"CD25,IDDM10,IL2R,IMD41,TCGFR,p55\"\\tAutoimmune Diseases\\t--\\tD001327\\t--\\t--\\t--\\tYY1\\t\"DELTA,GADEVS,INO80S,NF-E1,UCRBP,YIN-YANG-1\"\\tEMSA\\tThe allele-specific binding of YY1 was also confirmed by EMSA.\\trs12251836\\t6091281\\t\"Luciferase Reporter Assay,EMSA\"\\t>2KB\\t\\t\\t\\t12371', 'E_01_183\\t25248036\\t--\\thg19\\tchr2\\t217918268\\t217921910\\tHuman\\tBreast Cancer Cell\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq,3C,PCR\"\\t\"The strongest candidate for causality, SNP rs4442975, flanks a transcriptional enhancer that physically interacts with the promoter of IGFBP5. Furthermore, we demonstrate that rs4442975 is associated with allele_x0002_specific FOXA1 binding, chromatin looping and IGFBP5 expression. Our data suggest that the g-allele of rs4442975 confers increased breast cancer susceptibility through reduced IGFBP5 expression.\"\\tEnhancer\\tIGFBP5\\t3C\\t--\\t\"This SNP flanks a transcriptional enhancer that physically interacts with the promoter of IGFBP5 (encoding insulin-like growth factor-binding protein 5) and displays allele-specific gene expression, FOXA1 binding and chromatin looping. Evidence suggests that the g-allele confers increased breast cancer susceptibility through relative downregulation of IGFBP5, a gene with known roles in breast cell biology.\"\\tIBP5\\tBreast Cancer\\tDOID:1612\\tD001943\\t--\\t--\\t--\\tFOXA1\\t\"HNF3A,TCF3A\"\\t\"qPCR,ChIP-seq\"\\t\"To assess occupancy of FOXA1 in vivo, we conducted ChIP followed by allele-specific quantitative PCR (qPCR) in the heterozygous BT474 breast cancer cell line.\"\\trs4442975\\t217920769\\t\"ChIP-seq,ChIP,qPCR,ChIP-qPCR,3C\"\\t>2KB\\t\\t\\t\\t383262', 'E_01_184\\t25259561\\tDHS-44 Enhancer\\thg19\\tchr7\\t117075514\\t117075933\\tHuman\\tHuman Bronchial Epithelial Cell\\tLow throughput\\t\"Luciferase Reporter Assay,EMSA,RT-PCR,ChIP\"\\t\"To confirm that DHS-44kb contained an enhancer of CFTR expression, the 0.6-kb DHS fragment (hg 19, chr7:117075400C117076000) was PCR amplified and inserted into the enhancer site of the pGL3B 245 vector.These data suggest that the enhancer activity of the DHS-44kb element is specific to airway epithelial cells.\"\\tEnhancer\\tCFTR\\t--\\t\"RT-PCR,ChIP\"\\t\"To determine whether the SFN_x0002_triggered antioxidant response regulates CFTR expression directly through the DHS-44(279) cis-element, ChIP was used to measure the enrichment levels of the constitutive ARE binding factor Bach1, the SFN-induced ARE binding factor Nrf2, and their common partner MafK at this site.\"\\t\"ABC35,ABCC7,CF,CFTR/MRP,MRP7,TNR-CFTR,dJ760C5.1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"NFKB1,BACH1\"\\t\"CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50,BACH-1,BTBD24\"\\tEMSA\\t\"EMSAs were performed with 32P-labeled, double-stranded DNA probes for 44-I, 44-II, and 44-III,Moreover, when Bach1 was overexpressed in 16HBE14o- cells by transient transfection of a mouse Bach1 cDNA clone, a strong supershift was seen with the Bach1 antibody and probe 44-I.\"\\t--\\t--\\t--\\t>2KB\\t44292', 'E_01_185\\t25263592\\t--\\thg19\\tchr19\\t2482923\\t2484923\\tHuman\\tHEK-293\\tLow throughput\\t\"ChIP,qRT-PCR,3C\"\\t\"we performed Chromosome Conformation Capture (3C) to quantitatively measure the chromosomal interactions in the regions surrounding the Arc gene.As expected, the eRNA-producing Arc enhancer was the most prominent genomic locus that interacts with the Arc promoter in an activity-dependent manner.\"\\tEnhancer\\tGADD45B\\t3C\\tChIP\\t\"To corroborate our findings from 3C analysis,we next examined the effect of eRNA knockdown on the binding levels of the Mediator and cohesin complexes at both the Arc and', 'E_01_186\\t25267720\\t--\\thg19\\tchr10\\t43568986\\t43571244\\tHuman\\tNeural Crest Cells\\tLow throughput\\t\"RT-PCR,Luciferase Reporter Assay\"\\t\"To determine the probable function of SRY on the transactivation of RET by NKX2-1 and PAX3, we had exam_x0002_ined its effects in a reporter system with a luciferase gene directed', 'E_01_187\\t25267720\\t--\\thg19\\tchr10\\t43571244\\t43572713\\tHuman\\tNeural Crest Cells\\tLow throughput\\t\"RT-PCR,Luciferase Reporter Assay\"\\t\"To determine the probable function of SRY on the transactivation of RET by NKX2-1 and PAX3, we had exam_x0002_ined its effects in a reporter system with a luciferase gene directed', 'E_01_188\\t25303440\\t--\\thg19\\tchr7\\t94002773\\t94005073\\tHuman\\tFibroblasts\\tLow throughput\\tLuciferase Reporter Assay\\t\"TGF is a potent inducer of the human procollagen type I 2 chain gene ( COL1A2). We have recently described in detail the in vivo mechanism underlying the transcriptional control of COL1A2, through a complex interaction between its distal enhancer and proximal promoter, in response to TGF. Reporter gene constructs containing the wild-type human enhancer region and the heterologous thymidine kinase (TK) promoter, lacking a Smad-dependent T RE (21.1/18.8pLAC-TK), were used to', 'E_01_189\\t25381333\\t--\\thg19\\tchr22\\t42411333\\t42413643\\tHuman\\tHuman Primary Hepatocytes\\tLow throughput\\t\"4C,ChIP\"\\t\"To search for regions that interact with the CYP2D6 promoter, we performed 4C assays with the CYP2D6 promoter as an anchor.With the exception of R1 and R6, peak', 'E_01_190\\t25381333\\t--\\thg19\\tchr22\\t42411544\\t42415106\\tHuman\\tHuman Primary Hepatocytes\\tLow throughput\\t\"4C,ChIP\"\\t\"To search for regions that interact with the CYP2D6 promoter, we performed 4C assays with the CYP2D6 promoter as an anchor.With the exception of R1 and R6, peak', 'E_01_191\\t25381333\\t--\\thg19\\tchr22\\t42411769\\t42414470\\tHuman\\tHuman Primary Hepatocytes\\tLow throughput\\t\"4C,ChIP\"\\t\"To search for regions that interact with the CYP2D6 promoter, we performed 4C assays with the CYP2D6 promoter as an anchor.With the exception of R1 and R6, peak', 'E_01_192\\t25434007\\t--\\thg19\\tchr1\\t98509888\\t98511451\\tHuman\\tNeuroblastoma Cell Lines\\tLow+High throughput\\tLuciferase Reporter Assay\\t\"SNP 1:g.98515539A>T is within an ENCODE-annotated neuronal cell (NH-A and SKNSH) speci?c enhancer and promoter ~3.6 kb upstream of MIR137 (Figure 1A).We hypothesized that the risk allele of 1:g.98515539A>T reduced enhancer activity by interfering with YY1 binding. To test the hypothesis, we cloned the putative enhancer sequence (the transcribed minus strand) ?anking 1:g.98515539A>T into a luciferase reporter gene vector (pGL3-promoter) (Figure 2A; Table S6). As expected from the ENCODE functional annotation, the cloned sequence exhibited robust enhancer activity as indicated by luciferase expression in a human neuroblastoma cell line (SH-SY5Y), but not in HeLa cells. \"\\tEnhancer\\tMIR137\\t3C\\t--\\t\"SNP 1:g.98515539A>T is within an ENCODE--annotated neuronalcell(NH--Aand SKNSH) specificenhancer and promoter ~3.6 kb upstream of MIR137.We therefore performed the 3C assay 58C60 in SH--SY5Y cells to examine whether the 1:g.98515539A>T site can physically interact with core promoters of DPYD and LOC729987 (Figure 4A). We identified specific physical interaction of the enhancer sequence flanking 1:g.98515539A>T with other putative regulatory sequences upstream of MIR137/MIR2682, but not with the core promoters of DPYD or LOC729987 (Figure 4B). This suggests the functional 1:g.98515539A>T might influence expression of MIR137/MIR2682, but not of DPYD or LOC729987.', 'E_01_193\\t25434007\\t--\\thg19\\tchr1\\t98514701\\t98515055\\tHuman\\tNeuroblastoma Cell Lines\\tLow+High throughput\\t\"Luciferase Reporter Assay,3C,PCR,EMSA\"\\t\"SNP 1:g.98515539A>T is within an ENCODE--annotated neuronal cell (NH--A and SKNSH) speci?c enhancer and promoter ~3.6 kb upstream of MIR137 (Figure 1A).We hypothesized that the risk allele of 1:g.98515539A>T reduced enhancer activity by interfering with YY1 binding. To test the hypothesis, we cloned the putative enhancer sequence (the transcribed minus strand) ?anking 1:g.98515539A>T into a luciferase reporter gene vector (pGL3--promoter) (Figure 2A; Table S6). As expected from the ENCODE functional annotation, the cloned sequence exhibited robust enhancer activity as indicated by luciferase expression in a human neuroblastoma cell line (SH--SY5Y), but not in HeLa cells. \"\\tEnhancer\\tMIR137\\t3C\\t--\\t\"SNP 1:g.98515539A>T is within an ENCODE--annotated neuronalcell(NH--Aand SKNSH) specificenhancer and promoter ~3.6 kb upstream of MIR137.We therefore performed the 3C assay 58C60 in SH--SY5Y cells to examine whether the 1:g.98515539A>T site can physically interact with core promoters of DPYD and LOC729987 (Figure 4A). We identified specific physical interaction of the enhancer sequence flanking 1:g.98515539A>T with other putative regulatory sequences upstream of MIR137/MIR2682, but not with the core promoters of DPYD or LOC729987 (Figure 4B). This suggests the functional 1:g.98515539A>T might influence expression of MIR137/MIR2682, but not of DPYD or LOC729987.', 'E_01_194\\t25434007\\t--\\thg19\\tchr1\\t98516486\\t98522430\\tHuman\\tNeuroblastoma Cell Lines\\tLow+High throughput\\t\"Luciferase Reporter Assay,3C,PCR,EMSA\"\\t\"SNP 1:g.98515539A>T is within an ENCODE--annotated neuronal cell (NH--A and SKNSH) speci?c enhancer and promoter ~3.6 kb upstream of MIR137 (Figure 1A).We hypothesized that the risk allele of 1:g.98515539A>T reduced enhancer activity by interfering with YY1 binding. To test the hypothesis, we cloned the putative enhancer sequence (the transcribed minus strand) ?anking 1:g.98515539A>T into a luciferase reporter gene vector (pGL3--promoter) (Figure 2A; Table S6). As expected from the ENCODE functional annotation, the cloned sequence exhibited robust enhancer activity as indicated by luciferase expression in a human neuroblastoma cell line (SH--SY5Y), but not in HeLa cells. \"\\tEnhancer\\tMIR137\\t3C\\t--\\t\"SNP 1:g.98515539A>T is within an ENCODE--annotated neuronalcell(NH--Aand SKNSH) specificenhancer and promoter ~3.6 kb upstream of MIR137.We therefore performed the 3C assay 58C60 in SH--SY5Y cells to examine whether the 1:g.98515539A>T site can physically interact with core promoters of DPYD and LOC729987 (Figure 4A). We identified specific physical interaction of the enhancer sequence flanking 1:g.98515539A>T with other putative regulatory sequences upstream of MIR137/MIR2682, but not with the core promoters of DPYD or LOC729987 (Figure 4B). This suggests the functional 1:g.98515539A>T might influence expression of MIR137/MIR2682, but not of DPYD or LOC729987.', 'E_01_195\\t25434007\\t--\\thg19\\tchr1\\t98517055\\t98517909\\tHuman\\tNeuroblastoma Cell Lines\\tLow+High throughput\\t\"Luciferase Reporter Assay,3C,PCR,EMSA\"\\t\"SNP 1:g.98515539A>T is within an ENCODE--annotated neuronal cell (NH--A and SKNSH) speci?c enhancer and promoter ~3.6 kb upstream of MIR137 (Figure 1A).We hypothesized that the risk allele of 1:g.98515539A>T reduced enhancer activity by interfering with YY1 binding. To test the hypothesis, we cloned the putative enhancer sequence (the transcribed minus strand) ?anking 1:g.98515539A>T into a luciferase reporter gene vector (pGL3--promoter) (Figure 2A; Table S6). As expected from the ENCODE functional annotation, the cloned sequence exhibited robust enhancer activity as indicated by luciferase expression in a human neuroblastoma cell line (SH--SY5Y), but not in HeLa cells. \"\\tEnhancer\\tMIR137\\t3C\\t--\\t\"SNP 1:g.98515539A>T is within an ENCODE--annotated neuronalcell(NH--Aand SKNSH) specificenhancer and promoter ~3.6 kb upstream of MIR137.We therefore performed the 3C assay 58C60 in SH--SY5Y cells to examine whether the 1:g.98515539A>T site can physically interact with core promoters of DPYD and LOC729987 (Figure 4A). We identified specific physical interaction of the enhancer sequence flanking 1:g.98515539A>T with other putative regulatory sequences upstream of MIR137/MIR2682, but not with the core promoters of DPYD or LOC729987 (Figure 4B). This suggests the functional 1:g.98515539A>T might influence expression of MIR137/MIR2682, but not of DPYD or LOC729987.', 'E_01_196\\t25453756\\t--\\thg19\\tchr12\\t98559923\\t98563615\\tHuman\\tInduced Pluripotent Stem Cells \\tLow+High throughput\\t\"3C,Luciferase Reporter Assay\"\\t\"We identified 4 SNPs (rs2159100, rs12315711, rs11062170, rs4765905) that are in perfect LD with rs758170 and rs1006737, and lie within two predicted enhancers (spanning 1.4Kb and 1.85Kb), ?185Kb downstream from the genes TSS (Figure 5A, Figure 5B). To identify whether the predicted enhancer region might be capable of forming promoterCenhancer loops, we mapped its physical interaction with the CACNA1C TSS in human dorsolateral prefrontal cortex (n = 6) and hiPSC derived-neurons by chromosome conformation capture (3C) assay. Of the 17 restriction site regions tested, only one displayed an increased interaction with the TSS in human postmortem tissue (Primer #4 in Figure 5ACD). This enhancer region includes rs2159100 (MAF = 0.35, GWAS P value = 1.1  10?10) and rs12315711 (MAF=0.35, GWAS P value = 1.1  10?10) and demonstrates increased interaction frequency with the CACNA1C promoter (F (16, 83) = 5.52, P = 8  10?8). In addition to being in an enhancer region, rs2159100 also co-localizes with a 600bp DHS region. Thus, we examined its effect on transcriptional activity in in vitro experiments. Compared to the reference rs2159100 C allele, the risk T variant is associated with decreased transcriptional activity in HEK-293 and Neuro-2a cells (42% (P < 0.0001) and 23% (P < 0.05) reduction in luciferase activity, respectively) (Figure 5E).\"\\tEnhancer\\tCACNA1C \\t3C\\tLuciferase Reporter Assay\\t\"We identified 4 SNPs (rs2159100, rs12315711, rs11062170, rs4765905) that are in perfect LD with rs758170 and rs1006737, and lie within two predicted enhancers (spanning 1.4Kb and 1.85Kb), ?185Kb downstream from the genes TSS (Figure 5A, Figure 5B). To identify whether the predicted enhancer region might be capable of forming promoterCenhancer loops, we mapped its physical interaction with the CACNA1C TSS in human dorsolateral prefrontal cortex (n = 6) and hiPSC derived-neurons by chromosome conformation capture (3C) assay. Of the 17 restriction site regions tested, only one displayed an increased interaction with the TSS in human postmortem tissue (Primer #4 in Figure 5ACD). This enhancer region includes rs2159100 (MAF = 0.35, GWAS P value = 1.1  10?10) and rs12315711 (MAF=0.35, GWAS P value = 1.1  10?10) and demonstrates increased interaction frequency with the CACNA1C promoter (F (16, 83) = 5.52, P = 8  10?8). In addition to being in an enhancer region, rs2159100 also co-localizes with a 600bp DHS region. Thus, we examined its effect on transcriptional activity in in vitro experiments. Compared to the reference rs2159100 C allele, the risk T variant is associated with decreased transcriptional activity in HEK-293 and Neuro-2a cells (42% (P < 0.0001) and 23% (P < 0.05) reduction in luciferase activity, respectively) (Figure 5E).\"\\t\"CACH2,CACN2,CACNL1A1,CCHL1A1,CaV1.2,LQT8,TS,TS.LQT8\"\\tSchizophrenia\\tDOID:5419\\tD012559\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t96481817', 'E_01_197\\t25453756\\t--\\thg19\\tchr12\\t98562692\\t98566384\\tHuman\\tInduced Pluripotent Stem Cells \\tLow+High throughput\\t3C\\t\"We identified 4 SNPs (rs2159100, rs12315711, rs11062170, rs4765905) that are in perfect LD with rs758170 and rs1006737, and lie within two predicted enhancers (spanning 1.4Kb and 1.85Kb), ?185Kb downstream from the genes TSS (Figure 5A, Figure 5B).?To identify whether the predicted enhancer region might be capable of forming promoterCenhancer loops, we mapped its physical interaction with the CACNA1C TSS in human dorsolateral prefrontal cortex (n = 6) and hiPSC derived-neurons by chromosome conformation capture (3C) assay.?Of the 17 restriction site regions tested, only one displayed an increased interaction with the TSS in human postmortem tissue (Primer #4 in Figure 5ACD).?This enhancer region includes rs2159100 (MAF = 0.35, GWAS P value = 1.1  10?10) and rs12315711 (MAF=0.35, GWAS P value = 1.1  10?10) and demonstrates increased interaction frequency with the CACNA1C promoter (F (16, 83) = 5.52, P = 8  10?8).?The rs2159100/rs12315711 3C interaction was confirmed in hiPSC derived-neurons (Figure 5D).?\"\\tEnhancer\\tCACNA1C \\t3C\\t--\\t\"We identified 4 SNPs (rs2159100, rs12315711, rs11062170, rs4765905) that are in perfect LD with rs758170 and rs1006737, and lie within two predicted enhancers (spanning 1.4Kb and 1.85Kb), ?185Kb downstream from the genes TSS (Figure 5A, Figure 5B).?To identify whether the predicted enhancer region might be capable of forming promoterCenhancer loops, we mapped its physical interaction with the CACNA1C TSS in human dorsolateral prefrontal cortex (n = 6) and hiPSC derived-neurons by chromosome conformation capture (3C) assay.?Of the 17 restriction site regions tested, only one displayed an increased interaction with the TSS in human postmortem tissue (Primer #4 in Figure 5ACD).?This enhancer region includes rs2159100 (MAF = 0.35, GWAS P value = 1.1  10?10) and rs12315711 (MAF=0.35, GWAS P value = 1.1  10?10) and demonstrates increased interaction frequency with the CACNA1C promoter (F (16, 83) = 5.52, P = 8  10?8).?The rs2159100/rs12315711 3C interaction was confirmed in hiPSC derived-neurons (Figure 5D).?\"\\t\"CACH2,CACN2,CACNL1A1,CCHL1A1,CaV1.2,LQT8,TS,TS.LQT8\"\\tSchizophrenia\\tDOID:5419\\tD012559\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t96484586', 'E_01_198\\t25453903\\t--\\thg19\\tchr14\\t97410723\\t97443618\\tHuman\\t\"HeLa,293T\"\\tLow+High throughput\\t\"shRNA,RNA-seq,qRT-PCR,3C,Immunohistochemistry\"\\t\"Finally, we sought to address the impact of altered cis-regulatory element activity on the transcriptional landscape of Ewing sarcoma. We performed RNA sequencing (RNA-seq) in the Ewing sarcoma lines before and after EWS-FLI1 knock-down. By mapping EWS-FLI1 distal elements to the nearest expressed genes, we observed a strong relationship between changes in enhancer activity and changes in proximal gene expression (p < 10?10). We confirmed a subset of regulated gene targets by qRT-PCR.', 'E_01_199\\t25505291\\t--\\thg19\\tchr1\\t23881244\\t23881761\\tHuman\\tHuman Monocytes/Macrophages\\tLow throughput\\tLuciferase Reporter Assay\\t\"Using ECR browser (http://ecrbrowser.dcode.org/), we found two conserved regions,ECR1 located between -3177 and -2660 bp upstream of transcription start and ECR2 located between +4517 and 4662 bp downstream of the gene (sequences provided in Supplemental Table 1). ECR1 overlaps with the enhancer described by Shepherd et al. (32), whereas ECR2 has not been described to date.', 'E_01_200\\t25505291\\t--\\thg19\\tchr1\\t23888938\\t23889083\\tHuman\\tHuman Monocytes/Macrophages\\tLow throughput\\tLuciferase Reporter Assay\\t\"Using ECR browser (http://ecrbrowser.dcode.org/), we found two conserved regions,ECR1 located between -3177 and -2660 bp upstream of transcription start and ECR2 located between +4517 and 4662 bp downstream of the gene (sequences provided in Supplemental Table 1). ECR1 overlaps with the enhancer described by Shepherd et al. (32), whereas ECR2 has not been described to date.', 'E_01_201\\t25522777\\tMMP9 Enhancer\\thg19\\tchr20\\t44637014\\t44637468\\tHuman\\tHepatocellular Carcinoma\\tLow throughput\\tLuciferase Reporter Assay\\t\"The MMP9 promoter contains two AP-1 sites (located at -79 bp and -533 bp), a Sp1 site (located at -560 bp) and an NF-B site (located at -600 bp). To determine whether regulation of MMP9 is related to these cis-acting regulatory elements for transcription factors, several constructs with deletions or mutations were used and have been described in Materials and methods.', 'E_01_202\\t25604083\\t--\\thg19\\tchr17\\t69487560\\t69491400\\tHuman\\tHEK-293\\tLow throughput\\t\"Luciferase Reporter Assay,Transgenic mice\"\\t\"Finally, we performed transfections of reporter gene constructs under control of the minimal Sox9 promoter and additional XYSR fragments into various cell lines.For this, the 32.5 kb region was subdivided into 17 overlapping fragments,F1 to F17 (?gure 3). Whereas none of the conserved elements CNE1C7 showed Sertoli cell-speci?c activity (not shown),of all 24 constructs tested, F8 showed the highest luciferase activities in both Sertoli-like cell lines (see online supplementary ?gure S5A, B).In contrast, the same fragment had little in?uence on luciferase activity in both HEK293 and neuro-2a control cell lines (see online supplementary ?gure S5C, D). Even though F7 and F9 overlap with F8 (?gure 3 and online supplementary ?gure S6), their luciferase activities are weak in TM-4 and unremarkable in NT2/D1. Thus F8 appeared to be the best candidate for a testis-speci?c enhancer of the SOX9 gene.\"\\tEnhancer\\tSOX9\\t--\\t\"Luciferase Reporter Assay,Transgenic mice\"\\t\"Finally, we performed transfections of reporter gene constructs under control of the minimal Sox9 promoter and additional XYSR fragments into various cell lines.For this, the 32.5 kb region was subdivided into 17 overlapping fragments,F1 to F17 (?gure 3). Whereas none of the conserved elements CNE1C7 showed Sertoli cell-speci?c activity (not shown),of all 24 constructs tested, F8 showed the highest luciferase activities in both Sertoli-like cell lines (see online supplementary ?gure S5A, B).In contrast, the same fragment had little in?uence on luciferase activity in both HEK293 and neuro-2a control cell lines (see online supplementary ?gure S5C, D). Even though F7 and F9 overlap with F8 (?gure 3 and online supplementary ?gure S6), their luciferase activities are weak in TM-4 and unremarkable in NT2/D1. Thus F8 appeared to be the best candidate for a testis-speci?c enhancer of the SOX9 gene.\"\\t\"CMD1,CMPD1,SRA1,SRXX2,SRXY10\"\\tDisorders Of Sexual Development\\t--\\tD012734\\t--\\t--\\t--\\t\"SF1,WT1\"\\t\"BBP, D11S636, MBBP, ZCCHC25, ZFM1, ZNF162,AWT1, GUD, NPHS4, WAGR, WIT-2, WT33\"\\tLuciferase Reporter Assay\\t\"Such an enhancer should be under control of SRY, but could possibly also be in?uenced in its activity by other transcription factors present in the early gonad and essential for gonadal differentiation, including SF1 (also known as NR5A1), WT1, and GATA4. To test this assumption, a luciferase reporter containing F8 in front of the ?-globin TATA box was co-transfected in neuro-2a cells with expression plasmids for these factors. Of all factors tested, only SRY elicited a substantial 10-fold increase in luciferase activity. WT1 stimulated reporter activity two fold, whereas SF1 and GATA4 were completely ineffective (?gure 4A).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t627680', 'E_01_203\\t25622893\\thCNS1 Enhancer\\thg19\\tchr17\\t61927353\\t61928827\\tHuman\\t\"GC-spd(ts),Hepa 1-6,293T\"\\tLow throughput\\t\"PCR,Luciferase Reporter Assay\"\\t\"We first investigated enhancer activity of hCNS1. Because TSS of TCAM1P--I and TCAM1P--III was +401 and three TSSs of TCAM1P--II were close to +401 (Fig. 2E), we cloned about 2--kb sequence upstream of +401 as a major promoter of the TCAM1P gene and connected it to the luciferase gene (TCAM1P--Pro--luc). hCNS1 was amplified by genome PCR and cloned to upstream of the TCAM1P promoter (hCNS1--Pro--luc) or to downstream of the luciferase gene(Pro--luc--hCNS1 and Pro--luc--rev hCNS1).The activity of hCNS1--Pro--luc was significantly increased in all three cell lines compared to the construct without hCNS1 (Fig. 3A).', 'E_01_204\\t25628225\\t--\\thg19\\tchr17\\t6285838\\t6586838\\tHuman\\tHuman Liver Cell\\tLow throughput\\t\"Luciferase Reporter Assay,EMSA,ChIP\"\\t\"Utilizing cell-based Luciferaseassays,electrophoretic mobility shift assays,and Chromatin immunoprecipitation assays,we identified and functionally characterized two Enhancer modules located upstream of the SLC13A5 gene transcription start site that are associated with regulation of PXR-mediated SLC13A5 induction.\"\\tEnhancer\\tSLC13A5\\t--\\t\"Luciferase Reporter Assay,EMSA,ChIP\"\\t\"Utilizing cell-based Luciferaseassays,electrophoretic mobility shift assays,and Chromatin immunoprecipitation assays,we identified and functionally characterized two Enhancer modules located upstream of the SLC13A5 gene transcription start site that are associated with regulation of PXR-mediated SLC13A5 induction.\"\\t\"EIEE25,INDY,NACT,mIndy\"\\tSteatosis\\t--\\t--\\t--\\t--\\t--\\tNR1I2\\t\"BXR,ONR1,PAR,PAR1,PAR2,PARq,PRR,PXR,SAR,SXR\"\\tEMSA\\t\"Utilizing in vitro EMSA,we found that heterodimer of PXR and the retinoid X receptor binds specifically to DR4-1 and DR4-2 at intensities equal to or exceeding that of the everted-repeat (ER6) element from CYP3A4 promoter.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t151693', 'E_01_205\\t25628225\\t--\\thg19\\tchr17\\t6565038\\t6567038\\tHuman\\tHuman Liver Cell\\tLow throughput\\t\"Luciferase Reporter Assay,EMSA,ChIP\"\\t\"Utilizing cell-based Luciferaseassays,electro_x0002_phoretic mobility shift assays,and Chromatin immunoprecipitation assays,we identified and functionally characterized two Enhancer modules located upstream of the SLC13A5 gene transcription start site that are associated with regulation of PXR-mediated SLC13A5 induction.\"\\tEnhancer\\tSLC13A5\\t--\\t\"Luciferase Reporter Assay,EMSA,ChIP\"\\t\"Utilizing cell-based Luciferaseassays,electro_x0002_phoretic mobility shift assays,and Chromatin immunoprecipitation assays,we identified and functionally characterized two Enhancer modules located upstream of the SLC13A5 gene transcription start site that are associated with regulation of PXR-mediated SLC13A5 induction.\"\\t\"EIEE25,INDY,NACT,mIndy\"\\tFatty Liver Disease\\tDOID:9452\\tD005234\\t--\\t--\\t--\\tNR1I2\\t\"BXR,ONR1,PAR,PAR1,PAR2,PARq,PRR,PXR,SAR,SXR\"\\tEMSA\\t\"Utilizing in vitro EMSA,we found that heterodimer of PXR and the retinoid X receptor binds specifically to DR4-1 and DR4-2 at intensities equal to or exceeding that of the everted-repeat (ER6) element from CYP3A5 promoter.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t21993', 'E_01_206\\t25701871\\t--\\thg19\\tchr5\\t125345378\\t125345495\\tHuman\\tHIT DH5A cells (RBC)\\tLow throughput\\t\"Luciferase Reporter Assay,4C,PCR\"\\t\"To test the predicted enhancers experimentally, we perfor-ed dual-luciferase reporter assays with constructs containing regions A, B, C or D, in direct or reverse orientation, upstream of the LMNB1 promoter . Following transfection of the constructs into HEK293T human cell line, regions A and B (in both orientations) signi?cantly increased luciferase expression (A forward = 1.97  0.2; A reverse = 1.8  1.6; B forward = 2.52  0.25; B reverse = 1.85  0.10; mean  standard error; P < 0.01; Fig. 3C). In contrast, constructs containing regions C and D did not signi?cantly affect luciferase expression (Fig. 3C). Similar results were obtained when the same plasmids were transfected into NIH3T3 cell line (data not shown), suggesting that the enhancer activity of regions A and B could function across species, at least in mouse ?broblasts. Following these results, regions A and B were subsequently referred to as enhancer region A (Enh-A) and enhancer region B (Enh-B).\"\\tEnhancerLMNB1\\t4C\\t\"Luciferase Reporter Assay,PCR\"\\t\"To test the predicted enhancers experimentally, we perfor-ed dual-luciferase reporter assays with constructs containing regions A, B, C or D, in direct or reverse orientation, upstream of the LMNB1 promoter . Following transfection of the constructs into HEK293T human cell line, regions A and B (in both orientations) signi?cantly increased luciferase expression (A forward = 1.97  0.2; A reverse = 1.8  1.6; B forward = 2.52  0.25; B reverse = 1.85  0.10; mean  standard error; P < 0.01; Fig. 3C). In contrast, constructs containing regions C and D did not signi?cantly affect luciferase expression (Fig. 3C). Similar results were obtained when the same plasmids were transfected into NIH3T3 cell line (data not shown), suggesting that the enhancer activity of regions A and B could function across species, at least in mouse ?broblasts. Following these results, regions A and B were subsequently referred to as enhancer region A (Enh-A) and enhancer region B (Enh-B).\"\\t\"ADLD,LMN,LMN2,LMNB\"\\tAdult-onset Autosomal Dominant Demyelinating Leukodystrophy\\tDOID:0060785\\tC566813\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t766877', 'E_01_207\\t25701871\\t--\\thg19\\tchr5\\t125995862\\t125995972\\tHuman\\tHIT DH5A cells (RBC)\\tLow throughput\\t\"Luciferase Reporter Assay,4C,PCR\"\\t\"To test the predicted enhancers experimentally, we perfor-ed dual-luciferase reporter assays with constructs containing regions A, B, C or D, in direct or reverse orientation, upstream of the LMNB1 promoter . Following transfection of the constructs into HEK293T human cell line, regions A and B (in both orientations) signi?cantly increased luciferase expression (A forward = 1.97  0.2; A reverse = 1.8  1.6; B forward = 2.52  0.25; B reverse = 1.85  0.10; mean  standard error; P < 0.01; Fig. 3C). In contrast, constructs containing regions C and D did not signi?cantly affect luciferase expression (Fig. 3C). Similar results were obtained when the same plasmids were transfected into NIH3T3 cell line (data not shown), suggesting that the enhancer activity of regions A and B could function across species, at least in mouse ?broblasts. Following these results, regions A and B were subsequently referred to as enhancer region A (Enh-A) and enhancer region B (Enh-B).', 'E_01_208\\t25704602\\t \\thg19\\tchr10\\t123099588\\t123100426\\tHuman\\tGMSM-K (human embry-onic oral epithelial cells)\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"The same FGFR2 +254 construct used in the zebra?sh studies (GRCh37, chr10: 123,099,588C123,100,426) was modi?ed by PCR-mediated mutagenesis to test the de novo mutation at GRCh37, chr10: 123,099,960.', 'E_01_209\\t25704602\\t \\thg19\\tchr17\\t54754755\\t54758235\\tHuman\\tGMSM-K (human embry-onic oral epithelial cells)\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"One element, located +87 kb from the NOG translation start site, is an epithelial enhancer in zebra?sh (Figures 5E and 5F). The second element, at +105 kb, lacked consistent enhancer activity in transient transgenic zebra?sh assays. However, this element had low-level enhancer activity in vitro in human fetal oral epithelial cells (GMSM-K) (Figure 5G) and murine osteoblastic cells (MC3T3) (Figure S19). Interestingly, the enhancer activity of the +105 kb element in both cell types was signi?cantly lower with theNSCL/P-associated allele of rs227727 (i.e., T) than with the unassociated allele (i.e., A) (Figures 5G and S18).\"\\tEnhancer\\tNOG\\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"One element, located +87 kb from the NOG translation start site, is an epithelial enhancer in zebra?sh (Figures 5E and 5F). The second element, at +105 kb, lacked consistent enhancer activity in transient transgenic zebra?sh assays. However, this element had low-level enhancer activity in vitro in human fetal oral epithelial cells (GMSM-K) (Figure 5G) and murine osteoblastic cells (MC3T3) (Figure S19). Interestingly, the enhancer activity of the +105 kb element in both cell types was signi?cantly lower with theNSCL/P-associated allele of rs227727 (i.e., T) than with the unassociated allele (i.e., A) (Figures 5G and S18).\"\\t\"SYM1, SYNS1, SYNS1A\"\\tCleft Palate\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t85435', 'E_01_210\\t25704602\\t \\thg19\\tchr17\\t54775600\\t54778000\\tHuman\\tGMSM-K (human embry-onic oral epithelial cells)\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"One element, located +87 kb from the NOG translation start site, is an epithelial enhancer in zebra?sh (Figures 5E and 5F). The second element, at +105 kb, lacked consistent enhancer activity in transient transgenic zebra?sh assays. However, this element had low-level enhancer activity in vitro in human fetal oral epithelial cells (GMSM-K) (Figure 5G) and murine osteoblastic cells (MC3T3) (Figure S19). Interestingly, the enhancer activity of the +105 kb element in both cell types was signi?cantly lower with theNSCL/P-associated allele of rs227727 (i.e., T) than with the unassociated allele (i.e., A) (Figures 5G and S18).\"\\tEnhancer\\tNOG\\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"One element, located +87 kb from the NOG translation start site, is an epithelial enhancer in zebra?sh (Figures 5E and 5F). The second element, at +105 kb, lacked consistent enhancer activity in transient transgenic zebra?sh assays. However, this element had low-level enhancer activity in vitro in human fetal oral epithelial cells (GMSM-K) (Figure 5G) and murine osteoblastic cells (MC3T3) (Figure S19). Interestingly, the enhancer activity of the +105 kb element in both cell types was signi?cantly lower with theNSCL/P-associated allele of rs227727 (i.e., T) than with the unassociated allele (i.e., A) (Figures 5G and S18).\"\\t\"SYM1, SYNS1, SYNS1A\"\\tCleft Palate\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t105740', 'E_01_211\\t25769725\\tENH3 Enhancer\\thg19\\tchr6\\t45410618\\t45417584\\tHuman\\t\"B-CPAP,TPC1\"\\tLow throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP\"\\t\"Next, the DNA fragments corresponding to ENH1, 2, and 3 were cloned into the pGL3-P2 vector downstream of the luciferase reporter gene and their ability to activate the P2 promoter wasassessed by luciferase assay (Fig. 2F). Only ENH3 determined a strong induction of the reporter gene, suggesting a potent activity of this element on the P2 promoter. Noticeably, the activating effect of ENH3 on the P2 promoter was stronger in TPC1 than in BCPAP cells, in line with the higher level of RUNX2 expression detected in TPC1 as compared with BCPAP cells (Fig. 2B; ref. 16).Furthermore, ENH3 maintained the ability of activating the reporter gene when cloned in the inverted orientation and when cloned in the pGL3 vector containing the sP2 promoter.Finally, we have shown by chromatin immunoprecipitation (ChIP) experiments that the binding of RNA Polymerase II (RNA PolII) on the ENH3 was signi?cantly enriched as compared with two downstream RUNX2 exons (exons 5 and 7; Fig.3A and B).\"\\tEnhancer\\tRUNX2\\t--\\t\"Luciferase Reporter Assay,RT-PCR,ChIP\"\\t\"Next, the DNA fragments corresponding to ENH1, 2, and 3 were cloned into the pGL3-P2 vector downstream of the luciferase reporter gene and their ability to activate the P2 promoter wasassessed by luciferase assay (Fig. 2F). Only ENH3 determined a strong induction of the reporter gene, suggesting a potent activity of this element on the P2 promoter. Noticeably, the activating effect of ENH3 on the P2 promoter was stronger in TPC1 than in BCPAP cells, in line with the higher level of RUNX2 expression detected in TPC1 as compared with BCPAP cells (Fig. 2B; ref. 16).Furthermore, ENH3 maintained the ability of activating the reporter gene when cloned in the inverted orientation and when cloned in the pGL3 vector containing the sP2 promoter.Finally, we have shown by chromatin immunoprecipitation (ChIP) experiments that the binding of RNA Polymerase II (RNA PolII) on the ENH3 was signi?cantly enriched as compared with two downstream RUNX2 exons (exons 5 and 7; Fig.3A and B).\"\\t\"AML3,CBF-alpha-1,CBFA1,CCD,CCD1,CLCD,OSF-2,OSF2,PEA2aA,PEBP2aA\"\\t--\\t--\\t--\\tInhibition of ENH3 activity is a generalmechanismof action of HDACi in different types of cancer\\tLuciferase Reporter Assay\\t\"Finally, we tested whether the TSA-repressive effect was mediated by repression of ENH3 as we have shown in thyroid cancer cells. In all cell lines tested, the presence of ENH3 down-stream of the sP2 promoter determined the activation of the reporter gene as already observed in TPC1 and BCPAP cells (Fig.7C). Similarly, upon TSA treatment, the ENH3 activity was repressed in all cell lines, suggesting the possibility to extend to other cancer types the model of ENH3 functioning that we proposed in thyroid cancer.\"\\tJUND\\tAP-1\\tChIP\\t\"We performed supershift assay using anti-c-JUN antibodies to con-firm that AP1 binds to site 2,ChIP experiments using anti-c-JUN antibodies showed that c-JUN actively binds to ENH3 in a context of structured Chromatin.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t118048', 'E_01_212\\t25782671\\t--\\thg19\\tchr7\\t155845627\\t155847652\\tHuman\\tCaco-2\\tLow throughput\\tLuciferase Reporter Assa\"The 2026 bp sequence was subcloned into the expression vector pLS_Prom_SHH. As the human colon adenocarcinoma Caco2 cell line expresses SHH,we therefore performed transfections in these cells with the construct containing the 2026 bp sequence in forward (pLS_Prom_SHH_2026F) and reverse orientation (pLS_Prom_SHH_2026R). Both constructs show an approximately twofold decrease of the relative reporter activity, as compared with the control vector without regulatory element pLS_Prom_SHH (Figure 4). Thus, the 2026 bp sequence contains an element capable of repressing the transcriptional activity of the SHH promoter in cis, consistent with a silencer activity.\"\\tEnhancer\\tSHH\\t--\\tLuciferase Reporter Assa\"The 2026 bp sequence was subcloned into the expression vector pLS_Prom_SHH. As the human colon adenocarcinoma Caco2 cell line expresses SHH,we therefore performed transfections in these cells with the construct containing the 2026 bp sequence in forward (pLS_Prom_SHH_2026F) and reverse orientation (pLS_Prom_SHH_2026R). Both constructs show an approximately twofold decrease of the relative reporter activity, as compared with the control vector without regulatory element pLS_Prom_SHH (Figure 4). Thus, the 2026 bp sequence contains an element capable of repressing the transcriptional activity of the SHH promoter in cis, consistent with a silencer activity.\"\\t\"HHG1,HLP3,HPE3,MCOPCB5,SMMCI,ShhNC,TPT,TPTPS\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t251707', 'E_01_213\\t25786345\\tCXCR5 Enhancer\\thg19\\tchr11\\t118767466\\t118769582\\tHuman\\tMCF-7\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"Genomic fragment containing cxcr5 promoter was subcloned upstream of luciferase gene in pGL3-Basic reporter vector. Another genomic fragment containing the potential cxcr5 enhancer was subcloned downstream of the luciferase gene. In order to account for lower transfection efficiency of larger plasmids, the second construct included a control genomic fragment from cxcr5 locus which was similar in length to the enhancer fragment but had no characteristic features of a regulatory sequence. Functional comparison of these two constructs in MCF-7 and MCF-7-2si cells (Figure 3A) revealed a marginal modulating effect by the enhancer (15%) towards cxcr5 promoter activity. This effect, however, was stable in both cell lines, indicating that activity of the cxcr5 intronic enhancer in breast cancer cells was independent of the p53 status. \"\\tEnhancerCXCR5\\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"Genomic fragment containing cxcr5 promoter was subcloned upstream of luciferase gene in pGL3-Basic reporter vector. Another genomic fragment containing the potential cxcr5 enhancer was subcloned downstream of the luciferase gene. In order to account for lower transfection efficiency of larger plasmids, the second construct included a control genomic fragment from cxcr5 locus which was similar in length to the enhancer fragment but had no characteristic features of a regulatory sequence. Functional comparison of these two constructs in MCF-7 and MCF-7-2si cells (Figure 3A) revealed a marginal modulating effect by the enhancer (15%) towards cxcr5 promoter activity. This effect, however, was stable in both cell lines, indicating that activity of the cxcr5 intronic enhancer in breast cancer cells was independent of the p53 status. \"\\t\"BLR1,CD185,MDR15\"\\t--\\t--\\t--\\t--\\t--\\t--\\tNFASC\\t\"NEDCPMD,NF,NRCAML\"\\t\"ChIP,PCR\"\\t\"Map of cxcr5 promoter,predicted NF -binding sites and PCR products amplified in ChIP assay.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t14050', 'E_01_214\\t26211970\\tPRE3 Enhancer\\thg19\\tchr8\\t81286250\\t81293977\\tHuman\\tLymphoblastoid Cell Lines\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,PCR,3C\"\\tPRE2 (A) or PRE3 (B) was cloned upstream of a PAG1 promoter driven luciferase reporter with and without the candidate causal SNPs.Our 3C studies provided evidence for allele--specific chromatin looping between PRE2 and PAG1. These results demonstrate that PRE3 acts as a transcriptional enhancer in the presence of the rs2370615:C allergy predisposing allele.\\tEnhancerPAG1\\t3C\\tLuciferase Reporter Assay\\tPRE2 (A) or PRE3 (B) was cloned upstream of a PAG1 promoter driven luciferase reporter with and without the candidate causal SNPs.Our 3C studies provided evidence for allele--specific chromatin looping between PRE2 and PAG1. These results demonstrate that PRE3 acts as a transcriptional enhancer in the presence of the rs2370615:C allergy predisposing allele.\\t\"CBP,PAG\"\\tNational Institute of Allergy and Infectious Diseases\\t--\\tD054579\\t--\\t--\\t--\\tNFKB1\\t\"CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\tChIP-seq\\t\"Based on ENCODE ChIP-seq data for LCLs,17 two transcription factors (TFs) bind to PRE3 and overlap the putative functional variant rs2370615: the RelA (or p65[MIM: 164014]) subunit of the nuclear factor (NF-)TF,which is critical for innate and adaptive immune re_x0002_sponses,22 and the POU domain class 2 transcription factor2 (POU2F2 or Oct-2 [MIM: 164176]),which regulates B-cell-specific genes.\"\\trs7009110\\t81291879\\t\"ChIP-seq,Luciferase Reporter Assay,3C\"\\t>2KB\\t\\t\\t\\t589930', 'E_01_215\\t26214525\\tPROX1-11 Enhancer\\thg19\\tchr1\\t214150511\\t214151337\\tHuman\\tK-562\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"The C11 kb element was also cloned distally into the enhancer site of pGL4.12, together with a subcloned region of the Prox1 +4.5 kb element (containing GATA sites 4 and 5, see Methods for details) acting as a promoter. This addition resulted in a significant increase in luciferase activity, demonstrating that the Prox1 C11 kb element is capable of acting as an enhancer (Figure 1D). \"\\tEnhancer\\tPROX1\\t--\\tLuciferase Reporter Assay\\t\"The C11 kb element was also cloned distally into the enhancer site of pGL4.12, together with a subcloned region of the Prox1 +4.5 kb element (containing GATA sites 4 and 5, see Methods for details) acting as a promoter. This addition resulted in a significant increase in luciferase activity, demonstrating that the Prox1 C11 kb element is capable of acting as an enhancer (Figure 1D). \"\\tPROX1\\tLymphedema\\tDOID:4977\\tD008209\\t--\\t--\\t--\\t\"GATA2,FOXC2,NFATC1\"\\t\"DCML,IMD21,MONOMAC,NFE1B,FKHL14,LD,MFH-1,MFH1,NF-ATC,NF-ATc1.2,NFAT2,NFATc\"\\tChIP\\t\"ChIP demonstrates that GATA2, FOXC2, and NFATC1 ChIP at the PROX1 C11 kb enhancer in LECs and BECs, but not K-562 cells.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t10353', 'E_01_216\\t26214589\\t--\\thg19\\tchr10\\t64737032\\t64737909\\tHuman\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"We observed activating chromatin marks, signals for formaldehyde-assisted isolation of regulatory elements (FAIRE) and/or DNaseI hypersensitivity at five main loci: two loci corresponding to known EGR2 regulatory elements (MSE (myelinating Schwann cell enhancer) 30 and BoneE (bone enhancer) (unpublished data); Supplementary Fig. 8), one to the ADO promoter, and two to GGAA microsatellites (mSat1 and mSat2) that overlapped with EWSR1-FLI1 ChIP-Seq signals (Fig. 3a). Luciferase reporter assays indicated that BoneE and MSE had no or weak activity in Ewing sarcoma, respectively.\"\\tEnhancer\\tEGR2 \\t--\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"We observed activating chromatin marks, signals for formaldehyde-assisted isolation of regulatory elements (FAIRE) and/or DNaseI hypersensitivity at five main loci: two loci corresponding to known EGR2 regulatory elements (MSE (myelinating Schwann cell enhancer) 30 and BoneE (bone enhancer) (unpublished data); Supplementary Fig. 8), one to the ADO promoter, and two to GGAA microsatellites (mSat1 and mSat2) that overlapped with EWSR1-FLI1 ChIP-Seq signals (Fig. 3a). Luciferase reporter assays indicated that BoneE and MSE had no or weak activity in Ewing sarcoma, respectively.\"\\t\"AT591,CHN1,CMT1D,CMT4E,KROX20\"\\tEwing Sarcoma\\tDOID:3369\\tD012512\\t--\\t--\\t--\\tEWSR1\\t\"EWS,EWS-FLI1,bK984G1.4\"\\tqRT-PCR\\tChIP efficacy was validated by qRT-PCR using a CCND1 EWSR1-FLI1 binding site.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t165715', 'E_01_217\\t26216945\\tEBAs Enhancer\\thg19\\tchr2\\t98362566\\t98364423\\tHuman\\tHEK-293\\tLow+High throughput\\t\"ChIP-seq,Hi-C,Luciferase Reporter Assay\"\\t\"Similarly, we integrated topologically associated domain (TAD) boundaries inferred from Hi-C chromatin contact maps (34) and found that the MIR insulators are also closer to TAD boundaries (P < 5.4E-8, MannCWhitney test) (Fig. 1 C and D).These findings suggest the predicted MIR insulators have domain barrier function. In addition, we used the ChIACPET interaction data from human CD4+ T cells (35) to test whether the predicted MIR insulators can potentially block enhancerCpromoter interactions. We focused on ChIACPET interactions between enhancers and promoters that are proximal (<500 kb) to MIR-derived insulators and classified them into one-side interactions, i.e., the interactions anchors are restricted to one-side of a MIR insulator, and cross-interactions, i.e., the interactions anchors are separated by a MIR insulator (Fig. 1E).For the human EBA, a luciferase reporter construct transfected in human HEK 293 cells (Materials and Methods) was used to evaluate three predicted MIR insulators (SI Appendix, Table S2). All three MIR insulators tested here showed enhancer-blocking activity comparable to the 5 HS4 positive control (Fig. 2A) and much larger insulator activity than the second positive control, i.e., the minimal insulator sequence motif of 5 HS4 (II/III). \"\\tEnhancer\\tZAP70\\t--\\tLuciferase Reporter Assay\\t\"ll three MIR insulators tested here showed enhancer-blocking activity comparable to the 5 HS4 positive control (Fig. 2A) and much larger insulator activity than the second positive control, i.e., the minimal insulator sequence motif of 5 HS4 (II/III).The blocking of the repressive chromatin domain also appears to be functionally important to CD4+ T cells because one of the proximal genes in the adjacent active chromatin domain, i.e., gene ZAP70, is part of the T-cell receptor pathway and its promoter is involved with multiple active ChIACPET interactions (Fig. 2D).\"\\t\"ADMIO2,IMD48,SRK,STCD,STD,TZK,ZAP-70\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t33465', 'E_01_218\\t26229090\\t--\\thg19\\tchr3\\t186938165\\t186940165\\tHuman\\tLymphoma Cell\\tLow+High throughput\\t\"ChIP-seq,3C,qRT-PCR\"\\t\"This highlighted a known BCL6-interacting Enhancer region (6,16,17) 150-250 upstream of BCL6 (SE1),which overlaps the alternate breakpoint region,a recurrent site of BCL6 locus rearrangements (15). It also revealed a second Super-Enhancer overlapping the HGB-04 duplication at ?350 (SE2),and a third such region at ?500 (SE3).We used chromosome conformation capture (3C) analysis of GCB-DLBCL lymphoma cell lines (Supplementary Fig. S3A) to confirm looping of elements from super-enhancer regions to the BCL6 promoter, consistent with roles in activating this oncogene. Intriguingly, our PMBL case, HGB-04, showed a distinct acetylation pattern across these regions, with only the duplicated enhancer region, SE2, showing strong acetylation.', 'E_01_219\\t26229090\\t--\\thg19\\tchr3\\t187088165\\t187909165\\tHuman\\tLymphoma Cell\\tLow+High throughput\\t\"ChIP-seq,3C,qRT-PCR\"\\t\"This High throughputlighted a known BCL6-interacting Enhancer region (6,16,17) 150-250 upstream of BCL6 (SE1),which overlaps the alternate breakpoint region,a recurrent site of BCL6 locus rearrangements (15). It also revealed a second Super-Enhancer overlapping the HGB-04 duplication at ?350 (SE2),and a third such region at ?500 (SE3).We used chromosome conformation capture (3C) analysis of GCB-DLBCL lymphoma cell lines (Supplementary Fig. S3A) to confirm looping of elements from super-enhancer regions to the BCL6 promoter, consistent with roles in activating this oncogene. Intriguingly, our PMBL case, HGB-04, showed a distinct acetylation pattern across these regions, with only the duplicated enhancer region, SE2, showing strong acetylation.\"\\tSuper-Enhancer\\tBCL6\\t3C\\tqRT-PCR\\t\"This High throughputlighted a known BCL6-interacting Enhancer region (6,16,17) 150-250 upstream of BCL6 (SE1),which overlaps the alternate breakpoint region,a recurrent site of BCL6 locus rearrangements (15). It also revealed a second Super-Enhancer overlapping the HGB-04 duplication at ?350 (SE2),and a third such region at ?500 (SE3).We used chromosome conformation capture (3C) analysis of GCB-DLBCL lymphoma cell lines (Supplementary Fig. S3A) to confirm looping of elements from super-enhancer regions to the BCL6 promoter, consistent with roles in activating this oncogene. Intriguingly, our PMBL case, HGB-04, showed a distinct acetylation pattern across these regions, with only the duplicated enhancer region, SE2, showing strong acetylation.\"\\t\"BCL5A,BCL6,LAZ3,ZBTB27,ZNF51\"\\tB-Cell Lymphoma\\tDOID:707\\tD016393\\t--\\t--\\t--\\tMEF2B\\tRSRFR2\\tChIP-Seq\\t\"Rather, we observed strong, focal MEF2B binding at acetylated elements within BCL6 super-enhancer regions, including four sites in SE1, one in SE2, and two in SE3 (Fig. 3C). ChIP-Seq analysis in the Jeko-1 MCL line revealed a specific increase in H3K27ac at MEF2B binding sites in MEF2B transgene-expressing cells, but not at intervening elements within the BCL6 super-enhancers (Fig. 3E).These data support a direct role for MEF2B in the activation of enhancers that drive BCL6 expression in human lymphomas. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t59501', 'E_01_220\\t26229090\\t--\\thg19\\tchr3\\t187559165\\t187589165\\tHuman\\tLymphoma Cell\\tLow+High throughput\\t\"ChIP-seq,3C,qRT-PCR\"\\t\"This High throughputlighted a known BCL6-interacting Enhancer region (6,16,17) 150-250 upstream of BCL6 (SE1),which overlaps the alternate breakpoint region,a recurrent site of BCL6 locus rearrangements (15). It also revealed a second Super-Enhancer overlapping the HGB-04 duplication at ?350 (SE2),and a third such region at ?500 (SE3).We used chromosome conformation capture (3C) analysis of GCB-DLBCL lymphoma cell lines (Supplementary Fig. S3A) to confirm looping of elements from super-enhancer regions to the BCL6 promoter, consistent with roles in activating this oncogene. Intriguingly, our PMBL case, HGB-04, showed a distinct acetylation pattern across these regions, with only the duplicated enhancer region, SE2, showing strong acetylation.\"\\tSuper-Enhancer\\tBCL6\\t3C\\tqRT-PCR\\t\"This High throughputlighted a known BCL6-interacting Enhancer region (6,16,17) 150-250 upstream of BCL6 (SE1),which overlaps the alternate breakpoint region,a recurrent site of BCL6 locus rearrangements (15). It also revealed a second Super-Enhancer overlapping the HGB-04 duplication at ?350 (SE2),and a third such region at ?500 (SE3).We used chromosome conformation capture (3C) analysis of GCB-DLBCL lymphoma cell lines (Supplementary Fig. S3A) to confirm looping of elements from super-enhancer regions to the BCL6 promoter, consistent with roles in activating this oncogene. Intriguingly, our PMBL case, HGB-04, showed a distinct acetylation pattern across these regions, with only the duplicated enhancer region, SE2, showing strong acetylation.\"\\t\"BCL5A,BCL6,LAZ3,ZBTB27,ZNF51\"\\tB-Cell Lymphoma\\tDOID:707\\tD016393\\t--\\t--\\t--\\tMEF2B\\tRSRFR2\\tChIP-Seq\\t\"Rather, we observed strong, focal MEF2B binding at acetylated elements within BCL6 super-enhancer regions, including four sites in SE1, one in SE2, and two in SE3 (Fig. 3C). ChIP-Seq analysis in the Jeko-1 MCL line revealed a specific increase in H3K27ac at MEF2B binding sites in MEF2B transgene-expressing cells, but not at intervening elements within the BCL6 super-enhancers (Fig. 3E).These data support a direct role for MEF2B in the activation of enhancers that drive BCL6 expression in human lymphomas. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t135001', 'E_01_221\\t26232425\\t--\\thg19\\tchr20\\t42977441\\t42978040\\tHuman\\tHep G2\\tLow throughput\\tLuciferase Reporter Assa\"The HNF4a P1 promoter consists of the distal enhancer region extending around 26.5 kb fromthe transcription starting site and the proximal promoter region (Ladias et al., 1992; Parviz et al., 2003; Hatzis et al.,2006).', 'E_01_222\\t26257179\\t--\\thg19\\tchr4\\t56895793\\t56962351\\tHuman\\tHuman Embryonic Kidney Cell\\tLow+High throughput\\t\"ChIP-seq,5\\' RACE\"\\t\"As a representative example, we highlight cheRNA1345, which is produced from a locus ~15 kb downstream of Cep135 and overlaps with an experimentally validated tissue-specific Enhancer element (Figure 4E) (Visel et al., 2007). Peaks of H3K4me3, H3K27ac, and RNAPII decorate the TSS, which we validated by 5\\' RACE, and also bears a number of validated transcription factor binding sites (Gerstein et al., 2012). Similar patterns are observed with other cheRNA loci (Figure S3ACC).\"\\tEnhancer\\tCEP135\\t--\\t\"ChIP-seq,5\\' RACE\"\\t\"As a representative example, we highlight cheRNA1345, which is produced from a locus ~15 kb downstream of Cep135 and overlaps with an experimentally validated tissue-specific Enhancer element (Figure 4E) (Visel et al., 2007). Peaks of H3K4me3, H3K27ac, and RNAPII decorate the TSS, which we validated by 5\\' RACE, and also bears a number of validated transcription factor binding sites (Gerstein et al., 2012). Similar patterns are observed with other cheRNA loci (Figure S3ACC).\"\\t\"CEP4,KIAA0635,MCPH8\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t114099', 'E_01_223\\t26300977\\t--\\thg19\\tchr7\\t114541370\\t114542201\\tHuman\\t\"HEK-293,SK-N-MC\"\\tLow throughput\\tLuciferase Reporter Assay\\t\"To test whether these regions could act as Enhancer elements we cloned them into a reporter construct in front of a minimal promoter and luciferase reporter gene.Element 1, which had multiple chromatin signatures characteristic of an enhancer, was able to act as a functional enhancer in both cell-lines (Fig. 1d).In HEK293 cells we observed a 3 fold increase of luciferase expression in comparison to the empty vector control.In SK-N-MC cells the luciferase expression increased nearly 7 fold as compared to the control. \"\\tEnhancer\\tFOXP2\\t--\\tLuciferase Reporter Assay\\t\"To test whether these regions could act as Enhancer elements we cloned them into a reporter construct in front of a minimal promoter and luciferase reporter gene.Element 1, which had multiple chromatin signatures characteristic of an enhancer, was able to act as a functional enhancer in both cell-lines (Fig. 1d).In HEK293 cells we observed a 3 fold increase of luciferase expression in comparison to the empty vector control.In SK-N-MC cells the luciferase expression increased nearly 7 fold as compared to the control. \"\\t\"CAGH44,SPCH1,TNRC10\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t815422', 'E_01_224\\t26382291\\t--\\thg19\\tchr6\\t1614188\\t1617051\\tHuman\\tEmbryonic Stem Cell\\tLow+High throughput\\tEnhancer Assay\\tWe experimentally validate four novel Enhancers in this locus and demonstrate that these Enhancers show consistent activity during embryonic development in domains that overlap with the expression of FOXC1 and GMDS.\\tEnhancer\\t\"FOXC1,GMDS\"\\t--\\tEnhancer Assay\\tWe experimentally validate four novel Enhancers in this locus and demonstrate that these Enhancers show consistent activity during embryonic development in domains that overlap with the expression of FOXC1 and GMDS.\\t\"ARA,ASGD3,FKHL7,FREAC-3,FREAC3,IGDA,IHG1,IRID1,RIEG3,GMD,SDR3E1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tZIC1\\t\"CRS6,ZIC,ZNF201\"\\tX-Gal Staining Assay \\t\"An image from the UCSC Genome Browser showing the enhancer landscape near FOXC1 and GMDS.The lower panel is zoomed in to the regions immediately surrounding the enhancers, to highlight the ZIC1 binding sites. b E11.5 whole-mount images of ZIC mutants CE1C3. CE4 only yielded two X gal- positive embryos and was thus not used in the analysis. Tissues that have lacZ expression in at least half of the X-gal-positive embryos in WT but not in ZIC enhancers are noted in red. Arrowheads mark regions of interest.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t4940', 'E_01_225\\t26382291\\t--\\thg19\\tchr6\\t1615843\\t1617834\\tHuman\\tEmbryonic Stem Cell\\tLow+High throughput\\tEnhancer Assay\\tWe experimentally validate four novel Enhancers in this locus and demonstrate that these Enhancers show consistent activity during embryonic development in domains that overlap with the expression of FOXC1 and GMDS.\\tEnhancer\\t\"FOXC1,GMDS\"\\t--\\tEnhancer Assay\\tWe experimentally validate four novel Enhancers in this locus and demonstrate that these Enhancers show consistent activity during embryonic development in domains that overlap with the expression of FOXC1 and GMDS.\\t\"ARA,ASGD3,FKHL7,FREAC-3,FREAC3,IGDA,IHG1,IRID1,RIEG3,GMD,SDR3E1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tZIC1\\t\"CRS6,ZIC,ZNF201\"\\tX-Gal Staining Assay \\t\"An image from the UCSC Genome Browser showing the enhancer landscape near FOXC1 and GMDS.The lower panel is zoomed in to the regions immediately surrounding the enhancers, to highlight the ZIC1 binding sites. b E11.5 whole-mount images of ZIC mutants CE1C3. CE4 only yielded two X gal- positive embryos and was thus not used in the analysis. Tissues that have lacZ expression in at least half of the X-gal-positive embryos in WT but not in ZIC enhancers are noted in red. Arrowheads mark regions of interest.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t6159', 'E_01_226\\t26382291\\t--\\thg19\\tchr6\\t1619348\\t1621343\\tHuman\\tEmbryonic Stem Cell\\tLow+High throughput\\tEnhancer Assay\\tWe experimentally validate four novel Enhancers in this locus and demonstrate that these Enhancers show consistent activity during embryonic development in domains that overlap with the expression of FOXC1 and GMDS.\\tEnhancer\\t\"FOXC1,GMDS\"\\t--\\tEnhancer Assay\\tWe experimentally validate four novel Enhancers in this locus and demonstrate that these Enhancers show consistent activity during embryonic development in domains that overlap with the expression of FOXC1 and GMDS.\\t\"ARA,ASGD3,FKHL7,FREAC-3,FREAC3,IGDA,IHG1,IRID1,RIEG3,GMD,SDR3E1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tZIC1\\t\"CRS6,ZIC,ZNF201\"\\tX-Gal Staining Assay \\t\"An image from the UCSC Genome Browser showing the enhancer landscape near FOXC1 and GMDS.The lower panel is zoomed in to the regions immediately surrounding the enhancers, to highlight the ZIC1 binding sites. b E11.5 whole-mount images of ZIC mutants CE1C3. CE4 only yielded two X gal- positive embryos and was thus not used in the analysis. Tissues that have lacZ expression in at least half of the X-gal-positive embryos in WT but not in ZIC enhancers are noted in red. Arrowheads mark regions of interest.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t9666', 'E_01_227\\t26382291\\t--\\thg19\\tchr6\\t1701730\\t1704441\\tHuman\\tEmbryonic Stem Cell\\tLow+High throughput\\tEnhancer Assay\\tWe experimentally validate four novel Enhancers in this locus and demonstrate that these Enhancers show consistent activity during embryonic development in domains that overlap with the expression of FOXC1 and GMDS.\\tEnhancer\\t\"FOXC1,GMDS\"\\t--\\tEnhancer Assay\\tWe experimentally validate four novel Enhancers in this locus and demonstrate that these Enhancers show consistent activity during embryonic development in domains that overlap with the expression of FOXC1 and GMDS.\\t\"ARA,ASGD3,FKHL7,FREAC-3,FREAC3,IGDA,IHG1,IRID1,RIEG3,GMD,SDR3E1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tZIC1\\t\"CRS6,ZIC,ZNF201\"\\tX-Gal Staining Assay \\t\"An image from the UCSC Genome Browser showing the enhancer landscape near FOXC1 and GMDS.The lower panel is zoomed in to the regions immediately surrounding the enhancers, to highlight the ZIC1 binding sites. b E11.5 whole-mount images of ZIC mutants CE1C3. CE4 only yielded two X gal- positive embryos and was thus not used in the analysis. Tissues that have lacZ expression in at least half of the X-gal-positive embryos in WT but not in ZIC enhancers are noted in red. Arrowheads mark regions of interest.\"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t92406', 'E_01_228\\t26422229\\t--\\thg19\\tchr12\\t56728500\\t56735000\\tHuman\\tTHP-1\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"Our ChIP-seq analysis of markers for transcriptional enhancers in macrophages revealed the presence, in both uninfected and infected THP-1 cells, of an extended region encompassing the IL23A locus of elevated H3K4me1 (Fig. 7A). We screened between the PAN2 and IL23A loci for sequences with homology to extended core AHRE motifs (Fig. 7A). This revealed three potential elements lying 3675, 2025, and 1375 bp upstream of the IL23A transcription start site, one of which (22025) corresponds to an extended consensus sequence. ChIP experiments in THP-1 cells revealed that binding of AHR and its heterodimeric partners ARNT and RELB to these regions is markedly enhanced by infection (Fig. 7BCD), with, as expected, the strongest association of all three proteins observed at the extended consensus motif (22025 bp, Fig. 7C).\"\\tEnhancer\\tIL23A\\t--\\t\"ChIP-seq,ChIP\"\\t\"Our ChIP-seq analysis of markers for transcriptional enhancers in macrophages revealed the presence, in both uninfected and infected THP-1 cells, of an extended region encompassing the IL23A locus of elevated H3K4me1 (Fig. 7A). We screened between the PAN2 and IL23A loci for sequences with homology to extended core AHRE motifs (Fig. 7A). This revealed three potential elements lying 3675, 2025, and 1375 bp upstream of the IL23A transcription start site, one of which (22025) corresponds to an extended consensus sequence. ChIP experiments in THP-1 cells revealed that binding of AHR and its heterodimeric partners ARNT and RELB to these regions is markedly enhanced by infection (Fig. 7BCD), with, as expected, the strongest association of all three proteins observed at the extended consensus motif (22025 bp, Fig. 7C).\"\\t\"IL-23,IL-23A,IL23P19,P19,SGRF\"\\tMycobacterium Tuberculosis\\t--\\tD009169\\t--\\t--\\t--\\tAHR\\t\"RP85, bHLHe76\"\\tChIP\\tChIP analysis revealed that infection enhanced binding of AHR and ARNT to AHRE (31) in the IL10 promoter.\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t912', 'E_01_229\\t26813977\\t--\\thg19\\tchr6\\t30904342\\t30904788\\tHuman\\tEmbryonic Stem Cell\\tLow throughput\\t\"CRISPR/Cas9,Luciferase Reporter Assay,4C-seq\"\\t\"We designed 1964 sgRNA sequences (Supplemental Table S2) targeting these putative cis-regulatory elements with an average of 11 sgRNA per element (Supplemental Fig. S1A). To eliminate these false positives,we required that a positive cis-regulatory element should have at least two distinct sgRNAs enriched by twofold or more in the eGFP? population in at least three out of the four independent experiments (Supplemental Table S3; Supplemental Fig. S2A). By use of this criterion, no negative control sgRNA passed the filter, while six cis-regulatory elements were identified as positives (Fig. 2A).', 'E_01_230\\t26423156\\tCARDIAC Enhancer\\thg19\\tchr5\\t148785000\\t148815000\\tHuman\\tCardiac Progenitor Cell\\tLow+High throughput\\t\"ChIP-seq,Immunohistochemistry,Luciferase Reporter Assay,RIP\"\\t\"Secondly, ChIP-analysis demonstrates that these transcripts mapped to a highly active cardiac enhancer in both fetal and adult human hearts (Fig. 3B). Notably, this region was shown to harbor a car_x0002_diac SRF/NKX2.5-bound enhancer required for cardiac-specific expres_x0002_sion of miR-143 and -145 during cardiac development.Furthermore, this enhancer has been previously demonstrated to be a Notch-responsive enhancer, capable of modulating miR-143 and -145', 'E_01_231\\t26445536\\t--\\thg19\\tchr22\\t35686840\\t35687280\\tHuman\\tVascular Endothelial Cell\\tLow throughput\\t\"ChIP,RT-PCR,qPCR\"\\t\"To investigate whether the translocated Nrf2 interacted with the HO-1 promoter region, the primer sets for the enhancer region (E2) and promoter region near transcription start site were synthesized for ChIP assays (Figure 5A, upper panel schematic). The ChIP analyses demonstrated that AuNP treatment significantly induced Nrf2 binding to the human HO-1 E2 enhancer region, but not to the promoter region near the transcription start site. The binding to the enhancer region was concentration dependent, as determined by RT-PCR and real-time PCR (Figure 5B and C).\"\\tEnhancer\\tHMOX1\\t--\\t\"ChIP,RT-PCR,qPCR\"\\t\"To investigate whether the translocated Nrf2 interacted with the HO-1 promoter region, the primer sets for the enhancer region (E2) and promoter region near transcription start site were synthesized for ChIP assays (Figure 5A, upper panel schematic). The ChIP analyses demonstrated that AuNP treatment significantly induced Nrf2 binding to the human HO-1 E2 enhancer region, but not to the promoter region near the transcription start site. The binding to the enhancer region was concentration dependent, as determined by RT-PCR and real-time PCR (Figure 5B and C).\"\\t\"HMOX1D,HO-1,HSP32,bK286B10\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t89999', 'E_01_232\\t26501517\\tHS2 enhancer \\thg19\\tchr11\\t5301600\\t5302400\\tHuman\\tK-562\\tLow+High throughput\\t\"CRISPR/Cas9,qRT-PCR\"\\t\"To find optimal sgRNAs for repression of the HS2 enhancer by dCas9-KRAB, we designed a panel of 21 sgRNAs (Cr1CCr21) to cover the 400-bp core of the HS2 enhancer (Fig. 1a and Supplementary Table 1). Each sgRNA was transiently transfected into human K562 erythroid leukemia cells that were modified to stably express dCas9 or dCas9-KRAB (Supplementary Fig. 1a). The activity of each sgRNA was screened 3 d after transfection by quantitative real-time PCR (qRT-PCR) for expression of HBE1,', 'E_01_233\\t26560027\\t--\\thg19\\tchr11\\t8271697\\t8279492\\tHuman\\tNeuroblastoma Cell Lines\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"Acetylation of histone H3 at lysine 27 (H3K27ac) is a hallmark of active enhancers, and ChIP-seq analysis of SHSY5Y (G/G; not MYCN amplified), KELLY (G/?; MYCN amplified), BE2 (G/T; MYCN amplified) and NGP (G/T; MYCN amplified) neuroblastoma cells showed extensive H3K27 acetylation in the first intron of LMO1 across', 'E_01_234\\t26604133\\t--\\thg19\\tchr4\\t26075179\\t26135897\\tHuman\\tCD4+ T Cell\\tLow+High throughput\\t\"ChIP-seq,PCR,ELISA,qPCR\"\\t\"Genome-wide association studies of rheumatoid arthritis (RA) identi?ed a susceptibility locus, rs874040CC, which implicated the RBPJ gene. Here, chromatin state mapping generated using the chromHMM algorithm reveals strong enhancer regions containing DNase I hypersensitive sites overlapping the rs874040 linkage disequilibrium block in human memory, but not in na?ve CD4+ T cells. The rs874040 overlapping this chromatin state was associated with increased RBPJ expression in stimulated memory CD4+ T cells from healthy subjects homozygous for the risk allele (CC) compared with memory CD4+ T cells bearing the protective allele (GG).', 'E_01_235\\t26610392\\t--\\thg19\\tchr11\\t118540532\\t118575295\\tHuman\\tNormal Human Astrocyte\\tLow throughput\\t\"Luciferase Reporter Assay,3C\"\\t\"After testing candidate SNPs for allele-specific activity in a luciferase-based enhancer scanning assay, we established a subset of 10 functional SNPs in the promoters of PHLDB1 and DDX6, and in a putative enhancer element. Chromatin conformation capture (3C) identified a physical interaction between the enhancer element containing a functional SNP (rs73001406) and the promoter of the DDX6 gene.\"\\tEnhancerDDX6\\t3C\\tLuciferase Reporter Assay\\t\"After testing candidate SNPs for allele-specific activity in a luciferase-based enhancer scanning assay, we established a subset of 10 functional SNPs in the promoters of PHLDB1 and DDX6, and in a putative enhancer element. Chromatin conformation capture (3C) identified a physical interaction between the enhancer element containing a functional SNP (rs73001406) and the promoter of the DDX6 gene.\"\\t\"LL5A,HLR2,P54,RCK\"\\tGlioma\\tDOID:3070\\tD005910\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t80702', 'E_01_236\\t26615198\\t--\\thg19\\tchr7\\t117011495\\t117063767\\tHuman\\t\"Human Nasal Epithelial Cell,Skin Fibroblasts Cell\"\\tLow+High throughput\\t\"Hi-C,5C,Luciferase Reporter Assay,PCR\"\\t\"We tested whether any of the newly identiied looping regions displayed enhancer activity in a reporter assay. As shown in Figure 4A, two of our interacting regions significantly enhanced CFTR promoter activity.The fragment encompassing a DHS in the region E displayed a strong Enhancer function with almost 6-fold effect on the CFTR promoter.\"\\tEnhancer\\tCFTR\\t--\\tLuciferase Reporter Assay\\t\"As shown in Figure 4A, two of our interacting regions significantly enhanced CFTR promoter activity.The fragment encompassing a DHS in the region E displayed a strong Enhancer function with almost 6-fold effect on the CFTR promoter.\"\\t\"ABC35,ABCC7,CF,CFTR/MRP,MRP7,TNR-CFTR,dJ760C5.1\"\\tCystic Fibrosis\\tDOID:1485\\tD003550\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t82385', 'E_01_237\\t26615198\\t--\\thg19\\tchr7\\t117282955\\t117335227\\tHuman\\t\"Human Nasal Epithelial Cell,Skin Fibroblasts Cell\"\\tLow+High throughput\\t\"Hi-C,5C,Luciferase Reporter Assay,PCR\"\\t\"We tested whether any of the newly identiied looping regions displayed enhancer activity in a reporter assay. As shown in Figure 4A, two of our interacting regions significantly enhanced CFTR promoter activity.The fragment encompassing a DHS in the region E displayed a strong Enhancer function with almost 6-fold effect on the CFTR promoter.\"\\tEnhancer\\tCFTR\\t--\\tLuciferase Reporter Assay\\t\"As shown in Figure 4A, two of our interacting regions significantly enhanced CFTR promoter activity. The G region containing a DHS and a CTCF binding site also acted as an Enhancer.\"\\t\"ABC35,ABCC7,CF,CFTR/MRP,MRP7,TNR-CFTR,dJ760C5.1\"\\tEpithelial Cancer\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t189075', 'E_01_238\\t26626481\\tMYC Super-Enhancer\\thg19\\tchr8\\t130386715\\t130591515\\tHuman\\tAcute Myeloid Leukemia Cell\\tLow throughput\\t\"ChIP,ChIP-qPCR\"\\t\"BRD4 has been shown previously to regulate Myc expression in AML cell via a super-Enhancer (with individual Enhancer constituents E1-E5) located 1.7 Mb downstream of the Myc promoter. ChIP analysis revealed that H3K36 dimethylation, but not H3K36 mono- and tri-methylation, broadly correlated with NSD3-short occupancy at the Myc E1CE5 super-enhancer (Figures S5B and S5C and data not shown). Using the ectopically expressed FLAG-tagged NSD3-short, we con?rmed the association of this isoform with the Myc E1CE5 super-enhancer using ChIP with anti-FLAG antibodies (Figure 6A).\"\\tSuper-Enhancer\\tMYC\\t--\\t\"ChIP,ChIP-qPCR\"\\t\"BRD4 has been shown previously to regulate Myc expression in AML cell via a super-Enhancer (with individual Enhancer constituents E1-E5) located 1.7 Mb downstream of the Myc promoter. ChIP analysis revealed that H3K36 dimethylation, but not H3K36 mono- and tri-methylation, broadly correlated with NSD3-short occupancy at the Myc E1CE5 super-enhancer (Figures S5B and S5C and data not shown). Using the ectopically expressed FLAG-tagged NSD3-short, we con?rmed the association of this isoform with the Myc E1CE5 super-enhancer using ChIP with anti-FLAG antibodies (Figure 6A).\"\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1740801', 'E_01_239\\t26754925\\t--\\thg19\\tchr1\\t181117806\\t181122451\\tHuman\\tHCT 116\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,4C-seq\"\\t\"ChIP-seq analysis identified two p53 bind-ing sites 11 kb (RE1) and 46 kb (RE2) downstream of the IER5 gene (Fig.?1F).p53 binding to RE2 was strong,while binding to RE1 was relatively weak. In addition, H3K27 acetylation was detected at RE2, and was strongly increased by 5-FU treatment, showing that RE2 is an active enhancer that is highly activated upon DNA damage.We found sequences highly similar to the p53 consensus binding sequence at the center of RE2 (TRANSFAC match score; 0.94) but not in RE1 (TRANSFAC match score; 0.74, Fig.?1G). We generated DNA oligonucleotides representing RE1 and RE2, and created a version of RE2 mutated in the p53 consensus sequence (RE2 mut; sequences shown in Fig.?1G), and cloned each upstream of a luciferase reporter gene containing a minimal promoter. As shown in Fig.?1H, p53 strongly activated a reporter containing RE2, but not RE2 mut or RE1.', 'E_01_240\\t26813977\\t--\\thg19\\tchr6\\t31139599\\t31139921\\tHuman\\tEmbryonic Stem Cell\\tLow throughput\\t\"CRISPR/Cas9,Luciferase Reporter Assay\"\\t\"We designed 1964 sgRNA sequences (Supplemental Table S2) targeting these putative cis-regulatory elements with an average of 11 sgRNA per element (Supplemental Fig. S1A). To eliminate these false positives,we required that a positive cis-regulatory element should have at least two distinct sgRNAs enriched by twofold or more in the eGFP? population in at least three out of the four independent experiments (Supplemental Table S3; Supplemental Fig. S2A). By use of this criterion, no negative control sgRNA passed the filter, while six cis-regulatory elements were identified as positives (Fig. 2A).', 'E_01_241\\t26813977\\t--\\thg19\\tchr6\\t30646843\\t30647217\\tHuman\\tEmbryonic Stem Cell\\tLow throughput\\t\"CRISPR/Cas9,Luciferase Reporter Assay,4C-seq\"\\t\"We designed 1964 sgRNA sequences (Supplemental Table S2) targeting these putative cis-regulatory elements with an average of 11 sgRNA per element (Supplemental Fig. S1A). To eliminate these false positives,we required that a positive cis-regulatory element should have at least two distinct sgRNAs enriched by twofold or more in the eGFP? population in at least three out of the four independent experiments (Supplemental Table S3; Supplemental Fig. S2A). By use of this criterion, no negative control sgRNA passed the filter, while six cis-regulatory elements were identified as positives (Fig. 2A).', 'E_01_242\\t26422397\\t--\\thg19\\tchr18\\t60786011\\t60788011\\tHuman\\tHuman T Lymphoid Cell\\tLow throughput\\t\"3C,ChIP,RT-PCR\"\\t\"In the present study, we used chromosome conformation capture (3C) assays and molecular analyses to further investigate the function of the SATB1-mediated higher-order chromatin structure in co-regulation of the anti-apoptotic BCL2 gene and the pro-apoptotic NOXA gene located 3.4Mb downstream on Chromosome 18.We emonstrated that the mbr enhancer spatially juxtaposed the promoters of BCL2 and NOXA genes through SATB1-mediated chromatin-loop in Jurkat cells. Decreased SATB1 levels switched the mbr-BCL2 loop to mbr-NOXA loop, and thus changed expression of these two genes.', 'E_01_243\\t26813977\\t--\\thg19\\tchr6\\t31126537\\t31126685\\tHuman\\tEmbryonic Stem Cell\\tLow throughput\\t\"CRISPR/Cas9,Luciferase Reporter Assay,4C-seq\"\\t\"We designed 1964 sgRNA sequences (Supplemental Table S2) targeting these putative cis-regulatory elements with an average of 11 sgRNA per element (Supplemental Fig. S1A). To eliminate these false positives,we required that a positive cis-regulatory element should have at least two distinct sgRNAs enriched by twofold or more in the eGFP? population in at least three out of the four independent experiments (Supplemental Table S3; Supplemental Fig. S2A). By use of this criterion, no negative control sgRNA passed the filter, while six cis-regulatory elements were identified as positives (Fig. 2A).', 'E_01_244\\t26934861\\tACP10 Enhancer\\thg19\\tchr8\\t128412386\\t128415392\\tHuman\\t\"LNCaP,C4-2B\"\\tLow throughput\\t4C-seq\\t\"We examined the enrichment of H3K4 mono-,di-or tri-methylation,H3K36 tri-methylation and H3K27Ac marks around 4C interactome,and found that H3K4me1,H3K4me2,and H3K27Ac were enriched around identified 4C sites (+/? 500 kb) in both LNCaP and C4-2B cells.\"\\tEnhancer\\tPOU5F1B\\t4C\\t--\\t\"We examined the enrichment of H3K4 mono-,di-or tri-methylation,H3K36 tri-methylation and H3K27Ac marks around 4C interactome,and found that H3K4me1,H3K4me2,and H3K27Ac were enriched around identified 4C sites (+/? 500 kb) in both LNCaP and C4-2B cells. \"\\t\"OCT4-PG1,OCT4PG1,OTF3C,OTF3P1,POU5F1P1,POU5F1P4,POU5FLC20,POU5FLC8\"\\tProstate Cancer\\tDOID:10283\\tD011471\\t--\\t--\\t--\\t\"NKX3-1,FOXA1,AR\"\\t\"BAPX2,NKX3,NKX3.1,NKX3A,HNF3A,TCF3A,AIS,AR8,DHTR,HUMARA,HYSP1,KD,NR3C4,SBMA,SMAX1,TFM\"\\t\"4C,ChIP-seq\"\\t\"We thus compared the ChIP-seq profiles of the three factors and examined their peaks enrichment around our 4C sites (+ /? 500 kb), revealing that they were significantly enriched around the 4C loci compared to random sites in both LNCaP (NKX3.1: p = 7.214e-7, FOXA1: p = 7.871e-15, AR: p = 5.707e-12; Wilcoxon rank sum test, Fig.?5B) and C4-2B cells (NKX3.1: p = 4.593e-6, FOXA1: p = 6.989e-10, AR: p = 1.786e-11; Wilcoxon rank sum test, Fig.?5B). Therefore, we think these transcription factors can bind to accessible 4C regions and play a critical role in prostate cancer cell function.\"\\t--\\t--\\t--\\t>2KB\\t13967', 'E_01_245\\t26957309\\t--\\thg19\\tchr13\\t40839693\\t40853195\\tHuman\\tCD4+ T Cell\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\tDistinct chromatin signatures of active Enhancers and promoters. Examples of validated CD4+T cell Enhancers (highlighted in yellow) marked by H2BK20ac but not by H3K27ac\\tEnhancer\\tLINC00548\\t--\\tChIP-seq\\tDistinct chromatin signatures of active Enhancers and promoters. Examples of validated CD4+T cell Enhancers (highlighted in yellow) marked by H2BK20ac but not by H3K27acLINC00548\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t77799', 'E_01_246\\t26957309\\t--\\thg19\\tchr9\\t4600913\\t4608373\\tHuman\\tCD4+ T Cell\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\tDistinct chromatin signatures of active Enhancers and promoters. Examples of validated CD4+T cell Enhancers (highlighted in yellow) marked by H2BK20ac but not by H3K27ac\\tEnhancer\\tSLC1A1\\t--\\tChIP-seq\\tDistinct chromatin signatures of active Enhancers and promoters. Examples of validated CD4+T cell Enhancers (highlighted in yellow) marked by H2BK20ac but not by H3K27ac\\t\"DCBXA,EAAC1,EAAT3,SCZD18\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t114217', 'E_01_247\\t26960733\\tC11ORF82 Enhancer\\thg19\\tchr11\\t82608774\\t82612886\\tHuman\\tIMR-90\\tLow+High throughput\\t\"3C-qPCR,Hi-C\"\\t\"To further assess the performance of EpiTensor, we performed chromosome conformation capture coupled with quantitative PCR (3C-qPCR) on 14 randomly selected pairs from EpiTensor. As shown in Fig. 4, eigenvector 1 from tensor decomposition has three peaks, that is, loci i, ii and iii. Locii and iii correspond to the C11orf82 and RAB30-AS1 promoters,respectively, while locus ii corresponds to an active enhancer in IMR90 with H3K4me1/H3K27ac enrichment. \"\\tEnhancer\\tDDIAS\\t3C-qPCR\\tHi-C\\t\"To further assess the performance of EpiTensor, we performed chromosome conformation capture coupled with quantitative PCR (3C-qPCR) on 14 randomly selected pairs from EpiTensor. As shown in Fig. 4, eigenvector 1 from tensor decomposition has three peaks, that is, loci i, ii and iii. Locii and iii correspond to the C11orf82 and RAB30-AS1 promoters,respectively, while locus ii corresponds to an active enhancer in IMR90 with H3K4me1/H3K27ac enrichment. \"\\t\"C11orf82,noxin\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1906', 'E_01_248\\t26968549\\t--\\thg19\\tchr20\\t54937784\\t54938182\\tHuman\\tG1E-ER4 Cells\\tLow+High throughput\\t\"ChIP-seq,ChIP,Luciferase Reporter Assay\"\\t\"ChIP-seq analysis identiied two GATA-1 peaks at intronic region neighboring FAM210B in K562 cells (+1.9 and +4.0, relative to transcription start site) (Fig. 1c, upper) [1]. Quantitative ChIP analysis veriied GATA-1 chromatin occupancy at these loci, based on analyzes of both K562 cells and cord blood-derived primary erythroblasts (Fig. 1c, lower).', 'E_01_249\\t27333824\\tLAMC2 Enhancer\\thg19\\tchr1\\t183149939\\t183150336\\tHuman\\tIntestinal Epithelial Cell\\tLow throughput\\tLuciferase Reporter Assay\\t\"To determine whether CDX2 regulates the basal LAMC2 gene activity, a luciferase reporter gene driven by the LAMC2 promoter with and without the upstream LAMC2 enhancer identi?ed previously [Boyd et al., 2014] was transfected into Caco-2 cells without or with concomitant CDX2 expression plasmid. Over-expression of CDX2 increased the luciferase activity driven by the enhancer-less construct of LAMC2 by 2.5-fold (P?0.028) relative to cells transfected with the LAMC2 reporter vector alone, whereas the LAMC2 promoter with enhancer construct gave a fourfold increase (P?0.006) in reporter signal when co-transfected with CDX2 expression plasmid relative to cells transfected with the LAMC2 promoter/enhancer reporter vector alone (Fig. 1A).', 'E_01_250\\t27602360\\tNgR13\\' Enhancer\\thg19\\tchr22\\t20224442\\t20228467\\tHuman\\tNT2/D1\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"The presence of overlapping H3K4me1, H3K9ac, and DNase I hypersensitivity peaks suggests the presence of an enhancer in the NgR1 3  region. To investigate if this region has potential enhancer activity, chr22: 20224442C20228467 was PCR amplified and cloned into a luciferase reporter gene. Substantial increases in luciferase activity were observed for both constructs in the presence of the sodium channel blocker TTX and the NMDA receptor antagonist D-AP5, which blocks neuronal activity ( fig. 4 b).\"\\tEnhancer\\tRTN4R\\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"The presence of overlapping H3K4me1, H3K9ac, and DNase I hypersensitivity peaks suggests the presence of an enhancer in the NgR1 3  region. To investigate if this region has potential enhancer activity, chr22: 20224442C20228467 was PCR amplified and cloned into a luciferase reporter gene. Substantial increases in luciferase activity were observed for both constructs in the presence of the sodium channel blocker TTX and the NMDA receptor antagonist D-AP5, which blocks neuronal activity ( fig. 4 b).\"\\t\"NGR,NOGOR\"\\t--\\t--\\t--\\t\"The NgR1 3\\' region, which contains rs701428, is a neuronal activity-dependent enhancer, and that the minor A allele of rs701428 is susceptible to regulation of enhancer activity by MYBL2.\"\\t\"Luciferase Reporter Assay,PCR\"\\t\"The NgR1 3  Enhancer is downregulated by neuronal activation induced by KCl, and enhanced by blockade of neuronal activation by TTX and D-AP5.\"\\tMYBL2\\tBMYB; B-MYB\\tEMSA\\t\"To examine if the MYB transcription factor family binds to the region with the schizophrenia risk rs701428-A allele,we performed an electrophoretic mobility shift assay (EMSA). The labeled schizophrenia risk rs701428-A allele probe, but not the labeled reference-G allele probe, formed a binding complex with c-MYB, MYBL1, and', 'E_01_251\\t27651453\\tNGB Enhancer\\thg19\\tchr14\\t77635927\\t77639784\\tHuman\\tSH-SY5Y\\tlow throughput\\t\"3C,Luciferase Reporter Assay,ChIP,CRISPR/Cas9\"\\t\"By chromosome conformation capture we identi?ed two novel DREs located ?70 kb upstream and +100 kb downstream from the NGB gene. ENCODE database showed the presence of DNaseI hypersensitive and transcription factors binding sites in these regions. Further analyses using luciferase reporters and chromatin immunoprecipitation suggested that the ?70 kb region upstream of the NGB gene contained a neuronal-speci?c enhancer and GATA transcription factor binding sites.', 'E_01_252\\t27869826\\tIGF2 Enhancer\\thg19\\tchr11\\t2219531\\t2239062\\tHuman\\tHCC-15\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"We next performed ChIP-Seq at the IGF2 locus, detecting an accumulation of the active chromatin mark H3K27ac at a previously identified IGF2 enhancer66 herein referred to as IGF2 cognate enhancer (Fig. 5a, Supplementary Fig. 11). An even more pronounced H3K27ac peak, however, intersected with an element previously inferred to represent a lineage-specific super-enhancer in CRC cell lines (VACO-400 and VACO-9M)46.To verify enhancer function, we performed luciferase assays revealing enhancer activity of cloned fragments of this previously inferred super-enhancer46 in the HCT116 colon cancer but not in a control (HeLa) cancer cell line (Supplementary Fig. 12 and Supplementary Table 6).', 'E_01_253\\t27932455\\tP2RY2 Enhancer\\thg19\\tchr11\\t72905735\\t72906735\\tHuman\\tMCF-7\\tLow throughput\\t\"smFISH,ChIP,3C\"\"Here we monitored the expression and localization of eRNAs derived from the enhancers of the FOXC1 and P2RY2 loci, as well as the expression of their cognate mRNAs over a time course of estrogen induction in individual MCF-7 cells by smFISH. In the uninduced state,FOXC1 and P2RY2 eRNAs were expressed at low levels, independently of MLL1 and ER . Estrogen treatment induced the transcription of eRNAs and target mRNAs with similar kinetics, and required both chromatin modiication by MLL1 and ER recruitment.', 'E_01_254\\t27932455\\tFOXC1 Enhancer\\thg19\\tchr6\\t1636681\\t1642781\\tHuman\\tMCF-7\\tLow throughput\\t\"smFISH,ChIP\"\\t\"Here we monitored the expression and localization of eRNAs derived from the enhancers of the FOXC1 and P2RY2 loci, as well as the expression of their cognate mRNAs over a time course of estrogen induction in individual MCF-7 cells by smFISH. In the uninduced state,FOXC1 and P2RY2 eRNAs were expressed at low levels, independently of MLL1 and ER. Estrogen treatment induced the transcription of eRNAs and target mRNAs with similar kinetics, and required both chromatin modiication by MLL1 and ER recruitment.\"\\tEnhancer\\tFOXC1\\t--\\t\"smFISH,ChIP\"\\t\"Here we monitored the expression and localization of eRNAs derived from the enhancers of the FOXC1 and P2RY2 loci, as well as the expression of their cognate mRNAs over a time course of estrogen induction in individual MCF-7 cells by smFISH. In the uninduced state,FOXC1 and P2RY2 eRNAs were expressed at low levels, independently of MLL1 and ER. Estrogen treatment induced the transcription of eRNAs and target mRNAs with similar kinetics, and required both chromatin modiication by MLL1 and ER recruitment.\"\\t\"ARA,ASGD3,FKHL7,FREAC-3,FREAC3,IGDA,IHG1,IRID1,RIEG3\"\\t--\\t--\\t--\\teRNA accumulation at enhancerCpromoter loops is not required to sustain target gene transcription.\\t3C\\tChromosome conformation capture (3C) analyses conducted in the presence or absence of Enhancer-derived transcripts proposed that eRNAs mediate EnhancerCpromoter interactions .\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t29051', 'E_01_255\\t27980063\\t\\thg19\\tch3\\t189109942\\t189219942\\tHuman\\tMammary Epithelial Cell MCF10A and MCF12A\\tlow throughput\\t\"qPCR,ChIP\"\\t\"ChIP analyses of BRD4 and RNAPII occupancy revealed that decreased BRD4 occupanc on enhancers in response to JQ1 or OTX015 treatment leads to a signiicant concomitant reduction in RNAPII occupancy on these regions (Figure 4H, I and K, L). While BRD4 occupancy was not significantly affected around the Np63 TSS, the potential enhancers displayed a substantial reduction in the occupancy of both BRD4 and RNAPII (Figure 4HandI).', 'E_01_256\\t27980063\\t\\thg19\\tch3\\t189203942\\t189313942\\tHuman\\tMammary Epithelial Cell MCF10A and MCF12A\\tlow throughput\\t\"qPCR,ChIP\"\\t\"ChIP analyses of BRD4 and RNAPII occupancy revealed that decreased BRD4 occupanc on enhancers in response to JQ1 or OTX015 treatment leads to a signiicant concomitant reduction in RNAPII occupancy on these regions (Figure 4H, I and K, L). While BRD4 occupancy was not significantly affected around the Np63 TSS, the potential enhancers displayed a substantial reduction in the occupancy of both BRD4 and RNAPII (Figure 4HandI).', 'E_01_257\\t27980063\\t\\thg19\\tch1\\t24589812\\t24645812\\tHuman\\tMammary Epithelial Cell MCF10A and MCF12A\\tlow throughput\\t\"qPCR,ChIP\"\\t\"ChIP analyses of BRD4 and RNAPII occupancy revealed that decreased BRD4 occupanc on enhancers in response to JQ1 or OTX015 treatment leads to a signiicant concomitant reduction in RNAPII occupancy on these regions (Figure 4H, I and K, L). While BRD4 occupancy was not significantly affected around the Np63 TSS, the potential enhancers displayed a substantial reduction in the occupancy of both BRD4 and RNAPII (Figure 4HandI).', 'E_01_258\\t27989796\\tSR4 Enhancer\\thg19\\tchr17\\t69541883\\t69542989\\tHuman\\tTM4\\tLow throughput\\t\"Luciferase Reporter Assay,PCR,Transgenic mice,Immunofluorescence\"\\t\"Our previous work involving the genomic analysis of isolated DSD patients revealed a 78 kb minimal sex determining region (RevSex) far upstream of SOX9 that was duplicated in 46,XX and deleted in 46,XY DSDs. It was postulated that RevSex contains a gonadal enhancer. However, the most highly conserved sub-region within RevSex, called SR4, was neither responsive to sex determining factors in vitro nor active in the gonads of transgenic mice, suggesting that SR4 may not be functioning as a testicular enhancer. Interestingly, SR4 transgenic mice showed reporter activity in the genital tubercle, the primordium of the penis and clitoris, a previously unreported domain of Sox9 expression. SOX9 protein was detected in the genital tubercle, notably in the urethral plate epithelium,', 'E_01_259\\t28028313\\tGIMAP Enhancer\\thg19\\tchr7\\t150360500\\t150366500\\tHuman\\tT-acute lymphoblastic leukemia (T-ALL)\\tLow+High throughput\\t\"ChIP-seq,CRISPR/Cas9,PCR\"\\t\"DNA binding of the TAL1 complex was observed between the GIMAP2 and GIMAP6 genes across different TAL1-positive T-ALL cell lines (CCRF-CEM, RPMI-8402 and Jurkat) (Figures 1A and 1B, orange box). Notably, this region was also occupied by NOTCH1, which is another prevalent oncogene in T-ALL, with its binding partner RBPJ/CSL. .Importantly, there was substantial binding of RNA polymerase II (RNAP2) and mediator 1 (MED1) as well as extensive acetylation of histone H3 lysine 27 (H3K27Ac), constituting the formation of a super-enhancer in this region (Figure 1A, red box; Supplementary Figure 1C).', 'E_01_260\\t28082341\\tNBEAL2 Enhancer\\thg19\\tchr3\\t46988575\\t46990242\\tHuman\\tK-562\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,RT-PCR\"\\t\"Chromatin immunoprecipitation sequencing revealed 5 GATA binding sites in a regulatory region 31 kb upstream of NBEAL2 covered by a H3K4Me1 mark indicative of an Enhancer locus.Luciferase reporter constructs containing this region confirmed its Enhancer activity in K562 cells, and mutagenesis of the GATA1 binding sites resulted in significantly reduced Enhancer activity. \"\\tEnhancer\\tNBEAL2\\t--\\t\"Luciferase Reporter Assay,ChIP,RT-PCR\"\\t\"Chromatin immunoprecipitation sequencing revealed 5 GATA binding sites in a regulatory region 31 kb upstream of NBEAL2 covered by a H3K4Me1 mark indicative of an Enhancer locus.Luciferase reporter constructs containing this region confirmed its Enhancer activity in K562 cells, and mutagenesis of the GATA1 binding sites resulted in significantly reduced Enhancer activity. \"\\t\"BDPLT4,GPS\"\\t--\\t--\\t--\\t--\\t--\\t--\\tGATA1\\t\"ERYF1,GATA-1,GF-1,GF1,NF-E1,NFE1,XLANP,XLTDA,XLTT\"\\tImmunoblot\\tImmunoblot analysis for NBEAL2 with a rabbit polyclonal antibody and integrin 3 were performed (left) using total protein MK lysates at days 8 and 11 for the control and day 11 for GATA1 D218Y (right) using total protein MK lysates at day 13 for the control and GATA1 D218G.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t31763', 'E_01_261\\t28086087\\tYY1 Enhancer\\thg19\\tchr12\\t109986000\\t109986710\\tHuman\\tWT Mouse Embryonic Fibroblasts\\tLow+High throughput\\t\"ChIP-seq,ChIP-seq,RT-qPCR\"\\t\"Our data demonstrate that CBP binds to RNAs transcribed close to CBP binding sites such as at the YY1 Enhancer (Figure 6A);recruitment of CBP to active Enhancers results in binding to eRNAs, and that eRNA binding to the HAT domain of CBP stimulates its acetyltransferase activity', 'E_01_262\\t28123038\\t--\\thg19\\tchrX\\t55054584\\t55054678\\tHuman\\tK-562\\tLow throughput\\t\"3C,Luciferase Reporter Assay,RT-qPCR,CRISPR/Cas9\"\\t\"A luciferase reporter assay conirmed the enhancer activity of this GATA site in vitro (Figure 1E).', 'E_01_263\\t28123038\\t--\\thg19\\tchrX\\t55041578\\t55041679\\tHuman\\tK-562\\tLow throughput\\t\"3C,Luciferase Reporter Assay,RT-qPCR,CRISPR/Cas9\"\\t\"In addition to this int-1-GATA site,another GATA site in ALAS2 intron 8 (int-8-GATA) has also been reported to function as an enhancer through luciferase assays (35). Furthermore, the physical proximity between the int-1-GATA and int-8-GATA sites and between the promoter and the int-8-GATA site were veriied by 3C assays (Figure 4H and I), suggesting that the promoter, the int-1-GATA site and the int-8-GATA site could form a potential chromatin enhancer loop.', 'E_01_264\\t28210006\\t--\\thg19\\tchr1\\t92921568\\t92937818\\tHuman\\tHEL\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"Lysine-speci?c demethylase 1 (LSD1, also known as KDM1A) is the ?rst discovered histone lysine (K) demethylase and has a FAD-dependent activity to demethylase H3K4me1 and H3K4me2 but not H3K4me3.NCD25 and NCD38, small-molecule inhibitors of LSD1.', 'E_01_265\\t28244015\\tpCREB Enhancer\\thg19\\tchr3\\t181572712\\t181574712\\tHuman\\tMCF-7\\tLow throughput\\t\"ChIP,qPCR\"\\t\"To determine whether BPA-activated pCREB directly regulates SOX2 , ChIP was performed after a 5-minute treatment with either E2 or BPA. pCREB was enriched at an Enhancer region located 143 kilobases downstream of the SOX2 coding region only after BPA treatment (Figure 6), which correlates with gene expression\"\\tEnhancer\\tSOX2\\t--\\t\"ChIP,qPCR\"\\t\"To determine whether BPA-activated pCREB directly regulates SOX2 , ChIP was performed after a 5-minute treatment with either E2 or BPA. pCREB was enriched at an Enhancer region located 143 kilobases downstream of the SOX2 coding region only after BPA treatment (Figure 6), which correlates with gene expression\"\\t\"ANOP3, MCOPS3\"\\t--\\t--\\t--\\t--\\t--\\t--\\tDST\\t\"BP240,BPA,BPAG1,CATX-15,CATX15,D6S1101,DMH,DT,EBSB2,HSAN6,MACF2\"\\tChIP\\t\"To determine whether BPA-activated pCREB directly regulates SOX2, ChIP was performed after a 5-minute treatment with either E2 or BPA. pCREB was enriched at an Enhancer region located 143 kilobases downstream of the SOX2 coding region only after BPA treatment (Figure 6), which correlates with gene expression.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t144001', 'E_01_266\\t28260788\\tLMO1 Enhancer\\thg19\\tchr11\\t8240851\\t8242851\\tHuman\\t\"T-acute lymphoblastic leukemia (T-ALL),RPMI-8402 and Jurkat\"\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,ChIA-PET\"\\t\"To identify mutations in the T-ALL genome that might signi?cantly alter oncogene expression, we focused our search on aberrant,sample-speci?c enhancers in 10 different human T-ALL cell lines as identi?ed by H3K27ac ChIP-seq. The aberrant active enhancer we identi?ed was present upstream of the LMO1 gene in RPMI-8402 and Jurkat T-ALL cell lines.', 'E_01_267\\t28262837\\tFBP1 Enhancer\\thg19\\tchr9\\t97396388\\t97401553\\tHuman\\tHepatocellular Carcinoma\\tLow+High throughput\\t\"ChIP-seq,ChIP,RT-PCR\"\\t\"We noticed that there is a putative H3K4me1-positive enhancer region in the first intron of the FBP1 gene (Fig.?4E).Importantly, we found that at the same region there is a peak of histone H3 lysine-27 acetylation (H3K27Ac) (Fig.?4E), the level of which often correlates with transcriptional activity of the gene. We first examined whether treatment of the HDAC inhibitor NaBu affects H3K27Ac levels in this region. To this end, we erformed chromatin immunoprecipitation (ChIP) assays using H3K27Ac-specific antibody. As demonstrated in Fig.?4F, the H3K27Ac level was readily detectable in the enhancer region of FBP1 gene in both HepG2 and SK-Hep1 cell lines.However, consistent with the finding that NaBu treatment markedly increased expression of FBP1 mRNA (Fig.?2A), it also significantly increased H3K27Ac level in the FBP1 enhancer (Fig.?4F). Similar to the finding that HDAC1 and HDAC2 knockdown induced FBP1 expression, depletion of these proteins alone or together also largely increased H3K27Ac level at this locus (Fig.?4G). This data indicate that the FBP1 gene locus indeed harbors a regulatory region where H3K27Ac level can be modulated by HDAC proteins. Thus, HDAC inhibitor-induced upregulation of H3K27 acetylation in the enhancer of the FBP1 gene correlates with increased expression of FBP1 induced by HDAC inhibitors.\"\\tEnhancer\\tFBP1\\t--\\t\"ChIP-seq,ChIP,RT-PCR\"\\t\"We noticed that there is a putative H3K4me1-positive enhancer region in the first intron of the FBP1 gene (Fig.?4E).Importantly, we found that at the same region there is a peak of histone H3 lysine-27 acetylation (H3K27Ac) (Fig.?4E), the level of which often correlates with transcriptional activity of the gene. We first examined whether treatment of the HDAC inhibitor NaBu affects H3K27Ac levels in this region. To this end, we erformed chromatin immunoprecipitation (ChIP) assays using H3K27Ac-specific antibody. As demonstrated in Fig.?4F, the H3K27Ac level was readily detectable in the enhancer region of FBP1 gene in both HepG2 and SK-Hep1 cell lines.However, consistent with the finding that NaBu treatment markedly increased expression of FBP1 mRNA (Fig.?2A), it also significantly increased H3K27Ac level in the FBP1 enhancer (Fig.?4F). Similar to the finding that HDAC1 and HDAC2 knockdown induced FBP1 expression, depletion of these proteins alone or together also largely increased H3K27Ac level at this locus (Fig.?4G). This data indicate that the FBP1 gene locus indeed harbors a regulatory region where H3K27Ac level can be modulated by HDAC proteins. Thus, HDAC inhibitor-induced upregulation of H3K27 acetylation in the enhancer of the FBP1 gene correlates with increased expression of FBP1 induced by HDAC inhibitors.\"\\tFBP\\tHepatocellular carcinoma\\tDOID:684\\tD006528\\tHDAC-mediated suppression of FBP1 expression correlated with decreased histone H3K27Ac in the FBP1 enhancer.\\t\"ChIP,RT-PCR\"\\tReal-time PCR analysis of DNA mmunoprecipitated by control IgG or H3K27Ac antibody from HepG2 and SK-Hep1 cells transfected with control or HDAC1- and/or HDAC2-specific siRNAs. Cells harvested for ChIP assay at 48 hours after transfectio.\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t33557', 'E_01_268\\t29379199\\t--\\thg19\\tchr4\\t174441057\\t174442400\\tHuman\\tNeuroblastoma Cell Lines\\tLow+High throughput\\t\"ChIP-seq,Western blot\"\\t\"We observe that two hours after shutdown, MYCN is significantly depleted from promoters and enhancers (Fig. 1f; Supplementary Fig. 2c,e,f) - a result revealed by ChIP-Rx cell-count normalization.Although both traditional ChIP-seq and ChIP-Rx reveal global loss of MYCN at promoters and enhancers, we do observe global loss of additional marks (H3K4me3, CTCF, H3K27ac, and RNA Pol II) at 24 hours post MYCN shutdown only in ChIP-Rx data. In contrast, induction of MYCN in parental SHEP neuroblastoma cells (Fig. 1c,d) results in loading of MYCN at active promoters and enhancers, and is coincident with increases in global H3K27ac. \"\\tEnhancer\\tHAND2\\t--\\tChIP-seq\\t\"ChIP-seq tracks of MYCN and H3K27ac at 0, 2 and 6hrs post MYCN induction at RPL22, HAND2 and the upstream ID2 enhancer.Visual inspection of MYCN load at highly occupied genes revealed a diversity of binding profiles with some genes exhibiting promoter MYCN binding (RPL22) and others a more mixed array of promoter and enhancer binding (HAND2 & ID2). \"\\t\"DHAND2,Hed,Thing2,bHLHa26,dHand\"\\tNeuroblastoma\\tDOID:769\\tD009447\\t--\\t--\\t--\\tMYCN\\t\"MODED,N-myc,NMYC,ODED,bHLHe37\"\\t\"Western blot,ChIP-seq\"\\t\"Across MYCN-amplified neuroblastoma lines, we used chromatin immunoprecipitation coupled to high-throughput sequencing (ChIP-seq) to generate a consensus genome-wide map of ~10,000 regions that exhibit strong and consistent MYCN occupancy.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5922', 'E_01_269\\t28378740\\tCD47 SE E5\\thg19\\tchr3\\t107702615\\t107714185\\tHuman\\tMCF-7\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"By rank-ordering of enhancer regions based on H3K27ac enrichment, we discovered that T-cell acute lymphoblastic leukemia (T-ALL (RPMI18402, Jurkat and MOLT3)) diffuse large B-cell lymphoma (DLBCL (LY4)) and breast cancer (MCF7 and HCC1954) cell lines have SEs within B200 kb of CD47 (Fig. 1a).', 'E_01_270\\t28378740\\tCD47 SE E3.2\\thg19\\tchr3\\t107842838\\t107845592\\tHuman\\tMCF-7\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"By rank-ordering of enhancer regions based on H3K27ac enrichment, we discovered that T-cell acute lymphoblastic leukemia (T-ALL (RPMI18402, Jurkat and MOLT3)) diffuse large B-cell lymphoma (DLBCL (LY4)) and breast cancer (MCF7 and HCC1954) cell lines have SEs within B200 kb of CD47 (Fig. 1a).', 'E_01_271\\t28378740\\tCD47 SE E7\\thg19\\tchr3\\t107846969\\t107854683\\tHuman\\tMCF-7\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"By rank-ordering of enhancer regions based on H3K27ac enrichment, we discovered that T-cell acute lymphoblastic leukemia (T-ALL (RPMI18402, Jurkat and MOLT3)) diffuse large B-cell lymphoma (DLBCL (LY4)) and breast cancer (MCF7 and HCC1954) cell lines have SEs within B200 kb of CD47 (Fig. 1a).', 'E_01_272\\t28446439\\t--\\thg19\\tchr11\\t17666830\\t17677080\\tHuman\\tHuman Fibroblast Cell Lines\\tLow+High throughput\\t\"ChIP-seq,4C,Luciferase Reporter Assay\"\\t\"We found PAX3-FOXO1 most frequently occupied the strong Enhancer chromatin state , exemplified by known PAX3-FOXO1 target FGFR4 and oncogenes MYC, ALK and MET\"\\tSuper-Enhancer\\tMYOD1\\t4C\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"We found PAX3-FOXO1 most frequently occupied the strong Enhancer chromatin state , exemplified by known PAX3-FOXO1 target FGFR4 and oncogenes MYC, ALK and MET\"\\t\"MYF3,MYOD,PUM,bHLHc1\"\\tAlveolar Rhabdomyosarcoma\\tDOID:4051\\tD018232\\t--\\t--\\t--\\t\"PAX3,FOXO1\"\\t\"CDHS,HUP2,WS1,WS3,FKH1,FKHR,FOXO1A\"\\t\"ChIP-qPCR,ChIP-seq\"\\t\"To gain insight into the epigenetic consequences of PAX3-FOXO1, we mapped the landscape of active and repressive histone marks by sequencing DNA enriched by chromatin immunoprecipitation (ChIP-seq) from a patient derived fusion positive FP-RMS cell line RH4.', 'E_01_273\\t28511927\\t--\\thg19\\tchr11\\t355447\\t358949\\tHuman\\t\"A549,HEK-293\"\\tLow throughput\\t\"Luciferase Reporter Assay,CRISPR/Cas9,ChIP,EMSA,3C\"\\t\"Here, through public data mining, episomal luciferase reporter assay and in vivo CRISPR-Cas9 genome editing, we identified an IFN-responsive enhancer located 35 kb upstream of IFITM3 gene promoter upregulating the IFN-induced expression of IFITM1, 2 and 3 genes. Chromatin immunoprecipitation (ChIP), electrophoretic mobility shift assay (EMSA) and luciferase reporter assay demonstrated that signal transducers and activators of transcription (STAT) 1 bound to the enhancer with the treatment of IFN and was indispensable for the enhancer activity. Furthermore, using chromosome conformation capture technique, we revealed that the IFITM1, 2 and 3 genes physically clustered together and constitutively looped to the distal enhancer through long-range interactions in both HEK293 and A549 cells, providing structural basis for coordinated regulation of IFITM1, 2 and 3 by the enhancer. Finally, we showed that in vivo truncation of the enhancer impaired IFN-induced resistance to influenza A virus (IAV) infection. \"\\tEnhancer\\t\"IFITM1,IFITM2,IFITM3\"\\t\"CRISPR/Cas9,3C\"\\t\"Luciferase Reporter Assay,EMSA\"\\t\"Here, through public data mining, episomal luciferase reporter assay and in vivo CRISPR-Cas9 genome editing, we identified an IFN-responsive enhancer located 35 kb upstream of IFITM3 gene promoter upregulating the IFN-induced expression of IFITM1, 2 and 3 genes. Chromatin immunoprecipitation (ChIP), electrophoretic mobility shift assay (EMSA) and luciferase reporter assay demonstrated that signal transducers and activators of transcription (STAT) 1 bound to the enhancer with the treatment of IFN and was indispensable for the enhancer activity. Furthermore, using chromosome conformation capture technique, we revealed that the IFITM1, 2 and 3 genes physically clustered together and constitutively looped to the distal enhancer through long-range interactions in both HEK293 and A549 cells, providing structural basis for coordinated regulation of IFITM1, 2 and 3 by the enhancer.\"\\t\"927,CD225,DSPA2a,IFI17,LEU13,1-8D,DSPA2c,1-8U,DSPA2b,IP15\"\\t--\\t--\\t--\\tIn vivo truncation of the enhancer impaired IFN-induced resistance to influenza A virus (IAV) infection.\\t\"RT-PCR,Western blot\"\\t\"As IFITM1, 2 and 3 are very important for host to resist infection of IAV [4], [8], we infected wild-type and E2-3-truncated HEK293 cells with IAV Puerto Rico/8/1934 (PR8) at a MOI of 0.1 for 24 h. Although IFITM1, 2 and 3 were induced by PR8 virus in both wild-type and E2-3-truncated HEK293 cells, the inductions were greatly impaired upon truncation of E2-3 (Fig. 5A and B), further confirming the regulation of IFITM1, 2 and 3 expression by the enhancer E2-3. Surprisingly, we did not observe a significant effect on the replication of IAV when enhancer E2-3 was truncated, as revealed by the mRNA and protein levels of virus NP and progeny virus titers determined by TCID50 assay (Fig. S9). Nonetheless, when we treated wild-type and E2-3-truncated HEK293 cells with IFN respectively for 24 h before infection of PR8, NP mRNA and protein levels were found to be much higher in mutant HEK293 cells (Fig. 5C and D). Moreover, consistent with virus NP expression, the viral titers were also increased in E2-3-truncated cells (Fig. 5E), indicating that enhancer E2-3 was responsible for IFN-induced resistance to IAV infection.\"\\tSTAT1\\t\"CANDF7,IMD31A,IMD31B,IMD31C,ISGF-3,STAT91\"\\t\"ChIP,EMSA,3C\"\\t\"Chromatin immunoprecipitation (ChIP), electrophoretic mobility shift assay (EMSA) and luciferase reporter assay demonstrated that signal transducers and activators of transcription (STAT) 1 bound to the enhancer with the treatment of IFN and was indispensable for the enhancer activity. Furthermore, using chromosome conformation capture technique, we revealed that the IFITM1, 2 and 3 genes physically clustered together and constitutively looped to the distal enhancer through long-range interactions in both HEK293 and A549 cells, providing structural basis for coordinated regulation of IFITM1, 2 and 3 by the enhancer.', 'E_01_274\\t28536097\\t--\\thg19\\tchr1\\t221049243\\t221051243\\tHuman\\tHUVEC\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,ChIP\"\\t\"We also performed H3K27ac ChIP-seq and found that 94% of conserved ERG peaks overlapped H3K27ac-enriched regions, supporting their association with active enhancers. We also identified an ERG-bound enhancer, conserved in cows and humans, ~3.0 kb upstream of the gene H2.0-Like Homeobox (HLX) (Fig. 8B; Fig. S8).', 'E_01_275\\t28578223\\t--\\thg19\\tchr18\\t61533339\\t61545796\\tHuman\\t\"MonoMac6,K-562\"\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,qRT-PCR\"\\t\"A cluster of RNAs, not previously annotated, was found approximately 21 kb upstream of the SERPINB2 gene in SLE monocytes (Shi et al.,2014). The RNAs (772, 774, 775) formed a cluster near an enhancer, de?ned by H3K27ac in UCSC Genome Browser. 774 is centered on the H3K27ac peak, 775 is at the proximal edge of the H3K27ac peak and 772 is at the distal shores of the H3K27ac peak. We will refer to this region as Enhancer 1.', 'E_01_276\\t28637769\\t--\\thg19\\tChr11\\t45846943\\t45846959\\tHuman\\tU-2 OS\\tLow throughput\\tLuciferase Reporter Assa\"To identify components necessary for cell-autonomous circadian transcription of the hCry2 gene, a 3-kbp 5\\'-flanking region upstream from the hCry2 transcription start site was isolated and subcloned into the upstream of the luciferase gene, and the hCry2-driven luciferase activity was monitored in real time using a cell-based system (Figure 1A).To exclude systemic factors such as blood-borne factors and body temperature, it was necessary to examine cell-autonomous transcriptional regulatory mechanisms under an in vitro culture condition. The data clearly showed a robust circadian oscillation of the bioluminescence in U2OS human steosarcoma cells (Figure 1B), indicating that the cloned genomic region contained elements essential for circadian transcription.The 60-bp region is functionally divided into two; a 43-bp region (from -158 to -116) contains elements which enhance oscillation amplitude, while the 17-bp region (from -175 to -159) is indispensable for the generation of transcriptional oscillation.', 'E_01_277\\t28717659\\tSIX6 Enhancer\\thg19\\tchr14\\t60973427\\t60975430\\tHuman\\tEye\\tlow throughput\\t\"Transgenic zebrafish,Luciferase Reporter Assay\"\\t\"A evolutionarily conserved enhancer has been shown to control Six6 expression in the vertebrate retina (Conte et al. 2010).We ?rst inserted the entire control human enhancer sequence in the ZED transgenic vector (Bessa et al. 2009) and investigated if the plasmid could drive GFP reporter expression in the retina of zebra?sh embryos, similarly to what has been previously observed for the medaka orthologous region (Conte et al. 2010). We generated 38 F0 transgenic embryos, as determined by RFP expression in the somites (Fig. 4C) that acts as a positive control (Bessa et al.2009). About half of them (18) showed a clear GFP expression driven by the human enhancer element in the eye and occasionally in other CNS regions.\"\\tEnhancer\\tSIX6 \\t--\\tTransgenic zebrafish\\t\"A evolutionarily conserved enhancer has been shown to control Six6 expression in the vertebrate retina (Conte et al. 2010).We ?rst inserted the entire control human enhancer sequence in the ZED transgenic vector (Bessa et al. 2009) and investigated if the plasmid could drive GFP reporter expression in the retina of zebra?sh embryos, similarly to what has been previously observed for the medaka orthologous region (Conte et al. 2010). We generated 38 F0 transgenic embryos, as determined by RFP expression in the somites (Fig. 4C) that acts as a positive control (Bessa et al.2009). About half of them (18) showed a clear GFP expression driven by the human enhancer element in the eye and occasionally in other CNS regions.\"\\t\"MCOPCT2,ODRMD,OPTX2,Six9\"\\tPrimary Open-Angle Glaucoma\\tDOID:13550\\tD005902\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\trs33912345\\t60976537\\tLuciferase Reporter Assay\\t<2KB\\t\\t\\t\\t1509', 'E_01_278\\t28737489\\t--\\thg19\\tchr3\\t181432912\\t181433312\\tHuman\\tLung Adenocarcinoma\\tlow throughput\\t\"Luciferase Reporter Assay,ChIP-qPCR\"\\t\"To further delineate the molecular mechanism of SOX2 regulation by NFATc2, we screened, in silico, the genomic sequences spanning 5 kb up- and downstream of the SOX2 transcription start site (TSS), which identified 4 regions encompassing multiple conserved NFAT binding sequences (Figure 6-figure supplement 1A). Luciferase assays confirmed these regions are active transcriptional regulatory regions (Figure 6-figure supplement 1B). Further evaluation with respective SOX2 luciferase reporter revealed transcriptional activities were mediated by sites 1, 2, 4, and 5 (Figure 6B). Using H441 lung cancer cell line with NFATc2 transient overexpression, we observed only sites 1, 4 and 5 showed statistically significant increased reporter activities while those of sites 4 and 5 were reciprocally abolished by CSA treatment (Figure 6C).\"Enhancer\\tSOX2 \\t--\\t\"Luciferase Reporter Assay,ChIP-qPCR\"\\t\"To further delineate the molecular mechanism of SOX2 regulation by NFATc2, we screened, in silico, the genomic sequences spanning 5 kb up- and downstream of the SOX2 transcription start site (TSS), which identified 4 regions encompassing multiple conserved NFAT binding sequences (Figure 6-figure supplement 1A). Luciferase assays confirmed these regions are active transcriptional regulatory regions (Figure 6-figure supplement 1B). Further evaluation with respective SOX2 luciferase reporter revealed transcriptional activities were mediated by sites 1, 2, 4, and 5 (Figure 6B). Using H441 lung cancer cell line with NFATc2 transient overexpression, we observed only sites 1, 4 and 5 showed statistically significant increased reporter activities while those of sites 4 and 5 were reciprocally abolished by CSA treatment (Figure 6C).', 'E_01_279\\t28751304\\t--\\thg19\\tchr7\\t129854050\\t129895632\\tHuman\\tAdipose Progenitor Cell\\tLow+High throughput\\t\"ChIP-seq,FISH\"\\t\"We also searched for putative MIR335 short- and long-range Enhancers:to this end, we combined chromatin state modeling data from ENC ODE ChIP-seq experiments showing enrichment in H3K4me1 and/or H3K27ac, histone modifications found on inactive and active Enhancers, respectively (Fig. S5 A), with ENCODE genome-wide chromosome conformation capture of Hi-C data (Rao et al., 2014). The Hi-C data reveal multiple pairwise interactions of these predicted enhancers with the MIR335', 'E_01_280\\t28753427\\t--\\thg19\\tchr6\\t12490000\\t12590000\\tHuman\\tHUVEC\\tLow throughput\\t\"4C-seq,Luciferase Reporter Assay\"\\t\"The newly released ENCODE phase 3 data, however, includes four aortic artery samples. In these aortic samples, a strong enhancer signal overlies rs9349379 (Figure 2B), spanning 1 kb.', 'E_01_281\\t28881808\\t--\\thg19\\tchr4\\t77511788\\t77605824\\tHuman\\t\"LNCaP,CWR22Rv1\"\\tLow+High throughput\\t\"ChIP-seq,Dual-Luciferase Reporter Assay,ChIP,PCR\"\\t\"By analyzing AR ChIP-seq data in LNCaP cells, we found that there are two obvious AR binding peaks located within CXCL13 intron I.Corresponding to these two peaks of CHIP-seq results, there were two AR-binding sites (called as ARBS-1 and ARBS-2) in CXCL13 intron I.', 'E_01_282\\t28887413\\t--\\thg19\\tchr4\\t94419116\\t94437803\\tHuman\\t\"231/ERWt Cells,231/ERLQ Cells\"\\tLow+High throughput\\t\"3C-qPCR,ChIP-seq,ChIP-qPCR\"\\t3C-qPCR assays showing chromatin looping from a distal ER-binding site to the OTUB2 promoter in 231/ERWt and 231/ERLQ cells with 45 min of E2 treatment. \\tEnhancer\\tOTUB2\\t3C-qPCR\\t\"ChIP-seq,ChIP-qPCR\"\\t3C-qPCR assays showing chromatin looping from a distal ER-binding site to the OTUB2 promoter in 231/ERWt and 231/ERLQ cells with 45 min of E2 treatment. \\t\"C14orf137,OTB2,OTU2\"\\tBreast Cancer\\tDOID:1612\\tD001943\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t64263', 'E_01_283\\t28916223\\tDGKA Enhancer \\thg19\\tchr12\\t56330413\\t56330909\\tHuman\\tA9\\tLow+High throughput\\t\"ChIP-seq,PCR,ChIP\"\\t\"Analysis of publicly available data from the ENCODE project [18] (Fig. 1A), as well as our own ChIP experiments [8] revealed that the DGKA DMR carries H3K27ac. ChIP showed that JQ1 reduced the deposition of H3K27ac levels specifically at the DGKA enhancer site(DMR), but did not affect the upstream DGKA promoter sites 1 and 2 (Fig. 2C). \"\\tEnhancer\\tDGKA\\t--\\t\"ChIP,PCR\"\\t\"From various NHDFs,we selected three fibroblast lines (data not shown) with significant', 'E_01_284\\t28916725\\t\"rs6758183', 'E_01_285\\t28916725\\t\\thg19\\tchr6\\t31092587\\t31094587\\tHuman\\tHuman Corneal Epithelial Cells\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"Because of their importance in controlling cell typeCspecific gene expression, we also defined SEs, using the strength of the H3K27ac mark as described previously (5, 6). We identified 1154 SEs and 12,424 TEs in corneal epithelial cells as well as 76,205 distal regulatory regions, marked by H3K4me1 alone (Fig. 1A).', 'E_01_286\\t28973302\\t--\\thg19\\tchr8\\t27455903\\t27456103\\tHuman\\tHEK-293\\tLow throughput\\t\"Luciferase Reporter Assay,CRISPR/Cas9,EMSA,ChIP\"\\t\"We have previously shown that the alleles at rs2279590 differentially regulate clusterin (CLU) gene expression in lens capsule tissues. This polymorphism resides in an active regulatory region marked by H3K27Ac and DNase I hypersensitive site and is an eQTL for CLU expression. Here, we report the presence of an enhancer element in surrounding region of rs2279590. Deletion of a 115 base pair intronic region flanking the rs2279590 variant through CRISPRB Cas9 genome editing in HEK293 cells demonstrated a decreased clusterin gene expression. Electrophoretic mobility shift and chromatin immunoprecipitation assays show that rs2279590 with allele A constitutes a transcription factor binding site for heat shock factorB1 (HSF1) but not with allele G. By binding to allele A, HSF1 abrogates the enhancer effect of the locus as validated by reporter assays. Interestingly, rs2279590 locus have a widespread enhancer effect on two nearby genes, Protein tyrosine kinase 2 beta (PTK2B) and epoxide hydrolaseB2 (EPHX2); both of which have been previously associated with AD as risk factors.\"\\tEnhancer\\t\"CLU,EPHX2,PTK2BCRISPR/Cas9\\t\"Luciferase Reporter Assay,EMSA,ChIP\"\\t\"We have previously shown that the alleles at rs2279590 differentially regulate clusterin (CLU) gene expression in lens capsule tissues. This polymorphism resides in an active regulatory region marked by H3K27Ac and DNase I hypersensitive site and is an eQTL for CLU expression. Here, we report the presence of an enhancer element in surrounding region of rs2279590. Deletion of a 115 base pair intronic region flanking the rs2279590 variant through CRISPRB Cas9 genome editing in HEK293 cells demonstrated a decreased clusterin gene expression. Electrophoretic mobility shift and chromatin immunoprecipitation assays show that rs2279590 with allele A constitutes a transcription factor binding site for heat shock factorB1 (HSF1) but not with allele G. By binding to allele A, HSF1 abrogates the enhancer effect of the locus as validated by reporter assays. Interestingly, rs2279590 locus have a widespread enhancer effect on two nearby genes, Protein tyrosine kinase 2 beta (PTK2B) and epoxide hydrolaseB2 (EPHX2); both of which have been previously associated with AD as risk factors.\"\\t\"AAG4,APO-J,APOJ,CLI,CLU1,CLU2,KUB1,NA1/NA2,SGP-2,SGP2,SP-40,TRPM-2,TRPM2,CADTK,CAKB,FADK2,FAK2,PKB,PTK,PYK2,RAFTK\"\\tAlzheimer\\'s Disease\\tDOID:10652\\tD000544\\t--\\t--\\t--\\tHSF1\\tHSTF1\\t\"EMSA,ChIP\"\\t\"Electrophoretic mobility shift and chromatin immunoprecipitation assays show that rs2279590 with allele \"\"A\"\"constitutes a transcription factor binding site for heat shock factor-1 (HSF1) but not with allele G.By binding to allele A, HSF1 abrogates the Enhancer effect of the locus as validated by reporter assays.\"\\trs2279590\\t27456253\\t\"EMSA,ChIP\"\\tIntron\\t\\t\\t\\t1570', 'E_01_287\\t28991225\\t PGC-1 super-Enhancer\\thg19\\tchr4\\t23811733\\t23994829\\tHuman\\tCHL-1\\tLow+High throughput\\tChIP-seq\\tWe performed chromatin immunoprecipitation followed by DNA sequencing (ChIP-seq) using antibodies against H3K27ac and BRD4. \\tSuper-Enhancer\\tPPARGC1A\\t--\\tChIP-seq\\tWe performed chromatin immunoprecipitation followed by DNA sequencing (ChIP-seq) using antibodies against H3K27ac and BRD4.\\t\"LEM6,PGC-1(alpha),PGC-1alpha,PGC-1v,PGC1,PGC1A,PPARGC1\"\\tMelanoma\\tDOID:1909\\tD008545\\tBRD4 binding at super-enhancers that drive the expression of PGC-1 and SOX10\\t\"ChIP-seq,ChIP-qPCR\"\\t\"We provide evidence for a direct role of BRD4 binding at super-enhancers that drive the expression of PGC-1 and SOX10, a transcription factor involved in melanocyte development. \"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t109638', 'E_01_288\\t29035388\\tHGE(HER2 gene body Enhancer )\\thg19\\tchr17\\t37879571\\t37880263\\tHuman\\tK-562\\tLow throughput\\t\"DNaseI-seq,ChIP-PCR,ChIP,Luciferase Reporter Assay\"\\t\"With the ENCODE Regulation Super-track Settings, DNase I hypersensitivity sites, H3K4me1 and H3K27ac were found in the previously identified intron 1 enhancer.Chromatin immunoprecipitation (ChIP)-PCR amplicons are shown. NC: sequences to which nonspecific controls were generated.\"\\tEnhancer\\tERBB2\\t--\\t\"ChIP,Luciferase Reporter Assay\"\\t\"Chromatin immunoprecipitation of TFAP2C, H3K4me1 and H3K27ac were performed in SKBR3 cells and TFAP2C occupancies were confirmed at promoters 1 and 2, the intron 1 enhancer and HE.We tested the ability of the HGE to regulate the transcriptional activities of the HER2 promoters by luciferase reporter assay in 293T, SKBR3 and BT474 cells (Figure 2a). \"\\t\"CD340,HER-2,HER-2/neu,HER2,MLN19,NEU,NGL,TKR1\"\\tBreast Cancer\\tDOID:1612\\tD001943\\t--\\t--\\t--\\tTFAP2C\\t\"AP2-GAMMA,ERF1,TFAP2G,hAP-2g\"\\tCRISPR/Cas9\\tThe transcriptional activities of these constructs were analyzed using the luciferase assay in 293T and SKBR3 cells.We next carried out genomic editing using CRISPR-Cas9 system to determine the role of the TFAP2C-binding sites at the HGE in the regulation of HER2 expression.\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t35581', 'E_01_289\\t29091765\\t--\\thg19\\tchr3\\t180728712\\t180730712\\tHuman\\tNeural Stem Cells \\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR,4C-seq,Hi-C\"\\t\"To understand if loss of CTCF binding affects chromatin contacts at the SOX2 locus, we performed circularized chromosome conformation capture (4C-Seq)', 'E_01_290\\t29129929\\tBACH2 Enhancer\\thg19\\tchr6\\t91004796\\t91006944\\tHuman\\tB Cell\\tlow throughput\\t\"RT-PCR,ChIP-qPCR,Luciferase Reporter Assay\"\\t\"DNA immunoprecipitated by total ELK1 antibody or immunoglobulin G (IgG CTL) was amplified by real-time PCR using primers flanking the putative ELK1 binding site in BACH2 intron 1 (predicted BACH2 enhancer position), known ELK1 binding regions in MCL1, MYD88 and FOS genes, and negative control primers as a reference (negative site). ChIP analysis of histone modifications and p300 binding in IL-2-primed D3 B cells. We demonstrated an in vivo binding of ELK1 to this putative enhancer by chromatin immunoprecipitation followed by QPCR (ChIP-QPCR) in IL2-primed B cells collected at D3.\"\\tEnhancer\\tBACH2\\t--\\tLuciferase Reporter Assay\\t\"We decided to detail the functional DNA regulatory sequence by luciferase reporter assay in the context of primary human naive B cell activation. The BACH2 proximal promoter41 (minPBACH2, ?725; +146) ligated upstream from the nano-luciferase coding sequence (NanoLuc) demonstrated a 4.2??0.2 (n?=?15) fold more transcriptional activity than the promoterless NanoLuc vector (pNL1.1), whatever the time point of B-cell electroporation (from D0 to D3). The enhancer region (+1265; +1493) was inserted in the native and reverse orientation downstream from the minPBACH2. \"\\t\"BTBD25,IMD60\"\\t--\\t--\\t--\\t--\\t--\\t--\\tELK1\\tELK1\\tChIP-qPCR\\t\"ChIP assay of the in vivo binding of ELK1 in IL-2-primed D3 B cells. DNA immunoprecipitated by total ELK1 antibody or immunoglobulin G (IgG CTL) was amplified by real-time PCR using primers flanking the putative ELK1 binding site in BACH2 intron 1 (predicted BACH2 enhancer position), known ELK1 binding regions in MCL1, MYD88 and FOS genes, and negative control primers as a reference (negative site). Means  s.e.m. of four independent experiments.\"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t369624', 'E_01_291\\t29142074\\t--\\thg19\\tchr7\\t94002773\\t94005073\\tHuman\\tLung Myofibroblast\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\tThe structure of the COL1A2 far-upstream enhancer and the HS4 region is shown in Figure 1A.Enhancer activation was determined by measuring luciferase expression.\\tEnhancer\\tCOL1A2\\t--\\t\"Luciferase Reporter Assay,ChIP\"\\t\"We created two different luciferase reporter gene constructs under the control of the COL1A2 enhancer and proximal promoter.When the enhancer is engaged, it contacts the RNA polymerase complex forming at the collagen promoter. Thus, an RNA polymerase ChIP signal is a direct measure of enhancer engagement. \"\\t\"EDSARTH2,EDSCV,OI4\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSTAT3\\t\"ADMIO,ADMIO1,APRF,HIES\"\\tChIP-qPCR\\t\"ChIP analysis using qPCR for the binding of STAT3, JunB, and RNA polymerase II, in the HS4 region of SSc fibroblasts after treatment with SD1029, STAT3 siRNA (siSTAT3), or nontargeting siRNA (siNT). * = p < 0.05.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t19949', 'E_01_292\\t29149598\\t--\\thg19\\tchr12\\t47471429\\t47487762\\tHuman\\tSKmel147\\tLow+High throughput\\t\"ATAC-seq,ChIP-seq,Luciferase Reporter Assay,Transfection,CRISPR/Cas9\"\\t\"Using Assay for Transposase-Accessible Chromatin (ATAC)-seq (Buenrostro et al., 2013) and ChIP-seq for H3K4me1 and H3K27ac (Creyghton et al., 2010), we identified ~1,400 constituent enhancers within the SEs of melanoma cells (Table S5) and found them largely devoid of open chromatin, H3K27ac, and H3K4me1 in NHMs (Figures 5F and ?and5G).To test the functionality of the predicted enhancers (Figures 6B and ?and7A), we cloned E1CE5 upstream of a minimal promoter driving luciferase expression (Prescott et al., 2015) and trans-fected these constructs into SKmel147 cells. We next used CRISPR/Cas9 editing to individually delete the genomic regions containing E2CE3 and E4CE5.\"\\tSuper-Enhancer\\tAMIGO2\\tCRISPR/Cas9\\tLuciferase Reporter Assay\\t\"Schematic of AMIGO2 SE showing enhancers E1CE5 (B), deleted sequences for CRISPR-Cas9 editing (C), and DNA segments used for E3 luciferase assay (D). Luciferase reporter assays performed in SKmel147 cells for AMIGO2 enhancer elements E3ACC and E3A deleted for TF motifs (E3A-Del) \"\\t\"ALI1,AMIGO-2,DEGA\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"FOSL2,TEAD4\"\\t\"FRA2EFTR-2, RTEF1, TCF13L1, TEF-3, TEF3, TEFR-1, hRTEF-1B\"\\tImmunoblot\\t\"FOSL2 (left), TEAD4 (right), and AMIGO2 immunoblots of SKmel147 cells 96 hr post-transduction with shSCR or shFOSL2 (shF2 #1 and #2) and shSCR or shTEAD4+TEAD1 (shT4 #1+shT1 #1 and shT4 #2+shT1 #1). HSP90 and GAPDH were used as loading controls.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t10107', 'E_01_293\\t29151363\\t--\\thg19\\tchr19\\t34845032\\t34847032\\tHuman\\tProstate Cancer Cell Lines\\tLow+High throughput\\t\"WHG-STARR-seq,ChIP-seq,Luciferase Reporter Assay,ATAC-seq,RNA-seq\"\\t\"In the current study, we use a human prostate cancer cell line, LNCaP as a model to perform whole human genome STARR-seq (WHG-STARR-seq) to reliably obtain an assessment of enhancer activity.Furthermore, these datasets can only identify putative enhancers based on TF and histone chromatin immunoprecipitation (ChIP-seq) data and therefore do not necessarily exhaustively define functional enhancers, leading to false negatives. In order to validate the function of the active enhancers determined by WHG-STARR-seq, we selected 15 regions for all activity measurements and measured their activity using traditional Renilla luciferase reporter assays.\"\\tEnhancer\\tGPI\\t--\\t\"ATAC-seq,mRNA-seq,WHG-STARR-seq\"\\t\"Following treatment with TSA, we performed assay for transposase accessible chromatin sequencing (ATAC-seq) and messenger RNA-sequencing (mRNA-seq) experiments to measure the genome-wide chromatic accessibility and gene expression levels, respectively.Genomic snapshot displaying the GPI locus region as detected by WHG-STARR-seq. There is a strong enhancer region approximately 10 kb upstream of GPI transcriptional start site and another weaker enhancer region in the 3UTR of GPI. \"\\t\"AMF,GNPI,NLK,PGI,PHI,SA-36,SA36\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t9612', 'E_01_294\\t29155818\\t--\\thg19\\tchr13\\t28545205\\t28547205\\tHuman\\tColorectal Cancer\\tLow+High throughput\\t\"ChIP-seq,ChIP,EMSA,Luciferase Reporter Assay\"\\t\"To identify candidate cis-regulatory elements targeted by FOXA factors, we performed literature search, interrogated the ECR browser for evolutionary conserved regions (ECRs), and integrated this information with the results of the FOXA1 and FOXA2 ChIP-seq analysis.Moreover, the ENCODE data base lists several FOXA1 ChIP-seq peaks at the CDH1 locus in different cell lines, especially in the second intron. Notably, at the +7.8 kb enhancer region a FOXA1 ChIP-seq signal overlapped with an ECR (Fig 5A). To find out if FOXA factors could be involved in the function of the intronic CDH1 enhancer in CRC cells, we performed ChIP analyses.\"\\tEnhancer\\tCDX2\\t--\\t\"ChIP,Luciferase Reporter Assay\"\\t\"To test for binding of FOXA proteins at this site of the CDX2 locus in CRC cells we performed ChIP analyses. Indeed, this region was occupied by FOXA1 in LS174T and HT29 cells. Furthermore, FOXA3 was bound at the CDX2 +10.0 kb region in LS174T, SW480 and HT29 cells (Fig 5D). Even though binding did not correlate with CDX2 expression in the CRC cells (S10 Fig, panel B), this result suggests that FOXA proteins act upon a transcriptional enhancer at the CDX2 locus in CRC cells.To test the importance of the FOXA binding sites for CDH1, CDX2 and EPHB3 enhancer activity, we performed luciferase reporter assays in LS174T cells.\"\\t\"CDX-3,CDX2/AS,CDX3\"\\t--\\t--\\t--\\t--\\t--\\t--\\tFOXA1\\t\"HNF3A,TCF3A\"\\t\"EMSA,Luciferase Reporter Assay\"\\t\"EMSA to test for in vitro binding of FOXA1 and FOXA3 to the CDX2 +10.0 kb enhancer. (C) (Top) Scheme of the EPHB3 ?2.3 kb enhancer with its known transcription factor binding sites. TBE: TCF/LEF-binding element. (Bottom) EMSA to test for in vitro binding of FOXA1 and FOXA3 to the EPHB3 ?2.3 kb enhancer. (D, E, F) Luciferase reporter assay in LS174T cells with constructs covering the CDH1 (D), the CDX2 (E), and the EPHB3 (F) enhancer. Mutations of the respective FOX binding motifs are indicated by red crosses. Shown is the mean and SEM; n_x0015_3. RLA: relative luciferase activity. Statistical significance was calculated relative to the wild-type luciferase reporter constructs. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t10001', 'E_01_295\\t29155818\\t--\\thg19\\tchr16\\t68777928\\t68779928\\tHuman\\tColorectal Cancer\\tLow+High throughput\\t\"ChIP-seq,ChIP,EMSA,Luciferase Reporter Assay\"\\t\"To identify candidate cis-regulatory elements targeted by FOXA factors, we performed literature search, interrogated the ECR browser for evolutionary conserved regions (ECRs), and integrated this information with the results of the FOXA1 and FOXA2 ChIP-seq analysis.Moreover, the ENCODE data base lists several FOXA1 ChIP-seq peaks at the CDH1 locus in different cell lines, especially in the second intron. Notably, at the +7.8 kb enhancer region a FOXA1 ChIP-seq signal overlapped with an ECR (Fig 5A). To find out if FOXA factors could be involved in the function of the intronic CDH1 enhancer in CRC cells, we performed ChIP analyses.\"\\tEnhancer\\tCDH1\\t--\\t\"EMSA,Luciferase Reporter Assay\"\\t\"Since occupancy of the CDH1 +7.8 kb and EPHB3 ?2.3 kb enhancers by FOXA factors had not yet been described, we validated binding of FOXA1 and FOXA3 to the predicted FOXA binding motifs at these enhancers in vitro by EMSAs.To test the importance of the FOXA binding sites for CDH1, CDX2 and EPHB3 enhancer activity, we performed luciferase reporter assays in LS174T cells.\"\\t\"Arc-1,BCDS1,CD324,CDHE,ECAD,LCAM,UVO\"\\t--\\t--\\t--\\t--\\t--\\t--\\tFOXA1\\t\"HNF3A,TCF3A\"\\t\"EMSA,Luciferase Reporter Assay\"\\t\"EMSA to test for in vitro binding of FOXA1 and FOXA3 to the CDX2 +10.0 kb enhancer. (C) (Top) Scheme of the EPHB3 ?2.3 kb enhancer with its known transcription factor binding sites. TBE: TCF/LEF-binding element. (Bottom) EMSA to test for in vitro binding of FOXA1 and FOXA3 to the EPHB3 ?2.3 kb enhancer. (D, E, F) Luciferase reporter assay in LS174T cells with constructs covering the CDH1 (D), the CDX2 (E), and the EPHB3 (F) enhancer. Mutations of the respective FOX binding motifs are indicated by red crosses. Shown is the mean and SEM; n_x0015_3. RLA: relative luciferase activity. Statistical significance was calculated relative to the wild-type luciferase reporter constructs. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t7736', 'E_01_296\\t29155818\\t--\\thg19\\tchr3\\t184276287\\t184278287\\tHuman\\tColorectal Cancer\\tLow+High throughput\\t\"ChIP-seq,ChIP,EMSA,Luciferase Reporter Assay\"\\t\"To identify candidate cis-regulatory elements targeted by FOXA factors, we performed literature search, interrogated the ECR browser for evolutionary conserved regions (ECRs), and integrated this information with the results of the FOXA1 and FOXA2 ChIP-seq analysis.Moreover, the ENCODE data base lists several FOXA1 ChIP-seq peaks at the CDH1 locus in different cell lines, especially in the second intron. Notably, at the +7.8 kb enhancer region a FOXA1 ChIP-seq signal overlapped with an ECR (Fig 5A). To find out if FOXA factors could be involved in the function of the intronic CDH1 enhancer in CRC cells, we performed ChIP analyses.\"\\tEnhancer\\tEPHB3\\t--\\t\"ChIP,Luciferase Reporter Assay\"\\t\"In line with this, we detected by ChIP FOXA1 occupancy at the EPHB3 ?2.3 kb enhancer in LS174T and HT29 cells (Fig 5F). Furthermore, FOXA3 was found to be present at the EPHB3 enhancer at -2.3 kb in all CRC cells except for HCT116 cells.To test the importance of the FOXA binding sites for CDH1, CDX2 and EPHB3 enhancer activity, we performed luciferase reporter assays in LS174T cells.\"\\t\"EK2,ETK2,HEK2,TYRO6\"\\t--\\t--\\t--\\t--\\t--\\t--\\tFOXA1\\t\"HNF3A,TCF3A\"\\t\"EMSA,Luciferase Reporter Assay\"\\t\"EMSA to test for in vitro binding of FOXA1 and FOXA3 to the CDX2 +10.0 kb enhancer. (C) (Top) Scheme of the EPHB3 ?2.3 kb enhancer with its known transcription factor binding sites. TBE: TCF/LEF-binding element. (Bottom) EMSA to test for in vitro binding of FOXA1 and FOXA3 to the EPHB3 ?2.3 kb enhancer. (D, E, F) Luciferase reporter assay in LS174T cells with constructs covering the CDH1 (D), the CDX2 (E), and the EPHB3 (F) enhancer. Mutations of the respective FOX binding motifs are indicated by red crosses. Shown is the mean and SEM; n_x0015_3. RLA: relative luciferase activity. Statistical significance was calculated relative to the wild-type luciferase reporter constructs. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t2299', 'E_01_297\\t29236325\\t--\\thg19\\tchr4\\t55859107\\t55860311\\tHuman\\t\"Kasumi-1,U937-AE\"\\tLow throughput\\t\"PCR,Luciferase Reporter Assay,ChIP-qPCR,ChIP-3C\"\\t\"Intriguingly, we also identified a DHS site located at 30 kb downstream of TSS, which was also enriched by p300 (GSE6284728) and thus defined as the intronic enhancer (Figure 2a).To verify whether these co-factors binding to the promoter and intron regions of cKIT, we performed ChIP-qPCR assays with a series of primers for these regions using t(8;21) positive Kasumi-1 cells (Figure 2c).\"\\tEnhancer\\tKIT\\tChIP-3C\\t\"Luciferase Reporter Assay,ChIP-qPCR\"\\t\"We firstly detected the luciferase activity of c-KIT promoter or promoter plus intronic enhancer in Kasumi-1 cells.As shown in Figure 4c, the activity of c-KIT promoter plus intronic enhancer (pGL3-c-KIT-P+I) was significantly higher than that of the c-KIT promoter (pGL3-c-KIT-P).The motif analysis (Figure 2b) as well as', 'E_01_298\\t29326336\\t--\\thg19\\tchr10\\t63525013\\t63527013\\tHuman\\tT-acute lymphoblastic leukemia (T-ALL)\\tLow+High throughput\\t\"ChIA-PET,CRISPR/Cas9\"\\t\"A ChIP-seq analysis of the ARID5B protein showed that ARID5B binds directly to the NOTCH-driven MYC enhancer (N-Me), which has been reported previously to be activated by NOTCH1 in T-ALL cells.Analysis of transcriptional activity using a luciferase reporter assay with the N-Me sequence also demonstrated that ARID5B knockdown significantly inhibited the enhancer activity.Hence, these results indicated that ARID5B directly activates the expression of MYC in T-ALL.\"\\tEnhancer\\tARID5B\\tCRISPR/Cas9\\tChIA-PET\\t\"Importantly, a chromatinCchromatin interaction analysis by paired-end tag sequencing (ChIA-PET) for the cohesin protein demonstrated a cis-regulatory interaction between the ?135-kb element and the ARID5B transcriptional start site (TSS).In fact, genetic knockout of the ?135-kb element by the CRISPR/Cas9 method resulted in a marked down-regulation of ARID5B gene expression.Thus, these results indicate that the TAL1-bound region at ?135 kb is the enhancer controlling ARID5B expression in T-ALL cells.\"\\t\"DESRT, MRF-2, MRF2\"\\tT-Cell Acute Lymphoblastic Leukemia\\tDOID:5602\\tD054218\\tThe TAL1-bound region at ?135 kb is the Enhancer controlling ARID5B expression in T-ALL cells.\\tCRISPR/Cas9\\t\"In fact, genetic knockout of the -135-kb element by the CRISPR/Cas9 method resulted in a marked down-regulation of ARID5B gene expression\"\\tTAL1\\t\"SCL,TCL5,bHLHa17,tal-1\"\\tCRISPR/Cas9\\t\"In fact, genetic knockout of the ?135- kb element by the CRISPR/Cas9 method resulted in a marked down-regulation of ARID5B gene expression (Fig. 1G; Supplemental Fig. S1BCE). Thus, these results indicate that the TAL1-bound region at ?135 kb is the enhancer controlling ARID5B expression in T-ALL cells. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t282956', 'E_01_299\\t29348663\\tLEENE Enhancer\\thg19\\tchr14\\t56245000\\t56285000\\tHuman\\tEndothelial Cells (ECs)\\tLow+High throughput\\t\"ChIP-seq,qPCR,Hi-C\"\\t\"To test this possibility, we first attempted to identify the promoter/enhancer region of LEENE. In evaluating the genomic region surrounding LEENE, i.e., 20?kb up- and down-stream of the transcription start site (TSS) on chr14: 56,245,000C56,285,000, we observed the enrichments of H3K27ac and H3K4me1 in the HUVEC chromatin immunoprecipitation (ChIP)-seq data available in the ENCODE Database, indicating an enhancer state of this DNA region.We did find that PS led to significant increase in the H3K27ac in the LEENE region as measured by ChIP-qPCR, indicating the activation of LEENE as an enhancer in ECs subjected to PS vs. \"\\tEnhancer\\tLINC00520\\t--\\t\"ChIP-qPCR,qPCR,Hi-C\"\\t\"As shown in Fig. 1f, quantitative PCR (qPCR) with LEENE RNA-specific primers also revealed the significantly higher level of LEENE in ECs subjected to PS than to OS. To confirm the flow regulation of LEENE in ECs and explore its relevance to endothelial function, we tested whether LEENE is differentially regulated by tumor necrosis factor alpha (TNF), which exerts pro-inflammatory effects similar to OS, and atorvastatin (ATV), which confers endothelial protective effects similar to PS. Resembling the opposite effects of OS and PS, TNF decreased, while ATV increased the level of LEENE.These findings are in line with the differential levels of KLF2/KLF4-eNOS signaling.\"\\tC14orf34\\t--\\t--\\t--\\tLEENE-associated Enhancer plays a role in positive regulation of eNOS transcription\\tCRISPR/Cas9\\t\"As a result of the Enhancer ablation, the transcription of LEENE and eNOS was significantly suppressed, in both DMSO (a control vehicle) and ATV-treated ECs\"\\t\"KLF2,KLF4\"\\t\"LKLF,EZF,GKLF\"\\tChIP-qPCR\\t\"To confirm the association of such TFs on the promoter of LEENE, we performed ChIP-qPCR, which detected a robust binding between KLF4 and multiple regions within the promoter region of LEENE (marked by H3K4me3 peaks, Fig. 2a); these interactions were significantly increased by Ad-KLF4, which mimics the effect of PS and ATV.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t17148', 'E_01_300\\t29378668\\t--\\thg19\\tchr2\\t11633299\\t11641134\\tHuman\\tMCF-7\\tLow+High throughput\\t\"ChIP-seq,3C,ChIP-qPCR\"\\t\"Additionally, overlap with datasets containing histone ChIP-Seq data reveals the enrichment of histone marks corresponding to active/poised enhancers and depleted of those markings found at repressed/silenced enhancers.Validations were performed using conventional ChIP-qPCR.Compared to genomic background, global analysis of the high-confidence TDG peaks revealed that E2-dependent TDG binding was enriched at promoters as well as distal to promoters with approximately 60% occurring intergenically.However, overlapping TDG peaks with sites of E2-dependent ER localization revealed that 45% of TDG peaks occur at the same sites where ER localizes in response to E2.\"\\tEnhancer\\tGREB1\\t3C\\t--\\t\"ChIP using ER in the presence and absence of TDG as well as ? or + E2, showing that ER binding is unaltered during depletion of TDG. e Loss of TDG prevents enhancerCpromoter looping at the GREB1 locus. MCF7 cells were treated with siControl or siTDG, and then treated with 100 nM E2 for 1 h. 3C, semiquantitative method of measuring the looping between the GREB1 enhancer and promoter, revealed that E2-driven looping of the enhancer and promoter is disrupted upon TDG knockdown.\"\\tGREB1\\tBreast Cancer\\tDOID:1612\\tD001943\\t--\\t--\\t--\\tTDG\\thTDG\\tCRISPR/Cas9\\t\"Remarkably, at a subset of enhancers that E2 targets, we found that TDG depletion abrogates E2-mediated eRNA, disrupts 3-dimensional reorganization at ER-targets such as GREB1 and disrupts E2-mediated transcription of corresponding ER-target genes. To investigate whether TDG plays a functional role in E2 signaling in breast cancer, we engineered an MCF7 TDG knockout cell line using CRISPR technology and found that TDG knockout and depletion leads to defects in E2-mediated proliferation and sensitizes MCF7 cells to the anti-estrogen. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t37024', 'E_01_301\\t29379199\\t--\\thg19\\tchr2\\t8818311\\t8820522\\tHuman\\tNeuroblastoma Cell Lines\\tLow+High throughput\\t\"ChIP-seq,Western blot\"\\t\"We observe that two hours after shutdown, MYCN is significantly depleted from promoters and enhancers (Fig. 1f; Supplementary Fig. 2c,e,f) - a result revealed by ChIP-Rx cell-count normalization.Although both traditional ChIP-seq and ChIP-Rx reveal global loss of MYCN at promoters and enhancers, we do observe global loss of additional marks (H3K4me3, CTCF, H3K27ac, and RNA Pol II) at 24 hours post MYCN shutdown only in ChIP-Rx data. In contrast, induction of MYCN in parental SHEP neuroblastoma cells (Fig. 1c,d) results in loading of MYCN at active promoters and enhancers, and is coincident with increases in global H3K27ac. \"\\tEnhancer\\tID2\\t--\\tChIP-seq\\t\"ChIP-seq tracks of MYCN and H3K27ac at 0, 2 and 24hrs post MYCN shutdown at RPL22, HAND2 and the upstream ID2 enhancer.Profiling of the TH-MYCN genetically engineered neuroblastoma mouse model28 revealed MYCN binding at the promoters of classic MYC target genes (Npm1/Rpl22), as well as at enhancers for key neuroblastoma and/or neural crest associated genes (ID2 and GATA2).\"\\t\"GIG8,ID2A,ID2H,bHLHb26\"\\tNeuroblastoma\\tDOID:769\\tD009447\\t--\\t--\\t--\\tMYCN\\t\"MODED,N-myc,NMYC,ODED,bHLHe37\"\\t\"Western blot,ChIP-seq\"\\t\"Across MYCN-amplified neuroblastoma lines, we used chromatin immunoprecipitation coupled to high-throughput sequencing (ChIP-seq) to generate a consensus genome-wide map of ~10,000 regions that exhibit strong and consistent MYCN occupancy.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2695', 'E_01_302\\t29379199\\t--\\thg19\\tchr4\\t174435828\\t174438057\\tHuman\\tNeuroblastoma Cell Lines\\tLow+High throughput\\t\"ChIP-seq,Western blot\"\\t\"We observe that two hours after shutdown, MYCN is significantly depleted from promoters and enhancers (Fig. 1f; Supplementary Fig. 2c,e,f) - a result revealed by ChIP-Rx cell-count normalization.Although both traditional ChIP-seq and ChIP-Rx reveal global loss of MYCN at promoters and enhancers, we do observe global loss of additional marks (H3K4me3, CTCF, H3K27ac, and RNA Pol II) at 24 hours post MYCN shutdown only in ChIP-Rx data. In contrast, induction of MYCN in parental SHEP neuroblastoma cells (Fig. 1c,d) results in loading of MYCN at active promoters and enhancers, and is coincident with increases in global H3K27ac. \"\\tEnhancer\\tHAND2\\t--\\tChIP-seq\\t\"ChIP-seq tracks of MYCN and H3K27ac at 0, 2 and 6hrs post MYCN induction at RPL22, HAND2 and the upstream ID2 enhancer.Visual inspection of MYCN load at highly occupied genes revealed a diversity of binding profiles with some genes exhibiting promoter MYCN binding (RPL22) and others a more mixed array of promoter and enhancer binding (HAND2 & ID2). \"\\t\"DHAND2,Hed,Thing2,bHLHa26,dHand\"\\tNeuroblastoma\\tDOID:769\\tD009447\\t--\\t--\\t--\\tMYCN\\t\"MODED,N-myc,NMYC,ODED,bHLHe37\"\\t\"Western blot,ChIP-seq\"\\t\"Across MYCN-amplified neuroblastoma lines, we used chromatin immunoprecipitation coupled to high-throughput sequencing (ChIP-seq) to generate a consensus genome-wide map of ~10,000 regions that exhibit strong and consistent MYCN occupancy.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t10708', 'E_01_303\\t28378740\\t--\\thg19\\tchr3\\t107662655\\t107762655\\tHuman\\tMCF-7\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,ChIA-PET\"\\t\"By rank-ordering of enhancer regions based on H3K27ac enrichment, we discovered that T-cell acute lymphoblastic leukemia (T-ALL (RPMI18402, Jurkat and MOLT3)) diffuse large B-cell lymphoma (DLBCL (LY4)) and breast cancer (MCF7 and HCC1954) cell lines have SEs within ~200 kb of CD47 (Fig. 1a).', 'E_01_304\\t29385519\\t--\\thg19\\tchrx\\t106959618\\t106962917\\tHuman\\tU-2 OS\\tLow+High throughput\\t\"ChIP-qPCR,4C\"\\t\"However, in contrast to the effects on the hormone-induced levels, these effects do not consistently correlate with deletions at the GILZ GBS1C4 enhancer and might reflect difficulties to quantify these transcripts by qPCR due to low expression levels in the absence of hormone treatment.To test if deletion of an individual GBS influences GR occupancy at the edited enhancer, we analyzed GR binding by ChIP-qPCR in a representative clonal cell line in which we deleted the GILZ GBS1. \"\\tEnhancer\\tTSC22D3\\tCRISPR/Cas9\\t--\\t\"To experimentally test the contribution of the smallest regulatory units of gene regulation, individual GR binding sequences (GBSs), we used the CRISPR/Cas9-system (25) to delete selected GBSs in their endogenous genomic context. We chose a GBS located 1.5 kb upstream of the GR target gene GILZ (glucocorticoid induced leucine zipper, alias TSC22D3) and one GBS 1.5 kb upstream of the target gene DUSP1.\"\\t\"SOM,TFCP2L4,VWS2\"\\t--\\t--\\t--\\t--\\t--\\t--\\tNR3C1\\t\"GCCR,GCR,GCRST,GR,GRL\"\\tChIP-seq\\t\"To study the global connection between GR binding and GR-dependent gene regulation, we combined data from genome-wide GR binding experiments (Chromatin Immunoprecipitation followed by sequencing (ChIP-seq)) with RNA-seq data regarding gene expression changes upon GR-activation in A549 cells (3).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t4818', 'E_01_305\\t29408204\\t--\\thg19\\tchr3\\t69863244\\t70187654\\tHuman\\tMelanoma\\tLow+High throughput\\t\"ChIP-seq,Western blot,siRNA,qPCR\"\\t\"H3K27ac chromatin immunoprecipitation sequencing (ChIP-seq) profiling, which marks active enhancers, indicated the presence of SEs at the MITF gene locus in MITF-hi cells and substantially less enhancer signal at the same location in an MITF-lo cell line (LOXIMVI) and a normal human dermal fibroblast line.\"\\tSuper-Enhancer\\tMITF\\t--\\t\"qPCR,ChIP-seq\"\\t\"To identify a set of MITF high- and low-expressing cells, we performed quantitative real-time reverse transcriptase-PCR analysis on 18 melanoma lines and found 8 lines with high MITF (MITF-hi) (>2-fold mean normalized MITF levels) and 5 lines with low MITF (MITF-lo) (<?-fold mean normalized MITF levels) expression levels (Figure 1b). Analysis of MITF copy number in these lines showed only two lines with evidence of MITF gene amplification.\"\\t\"CMM8,COMMAD,MI,WS2,WS2A,bHLHe32\"\\tMelanoma\\tDOID:1909\\tD008545\\t--\\t--\\t--\\tCDK7\\t\"CAK,CAK1,CDKN7,HCAK,MO15,STK1,p39MO15\"\\tsiRNA\\t\"To verify that CDK7 blockade can lead to growth suppression, we subjected SK-MEL-5, SK-MEL-30, and UACC257 cells to small interfering RNA (siRNA) against CDK7 and observed consistent decreases in cell growth (see Supplementary Figure S6 online) compared with siRNA nontargeting controls. Cell lines with lower MITF levels (LOXIMVI, A375, SK-MEL-63) (see Supplementary Figure S7 online) were also sensitive to THZ1, exhibited appropriate CDK7 targeting, and showed an apoptotic response when exposed to the agent.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t236864', 'E_01_306\\t29416716\\t--\\thg19\\tchr14\\t99770698\\t99782638\\tHuman\\tA673/TR/shEF1 Cells\\tLow throughput \\t\"Luciferase Reporter Assay,qRT-PCR\"\\t\" Expression of these genes appeared to be induced by EWSR1-FLI1-bound super-enhancers, which showed high activity in reporter assays.In fact, EWSR1-FLI1 is known to convert non-functional GGAA-microsatellites into potent enhancers to steer a large proportion of its target genes [24C26]. Strong EWSR1-FLI1-dependent enhancer activity of these GGAA-microsatellites in luciferase reporter assays was consistently observed. \"\\tSuper-Enhancer\\tBCL11B\\t--\\t\"qRT-PCR,ChIP-seq\"\\t\"These data in cell lines suggested that ATP1A1, BCL11B, and GLG1 may be direct EWSR1-FLI1 target genes. Testing this hypothesis involved analyzing available ChIP-Seq and DNase-Seq data generated in Ewing sarcoma cell lines, which showed strong EWSR1-FLI1-binding to GGAA-microsatellites close to these genes.\"\\t\"ATL1,ATL1-alpha,ATL1-beta,ATL1-delta,ATL1-gamma,CTIP-2,CTIP2,IDDFSTA,IMD49,RIT1,ZNF856B,hRIT1-alpha\"\\tEwing Sarcoma\\tDOID:3369\\tD012512\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t141045', 'E_01_307\\t29416716\\t--\\thg19\\tchr1\\t116896120\\t116904325\\tHuman\\tA673/TR/shEF1 Cells\\tLow throughput\\t\"Luciferase Reporter Assay,qRT-PCR\"\\t\" Expression of these genes appeared to be induced by EWSR1-FLI1-bound super-enhancers, which showed high activity in reporter assays.In fact, EWSR1-FLI1 is known to convert non-functional GGAA-microsatellites into potent enhancers to steer a large proportion of its target genes [24C26]. Strong EWSR1-FLI1-dependent enhancer activity of these GGAA-microsatellites in luciferase reporter assays was consistently observed. \"\\tSuper-Enhancer\\tATP1A1\\t--\\t\"qRT-PCR,ChIP-seq\"\\t\"These data in cell lines suggested that ATP1A1, BCL11B, and GLG1 may be direct EWSR1-FLI1 target genes. Testing this hypothesis involved analyzing available ChIP-Seq and DNase-Seq data generated in Ewing sarcoma cell lines, which showed strong EWSR1-FLI1-binding to GGAA-microsatellites close to these genes.\"\\t\"CMT2DD,HOMGSMR2\"\\tEwing Sarcoma\\tDOID:3369\\tD012512\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t15571', 'E_01_308\\t29416716\\t--\\thg19\\tchr16\\t74568579\\t74583481\\tHuman\\tA673/TR/shEF1 Cells\\tLow throughput\\t\"Luciferase Reporter Assay,qRT-PCR\"\\t\" Expression of these genes appeared to be induced by EWSR1-FLI1-bound super-enhancers, which showed high activity in reporter assays.In fact, EWSR1-FLI1 is known to convert non-functional GGAA-microsatellites into potent enhancers to steer a large proportion of its target genes [24C26]. Strong EWSR1-FLI1-dependent enhancer activity of these GGAA-microsatellites in luciferase reporter assays was consistently observed. \"\\tSuper-Enhancer\\tGLG1\\t--\\t\"qRT-PCR,ChIP-seq\"\\t\"These data in cell lines suggested that ATP1A1, BCL11B, and GLG1 may be direct EWSR1-FLI1 target genes. Testing this hypothesis involved analyzing available ChIP-Seq and DNase-Seq data generated in Ewing sarcoma cell lines, which showed strong EWSR1-FLI1-binding to GGAA-microsatellites close to these genes.\"\\t\"CFR-1,ESL-1,MG-160,MG160\"\\tEwing Sarcoma\\tDOID:3369\\tD012512\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t94706', 'E_01_309\\t29440643\\t--\\thg19\\tchr6\\t138226720\\t138233280\\tHuman\\t\"B Cell,Monocytes\"\\tLow+High throughput\\t\"CRISPR/Cas9,Hi-C,ChIP-seq\"\\t\"Recent studies have begun to address mammalian enhancer function using CRISPR-mediated genome editing, including in vivo analysis of mouse genes.To identify enhancers activated by inflammatory stimuli,we analyzed ATAC-seq and ChIP-seq data sets generated in our laboratory (GSE43036, GSE9836926, and GSE104638) for chromatin accessibility, binding of PU.1 and C/EBP (which bind to regulatory elements and help define enhancers in myeloid cells), and induction of acetylation of histone 3 lysine 27 (H3K27-Ac), which indicates enhancer activation, at enhancer peaks after LPS stimulation of primary monocytes.\"\\tEnhancer\\tTNFAIP3\\t--\\t\"Hi-C,ChIP-seq\"\\t\"To guide selection of BACs for generation of humanized transgenic mice, we analyzed chromatin conformation, accessibility, and enhancer-related histone marks at the TNFAIP3 locus using our laboratorys (GSE4303620, GSE10038321 and previously unpublished data, GSE104628) and ENCODE and NIH Roadmap ChIP-seq data (available at genome.ucsc.edu and roadmapepigenomics.org) and Hi-C data.Analysis of pre-existing high-resolution Hi-C (chromatin conformation) data in GM12878 B cells identified a 305?kb topologically associating domain (TAD) flanked by CTCF sites roughly centered around the TNFAIP3 gene body.\"\\t\"A20,AISBL,OTUD7C,TNFA1P2\"\\tAutoimmune Diseases\\t--\\tD001327\\tThese findings provide insights into enhancers that regulate human A20 expression to prevent inflammatory pathology and autoimmunity.\\t\"PCR,CRISPR/Cas9\"\\t\"Collectively, the results show that deletion of the TT>A Enhancer at the TNFAIP3 locus results in an autoimmune/inflammatory phenotype most clearly evident as athritis.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t41676', 'E_01_310\\t29453285\\t--\\thg19\\tchr7\\t45920458\\t45923258\\tHuman\\tHep G2\\tLow throughput\\t\"FAIRE-qPCR,Luciferase Reporter Assay\"\\t\"The effect of cAMP on the changes of chromatin structure in the IGFBP-1 enhancer region were examined by FAIRE-qPCR. cAMP significantly increased the relative ratio of FAIRE enrichment in the IGFBP-1 enhancer region,indicating that the chromatin structure of the IGFBP-1 enhancer region becomes looser during decidualization.\"\\tEnhancer\\tIGFBP1\\tCRISPR/Cas9\\t--\\twe used HepG2 cells to examine the endogenous function of the IGFBP-1 enhancer region by a genome editing approach.The enhancer-deleted clones showed significantly lower expression of IGFBP-1 mRNA than the wild-type clones . These results demonstrate that the enhancer region is responsible for IGFBP-1 mRNA expression.\\t\"AFBP,IBP1,IGF-BP25,PP12,hIGFBP-1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tFOXO1\\t\"FKH1, FKHRA, FOXO1\"\\tChIP\\t\"A ChIP assay revealed that cAMP increased the recruitment of the transcriptional regulators CCAAT Enhancer-binding protein  (C/EBP), forkhead box O1 (FOXO1), and p300 to the IGFBP-1 Enhancer in ESCs.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t6100', 'E_01_311\\t29507293\\t--\\thg19\\tchr15\\t66074477\\t66098025\\tHuman\\t\"Erythroleukemia Cell,B lymphoblastoid Cell\"\\tLow+High throughput\\t\"Hi-C,ChIP-seq\"\\t\"Finally, if an enhancer element within hierarchical SEs overlaps with a bin associated with a z-score greater than the threshold H-score, the element is referred to a hub enhancer, whereas the remaining enhancers at the same SE are termed non-hub enhancers\"\\tSuper-Enhancer\\t\"MYO1D,SMYD3 \"\\tCRISPR/Cas9\\t--\\t\"we employed CRISPR-Cas9-mediated genome engineering to delete individual hub or non-hub enhancers with paired sgRNAs flanking the enhancer elements at the MYO1D SE.We observed that 3 of the 5 genes within the SE-containing TAD domain MYO1D,TMEM98 and SPACA3)displayed significant downregulation in mRNA expression, whereas the other two genes (PSMD11 and CDK5R1) remained unaffected , suggesting that the MYO1D SE may regulate only a subset of genes within the same TAD domain.\"\\t\"PPP1R108,myr4,KMT3E,ZMYND1,ZNFN3A1,bA74P14.1\"\\t--\\t--\\t--\\tHub enhancers are the major constituents responsible for SE functional and structural organization.\\t\"ChIP-seq,CRISPR/Cas9\"\\tThe signals for H3K27ac and DNase I hypersensitivity are slightly higher at hub than other types of Enhancers. \\t\"GATA1,TAL1,PAX5,EBF1\"\\t\"ERYF1,GATA-1,GF-1,GF1,NF-E1,NFE1,XLANP,XLTDA,XLTT,SCL,ALL3,BSAPTCL5,bHLHa17,tal-1,ALL3,BSAP,COE1,EBF,O/E-1,OLF1\"\\tChIP-seq\\t\"Hub Enhancers contain significantly higher ChIP-seq binding signals for lineage-regulating master regulators than non-hub Enhancers, such as GATA1 and TAL1 in K562 cells, and PAX5 and EBF1 in GM12878 cells.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t35266712', 'E_01_312\\t29507293\\t--\\thg19\\tchr17\\t31209116\\t31240452\\tHuman\\t\"Erythroleukemia Cell,B lymphoblastoid Cell\"\\tLow+High throughput\\t\"Hi-C,ChIP-seq,CRISPR/Cas9\"\\t\"Finally, if an enhancer element within hierarchical SEs overlaps with a bin associated with a z-score greater than the threshold H-score, the element is referred to a hub enhancer, whereas the remaining enhancers at the same SE are termed non-hub enhancers\"\\tSuper-Enhancer\\t\"MYO1D,SMYD3 \"\\tCRISPR/Cas9\\t--\\t\"we employed CRISPR-Cas9-mediated genome engineering to delete individual hub or non-hub enhancers with paired sgRNAs flanking the enhancer elements at the MYO1D SE.We observed that 3 of the 5 genes within the SE-containing TAD domain MYO1D,TMEM98 and SPACA3)displayed significant downregulation in mRNA expression, whereas the other two genes (PSMD11 and CDK5R1) remained unaffected , suggesting that the MYO1D SE may regulate only a subset of genes within the same TAD domain.\"\\t\"PPP1R108,myr4,KMT3E,ZMYND1,ZNFN3A1,bA74P14.1\"\\t--\\t--\\t--\\tHub enhancers are the major constituents responsible for SE functional and structural organization.\\t\"ChIP-seq,CRISPR/Cas9\"\\tThe signals for H3K27ac and DNase I hypersensitivity are slightly higher at hub than other types of Enhancers.\\t\"GATA1,TAL1,PAX5,EBF1\"\\t\"ERYF1,GATA-1,GF-1,GF1,NF-E1,NFE1,XLANP,XLTDA,XLTT,SCL,ALL3,BSAPTCL5,bHLHa17,tal-1,ALL3,BSAP,COE1,EBF,O/E-1,OLF1\"\\tChIP-seq\\t\"Hub Enhancers contain significantly higher ChIP-seq binding signals for lineage-regulating master regulators than non-hub Enhancers, such as GATA1 and TAL1 in K562 cells, and PAX5 and EBF1 in GM12878 cells.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t405245', 'E_01_313\\t29511351\\tHPSE eRNA\\thg19\\tchr4\\t84277627\\t84281723\\tHuman\\tAGS\\tLow+High throughput\\t\"3C,ChIP-seq,qPCR\"\\t\"Overlapping analysis of FANTOM5 expression atlas and chromatin interaction dataset derived from 4DGenome revealed one potential enhancer region (chr4:84277627C84281723) adjacent to human HPSE promoter(chr4: 84255915C84256935).', 'E_01_314\\t29628309\\t--\\thg19\\tchr3\\t150450617\\t150456921\\tHuman\\tMCF-7\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"We next assessed whether those estrogen-induced JMJD6 binding sites shared enhancer characteristics, including highly enriched H3K4me1/2 but low levels of H3K4me3. Heatmap and tag density plots confirmed they were, indeed, ERa-bound enhancers . Furthermore, it was reported recently that a group of enhancers, namely active enhancers, were essential for the transcriptional activation of estrogeninduced coding target genes\"\\tEnhancer\\tSIAH2\\t--\\t\"ChIP-qPCR,qRT-PCR\"\\t\"To validate JMJD6 effects on eRNA production, total RNA extracted from MCF7 cells transfected with control siRNA or two independent siRNAs pecifically against JMJD6 in the presence or absence of strogen treatment were subjected to qRT-PCR analysis using primers specifically targeting several ERa and JMJD6 co-bound active enhancers nearby estrogen-induced coding genes, such as FOXC1, SIAH2, GREB1, NRIP1, and SMAD7.\"\\thSiah2\\tBreast Cancer\\tDOID:1612\\tD001943\\t--\\t--\\t--\\t\"JMJD6,ESR1\"\\t\"PSR,PTDSR,PTDSR1,ER,ESR,ESRA,ESTRR,Era,NR3A1\"\\t\"siRNA,RT-qPCR\"\\t\"To validate JMJD6 effects on eRNA production, total RNA extracted from MCF7 cells transfected with control siRNA or two independent siRNAs specifically against JMJD6 in the presence or absence of estrogen treatment were subjected to RT-qPCR analysis using primers specifically targeting several ER and JMJD6 co-bound active enhancers nearby estrogen-induced coding genes.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5140', 'E_01_315\\t29628309\\t--\\thg19\\tchr6\\t1679010\\t1683521\\tHuman\\tMCF-7\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"We next assessed whether those estrogen-induced JMJD6 binding sites shared enhancer characteristics, including highly enriched H3K4me1/2 but low levels of H3K4me3. Heatmap and tag density plots confirmed they were, indeed, ERa-bound enhancers . Furthermore, it was reported recently that a group of enhancers, namely active enhancers, were essential for the transcriptional activation of estrogeninduced coding target genes\"\\tEnhancer\\tFOXC1\\t--\\t\"ChIP-qPCR,qRT-PCR\"\\t\"To validate JMJD6 effects on eRNA production, total RNA extracted from MCF7 cells transfected with control siRNA or two independent siRNAs pecifically against JMJD6 in the presence or absence of strogen treatment were subjected to qRT-PCR analysis using primers specifically targeting several ERa and JMJD6 co-bound active enhancers nearby estrogen-induced coding genes, such as FOXC1, SIAH2, GREB1, NRIP1, and SMAD7.\"\\t\"ARA,ASGD3,FKHL7,FREAC-3,FREAC3,IGDA,IHG1,IRID1,RIEG3\"\\tBreast Cancer\\tDOID:1612\\tD001943\\t--\\t--\\t--\\t\"JMJD6,ESR1\"\\t\"PSR,PTDSR,PTDSR1,ER,ESR,ESRA,ESTRR,Era,NR3A1\"\\t\"siRNA,RT-qPCR\"\\t\"To validate JMJD6 effects on eRNA production, total RNA extracted from MCF7 cells transfected with control siRNA or two independent siRNAs specifically against JMJD6 in the presence or absence of estrogen treatment were subjected to RT-qPCR analysis using primers specifically targeting several ER and JMJD6 co-bound active enhancers nearby estrogen-induced coding genes.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t70586', 'E_01_316\\t29641996\\t--\\thg19\\tchr8\\t128206162\\t128314310\\tHuman\\tHCT 116\\tLow+High throughput\\t\"4C,ChIP-seq\"\\t\"Among the DNA loop structures observed in these datasets, a large 2.8 Mb DNA loop was evident in all four cell types. This loop connects CTCF sites encompassing the MYC gene and qualifies as an insulated neighborhood. The DNA anchor sites of this 2.8 Mb DNA loop occur at the boundaries of a TAD found in all cells.\"\\tSuper-Enhancer\\tMYC\\tCRISPR/Cas9\\t\"ChIA-PET,ChIP-qPCR\"\\tCTCF chromatin immunoprecipitation quantative polymerase chain reaction (ChIP-qPCR) showed complete loss of CTCF binding to this site in K562 and HCT-116 cells and a 60%C70% reduction in CTCF binding at this site in Jurkat and MCF7 cells. RNA analysis revealed a 70%C80% reduction of endogenous MYC mRNA in the absence of the enhancer-docking site in all of these cell types.\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\t--\\t--\\t--\\tThe CTCF site in the promoter-proximal region of MYC is important for optimal interaction with distal Enhancers and supports the idea that this CTCF site functions as an Enhancer-docking site.\\t\"4C,CRISPR/Cas9,ChIP-qPCR\"\\t\"Similar results were obtained in HCT-116 cells, where the viewpoint was centered on the super-Enhancer located -0.4 Mb upstream of the MYC gene\"\\tCTCF\\tMRD21\\tCRISPR/Cas9\\t\"An inducible CRISPR/Cas9 perturbation model showed reduced proliferation for these four cell types upon induction of CTCF-site deletions .These results indicate that the CTCF motif in the MYC Enhancer-docking site is necessary for CTCF binding, for high levels of MYC expression and for cellular proliferation.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t488078', 'E_01_317\\t29641996\\t--\\thg19\\tchr8\\t129046628\\t129096067\\tHuman\\tK-562\\tLow+High throughput\\t\"4C,ChIP-seq\"\\t\"Among the DNA loop structures observed in these datasets, a large 2.8 Mb DNA loop was evident in all four cell types. This loop connects CTCF sites encompassing the MYC gene and qualifies as an insulated neighborhood. The DNA anchor sites of this 2.9 Mb D?\"\\tSuper-Enhancer\\tMYC\\tCRISPR/Cas9\\t\"ChIA-PET,ChIP-qPCR\"\\tCTCF chromatin immunoprecipitation quantative polymerase chain reaction (ChIP-qPCR) showed complete loss of CTCF binding to this site in K562 and HCT-116 cells and a 60%C70% reduction in CTCF binding at this site in Jurkat and MCF8 cells. RNA analysis rev?\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\t--\\t--\\t--\\tThe CTCF site in the promoter-proximal region of MYC is important for optimal interaction with distal Enhancers and supports the idea that this CTCF site functions as an Enhancer-docking site.\\t\"4C,CRISPR/Cas9,ChIP-qPCR\"\\tThe 4C-seq data in K562 cells indicated that the MYC Enhancer docking site interacts predominantly with distal super-Enhancers located -0.3 Mb and -2 Mb downstream of the MYC gene and that the majority of these interactions were significantly reduced when the putative CTCF motif was perturbed .\\tCTCF\\tMRD21\\tCRISPR/Cas9\\t\"An inducible CRISPR/Cas9 perturbation model showed reduced proliferation for these four cell types upon induction of CTCF-site deletions .These results indicate that the CTCF motif in the MYC Enhancer-docking site is necessary for CTCF binding, for high levels of MYC expression and for cellular proliferation.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t323034', 'E_01_318\\t29641996\\t--\\thg19\\tchr8\\t130569971\\t130616320\\tHuman\\tK-562\\tLow+High throughput\\t\"4C,ChIP-seq\"\\t\"Among the DNA loop structures observed in these datasets, a large 2.8 Mb DNA loop was evident in all four cell types. This loop connects CTCF sites encompassing the MYC gene and qualifies as an insulated neighborhood. The DNA anchor sites of this 210 Mb D?\"\\tSuper-Enhancer\\tMYC\\tCRISPR/Cas9\\t\"ChIA-PET,ChIP-qPCR\"\\tCTCF chromatin immunoprecipitation quantative polymerase chain reaction (ChIP-qPCR) showed complete loss of CTCF binding to this site in K562 and HCT-116 cells and a 60%C70% reduction in CTCF binding at this site in Jurkat and MCF9 cells. RNA analysis rev?\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\t--\\t--\\t--\\tThe CTCF site in the promoter-proximal region of MYC is important for optimal interaction with distal Enhancers and supports the idea that this CTCF site functions as an Enhancer-docking site.\\t\"4C,CRISPR/Cas9,ChIP-qPCR\"\\tThe 4C-seq data in K562 cells indicated that the MYC Enhancer docking site interacts predominantly with distal super-Enhancers located -0.3 Mb and -2 Mb downstream of the MYC gene and that the majority of these interactions were significantly reduced when the putative CTCF motif was perturbed .\\tCTCF\\tMRD21\\tCRISPR/Cas9\\t\"An inducible CRISPR/Cas9 perturbation model showed reduced proliferation for these four cell types upon induction of CTCF-site deletions .These results indicate that the CTCF motif in the MYC Enhancer-docking site is necessary for CTCF binding, for high levels of MYC expression and for cellular proliferation.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1844832', 'E_01_319\\t29641996\\t--\\thg19\\tchr8\\t130693569\\t130739918\\tHuman\\tK-562\\tLow+High throughput\\t\"4C,ChIP-seq\"\\t\"Among the DNA loop structures observed in these datasets, a large 2.8 Mb DNA loop was evident in all four cell types. This loop connects CTCF sites encompassing the MYC gene and qualifies as an insulated neighborhood. The DNA anchor sites of this 211 Mb D?\"\\tSuper-Enhancer\\tMYC\\tCRISPR/Cas9\\t\"ChIA-PET,ChIP-qPCR\"\\tCTCF chromatin immunoprecipitation quantative polymerase chain reaction (ChIP-qPCR) showed complete loss of CTCF binding to this site in K562 and HCT-116 cells and a 60%C70% reduction in CTCF binding at this site in Jurkat and MC10 cells. RNA analysis rev\\t\"MRTL,MYCC,bHLHe39,c-Myc\"\\t--\\t--\\t--\\tThe CTCF site in the promoter-proximal region of MYC is important for optimal interaction with distal Enhancers and supports the idea that this CTCF site functions as an Enhancer-docking site.\\t\"4C,CRISPR/Cas9,ChIP-qPCR\"\\tThe 4C-seq data in K562 cells indicated that the MYC Enhancer docking site interacts predominantly with distal super-Enhancers located -0.3 Mb and -2 Mb downstream of the MYC gene and that the majority of these interactions were significantly reduced when the putative CTCF motif was perturbed .\\tCTCF\\tMRD21\\tCRISPR/Cas9\\t\"An inducible CRISPR/Cas9 perturbation model showed reduced proliferation for these four cell types upon induction of CTCF-site deletions .These results indicate that the CTCF motif in the MYC Enhancer-docking site is necessary for CTCF binding, for high levels of MYC expression and for cellular proliferation.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1968430', 'E_01_320\\t29695788\\tOCT4 Intronic Enhancer\\thg19\\tchr6\\t31137269\\t31137697\\tHuman\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ATAC-seq,Luciferase Reporter Assay,ChIP-seq\"\\t\"In human pre-implantation blastocyst, however, neither enhancer appears open, whereas two putative enhancers appear downstream of the POU5F1 TSS. Each of these peaks contains a cluster of AP2 sites and a KLF site, indicating that they could be opened by the combinatorial activity of these transcription factors during preimplantation development.\"\\tEnhancer\\tPOU5F1\\tCRISPR/Cas9\\t--\\t\"We found normal expression of OCT4 and self-renewal in the primed state, but a dramatic loss of OCT4 expression accompanied by differentiation upon reversion to the naive state. This indicates a potential direct role for TFAP2C in regulating the pluripotency master-regulator OCT4 by binding to a previously unknown enhancer, which in turn is likely to be important for preimplantation OCT4 expression.\"\\t\"OCT3, OCT4, OTF-3, OTF3, OTF4, Oct-3, Oct-4\"\\t--\\t--\\t--\\tTFAP2C functions to maintain pluripotency and repress neuroectodermal differentiation during the transition from primed to naive by facilitating the opening of enhancers proximal to pluripotency factors.\\t\"Luciferase Reporter Assay,CRISPR/Cas9\"\\t\"This indicates a potential direct role for TFAP2C in regulating the pluripotency master-regulator OCT4 by binding to a previously unknown Enhancer, which in turn is likely to be important for preimplantation OCT4 expression.\"\\tTFAP2C\\t\"AP2-GAMMA,ERF1,TFAP2G,hAP-2g\"\\tLuciferase Reporter Assay\\t\"Luciferase activity from a pGL3 construct in which WT or mutant Intron Element 1 had been cloned, normalized to signal from a pGL3 construct with no enhancer. Results are shown from two independent experiment, except for the AP2 sample, for which there are n=3 replicates from two experiments. All signals were first normalized for Renilla signal.\"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t5370', 'E_01_321\\t29706059\\tAHRR Enhancer\\thg19\\tchr5\\t372500\\t376000\\tHuman\\tCirculating Monocytes Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"We tested total RNA samples from a group of CRU-mRNA with available RNA (n=11smokers/10 nonsmokers) and observed three ofthe SM-DMR locations(AHRR, C5orf55-EXOC-AS, and SASH1) showedup-regulation of correspondinge RNA expression as well as mRNA.\"\\tEnhancer\\tAHRR\\t--\\tRT-PCR\\tThe Figure6C bar graph displays the significants moking-associated fold-change results for each RT-qPCR relative to their genome position in the AHRR region.\\t\"AHH,AHHR,bHLHe77\"\\tCardiovascular Cancer\\tDOID:176\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t69960', 'E_01_322\\t29765516\\tERG Super-Enhancer\\thg19\\tchr21\\t40003867\\t40015913\\tHuman\\tHEL\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq\"\\t\"Chromatin-immunoprecipitation followed by quantitative PCR (ChIP-qPCR) analysis for GFI1B and RUNX1 was performed at highly conserved noncoding areas including upstream, promotor, 3 UTR, and SE regions.\"\\tSuper-Enhancer\\tERG\\t--\\tChIP-qPCR\\t\"Furthermore,ChIP-qPCR analysis for H3K27ac revealed that the level of H3K27ac was elevated only at the SE region after the NCD38 treatment.\"\\t\"erg-3,p55\"\\tLeukemia\\tDOID:1240\\tD007938\\t--\\t--\\t--\\t\"GFI1B,RUNX1,KDM1A\"\\t\"BDPLT17,ZNF163B,AML1, AML1-EVI-1, AMLCR1, CBF2alpha, CBFA2, EVI-1, PEBP2aB, PEBP2alpha,AOF2, BHC110, CPRF, KDM1, LSD1\"\\tChIP-qPCR\\t\"ChIP-qPCR analysis for GFI1B, RUNX1, or control IgG in the conserved non-coding regions at the ERG locus in HEL cells.ChIP-qPCR analysis for GFI1B (C), LSD1 (D), and CoREST (E) at the ERG-SE locus in HEL cells treated with DMSO or NCD38 for 24 hours. All ChIP-qPCR assays were performed independently twice and the means are displayed.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t270708', 'E_01_323\\t29898989\\tMLH1 Enhancer\\thg19\\tchr3\\t36999364\\t37001216\\tHuman\\t\"SW620,RKO\"\\tLow throughput\\t\"ChIP-qPCR,3C,Luciferase Reporter Assay,CRISPR/Cas9\"\\t\"We provide the first description of an enhancer for the MLH1 gene. This -35kb enhancer was CTCF-bound, and disruption of the consensus binding site within the enhancer significantly impairs MLH1 expression in SW620 colorectal carcinoma cells.\"\\tEnhancer\\tMLH1\\t\"3C,CRISPR/Cas9\"\\t\"ChIP-qPCR,Luciferase Reporter Assay\"\\t\"We deleted a DNA fragment corresponding to the -35kb region from SW620 cells,showed that the -35kb region functions as an MLH1 Enhancer in SW620 colorectal cells.\"\\t\"COCA2,FCC2,HNPCC,HNPCC2,hMLH1\"\\tLynch Syndrome\\tDOID:3883\\tD003123\\t--\\t--\\t--\\tCTCF\\tMRD21\\tCRISPR/Cas9\\t\"To evaluate the impact on endogenous MLH1 expression, we next deleted a region containing the CTCF binding motif in SW620 and K562 cells using CRISPR-Cas9 (Fig 3A, CTCF). Similar to deletion of the entire Enhancer, we noted significant reduction in MLH1 expression in SW620 but not in K562 cells.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t34550', 'E_01_324\\t29903739\\tCOUP-TFII Enhancer\\thg19\\tchr15\\t96725740\\t96737960\\tHuman\\tE11.5 Heart\\tLow+High throughput\\t\"ChIP-seq,Hi-C\"\\t\"we made use of our TBX20 ChIP-Seq analysis in E11.5 hearts and attributed TBX20 bound sites to the nearest expressed gene in E11.5 control or Tbx20 cKO cardiomyocytes. Next, we overlaid our RNA-Seq data with that of our TBX20 ChIP-Seq analysis in E11.5 hearts and identified 548 genes that were differentially expressed in Tbx20 cKO cardiomyocytes and were marked by one or more TBX20 binding events in the vicinity of the gene.\"\\tEnhancer\\tNR2F2\\t--\\tHi-C\\t\"We next established that this Enhancer was functionally connected with COUP-TFII. We performed a promoter-based Capture Hi-C in iPSC-derived human cardiomyocytes to identify long-range physical interactions between genes and Enhancers. We observed that this Enhancer directly loops and contacts the COUP-TFII promoter,140-Kb away,confirming that this is a COUP-TFII Enhancer.\"\\t\"ARP-1,ARP1,CHTD4,COUPTF2,COUPTFB,COUPTFII,NF-E3,SVP40,TFCOUP2\"\\tCongenital Heart Disease\\tDOID:1682\\tD006330\\tThis Enhancer interacted with the NR2F2 promoter in human cardiomyocytes and conferred atrial specific gene expression in a transgenic mouse in a TBX20 dependent manner.\\tHi-C\\t\"Taken together, these results linked TBX20 binding to an evolutionary conserved Enhancer that regulates COUP-TFII expression in developing atrial cardiomyocytes, uncovering a mechanism by which COUP-TFII expression is TBX20 dependent.\"\\tTBX20\\tASD4\\tX-Gal Staining Assay \\tSection analysis of COUP-TFII enh1::lacZ transgenic embryo demonstrates X-gal staining in developing atrial myocardium (arrows) and caval vein (arrowheads). \\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t137306', 'E_01_325\\t29912188\\t--\\thg19\\tchr12\\t132285987\\t132290767\\tHuman\\tIshikawa Cell\\tLow+High throughput\\tChIP-seqEnsure that each regulatory element to be targeted with Enhancer-i has at least 4 unique oligonucleotides designed for it. Order these sequences along with the U6_internal primers\\tEnhancer\\tMMP17\\tCRISPR/Cas9\\t--\\t\"To determine the contributions of ER-bound Enhancers near the estrogen_x0002_regulated gene MMP17,which has 3 binding sites nearby as defined by ChIP-seq.Figure 2D shows results from an Enhancer dissection experiment, where multiple Enhancers nearby MMP17 are targeted alone and in combination using Enhancer-i.\"\\t\"MMP-17,MT4-MMP,MT4MMP,MTMMP4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t24560', 'E_01_326\\t29912188\\t--\\thg19\\tchr12\\t132281970\\t132287430\\tHuman\\tIshikawa Cell\\tLow+High throughput\\tChIP-seqEnsure that each regulatory element to be targeted with Enhancer-i has at least 4 unique oligonucleotides designed for it. Order these sequences along with the U6_internal primers\\tEnhancer\\tMMP17\\tCRISPR/Cas9\\t--\\t\"To determine the contributions of ER-bound Enhancers near the estrogen_x0002_regulated gene MMP17,which has 3 binding sites nearby as defined by ChIP-seq.Figure 2D shows results from an Enhancer dissection experiment, where multiple Enhancers nearby MMP18 are t.\"\\t\"MMP-17,MT4-MMP,MT4MMP,MTMMP4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t28237', 'E_01_327\\t29912188\\t--\\thg19\\tchr12\\t132342570\\t132347690\\tHuman\\tIshikawa Cell\\tLow+High throughput\\tChIP-seqEnsure that each regulatory element to be targeted with Enhancer-i has at least 4 unique oligonucleotides designed for it. Order these sequences along with the U6_internal primers\\tEnhancer\\tMMP17\\tCRISPR/Cas9\\t--\\t\"To determine the contributions of ER-bound Enhancers near the estrogen_x0002_regulated gene MMP17,which has 3 binding sites nearby as defined by ChIP-seq.Figure 2D shows results from an Enhancer dissection experiment, where multiple Enhancers nearby MMP19 are t.\"\\t\"MMP-17,MT4-MMP,MT4MMP,MTMMP4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t32193', 'E_01_328\\t30008316\\t--\\thg19\\tchr1\\t160536893\\t160537651\\tHuman\\t\"HL-60,293T\"\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"Notably, INTS11-gained regions were largely enriched for proximal promoters(62.2%), mirroring the known RNAPII-associated activity of the complex, and displayed robust binding of INTS13 as well. Conversely, INTS13-gained regions were largely found distal from the TSS of protein-coding genes (82.5%) and only partially overlapped with INTS11.\"\\tEnhancer\\tCD84\\t\"CRISPR/Cas9,3C\"\\tqRT-PCR\\t\"We performed chromosome conformation capture (3C) on the enhancer of CSF1R to infer the consequences of INTS13 depletion on genome architecture. During differentiation of HL-60 cells, we detected a robust interaction between the intronic enhancer of CSF1R and the proximal promoter, as compared to other regions within the CSF1R gene or with the promoter of neighboring genes.\"\\t\"LY9B,SLAMF5,hCD84,mCD84\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"INTS13,EGR1,NAB2\"\\t\"ASUN,C12orf11,GCT1,Mat89Bb,NET48,SPATA30,AT225,G0S30,KROX-24,NGFI-A,TIS8,ZIF-268,ZNF225,MADER\"\\t\"Immunoprecipitation,Colony-forming Unit (CFU) Assay\"\\tImmunoprecipitation of INTS13 in undifferentiated (CTRL) and differentiated (PMA) HL-60 cells confirms the interaction between INTS13 and NAB2 after PMA treatment.Colony-forming unit (CFU) assay of cord blood derived CD34+ cells infected with shRNAs against NAB2 and EGR1 shows that the number of monocytic/macrophagic colonies is significantly reduced in both NAB2- and EGR1-depleted cells when compared to control.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t26389', 'E_01_329\\t30008316\\t--\\thg19\\tchr1\\t160542651\\t160543257\\tHuman\\t\"HL-60,293T\"\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"Notably, INTS11-gained regions were largely enriched for proximal promoters(62.2%), mirroring the known RNAPII-associated activity of the complex, and displayed robust binding of INTS13 as well. Conversely, INTS14-gained regions were largely found distal\"\\tEnhancer\\tCD84\\t\"CRISPR/Cas9,3C\"\\tqRT-PCR\\t\"We performed chromosome conformation capture (3C) on the enhancer of CSF1R to infer the consequences of INTS13 depletion on genome architecture. During differentiation of HL-60 cells, we detected a robust interaction between the intronic enhancer of CSF1R and the proximal promoter, as compared to other regions within the CSF1R gene or with the promoter of neighboring genes.\"\\t\"LY9B,SLAMF5,hCD84,mCD84\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"INTS13,EGR1,NAB2\"\\t\"ASUN,C12orf11,GCT1,Mat89Bb,NET48,SPATA30,AT225,G0S30,KROX-24,NGFI-A,TIS8,ZIF-268,ZNF225,MADER\"\\t\"Immunoprecipitation,Colony-forming Unit (CFU) Assay\"\\tImmunoprecipitation of INTS13 in undifferentiated (CTRL) and differentiated (PMA) HL-60 cells confirms the interaction between INTS13 and NAB2 after PMA treatment.Colony-forming unit (CFU) assay of cord blood derived CD34+ cells infected with shRNAs against NAB2 and EGR1 shows that the number of monocytic/macrophagic colonies is significantly reduced in both NAB2- and EGR1-depleted cells when compared to control.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t32071', 'E_01_330\\t30008316\\t--\\thg19\\tchr5\\t149457542\\t149458807\\tHuman\\t\"HL-60,293T\"\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\t\"Notably, INTS11-gained regions were largely enriched for proximal promoters(62.2%), mirroring the known RNAPII-associated activity of the complex, and displayed robust binding of INTS13 as well. Conversely, INTS15-gained regions were largely found distal\"\\tEnhancer\\tCSF1R\\t\"CRISPR/Cas9,3C\"\\tqRT-PCR\\t\"We performed chromosome conformation capture (3C) on the enhancer of CSF1R to infer the consequences of INTS13 depletion on genome architecture. During differentiation of HL-60 cells, we detected a robust interaction between the intronic enhancer of CSF1R and the proximal promoter, as compared to other regions within the CSF1R gene or with the promoter of neighboring genes.\"\\t\"BANDDOS,C-FMS,CD115,CSF-1R,CSFR,FIM2,FMS,HDLS,M-CSF-R\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"INTS13,EGR1,NAB2\"\\t\"ASUN,C12orf11,GCT1,Mat89Bb,NET48,SPATA30,AT225,G0S30,KROX-24,NGFI-A,TIS8,ZIF-268,ZNF225,MADER\"\\t3C\\tWe performed Chromosome Conformation Capture (3C) on the enhancer of CSF1R to infer the consequences of INTS13 depletion on genome architecture.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t25322', 'E_01_331\\t30012865\\tHBG-4kb HS site\\thg19\\tchr11\\t5279732\\t5280272\\tHuman\\tK-562\\tLow+High throughput\\tChIP-seq\\tWe generated a synthetic zinc finger DNA-binding domain (ZF) targeting the HBG-4kb HS. The HBG-4kb ZF interacted with the target site in vitro and in the context of cells with high affinity and specificity.\\tEnhancer\\tHBG2\\t--\\tPCR\\t\"The regions that we analyzed were the HBG-4kb HS,a 327 region located 4 kb upstream (HBG-8kb) or 1 and 2 kb downstream(HBG-3kb and HBG-2kb) of 328 the HBG-4kb HS,the G-(HBG2) and A-(HBG1) globin gene promoter regions.\"\\t\"HBG-T1,TNCY\"\\tErythrocytic Leukemia\\t--\\t--\\tthe HBG-4kb HS site is important for fetal globin production and suggest that it may act by opening chromatin in a directional manner.\\tCRISPR/Cas9\\t\"Expression of the adult -globin, the embryonic -globin, and the GATA1 genes was not affected by the HBG-4kb ZF. Delivery of the Control ZF did not cause any change in expression of the -globin, -globin, -globin, or GATA1 genes.\"\\t\"USF1,USF2,EGR1,MAFK,NFE2\"\\t\"FCHL,FCHL1,HYPLIP1,MLTF,MLTFI,UEF,bHLHb11,FIP,bHLHb12,AT225,G0S30,KROX-24,NGFI-A,TIS8,ZIF-268,ZNF225,NFE2U,P18,NF-E2,p45\"\\tChIP\\t\"Reduced binding of transcription factors with the HBG-4kb HS in K562 cells expressing the HBG-4kb ZF. K562 cells (Untreated) or K562 cells exposed to 500 nM HBG-4kb ZF for 12 h were subjected to ChIP using antibodies specific for EGR1, USF1, USF2, NF-E2 (p45), MafK, or the negative control, IgG.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5582', 'E_01_332\\t30033119\\t--\\thg19\\tchr8\\t38291071\\t38304326\\tHuman\\tEmbryonic Stem Cell\\tLow+High throughput\\tChIP-STARR-seq\\t\"Visual inspection of selected genomic regions illustrates the broad spectrum of enhancer activity measured by ChIP-STARR-seq. For instance, ChIP-seq for NANOG indicates two strong binding sites up- and downstream of SOX2, but only the downstream binding site resulted in ChIP-STARR-seq RNA in excess of plasmid abundance.\"\\tSuper-Enhancer\\tFGFR1\\tCRISPR/Cas9\\t\"Luciferase Reporter Assay,qRT-PCR\"\\tLuciferase assays confirmed spatially restricted Enhancer activity of DNA in the neighborhood of the central active region of the FGFR1 SE.\\t\"BFGFR,CD331,CEK,ECCL,FGFBR,FGFR-1,FLG,FLT-2,FLT2,HBGFR,HH2,HRTFDS,KAL2,N-SAM,OGD,bFGF-R-1\"\\t--\\t--\\t--\\t\"Although transposable elements associate with putative enhancers, only some exhibit activity.\"\\t\"Luciferase Reporter Assay,CRISPR/Cas9\"\\tLuciferase assays confirmed spatially restricted Enhancer activity of DNA in the neighborhood of the central active region of the FGFR1 SE\\tPOU5F1 \\t\"OCT3,OCT4,OTF-3,OTF3,OTF4,Oct-3,Oct-4,ANOP3,MCOPS3,NANOG\"\\tChIP-qPCR\\t\"To investigate the functional potential of enhancers in ESCs, we first focused on primed H9 ESCs (Figures S1A and S1B) and performed ChIP for NANOG, OCT4, H3K4me1 and H3K27ac.ChIP-qPCR and ChIP-seq were similar to previous results (Figures S1C and 1D).Applying this threshold to regions bound by NANOG, OCT4, H3K4me1, H3K27ac, or combinations of these factors indicates that only a minority of ChIP-seq peaks showed enhancer activity (Figure 2D and S3C), with regions bound by OCT4 having the highest proportion of high activity enhancers.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t29044', 'E_01_333\\t30119690\\t--\\thg19\\tchr16\\t86422446\\t86426834\\tHuman\\tHEK-293\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,CRISPR/Cas9\"\\t\"Notably, two of these five sgRNAs,sgRNAs-AP169 and sgRNA-AP171, are independent sgRNAs that target the same enhancer region,hence increasing the confidence that these are true-positive hits. ENCODE ChIP-seq data confirmed a strong binding of both FOS and JUN to this region.\"\\tEnhancer\\tFOXF1\\tCRISPR/Cas9\\tqRT-PCR\\t\"Furthermore,we identified FOXF1 as the tar_x0002_get gene of this Enhancer and demonstrated that it regu_x0002_lates the senescence phenotype.\"\\t\"ACDMPV,FKHL5,FREAC1\"\\t--\\t--\\t--\\tWe identify FOXF1 as the gene regulated by this enhancer and demonstrate that FOXF1 mediates EnhAP1-OIS1 effect on the senescence phenotype.\\t\"qRT-PCR,Western blot\"\\t\"the physical interactions between EnhAP1-OIS1 and the promoter region of FOXF1 were significantly stronger , suggesting a robust transcriptional regulation of EnhAP1-OIS1.\"\\tJUN\\t\"AP-1,AP1,c-Jun,cJUN,p39\"\\t\"CRISPR/Cas9,Luciferase Reporter Assay\"\\t\"We thus constructed a CRISPR-Cas9 sgRNA library designed to target senescence-induced Enhancers that are putatively regulated by AP-1 and used it in a functional screen.That screen was confined to regions that are directly bound by p53 as detected by p53 chromatin immunoprecipitation-sequencing (ChIP-seq) analysis and therefore missed Enhancers that are critical for OIS enforcement and are regulated by other TFs, in either a p53-dependent or -independent manner.--\\t--\\t--\\t>2KB\\t\\t\\t\\t119492', 'E_01_334\\t30139998\\t--\\thg19\\tchr20\\t30244800\\t30316534\\tHuman\\tMDA-MB-231\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,Luciferase Reporter Assay\"\\t\"In contrast to the broad RING1B ChIP-seq signals in pluripotent cells, RING1B peaks in the breast cell lines were narrow, resembling ChIP-seq signals of transcription factors. Therefore, we assessed whether RING1B is recruited to specific transcription factor-binding sites at SEs.\"\\tSuper-Enhancer\\tBCL2L1\\t--\\tqRT-PCR\\t\"Indeed,we observed RING1B recruitment at SE regions near BCL2L1 in MDA-MB-231 and ESR1 in T47D23.\"\\t\"BCL-XL/S,BCL2L,BCLX,Bcl-X,PPP1R52\"\\tBreast Cancer\\tDOID:1612\\tD001943\\tRING1B regulates Enhancer activity and gene transcription not only by promoting the expression of oncogenes but also by regulating chromatin accessibility.\\tATAC-seq\"RING1B was recruited to regions targeted by transcription factors and its depletion deregulated breast cancer signaling pathways as well as FOXA1 and ER localization to chromatin, we next hypothesized that RING1B regulates transcriptional programs in breast cancer by orchestrating chromatin accessibility. To test', 'E_01_335\\t30139998\\t--\\thg19\\tchr6\\t37997692\\t38001538\\tHuman\\tT-47D\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,Luciferase Reporter Assay\"\\t\"In contrast to the broad RING1B ChIP-seq signals in pluripotent cells, RING1B peaks in the breast cell lines were narrow, resembling ChIP-seq signals of transcription factors. Therefore, we assessed whether RING1B is recruited to specific transcription factor-binding sites at SEs.\"\\tSuper-Enhancer\\tFOXA1\\t--\\tqRT-PCR\\t\"In T47D cells, RING1B bound to a putative SE downstream of FOXA1, suggesting that it plays an activating role in regulating FOXA1 expression.\"\\t\"HNF3A,TCF3A\"\\tBreast Cancer\\tDOID:1612\\tD001943\\tRING1B regulates Enhancer activity and gene transcription not only by promoting the expression of oncogenes but also by regulating chromatin accessibility.\\tATAC-seq\\t\"RING1B was recruited to regions targeted by transcription factors and its depletion deregulated breast cancer signaling pathways as well as FOXA1 and ER localization to chromatin, we next hypothesized that RING1B regulates transcriptional programs in breast cancer by orchestrating chromatin accessibility. To test', 'E_01_336\\t30139998\\t--\\thg19\\tchr6\\t38010769\\t38019538\\tHuman\\tT-47D\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,Luciferase Reporter Assay\"\\t\"In contrast to the broad RING1B ChIP-seq signals in pluripotent cells, RING1B peaks in the breast cell lines were narrow, resembling ChIP-seq signals of transcription factors. Therefore, we assessed whether RING1B is recruited to specific transcription factor-binding sites at SEs.\"\\tSuper-Enhancer\\tFOXA1\\t--\\tqRT-PCR\\t\"In T47D cells, RING1B bound to a putative SE downstream of FOXA1, suggesting that it plays an activating role in regulating FOXA1 expression.\"\\t\"HNF3A,TCF3A\"\\tBreast Cancer\\tDOID:1612\\tD001943\\tRING1B regulates Enhancer activity and gene transcription not only by promoting the expression of oncogenes but also by regulating chromatin accessibility.\\tATAC-seq\\t\"RING1B was recruited to regions targeted by transcription factors and its depletion deregulated breast cancer signaling pathways as well as FOXA1 and ER localization to chromatin, we next hypothesized that RING1B regulates transcriptional programs in breast cancer by orchestrating chromatin accessibility. To test', 'E_01_337\\t30139998\\t--\\thg19\\tchr6\\t38032307\\t38035230\\tHuman\\tT-47D\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,Luciferase Reporter Assay\"\\t\"In contrast to the broad RING1B ChIP-seq signals in pluripotent cells, RING1B peaks in the breast cell lines were narrow, resembling ChIP-seq signals of transcription factors. Therefore, we assessed whether RING1B is recruited to specific transcription factor-binding sites at SEs.\"\\tSuper-Enhancer\\tFOXA1\\t--\\tqRT-PCR\\t\"In T47D cells, RING1B bound to a putative SE downstream of FOXA1, suggesting that it plays an activating role in regulating FOXA1 expression.\"\\t\"HNF3A,TCF3A\"\\tBreast Cancer\\tDOID:1612\\tD001943\\tRING1B regulates Enhancer activity and gene transcription not only by promoting the expression of oncogenes but also by regulating chromatin accessibility.\\tATAC-seq\\t\"RING1B was recruited to regions targeted by transcription factors and its depletion deregulated breast cancer signaling pathways as well as FOXA1 and ER localization to chromatin, we next hypothesized that RING1B regulates transcriptional programs in breast cancer by orchestrating chromatin accessibility. To test', 'E_01_338\\t30139998\\t--\\thg19\\tchr6\\t151917241\\t152028736\\tHuman\\tT-47D\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,Luciferase Reporter Assay\"\\t\"In contrast to the broad RING1B ChIP-seq signals in pluripotent cells, RING1B peaks in the breast cell lines were narrow, resembling ChIP-seq signals of transcription factors. Therefore, we assessed whether RING1B is recruited to specific transcription factor-binding sites at SEs.\"\\tSuper-Enhancer\\tESR1\\t--\\tqRT-PCR\\t\"Indeed,we observed RING1B recruitment at SE regions near BCL2L1 in MDA-MB-231 and ESR1 in T47D23.\"\\t\"ER,ESR,ESRA,ESTRR,Era,NR3A1\"\\tBreast Cancer\\tDOID:1612\\tD001943\\tRING1B regulates Enhancer activity and gene transcription not only by promoting the expression of oncogenes but also by regulating chromatin accessibility.\\tATAC-seq\\t\"RING1B was recruited to regions targeted by transcription factors and its depletion deregulated breast cancer signaling pathways as well as FOXA1 and ER localization to chromatin, we next hypothesized that RING1B regulates transcriptional programs in breast cancer by orchestrating chromatin accessibility. To test', 'E_01_339\\t30397001\\tgtPBREM Enhancer\\thg19\\tchr2\\t234665349\\t234665739\\tHuman\\tHep G2\\tLow throughput\\t\"Luciferase Reporter Assay,qRT-PCR\"\\t\"These observations suggested that a specific enhancer region, such as neighboring transcriptional factors, is required for CAR/CCAR1 interaction.\"\\tEnhancer\\tUGT1A1\\t--\\tChIP\\tThe phenobarbital-responsive Enhancer module of UGT1A1 (gtPBREM)-driven Luciferaseplasmid (gtPBREM-Luc.) containing a 290-bp distal Enhancer module (23570/23180) was constructed based on pGL3-promoter (Promega).\\t\"BILIQTL1,GNT1,HUG-BR1,UDPGT,UDPGT1-1,UGT1,UGT1A\"\\tHepatocellular Carcinoma\\tDOID:684\\tD006528\\tCCAR1 enhances transcriptional activity of CAR and acts as an Enhancer-dependent selective cofactor of CAR for UGT1A1 expression.\\tLuciferase Reporter Assay\\tThe phenobarbital-responsive Enhancer module of UGT1A1 (gtPBREM)-driven luciferase reporter plasmid (gtPBREM-Luc.) containing a 290-bp distal Enhancer module (-23570/-23180) was constructed based on pGL3-promoter (Promega).\\tCCAR1\\tCCAR1\\tChIP\\t\"CCAR1 regulates CAR-mediated mRNA expression (Figs. 4 and 5) and reporter activity (Figs. 2 and 3) of UGT1A1 but not CYP2B6.We speculated that CCAR1 is differentially recruited to CAR-responsive Enhancer regions. To investigate this hypothesis,we carried out ChIP experiments in HepTR/CAR cells.\"\\t--\\t--\\t--\\t>2KB\\t3374', 'E_01_340\\t30511964\\t--\\thg19\\tchr14\\t76419740\\t76421740\\tHuman\\tLNCaP\\tLow+High throughput\\t\"ChIP-seq,ChIP-qPCR\"\\tAnalysis of FOXA1 ChIP-seq previously performed in LNCaP cells identified strong FOXA1 binding at an enhancer of approximately 3.7 kb upstream of the TGFB3 gene promoter.\\tEnhancer\\tTGFB3\\t--\\tChIP-qPCR\\t\"Analysis of FOXA1 ChIP-seq previously performed in LNCaP cells identified strong FOXA1 binding at an Enhancer of approximately 3.7 kb upstream of the TGFB3 gene promoter.Importantly, qRT-PCR analysis demonstrated that CRISPR-mediated Enhancer deletion in only one-fourth of the cell population was able to substantially restore the transcription of TGFB3 gene, comparing cells transfected with TGFB3-targeting gRNAs to those with control gRNAs.\"\\t\"ARVD,ARVD1,LDS5,RNHF,TGF-beta3\"\\tProstate Cancer\\tDOID:10283\\tD011471\\t--\\t--\\t--\\tFOXA1\\t\"HNF3A,TCF3A\"\\tCRISPR/Cas9\\t\"To demonstrate that FOXA1 binding at the upstream Enhancer indeed inhibits TGFB3 gene transcription in vivo, we used a lentiviral CRISPR/Cas9 editing system to delete the Enhancer elements with small guide RNAs.Taken together, these results strongly support that FOXA1 binds to an upstream Enhancer to directly repress TGFB3 transcription.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3663', 'E_01_341\\t30518632\\t--\\thg19\\tchr1\\t47704791\\t47705107\\tHuman\\tT-acute lymphoblastic leukemia (T-ALL)\\tLow throughput\\t\"ChIP-qPCR,qRT-PCR,ChIP,Transgenic mice\"\\t\"We found robust cytotoxicity against T-ALL cells, but not normal bone marrow progenitors, for two N-alkylated TCP derivatives, S2116 and S2157. The two compounds induced apoptosis in TCP-resistant T-ALL cells in vitro and in vivo by repressing transcription of the NOTCH3 and TAL1 genes through increased H3K9 methylation and reciprocal H3K27 deacetylation at super-enhancer regions.\"\\tSuper-Enhancer\\tTAL1\\t--\\t\"qRT-PCR,ChIP\"\\t\"we found that both S2116 and S2157 readily increased the methylation level of H3K9 and reciprocally reduced the acetylation level of H3K27 at Super-Enhancer regions of the NOTCH3 and TAL1 genes (GRCh38/hg38: 15,198,031-15,197,862 and GRCh38/hg38: 47,239,435-47,239,119,respectively) using ChIP assays.In addition,global ChIP-seq analyses revealed that the acetylation level of H3K27 was readily decreased by LSD1 inhibition through the entire range of NOTCH3 and TAL1 Enhancers.\"\\t\"SCL,TCL5,bHLHa17,tal-1\"\\tCentral Nervous System Leukemia\\tDOID:12969\\t--\\t--\\t--\\t--\\tZEB2\\t\"HSPC082,SIP-1,SIP1,SMADIP1,ZFHX1B\"\\tRT-PCR\\t\"We used the Expression Assays(Hs01097987 for TAL1, Hs01128537 for NOTCH3, Hs00207691 for ZEB2, and Hs01922876 forGAPDH) and TaqMan Universal PCR Master Mix (Applied Biosystems, Waltham, MA) for real-time quantitative RT-PCR (RQ-PCR).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t22988', 'E_01_342\\t30518632\\t--\\thg19\\tchr19\\t15308673\\t15308842\\tHuman\\tT-acute lymphoblastic leukemia (T-ALL)\\tLow throughput\\t\"ChIP-qPCR,qRT-PCR,ChIP,Transgenic mice\"\\t\"We found robust cytotoxicity against T-ALL cells, but not normal bone marrow progenitors, for two N-alkylated TCP derivatives, S2116 and S2157. The two compounds induced apoptosis in TCP-resistant T-ALL cells in vitro and in vivo by repressing transcription of the NOTCH3 and TAL1 genes through increased H3K9 methylation and reciprocal H3K27 deacetylation at super-enhancer regions.\"\\tSuper-Enhancer\\tNOTCH3\\t--\\t\"qRT-PCR,ChIP\"\\t\"we found that both S2116 and S2157 readily increased the methylation level of H3K9 and reciprocally reduced the acetylation level of H3K27 at Super-Enhancer regions of the NOTCH3 and TAL1 genes (GRCh38/hg38: 15,198,031-15,197,862 and GRCh38/hg38: 47,239,435-47,239,119,respectively) using ChIP assays.In addition,global ChIP-seq analyses revealed that the acetylation level of H3K27 was readily decreased by LSD1 inhibition through the entire range of NOTCH3 and TAL1 Enhancers.\"\\t\"CADASIL,CADASIL1,CASIL,IMF2,LMNS\"\\tCentral Nervous System Leukemia\\tDOID:12969\\t--\\t--\\t--\\t--\\tZEB2\\t\"HSPC082,SIP-1,SIP1,SMADIP1,ZFHX1B\"\\tRT-PCR\\t\"We used the Expression Assays(Hs01097987 for TAL1, Hs01128537 for NOTCH3, Hs00207691 for ZEB2, and Hs01922876 forGAPDH) and TaqMan Universal PCR Master Mix (Applied Biosystems, Waltham, MA) for real-time quantitative RT-PCR (RQ-PCR).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t38315', 'E_01_343\\t30566872\\t--\\thg19\\tchr10\\t33492690\\t33519530\\tHuman\\tNormal Epidermal Cell\\tLow+High throughput\\t\"siRNA,ChIP-seq\"\\t\"Two clusters of active enhancers were bound by p63 . Regions in C3 showed higher p63-binding signals.nearby genes were involved in apoptosis and epidermis development. Regions in C4 had relatively lower p63-binding signals, and nearby genes were involved in keratinocyte differentiation.\"\\tEnhancer\\tNRP1\\t--\\tsiRNA\\t\"Indeed, we observed a clear decrease of H3K27ac signals at Enhancers that had higher H3K27ac signals and bound by RUNX1 in p63 mutant keratinocytes (Figure 6B; n = 6,035) in two biological replicas, such as Enhancers near NRP1, which is involved in angiogenesis .\"\\t\"BDCA4,CD304,NP1,NRP,VEGF165R\"\\tEEC syndrome\\tDOID:0060782\\tC536189\\t--\\t--\\t--\\t\"TP63,RUNX1\"\\t\"AIS,B(p51A),B(p51B),EEC3,KET,LMS,NBP,OFC8,RHS,SHFM4,TP53CP,TP53L,TP73L,p40,p51,p53CP,p63,p73H,p73L,AML1, AML1-EVI-1, AMLCR1, CBF2alpha, CBFA2, EVI-1, PEBP2aB, PEBP2alpha\"\\t\"siRNA,qRT-PCR\"\\tGene expression analysis by qRT-PCR of TP63 and RUNX1 expression in TP63 knockdown (siTP63) compared to non-targeting siRNA (siNT) in control keratinocytes.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t39692', 'E_01_344\\t30566872\\t--\\thg19\\tchr10\\t33512793\\t33539633\\tHuman\\tNormal Epidermal Cell\\tLow+High throughput\\t\"siRNA,ChIP-seq\"\\t\"Two clusters of active enhancers were bound by p63 . Regions in C3 showed higher p63-binding signals.nearby genes were involved in apoptosis and epidermis development. Regions in C4 had relatively lower p63-binding signals, and nearby genes were involved in keratinocyte differentiation.\"\\tEnhancer\\tNRP1\\t--\\tsiRNA\\t\"Indeed, we observed a clear decrease of H3K27ac signals at Enhancers that had higher H3K27ac signals and bound by RUNX1 in p63 mutant keratinocytes (Figure 6B; n = 6,035) in two biological replicas (Figures 6H and S7E), such as Enhancers near NRP1, which is involved in angiogenesis (Figure 6I).\"\\t\"BDCA4,CD304,NP1,NRP,VEGF165R\"\\tEEC syndrome\\tDOID:0060782\\tC536189\\t--\\t--\\t--\\t\"TP63,RUNX1\"\\t\"AIS,B(p51A),B(p51B),EEC3,KET,LMS,NBP,OFC8,RHS,SHFM4,TP53CP,TP53L,TP73L,p40,p51,p53CP,p63,p73H,p73L,AML1, AML1-EVI-1, AMLCR1, CBF2alpha, CBFA2, EVI-1, PEBP2aB, PEBP2alpha\"\\t\"siRNA,qRT-PCR\"\\tGene expression analysis by qRT-PCR of TP63 and RUNX1 expression in TP63 knockdown (siTP63) compared to non-targeting siRNA (siNT) in control keratinocytes.\\t--\\t--\\t--\\t>2KB\\t\\t\\t59795', 'E_01_345\\t30566872\\t--\\thg19\\tchr10\\t33637535\\t33664375\\tHuman\\tNormal Epidermal Cell\\tLow+High throughput\\t\"siRNA,ChIP-seq\"\\t\"Two clusters of active enhancers were bound by p63 . Regions in C3 showed higher p63-binding signals.nearby genes were involved in apoptosis and epidermis development. Regions in C4 had relatively lower p63-binding signals, and nearby genes were involved in keratinocyte differentiation.\"\\tEnhancer\\tNRP1\\t--\\tsiRNA\\t\"Indeed, we observed a clear decrease of H3K27ac signals at Enhancers that had higher H3K27ac signals and bound by RUNX1 in p63 mutant keratinocytes (Figure 6B; n = 6,035) in two biological replicas (Figures 6H and S7E), such as Enhancers near NRP1, which is involved in angiogenesis (Figure 6I).\"\\t\"BDCA4,CD304,NP1,NRP,VEGF165R\"\\tEEC syndrome\\tDOID:0060782\\tC536189\\t--\\t--\\t--\\t\"TP63,RUNX1\"\\t\"AIS,B(p51A),B(p51B),EEC3,KET,LMS,NBP,OFC8,RHS,SHFM4,TP53CP,TP53L,TP73L,p40,p51,p53CP,p63,p73H,p73L,AML1, AML1-EVI-1, AMLCR1, CBF2alpha, CBFA2, EVI-1, PEBP2aB, PEBP2alpha\"\\t\"siRNA,qRT-PCR\"\\tGene expression analysis by qRT-PCR of TP63 and RUNX1 expression in TP63 knockdown (siTP63) compared to non-targeting siRNA (siNT) in control keratinocytes.\\t--\\t--\\t--\\t>2KB\\t\\t\\t184537', 'E_01_346\\t30568279\\t--\\thg19\\tchr14\\t106327635\\t106328903\\tHuman\\tGM12878\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"Surprisingly, this enrichment stayed constant thereafter for promoter regions, and increased modestly for enhancer regions, ultimately surpassing the enrichment seen for promoters. In fact, even though promoter chromatin states were more enriched at intermediate HiDRA activity levels,enhancer chromatin states were the most enriched at the highest HiDRA activity levels , suggesting that enhancer elements have a greater dynamic range of regulatory activity potential, which has implications for the regulatory architecture of genes.\"\\tEnhancer\\tIGHE\\t--\\tLuciferase Reporter Assay\\tWe observed that the peak of HiDRA activity is centered precisely within the region previously identi?ed as driving Enhancer activity in low-throughput luciferase assays (Fig. 2c).\\tIgE\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t261866', 'E_01_347\\t30568279\\t--\\thg19\\tchr9\\t126900000\\t126900625\\tHuman\\tGM12878\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"Surprisingly, this enrichment stayed constant thereafter for promoter regions, and increased modestly for enhancer regions, ultimately surpassing the enrichment seen for promoters. In fact, even though promoter chromatin states were more enriched at intermediate HiDRA activity levels,enhancer chromatin states were the most enriched at the highest HiDRA activity levels , suggesting that enhancer elements have a greater dynamic range of regulatory activity potential, which has implications for the regulatory architecture of genes.\"\\tEnhancer\\tNEK6\\t--\\tLuciferase Reporter Assay\\tA visualization of 14 luciferase-tested Enhancers in the serine/threonine kinase NEK6 locus shows a strong correspondence between luciferase assay results and HiDRA active regions.\\tSID6-1512\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t119571', 'E_01_348\\t30568279\\t--\\thg19\\tchr9\\t126964375\\t126965000\\tHuman\\tGM12878\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"Surprisingly, this enrichment stayed constant thereafter for promoter regions, and increased modestly for enhancer regions, ultimately surpassing the enrichment seen for promoters. In fact, even though promoter chromatin states were more enriched at intermediate HiDRA activity levels,enhancer chromatin states were the most enriched at the highest HiDRA activity levels , suggesting that enhancer elements have a greater dynamic range of regulatory activity potential, which has implications for the regulatory architecture of genes.\"\\tEnhancer\\tNEK6\\t--\\tLuciferase Reporter Assay\\tA visualization of 14 luciferase-tested Enhancers in the serine/threonine kinase NEK7 locus shows a strong correspondence between luciferase assay results and HiDRA active regions.\\tSID6-1512\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t55196', 'E_01_349\\t30568279\\t--\\thg19\\tchr9\\t126980625\\t126981125\\tHuman\\tGM12878\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"Surprisingly, this enrichment stayed constant thereafter for promoter regions, and increased modestly for enhancer regions, ultimately surpassing the enrichment seen for promoters. In fact, even though promoter chromatin states were more enriched at intermediate HiDRA activity levels,enhancer chromatin states were the most enriched at the highest HiDRA activity levels , suggesting that enhancer elements have a greater dynamic range of regulatory activity potential, which has implications for the regulatory architecture of genes.\"\\tEnhancer\\tNEK6\\t--\\tLuciferase Reporter Assay\\tA visualization of 14 luciferase-tested Enhancers in the serine/threonine kinase NEK8 locus shows a strong correspondence between luciferase assay results and HiDRA active regions.\\tSID6-1512\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t39009', 'E_01_350\\t30568279\\t--\\thg19\\tchr9\\t127001250\\t127001875\\tHuman\\tGM12878\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"Surprisingly, this enrichment stayed constant thereafter for promoter regions, and increased modestly for enhancer regions, ultimately surpassing the enrichment seen for promoters. In fact, even though promoter chromatin states were more enriched at intermediate HiDRA activity levels,enhancer chromatin states were the most enriched at the highest HiDRA activity levels , suggesting that enhancer elements have a greater dynamic range of regulatory activity potential, which has implications for the regulatory architecture of genes.\"\\tEnhancer\\tNEK6\\t--\\tLuciferase Reporter Assay\\tA visualization of 14 luciferase-tested Enhancers in the serine/threonine kinase NEK9 locus shows a strong correspondence between luciferase assay results and HiDRA active regions.\\tSID6-1512\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t18321', 'E_01_351\\t30568279\\t--\\thg19\\tchr9\\t127047500\\t127048375\\tHuman\\tGM12878\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"Surprisingly, this enrichment stayed constant thereafter for promoter regions, and increased modestly for enhancer regions, ultimately surpassing the enrichment seen for promoters. In fact, even though promoter chromatin states were more enriched at intermediate HiDRA activity levels,enhancer chromatin states were the most enriched at the highest HiDRA activity levels , suggesting that enhancer elements have a greater dynamic range of regulatory activity potential, which has implications for the regulatory architecture of genes.\"\\tEnhancer\\tNEK6\\t--\\tLuciferase Reporter Assay\\tA visualization of 14 luciferase-tested Enhancers in the serine/threonine kinase NEK10 locus shows a strong correspondence between luciferase assay results and HiDRA active regions.\\tSID6-1512\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t28054', 'E_01_352\\t30590035\\tPOU5F1 proximal Enhancer\\thg19\\tchr6\\t31139519\\t31140167\\tHuman\\tHuman Primordial Germ Cell\\tLow+High throughput\\t\"ATAC-seq,ChIP-seq\"\\t\"These data show that the PE, DE, and NE are all open in hPGCLCs at day 4 of aggregate differentiation, with the NE being more open in hPGCs than the DE and PE. This observation raises the possibility that restriction of OCT4 expression between days 2 and 3 of aggregate differentiation may be due to NE and/or the DE enhancer activation at the OCT4 locus. Given that the NE is bound by TFAP2C in groundstate naive pluripotent stem cells, whereas the DE is not , we next confirmed that the OCT4 NE is also a target of TFAP2C during hPGCLC differentiation.\"\\tEnhancer\\tPOU5F1\\t--\\tChIP-qPCR\\t\"Screenshot of ATAC-seq and TFAP2C ChIP-seq signals showing three Enhancers at the POU5F1 locus (encoding OCT4). Shaded boxes highlight the naive Enhancer(NE),proximalEnhancer(PE),anddistalEnhancer(DE)atthePOU5F1locus.DEdeletion andNEdeletion indicategenomicregionsthatweredeleted by CRISPR/Cas9-mediated genome editing. Primers for ChIP-qPCR (P1-2,P3-4,P5-6) of NE and control regions are shown.\"\\t\"OCT3, OCT4, OTF-3, OTF3, OTF4, Oct-3, Oct-4\"\\t--\\t--\\t--\\t\"TFAP2C regulates human germline cell formation is opening naive-specific Enhancers, with one of these Enhancers corresponding to the NE at the OCT4 locus.\"\\tCRISPR/Cas9\\t\"Using CRISPR/Cas9, we show that deleting the TFAP2C-bound naive enhancer at the OCT4 locus (also called POU5F1) results in impaired OCT4 expression and a negative effect on hPGCLC identity.\"\\tTFAP2C\\t\"AP2-GAMMA,ERF1,TFAP2G,hAP-2g\"\\tCRISPR/Cas9\\t\"Using CRISPR/Cas9, we show that deleting the TFAP2C-bound naive Enhancer at the OCT4 locus (also called POU5F1) results in impaired OCT4 expression and a negative effect on hPGCLC identity.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t7730', 'E_01_353\\t30590035\\tPOU5F1 naive Enhancer\\thg19\\tchr6\\t31137056\\t31138028\\tHuman\\tHuman Primordial Germ Cell\\tLow+High throughput\\t\"ATAC-seq,ChIP-seq\"\\t\"These data show that the PE, DE, and NE are all open in hPGCLCs at day 4 of aggregate differentiation, with the NE being more open in hPGCs than the DE and PE. This observation raises the possibility that restriction of OCT4 expression between days 3 and\"\\tEnhancer\\tPOU5F1\\t--\\tChIP-qPCR\\t\"Screenshot of ATAC-seq and TFAP2C ChIP-seq signals showing three Enhancers at the POU5F1 locus (encoding OCT4). Shaded boxes highlight the naive Enhancer(NE),proximalEnhancer(PE),anddistalEnhancer(DE)atthePOU5F1locus.DEdeletion andNEdeletion indicategenomicregionsthatweredeleted by CRISPR/Cas9-mediated genome editing. Primers for ChIP-qPCR (P1-2,P3-4,P5-6) of NE and control regions are shown.\"\\t\"OCT3, OCT4, OTF-3, OTF3, OTF4, Oct-3, Oct-4\"\\t--\\t--\\t--\\t\"The PE, DE, and NE are all open in hPGCLCs at day 4 of aggregate differentiation.\"\\tATAC-seq\\t\"First, to determine whether the NE is also open in human germline cells, we compared ATAC-seq peaks for the NE in hPGCLCs and hPGCs to this region in naive and primed hESCs (Figure 5C). These data show that the PE, DE, and NE are all open in hPGCLCs at day 4 of aggregate differentiation, with the NE being more open in hPGCs than the DE and PE.\"\\tTFAP2C\\t\"AP2-GAMMA,ERF1,TFAP2G,hAP-2g\"\\tCRISPR/Cas9\\t\"Using CRISPR/Cas9, we show that deleting the TFAP2C-bound naive Enhancer at the OCT4 locus (also called POU5F1) results in impaired OCT4 expression and a negative effect on hPGCLC identity.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5429', 'E_01_354\\t30590035\\tPOU5F1 distal Enhancer\\thg19\\tchr6\\t31140556\\t31141204\\tHuman\\tHuman Primordial Germ Cell\\tLow+High throughput\\t\"ATAC-seq,ChIP-seq,ChIP-qPCR\"\\t\"Using chromatin immunoprecipitation sequencing (ChIP-seq) for TFAP2C in 5i/L/FA cultured naive hPSCs (Pastor et al., 2018), we found that the 1,892 hPGC and hPGCLC overlapping peaks are also bound by TFAP2C in 5i/L/FA cultured cells (Figure 3C).Analysis of the 1,892 peaks enriched in the naive-specific and germline cell-specific intersect group revealed a significant enrichment of AP2 motifs, as expected (Figure S3B).This observation raises the possibility that restriction of OCT4 expression between days 2 and 3 of aggregate differentiation may be due to NE and/or the DE enhancer activation at the OCT4 locus. Given that the NE is bound by TFAP2C in ground-state naive pluripotent stem cells, whereas the DE is not (Figure 5C), we next confirmed that the OCT4 NE is also a target of TFAP2C during hPGCLC differentiation. To do this, we performed ChIP-qPCR on day 4 aggregates containing hPGCLCs and discovered that TFAP2C is bound to the NE, whereas it is not bound to a genomic region that does not contain AP2 sites (Figure 5D). Taken together, these experiments suggest that OCT4 regulation during hPGCLC differentiation may involve enhancer activation at the DE as well as TFAP2C-bound enhancer activation at the NE.\"\\tEnhancer\\tPOU5F1\\t--\\tChIP-qPCR\\t\"Screenshot of ATAC-seq and TFAP2C ChIP-seq signals showing three Enhancers at the POU5F1 locus (encoding OCT4). Shaded boxes highlight the naive Enhancer(NE),proximalEnhancer(PE),anddistalEnhancer(DE)atthePOU5F1locus.DEdeletion andNEdeletion indicategenomicregionsthatweredeleted by CRISPR/Cas9-mediated genome editing. Primers for ChIP-qPCR (P1-2,P3-4,P5-6) of NE and control regions are shown.\"\\t\"OCT3, OCT4, OTF-3, OTF3, OTF4, Oct-3, Oct-4\"\\t--\\t--\\t--\\tOCT4 regulation during hPGCLC differentiation may involve enhancer activation at the DE as well as TFAP2C-bound enhancer activation at the NE.\\t\"ATAC-seq,ChIP-qPCR\"\\t\"Taken together, these experiments suggest that OCT4 regulation during hPGCLC differentiation may involve enhancer activation at the DE as well as TFAP2C-bound', 'E_01_355\\t18695675\\tIgH 3\\' Enhancer\\thg19\\tchr14\\t105051517\\t105351516\\tHuman\\t\"DHL-4,DHL-9\"\\tLow+High throughput\\t\"3C,PCR,ChIP\"\\t\"To determine whether the IgH 3\\' enhancers interact with the bcl-2 promoter region, 3C analysis was performed with primers near the bcl-2 ranscriptional start sites on chromosome 18 and primers on chromosome 14 (105,051,517-105,351,516). The region of chromosome 14 is centered approximately between the two IgH 3\\' enhancer regions downstream of C1 and C2.\"\\tEnhancer\\tBCL2\\t3C\\tPCR\\t\"To determine whether the IgH 3\\' Enhancers interact with the bcl-2 promoter region, 3C analysis was performed with primers near the bcl-2 transcriptional start sites on chromosome 18 and primers on chromosome 14.\"\\t\"Bcl-2,PPP1R50\"\\tLymphoma\\tDOID:0060058\\tD008223\\t\"The IgH 3\\' enhancers physically interact with the bcl-2 promoter region over a 350 kb genomic region in t(14;18) lymphoma cells. The physical', 'E_01_356\\t20093418\\tFKBP51 Enhancer\\thg19\\tchr6\\t35628362\\t35641362\\tHuman\\tA549\\tLow throughput\\t\"ChIP,Transgenic mice\"\\tOur quantitative chromatin immunoprecipitation scans and enhancer activity analyses indicate that activation of the FKBP51 locus by glucocorticoids in vivo is triggered by the loading of GR to enhancers at about 34 kb 5 and about 87 kb 3 of the transcription start site.\\tEnhancerFKBP4\\t--\\t\"qRT-PCR,Western blot\"\\tSchematic locations of the GREs and regions analyzed in reporter and ChIP assays.\\t\"FKBP51,FKBP52,FKBP59,HBI,Hsp56,PPIase,p52\"\\t--\\t--\\t--\\t--\\t--\\t--\\tCTCF\\tMRD21\\tChIP\\tWe used the genome-wide CTCF-binding information from the CTCF-binding site database for choosing the regions to be ChIP scanned with anti-CTCF antibody.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t32730755', 'E_01_357\\t20952403\\tRET Enhancer\\thg19\\tchr10\\t43569045\\t43569456\\tHuman\\tSK-N-BE\\tLow throughput\\tChIP\\t\"We then performed chromatin immunoprecipitation (ChIP) experiments to verify the presence of RARa, in RA-stimulated and unstimulated cells, at three different regions of RET locus:the enhancer region located about 3.4-kb upstream from the TSS, containing functional SOX10 and Pax3 binding sites and displaying a putative complete site for RARa binding;\"\\tEnhancer\\tRET\\t--\\t\"Luciferase Reporter Assay,qRT-PCR\"\\t\"The general structure of human RET gene and the main regulatory elements have been partially investigated and it has been reported that a conserved Enhancer region, located about 3000-bp upstream from a promoter region, contains binding sites for Sox10 and Pax3 transcription factors.\"\\t\"CDHF12,CDHR16,HSCR1,MEN2A,MEN2B,MTC1,PTC,RET-ELE1\"\\t--\\t--\\t--\\tThe primary epigenetic determinants of RA-induced RET activation differ between Enhancer and promoter regions.\\t\"Luciferase Reporter Assay,qRT-PCR,ChIP\"\\t\"Then we performed luciferase transcriptional assays in order to investigate whether a 3.4-kb RET gene regulatory region,located upstream from TSS and including both promoter and Enhancer,was sufficient for RA responsiveness.The results,shown in Figure 2A,demonstrated that plasmid constructs containing the 3.4-kb RET region upstream the luciferase gene were more active than empty plasmids(pGL3-basic);however, no significant increase of luciferase activity was observed when trans_x0002_fected cells were treated with RA.\"\\t\"SOX10,PAX3\"\\t\"DOM,PCWH,WS2E,WS4,WS4C,CDHS,HUP2,WS1,WS3\"\\tChIP\\t\"We then performed chromatin immunoprecipitation (ChIP) experiments to verify the presence of RARa, in RA-stimulated and un_x0002_stimulated cells, at three different regions of RET locus (Figure 1A):(i) the Enhancer region located about 3.4-kb upstream from the TSS, containing functional SOX10 and Pax3 binding sites.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3265', 'E_01_358\\t22543974\\t--\\thg19\\tchr12\\t114463712\\t114464080\\tHuman\\t\"HeLa,293T\"\\tLow+High throughput\\t\"EMSA,PCR,ChIP-seq\"\\t\"This strategy considered multiple genome-wide ChIP-seq data sets including the enhancer associated transcriptional co-activator p300, cardiac transcription factors Mef2a, TBX5, Gata4, Nkx2.5, SRF and computationally predicted enhancers, as well as the enhancer-associated histonemodificationH3K4me1and evolutionary conservation to identify and prioritize likely functional non-coding regions.\"\\tEnhancer\\tTBX5\\t--\\t\"EMSA,PCR\"\\t\"We identified three elements (CREs 2, 9, 16) with patterns of Enhancer activity overlapping the endogenous expression pattern of TBX5 and the Enhancer-trap BACs in which they were contained.\"\\tHOS\\tCongenital Heart Disease\\tDOID:1682\\tD006330\\t\"A significant number of CHD associated with TBX5 dysfunction might arise from non-coding mutations in TBX5 heart enhancers, effectively decoupling the heart and hand phenotypes of the Holt-Oram syndrome.\"\\t\"EMSA,ChIP-seq\"\\t\"Notably, all lines exhibited weak expression in the left atrium and almost no ex_x0002_pression in the right atrium, suggesting that the bulk of TBX5 atrial expression is regulated by Enhancers outside the genomic region covered by this BAC. \"\\tTBX5\\tHOS\\tEMSA\\t\"We hypothesized that the variant nucleotide disrupts transcrip_x0002_tion factor binding to Enhancer 9. To test this, we performed an electrophoretic mobility shift assay using short DNA probes spanning either the wild-type (G) or variant (T) allele.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t327838', 'E_01_359\\t22543974\\t--\\thg19\\tchr12\\t114701207\\t114704691\\tHuman\\t\"HeLa,293T\"\\tLow+High throughput\\t\"EMSA,PCR,ChIP-seq\"\\t\"This strategy considered multiple genome-wide ChIP-seq data sets including the enhancer associated transcriptional co-activator p300, cardiac transcription factors Mef2a, TBX5, Gata4, Nkx2.6, SRF and computationally predicted enhancers, as well as the enh?\"\\tEnhancerTBX5\\t--\\t\"EMSA,PCR\"\\t\"We identified three elements (CREs 2, 9, 16) with patterns of Enhancer activity overlapping the endogenous expression pattern of TBX5 and the Enhancer-trap BACs in which they were contained.\"\\tHOS\\tCongenital Heart Disease\\tDOID:1682\\tD006330\\t\"Asignificant number of CHD associated with TBX5 dysfunction might arise from non-coding mutations in TBX5 heart enhancers, effectively decoupling the heart and hand phenotypes of the Holt-Oram syndrome.\"\\t\"EMSA,ChIP-seq\"\\t\"Notably, all lines exhibited weak expression in the left atrium and almost no ex_x0002_pression in the right atrium, suggesting that the bulk of TBX5 atrial expression is regulated by Enhancers outside the genomic region covered by this BAC. \"\\tTBX5\\tHOS\\tEMSA\\t\"We hypothesized that the variant nucleotide disrupts transcrip_x0002_tion factor binding to Enhancer 9. To test this, we performed an electrophoretic mobility shift assay using short DNA probes spanning either the wild-type (G) or variant (T) allele.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t88785', 'E_01_360\\t22543974\\t--\\thg19\\tchr12\\t114853271\\t114858238\\tHuman\\t\"HeLa,293T\"\\tLow+High throughput\\t\"EMSA,PCR,ChIP-seq\"\\t\"This strategy considered multiple genome-wide ChIP-seq data sets including the enhancer associated transcriptional co-activator p300, cardiac transcription factors Mef2a, TBX5, Gata4, Nkx2.7, SRF and computationally predicted enhancers, as well as the enh?\"\\tEnhancerTBX5\\t--\\t\"EMSA,PCR\"\\t\"We identified three elements (CREs 2, 9, 16) with patterns of Enhancer activity overlapping the endogenous expression pattern of TBX5 and the Enhancer-trap BACs in which they were contained.\"\\tHOS\\tCongenital Heart Disease\\tDOID:1682\\tD006330\\t\"A significant number of CHD associated with TBX5 dysfunction might arise from non-coding mutations in TBX5 heart enhancers, effectively decoupling the heart and hand phenotypes of the Holt-Oram syndrome.\"\\t\"EMSA,ChIP-seq\"\\t\"Notably, all lines exhibited weak expression in the left atrium and almost no ex_x0002_pression in the right atrium, suggesting that the bulk of TBX5 atrial expression is regulated by Enhancers outside the genomic region covered by this BAC. \"\\tTBX5\\tHOS\\tEMSA\\t\"We hypothesized that the variant nucleotide disrupts transcrip_x0002_tion factor binding to Enhancer 9. To test this, we performed an electrophoretic mobility shift assay using short DNA probes spanning either the wild-type (G) or variant (T) allele.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t64021', 'E_01_361\\t23108396\\t--\\thg19\\tchr11\\t69468761\\t69470761\\tHuman\\t\"MDA-MB-23,MDA-MB-436\"\\tLow+High throughput\\t\"3C,ChIP,qPCR\"\\t\"ChIP confirmed that H2A.Z siRNA treatment lead to markedly reduced binding of H2A.Z to the CCND1 TSS. Concomitant to increased mRNA levels, polymerase II (pol II) recruitment to the CCND1 TSS slightly increased.\"\\tEnhancer\\tCCND1\\t3C\\tqPCR\\t\"We conclude that enh2 has an active role in ER-independent CCND1 activation.To investigate whether the CCND1 TSS interacts with enh2, we used a chromatin conformation capture (3C) assay,which detects physical proximity between distal DNA sites by ligation of cross-linked restricted DNA fragments.\"\\t\"BCL1,D11S287E,PRAD1,U21B31\"\\t--\\t--\\t--\\tThe enh2 has an active role in ER-independent CCND1 activation.\\tChIP\\t\"In addition, CCND1 activation in siH2AZ-transfected MDA-MB231 cells was accompanied by a strong recruitment of pol II to enh2 (Figure 1c, right panel), as described previously for E2 stimulated MCF-7 cells. We conclude that enh2 has an active role in ER-independent CCND1 activation.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t13889', 'E_01_362\\t23637611\\t--\\thg19\\tchr11\\t69468873\\t69470873\\tHuman\\tMCF-7\\tLow throughput\\t\"ChIP,PCR,3C\"\\t\"Long-range chromatin interactions between ERa recognition sequences and enhancers have been proposed to regulate ERa target genes in breast cancer cells. The main enhancer regulating CCND1 is located at the 39 end of the gene, 14 kb distant from the promoter.\"\\tEnhancer\\tCCND1\\t3C\\tPCR\\t\"The main Enhancer regulating CCND1 is located at the 3\\' end of the gene,14 kb distant from the promoter.\"\\t\"BCL1,D11S287E,PRAD1,U21B31\"\\tBreast Tumor\\tDOID:3459\\tD001943\\t--\\t--\\t--\\tKAT5\\t\"ESA1,HTATIP,HTATIP1,PLIP,TIP,TIP60,ZC2HC5,cPLA2\"\\tChIP\\tTIP60 binding to the CCND1 promoter was analyzed by ChIP and shown as percent of input (n = 2).\\t--\\t--\\t--\\t>2KB\\t\\t\\t14001', 'E_01_363\\t24443471\\t--\\thg19\\tchr7\\t116374406\\t116376406\\tHuman\\tCOLO 829\\tLow throughput\\t\"3C,ChIP\"\\t\"Chromosome conformation capture (3C)was performed in COLO829 cells in the presence or absence of PLX4032 using the MET +63-kb enhancer as an anchor region. We observed that BRAF inhibition strikingly increased the interaction between the MET +63-kb enhancer and the MET TSS, demonstrating that this lineage-specific enhancer undergoes inducible chromatin looping in response to oncogene withdrawal in melanoma cells.\"\\tEnhancer\\tMET\\t3C\\tChIP\\tChromosome conformation capture (3C)was performed in COLO829 cells in the presence or absence of PLX4032 using the MET +63-kb Enhancer as an anchor region.We observed that BRAF inhibition strikingly increased the interaction between the MET +63-kb Enhancer and the MET TSS.\\t\"AUTS9,DFNB97,HGFR,RCCP2,c-Met\"\\tMelanoma\\tDOID:1909\\tD008545\\tThis enhancer displays inducible chromatin looping with the MET promoter to up-regulate MET expression upon BRAF inhibition.\\t\"ChIP,3C\"\\t\"ChIP and 3C analysis were performed on wild-type or Enhancer-edited cells treated with PLX4032.A significant decrease in both H3K27 acetylation,a histone modification associated with active Enhancer state,and Enhancer-promoter chromatin looping was observed in Enhancer-edited cells. \"\\tMITF\\t\"CMM8,COMMAD,MI,WS2,WS2A,bHLHe32\"\\t\"ChIP-seq,3C\"\\t\"To test the necessity of MITF for inducible chromatin looping between the MET +63-kb Enhancer and MET TSS in response to BRAF inhibition, 3C was performed in COLO829 cells in the presence or absence of PLX4032 and MITF depletion.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t62963', 'E_01_364\\t24498324\\t--\\thg19\\tchr11\\t47363009\\t47364009\\tHuman\\tHuman Myeloid Cells\\tLow throughput\\tChIP\\t\"Using ChIP we focused on loci with more than 2-fold occupancy relative to control antibody. We dentified diffuse occupancy of CTCF with peaks involving amplicons -16.6 (downstream URE), -14.4 (Enhancer), and -11 (Element) as well as in the neighboring mplicons: -15.6, -13.7, -13.4, -13.3, -12.4 and also close to the promoter at -0.15.\"\\tEnhancer\\tSPI1\\t--\\t\"RT-PCR,Western blot,Luciferase Reporter Assay\"\\t\"To conclude this part,CTCF and SMARCA5 co-occupy its newly validated target SPI1 using ChIP assay at the -14.4 Enhancer upon AZA-mediated DNA demethylation.\"\\t\"OF,PU.1,SFPI1,SPI-1,SPI-A\"\\tAcute Myeloid Leukemia\\tDOID:9119\\tD015470\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t12899', 'E_01_365\\t25453903\\t--\\thg19\\tchr20\\t21452815\\t21462458\\tHuman\\t\"HeLa,293T\"\\tLow+High throughput\\t\"ChIP-seq,PCR,3C\"\\t\"WDR5 occupancy at activated EWS-FLI1 sites in SKNMC and A673 cells was also detected by ChIP-seq. Accordingly, ChIP-seq profiling of MSCs before and after fusion gene induction demonstrated that EWS-FLI1 recruits WDR5 to activated enhancers.\"\\tEnhancer\\tNKX2-2\\t3C\\tPCR\\t\"Genes proximal to EWS-FLI1 bound Enhancers include known regulators with critical functions in Ewing sarcoma,such as CCND1 and NKX2-2,as well as many novel targets.\"\\t\"NKX2.2,NKX2B\"\\tEwing Sarcoma\\tDOID:3369\\tD012512\\t--\\t--\\t--\\tELF1\\t\"EFTUD1,RIA1\"\\tChIP-seq\\t\"We focused on the ETS-related transcription factor ELF1,which has similar sequence preference to FLI1 and is expressed in A673 and SKNMC cells. We surveyed ELF1 binding in SKNMC cells by ChIP-seq before and after EWS-FLI1 knock-down.\"\\t--\\t--\\t--\\t>2KB\\t\\t34017', 'E_01_366\\t26166704\\t--\\thg19\\tchr21\\t42668000\\t42671000\\tHuman\\tBreast Cancer Cell\\tLow+High throughput\\t\"ChIP-seq,ChIP,GRO-seq\"\\t\"Previous work from our lab and others established that a subgroup of ER-/H3K27Ac co-bound enhancers (n=1,248,), exhibiting E2-upregulated eRNAs, high intensity of ER- binding and close proximity to estrogen target genes, constitute the major E2-activated functional enhancers in MCF-7 cells, referred to as E2-induced \"\"active enhancers\"\" or eRNA+ enhancers. \"\\tEnhancer\\tTFF1\\t3C\\tChIP-qPCR\\tGenome browser image showing the results of GRO-Seq and Pol II ChIP-Seq at TFF1 locus in the presence of condensin knockdown ( siNCAPG or siNCAPD3 ) vs siCTL transfected cell.\\t\"BCEI,D21S21,HP1.A,HPS2,pNR-2,pS2\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1112890', 'E_01_367\\t26445536\\t--\\thg19\\tchr22\\t35736840\\t35737280\\tHuman\\tEndothelial Cells (ECs)\\tLow+High throughput\\t\"ChIP,PCR\"\\t\"The ChIP analyses demonstrated that AuNP treatment significantly induced Nrf2 binding to the human HO-1 E2 enhancer region, but not to the promoter region near the transcription start site. \"\\tEnhancer\\tHMOX1\\t--\\t\"Western blot,Immunofluorescence\"\\tThe Chromatin immunoprecipitation assay revealed that the translocated Nrf2 bound to the antioxidant-response element located in the E2 Enhancer region of the HO-1 gene promoter and acted as a transcrip- tion factor.\\t\"HMOX1D,HO-1,HSP32,bK286B10\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t39999', 'E_01_368\\t26829750\\t--\\thg19\\tchr9\\t14538685\\t14540685\\tHuman\\tMCF-7\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,ChIP-seq\"\\tHere we identify super-enhancer translocations that drive overexpression of the oncogenic transcription factor MYB as a recurrent theme in adenoid cystic carcinoma (ACC). Whole-genome sequencing data and chromatin maps highlight distinct chromosomal rearrangements that juxtapose super-enhancers to the MYB locus. Chromosome conformation capture confirms that the translocated enhancers interact with the MYB promoter.\\tEnhancer\\tNFIB\\t3C\\t\"ChIP,ChIP-seq\"\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C). First, we examined an ACC with a translocation involving MYB and the NFIB locus. \"\\t\"CTF,HMGIC/NFIB,MACID,NF-I/B,NF1-B,NFI-B,NFI-RED2,NFIB3,NFIB\"\\tAdenoid Cystic Carcinoma\\tDOID:0080202\\tD003528\\tHere we identify the juxtaposition of super-enhancer regions to the MYB locus as the unifying feature of ACC translocations.Detailed genomic and epigenomic analyses of ACCs identify alternate rearrangements that translocate super-enhancers in the NFIB and\\t3C\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).These data suggest that the translocations reposition super-enhancers that subsequently loop to the MYB promoter and sustain high-level MYB expression.\"\\tTP63 \\t\"AIS, B(p51A), B(p51B), EEC3, KET, LMS, NBP, OFC8, RHS, SHFM4, TP53CP, TP53L, TP73L, p40, p51, p53CP, p63, p73H, p73L\"\\tChIP-seq\\t\"To directly test whether TP63 co-binds with MYB, we mapped binding of this transcription factor by ChIP-seq.Remarkably, we found that 81% of TP63-binding sites in ACC were co-bound by MYB.These data suggest that MYB,TP63 and Notch signaling may coordinately orchestrate the diverse expression programs in ACC.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t457843', 'E_01_369\\t26829750\\t--\\thg19\\tchr9\\t14923979\\t14925979\\tHuman\\tMCF-7\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,ChIP-seq\"\\tHere we identify super-enhancer translocations that drive overexpression of the oncogenic transcription factor MYB as a recurrent theme in adenoid cystic carcinoma (ACC). Whole-genome sequencing data and chromatin maps highlight distinct chromosomal rearrangements that juxtapose super-enhancers to the MYB locus. Chromosome conformation capture confirms that the translocated enhancers interact with the MYB promoter.\\tEnhancer\\tNFIB\\t3C\\t\"ChIP,ChIP-seq\"\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C). First, we examined an ACC with a translocation involving MYB and the NFIB locus. \"\\t\"CTF,HMGIC/NFIB,MACID,NF-I/B,NF1-B,NFI-B,NFI-RED2,NFIB3,NFIB\"\\tAdenoid Cystic Carcinoma\\tDOID:0080202\\tD003528\\tHere we identify the juxtaposition of super-enhancer regions to the MYB locus as the unifying feature of ACC translocations.Detailed genomic and epigenomic analyses of ACCs identify alternate rearrangements that translocate super-enhancers in the NFIB and\\t3C\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).These data suggest that the translocations reposition super-enhancers that subsequently loop to the MYB promoter and sustain high-level MYB expression.\"\\tTP63 \\t\"AIS, B(p51A), B(p51B), EEC3, KET, LMS, NBP, OFC8, RHS, SHFM4, TP53CP, TP53L, TP73L, p40, p51, p53CP, p63, p73H, p73L\"\\tChIP-seq\\t\"To directly test whether TP63 co-binds with MYB, we mapped binding of this transcription factor by ChIP-seq.Remarkably, we found that 81% of TP63-binding sites in ACC were co-bound by MYB.These data suggest that MYB,TP64 and Notch signaling may coordinately orchestrate the diverse expression programs in ACC.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t843137', 'E_01_370\\t26829750\\t--\\thg19\\tchr9\\t15066136\\t15068136\\tHuman\\tMCF-7\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,ChIP-seq\"\\tHere we identify super-enhancer translocations that drive overexpression of the oncogenic transcription factor MYB as a recurrent theme in adenoid cystic carcinoma (ACC). Whole-genome sequencing data and chromatin maps highlight distinct chromosomal rearrangements that juxtapose super-enhancers to the MYB locus. Chromosome conformation capture confirms that the translocated enhancers interact with the MYB promoter.\\tEnhancer\\tNFIB\\t3C\\t\"ChIP,ChIP-seq\"\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C). First, we examined an ACC with a translocation involving MYB and the NFIB locus. \"\\t\"CTF,HMGIC/NFIB,MACID,NF-I/B,NF1-B,NFI-B,NFI-RED2,NFIB3,NFIB\"\\tAdenoid Cystic Carcinoma\\tDOID:0080202\\tD003528\\tHere we identify the juxtaposition of super-enhancer regions to the MYB locus as the unifying feature of ACC translocations.Detailed genomic and epigenomic analyses of ACCs identify alternate rearrangements that translocate super-enhancers in the NFIB and\\t3C\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).These data suggest that the translocations reposition super-enhancers that subsequently loop to the MYB promoter and sustain high-level MYB expression.\"\\tTP63 \\t\"AIS, B(p51A), B(p51B), EEC3, KET, LMS, NBP, OFC8, RHS, SHFM4, TP53CP, TP53L, TP73L, p40, p51, p53CP, p63, p73H, p73L\"\\tChIP-seq\\t\"To directly test whether TP63 co-binds with MYB, we mapped binding of this transcription factor by ChIP-seq.Remarkably, we found that 81% of TP63-binding sites in ACC were co-bound by MYB.These data suggest that MYB,TP65 and Notch signaling may coordinately orchestrate the diverse expression programs in ACC.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t985294', 'E_01_371\\t26829750\\t--\\thg19\\tchr1\\t92170642\\t92172642\\tHuman\\tMCF-7\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,ChIP-seq\"\\tHere we identify super-enhancer translocations that drive overexpression of the oncogenic transcription factor MYB as a recurrent theme in adenoid cystic carcinoma (ACC). Whole-genome sequencing data and chromatin maps highlight distinct chromosomal rearrangements that juxtapose super-enhancers to the MYB locus. Chromosome conformation capture confirms that the translocated enhancers interact with the MYB promoter.\\tEnhancer\\tTGFBR3\\t3C\\t\"ChIP,ChIP-seq\"\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).We also examined a second ACC with a MYB-TGFBR3 translocation. In this case, seven of the nine H3K27ac peaks tested interacted with the MYB promoter.\"\\t\"BGCAN, betaglycan\"\\tAdenoid Cystic Carcinoma\\tDOID:0080202\\tD003528\\tHere we identify the juxtaposition of super-enhancer regions to the MYB locus as the unifying feature of ACC translocations.Detailed genomic and epigenomic analyses of ACCs identify alternate rearrangements that translocate super-enhancers in the NFIB and\\t3C\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).These data suggest that the translocations reposition super-enhancers that subsequently loop to the MYB promoter and sustain high-level MYB expression.\"\\tTP63 \\t\"AIS, B(p51A), B(p51B), EEC3, KET, LMS, NBP, OFC8, RHS, SHFM4, TP53CP, TP53L, TP73L, p40, p51, p53CP, p63, p73H, p73L\"\\tChIP-seq\\t\"To directly test whether TP63 co-binds with MYB, we mapped binding of this transcription factor by ChIP-seq.Remarkably, we found that 81% of TP63-binding sites in ACC were co-bound by MYB.These data suggest that MYB,TP66 and Notch signaling may coordinately orchestrate the diverse expression programs in ACC.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t25742', 'E_01_372\\t26829750\\t--\\thg19\\tchr1\\t92254801\\t92256801\\tHuman\\tMCF-7\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,ChIP-seq\"\\tHere we identify super-enhancer translocations that drive overexpression of the oncogenic transcription factor MYB as a recurrent theme in adenoid cystic carcinoma (ACC). Whole-genome sequencing data and chromatin maps highlight distinct chromosomal rearrangements that juxtapose super-enhancers to the MYB locus. Chromosome conformation capture confirms that the translocated enhancers interact with the MYB promoter.\\tEnhancer\\tTGFBR3\\t3C\\t\"ChIP,ChIP-seq\"\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).We also examined a second ACC with a MYB-TGFBR3 translocation. In this case, seven of the nine H3K27ac peaks tested interacted with the MYB promoter.\"\\t\"BGCAN, betaglycan\"\\tAdenoid Cystic Carcinoma\\tDOID:0080202\\tD003528\\tHere we identify the juxtaposition of super-enhancer regions to the MYB locus as the unifying feature of ACC translocations.Detailed genomic and epigenomic analyses of ACCs identify alternate rearrangements that translocate super-enhancers in the NFIB and\\t3C\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).These data suggest that the translocations reposition super-enhancers that subsequently loop to the MYB promoter and sustain high-level MYB expression.\"\\tTP63 \\t\"AIS, B(p51A), B(p51B), EEC3, KET, LMS, NBP, OFC8, RHS, SHFM4, TP53CP, TP53L, TP73L, p40, p51, p53CP, p63, p73H, p73L\"\\tChIP-seq\\t\"To directly test whether TP63 co-binds with MYB, we mapped binding of this transcription factor by ChIP-seq.Remarkably, we found that 81% of TP63-binding sites in ACC were co-bound by MYB.These data suggest that MYB,TP67 and Notch signaling may coordinately orchestrate the diverse expression programs in ACC.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t109901', 'E_01_373\\t26829750\\t--\\thg19\\tchr1\\t92331533\\t92333533\\tHuman\\tMCF-7\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,ChIP-seq\"\\tHere we identify super-enhancer translocations that drive overexpression of the oncogenic transcription factor MYB as a recurrent theme in adenoid cystic carcinoma (ACC). Whole-genome sequencing data and chromatin maps highlight distinct chromosomal rearrangements that juxtapose super-enhancers to the MYB locus. Chromosome conformation capture confirms that the translocated enhancers interact with the MYB promoter.\\tEnhancer\\tTGFBR3\\t3C\\t\"ChIP,ChIP-seq\"\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).We also examined a second ACC with a MYB-TGFBR3 translocation. In this case, seven of the nine H3K27ac peaks tested interacted with the MYB promoter.\"\\t\"BGCAN, betaglycan\"\\tAdenoid Cystic Carcinoma\\tDOID:0080202\\tD003528\\tHere we identify the juxtaposition of super-enhancer regions to the MYB locus as the unifying feature of ACC translocations.Detailed genomic and epigenomic analyses of ACCs identify alternate rearrangements that translocate super-enhancers in the NFIB and\\t3C\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).These data suggest that the translocations reposition super-enhancers that subsequently loop to the MYB promoter and sustain high-level MYB expression.\"\\tTP63 \\t\"AIS, B(p51A), B(p51B), EEC3, KET, LMS, NBP, OFC8, RHS, SHFM4, TP53CP, TP53L, TP73L, p40, p51, p53CP, p63, p73H, p73L\"\\tChIP-seq\\t\"To directly test whether TP63 co-binds with MYB, we mapped binding of this transcription factor by ChIP-seq.Remarkably, we found that 81% of TP63-binding sites in ACC were co-bound by MYB.These data suggest that MYB,TP68 and Notch signaling may coordinately orchestrate the diverse expression programs in ACC.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t186633', 'E_01_374\\t26829750\\t--\\thg19\\tchr1\\t92359751\\t92361751\\tHuman\\tMCF-7\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,ChIP-seq\"\\tHere we identify super-enhancer translocations that drive overexpression of the oncogenic transcription factor MYB as a recurrent theme in adenoid cystic carcinoma (ACC). Whole-genome sequencing data and chromatin maps highlight distinct chromosomal rearrangements that juxtapose super-enhancers to the MYB locus. Chromosome conformation capture confirms that the translocated enhancers interact with the MYB promoter.\\tEnhancer\\tTGFBR3\\t3C\\t\"ChIP,ChIP-seq\"\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).We also examined a second ACC with a MYB-TGFBR3 translocation. In this case, seven of the nine H3K27ac peaks tested interacted with the MYB promoter.\"\\t\"BGCAN, betaglycan\"\\tAdenoid Cystic Carcinoma\\tDOID:0080202\\tD003528\\tHere we identify the juxtaposition of super-enhancer regions to the MYB locus as the unifying feature of ACC translocations.Detailed genomic and epigenomic analyses of ACCs identify alternate rearrangements that translocate super-enhancers in the NFIB and\\t3C\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).These data suggest that the translocations reposition super-enhancers that subsequently loop to the MYB promoter and sustain high-level MYB expression.\"\\tTP63 \\t\"AIS, B(p51A), B(p51B), EEC3, KET, LMS, NBP, OFC8, RHS, SHFM4, TP53CP, TP53L, TP73L, p40, p51, p53CP, p63, p73H, p73L\"\\tChIP-seq\\t\"To directly test whether TP63 co-binds with MYB, we mapped binding of this transcription factor by ChIP-seq.Remarkably, we found that 81% of TP63-binding sites in ACC were co-bound by MYB.These data suggest that MYB,TP69 and Notch signaling may coordinately orchestrate the diverse expression programs in ACC.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t214851', 'E_01_375\\t26829750\\t--\\thg19\\tchr1\\t92392919\\t92394919\\tHuman\\tMCF-7\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,ChIP-seq\"\\tHere we identify super-enhancer translocations that drive overexpression of the oncogenic transcription factor MYB as a recurrent theme in adenoid cystic carcinoma (ACC). Whole-genome sequencing data and chromatin maps highlight distinct chromosomal rearrangements that juxtapose super-enhancers to the MYB locus. Chromosome conformation capture confirms that the translocated enhancers interact with the MYB promoter.\\tEnhancer\\tTGFBR3\\t3C\\t\"ChIP,ChIP-seq\"\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).We also examined a second ACC with a MYB-TGFBR3 translocation. In this case, seven of the nine H3K27ac peaks tested interacted with the MYB promoter.\"\\t\"BGCAN, betaglycan\"\\tAdenoid Cystic Carcinoma\\tDOID:0080202\\tD003528\\tHere we identify the juxtaposition of super-enhancer regions to the MYB locus as the unifying feature of ACC translocations.Detailed genomic and epigenomic analyses of ACCs identify alternate rearrangements that translocate super-enhancers in the NFIB and\\t3C\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).These data suggest that the translocations reposition super-enhancers that subsequently loop to the MYB promoter and sustain high-level MYB expression.\"\\tTP63 \\t\"AIS, B(p51A), B(p51B), EEC3, KET, LMS, NBP, OFC8, RHS, SHFM4, TP53CP, TP53L, TP73L, p40, p51, p53CP, p63, p73H, p73L\"\\tChIP-seq\\t\"To directly test whether TP63 co-binds with MYB, we mapped binding of this transcription factor by ChIP-seq.Remarkably, we found that 81% of TP63-binding sites in ACC were co-bound by MYB.These data suggest that MYB,TP70 and Notch signaling may coordinately orchestrate the diverse expression programs in ACC.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t248019', 'E_01_376\\t26829750\\t--\\thg19\\tchr1\\t92567177\\t92569177\\tHuman\\tMCF-7\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-PCR,ChIP,ChIP-seq\"\\tHere we identify super-enhancer translocations that drive overexpression of the oncogenic transcription factor MYB as a recurrent theme in adenoid cystic carcinoma (ACC). Whole-genome sequencing data and chromatin maps highlight distinct chromosomal rearrangements that juxtapose super-enhancers to the MYB locus. Chromosome conformation capture confirms that the translocated enhancers interact with the MYB promoter.\\tEnhancer\\tTGFBR3\\t3C\\t\"ChIP,ChIP-seq\"\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).We also examined a second ACC with a MYB-TGFBR3 translocation. In this case, seven of the nine H3K27ac peaks tested interacted with the MYB promoter.\"\\t\"BGCAN, betaglycan\"\\tAdenoid Cystic Carcinoma\\tDOID:0080202\\tD003528\\tHere we identify the juxtaposition of super-enhancer regions to the MYB locus as the unifying feature of ACC translocations.Detailed genomic and epigenomic analyses of ACCs identify alternate rearrangements that translocate super-enhancers in the NFIB and\\t3C\\t\"To test whether specific enhancers within the translocated super_x0002_enhancers might activate the MYB promoter, we examined their physical proximity to the promoter using chromosome conforma_x0002_tion capture (3C).These data suggest that the translocations reposition super-enhancers that subsequently loop to the MYB promoter and sustain high-level MYB expression.\"\\tTP63 \\t\"AIS, B(p51A), B(p51B), EEC3, KET, LMS, NBP, OFC8, RHS, SHFM4, TP53CP, TP53L, TP73L, p40, p51, p53CP, p63, p73H, p73L\"\\tChIP-seq\\t\"To directly test whether TP63 co-binds with MYB, we mapped binding of this transcription factor by ChIP-seq.Remarkably, we found that 81% of TP63-binding sites in ACC were co-bound by MYB.These data suggest that MYB,TP71 and Notch signaling may coordinately orchestrate the diverse expression programs in ACC.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t422277', 'E_01_377\\t26886256\\tES-specific enhancers (ESSEs)\\thg19\\tchr6\\t43891400\\t43898360\\tHuman\\tK-562\\tLow+High throughput\\tChIP-seq\\t\"We termed these promoter-distal elements, which carry the H3K4me1 mark in ES but not in hematopoietic cells,the ES-specific enhancers (ESSEs) of the hematopoietic cell lineage.\"\\tenhancer\\tVEGFA \\t\"4C-seq,CRISPR/Cas9\"\\tLuciferase Reporter Assay\\t\"To determine whether this ESSE is able to enhance transcription, a 1,500 bp fragment surrounding the ESSE methylation site was cloned upstream to a minimal SV40 promoter in a luciferase reporter plasmid and transfected to K562 cells.Indeed, the ESSE segment significantly enhanced expression when placed upstream or downstream to the minimal promoter. Thus,the DNA element surrounding the ESSE site is a transcriptional enhancer in leukemia cells.The Circularized Chromosome Conformation Capture (4C-seqencing) assay allowed mapping the interactions between the VEGFA promoter and DNA sequences across the K562 cell genome.To determine whether VEGFA expression depends upon the ESSE, we mutated the ESSE site in K562 cells by utilization of the CRISPR/Cas9 genome-editing system.\"\\t\"MVCD1, VEGF, VPF\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSTAT1/2\\t\"CANDF7,IMD31A,IMD31B,IMD31C,ISGF-3,STAT91,IMD44,ISGF-3,P113,STAT113\"\\tChIP-seq\\tFocused (<600 bp) cluster of transcription factor binding sites (Chip-seq data) within the predicted enhancer. The factors bound in K562 cells are listed. This is the only cluster within 20 kb around the ESSE.--\\t--\\t--\\t>2kb\\t\\t\\t\\t156934', 'E_01_378\\t26916345\\tPRE\\thg19\\tchr1\\t67739940\\t67741075\\tHuman\\tLymphoblastoid Cell Lines\\tlow throughput\\tDNase I Hypersensitivity Assay\\t\"We identified a PRE (Chr1:67739940C67741075) located 14.2 kb downstream of IL23R and 33.1 kb upstream of IL12RB2.This region is DNase I hypersensitive (in Th1-cells, but not Th17-cells) and exhibits TF binding (lymphoblastoid cell line; GM12878) and enhancer-associated H3K4me1 methylation (CD4+ CD25? IL-17A? T-cells; PMA and ionomycin-stimulated).\"\\tenhancer\\tIL23R\\t--\\tDNase I Hypersensitivity Assay\\t\"We identified a PRE (Chr1:67739940C67741075) located 14.2 kb downstream of IL23R and 33.1 kb upstream of IL12RB2.This region is DNase I hypersensitive (in Th1-cells, but not Th17-cells) and exhibits TF binding (lymphoblastoid cell line; GM12878) and enhancer-associated H3K4me1 methylation (CD4+ CD25? IL-17A? T-cells; PMA and ionomycin-stimulated).\"\\tIL23R\\tAnkylosing Spondylitis\\tDOID:7147\\tD013167\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\trs11209026\\t67705958\\t\"DNase I Hypersensitivity Assay,Luciferase Reporter Assay\"\\t>2kb\\t\\t\\t\\t135918', 'E_01_379\\t27149122\\t\\thg19\\tchr5\\t110175800\\t110176600\\tHuman\\t293T\\tlow throughput\\tLuciferase Reporter Assa\"To experimentally test the effect of rs9885413 on enhancer activity, the 100 bp region flanking the SNP (50 bp on either side) was cloned into a reporter vector and transfected into HEK293 cells expressing NHLH1 (S1 Text). Luciferase activity measured after 24 hours was 4-fold higher with a construct corresponding to the risk allele as compared to the wild-type allele (S4 Fig, P < 0.001), indicating that the risk allele of rs9885413 substantially increases enhancer activity.\"\\tenhancer\\tTSLP\\t--\\tLuciferase Reporter Assay\\t\"To experimentally test the effect of rs9885413 on enhancer activity, the 100 bp region flanking the SNP (50 bp on either side) was cloned into a reporter vector and transfected into HEK293 cells expressing NHLH1 (S1 Text). Luciferase activity measured after 24 hours was 4-fold higher with a construct corresponding to the risk allele as compared to the wild-type allele (S4 Fig, P < 0.001), indicating that the risk allele of rs9885413 substantially increases enhancer activity.\"\\tTSLP\\tHeart Failure\\t--\\tD006333\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\trs9885413\\t110176128\\t\"Transfection,Luciferase Reporter Assay\"\\t>2kb\\t\\t\\t\\t229578', 'E_01_380\\t26581162\\t--\\thg19\\tchr3\\t108015219\\t108015290\\tHuman\\t293T\\tLow throughput\\t\"Luciferase Reporter Assay,Transient Transfection Assay,RNA-seq,PCR\"\\t\"In this study, we identify a targeted hotspot for CRISPR acti_x0002_vation within the long terminal repeat (LTR) enhancer region,at the junction between two nuclear factor (NF)-B transcrip_x0002_tion factorCbinding sites.Twenty-three possible NGG Streptococcus pyogenes(Sp) Cas9 photospacer adjacent motif (PAM) sites for targeting of sgRNAs were identified in the U3 region of the LTR, upstream of the HIV transcriptional start site(TSS) (?450 to 0 bp; HIV genome HXB2) within the enhancer and modulatory region of the promoter. Enhancer\\tHHLA2\\tCRISPR/Cas9\\tLuciferase Reporter Assay\\tAll sgRNAs were named according to the 3 adjacent sense strand nucleotide cleavage site catalyzed by nuclease-active Cas9 (Figure? 1a) and were screened together with dCas9-VP64 or dCas9-VP160 for their activation properties by transient transfection with the reporter NL4-3.Luc.R_x005f_x0002_E-a full-length HIV molec_x0002_ular clone where luciferase is driven by the viral LTR.\\t\"B7-H5,B7-H7,B7H7,B7y\"\\t--\\t--\\t--\\t--\\t--\\t--\\tNFKB1\\t\"CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\tCRISPR/Cas9\\t\"Since sg362F fully occupies the sequence across both NF-BCbinding sites, we next determined if CRISPR activation via the sg362F target site was coupled to NF-B function. these data strongly support a conclusion where sg362F/dCas9-VP64 functions independently of NF-B as a stand-alone transactivator of LTR activation. \"\\t--\\t--\\t--\\t<2KB\\t\\t\\t98', 'E_01_381\\t28905448\\t--\\thg19\\tchr11\\t109985003\\t109985222\\tHuman\\tOsteosarcoma Stem Cells\\tLow throughput\\t\"ChIP-PCR,Luciferase Reporter Assay\"\\t\"Interestingly, mutation of the Sox2 binding site at mt 11, whose sequence is not con\\\\', 'E_01_382\\t29348663\\tBY707159.1 Enhancer\\thg19\\tchr7\\t150700000\\t150705000\\tHuman\\tEndothelial Cells (ECs)\\tLow+High throughput\\t\"ChIP-seq,qPCR,4C-seq,Hi-C\"\\t\"Combining RNA-sequencing and chromatin conformation capture methods, we demonstrate that LEENE is co-regulated with eNOS and that its enhancer resides in proximity to eNOS promoter in endothelial cells (ECs).\"\\tEnhancer\\tNOS3\\tCRISPR/Cas9\\t\"ChIP-seq,qPCR,ChIP-qPCR\"\\t\"Consistent with the Hi-C data, 4C-seq also revealed the chromosomal proximity between the LEENE enhancer and eNOS promoter and this interaction is substantially', 'E_01_383\\t26745862\\t\\thg19\\tchr6\\t36640937\\t36642937\\tHuman\\t\"MCF-7,A549,HeLa,293T\"\\tLow throughput\\t\"ChIP,qPCR\"\\t\"Our ChIP experiments showed that prior to activation p53 was exclusively found at the upstream enhancer that harbors a high-affinity, near-consen_x0002_sus binding site for p53 (Fig 5A). Occupancy of this site increased incrementally after 1.5 and 4 hours of CDK9 inhibition (Fig 5B). In contrast to this pre-loaded \"\"primary\"\" enhancer, treat_x0002_ment with 067 was followed by de novo recruitment of p53 to additional, secondary binding sites, most notably to an intronic region that maps to position +4 kb downstream of the p21 TSS and contains another near-consensus p53 motif (Fig 5A). After four hours of treatment,p53 occupancy of this site reached similar levels as found at the upstream enhancer (Fig 5B).Enhancer\\tCDKN1A\\t--\\tChIP-qPCR\\t\"Our ChIP experiments showed that prior to activation p53 was exclusively found at the upstream enhancer that harbors a high-affinity, near-consen_x0002_sus binding site for p53 (Fig 5A). Occupancy of this site increased incrementally after 1.5 and 4 hours of CDK9 inhibition (Fig 5B). In contrast to this pre-loaded \"\"primary\"\" enhancer, treat_x0002_ment with 067 was followed by de novo recruitment of p53 to additional, secondary binding sites, most notably to an intronic region that maps to position +4 kb downstream of the p21 TSS and contains another near-consensus p53 motif (Fig 5A). After four hours of treatment,p53 occupancy of this site reached similar levels as found at the upstream enhancer (Fig 5B).\"\\t\"CAP20, CDKN1, CIP1, MDA-6,?P21, SDI1, WAF1, p21CIP1\"\\tTumour\\t--\\t--\\t--\\t--\\t--\\tTP53\\t\"BCC7, BMFS5, LFS1,?P53, TRP53\"\\tqPCR\\t\"MCF7 cells, which harbour functional wild-type p53, were treated for 20 minutes to 4 hours with 10 M 067 and both nascent unspliced precursor (pre) and mature spliced mRNA was analyzed by quantitative PCR (qPCR). Inhibitor treatment led to significant(2.5-fold) repression of nascent RNA synthesis of the p53 target gene p21 within 20 minutes (Fig1A).\"\\t--\\t--\\t--\\tIntron\\t2299', 'E_01_384\\t26745862\\t\\thg19\\tchr6\\t36647237\\t36649237\\tHuman\\t\"MCF-7,A549,HeLa,293T\"\\tLow throughput\\t\"ChIP,qPCR\"\\t\"Our ChIP experiments showed that prior to activation p53 was exclusively found at the upstream enhancer that harbors a high-affinity, near-consen_x0002_sus binding site for p53 (Fig 5A). Occupancy of this site increased incrementally after 1.5 and 4 hours of CDK9 inhibition (Fig 5B). In contrast to this pre-loaded \"\"primary\"\" enhancer, treat_x0002_ment with 067 was followed by de novo recruitment of p53 to additional, secondary binding sites, most notably to an intronic region that maps to position +4 kb downstream of the p21 TSS and contains another near-consensus p53 motif (Fig 5A). After four hours of treatment,p53 occupancy of this site reached similar levels as found at the upstream enhancer (Fig 5B).Enhancer\\tCDKN1A\\t--\\tChIP-qPCR\\t\"Our ChIP experiments showed that prior to activation p53 was exclusively found at the upstream enhancer that harbors a high-affinity, near-consen_x0002_sus binding site for p53 (Fig 5A). Occupancy of this site increased incrementally after 1.5 and 4 hours of CDK9 inhibition (Fig 5B). In contrast to this pre-loaded \"\"primary\"\" enhancer, treat_x0002_ment with 067 was followed by de novo recruitment of p53 to additional, secondary binding sites, most notably to an intronic region that maps to position +4 kb downstream of the p21 TSS and contains another near-consensus p53 motif (Fig 5A). After four hours of treatment,p53 occupancy of this site reached similar levels as found at the upstream enhancer (Fig 5B).\"\\t\"CAP20, CDKN1, CIP1, MDA-6,?P21, SDI1, WAF1, p21CIP1\"\\tTumour\\t--\\t--\\t--\\t--\\t--\\tTP53\\t\"BCC7, BMFS5, LFS1,?P53, TRP53\"\\tqPCR\\t\"MCF7 cells, which harbour functional wild-type p53, were treated for 20 minutes to 4 hours with 10 M 067 and both nascent unspliced precursor (pre) and mature spliced mRNA was analyzed by quantitative PCR (qPCR). Inhibitor treatment led to significant(2.5-fold) repression of nascent RNA synthesis of the p53 target gene p21 within 20 minutes (Fig1A).\"\\t--\\t--\\t--\\tIntron\\t4001', 'E_01_385\\t2116990\\t\\thg19\\tchr11\\t5288664\\t5288684\\tHuman\\t\"K-562,HeLa\"\\tlow throughput\\t\"Transient Assay,DNaseI-seq,Luciferase Reporter Assay\"\\t\"When the same constructs were transfected into HeLa cells, the HS II enhancer had modest activity (relative activity: no HS II, 1.0 + 0.1;HS II (1455 bp), 2.8 + 0.8; HS II (46 bp), 3.9 + 0.1; HS IIenh-, 0.7 _ 0.1. Consistent with the marked increase in luciferase activity following induction, correctly initiated luciferase transcripts were seen in the uninduced state only after long exposures. These data suggest that the HS II enhancer element may have a role in the increase in globin gene expression that occurs during erythroid maturation.\"\\tenhancerHBE1\\t--\\tLuciferase Reporter Assay\\t\"The enhancer consists of tandem AP-l-binding sites, phased 10 bp apart, which are both', 'E_01_386\\t2339058\\t\\thg19\\tchr11\\t5268575\\t5268621\\tHuman\\tK-562\\tlow throughput\\tDNase I Hypersensitivity Assay\\tA 46 bp fragment was made that contained this sequence (46bpE) and a 1.5 kb fragment was made (HS II-E) that was identical to the wildtype HS II fragment except that a portion of the minimal enhancer was replaced by a plasmid polylinker ablating both API sites (fig.7). This data clearly shows that the 46 bp minimal enhancer is both necessary and sufficient for gamma-globin gene induction with hemin induced erythroid maturation in K562 cells.\\tenhancer\\tHBG1\\t--\\t\"DNase I Hypersensitivity Assay,Southern blot,PCR\"\\t\"To determine the effect of fragments containing LAR sequences on globin gene expression, mRNA from a marked gamma_x0002_globin gene linked to LAR fragments was assayed in stably transfected K562 erythroleukemia cells. DNasel hypersensitive site 11 (HS 11), located 10.9 kb upstream', 'E_01_387\\t10523648\\t--\\thg19\\tchr11\\t5266021\\t5266708\\tHuman\\tK-562\\tLow throughput\\t\"Transfection,RNase Protection Assay,DNase I Hypersensitivity Assay\"\\t\"We investigated the requirements for enhancer-promoter communication by using the human b-globin locus control region (LCR) DNase I-hypersensitive site 2 (HS2) enhancer and the ?epsilon-globin gene in chromatinized minichromosomes in erythroid cells.Activation of globin genes during development is accompanied by local_x0002_ized alterations of chromatin structure, and CACCC binding factors and GATA-1, which interact with both globin promoters and the LCR, are believed to be critical for globin gene transcription activation.\"\\tEnhancerHBE1\\t--\\tDNase I Hypersensitivity Assay\\t\"We found that an HS2 element mutated in its GATA motif failed to remodel the ?-globin promoter or activate tran_x0002_scription yet HS2 nuclease accessibility did not change. Accessibility and transcription were reduced at promoters with mutated GATA-1 or CACCC sites. Strikingly, these mutations also resulted in reduced accessibility at HS2.\"\\tHBE\\t--\\t--\\t--\\t\"Further, at least in this instance, transcription activation and promoter remodeling by a distant enhancer are not separable.\"\\tDNase I Hypersensitivity Assay\\t\"To determine whether the HS2 mutations affected transcription, -globin RNA levels were measured by RNase protection as illustrated in Fig. 4A. The minichromosomal -globin gene is not transcribed in the absence of an enhancer, while inclusion of HS2 results in transcription of the linked gene.The results suggest that in chromatin,multiple interactions contribute to enhancer-promoter communication.\"\\t\"GATA1,NFE2\"\\t\"ERYF1,GATA-1,GF-1,GF1,NF-E1,NFE1,XLANP,XLTDA,XLTT,NF-E2,p45\"\\tDNase I Hypersensitivity Assay\\t\"Because of the reduced DNase I sensitivity of HS2 in the absence of a globin gene, cleavage at sites close to the HS2 GATA-1 and NF-E2 binding motifs could be visualized at the', 'E_01_388\\t10688646\\t--\\thg19\\tchr16\\t80617\\t80837\\tHuman\\tK-562\\tLow throughput\\t\"ChIP,PCR,Transfection\"\\t\"In this study,we have further investigated the molecular basis of this model. First, human erythroid K562 cells stablyintegrated with various HS-40 mutants cis linked to a human a-globin promoter-growth hormone hybrid gene were analyzed by genomic footprinting and expression analysis. By the assay, we demonstrate that factors bound at different motifs of HS-40 indeed act in concert to build a fully functional enhanceosome.Thus,modification of factor binding at a single motif could drastically change the configuration and function of the HS-40 enhanceosome.\"\\tEnhancer \\tGATA1\\t--\\t\"ChIP,PCR\"\\t\"Substitution of 3 bp in the factor-binding cores of 59-NA and GATA-1(c) motifs, as well as the 1-bp mutation of 39-NA, had no apparent effects on the presence of the HS-40 genomic footprints (Fig. 2C, E, and G, respectively).On the other hand, 3-bp mutation of either 39-NA or the GT motif greatly affected the genomic footprints. In particular, the HS-40 enhancer carrying the GT mutation becomes empty(Fig. 2F), while factor binding at both the GT and GATA-1(c)motifs of HS-40 with the 39-NA(I) mutation was abolished(Fig. 2D).\"\\t\"ERYF1,GATA-1,GF-1,GF1,NF-E1,NFE1,XLANP,XLTDA,XLTT\"\\t--\\t--\\t--\\t\"The HS-40 enhancer is the major cis-acting regulatory element responsible for the developmental stage- and erythroid lineage-specific expression of the human a-like globin genes, the embryonic z and the adult a2/a/1.\"\\t\"Footprinting,ChIP\"\\t\"A model has been proposed in which competitive factor binding at one of the HS-40 motifs, 3-NA, modulates the capability of HS-40 to activate the embryonic -globin promoter. Furthermore, this modulation was thought to be mediated through configurational changes of the HS-40 enhanceosome during development.\"\\tNFE2\\t\"NF-E2, p45\"\\t\"ChIP,EMSA\"\\t\"Finally, we show by chromatin immunoprecipitation experiments that only binding of NF-E2, not AP1,could be detected in vivo in K562 cells around the HS-40 region. These data exclude a role for AP1 in the developmental regulation of the human a-globin locus via the 3*-NA motif of HS-40 in embryonic/fetal erythroid cells. \"\\t--\\t--\\t--\\t>2kb\\t\\t\\t\\t48564255', 'E_01_389\\t2304460\\t--\\thg19\\tchr11\\t5248824\\t5249485\\tHuman\\tEmbryonic Blood Cells\\tLow throughput\\tDNase I Hypersensitivity Assay\\t\"We found that sequences between -201 and -136 are essential for expression of the G-globin gene,whereas those upstream of -201 have little effect on the level or tissue or stage specificity of G-globin expression. The G-globin upstream sequences from -201 to -136 were, furthermore, capable of activating a linked -globin gene in embryonic blood cells; however, a G-globin fragment from -383 to -206 was similarly active in this assay, and the complete fragment from -383 to -136 was considerably more active than either of the smaller fragments, suggesting the presence of multiple cis-acting elements for embryonic blood cells. Our data also suggested the possibility of a negative regulatory element between -201 and -136. \"\\tEnhancer \\t\"HBG1,HBB\"\\t--\\tDNase I Hypersensitivity Assay\\t\"Finally,we observed that removal of the ,-globin3\\'-flanking sequences,including the 3\\' enhancer,from the G-globin upstream--globin hybrid gene resulted in a 25-fold reduction', 'E_01_390\\t10891470\\t\\thg19\\tchr11\\t5255062\\t5258441\\tHuman\\tK-562\\tLow throughput \\t\"Luciferase Reporter Assay,ChIP,PCR,Southern blot\"\\t\"To determine whether NF-E2 interacts directly with HS2 of the endogenous -globin LCR , we used a ChIP assay.The immunoprecipi_x0002_tated DNA was analyzed by PCR with primers specific for HS2, the Ag-globin promoter, and the HS12 enhancer element of the IgH locus.These', 'E_01_391\\t9878258\\t\\thg19\\tchr3\\t108296550\\t108297750\\tHuman\\tK-562\\tLow throughput \\t\"CAT Assay,PCR,RT-PCR\"\\t\"Functional tests with the CAT reporter gene assays demonstrate that the human 59 HS5 LTR activates the cis-linked CAT gene and possesses enhancer and pro_x0002_moter activities in erythroid cells.We found an LTR retrotranspo_x0002_son belonging to the ERV-9 family of human endoge_x0002_nous retroviruses in the apparent 5* boundary area of the LCR. This ERV-9 LTR contains an unusual U3 en_x0002_hancer region composed of 14 tandem repeats with recurrent GATA, CACCC, and CCAAT motifs.\"\\tEnhancer\\tHBB\\t--\\t\"CAT Assay,RT-PCR\"\\tWe next determined whether the endogenous 59 HS5 LTR also exhibits enhancer and promoter activities and can activate the transcription of the downstream R region and the flanking genomic DNA in the b-globin LCR.The above RT-PCR results indicate that the endog_x0002_enous 59 HS5 LTR possesses apparent enhancer and promoter activities and is capable of promoting the transcription of the R and U5 regions in the LTR and of further downstream genomic DNA in the LCR.\\t\"CD113t-C,ECYT6,beta-globin\"\\t--\\t--\\t--\\t\"In both recombinant constructs and the endog_x0002_enous human genome, the LTR enhancer and promoter activate the transcription of cis-linked DNA preferentially in erythroid cells.\"\\tCAT Assay\\tFunctional tests with the CAT reporter gene assays demonstrate that the human 59 HS5 LTR activates the cis-linked CAT gene and possesses enhancer and pro_x0002_moter activities in erythroid cells.\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2kb\\t\\t\\t\\t103050454', 'E_01_392\\t10595394\\t\\thg19\\tchr11\\t5250264\\t5250295\\tHuman\\t\"K-562,HEL\"\\tLow throughput \\t\"Luciferase Reporter Assay,EMSA\"\"DNA footprinting analyses and electrophoretic mobility shift assays (EMSAs) indicate that two sites for which PE has a high affinity, 3\\' PE1 and 3\\' PE2, span from +1992 to + 2023 and +2263 to +2288, respectively, and contain the consensus sequence ATTANNNGGAANNCT(N)NNNNTAATGG. Three low-affinity sites,3\\' PE1 (+2245 to +2225),3\\' PE3 (+2432 to +2447), and 3\\' PE4 (+2479 to +2505),contain the consensus sequences 59 -AAAAN(T/A)A(A/T)TT-3\\'. \"\\tEnhancer\\tHBG1\\t--\\tLuciferase Reporter Assay\\t\"A large nuclear protein complex, termed PE (for -globin promoter and enhancer binding factor), binds to five sites located 5\\' and 3\\' of the human -globin gene. Transient cotransfections of a HOXB2 cDNA and a c -globinCluciferase reporter gene con_x0002_struct into cells expressing SATB1 suggested that SATB1 has a positive and HOXB2 a negative regulatory', 'E_01_393\\t10595394\\t\\thg19\\tchr11\\t5250535\\t5250560\\tHuman\\t\"K-562,HEL\"\\tLow throughput \\t\"Luciferase Reporter Assay,EMSA\"\"DNA footprinting analyses and electrophoretic mobility shift assays (EMSAs) indicate that two sites for which g PE has a high affinity, 3\\' PE1 and 3\\' PE2, span from +1992 to + 2023 and +2263 to +2288, respectively, and contain the consensus sequence ATTANNNGGAANNCT(N)NNNNTAATGG. Three low-affinity sites,3\\' PE1 (+2245 to +2225),3\\' PE3 (+2432 to +2447), and 3\\' PE4 (+2479 to +2505),contain the consensus sequences 59 -AAAAN(T/A)A(A/T)TT-3\\'. \"\\tEnhancer\\tHBG1\\t--\\tLuciferase Reporter Assay\\t\"A large nuclear protein complex, termed PE (for -globin promoter and enhancer binding factor), binds to five sites located 5\\' and 3\\' of the human -globin gene. Transient cotransfections of a HOXB2 cDNA and a c -globinCluciferase reporter gene con_x0002_struct into cells expressing SATB1 suggested that SATB1 has a positive and HOXB2 a negative regulatory', 'E_01_394\\t10595394\\t\\thg19\\tchr11\\t5248027\\t5248047\\tHuman\\t\"K-562,HEL\"\\tLow throughput \\t\"Luciferase Reporter Assay,EMSA\"\"DNA footprinting analyses and electrophoretic mobility shift assays (EMSAs) indicate that two sites for which g PE has a high affinity, 3\\' PE1 and 3\\' PE2, span from +1992 to + 2023 and +2263 to +2288, respectively, and contain the consensus sequence ATTANNNGGAANNCT(N)NNNNTAATGG. Three low-affinity sites,3\\' PE1 (+2245 to +2225),3\\' PE3 (+2432 to +2447), and 3\\' PE4 (+2479 to +2505),contain the consensus sequences 59 -AAAAN(T/A)A(A/T)TT-3\\'. \"\\tEnhancer\\tHBG1\\t--\\tLuciferase Reporter Assay\\t\"A large nuclear protein complex, termed PE (for -globin promoter and enhancer binding factor), binds to five sites located 5\\' and 3\\' of the human -globin gene. Transient cotransfections of a HOXB2 cDNA and a c -globinCluciferase reporter gene con_x0002_struct into cells expressing SATB1 suggested that SATB1 has a positive and HOXB2 a negative regulatory', 'E_01_395\\t10595394\\t\\thg19\\tchr11\\t5250704\\t5250719\\tHuman\\t\"K-562,HEL\"\\tLow throughput \\t\"Luciferase Reporter Assay,EMSA\"\"DNA footprinting analyses and electrophoretic mobility shift assays (EMSAs) indicate that two sites for which g PE has a high affinity, 3\\' PE1 and 3\\' PE2, span from +1992 to + 2023 and +2263 to +2288, respectively, and contain the consensus sequence ATTANNNGGAANNCT(N)NNNNTAATGG. Three low-affinity sites,3\\' PE1 (+2245 to +2225),3\\' PE3 (+2432 to +2447), and 3\\' PE4 (+2479 to +2505),contain the consensus sequences 59 -AAAAN(T/A)A(A/T)TT-3\\'. \"\\tEnhancer\\tHBG1\\t--\\tLuciferase Reporter Assay\\t\"A large nuclear protein complex, termed PE (for -globin promoter and enhancer binding factor), binds to five sites located 5\\' and 3\\' of the human -globin gene. Transient cotransfections of a HOXB2 cDNA and a c -globinCluciferase reporter gene con_x0002_struct into cells expressing SATB1 suggested that SATB1 has a positive and HOXB2 a negative regulatory', 'E_01_396\\t10595394\\t\\thg19\\tchr11\\t5250751\\t5250777\\tHuman\\t\"K-562,HEL\"\\tLow throughput \\t\"Luciferase Reporter Assay,EMSA\"\"DNA footprinting analyses and electrophoretic mobility shift assays (EMSAs) indicate that two sites for which g PE has a high affinity, 3\\' PE1 and 3\\' PE2, span from +1992 to + 2023 and +2263 to +2288, respectively, and contain the consensus sequence ATTANNNGGAANNCT(N)NNNNTAATGG. Three low-affinity sites,3\\' PE1 (+2245 to +2225),3\\' PE3 (+2432 to +2447), and 3\\' PE4 (+2479 to +2505),contain the consensus sequences 59 -AAAAN(T/A)A(A/T)TT-3\\'. \"\\tEnhancer\\tHBG1\\t--\\tLuciferase Reporter Assay\\t\"A large nuclear protein complex, termed PE (for -globin promoter and enhancer binding factor), binds to five sites located 5\\' and 3\\' of the human -globin gene.Transient cotransfections of a HOXB2 cDNA and a c -globinCluciferase reporter gene con_x0002_struct into cells expressing SATB1 suggested that SATB1 has a positive and HOXB2 a negative regulatory', 'E_01_397\\t12861010\\t\\thg19\\tchr11\\t5293686\\t5294250\\tHuman\\tK-562\\tLow throughput \\t\"PCR,DNaseI-seq\"\\t\"We have also found DNase I HSs at kb -85.5 (HS-85.5) and -84.5 in mouse, but there is no corresponding se_x0002_quence or structure in human for these HSs.By comparison,neither the active _x0002_maj-globin gene promoter nor a region located ~1 kb from HS5 shows significant enrichment. HS-85.5 exhibits a modest (twofold) enrichment.\"\\tEnhancer\\tLOC110006319\\t--\\t\"DNaseI-seq,Southern blot,PCR\"\\t\"In ery_x0002_throid cells, the active _x0002_-globin locus contains several DNase I HSs, which have been shown to map to sequences with regu_x0002_latory functionIn addition, histone hyperacetylation anddimethylation of histone H3 K4 are not uniform features of the nuclease-sensitive mouse _x0001_-globin domain but rather define distinct subdomains within it. Our results reveal a complex chromatin landscape for the active', 'E_01_398\\t17637499\\t\\thg19\\tchr11\\t5211019\\t5213983\\tHuman\\tK-562\\tLow throughput \\tLuciferase Reporter Assay\\t\"Enhancer activity of reporter constructs was determined with the dual luciferase system in transient transfection assays. Notch signaling transactivated a reporter construct harboring a con_x0002_served RBP-J (CBF1) binding site in the hypersensitive site 2 (HS2) of human b-globin.', 'E_01_399\\t17283048\\t\\thg19\\tchr11\\t5264845\\t5266345\\tHuman\\tK-562\\tLow throughput \\t\"FISH,Transfection\"\\t\"Minichromosomes, maintained at stable, moderate copy num_x0002_bers in K562 cells, contained the strong HS2 enhancer of the -globin LCR and the -globin gene in the form of a', 'E_01_400\\t15371553\\tHS2 enhancer\\thg19\\tchr11\\t5271421\\t5273421\\tHuman\\tK-562\\tLow throughput \\tRNase Protection Assay\\tThe minichromosomes were stably maintained in human erythroid K562 cells in which the endogenous e-globin gene is actively transcribed. RNase protection assays were performed to test the influence of the insulators on enhancer activation of the e-globin gene.\\tenhancer\\tHBE1\\t--\\tLuciferase Reporter Assay\\t\"The minichromosomes were stably maintained in human erythroid K562 cells in which the', 'E_01_401\\t14585970\\tHS2 enhancer\\thg19\\tchr11\\t5286080\\t5288080\\tHuman\\tK-562\\tLow throughput \\t\"DNase I Hypersensitivity Assay,RT-PCR,ChIP\"\\t\"To investigate the chromatin anatomy of a more complete enhancer-gene locus, we created minichromosomes linking -globin to a longer 1.46-kb KpnI-to-BglII LCR HS2 enhancer fragment. Formation of HS2 and the -globin promoter DNase I hypersensitive site are enhancer dependent.We compared the transcriptionally active locus to one in which HS2 was inactivated by mutations in the core NF-E2 sites. In contrast to inactive templates, nucleosomes were mobilized in discrete areas of the active locus, including the HS2 core and the proximal promoter.\"\\tenhancerHBE1\\t--\\tDNase I Hypersensitivity Assay\\t\"We studied nucleosome remodeling and covalent histone modification mediated by the-globin locus control region HS2 enhancer at nucleosome-level resolution throughout a 5.5-kb globin gene model locus in vivo in K562 cells.We compared the transcriptionally active locus to one in which HS2 was inactivated by mutations in the core NF-E2 sites. In contrast to inactive templates, nucleosomes were mobilized in discrete areas of the active locus, including the HS2 core and the proximal promoter. Large differences in restriction enzyme accessibility between the active and inactive templates were limited to the regions of nucleosome mobilization, which subsumed the DNase I hypersensitive sites.\"\\tHBE\\t--\\t--\\t--\\t--\\t--\\t--\\tNFE2\\t\"NF-E2, p45\"\\tRNase Protection Assay\\t\"To investigate the effect on transcription activation of the NF-E2 mutation in the context of an extended HS2, we performed RNase protection assays on RNA isolated from clones with the wild-type (HS2L) or mutated [HS2L(mut)] enhancer.A representative experiment is de_x0002_picted in Fig. 2A, and the results of multiple determinations for six different clones of HS2L and HS2L(mut) are summa_x0002_rized in Fig. 2B. Abundant transcription of -globin RNA is seen when the gene is linked to the longer 1.46-kb HS2 se_x0002_quence (HS2L). Destruction of the tandem NF-E2 binding sites in HS2 abolished transcription of the -globin gene [HS2L(mut)].\"\\t--\\t--\\t--\\t>2kb\\t\\t\\t\\t17578', 'E_01_402\\t27149122\\t--\\thg19\\tchr5\\t110175120\\t110177136\\tHuman\\tBlood\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"To experimentally test the effect of rs9885413 on enhancer activity, the 100 bp region flank_x0002_ing the SNP (50 bp on either side) was cloned into a reporter vector and transfected intoHEK293 cells expressing NHLH1 (S1 Text). Luciferase activity measured after 24 hours was', 'E_01_403\\t27207652\\t--\\thg19\\tchr11\\t95817801\\t95818824\\tHuman\\t\"MCF10CA1h,MCF10CA1a\"\\tLow+High throughput\\t\"ChIP,qPCR,Transfection,Western blot\"\\t\"The difference in DNA meth_x0002_ylation within MAML2 in RSV treated versus control cells was', 'E_01_404\\t26919345\\tPRE\\thg19\\tchr1\\t67739940\\t67741075\\tHuman\\t\"GM12878,Th17,Th1\"\\tLow throughput\\tDNase I Hypersensitivity Assay\\t\"We identified a PRE (Chr1:67739940C67741075) located 14.2 kb downstream of IL23R and 33.1 kb upstream of IL12RB2.This region is DNase I hypersensitive (in Th1--cells, but not Th17--cells) and exhibits TF binding (lymphoblastoid cell--line; GM12878) and enhancer--associated H3K4me1 methylation (CD4+ CD25? IL--17A? T--cells; PMA and ionomycin--stimulated). \"\\tenhancer\\tIL23R-IL12RB2\\t--\\tDNase I Hypersensitivity Assay\\t\"We identified a PRE (Chr1:67739940C67741075) located 14.2 kb downstream of IL23R and 33.1 kb upstream of IL12RB2.This region is DNase I hypersensitive (in Th1-cells, but not Th17-cells) and exhibits TF binding (lymphoblastoid cell-line; GM12878) and enhancer-associated H3K4me1 methylation (CD4+ CD25? IL-17A? T-cells; PMA and ionomycin-stimulated). \"\\tIL23R-IL12RB2\\tAnkylosing Spondylitis\\tDOID:7147\\tD013167\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\trs11209032\\t67740092\\t\"DNase I Hypersensitivity Assay,ChIP-qPCR,Luciferase Reporter Assay\"\\t--\\t\\t\\t\\t--', 'E_01_405\\t26427657\\t--\\thg19\\tchr3\\t78597238\\t79767998\\tHuman\\t\"Colorectal Cancer,Breast Cancer Cell\"\\tLow+High throughput\\t\"qPCR,ChIP-seq\"\\t\"We identified an identical germline deletion of the ROBO1 gene, which was confirmed by an independent microarray platform and qPCR analysis in three unrelated patients with BC and/or CRC. The ROBO1 deletion identified comprises an intronic region (intron 4) of four protein-coding splice variants of ROBO1\"\\tEnhancer\\tROBO1\\t--\\t\"qPCR,ChIP-seq\"\\t\"We identified an identical germline deletion of the ROBO1 gene, which was confirmed by an independent microarray platform and qPCR analysis in three unrelated patients with BC and/or CRC. The ROBO1 deletion identified comprises an intronic region (intron 4) of four protein-coding splice variants of ROBO1\"\\t\"AW494633, AW742721, DUTT1, Gm310\"\\t\"Colorectal Cancer,Breast Cancer\"\\t\"DOID:9256,DOID:1612\"\\tD001943\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2kb\\t\\t\\t\\t536230', 'E_01_406\\t26776159\\t--\\thg19\\tchr9\\t91737604\\t92057238\\tHuman\\tPrimary Human BJ Fibroblasts\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-qPCR,GRO-seq,ChIP-seq,4C\"\\t\"Thus,we assessed whether p53BERs can induce lincRNAs with essential roles within the p53 pathway. We first genome-wide mapped p53BERs using both publically available p53 chromatin immuno_x0002_precipitation sequencing (ChIP-seq) data sets10 and enhancer domains epigenetically defined by the Broad Institute segmentation.\"\\tEnhancer\\tlincRNA00475\\t4C\\t--\\t\"we identified and characterized a novel p53-bound intronic enhancer that controls the expression of its host, the lincRNA00475(linc-475). We demonstrate the requirement of linc-475 for the proper induction of a p53-dependent cell cycle inhibitory response.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t\"CTCF,TP53\"\\t\"MRD21,BCC7,BMFS5,LFS1,P53,TRP53\"\\tChIP-qPCR\\t\"In addition,we assessed whether CTCF could influence the p53-dependent regulation of linc-475. To tackle this question, we first evaluated and confirmed the binding of CTCF upstream of linc-475 using ChIPCquantitative PCR.Next, we tested whether knocking down CTCF could interfere with the expression levels of linc-475. \"\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_01_407\\t26780995\\t--\\thg19\\tchr19\\t33299481\\t33300688\\tHuman\\tBreast Cancer Cell\\tLow+High throughput\\t\"DNase I Hypersensitivity Assay,ATAC-seq,ChIP-seq\"\\t\"ChIP-seq of TFs predicts a subset of putative enhancers (reviewed in [13]) whereas ChIP_x0002_seq of p300 covers enhancers more ubiquitously [28]. High-throughput profiling of DNase I hypersensitive sites (DHSs) allows identification of enhancers [15], although DHSs also include other regulatory DNA regions such as promoters, insulators, and silencers.\"\\tEnhancer\\tCEP89\\t--\\t\"GWAS,ChIP-seq\"\\t\"It was found that rs10411210, which is associated with colorectal cancer [76], is located on an enhancer linked to the centrosomal protein 89kDa(CEP89) gene, which is listed in the COSMIC database. Although rs10411210 was mapped to the rhophilin, Rho GTPase binding protein 2 (RHPN2) gene based on the genomic distance,FANTOM5 CAGE data indicated that an enhancer covering rs10411210 targeted the CEP89 gene in addition to the RHPN2 gene based on the correlation of the transcription of the enhancer and that of target genes.\"\\t\"CCDC123, CEP123Colorectal Cancer\\tDOID:9256\\t--\\t--\\t--\\t--\\tEP300\\t\"KAT3B,MKHK2,RETS2,P300\"\\tChIP-seq\\t\"ChIP-seq of TFs predicts a subset of putative enhancers (reviewed in [13]) whereas ChIP_x0002_seq of p300 covers enhancers more ubiquitously [28]. High-throughput profiling of DNase I hypersensitive sites (DHSs) allows identification of enhancers [15], although DHSs also include other regulatory DNA regions such as promoters, insulators, and silencers.\"\\trs10411210 \\t33532300\\t\"DNase I Hypersensitivity Assay,ATAC-seq,ChIP-seq\"\\t>2KB\\t\\t\\t\\t69819', 'E_01_408\\t26795348\\t--\\thg19\\tchr2\\t202120390\\t202125986\\tHuman\\tHuman Mammary Epithelial Normal Breast Cell LineLow+High throughput\\t\"PCR,Luciferase Reporter Assay,ChIP-seq\"\\t\"Variants rs3769823, rs3769821, and rs10197246 were chosen to investigate for ability to affect enhancer activity. To create the enhancer constructs, we performed PCR using genomic DNA from T-47D, which is homozygous for risk alleles at all three SNPs, and MCF10A, which is homozygous for neutral alleles, using the following primer pairs.\"\\tEnhancer\\tCASP8\\t--\\tDNase I Hypersensitivity Assay\\t\"To prioritize further, we used expression-quantitative trait locus analysis of RNA sequencing from breast tissues, gene regulation annota_x0002_tions from the ENCODE consortium, and functional assays for differential enhancer activities. Notably, we implicate three regulatory variants at 2q33 that target CASP8 (rs3769823, rs3769821 in CASP8, and rs10197246 in ALS2CR12) as func_x0002_tionally relevant.\"\\t\"CASP-8, FLICE, MACH, Mch5\"\\tBreast Cancer\\tDOID:1612\\tD001943\\t--\\t--\\t--\\t\"STAT3,MYC,CTCF\"\\t\"1110034C02Rik, AW109958, Aprf,MRTLC,bHLHe39,c-Myc,MYC,MRD21\"\\t\"DNase I Hypersensitivity Assay,ChIP-seq\"\\t\"The first two reside 435 bp apart in a promoter proximal region that exhibits DNAse I hypersensitivity in mammary epithelial and breast cancer cell lines, and binds multiple transcription factors(STAT3, MYC, Pol2, and CTCF) based on ENCODE data (22).Also notable are results from an eQTL study in blood (23),indicating highly significant decreased CASP8 expression associ_x0002_ated with these cFSVs and suggesting the expression signature is also readily observed in blood.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t25022', 'E_01_409\\t26809031\\t--\\thg19\\tchr12\\t131949637\\t131949801\\tHuman\\tNormal Human Bronchial Epithelial Cells\\tLow throughput\\t\"RT-qPCR,ChIP,EMSA\"\\tDNA sequence alignment of the E1A enhancer regions of divergent Ad serotypes revealed that the E2F binding site located between nt 270C290 is highly conserved.\\tEnhancer\\tEP400\\t--\\tqPCR\\t\"This finding indicated that an IFN-induced repressor binding site is located in the downstream half of the E1A enhancer region, corresponding to Ad5 nt 270C358.\"\\t\"1700020J09Rik, AU023439, mDomino, mKIAA1498, p400\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"E2F1,SP1\"\\t\"E2F-1,RBAP1,RBBP3,RBP3,SP1\"\\t\"ChIP,EMSA\"\\tThe cellular transcription factor Sp1 binds to several sites in the Ad5 inverted terminal repeat (ITR)[26] that are adjacent to the E1A enhancer region. We analyzed if Sp1 binding to the ITR was altered by IFN signaling in vivo. Sp1 bound to the Ad5 ITR at similar levels in the presence and absence of IFNs (Fig 3B). This result suggests that reduced binding of GABP to the E1A enhancer following IFN treatment is target specific and not due to reduced global accessibility of the left-end of the Ad5 genome.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t484746', 'E_01_410\\t26814967\\t--\\thg19\\tchr8\\t128310315\\t128312315\\tHuman\\tMedulloblastoma Cell Lines\\tLow+High throughput\\t\"qPCR,ChIP-seq,RNA-seq,4C-seq\"\\t\"To deeply characterize the active cis-regulatory circuitry of a single disease entity, here medul_x0002_loblastoma, we performed high-resolution chromatin immuno_x0002_precipitation with sequencing (ChIP-seq) for active enhancers (H3K27ac) in 28 primary tumour specimens and three established cell lines. \"\\tEnhancer\\tMYC\\t--\\tChIP-seq\\t\"Given the apparent limitations of using cell lines to faithfully study the tumour epigenome, and the recognized subgroup-dependent hetero_x0002_geneity of medulloblastoma, we collected a series of 28 treatment_x0002_naive, fresh-frozen medulloblastoma specimens and profiled the active enhancer landscape by H3K27ac ChIP-seq.\"\\t\"AU0167572, Niard, Nird, bHLHe39, Myc\"\\tMedulloblastoma\\tDOID:0050902\\tD008527\\t--\\t--\\t--\\t\"POU5F1,SOX2,NANOG\"\\t\"OCT3,OCT4,OTF-3,OTF3,OTF4,Oct-3,Oct-4,ANOP3,MCOPS3,NANOG\"\\tChIP-seq\\t\"Computational reconstruction of core regulatory circuitry identified a master set of transcription factors, validated by ChIP-seq, that is responsible for subgroup divergence, and implicates candidate cells of origin for Group 4. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t436999', 'E_01_411\\t26814967\\t--\\thg19\\tchr2\\t29511793\\t29765640\\tHuman\\tMedulloblastoma Cell Lines\\tLow+High throughput\\t\"qPCR,ChIP-seq,RNA-seq,4C-seq\"\\t\"To deeply characterize the active cis-regulatory circuitry of a single disease entity, here medul_x0002_loblastoma, we performed high-resolution chromatin immuno_x0002_precipitation with sequencing (ChIP-seq) for active enhancers (H3K27ac) in 29 primary tumour specimens and three established cell lines. \"\\tEnhancer\\tALK\\t--\\tChIP-seq\\t\"To validate the robustness of our methods, we used 4C-seq19 to query Group 3-specific enhancerCpromoter interactions for enhancers showing conserved activity in both primary Group 3 tumours and cell lines. This approach con_x0002_firmed enhancerCpromoter interactions for both TGFBR1 and SMAD9in the Group 3 cell line HD-MB03, a low-passage line more faithful to primary Group 3 tumours than older models.\"\\t\"CD246, NBLST3\"\\tMedulloblastoma\\tDOID:0050902\\tD008527\\t--\\t--\\t--\\t\"POU5F1,SOX2,NANOG\"\\t\"OCT3,OCT4,OTF-3,OTF3,OTF4,Oct-3,Oct-4,ANOP3,MCOPS3,NANOG\"\\tChIP-seq\\t\"Computational reconstruction of core regulatory circuitry identified a master set of transcription factors, validated by ChIP-seq, that is responsible for subgroup divergence, and implicates candidate cells of origin for Group 4. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t223077', 'E_01_412\\t26814967\\t--\\thg19\\tchr5\\t893433\\t1083433\\tHuman\\tMedulloblastoma Cell Lines\\tLow+High throughput\\t\"qPCR,ChIP-seq,RNA-seq,4C-seq\"\\t\"To deeply characterize the active cis-regulatory circuitry of a single disease entity, here medul_x0002_loblastoma, we performed high-resolution chromatin immuno_x0002_precipitation with sequencing (ChIP-seq) for active enhancers (H3K27ac) in 30 primary tumour specimens and three established cell lines. \"\\tSuper-Enhancer\\tNKD2\\t--\\tChIP-seq\\t\"Compared to typ_x0002_ical enhancers, SEs showed higher occupancy of BRD4 and greater enhancer signal dynamic range between subgroups.Targets of differential enhancers contained within SEs included a large fraction of established medullo_x0002_blastoma signature genes, as well as novel candidates.Medulloblastoma SEs were inferred to regulate known Cancer Gene Census genes, including the aforemen_x0002_tioned ALK in WNT, SMO and NTRK3 in SHH, LMO1, LMO2, and MYC in Group 3, and ETV4 and PAX5 in Group 4, among others. \"\\tNaked2\\tMedulloblastoma\\tDOID:0050902\\tD008527\\t--\\t--\\t--\\t\"POU5F1,SOX2,NANOG\"\\t\"OCT3,OCT4,OTF-3,OTF3,OTF4,Oct-3,Oct-4,ANOP3,MCOPS3,NANOG\"\\tChIP-seq\\t\"Computational reconstruction of core regulatory circuitry identified a master set of transcription factors, validated by ChIP-seq, that is responsible for subgroup divergence, and implicates candidate cells of origin for Group 4. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t20508', 'E_01_413\\t26814967\\t--\\thg19\\tchr21\\t47716490\\t47892045\\tHuman\\tMedulloblastoma Cell Lines\\tLow+High throughput\\t\"qPCR,ChIP-seq,RNA-seq,4C-seq\"\\t\"To deeply characterize the active cis-regulatory circuitry of a single disease entity, here medul_x0002_loblastoma, we performed high-resolution chromatin immuno_x0002_precipitation with sequencing (ChIP-seq) for active enhancers (H3K27ac) in 31 primary tumour specimens and three established cell lines. \"\\tSuper-Enhancer\\tPCNT\\t--\\tChIP-seq\\t\"Compared to typ_x0002_ical enhancers, SEs showed higher occupancy of BRD4 and greater enhancer signal dynamic range between subgroups.Targets of differential enhancers contained within SEs included a large fraction of established medullo_x0002_blastoma signature genes, as well as novel candidates.Medulloblastoma SEs were inferred to regulate known Cancer Gene Census genes, including the aforemen_x0002_tioned ALK in WNT, SMO and NTRK3 in SHH, LMO1, LMO2, and MYC in Group 3, and ETV4 and PAX5 in Group 4, among others. \"\\t\"KEN, MOPD2, PCN2, PCNTB, PCTN2, SCKL4, PCNT\"\\tMedulloblastoma\\tDOID:0050902\\tD008527\\t--\\t--\\t--\\t\"POU5F1,SOX2,NANOG\"\\t\"OCT3,OCT4,OTF-3,OTF3,OTF4,Oct-3,Oct-4,ANOP3,MCOPS3,NANOG\"\\tChIP-seq\\t\"Computational reconstruction of core regulatory circuitry identified a master set of transcription factors, validated by ChIP-seq, that is responsible for subgroup divergence, and implicates candidate cells of origin for Group4. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t60292', 'E_01_414\\t26814967\\t--\\thg19\\tchr2\\t221040922\\t221069549\\tHuman\\tMedulloblastoma Cell Lines\\tLow+High throughput\\t\"qPCR,ChIP-seq,RNA-seq,4C-seq\"\\t\"To deeply characterize the active cis-regulatory circuitry of a single disease entity, here medul_x0002_loblastoma, we performed high-resolution chromatin immuno_x0002_precipitation with sequencing (ChIP-seq) for active enhancers (H3K27ac) in 32 primary tumour specimens and three established cell lines. \"\\tSuper-Enhancer\\tHLX\\t--\\tChIP-seq\\t\"Compared to typ_x0002_ical enhancers, SEs showed higher occupancy of BRD4 and greater enhancer signal dynamic range between subgroups.Targets of differential enhancers contained within SEs included a large fraction of established medullo_x0002_blastoma signature genes, as well as novel candidates.Medulloblastoma SEs were inferred to regulate known Cancer Gene Census genes, including the aforemen_x0002_tioned ALK in WNT, SMO and NTRK3 in SHH, LMO1, LMO2, and MYC in Group 3, and ETV4 and PAX5 in Group 4, among others. \"\\t\"HB241, HLX\"\\tMedulloblastoma\\tDOID:0050902\\tD008527\\t--\\t--\\t--\\t\"POU5F1,SOX2,NANOG\"\\t\"OCT3,OCT4,OTF-3,OTF3,OTF4,Oct-3,Oct-4,ANOP3,MCOPS3,NANOG\"\\tChIP-seq\\t\"Computational reconstruction of core regulatory circuitry identified a master set of transcription factors, validated by ChIP-seq, that is responsible for subgroup divergence, and implicates candidate cells of origin for Group 4. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2493', 'E_01_415\\t26814967\\t--\\thg19\\tchr5\\t121568382\\t121839811\\tHuman\\tMedulloblastoma Cell Lines\\tLow+High throughput\\t\"qPCR,ChIP-seq,RNA-seq,4C-seq\"\\t\"To deeply characterize the active cis-regulatory circuitry of a single disease entity, here medul_x0002_loblastoma, we performed high-resolution chromatin immuno_x0002_precipitation with sequencing (ChIP-seq) for active enhancers (H3K27ac) in 33 primary tumour specimens and three established cell lines. \"\\tSuper-Enhancer\\tSNCAIP\\t--\\tChIP-seq\\t\"Compared to typ_x0002_ical enhancers, SEs showed higher occupancy of BRD4 and greater enhancer signal dynamic range between subgroups.Targets of differential enhancers contained within SEs included a large fraction of established medullo_x0002_blastoma signature genes, as well as novel candidates.Medulloblastoma SEs were inferred to regulate known Cancer Gene Census genes, including the aforemen_x0002_tioned ALK in WNT, SMO and NTRK3 in SHH, LMO1, LMO2, and MYC in Group 3, and ETV4 and PAX5 in Group 4, among others. \"\\t\"SYPH1, Sph1\"\\tMedulloblastoma\\tDOID:0050902\\tD008527\\t--\\t--\\t--\\t\"POU5F1,SOX2,NANOG\"\\t\"OCT3,OCT4,OTF-3,OTF3,OTF4,Oct-3,Oct-4,ANOP3,MCOPS3,NANOG\"\\tChIP-seq\\t\"Computational reconstruction of core regulatory circuitry identified a master set of transcription factors, validated by ChIP-seq, that is responsible for subgroup divergence, and implicates candidate cells of origin for Group 4. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t56642', 'E_01_416\\t26817450\\t--\\thg19\\tchr1\\t230293530\\t230305966\\tHuman\\tHep G2\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay,EMSA\"\\tEMSA and luciferase assays were used to validate the allele specific binding and to test the enhancer activity of the regulatory element harboring the AS-SNP rs4846913 as well as the neighboring rs2144300 which are in high LD with rs4846914.\\tEnhancerGALNT2\\t--\\tGWAS\\tUsing luciferase assays we found that rs4846913 and the neighboring rs2144300 displayed allele specific enhancer activity. We propose that an inhibitor binds preferentially to the rs4846913-C allele with an inhibitory boost from the synergistic binding of other TFs at the neighboring SNP rs2144300. These events influence the transcription level of GALNT2.\\tGalNAc-T2\\t--\\t--\\t--\\t--\\t--\\t--\\tZBTB3\\tZBTB3\\t\"EMSA,Luciferase Reporter Assay\"\\tUsing luciferase assays we found that rs4846913 and the neighboring rs2144300 displayed allele specific enhancer activity. We propose that an inhibitor binds preferentially to the rs4846913-C allele with an inhibitory boost from the synergistic binding of other TFs at the neighboring SNP rs2144300.\\t\"rs4846913,rs2144300\"\\t\"230294715,230294916\"\\t\"ChIP-seq,Luciferase Reporter Assay,EMSA\"\\t>2KB\\t\\t\\t\\t96792', 'E_01_417\\t26818267\\t--\\thg19\\tchr22\\t18897006\\t18899006\\tHuman\\tDLBCL Cells\\tLow+High throughput\\t\"Luciferase Reporter Assay,PCR,ChIP-seq\"\\t\"Deletion of the LTR as well as insertion of the chimpanzee orthologous regulatory sequence imparts the activity of the human PRODH pro_x0002_moter in luciferase assays, showing that the LTR of hsERV_PRODH is critical for maintaining human_x0002_specific and high expression of PRODH.\"\\tEnhancer\\tPRODH\\t--\\tLuciferase Reporter Assay\\t2.2 kb upstream of the PRODH promoter the LTR of a hsERV (hsERV_PRODH) functions as a neuron-specific enhancer in hippocampus where it regulates PRODH expression in synergy with a CpG island that overlaps PRODH exon 2.\\t\"HSPOX2, PIG6, POX1, PRODH2, TP53I6, PRODH\"\\tDiffuse Large B-Cell Lymphoma\\tDOID:0050745\\tD016403\\tSeveral studies have shown that human endogenous retroviruses and endogenous retrovirus-like repeats (here collec_x0002_tively HERVs) impose direct regulation on human genes through enhancer and promoter motifs present in their long terminal repeats (LTRs\\tLuciferase Reporter Assay\\t\"Deletion of the LTR as well as insertion of the chimpanzee orthologous regulatory sequence imparts the activity of the human PRODH pro_x0002_moter in luciferase assays, showing that the LTR of hsERV_PRODH is critical for maintaining human_x0002_specific and high expression of PRODH.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2200', 'E_01_418\\t26859151\\t--\\thg19\\tchr5\\t172196326\\t172199378\\tHuman\\t\"MCF-7,MDA-MB-231\"\\tLow+High throughput\\tChIP-seq\\t\"Similar co-localisation studies were conducted in MDA-MB-231 cells to further understand', 'E_01_419\\t26859151\\t--\\thg19\\tchr8\\t29203575\\t29214287\\tHuman\\t\"MCF-7,MDA-MB-231\"\\tLow+High throughput\\tChIP-seq\\t\"DUSP4 co-localised with the enhancer mark H3K27ac [40] in MCF-7/PMA (PCC = 0.40) but not in non-stimulated MCF-7 cells (PCC = -0.30). Moreover, DUSP4 co-localised with the enhancer mark H3K4me1 in MCF-7 cells (PCC = 0.35), which increased in MCF-7/PMA cells (PCC = 0.44) (Fig 2D). In comparison, there was absent or minimal co-localisation of DUSP4 with H3K4me3,', 'E_01_420\\t26864944\\t--\\thg19\\tchr14\\t24435744\\t24437744\\tHuman\\t\"Hep G2,HL-7702\"\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\t\"we searched for and identified a putative enhancer region located 13.8 kb downstream from the TSS of DHRS4-AS117,18. We performed chromatin immunoprecipitation (ChIP) to confirm the enrichment of histone modifications in hepatocellular carcinoma cells (HepG2) and normal hepatocytes (HL7702) (Fig.?1D). We then examined the potential enhancement of luciferase reporter activity by using luciferase reporter assays in HepG2 and HL7702 cells (Fig.?1E). \"\\tEnhancer\\tDHRS4-AS1\\t3C\\t\"Luciferase Reporter Assay,ChIP,qPCR\"\\t\"In a search of the DHRS4-AS1 region for nearby enhancers, we identified an enhancer located 13.8 kb downstream of the DHRS4-AS1 transcriptional start site. We further showed, by using a chromosome conformation capture (3C) assay, that this enhancer is capable of physically interacting with the DHRS4-AS1 promoter through chromosomal looping. \"\\t\"AS1DHRS4,C14orf167,C14orf67,DHRS4AS1,PRO1488\"\\t--\\t--\\t--\\t--\\t--\\t--\\tEP300\\t\"KAT3B,MKHK2,RSTS2,p300\"\\tChIP\\t\"RNA pol II and the transcriptional coactivator p300/CBP occupy enhancer regions and target gene promoters to maintain chromatin loops29. We next characterized the effect of AS1eRNA knockdown on the binding levels of RNA pol II and p300 at the AS1 enhancer and the', 'E_01_421\\t26884396\\t--\\thg19\\tchr14\\t61121917\\t61123917\\tHuman\\tNephron Progenitors\\tLow+High throughput\\tChIP-seq\\t\"ChIP-seq was performed on 17 week fetal kidneys and E16.5 mouse kidneys to assess chromatin marks associated with active genes and enhancers (H3K27ac) and transcriptionally silenced chromatin (H3K27me3). Similarly, the human SIX1 locus was bound by SIX2 at multiple conserved elements and displayed prominent H3K27ac throughout the gene body and the SIX2-bound regions (Fig. 3A).\"\\tEnhancer\\tSIX1\\t--\\tRNA-seq\\t\"Using RNA-seq data from nephron progenitors, we compared expression of genes between human', 'E_01_422\\t26884396\\t--\\thg19\\tchr14\\t61106417\\t61108417\\tHuman\\tNephron Progenitors\\tLow+High throughput\\tChIP-seq\\t\"ChIP-seq was performed on 17 week fetal kidneys and E16.5 mouse kidneys to assess chromatin marks associated with active genes and enhancers (H3K27ac) and transcriptionally silenced chromatin (H3K27me3). Similarly, the human SIX1 locus was bound by SIX2 at multiple conserved elements and displayed prominent H3K27ac throughout the gene body and the SIX2-bound regions (Fig. 3A).\"\\tEnhancer\\tSIX1\\t--\\tRNA-seq\\t\"Using RNA-seq data from nephron progenitors, we compared expression of genes between human', 'E_01_423\\t26679052\\t--\\thg19\\tchr8\\t140657900\\t141002079\\tHuman\\t5637\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,RT-PCR\"\\t\"Realtime PCR analysis showed that antrocin downregulated the expression of mRNA of several MMPs, including MMP-2. Moreover, the phosphorylation of ERK and c-Fos were also attenuated by antrocin\"\\tEnhancer\\tFAK \\t--\\t\"Luciferase Reporter Assay,ChIP,RT-PCR\"\\t\"Realtime PCR analysis showed that antrocin downregulated the expression of mRNA of several MMPs, including MMP-2. Moreover, the phosphorylation of ERK and c-Fos were also attenuated by antrocin\"\\t\"FADK, FAK, FAK1, FRNK, PPP1R71, p125FAK, pp125FAK\"\\tBladder Cancer\\tDOID:11054\\tD001749\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2kb\\t\\t\\t\\t838491', 'E_01_424\\t26505625\\t--\\thg19\\tchr7\\t95210132\\t95260132\\tHuman\\tccRCC Cells\\tLow+High throughput\\t\"ChIP,ChIP-seq\"\\t\"rs11762213 alone is found in a regulatory element of MET likely in an enhancer domain, and this has implications for its interaction both with MET itself and with potentially other cis-acting target genes.To further characterize the functional significance of rs11762213 we used our previously generated genome-wide chromatin annotation maps using cultured human proximal tubular epithelial cells (HKC8) and overlaid them with previously generated gene regulatory annotation maps from a panel of ChIP-seq data using the hidden Markov model-based ChromHMM chromatin segmentation program (Figure 4). Notably, rs11762213 maps to an H3K4me1 histone modification mark, which serves as an enhancer marker and is therefore consistent with the hypothesis that the SNP has a regulatory function.Mapping of rs11762213 to regulatory regions within the genome suggests that it may impact a DNA enhancer region.\"\\tEnhancer\\tMET\\t--\\t\"ChIP,RNA-seq\"\\t\"Next, we assessed the impact of rs11762213 on MET steady-state mRNA and protein tumor expression using available RNA seq data and reverse phase protein array (RPPA) data from the TCGA. We found no difference in tumor MET expression by SNP status (p=0.47 Mann Whitney) including all detected MET isoforms (n=18) (Supplemental figure 2A). Since rs11762213 is located in the coding region of exon 2 we further explored exon level expression differences by genotype and again did not find any difference by genotype for exon 2 MET expression (p=0.29) of any other exon (Supplemental figure 3). Additionally, we did not see differences in tumor MET protein (p=0.88) or MET phospho-protein expression (Y1235) (Supplemental figure 2B and 2C).rs11762213 was associated with higher normal tissue MET expression (p=0.019), however, in an independent normal kidney Affymetrix mRNA array data set this finding did not validate (n=95) (data not shown).\"\\t\"AI838057,HGF,HGFR,Par4,c-Met\"\\tClear Cell Renal Cell Carcinoma\\tDOID:4467\\tD002292\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_01_425\\t26443750\\t--\\thg19\\tchr1\\t153628434\\t153632039\\tHuman\\tHEKa\\tLow throughput\\t\"RT-PCR,Western blot\"\\t\"Methylation analysis was also performed for three other EDC genes of known expression pattern (involucrin, loricrin, and NICE-1) and a recently identified evolutionary conserved region with defined enhancer properties.\"\\tEnhancer\\tLCE1A\\t--\\t\"RT-PCR,Western blot\"\\t\"Methylation analysis was also performed for three other EDC genes of known expression pattern (involucrin, loricrin, and NICE-1) and a recently identified evolutionary conserved region with defined enhancer properties.\"\\tLEP1\\tSkin Barrier\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_001\\t25209250\\t--\\tmm10\\tchr10\\t52388022\\t52388224\\tMouse\\tP19\\tLow throughput\\tLuciferase Reporter AssaLuciferase constructs containing the Nepn TSS and upstream promoter regions were assayed in P19 cells treated with DMSO or RA (0.1 M). The length of the upstream region for each construct is indicated in the graphic. Relative luciferase activity represents the fold increase observed in the presence of RA relative to DMSO.\\tEnhancer\\tNepn\\tCRISPR/Cas9\\t\"Luciferase Reporter Assay,RT-PCR\"\\tLuciferase assays indicated that promoter activity of the construct (2.3 kb) that contains both Sox17-binding motif and RARE was dramatically enhanced with ectopic expression of Sox17 in the presence of RA.\\t\"Npn,periolin,5730521E12Rik\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Sox17,Rara\"\\t\"Sox17,Nr1b1,RAR,RARalpha1\"\\t\"Luciferase Reporter Assay,EMSA\"\\t\"The luciferase activity of these constructs was evaluated in P19 cells in the presence of ectopically expressed Sox17. A significant drop in Sox17-inducible activity occurred when the sequence between ?842?bp and ?640?bp upstream of the Nepn TSS was deleted, suggesting that an enhancer region that is dependent on Sox17 lies in this region.EMSA assay of nuclear protein binding to the proposed RARE from the Nepn promoter. Myc-tagge RAR was ectopically expressed in P19 cells treated with RA (0.1 M) and nuclear extracts were incubated with 32P-labelled RARE from the Nepn promoter. Competition with a 100-fold excess of unlabeled wild-type or mutated RARE is indicated. Anti-Myc antibodies were included in the binding assay as indicated. Sindicates the RAR-DNA complex; SS indicates the super shifted complex in the presence of anti-Myc antibody. **P<0.01. Mut, mutated; NE, nuclear extracts; Wt, wild type.\"\\t--\\t--\\t--\\t<2KB\\t742', 'E_02_002\\t18552207\\t--\\tmm10\\tchr7\\t126406352\\t126406616\\tMouse\\t\"B Cell,T Cell\"\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\t\"Transient transfection assays in different mouse cell lines. A mouse Cd19 promoter reporter construct (_x005f_x005f_x005f_x005f_x005f_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x005f_x005F_x0007_7 to _x005F_x0007_200 bp; e) or a construct containing the _x0007_2 kb DHS (_x0007_1832 to _x0007_2096 bp) combined with the Cd19 promoter (f) was transiently transfected into various cell lines. The', 'E_02_003\\t18984737\\t-cell specific Enhancer\\tmm10\\tchr3\\t51508734\\t51512187\\tMouse\\t\"TC-3,INS-1,TC1.6,NIH3T3\"\\tLow throughput\\tLuciferase Reporter Assay\\t\"To directly test whether this region of the Setd7 gene might be regulated in -cells, a DNA fragment containing', 'E_02_004\\t19041414\\t--\\tmm10\\tchr15\\t97977739\\t97977942\\tMouse\\tHeLa\\tLow throughput\\t\"Luciferase Reporter Assay,EMSA\"\\t\"Transient transfections of Sox9, Smad3, and T_x0002_R-I(TD) did not increase luciferase activities of pGL3-B plasmids in SW1353 cells (pGL3-B). In pGL3-585E systems, Sox9 enhanced a relative luciferase activity to a level as high as 2.2-fold over the control. Cotransfection of Smad3 augmented a luciferase activity up to 2.3-fold higher level of Sox9-transfected cells. The additional transfection of constitutively active form of T_x0002_R-I(TD) induced an approximately 36% increase of the activity in Sox9- and Smad3-transfected SW1353 cells. Luciferase activities of pGL3-585E were not increased in the absence of Sox9. Note that Smad3 and T_x0002_R-I(TD) synergistically activated the native Col2a1 reporter-mediated transcription in a Sox9-dependent manner. \"\\tEnhancer\\tCol2a1\\t--\\t\"Luciferase Reporter Assay,EMSA\"\\tPurified Sox9 associated with the Col2a1 enhancer probe in EMSA.\\t\"Col2,Col2a,Col2a-1,Del1,Dmm,Lpk,M100413,Rgsc413,Rgsc856\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSox9\\t\"2010306G03Rik,AV220920,mKIAA4243\"\\tEMSA\\tPurified Sox9 associated with the Col2a1 enhancer probe in EMSA. The unlabeled competitor decreased the signal of Sox9CDNA complex. Supershifted band was observed in the presence of anti-Sox9 antibody.\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t2240', 'E_02_005\\t19285986\\t--\\tmm10\\tchr8\\t105528423\\t105529045\\tMouse\\tN38\\tLow throughput\\t\"Transgenic mice,Luciferase Reporter Assay\"\\t\"Specifically, Diiib had as much activity as subregion Diii on its own but only in the h295R cells, while subregion Diiia had the same activity as the parent Diii region but only in the N38 cells.\"\\tEnhancer\\tAgrp\\t--\\t\"Transgenic mice,Luciferase Reporter Assay,qPCR\"\\tqPCR shows robust expression of AgRP in the tongue but at lower levels than in the hypothalamus. (c) Immunohistochemistry of mouse tongue sections shows AgRP expression in epithelial cells of wild-type (AgRP+/+) but not AgRP-deficient (AgRP?/?) mice\\t\"Agrt,Art\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t37960', 'E_02_006\\t19345119\\tMHB Enhancer\\tmm10\\tchr4\\t44520195\\t44520234\\tMouse\\tLymphopoiesis B Cell\\tLow throughput\\t\"Transfection,ChIP,EMSA\"\\t\"By using transgenic analysis, we have previously identified a 435 bp enhancer (located 5.6 kb upstream of exon 1A), which is sufficient to drive reporter gene expression in the developing MHB region.To confirm these footprinting data, we performed ChIP analysis with unstimulated cells of the mature B cell line WEHI-279 (Figure 7C; Figure S14A) and with in vitro cultured pro-B cells (Figure S14B) to screen all HS sites of the Pax5 locus for binding of PU.1, IRF4, IRF8, NF-kB, and Myc. Electrophoretic mobility shift assays(EMSAs)with an oligonucleotide probe containing the EBF1 recognition sequence of HS-7 revealed efficient protein binding to this site with a pro-B cell nuclear extract.\"\\tEnhancer\\tPax5\\t--\\tTransgenic mice\\t\"A DNA fragment containing the entire Pax5 promoter region from the MHB enhancer (at position -6,074) to exon 2 (at position +13,368) was fused in-frame in exon 2 to a Gfp reporter gene to generate transgene PromGFP.GFP expression was determined by flow cytometry in different B cell types of a PromGFP transgenic mouse. PromGFP transgenic mice failed to express GFP in pro-B and mature B cells, but gave rise to low GFP expression in pre-B and immature B cells(Figure 4A) similar to the BAC1-DIn5 transgene .\"\\t\"BSAP,EBB-1,KLP,Pax-5\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Spi1,Irf4,Irf8,Nfkb1\"\\t\"Dis-1,Dis1,PU.1,Sfpi-1,Sfpi1,Spi-1,Tcfpu1,Tfpu.1,AI385587,IRF-4,LSIRF,NF-EM5,Spip,AI893568,ICSBP,IRF-8,Icsbp1,Myls,NF-KB1,NF-kappaB,NF-kappaB1,p105,p50,p50/p105\"\\t\"ChIP,DMS Footprinting Assay\"\\t\"PU.1, IRF4, IRF8, and NF-kB Control the Activity of the Pax5 Enhancer.To confirm these footprinting data, we performed ChIP analysis with unstimulated cells of the mature B cell line WEHI-279 and with in vitro cultured pro-B cells to screen all HS sites of the Pax5 locus for binding of PU.1, IRF4, IRF8, NF-kB, and Myc. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t6053', 'E_02_007\\t20042469\\tCNS-2 Enhancer\\tmm10\\tchr3\\t37225520\\t37228865\\tMouse\\tCD4+ T Cell\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP-qPCR,ChIP,PCR\"\\t\"To determine the function of IL-21P and CNS-2 in the transcriptional control of IL-21 expression by c-Maf, we performed a reporter assay. IL-21P-Luci containing a fragment of IL-21P (positions C261 to 17), IL-21P-Luci-CNS-2 containing IL-21P and CNS-2 (position +2931 to +3443), or control pGL3 was transfected into EL4 cells, together with pcDNA3 c-Maf or control pcDNA3, and the luciferase activity of the reporter construct was evaluated with or without PMA+I stimulation. As shown in Figure3B, IL-21P-Luci responded to PMA+I stimulation, and c-Maf enhanced the activity of IL-21P-Luci (n=3; P<0.05) further. IL-21P-Luci-CNS-2 responded much greater to c-Maf in the presence of PMA+I stimulation (Fig. 3B). These results suggest that CNS-2 functions as an enhancer of the IL-21 gene and that c-Maf activates IL-21P as well as CNS-2.After cells were stimulated with or without PMA+I for 1 h, ChIP-qPCR assay for IL-21P, CNS-2, and CNS-4 was performed with anti-c-Maf anti-body or control rabbit IgG. Results are expressed as the percent input for each ChIP fraction.\"\\tEnhancer\\tIl21\\t--\\tChIP\\t\"To determine whether c-Maf actually binds to IL-21P and the CNS-2 enhancer in CD4 + T cells, we performed ChIP analysis.These results suggest that CNS-2 functions as an Enhancer of the IL-21 gene and that c-Maf acti_x005F_x0002_vates IL-21P as well as CNS-2.\"\\tIL-21\\t--\\t--\\t--\\tc-Maf induces IL-21 production directly in CD4_x005f_x0002_ T cells by activating IL-21P and the CNS-2 Enhancer and that TGF- suppresses c-Maf-induced IL-21 production in CD4_x0002_ T cells.\\t\"Luciferase Reporter Assay,PCR\"\\t\"Taken together with the data shown in Figure 3, these results suggest that c-Maf binds di_x005f_x005f_x005f_x005F_x005f_x005F_x005f_x005F_x0002_rectly to IL-21P and the CNS-2 Enhancer in CD4_x005F_x0001_ T cells after IL-6 stimulation and that TGF-_x0001_ enhances IL-6-induced c-Maf binding to IL-21P and the Enhancer.\"\\tMaf\\t\"2810401A20Rik,A230108G15Rik,AW047063,c-maf\"\\tChIP\\t\"To determine whether c-Maf actually binds to IL-21P and the CNS-2 Enhancer in CD4 T cells, we performed ChIP analysis.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t4435', 'E_02_008\\t20864515\\t--\\tmm10\\tchr6\\t124718594\\t124718929\\tMouse\\t\"Hep G2,NMuLi\"\\tLow throughput\\t\"ChIP,Luciferase Reporter Assay\"\\t\"By ChIP, two of these sites bound Prep1 (Fig. 7A; positions nucleotide [nt] ?2,489 to ?2,139, nt ?2,113 to ?1,778), but re-ChIP assays revealed that Pbx1 was simultaneously present only at position nt ?2,113 to ?1,778.To investigate whether this last region features enhancer activity, we subsequently cloned this fragment in the pgl3 basic construct upstream the luciferase gene (pgl3LUC). The construct was then cotransfected in HeLa cells together with the Prep1, Pbx1 cDNAs or both and luciferase activity was measured (Fig. 7B).Prep1 and Pbx1 increased the SHP1 reporter activity, respectively, by 7.1-fold and 6-fold. Simultaneous cotransfection of the two plasmids caused an almost 30-fold induction, indicating SHP1 transcriptional regulation by the Prep1/Pbx1 complex.\"\\tEnhancer\\tNr0b2\\t--\\tLuciferase Reporter Assay\\t\"lin-dependent IR phosphorylation and glycogen accumulation. Both Prep1 and Pbx1 bind SHP1 promoter at a site located between nucleotides -2,113 and -1,778. This fragment features Enhancer activity and induces luciferase function by 7-, 6-, and 30-fold, respectively,in response to Prep1, Pbx1, or both.\"\\t\"SHP,SHP-1,Shp1\"\\tDiabetes Mellitus\\tDOID:9351\\tD003920\\t\"Both Prep1 and Pbx1 bind SHP1 promoter at a site located between nucleotides -2,113 and -1,778. This fragment features Enhancer activity and induces luciferase function by 7-, 6-, and 30-fold, respectively, in response to Prep1, Pbx1, or both.\"\\t\"Luciferase Reporter Assay,ChIP\"\\t\"In these assays, a fragment containing single Prep1/Pbx1 binding site displays a powerful Enhancer function. In addition, ChIp experiments with Prep1 and Pbx1 antibod_x005f_x0002_ies showed that these proteins bind the SHP1 regulatory region, suggesting that SHP1 gene is a target of the Prep1/Pbx1 complex and not just Prep1.\"\\t\"Pknox1,Pbx1\"\\t\"D17Wsu76e,PREP1,2310056B04Rik,D230003C07Rik,Pbx-1\"\\t\"Luciferase Reporter Assay,ChIP\"\\t\"In these assays, a fragment containing single Prep1/Pbx1 binding site displays a powerful Enhancer function. The construct was then cotransfected in HeLa cells together with the Prep1, Pbx1 cDNAs or both and luciferase activity was measured In addition, ChIp experiments with Prep1 and Pbx1 antibodies showed that these proteins bind the SHP1 regulatory region, suggesting that SHP1 gene is a target of the prep1/Pbx1 complex and not just Prep1.\"\\t--\\t--\\t--\\t>2KB\\t\\t8834613', 'E_02_009\\t20878775\\t--\\tmm10\\tchr13\\t53463266\\t53463541\\tMouse\\t\"C2C12,Primary Bone MarrowCMesenchymal Stem CellLow throughput\\t\"PCR,Luciferase Reporter Assay\"\\tThe murine Dlx5 promoter from -1036 to +30 was obtained by polymerase chain reaction (PCR) and cloned in the pGL2basic vector. Murine Msx2 and Runx2 enhancers (from -3340 to -3615 for Msx2 and from -1170 to -1388 for Runx2) also were obtained by PCR and subcloned in the minimal promoter of c-fos promoter.\\tEnhancer\\tMsx2\\t--\\tLuciferase Reporter Assay\\t\"The Msx2 gene shows a conserved region 3.5 kb upstream of the transcriptional start site, which also includes a combination of a TCF/LEF1 box and both GC-rich and CAGA SBEs. This region has been shown to be necessary and sufficient to confer BMP responsiveness in vitro and in vivo.Finally, a similar enhancer is located 1 kb upstream of the P1 osteoblast-specific promoter of the Runx2 gene (Supplemental Fig. S3). To test whether these regions have the ability to render responsiveness to Wnt3a and/or BMP-2, we generated the corresponding luciferase constructs and assayed them in C2C12 cells. \"\\t\"BB122635,Hox-8,Hox8,Hox8.1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Msx2,Dlx3,Dlx5,Atf4,Runx2,Sp7\"\\t\"BB122635,Hox-8,Hox8,Hox8.1,AV237891,Dlx-3,AI385752,Atf-4,C/ATF,CREB2,TAXREB67,AML3,Cbf,Cbfa-1,Cbfa1,LS3,Osf2,PEBP2aA,Pebp2a1,Pebpa2a,6430578P22Rik,C22,Osx\"\\t\"qRT-PCR,Luciferase Reporter Assay,ChIP\"\\t\"To confirm that cooperative transcriptional interactions between canonical Wnt signaling and the Smad pathway on osteogenic genes occur in vivo, ChIP assays were performed in C2C12 cells.As shown in Fig.7D,TCF4,b-catenin,and Smad1 and Smad4 interacted with the responsive regions of Dlx5, Msx2, and Runx2 genes in vivo.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3476', 'E_02_010\\t20878775\\t--\\tmm10\\tchr17\\t44494486\\t44495486\\tMouse\\t\"C2C12,Primary Bone MarrowCMesenchymal Stem CellLow throughput\\t\"PCR,Luciferase Reporter Assay\"\\tThe murine Dlx5 promoter from -1036 to +30 was obtained by polymerase chain reaction (PCR) and cloned in the pGL2basic vector. Murine Msx2 and Runx2 enhancers (from -3340 to -3615 for Msx2 and from -1170 to -1388 for Runx2) also were obtained by PCR and subcloned in the minimal promoter of c-fos promoter.\\tEnhancer\\tRunx2\\t--\\tLuciferase Reporter Assay\\t\"The Msx2 gene shows a conserved region 3.5 kb upstream of the transcriptional start site, which also includes a combination of a TCF/LEF1 box and both GC-rich and CAGA SBEs. This region has been shown to be necessary and sufficient to confer BMP responsiveness in vitro and in vivo.Finally, a similar enhancer is located 1 kb upstream of the P1 osteoblast-specific promoter of the Runx2 gene (Supplemental Fig. S3). To test whether these regions have the ability to render responsiveness to Wnt3a and/or BMP-2, we generated the corresponding luciferase constructs and assayed them in C2C12 cells. \"\\t\"AML3,Cbf,Cbfa-1,Cbfa1,LS3,Osf2,PEBP2aA,Pebp2a1,Pebpa2a\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Msx2,Dlx3,Dlx5,Atf4,Runx2,Sp7\"\\t\"BB122635,Hox-8,Hox8,Hox8.1,AV237891,Dlx-3,AI385752,Atf-4,C/ATF,CREB2,TAXREB67,AML3,Cbf,Cbfa-1,Cbfa1,LS3,Osf2,PEBP2aA,Pebp2a1,Pebpa2a,6430578P22Rik,C22,Osx\"\\t\"qRT-PCR,Luciferase Reporter Assay,ChIP\"\\t\"To confirm that cooperative transcriptional interactions between canonical Wnt signaling and the Smad pathway on osteogenic genes occur in vivo, ChIP assays were performed in C2C12 cells.As shown in Fig.7D,TCF4,b-catenin,and Smad1 and Smad4 interacted with the responsive regions of Dlx5, Msx2, and Runx2 genes in vivo.\"\\t--\\t--\\t--\\t<2KB\\t1000', 'E_02_011\\t20878775\\t--\\tmm10\\tchr17\\t44494598\\t44494816\\tMouse\\t\"C2C12,Primary Bone MarrowCMesenchymal Stem CellLow throughput\\t\"PCR,Luciferase Reporter Assay\"\\tThe murine Dlx5 promoter from -1036 to +30 was obtained by polymerase chain reaction (PCR) and cloned in the pGL2basic vector. Murine Msx2 and Runx2 enhancers (from -3340 to -3615 for Msx2 and from -1170 to -1388 for Runx2) also were obtained by PCR and subcloned in the minimal promoter of c-fos promoter.\\tEnhancer\\tRunx2\\t--\\tLuciferase Reporter Assay\\t\"The Msx2 gene shows a conserved region 3.5 kb upstream of the transcriptional start site, which also includes a combination of a TCF/LEF1 box and both GC-rich and CAGA SBEs. This region has been shown to be necessary and sufficient to confer BMP responsiveness in vitro and in vivo.Finally, a similar enhancer is located 1 kb upstream of the P1 osteoblast-specific promoter of the Runx2 gene (Supplemental Fig. S3). To test whether these regions have the ability to render responsiveness to Wnt3a and/or BMP-2, we generated the corresponding luciferase constructs and assayed them in C2C12 cells. \"\\t\"AML3,Cbf,Cbfa-1,Cbfa1,LS3,Osf2,PEBP2aA,Pebp2a1,Pebpa2a\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Msx2,Dlx3,Dlx5,Atf4,Runx2,Sp7\"\\t\"BB122635,Hox-8,Hox8,Hox8.1,AV237891,Dlx-3,AI385752,Atf-4,C/ATF,CREB2,TAXREB67,AML3,Cbf,Cbfa-1,Cbfa1,LS3,Osf2,PEBP2aA,Pebp2a1,Pebpa2a,6430578P22Rik,C22,Osx\"\\t\"qRT-PCR,Luciferase Reporter Assay,ChIP\"\\t\"To confirm that cooperative transcriptional interactions between canonical Wnt signaling and the Smad pathway on osteogenic genes occur in vivo, ChIP assays were performed in C2C12 cells.As shown in Fig.7D,TCF4,b-catenin,and Smad1 and Smad4 interacted with the responsive regions of Dlx5, Msx2, and Runx2 genes in vivo.\"\\t--\\t--\\t--\\t<2KB\\t1279', 'E_02_012\\t20942803\\tSlc30a8 Enhancer\\tmm10\\tchr15\\t52315678\\t52316298\\tMouse\\tIslet B-Cell\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"To address this hypothesis these regions, designated intronic enhancers A and B, respectively, were isolated using PCR and ligated 5 of a heterologous thymidine kinase (TK)-luciferase fusion gene containing TK genomic sequence between ?105 and +51, relative to the transcription start site.Luciferase expression directed by these fusion genes was then analyzed by transient transfection of TC-3 cells, an islet beta cell-derived line [31], TC-6 cells, an islet alpha cell-derived line [32, 33] and HeLa cells, a cervix-derived cell line.\"\\tEnhancer\\tSlc30a8\\t--\\t\"PCR,Luciferase Reporter Assay\"\\t\"Luciferase expression directed by these fusion genes was then analyzed by transient transfection of TC-3 cells, an islet beta cell-derived line [31], TC-6 cells, an islet alpha cell-derived line [32, 33] and HeLa cells, a cervix-derived cell line. Figure 2 shows that in TC-3 cells, but not TC-6 or HeLa cells, intronic enhancer A elevated reporter gene expression beyond that driven by the TK-luciferase fusion gene alone, indicating that this region is an islet beta cell-specific enhancer. In contrast, intronic enhancer B had no effect on fusion gene expression in any of the cell lines.\"\\t\"C820002P14Rik,ZnT-8,ZnT8\"\\t--\\t--\\t--\\t--\\t--\\t--\\tPbx1\\t\"2310056B04Rik,D230003C07Rik,Pbx-1\"\\t\"ChIP,EMSA\"\\t\"Gel retardation and chromatin immunoprecipitation (ChIP) assays revealed that the islet-enriched transcription factor Pdx-1 binds Enhancer A in vitro and in situ, respectively.\"\\t--\\t--\\t--\\tIntron\\t20436', 'E_02_013\\t21056086\\t--\\tmm10\\tchr4\\t136140539\\t136140546\\tMouse\\t\"Immortalized Murine Gonadotrope L??T2 Cell,Ovarian Cancer Cell\"\\tLow throughput\\t\"Transfection,EMSA,Luciferase Reporter Assay,ChIP\"\\t\"In silico analyses suggested that the upstream enhancer in the human ID3 gene might be conserved in murine Id3. In that report, an enhancer was described upstream of the promoter region we investigated here in mouse. We obtained the reporter constructs used in Shepherd et al. (2008) and examined their BMP2 induction in LT2cells(Fig.6(A)). The data reflect the mean (+SEM) of three independent experiments and are presented relative to untreated cells transfected with the empty vector, pGL3-Basic. EMSAs were performed with the indicated radio-labeled (*) probes corresponding to ?568/?502 of the murine Id3 promoter\"\\tEnhancerId3\\t--\\tChIP\\tThe analysis of the human gene revealed a BMP response element (BRE) within a distal Enhancer (?2632/?2625) that when mutated (TGGCGCCTGGTGCT) greatly reduced the fold BMP4 response.We identified the same sequence in the murine gene (?3283/?3276) and observed through ChIP analysis that BMP2 stimulated recruitment of SMAD1 to this region of the endogenous Id3 gene in LT2 cells.\\t\"Hlh462,Idb3,bHLHb25\"\\t--\\t--\\t--\\tA more distal Enhancer was shown to mediate BMP4-induction of the human ID3 gene in ovarian cancer cells.\\tChIP\\t\"We identified the same sequence in the murine gene (?3283/?3276) and observed through ChIP analysis that BMP2 stimulated recruitment of SMAD1 to this region of the endogenous Id3 gene in L_x005f_x0002_T2 cells. Mutation of the distal BRE in the longer murine Id3 reporter greatly reduced, but did not abolish, BMP2 induction.\"\\tSmad1\\t\"AI528653,Mad1,Madh1,Madr1,Mlp1,MusMLP,dwf-A,mMad1\"\\tChIP\\t\"We identified the same sequence in the murine gene (?3283/?3276) and observed through ChIP analysis that BMP2 stimulated recruitment of SMAD1 to this region of the endogenous Id3 gene in LT2 cells. Mutation of the distal BRE in the longer murine Id3 reporter greatly reduced, but did not abolish, BMP2 induction.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3278', 'E_02_014\\t21331042\\t--\\tmm10\\tchr3\\t34563695\\t34565695\\tMouse\\tAxial Stem Cell\\tLow throughput\\t\"EMSA,Immunostaining,Transgenic mice\"\\t\"Our earlier studies have indicated that among a number of enhancers regulating Sox2, enhancer N1 is responsible for Sox2 activation in the caudally extendin neural plate5,6 (Fig.1a-c; Supplementary Fig.1).\"\\tEnhancer\\tSox2\\t--\\t\"EMSA,Immunostaining\"\\t\"Finally, we asked whether the suppression of Sox2 enhancer N1 activation by Tbx6 involves direct interaction of the Tbx6 protein with the enhancer N1 DNA sequence. Various overlapping fragments of the N1 sequence were tested for Tbx6 binding using electrophoretic mobility shift assays (EMSA). Exogenous Sox2-HA was successfully expressed in the caudal-most mesodermal compartment (Fig.4b), as indicated by HA-tag immunostaining and the mesodermal Sox2 immunostaining (Fig.4c-e). This result demonstrates that Sox2 expression is sufficient for the initiation of neural tube development from the primitive paraxial mesoderm, even in wild type embryos.\"\\t\"Sox-2,lcc,ysb\"\\t--\\t--\\t--\\t--\\t--\\t--\\tTbx6\\trv\\tEMSA\\tVarious overlapping fragments of the N1 sequence were tested for Tbx6 binding using electrophoretic mobility shift assays (EMSA)--\\t--\\t--\\t>2KB\\t\\t\\t\\t85299', 'E_02_015\\t21368045\\t--\\tmm10\\tchr14\\t67742375\\t67743103\\tMouse\\t\"Epithelial Cell,Granulosa Cells,GT1-7,GN11,Embryonic Stem Cell\"\\tLow throughput\\t\"RT-PCR,Luciferase Reporter Assay\"\\t\"Furthermore, studies in mice bearing either ?3446/+23 or ?2078/+23 bp of the mGnRH promoter fused to the LUC reporter defined a critical enhancer region for the in vivo expression of hypothalamic mGnRH (Kim et al., 2002).The quantitative RT-PCR primer sequences are available on request.\"\\tEnhancer\\tGnrh1\\t--\\tLuciferase Reporter Assay\\t\"In our previous study, using the luciferase (LUC) reporter gene, we found that the proximal ?1005 bp of the mouse GnRH (mGnRH) promoter contains the critical elements for appropriate neuronal and ovarian expression of mGnRH (Kim et al., 2002). Furthermore, studies in mice bearing either ?3446/+23 or ?2078/+23 bp of the mGnRH promoter fused to the LUC reporter defined a critical enhancer region for the in vivo expression of hypothalamic mGnRH (Kim et al., 2002).\"\\t\"Gnrh,Gnrh2,LHRH,Lhrh1,Lnrh,hpg\"\\t--\\t--\\t--\\tThis work supports the hypothesisthatthe region ofthe GnRH promoter contained between -2806 and -2078 bp acts as a cell-specific Enhancer in the GnRH neuron and as a repressor in the ovary.\\tLuciferase Reporter Assay\\t\"In this work, to further characterize the mGnRH promoter,we have constructed an additional transgenic mouse bearing a -2806/+23 bp fragment fused to the LUC reporter gene. This model demonstrates that critical hypothalamic Enhancer se_x005f_x0002_quences and ovarian repressor elements for the in vivo expression of mGnRH are located between -2806 and -2078 bp on the mGnRH promoter.\"\\t\"Oct1,Gata4\"\\t\"2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1,Gata-4\"\\tLuciferase Reporter Assay\\tOct-1 and GATA-4 were identified as important transcription factors that bind to the neuron-specific Enhancer and regulate GnRH expression in vitro.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2441', 'E_02_016\\t21454523\\tLCR\\tmm10\\tchr7\\t103823354\\t103846754\\tMouse\\t745A\\tLow throughput\\t\"ChIP,qRT-PCR,3C\"\\t\"The presence of the cohesin holo-complex was confirmed by ChIP using antibodies specific for SMC1 and SA1.Most notably, HS2 functions as an enhancer, and the outer HS5 functions as an insulator. B, adult -globin gene induction following DMSO treatment. End point RT-PCR of -major gene induction and q-RT-PCR analysis of -major and -minor genes in comparison to GAPDH were examined over 4 days of DMSO treatment. Q-RT-PCR values were normalized to ribonuclease/angiogenin inhibitor 1 (rnh1). \"\\tEnhancer\\tHbb-y\\t3C\\tChIP\\t3C and ChIP-loop analysis of the HS2 and -major promoter interaction at 4 days after DMSO treatment. The interactions of a region containing HS4 and HS5 (HS4/5) with either 3HS1 or HS-62 are compared.\\t\"Ey,Hby\"\\tHaploinsufficiency\\t--\\tD057895\\tNipped-B-like (Nipbl) bind to the locus control region (LCR) at the CTCF insulator and distal enhancer regions as well as at the specific target globin gene that undergoes activation upon differentiation\\t\"ChIP,3C,PCR\"\\t\"The results indicate that cohesin binds to these regions when they interact. Taken together, cohesin and Nipbl are recruited not only to the HS5 insulator, but also to the Enhancer and the active _x005f_x0001_-globin gene regions. This correlates with the induction of the interaction of these domains and -globin gene activation.\"\\t\"Nipbl,Ctcf,Nfe2\"\\t\"Idn3,AW108038,NF-E2,NF-E2/P45,p45,p45NFE2,p45nf-e2\"\\tChIP\\t\"Differentiation-induced Cohesin and Nipbl Binding at the -globin Locus Is Associated with LCR Enhancer-globin Gene Interactions.,ChIP analysis of cohesin (Rad21),Nipbl,CTCF, and NF-E2 binding to the -globin locus.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t16699', 'E_02_017\\t21464046\\t--\\tmm10\\tchrx\\t7954360\\t7956360\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\t\"Luciferase Reporter Assay,RT-PCR\"\\tThe 3.9 kb Gata1 hematopoietic enhancer and minigene was previously described.We have previously generated a transgenic mouse model using an Nkx2-5 cardiac specific enhancer to direct EYFP reporter expression to the developing heart.Transcriptional assay in K562 cells reveal a dose-dependent repression of luciferase activity by Nkx2-5 using the 3.9 kb erythroid enhancer.\\tEnhancer\\tGata1\\t--\\tTransfection\\t\"Utilizing transgenic technologies to isolate Nkx2-5 expressing cells, we observed an induction of the erythroid molecular program, including Gata1, in the Nkx2-5 null embryos. We demonstrated that Nkx2-5 binds to the Gata1 gene enhancer and represses the transcriptional activity of the Gata1 gene.\"\\t\"Gata-1,Gf-1,eryf1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tNkx2-5\\t\"MyoR,bHLHa22,Tbx5,Csx,Nkx-2.5,Nkx2.5,tinman\"\\tLuciferase Reporter Assay\\t\"Using transcriptional assays,we showed that one of the targets of Nkx2-5 is the Gata1 gene,an essential transcription factor in erythroid differentiation.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3899', 'E_02_018\\t21486496\\t--\\tmm10\\tchr5\\t115358083\\t115360083\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"The deleted BAC constructs were introduced into ESCs, and stable transformants were established for each BAC reporter construct. In a similar manner to the RA treatment and Luciferase reporter assay conducted with EBs (shown in Figure ?Figure2A),the Msi1 transcriptional activities were quantified using the deletion-containing reporters D1-D5 and the full-length BAC reporter D0. These results suggested that the 10-kb region from 55 kb to 65 kb upstream of the TSS might contain Msi1 transcriptional enhancers, and we named this region the \\'upstream 10-kb enhancer.\\'\"\\tEnhancer\\tMsi1\\t--\\tLuciferase Reporter Assay\\t\"We then used homologous recombination techniques to insert the ffLuc reporter gene into the Msi1 translational initiation site of the RP24-132L16 BAC (Figure1A).The ffLuc reporter gene encodes a fusion protein of the fluorescent protein Venus and firefly Luciferase.This reporter gene both visualized Msi1 transcriptional activity in vivo, and allowed us to use Luciferase bioluminescence to quantify the level of transcriptional activity.\"\\t\"Msi1h,Musahi1,m-Msi-1\"\\t--\\t--\\t--\\tThe intronic Enhancer can transactivate Msi1 gene expression with cell-type specificity markedly similar to the endogenous Msi1 expression patterns.\\tLuciferase Reporter Assay\\t\"Furthermore, when the region after the transcription site (exons and introns coding region) was deleted, the Msi1 transcriptional activity diminished even in the presence of theupstream 10-kb Enhancer (Figure 3C). These results indicated two regions responsible for Msi2 transcription.\"\\t\"Qsox1,Tfap2a\"\\t\"1300003H02Rik,QSOX,Qscn6,SOx,b2b2673Clo,AP-2,AP2alpha,Ap-2(a),Ap2,Ap2tf,Tcfap2aLuciferase Reporter Assay\\t\"Through these investigations, we have identified a new Msi1 transcription Enhancer element in NS/PCs. This region is located in the 595-bp region within the sixth intron of the Msi1 gene, and contains SOX- and AP-2-binding sites.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1', 'E_02_019\\t21487009\\tTCCR Enhancer\\tmm10\\tchr14\\t78384835\\t78446843\\tMouse\\tT Cell\\tLow throughput\\t\"ChIP-chip,3C,ChIP,Luciferase Reporter Assay,PCR\"\\t\"We used ChIP-chip analysis in mouse primary T cells and a T cell hybridoma to define the regulatory enhancers responsible for this up-regulation and to characterize their properties.Our previous ChIP-chip analyses of the mouse Rankl locus suggested that the long distance Rankl enhancers were routinely marked in osteoblasts by both basal and 1,25(OH)2D3-, PTH-, or cytokine-inducible levels of H4ac as well as increased RNA pol II density.', 'E_02_020\\t21487009\\tmRL-D5 Enhancer\\tmm10\\tchr14\\t78351421\\t78354313\\tMouse\\tT Cell\\tLow throughput\\t\"ChIP-chip,3C,ChIP,Luciferase Reporter Assay,PCR\"\\t\"We used ChIP-chip analysis in mouse primary T cells and a T cell hybridoma to define the regulatory enhancers responsible for this up-regulation and to characterize their properties.Our previous ChIP-chip analyses of the mouse Rankl locus suggested that the long distance Rankl enhancers were routinely marked in osteoblasts by both basal and 1,25(OH)2D3-, PTH-, or cytokine-inducible levels of H4ac as well as increased RNA pol II density.', 'E_02_021\\t21527504\\t--\\tmm10\\tchr14\\t67745496\\t67746000\\tMouse\\tGT1-1\\tLow throughput\\t\"Luciferase Reporter Assay,Transfection,ChIP,qRT-PCR,EMSA\"\\t\"To examine the enhancer activity of intron A, we constructed reporter vectors containing the full-length mouse intron A sequence (1003 bp) in either the forward or reversed direction followed by the simian vacuolating virus 40 (SV40) minimal promoter (SV40p) driving luciferase expression.These constructs were transfected into GnRH-expressing GT1-1 cells, and luciferase activity was then measured. Cells harboring the vectors with the full-length intron A exhibited luciferase activity significantly increased by 2- to 3-fold compared with those harboring only the SV40p (the empty vector in this study) (Fig. 1B). These findings strongly suggest that GnRH intron A contains a putative transcriptional enhancer that is likely involved in GnRH gene regulation.\"\\tEnhancer\\tGnrh1\\t--\\tLuciferase Reporter Assay\\t\"Schematic diagrams of the reporter constructs and their relative luciferase activities in comparison with those of SV40p-Ex1-Luc and SV40p-Ex1IA-315-Luc are shown in Fig. 4. Thus, these results strongly suggest that the region between 12 and 25 nt in the mouse GnRH intron A harbors a key cis-element for the enhancer function.\"\\t\"Gnrh,Gnrh2,LHRH,Lhrh1,Lnrh,hpg\"\\t--\\t--\\t--\\tClass-C SOX Transcription Factors Control GnRH Gene Expression via the Intronic Transcriptional Enhancer.\\t\"Luciferase Reporter Assay,ChIP,EMSA\"\\t\"Chromatin immunoprecipitation shows that endogenous SOX-C factors recognize and bind to the intronic Enhancer in GT1-1 cells and the hypothalamus. Taken together,these findings demonstrate that SOX-C transcription factors function as important transcriptional regulators of cell type-specific GnRH gene expression by acting on the intronic transcriptional Enhancer.\"\\t\"Sox4,Sox11,Sox12\"\\t\"AA682046,Sox-4,1110038H03Rik,6230403H02Rik,AI836553,end1,2010205A02Rik\"\"ChIP,EMSA\"\\t\"Chromatin immunoprecipitation (ChIP) assay re vealed that SOX11 binds to the Ex1-proximal Enhancer region of intron A, and overexpression of SOX11 in creases this binding in GT1-1 cells.In accor_x005f_x0002_dance with the result in the reporter assay, EMSA revealed that in vitro translated SOX4 and SOX11 proteins can recognize both SOX-binding sites in mouse intronA; the binding was completely abrogated when both sites were mutated but partially diminished in the probes with a deletion of one of them.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t568', 'E_02_022\\t21549805\\t--\\tmm10\\tchr18\\t61099404\\t61099740\\tMouse\\tB Cell\\tLow throughput\\t\"Luciferase Reporter Assay,EMSA,ChIP\"\\t\"luciferase expression was driven by FIRE sequences, and tested these constructs for their responsiveness to PAX5 expression in the RAW264 macrophage cell line and the 38B9 B-cell line using transient transfection assays.When testing the binding of PAX5 across Csf1r by ChIPassays in primary pro-B cells, we noticed that PAX5 did not just bind to the promoter, but also associated with the intronic enhancer (FIRE).This observation was confirmed by EMSA experiments, demonstrating that none of the DNA sequences present on FIRE were capable of competing for the binding of PAX5 to DNA containing a PAX5 consensus sequence.\"\\tEnhancer\\tCsf1r\\t--\\t\"Immunoprecipitation,Transfection,ChIP\"\\t\"We performed PAX5-specific chromatin immunoprecipitation analyses across the Csfr1 locus. We investigated the role of PAX5 in regulating Csf1r sense and antisense promoter activity by transient transfections and by employing a Pax5/ proLBcell line expressing an inducible PAX5 protein. When testing the binding of PAX5 across Csf1r by ChIPassays in primary pro-B cells, we noticed that PAX5 did not just bind to the promoter, but also associated with the intronic enhancer (FIRE).', 'E_02_023\\t21566115\\tIl12b Enhancer \\tmm10\\tchr11\\t44390342\\t44392251\\tMouse\\t\"Myeloid Cells,T Cell\"\\tLow throughput\\t\"Luciferase Reporter Assay,qRT-PCR\"\\tWe focused on a conserved enhancer about 10kb upstream of the Il12b start site that is targeted by TLR signaling to increase Il12b transcription in response to LPS (33). We made luciferase reporter constructs that fused the Il12b promoter to the upstream enhancer.\\tEnhancer\\tIl12b\\t--\\t\"Luciferase Reporter Assay,qRT-PCR\"\\t\"To search for transcription factors that could be involved in the suppression of Il12b transcription, we used qRT-PCR to measure the expression of transcription factor mRNAs induced by IL-10 in the presence of an inflammatory costimulus.We made luciferase reporter constructs that fused the Il12b promoter to the upstream enhancer.\"\\t\"Il-12b,Il-12p40,Il12p40,p40\"\\tAutoimmune and Inflammatory Diseases\\t--\\t--\\t--\\t--\\t--\\tNfil3\\t\"AV225605,E4BP4\"\\t\"qRT-PCR,Luciferase Reporter Assay\"\\tA NFIL3 binding site is identified in the enhancer region of Il12b.\\t--\\t--\\t--\\t>2KB\\t\\t8765', 'E_02_024\\t21652712\\t--\\tmm10\\tchr5\\t147267614\\t147267639\\tMouse\\t\"HIT,Hep G2,INS-1\"\\tLow throughput\\t\"Transfection,Luciferase Reporter Assay,EMSA,ChIP,qPCR\"\\t\"Previous studies have identified several transcriptional factors for the Pdx-1 promoter that consists of two important regulatory regions as follows: the proximal and distal enhancer regions.To delineate the sites responsible for SREBP-lc suppression o Pdx-1, we generated a series of 5\\'-deletion constructs and analyzed these reporters by transfection into HIT cells (Fig.2C). SREBP-lc-mediated inhibition was completely abolished In PDX-1 (-93)-Luc. \"\\tEnhancer\\tPdx1\\t--\\t\"Luciferase Reporter Assay,EMSA\"\\t\"SREBP-lc-mediated inhibition was completely abolished In PDX-1 (-93)-Luc. In the distal enhancer region, EMSA analysis showed that potent binding of PDX-1 to the probe containing a PDX-1 -binding site in area I (-2517 to - 2492 bp) was dose-dependently inhibited by SREBP-lc (Fig. 6\"\\t\"IDX-1,IPF-1,Ipf1,Mody4,STF-1,pdx-1\"\\tDiabetes Mellitus\\tDOID:9351\\tD003920\\t--\\t--\\t--\\tPdx1\\t\"IDX-1,IPF-1,Ipf1,Mody4,STF-1,pdx-1\"\\tEMSA\\t\"In the distal Enhancer region, EMSA analysis showed that potent binding of PDX-1 to the probe containing a PDX-1-binding site in area I (-2517 to - 2492 bp) was dose-dependently inhibited by SREBP-lc.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2503', 'E_02_025\\t21652712\\t--\\tmm10\\tchr5\\t147267670\\t147267721\\tMouse\\t\"HIT,Hep G2,INS-1\"\\tLow throughput\\t\"Transfection,Luciferase Reporter Assay,EMSA,ChIP,qPCR\"\\t\"Previous studies have identified several transcriptional factors for the Pdx-1 promoter that consists of two important regulatory regions as follows: the proximal and distal enhancer regions.To delineate the sites responsible for SREBP-lc suppression o Pdx-1, we generated a series of 5\\'-deletion constructs and analyzed these reporters by transfection into HIT cells (Fig.2C). SREBP-lc-mediated inhibition was completely abolished In PDX-1 (-93)-Luc. \"\\tEnhancer\\tPdx1\\t--\\t\"Luciferase Reporter Assay,EMSA\"\\t\"SREBP-lc-mediated inhibition was completely abolished In PDX-1 (-93)-Luc. In the distal enhancer region, EMSA analysis showed that potent binding of PDX-1 to the probe containing a PDX-1 -binding site in area I (-2517 to - 2492 bp) was dose-dependently inhibited by SREBP-lc (Fig. 6\"\\t\"IDX-1,IPF-1,Ipf1,Mody4,STF-1,pdx-1\"\\tDiabetes Mellitus\\tDOID:9351\\tD003920\\t--\\t--\\t--\\tPdx1\\t\"IDX-1,IPF-1,Ipf1,Mody4,STF-1,pdx-1\"\\tEMSA\\t\"In the distal Enhancer region,EMSA analysis showed that potent binding of PDX-1 to the probe containing a PDX-1 -binding site in area I (-2517 to - 2492 ) was dose-dependently inhibited by SRE -lc.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2434', 'E_02_026\\t21652712\\t--\\tmm10\\tchr5\\t147267701\\t147267751\\tMouse\\t\"HIT,Hep G2,INS-1\"\\tLow throughput\\t\"Transfection,Luciferase Reporter Assay,EMSA,ChIP,qPCR\"\\t\"Previous studies have identified several transcriptional factors for the Pdx-1 promoter that consists of two important regulatory regions as follows: the proximal and distal enhancer regions.To delineate the sites responsible for SREBP-lc suppression o Pdx-1, we generated a series of 5\\'-deletion constructs and analyzed these reporters by transfection into HIT cells (Fig.2C). SREBP-lc-mediated inhibition was completely abolished In PDX-1 (-93)-Luc. \"\\tEnhancer\\tPdx1\\t--\\t\"Luciferase Reporter Assay,EMSA\"\\t\"SREBP-lc-mediated inhibition was completely abolished In PDX-1 (-93)-Luc. In the distal enhancer region, EMSA analysis showed that potent binding of PDX-1 to the probe containing a PDX-1 -binding site in area I (-2517 to - 2492 bp) was dose-dependently inhibited by SREBP-lc (Fig. 6\"\\t\"IDX-1,IPF-1,Ipf1,Mody4,STF-1,pdx-1\"\\tDiabetes Mellitus\\tDOID:9351\\tD003920\\t--\\t--\\t--\\tPdx1\\t\"IDX-1,IPF-1,Ipf1,Mody4,STF-1,pdx-1\"\\tEMSA\\t\"In the distal Enhancer region,EMSA analysis showed that potent binding of PDX-1 to the probe containing a PDX-1 -binding site in area I (-2517 to - 2492 ) was dose-dependently inhibited by SRE -lc.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2404', 'E_02_027\\t21673114\\t--\\tmm10\\tchr14\\t55518040\\t55518674\\tMouse\\tHEK-293\\tLow throughput\\t\"Transfection,EMSA,PCR\"\\t\"To refine the cis-regulatory elements involved in controlling Nrl expression further, we tested the sequences within the three conserved clusters by in vivo transfection.\"\\tEnhancer\\tNrl\\t--\\tEMSA\\t\"In silico analysis using the TRANSFAC database (46) revealed a number of conserved putative transcription factor binding sites within B2(transcriptionally active sequence in clusterB)and clusterA(Fig.3,A and B, respectively). To identify the involvement of specific transcription factors, we performed EMSA using several different oligonucleotides spanning the conserved sequence elements(Fig. 4).\"\\tD14H14S46E\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Rorb,Crx,,Nrl,Nr2e3\"\\t\"Nr1f2,RZR-beta,RZRBeta,hstp,Rorb,Crx1,D14H14S46E,A930035N01Rik,PNR,RNR,rd7\"\\tEMSA\\t\"Electrophoretic mobility shift assays using mouse retinal nuclear extracts, in combination with specific antibodies,demonstrate the binding of retinoid-related orphan nuclear receptor(ROR),cone rod homeobox,orthodenticle homolog 2,and cyclic AMP response element-binding protein to predicted consensus elements within clusters A and B.Our studies demonstrate Nrl as a direct transcriptional target of ROR and suggest that combinatorial action of multiple regulatory factors modulates the expression of Nrl in developing and mature retina.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t620', 'E_02_028\\t21797989\\t--\\tmm10\\tchr7\\t19409838\\t19410044\\tMouse\\tMM14\\tLow throughput\\t\"EMSA,Immunofluorescence,ChIP\"\\tOf particular interest was a 95-bp region (+901 to +995) that was subsequently proven to exhibit the properties of a transcriptional enhancer (Figure 1).\\tEnhancer\\tCkm\\t--\\tChIP\\tThe chromatin immunoprecipitation (ChIP) primer pairs (black lines) that span the 5-enhancer sequence were used as positive controls for MyoD and myogenin binding to functional E-boxes.\\t\"CPK-Mm,M-CK,MCK,?Ckm\"\\t--\\t--\\t--\\t--\\t--\\t--\\tMef2\\t\"5430401D19Rik,9930028G15Rik,AV011172,Mef2\"\\tChIP-seq\\tMEF2 ChIP-seq occupancy at the 6.5- MCK regulatory region in differentiated C2 C12 cell shows that MEF2 is present at all three control regions.\\t--\\t--\\t--\\t<2KB\\t1152', 'E_02_029\\t21798992\\t--\\tmm10\\tchr15\\t52315678\\t52316298\\tMouse\\t\"TC-3,TC-6\"\\tLow throughput\\t\"Luciferase Reporter Assay,PCR\"\\t\"The analysisof SLC30A8-luciferase expression in bTC-3 cells revealed that the proximal promoter region, located between K6154 and K1, relativeto the translation startsite, was onlyactivein stablebut not transient transfections.To assess whether chromatin structure was essential for human SLC30A8 and mouse Slc30a8 promoter function, luciferase fusion genes containing either the human SLC30A8 sequence from K6154 to K1 or the mouse Slc30a8 sequence from K1803 to K1 were analyzed in the context of the pGL4 vector.\"\\tEnhancer\\tSlc30a8\\t--\\t\"PCR,Luciferase Reporter Assay\"\\t\"To determine whether the C16 579 and C16 954 region of intron 2 in the human SLC30A8 gene also represents a transcriptional enhancer, this region was isolated using PCR and ligated 5 0 of the K150/C3 G6PC2-luciferase fusion gene described above.\"\\t\"C820002P14Rik,ZnT-8,ZnT8\"\\tType 2 Diabetes Mellitus\\tDOID:9352\\tD003924\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t20436', 'E_02_030\\t21798992\\t--\\tmm10\\tchr15\\t5249678\\t5250298\\tMouse\\tTC-3\\tLow throughput\\tLuciferase Reporter Assay\\t\"The analysisof SLC30A8-luciferase expression in bTC-3 cells revealed that the proximal promoter region, located between K6154 and K1, relativeto the translation startsite, was onlyactivein stablebut not transient transfections.To assess whether chromatin structure was essential for human SLC30A8 and mouse Slc30a8 promoter function, luciferase fusion genes containing either the human SLC30A8 sequence from K6154 to K1 or the mouse Slc30a8 sequence from K1803 to K1 were analyzed in the context of the pGL4 vector.\"\\tEnhancer\\tSlc30a8\\t--\\t\"PCR,Luciferase Reporter Assay\"\\t\"To determine whether the C16 579 and C16 954 region of intron 2 in the human SLC30A8 gene also represents a transcriptional enhancer, this region was isolated using PCR and ligated 5 0 of the K150/C3 G6PC2-luciferase fusion gene described above.\"\\t\"C820002P14Rik,ZnT-8,ZnT8\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t47045564', 'E_02_031\\t21822303\\t--\\tmm10\\tchr3\\t34646162\\t34646551\\tMouse\\tMCF-7\\tLow throughput\\tLuciferase Reporter AssaWe transfected MCF7 and T47D cells with luciferase reporter vectors for the upstream distal enhancer and the core promoter to test whether induction of Sox2 protein was achieved through transcriptional activation of Sox2 promoter.\\tEnhancer\\tSox2\\t--\\tLuciferase Reporter AssaWe transfected MCF7 and T47D cells with luciferase reporter vectors for the upstream distal enhancer and the core promoter to test whether induction of Sox2 protein was achieved through transcriptional activation of Sox2 promoter.\\t\"Sox-2,lcc,ysb\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3637', 'E_02_032\\t21937508\\tu-PAR Enhancer\\tmm10\\tchr7\\t24461573\\t24462573\\tMouse\\tColon Cancer Cell\\tLow throughput\\t\"RT-PCR,ChIP\"\\t\"In order to gain insight into the epigenetic regulation of u-PAR expression, ChIP assays were employed in an unbiased screen to identify histone modifications or', 'E_02_033\\t22072553\\t--\\tmm10\\tchr1\\t38469166\\t38512500\\tMouse\\tBM Cell\\tLow throughput\\t\"Transfection,ChIP-chip,Luciferase Reporter Assay\"\\t\"Furthermore, a subset of these sites shows enhancer activity in transient transfection assays. ChIP-chip data were analyzed with MA2C 36 and a customized R package.These studies showed that H3 and H4 acetylation and p300/CBP binding is centered on the H/M peaks and is flanked by regions of histone H3K4me1 in a bimodal distribution characteristic of enhancer sequences (Figure 3B). 3To test for potential enhancer (or repressor activity), 22 Hoxa9/Meis1 binding sites as well as 2 control regions randomly selected in the genome where there is no Hoxa9/Meis1 binding were cloned as ~1000-bp fragments. These were inserted into the multiple cloning site of the pTAL-Lucvector,electroporated into myeloblastic K562 cells and were analyzed by Dual Luciferase Reporter Assays (Promega; supplemental Figure 3).\"\\tEnhancer\\tAff3\\t--\\tLuciferase Reporter Assay\\t\"To test for potential enhancer (or repressor activity), 22 Hoxa9/Meis1 binding sites as well as 2 control regions randomly selected in the genome where there is no Hoxa9/Meis1 binding were cloned as ~1000-bp fragments. These were inserted into the multiple cloning site of the pTAL-Lucvector,electroporated into myeloblas-', 'E_02_034\\t22264824\\t--\\tmm10\\tchr11\\t32244715\\t32246895\\tMouse\\tPrimary Erythroid Cell\\tLow+High throughput\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\tEnhancer\\tNprl3\\t--\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\t\"Aag,CGTHBA,HS-26,HS-40,Mare,Phg,Prox1,m(alpha)RE\"\\t--\\t--\\t--\\t\"Intragenic Enhancers frequently act as alternative tissue_x005f_x0002_specific promoters producing a class of abundant,spliced, multiexonic poly(A)+ RNAs (meRNAs) which reflect the host genes structure.\"\\t\"ChIP,RT-PCR,ChIP-seq\"\\t\"To address this problem, we analyzed transcrip_x005f_x0002_tion of the Nprl3 gene after deleting its constitutive promoter. This revealed that intragenic Enhancers act as highly active, alternative tissue-specific promoters for the gene containing them. \"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t13843', 'E_02_035\\t22264824\\t--\\tmm10\\tchr11\\t32249976\\t32252156\\tMouse\\tPrimary Erythroid Cell\\tLow+High throughput\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\tEnhancer\\tNprl3\\t--\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\t\"Aag,CGTHBA,HS-26,HS-40,Mare,Phg,Prox1,m(alpha)RE\"\\t--\\t--\\t--\\t\"Intragenic Enhancers frequently act as alternative tissue_x005f_x0002_specific promoters producing a class of abundant,spliced, multiexonic poly(A)+ RNAs (meRNAs) which reflect the host genes structure.\"\\t\"ChIP,RT-PCR,ChIP-seq\"\\t\"To address this problem, we analyzed transcrip_x005f_x0002_tion of the Nprl3 gene after deleting its constitutive promoter. This revealed that intragenic Enhancers act as highly active, alternative tissue-specific promoters for the gene containing them. \"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t19104', 'E_02_036\\t22264824\\t--\\tmm10\\tchr11\\t32255189\\t32257369\\tMouse\\tPrimary Erythroid Cell\\tLow+High throughput\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\tEnhancer\\tNprl3\\t--\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\t\"Aag,CGTHBA,HS-26,HS-40,Mare,Phg,Prox1,m(alpha)RE\"\\t--\\t--\\t--\\t\"Intragenic Enhancers frequently act as alternative tissue_x005f_x0002_specific promoters producing a class of abundant,spliced, multiexonic poly(A)+ RNAs (meRNAs) which reflect the host genes structure.\"\\t\"ChIP,RT-PCR,ChIP-seq\"\\t\"To address this problem, we analyzed transcrip_x005f_x0002_tion of the Nprl3 gene after deleting its constitutive promoter. This revealed that intragenic Enhancers act as highly active, alternative tissue-specific promoters for the gene containing them. \"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t24317', 'E_02_037\\t22264824\\t--\\tmm10\\tchr11\\t32268128\\t32270308\\tMouse\\tPrimary Erythroid Cell\\tLow+High throughput\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\tEnhancer\\tNprl3\\t--\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\t\"Aag,CGTHBA,HS-26,HS-40,Mare,Phg,Prox1,m(alpha)RE\"\\t--\\t--\\t--\\t\"Intragenic Enhancers frequently act as alternative tissue_x005f_x0002_specific promoters producing a class of abundant,spliced, multiexonic poly(A)+ RNAs (meRNAs) which reflect the host genes structure.\"\\t\"ChIP,RT-PCR,ChIP-seq\"\\t\"To address this problem, we analyzed transcrip_x005f_x0002_tion of the Nprl3 gene after deleting its constitutive promoter. This revealed that intragenic Enhancers act as highly active, alternative tissue-specific promoters for the gene containing them. \"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t37256', 'E_02_038\\t22264824\\tDHS-12 Enhancer\\tmm10\\tchr11\\t32264052\\t32266232\\tMouse\\tPrimary Erythroid Cell\\tLow+High throughput\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\tEnhancer\\tNprl3\\t--\\t\"RT-PCR,ChIP,ChIP-seq\"\\t\"Beyond this AFE the isoform structure appears identical to the full-length transcript. Interestingly, the AFE coincides with one of the known intragenic enhancers.First, we tested wild-type mouse tissues by RT-PCR to verify the existence of this alternative transcript. \"\\t\"Aag,CGTHBA,HS-26,HS-40,Mare,Phg,Prox1,m(alpha)RE\"\\t--\\t--\\t--\\t\"Intragenic Enhancers frequently act as alternative tissue_x005f_x0002_specific promoters producing a class of abundant,spliced, multiexonic poly(A)+ RNAs (meRNAs) which reflect the host genes structure.\"\\t\"ChIP,RT-PCR,ChIP-seq\"\\t\"To address this problem, we analyzed transcrip_x005f_x0002_tion of the Nprl3 gene after deleting its constitutive promoter. This revealed that intragenic Enhancers act as highly active, alternative tissue-specific promoters for the gene containing them. \"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t33180', 'E_02_039\\t22323542\\t--\\tmm10\\tchr6\\t70763764\\t70765090\\tMouse\\t\"S194,MPC-11\"\\tLow throughput\\t\"DNase I Hypersensitivity Assay,Luciferase Reporter Assay,Transfection\"\\t\"We identified a DNA sequence that exhibited B cell-specific, longrange interactions with Ei and E39 using chromosome conformation-capture technology.Mapping and sequence analysis of DNase I-hypersensitive site HS10.To determine whether HS10 corresponded to a new transcriptional enhancer in the locus, we performed transient-transfection assays with a luciferase reporter gene containing a Vk gene promoter (Fig. 2A, construct 1).\"\\tEnhancer\\tIgk\\t--\\tRT-PCR\\t\"Finally,to confirm the above results we performed real-timePCR assays to quantitate Igk gene transcript levels in plasma cell populations before and 5 d after immunization.\"\\tkappa\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3208791', 'E_02_040\\t22323542\\tEi Enhancer\\tmm10\\tchr6\\t70725576\\t70726090\\tMouse\\t\"S194,MPC-11\"\\tLow throughput\\t\"DNaseI-seq,Luciferase Reporter Assay,PCR\"\\t\"We identified a DNA sequence that exhibited B cell-specific, longrange interactions with Ei and E39 using chromosome conformation-capture technology.Mapping and sequence analysis of DNase I-hypersensitive site HS10.To determine whether HS10 corresponded to a new transcriptional enhancer in the locus, we performed transient-transfection assays with a luciferase reporter gene containing a Vk gene promoter (Fig. 2A, construct 1).\"\\tEnhancer\\tIgk\\t--\\tRT-PCR\\t\"Finally,to confirm the above results we performed real-timePCR assays to quantitate Igk gene transcript levels in plasma cell populations before and 5 d after immunization.\"\\tkappa\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3170197', 'E_02_041\\t22323542\\tEd Enhancer\\tmm10\\tchr6\\t70743710\\t70744950\\tMouse\\t\"S194,MPC-11\"\\tLow throughput\\t\"DNaseI-seq,Luciferase Reporter Assay,PCR\"\\t\"We identified a DNA sequence that exhibited B cell-specific, longrange interactions with Ei and E39 using chromosome conformation-capture technology.Mapping and sequence analysis of DNase I-hypersensitive site HS10.To determine whether HS10 corresponded to a new transcriptional enhancer in the locus, we performed transient-transfection assays with a luciferase reporter gene containing a Vk gene promoter (Fig. 2A, construct 1).\"\\tEnhancer\\tIgk\\t--\\tRT-PCR\\t\"Finally,to confirm the above results we performed real-timePCR assays to quantitate Igk gene transcript levels in plasma cell populations before and 5 d after immunization.\"\\tkappa\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3188694', 'E_02_042\\t22323542\\tE3\\' Enhancer\\tmm10\\tchr6\\t70735256\\t70736064\\tMouse\\t\"S194,MPC-11\"\\tLow throughput\\t\"DNaseI-seq,Luciferase Reporter Assay,PCR\"\\t\"We identified a DNA sequence that exhibited B cell-specific, longrange interactions with Ei and E39 using chromosome conformation-capture technology.Mapping and sequence analysis of DNase I-hypersensitive site HS10.To determine whether HS10 corresponded to a new transcriptional enhancer in the locus, we performed transient-transfection assays with a luciferase reporter gene containing a Vk gene promoter (Fig. 2A, construct 1).\"\\tEnhancer\\tIgk\\t--\\tRT-PCR\\t\"Finally,to confirm the above results we performed real-timePCR assays to quantitate Igk gene transcript levels in plasma cell populations before and 5 d after immunization.\"\\tkappa\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3180024', 'E_02_043\\t22378757\\t--\\tmm10\\tchr10\\t62128342\\t62130342\\tMouse\\tGerm Cell\\tLow throughput\\t\"ChIP,PCR,Luciferase Reporter Assay\"\\t\"To investigate this further, we used a ChIP method to determine whether STAT3 binds Neurog3 regulatory elements in THY1+ spermatogonia. PCR analysis', 'E_02_044\\t22396734\\t--\\tmm10\\tchr4\\t115100426\\t115102426\\tMouse\\t416B\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,EMSA\"\\t\"To investigate whether the +45 region functions as a transcriptional enhancer, luciferase reporter constructs were generated using a 1.76 kb genomic fragment encompassing the complete +45 element and their activity was assessed in 416B cells hematopoietic progenitor cells (Figure 1B).\"\\tEnhancer\\tTal1\\t--\\t\"DNaseI-seq,EMSA,ChIP\"\\t\"Here we report identification of a DNaseI hypersensitive site at the 39 end of the Scl/Map17 domain and 45 kb downstream of the Scl transcription start site.To investigate whether CTCF binding is responsible for the footprinting results observed in Figure 3A, we performed electrophoretic mobility shift assay (EMSA) and ChIP-on-chip experiments.\"\\t\"Hpt,SCL/tal-1,Scl,bHLHa17,tal-1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tCtcf\\tAW108038\\tChIP-seq\\tAnalysis of publicly available CTCF ChIP-seq data from the ENCODE consortium clearly demonstrates CTCF binding to the +45 element across a wide range of mouse tissues consistent with the presence of a ubiquitous CTCF site.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t45001', 'E_02_045\\t22401818\\tScl +19 Enhancer\\tmm10\\tchr4\\t115074426\\t115076426\\tMouse\\t\"Hematopoietic Stem Cell,Progenitor Cell\"\\tLow throughput\\t\"RT-PCR,qPCR,ChIP\"\\t\"Real-time semi-quantitative PCR performed with primers specific for the Scl gene [16] showed that expression levels of Scl in erythroid (Ter119 ? ), myeloid (Mac-1 ? ), megakaryocytic (CD41 ? ), and T-cell (CD4 ? ) lineages were similar for WTand Scl D19/D19 mice (Fig. 1D). To determine whether deletion of the Scl +19', 'E_02_046\\t22701669\\tCONS3.8 Enhancer\\tmm10\\tchr14\\t47056226\\t47056692\\tMouse\\tLS-8 Cells\\tLow throughput\\t\"EMSA,Phylogenetic Footprinting\"\\tIn vitro electrophoretic mobility shift assays (EMSA) and in vivo transgenic reporter mutational analyses revealed that this site supports Pitx binding and that the site is necessary to recapitulate aspects of endogenous Bmp4 expression in developing craniofacial and limb tissues.\\tEnhancer\\tBmp4\\t--\\t\"ChIP,PCR\"\\t\"Pitx2 chromatin immunoprecipitation (ChIP) demonstrate direct binding of Pitx2 to this Bmp4 enhancer site in a dental epithelial cell line.Using primers that target the Pitx1/2 binding motif in the 396 bp Bmp4 enhancer sequence (CONS3.8), DNA purified from crosslinked LS8 chromatin after immunoprecipitation with an anti-Pitx2 antibody yielded a 4.7-fold increase in amplicon abundance, relative to an IgG control, by PCR and qPCR (Figure 6A, B).\"\\t\"Bmp-4,Bmp2b,Bmp2b-1,Bmp2b1\"\\t--\\t--\\t--\\tPitx2 ChIP demonstrated direct binding of Pitx2 to this Bmp4 enhancer site in a dental epithelial cell line.\\t\"ChIP,PCR,qPCR\"\\t\"Using primers that target the Pitx1/2 binding motif in the 396 bp Bmp4 Enhancer sequence (CONS3.8), DNA purified from crosslinked LS8 chromatin after immunoprecipitation with an anti-Pitx2 antibody yielded a 4.7-fold increase in amplicon abundance, relative to an IgG control, by PCR and qPCR.\"\\tPitx2\\t\"9430085M16Rik,Brx1,Brx1b,Munc30,Otlx2,Ptx2,Rieg\"\\t\"EMSA,ChIP\"\\t\"Electrophoretic Mobility Shift Assay (EMSA) exhibits robust binding of Pitx1 protein to both a positive control bicoid DNA sequence and to the consensus Pitx1/2 binding site in with a 25 probe sequence in the CONS3.8 sequenceTo determine whether Pitx2 physically binds the CONS3.8 Bmp4 Enhancer in living cells,we performed Chromatin Immunoprecipitation (ChIP) assays in LS8 mouse dental epithelial cells.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t672935', 'E_02_047\\t22728936\\tScn5a Enhancer\\tmm10\\tchr9\\t119469800\\t119469811\\tMouse\\tHL-1\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"We tested the ability of this putative enhancer to activate transcription in a TBX5-dependent manner. TBX5 dramatically upregulated enhancer-dependent luciferase reporter expression from this genomic region (chr9:119,378,051C119,379,479;NCBI build 37/mm9) in vitro (Figure 6B). \"\\tEnhancer\\tScn5a\\t--\\tLuciferase Reporter Assay\\t\"We tested the ability of this putative enhancer to activate transcription in a TBX5-dependent manner. TBX5 dramatically upregulated enhancer-dependent luciferase reporter expression from this genomic region (chr9:119,378,051C119,379,479;NCBI build 37/mm9) in vitro (Figure 6B). \"\\t\"Nav1.5,Nav1.5c,SkM1,SkM2,mH1\"\\tCardiac Conduction System DiseasDOID:10273\\tD000075224\\t\"TBX5-responsive Enhancer downstream of Scn5a sufficient to drive VCS expression in vivo, dependent on canonical T-box binding sites. \"\\tChIP-seq\\t\"Here, we found a direct molecular link between TBX5 and Scn5a via a TBX5-respon_x005f_x0002_sive downstream Enhancer that was sufficient to direct VCS-specif-ic gene expression. Our results identified a TBX5-Scn5a molecular network essential for function of the mature VCS. \"\\tTbx5\\tTbx5\\tChIP-seq\\tWe bioinformatically interrogated the Scn5a locus to identify potential TBX5-responsive Enhancers using the overlap of 4 independent data sets: (a) evolutionary conservation (21);(b) ChIP-seq studies identifying TBX5 binding sites in the atrial cardiomyocyte HL-1 cell line.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t13597', 'E_02_048\\t22728936\\tScn5a Enhancer\\tmm10\\tchr9\\t119469879\\t119469890\\tMouse\\tHL-1\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"We tested the ability of this putative enhancer to activate transcription in a TBX5-dependent manner. TBX5 dramatically upregulated enhancer-dependent luciferase reporter expression from this genomic region (chr9:119,378,051C119,379,479;NCBI build 37/mm9) in vitro (Figure 6B). \"\\tEnhancer\\tScn5a\\t--\\tLuciferase Reporter Assay\\t\"We tested the ability of this putative enhancer to activate transcription in a TBX5-dependent manner. TBX5 dramatically upregulated enhancer-dependent luciferase reporter expression from this genomic region (chr9:119,378,051C119,379,479;NCBI build 37/mm9) in vitro (Figure 6B). \"\\t\"Nav1.5,Nav1.5c,SkM1,SkM2,mH1\"\\tCardiac Conduction System DiseasDOID:10273\\tD000075224\\t\"TBX5-responsive Enhancer downstream of Scn5a sufficient to drive VCS expression in vivo, dependent on canonical T-box binding sites. \"\\tChIP-seq\\t\"Here, we found a direct molecular link between TBX5 and Scn5a via a TBX5-respon_x005f_x0002_sive downstream Enhancer that was sufficient to direct VCS-specif-ic gene expression. Our results identified a TBX5-Scn5a molecular network essential for function of the mature VCS. \"\\tTbx5\\tTbx5\\tChIP-seq\\tWe bioinformatically interrogated the Scn5a locus to identify potential TBX5-responsive Enhancers using the overlap of 4 independent data sets: (a) evolutionary conservation (21);(b) ChIP-seq studies identifying TBX5 binding sites in the atrial cardiomyocyte HL-1 cell line.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t13518', 'E_02_049\\t22728936\\tScn5a Enhancer\\tmm10\\tchr9\\t119469902\\t119469913\\tMouse\\tHL-1\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"We tested the ability of this putative enhancer to activate transcription in a TBX5-dependent manner. TBX5 dramatically upregulated enhancer-dependent luciferase reporter expression from this genomic region (chr9:119,378,051C119,379,479;NCBI build 37/mm9) in vitro (Figure 6B). \"\\tEnhancer\\tScn5a\\t--\\tLuciferase Reporter Assay\\t\"We tested the ability of this putative enhancer to activate transcription in a TBX5-dependent manner. TBX5 dramatically upregulated enhancer-dependent luciferase reporter expression from this genomic region (chr9:119,378,051C119,379,479;NCBI build 37/mm9) in vitro (Figure 6B). \"\\t\"Nav1.5,Nav1.5c,SkM1,SkM2,mH1\"\\tCardiac Conduction System DiseasDOID:10273\\tD000075224\\t\"TBX5-responsive Enhancer downstream of Scn5a sufficient to drive VCS expression in vivo, dependent on canonical T-box binding sites. \"\\tChIP-seq\\t\"Here, we found a direct molecular link between TBX5 and Scn5a via a TBX5-respon_x005f_x0002_sive downstream Enhancer that was sufficient to direct VCS-specif-ic gene expression. Our results identified a TBX5-Scn5a molecular network essential for function of the mature VCS. \"\\tTbx5\\tTbx5\\tChIP-seq\\tWe bioinformatically interrogated the Scn5a locus to identify potential TBX5-responsive Enhancers using the overlap of 4 independent data sets: (a) evolutionary conservation (21);(b) ChIP-seq studies identifying TBX5 binding sites in the atrial cardiomyocyte HL-1 cell line.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t13495', 'E_02_050\\t22761896\\t--\\tmm10\\tchr14\\t67741735\\t67744661\\tMouse\\tGT1-7\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,PCR\"\\t\"the luciferase activities in GT1C7 cells were noticeably decreased without HDACIs treatments, suggesting that potential enhancer elements located in the regions (-3446,-2087 bp and -1134,-520 bp). \"\\tEnhancer\\tGnrh1\\t--\\t\"Western blot,Immunofluorescence,ChIP,EMSA\"\\t\"The Otx2 protein levels were also obviously reduced in GT1C7 cells showed by western blot analysis (Figure 5B) and immunofluorescence staining (Figure 5C).The results of ChIP assay showed that the binding ability of Otx2 to neuron-specific elements (2356,2249 bp) in Gnrh1 promoter obviously decreased after HDACIs treatments (Figure 6B).To further analyze the affinities of Otx2 to the two conserved binding sites in mouse Gnrh1 promoter respectively, EMSA was performed with probes containing the 2268,2239 bp and 2330,2301 bp sequences [20](Figure 6C, above).\"\\t\"Gnrh,Gnrh2,LHRH,Lhrh1,Lnrh,hpg\"\\t--\\t--\\t--\\t--\\t--\\t--\\tOtx2\\tE130306E05Rik\\t\"ChIP,EMSA\"\\tChromatin immunoprecipitation and electrophoretic mobility shift assays demonstrated that the DNA-binding activity of Otx2 to Gnrh1 gene was suppressed by TSA and VPA.\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1982', 'E_02_051\\t22877652\\tGnRH-E1 Enhancer\\tmm10\\tchr14\\t67743318\\t67743610\\tMouse\\tGT1-7\\tLow throughput\\t\"ChIP,PCR,Luciferase Reporter Assay,EMSA\"\\t\"To determine whether AR interacts with endogenous chromatin in the GnRH-E1 region in GT1-7 cells, ChIP assays were performed. GT1-7 cells were transiently', 'E_02_052\\t22877652\\tGnRH-E2 Enhancer\\tmm10\\tchr14\\t67742046\\t67742550\\tMouse\\tGT1-7\\tLow throughput\\t\"ChIP,PCR,Luciferase Reporter Assay,EMSA\"\\t\"To determine whether AR interacts with endogenous chromatin in the GnRH-E1 region in GT1-7 cells, ChIP assays were performed. GT1-7 cells were transiently', 'E_02_053\\t22877652\\tGnRH-E3 Enhancer\\tmm10\\tchr14\\t67740982\\t67741286\\tMouse\\tGT1-7\\tLow throughput\\t\"ChIP,PCR,Luciferase Reporter Assay,EMSA\"\\t\"To determine whether AR interacts with endogenous chromatin in the GnRH-E1 region in GT1-7 cells, ChIP assays were performed. GT1-7 cells were transiently', 'E_02_054\\t23024062\\t--\\tmm10\\tchr1\\t60908875\\t60908895\\tMouse\\tT Cell\\tLow throughput\\tLuciferase Reporter Assa\"T cells were stimulated with CPT after transfection of CTLA4 luciferase reporter constructs (including -1221, -343, -238, -167, and-63 bp relative to the CTLA4 transcription start site). suggesting the presence of an enhancer binding site in this region. \"\\tEnhancer\\tCtla4\\t--\\tChIP\\t\"The promoter region from 2150 to 2130 bp contains a putative CREB binding site, a well known downstream transcription factor of cAMP signaling. To define whether the putative CREB binding site (2150 to 2143 bp) is responsive to cAMP, we performed ChIP assay with CREB antibody in T cells, with or without CPT treatment.\"\\t\"Cd152,Ctla-4,Ly-56\"\\t--\\t--\\t--\\t--\\t--\\t--\\tCreb1\\t\"2310001E10Rik,3526402H21Rik,AV083133,Creb,Creb-1\"\\tChIP\\t\"The promoter region from -150 to -130 bp contains a putative CREB binding site, a well known downstream transcription factor of cAMP signaling.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t139', 'E_02_055\\t23154183\\tIl12b Enhancer\\tmm10\\tchr11\\t44390063\\t44390123\\tMouse\\tHEK-293\\tLow throughput\\t\"ChIP,Luciferase Reporter Assay\"\\t\"We then examined by ChIP assay the binding of IRFs and other transcription factors to the enhancer region in mouse peritoneal macrophages following stimulation with either LPS for TLR4 activation or poly(I:C) for RLR activation. Interestingly,TLR4 stimulation resulted in the recruitment of IRF5, C/EBPb and Oct-1/-2 transcription factors to the enhancer.\"\\tEnhancer\\tIl12b\\t--\\tTransfection\\t\"The above observations suggest that, similar to the Il12b promoter, TLR-activated IRF5 also activates the enhancer through binding to the ISREs, whereas the binding of the RLR-activated IRF3 suppresses the IRF5-mediated activation. To test this, we carried out a transient reporter assay.\"\\t\"Il-12b,Il-12p40,Il12p40,p40\"\\t--\\t--\\t--\\t--\\t--\\t--\\tIrf4\\t\"LSIRF,MUM1,NF-EM5,SHEP8\"\\tChIP\\t\"We then examined by ChIP assay the bindingof IRFs and other transcription factors to the Enhancer region in mouse peritoneal macrophages following stimulation with either LPS for TLR4 activation or poly(I:C) for RLR activation. Interestingly,TLR4 stimulation resulted in the recruitment of IRF5, C/EBPb and Oct-1/- transcription factors to the Enhancer.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t9969', 'E_02_056\\t23307821\\t--\\tmm10\\tchr17\\t26829229\\t26831311\\tMouse\\tCardiac Progenitor Cell\\tLow throughput\\t\"ChIP,PCR,Luciferase Reporter Assay,EMSA\"\\t\"To examine in detail the potential binding of YY1 to Nkx2.5 enhancer, we performed an electromobility shift assay (EMSA) using the 2.1 kb cardiac enhancer located within -9435 and -7353 bases upstream from the murine NKX2.5 transcriptional start site.Binding of site #1 to YY1 was further confirmed using a DNA supershift assay (Figure 2C). To corroborate these in vitro data with their binding to the Nkx2.5 cardiac enhancer in vivo , we performed chromatin immunoprecipitation (ChIP) and PCR analysis using anti-YY1 antibodies on day 6 differentiated NK ESCs (Figure 2D), FACS-purified eGFP+ CPCs and eGFP- cells from day 6 differentiated NK ESCs (Figure 2E), and from embryonic day 9.5 hearts and body.', 'E_02_057\\t23321477\\t--\\tmm10\\tchr6\\t122726952\\t122727236\\tMouse\\tN2a\\tLow throughput\\t\"Transfection,Luciferase Reporter Assay,PCR,EMSA,ChIP\"\\tTransient transfection assays in N2A neuroblasts using murine glut3-luciferase reporter constructs mapped the hypoxia-induced enhancer activities to -857- to -573-bp and -203- to -177-bp regions.\\tEnhancer\\tSlc2a3\\t--\\tChIP\\t\"Hypoxia-exposed N2A nuclear extracts demonstrated an increase in HIF-1a and p-Creb binding to HRE (-828 to -824 bp) and AP-1 (-187 to -180 bp) cis-elements, respectively, in electromobility shift and supershift assays, which was confirmed by chromatin immunoprecipitation assays.\"\\t\"AA408729,AL023014,AL024341,AU040424,C78366,Glut-3,Glut3\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Hif1a,Creb1\"\\t\"AA959795,HIF-1-alpha,HIF1-alpha,HIF1alpha,MOP1,bHLHe78,2310001E10Rik,3526402H21Rik,AV083133,Creb,Creb-1\"\\t\"ChIP,EMSA\"\\t\"Hypoxia-exposed N2A nuclear extracts demonstrated an increase in HIF-1 and p-Creb binding to HRE (-828 to -824 bp) and AP-1 (-187 to -180 bp) cis-elements, respectively, in electromobility shift and supershift assays, which was confirmed by chromatin immunoprecipitation assays.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t714', 'E_02_058\\t23321477\\t--\\tmm10\\tchr6\\t122727606\\t122727632\\tMouse\\tN2a\\tLow throughput\\t\"Transfection,Luciferase Reporter Assay,PCR,EMSA,ChIP\"\\tTransient transfection assays in N2A neuroblasts using murine glut3-luciferase reporter constructs mapped the hypoxia-induced enhancer activities to -857- to -573-bp and -203- to -177-bp regions.\\tEnhancer\\tSlc2a3\\t--\\tChIP\\t\"Hypoxia-exposed N2A nuclear extracts demonstrated an increase in HIF-1a and p-Creb binding to HRE (-828 to -824 bp) and AP-1 (-187 to -180 bp) cis-elements, respectively, in electromobility shift and supershift assays, which was confirmed by chromatin immunoprecipitation assays.\"\\t\"AA408729,AL023014,AL024341,AU040424,C78366,Glut-3,Glut3\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Hif1a,Creb1\"\\t\"AA959795,HIF-1-alpha,HIF1-alpha,HIF1alpha,MOP1,bHLHe78,2310001E10Rik,3526402H21Rik,AV083133,Creb,Creb-1\"\\t\"ChIP,EMSA\"\\t\"Hypoxia-exposed N2A nuclear extracts demonstrated an increase in HIF-1 and p-Creb binding to HRE (-828 to -824 bp) and AP-1 (-187 to -180 bp) cis-elements, respectively, in electromobility shift and supershift assays, which was confirmed by chromatin immunoprecipitation assays.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t189', 'E_02_059\\t23642367\\tTal1 +40k Enhancer\\tmm10\\tchr4\\t115095426\\t115097426\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\t\"ChIP,EMSA\"\\t\"Several Tal1 enhancers have been identified, but only the +40k enhancer region contains E-box motifs (Ogilvy et al., 2007), potential binding sites for Mesp1 (Figure 3B, left panel).\"\\tEnhancer\\tTal1\\t--\\t\"ChIP,EMSA\"\\t\"We performed ChIP in day 3 EBs after a prior 24 hr Mesp1 pulse and found that Mesp1 was enriched at this cis -regulatory element (Figure 3B, right panel) with a concurrent upregulation of Tal1 expression (Supplementary Figure S3B).We next performed electrophoretic mobility shift assays to confirm the direct interaction between Mesp1 and the Tal1 +40k Enhancer.\"\\t\"Hpt,SCL/tal-1,Scl,bHLHa17,tal-1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tNkx2-5\\t\"Csx,Nkx-2.5,Nkx2.5,tinman\"\\tRT-PCR\\t\"At the protein level,the late pulse of Mesp1 elevated the expression of Nkx2.5,a cardiogenic transcription factor,and downstream markers,cTnT,cTnI and sarcomeric -actinin.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t40001', 'E_02_060\\t23993744\\tCE Enhancer\\tmm10\\tchr7\\t46352720\\t46353957\\tMouse\\tC2C12\\tLow+High throughput\\t\"siRNA,ChIP-qPCR,qRT-PCR,ChIP-seq,DNaseI-seq\"\\t\"To this end, we devised a screening approach where ten different small interfering RNA duplexes (siRNA) were designed to target various regions upstream of MYOD1 with proximity to MyoD+/MyoG+/H3K4me1+ peaks (Figure 3A, #1C10). As controls, siRNAs were designed to target four eRNA+ regions on other chromosomes (Figure S3) and another against green fluorescent protein (GFPi). Transfected C2C12 cells were placed in differentiation media (DM) for 4C6hrs and harvested thereafter for analysis. \"\\tEnhancer\\tMyod\\t--\\t\"ChIP-seq,RNA-seq\"\\t\"Within this region, the concerted activity of two DNA Enhancer elements, CE and DRR, specifies the level of MyoD expression in the myogenic lineage.In this context, we surveyed for the enhancer signature (i.e. H3K4me1) within ~1C2kb of MyoG+ sites. We obtained and analyzed ChIP-Seq data for H3K4me1 (Table S1) and observed the majority (~80%) of the extragenic MyoG+ peaks within H3K4me1+ domains (Figure 1BCD).To ascertain whether these regions were transcriptionally active, we performed paired-end RNA-Seq (PE-Seq)from ribosome-depleted RNA fraction from MT (76bp/end, see Table S1 for number of reads). \"\\t\"AI503393,MYF3,MyoD,?Myod-1,bHLHc1\"\\t--\\t--\\t--\\tThe eRNAs regulate genomic access of the transcriptional complex to defined regulatory regions.\\tChIP\\t\"The above findings prompted us to evaluate whether RNA-assisted PolII recruitment could also explain the effect of DRRRNA on MyoG expression (Figure 4A,B). Indeed, ChIP experiments revealed that DRRi resulted in the reduction of PolII at MYOG, but not at MYOD1. \"\\t\"Mef2a,Foxo1,Myod1,Myog\"\\t\"A430079H05Rik,AI876417,Afxh,FKHR,Fkhr1a,Foxo1,MYF4,bHLHc3,myo,AI503393,MYF3,MyoD,Myod-1,bHLHc1\"\\t\"qRT-PCR,ChIP-seq\"\\t\"Similar to other nuclear eRNAs examined (FigureS1GCI), DRRRNA levels were significantly reduced following MyoDi and to a lesser extent by MyoGi (Figure 2D), suggesting that its transcriptional regulation is under the concerted control of these transcription factors.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t23134', 'E_02_061\\t23993744\\tDRR Enhancer\\tmm10\\tchr7\\t46370375\\t46374302\\tMouse\\tC2C12\\tLow+High throughput\\t\"siRNA,ChIP-qPCR,qRT-PCR,ChIP-seq,DNaseI-seq\"\\t\"To this end, we devised a screening approach where ten different small interfering RNA duplexes (siRNA) were designed to target various regions upstream of MYOD1 with proximity to MyoD+/MyoG+/H3K4me1+ peaks (Figure 3A, #1C10). As controls, siRNAs were designed to target four eRNA+ regions on other chromosomes (Figure S3) and another against green fluorescent protein (GFPi). Transfected C2C12 cells were placed in differentiation media (DM) for 4C6hrs and harvested thereafter for analysis. \"\\tEnhancer\\tMyod\\t--\\t\"ChIP-seq,RNA-seq\"\\t\"Within this region, the concerted activity of two DNA Enhancer elements, CE and DRR, specifies the level of MyoD expression in the myogenic lineage.In this context, we surveyed for the enhancer signature (i.e. H3K4me1) within ~1C2kb of MyoG+ sites. We obtained and analyzed ChIP-Seq data for H3K4me1 (Table S1) and observed the majority (~80%) of the extragenic MyoG+ peaks within H3K4me1+ domains (Figure 1BCD).To ascertain whether these regions were transcriptionally active, we performed paired-end RNA-Seq (PE-Seq)from ribosome-depleted RNA fraction from MT (76bp/end, see Table S1 for number of reads). \"\\t\"AI503393,MYF3,MyoD,?Myod-1,bHLHc1\"\\t--\\t--\\t--\\tThe eRNAs regulate genomic access of the transcriptional complex to defined regulatory regions.\\tChIP\\t\"The above findings prompted us to evaluate whether RNA-assisted PolII recruitment could also explain the effect of DRRRNA on MyoG expression (Figure 4A,B). Indeed, ChIP experiments revealed that DRRi resulted in the reduction of PolII at MYOG, but not at MYOD1. \"\\t\"Mef2a,Foxo1,Myod1,Myog\"\\t\"A430079H05Rik,AI876417,Afxh,FKHR,Fkhr1a,Foxo1,MYF4,bHLHc3,myo,AI503393,MYF3,MyoD,Myod-1,bHLHc1\"\\t\"qRT-PCR,ChIP-seq\"\\t\"Similar to other nuclear eRNAs examined (FigureS1GCI), DRRRNA levels were significantly reduced following MyoDi and to a lesser extent by MyoGi (Figure 2D), suggesting that its transcriptional regulation is under the concerted control of these transcription factors.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t4134', 'E_02_062\\t24000349\\t--\\tmm10\\tchr17\\t40206839\\t40206900\\tMouse\\t\"Somatic Spleen Cells,Spermatogenic Germ Cells\"\\tLow throughput\\t\"ChIP,qRT-PCR\"\\t\"This reflects the fact that both the human and mouse Pgk2 gene promoters contain highly conserved regulatory elements including a GC-box, a CAAT-box and an enhancer region just upstream from the CAAT-box.\"\\tEnhancer\\tPgk2\\t--\\t\"ChIP,qRT-PCR\"\\t\"Our results indicate there is an ordered series of molecular events that lead to activation of transcription of the Pgk2 gene in spermatogenic cells, and that these events are signaled by distinct elements in the Pgk2 promoter/Enhancer region.\"\\t\"Pgk-2,Tcp-2\"\\t--\\t--\\t--\\tActivation of Pgk2 transcription is regulated by testis-specific demethylation of DNA and binding of testis-specific transcription factors to Enhancer and core promoter elements. \\t\"ChIP,qRT-PCR\"\\t\"Our results indicate there is an ordered series of molecular events that lead to activation of transcription of the Pgk2 gene in spermatogenic cells, and that these events are signaled by distinct elements in the Pgk2 promoter/Enhancer region.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t147', 'E_02_063\\t24014243\\t--\\tmm10\\tchr5\\t107680655\\t107682655\\tMouse\\t\"Hematopoietic Cell,B Cell,T Cell\"\\tLow throughput\\t\"ChIP,PCR,Luciferase Reporter Assay\"\\t\"To investigate whether Mysm1 regulates Gfi1 expression by binding to the regulatory elements of Gfi1 in HSC, we performed chromatin immunoprecipitation (ChIP) assays along the Gfi1 regulatory elements. A panel of PCR primers to encompass the 23.4-kb promoter and 235-kb enhancer of the Gfi1 locus was designed (Figure 5A).\"\\tEnhancer\\tGfi1\\t--\\t\"ChIP,PCR\"\\t\"To investigate whether Mysm1 regulates Gfi1 expression by binding to the regulatory elements of Gfi1 in HSC, we performed chromatin immunoprecipitation (ChIP) assays along the Gfi1 regulatory elements. A panel of PCR primers to encompass the 23.4-kb promoter and 235-kb enhancer of the Gfi1 locus was designed (Figure 5A).\"\\t\"AW495828,Gfi-1,Pal-1,Pal1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Gata2,Runx1\"\\t\"Gata-2,AML1,CBF-alpha-2,Cbfa2,Pebp2a2,Pebpa2b\"\\t\"Luciferase Reporter Assay,ChIP,PCR\"\\tMechanistic studies revealed that Mysm1 modulates histone modifications and directs the recruitment of key transcriptional factors such as Gata2 and Runx1 to the Gfi1 locus in HSCs. We found that Mysm1 directly associates with the Gfi1 Enhancer element and promotes its transcription through Gata2 and Runx1 transactivation.\\t--\\t--\\t--\\t>2KB\\t35000', 'E_02_064\\t24040411\\tECR18 Enhancer\\tmm10\\tchr7\\t6502892\\t6504892\\tMouse\\t\"HeLa,HEK-293\"\\tLow+High throughput\\t\"ChIP-seq,3C,Luciferase Reporter Assay,PCR,ChIP\"\\t\"In vitro assays also reveal ECR18 as a potential enhancer or repressor for the promoter of Peg3.The identified ECRs were also examined in terms of their histone modifications using the ChIP-seq data set of the ENCODE project.To further characterize the predicted functions of the ECRs, we performed the Chromatin Conformation Capture (3C) approach with the tissues derived from mouse.', 'E_02_065\\t24157522\\t--\\tmm10\\tchr5\\t28476034\\t28479024\\tMouse\\tColon Adenocarcinoma Cell Lines\\tLow+High throughput\\t\"RT-PCR,ChIP-seq,Transient Transfection Assay,Luciferase Reporter Assay,3C,DNaseI-seq\"\\t\"The ChIP-seq data of the ENCODE project show that H3K4me1, but not H3K4me3, marks all four DNaseI HS sites in the adult lung.To examine whether each of the DNase I HS sites is responsible for upregulating the expression of Shh, we employed a transgenic reporter assay using BAC clones (Figs. 1A and 3A).\"\\tEnhancer\\tShh\\t3C\\tLuciferase Reporter Assay\\t\"To determine whether SLGE physically interacts with the Shh promoter in association with transcriptional activity, we conducted a 3C assay, which has been used to measure associations between two genomic regions. Assessment of the functional properties of SLGE by 3C and luciferase assays.\"\\t\"9530036O11Rik,Dsh,Hhg1,Hx,Hxl3,M100081NC,Shh\"\\t--\\t--\\t--\\t\"Shh expression in the endodermal epithelial lining, which is enabled by the coordinated actions of the Enhancers, is necessary for proper endodermal development.\"\\t\"RT-PCR,Luciferase Reporter Assay,3C\"\\t\"We identified a novel, less-conserved cis-regulatory element for Shh expres_x005f_x0002_sion in the esophageal epithelium, as well as in the lung and intestinal epithelium, in a region 100 Kb upstream of the Shh CDS. The new element compensates for the Enhancer activi_x0002_ties of the three previously identified Enhancers, and the combined activities of all four Enhancers permit the seamless Shh expression in the epithelial linings of the whole endoder_x0002_mal tissues in the mouse embryo.\"\\t\"Rbfox3,Socs1\"\\t\"Fox-3,Hrnbp3,NeuN,Neuna60,Cish1,Cish3,JAB,SOCS-1,SSI-1\"\\tDNaseI-seq\\t\"Putative binding sites for Fox transcription factors and STAT are found in the SLGE.In human cancer cell lines, including Caco-2, putative Fox- and STAT-binding sites are found in some of the DNase I HS sites around the SHH CDS.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t20690', 'E_02_066\\t24157522\\t--\\tmm10\\tchr5\\t28495262\\t28497421\\tMouse\\tColon Adenocarcinoma Cell Lines\\tLow+High throughput\\t\"RT-PCR,ChIP-seq,Transient Transfection Assay,Luciferase Reporter Assay,3C,DNaseI-seq\"\\t\"The ChIP-seq data of the ENCODE project show that H3K4me1, but not H3K4me3, marks all four DNaseI HS sites in the adult lung.To examine whether each of the DNase I HS sites is responsible for upregulating the expression of Shh, we employed a transgenic reporter assay using BAC clones (Figs. 1A and 3A).\"\\tEnhancer\\tShh\\t3C\\tLuciferase Reporter Assay\\t\"To determine whether SLGE physically interacts with the Shh promoter in association with transcriptional activity, we conducted a 3C assay, which has been used to measure associations between two genomic regions. Assessment of the functional properties of SLGE by 3C and luciferase assays.\"\\t\"9530036O11Rik,Dsh,Hhg1,Hx,Hxl3,M100081NC,Shh\"\\t--\\t--\\t--\\t\"Shh expression in the endodermal epithelial lining, which is enabled by the coordinated actions of the Enhancers, is necessary for proper endodermal development.\"\\t\"RT-PCR,Luciferase Reporter Assay,3C\"\\t\"We identified a novel, less-conserved cis-regulatory element for Shh expres_x005f_x0002_sion in the esophageal epithelium, as well as in the lung and intestinal epithelium, in a region 100 Kb upstream of the Shh CDS. The new element compensates for the Enhancer activi_x0002_ties of the three previously identified Enhancers, and the combined activities of all four Enhancers permit the seamless Shh expression in the epithelial linings of the whole endoder_x0002_mal tissues in the mouse embryo.\"\\t\"Rbfox3,Socs1\"\\t\"Fox-3,Hrnbp3,NeuN,Neuna60,Cish1,Cish3,JAB,SOCS-1,SSI-1\"\\tDNaseI-seq\\t\"Putative binding sites for Fox transcription factors and STAT are found in the SLGE.In human cancer cell lines, including Caco-2, putative Fox- and STAT-binding sites are found in some of the DNase I HS sites around the SHH CDS.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t39503', 'E_02_067\\t24157522\\t--\\tmm10\\tchr5\\t28500370\\t28502363\\tMouse\\tColon Adenocarcinoma Cell Lines\\tLow+High throughput\\t\"RT-PCR,ChIP-seq,Transient Transfection Assay,Luciferase Reporter Assay,3C,DNaseI-seq\"\\t\"The ChIP-seq data of the ENCODE project show that H3K4me1, but not H3K4me3, marks all four DNaseI HS sites in the adult lung.To examine whether each of the DNase I HS sites is responsible for upregulating the expression of Shh, we employed a transgenic reporter assay using BAC clones (Figs. 1A and 3A).\"\\tEnhancer\\tShh\\t3C\\tLuciferase Reporter Assay\\t\"To determine whether SLGE physically interacts with the Shh promoter in association with transcriptional activity, we conducted a 3C assay, which has been used to measure associations between two genomic regions. Assessment of the functional properties of SLGE by 3C and luciferase assays.\"\\t\"9530036O11Rik,Dsh,Hhg1,Hx,Hxl3,M100081NC,Shh\"\\t--\\t--\\t--\\t\"Shh expression in the endodermal epithelial lining, which is enabled by the coordinated actions of the Enhancers, is necessary for proper endodermal development.\"\\t\"RT-PCR,Luciferase Reporter Assay,3C\"\\t\"We identified a novel, less-conserved cis-regulatory element for Shh expres_x005f_x0002_sion in the esophageal epithelium, as well as in the lung and intestinal epithelium, in a region 100 Kb upstream of the Shh CDS. The new element compensates for the Enhancer activi_x0002_ties of the three previously identified Enhancers, and the combined activities of all four Enhancers permit the seamless Shh expression in the epithelial linings of the whole endoder_x0002_mal tissues in the mouse embryo.\"\\t\"Rbfox3,Socs1\"\\t\"Fox-3,Hrnbp3,NeuN,Neuna60,Cish1,Cish3,JAB,SOCS-1,SSI-1\"\\tDNaseI-seq\\t\"Putative binding sites for Fox transcription factors and STAT are found in the SLGE.In human cancer cell lines, including Caco-2, putative Fox- and STAT-binding sites are found in some of the DNase I HS sites around the SHH CDS.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t44528', 'E_02_068\\t24157522\\t--\\tmm10\\tchr5\\t28561915\\t28564905\\tMouse\\tColon Adenocarcinoma Cell Lines\\tLow+High throughput\\t\"RT-PCR,ChIP-seq,Transient Transfection Assay,Luciferase Reporter Assay,3C,DNaseI-seq\"\\t\"The ChIP-seq data of the ENCODE project show that H3K4me1, but not H3K4me3, marks all four DNaseI HS sites in the adult lung.To examine whether each of the DNase I HS sites is responsible for upregulating the expression of Shh, we employed a transgenic reporter assay using BAC clones (Figs. 1A and 3A).\"\\tEnhancer\\tShh\\t3C\\tLuciferase Reporter Assay\\t\"To determine whether SLGE physically interacts with the Shh promoter in association with transcriptional activity, we conducted a 3C assay, which has been used to measure associations between two genomic regions. Assessment of the functional properties of SLGE by 3C and luciferase assays.\"\\t\"9530036O11Rik,Dsh,Hhg1,Hx,Hxl3,M100081NC,Shh\"\\t--\\t--\\t--\\t\"Shh expression in the endodermal epithelial lining, which is enabled by the coordinated actions of the Enhancers, is necessary for proper endodermal development.\"\\t\"RT-PCR,Luciferase Reporter Assay,3C\"\\t\"We identified a novel, less-conserved cis-regulatory element for Shh expres_x005f_x0002_sion in the esophageal epithelium, as well as in the lung and intestinal epithelium, in a region 100 Kb upstream of the Shh CDS. The new element compensates for the Enhancer activi_x0002_ties of the three previously identified Enhancers, and the combined activities of all four Enhancers permit the seamless Shh expression in the epithelial linings of the whole endoder_x0002_mal tissues in the mouse embryo.\"\\t\"Rbfox3,Socs1\"\\t\"Fox-3,Hrnbp3,NeuN,Neuna60,Cish1,Cish3,JAB,SOCS-1,SSI-1\"\\tDNaseI-seq\\t\"Putative binding sites for Fox transcription factors and STAT are found in the SLGE.In human cancer cell lines, including Caco-2, putative Fox- and STAT-binding sites are found in some of the DNase I HS sites around the SHH CDS.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t106571', 'E_02_069\\t24204311\\tECR9 Enhancer\\tmm10\\tchr19\\t10215891\\t10216703\\tMouse\\tOligodendroglial Cell\\tLow throughput\\t\"ChIP,Luciferase Reporter Assay,EMSA,qPCR\"\\t\"Considering the large number of potential Sox binding sites and the low predictive power of their presence, we decided to assess regulatory potential and Sox10-responsiveness of the ECR in luciferase assays.To verify that Sox10 is bound to ECR9 in OL we next performed chromatin immunoprecipitation (ChIP) experiments. Some were clustered so that eight oligonucleotides were sufficient to cover all sites in electrophoretic mobility shift assays (EMSA).\"\\tEnhancer\\tMyrf\\t--\\tImmunoprecipitation\\t To address this issue we performed immunoprecipitation experiments with anti-Sox10 antibodies on extracts of the OL-like OLN93 cell line.\\t\"6030439E18,Gm1804,Gm98,Mrf\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSox10\\t\"Dom,Sox21,gt\"\\tChIP\\tTo verify that Sox10 is bound to ECR9 in OL we next performed chromatin immunoprecipitation (ChIP) experiments.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t8027', 'E_02_070\\t24257627\\t--\\tmm10\\tchr2\\t119321593\\t119322529\\tMouse\\t\"293T,Progenitor Cell\"\\tLow throughput\\t\"ChIP,PCR,EMSA\"\\t\"In addition, we carried out chromatin immunoprecipitation (ChIP) assays to show that both Ascl1 and Neurog1 were able to occupy in vivo the critical E-box region of the CR1 Enhancer in cell culture and mouse embryonic neural tubes, whereas no enrichment was shown for control DNA in Dll4 3 UTR.\"\\tEnhancer\\tDll4\\t--\\tEMSA\\t\"We performed an electrophoretic mobility shift assay (EMSA) to test whether Ascl1 and Neurog proteins were able to directly bind to the minimal enhancer region using an oligo probe containing the critical E-box (Probe 2) and a control upstream probe that lacks an E-box (Probe 1)', 'E_02_071\\t24257627\\t--\\tmm10\\tchr2\\t119321593\\t119322529\\tMouse\\t\"293T,Progenitor Cell\"\\tLow throughput\\t\"ChIP,PCR,EMSA\"\\t\"we tested whether Foxn4, Ascl1 and Neurog1 were able to activate gene expression through an evolutionarily conserved Dll4 enhancer CR1 found to be active in the retina. In the chick spinal cord, CR1 was able to drive DsRed reporter expression in a pattern that mimics that of the mouse Dll4 along the dorsoventral axis, predominantly in the V2 domain\"\\tEnhancer\\tDll4\\t--\\t\"ChIP,PCR,EMSA\"\\t\"In addition, we carried out chromatin immunoprecipitation (ChIP) assays to show that both Ascl1 and Neurog1 were able to occupy in vivo the critical E-box region of the CR1 Enhancer in cell culture and mouse embryonic neural tubes, whereas no enrichment was shown for control DNA in Dll4 3 UTR.\"\\tDelta4\\t--\\t--\\t--\\t\"Foxn4, Ascl1 and Neurog1 were able to activate gene expression through an evolutionarily conserved Dll4 Enhancer CR1 found to be active in the retina. \"\\t\"ChIP,EMSA\"\\t\"In addition, we carried out chromatin immunoprecipitation (ChIP) assays to show that both Ascl1 and Neurog1 were able to occupy in vivo the critical E-box region of the CR1 Enhancer in cell culture and mouse embryonic neural tubes, whereas no enrichment was shown for control DNA in Dll4 3 UTR.Thus, Ascl1 and Neurog factors may bind to the same E-box in the Dll4 Enhancer but have differential effects on Dll4 gene expression.\"\\t\"Foxn4,Ascl1\"\\t\"AI225900,ASH1,Mash1,bHLHa46\"\\t\"EMSA,ChIP\"\\tWe performed an electrophoretic mobility shift assay (EMSA) to test whether Ascl1 and Neurog proteins were able to directly bind to the minimal Enhancer region using an oligo probe containing the critical E-box (Probe 2) and a control upstream probe that lacks an E-box.\\t--\\t--\\t--\\t>2KB\\t\\t\\t3722', 'E_02_072\\t30146478\\t--\\tmm10\\tchr11\\t111838889\\t111839334\\tMouse\\tCartilage\\tLow+High throughput\\tChIP-qPC\"We next performed the ChIP-qPCR using anti-H3K4me1 and H3K27ac antibody in all five regions and found that RCSE regions were significantly enriched in both H3K4me1 and H3K27ac ChIP, suggesting the potential enhancer.\"\\tEnhancer\\tSox9\\t3C\\tPCR\\tThese data suggest that there is a long-range interaction between the Sox9 promoter and RCSE region in primary chondrocytes.\\t\"2010306G03Rik,AV220920,mKIAA4243\"\\t--\\t--\\t--\\tTG mice with the Sox9 promoter and 43 RCSE region.These data suggest that the RCSE region functions as a costal- and sternum cartilage-specific Enhancer.\\tTransgenic mice\\tTG mice with the Sox9 promoter and 43 RCSE region.These data suggest that the RCSE region functions as a costal- and sternum cartilage-specific Enhancer.\\tStat3\\t\"1110034C02Rik,AW109958,Aprf\"\\t\"Luciferase Reporter Assay,Immunostaining\"\\t\"Furthermore, transcription factors that regulate Sox9 expres sion were screened using a CRISPR/Cas9-chromatin immuno precipitation (ChIP)-mass spectrometry (MS) system targeting the RCSE region in chondrocytes; signal transducer and acti vator of transcription 3 (Stat3) was identified as a transacting fac tor for the Sox9 Enhancer.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t943097', 'E_02_073\\t24324617\\t--\\tmm10\\tchr16\\t59471850\\t59472124\\tMouse\\tEL4\\tLow+High throughput\\t\"ChIP,Luciferase Reporter Assay\"\\tThese results suggested: (1) that region (264/+19) contains an element that mediates strong reporter activity suggestive of an enhancer; and (2) that fragment(+19/+354) contains a Mina promoter comparable in activity to the SV40 promoter.\\tEnhancer\\tRiox2\\t--\\tqRT-PCR\\t\"In summary, two promoters were identified in a ,2 kb Mina TSS-spanning genomic fragment: P1 in region (264/+19) and P2 in region (+150/+280). Further, we inferred the existence of a P1-specific Enhancer E1 located in region (+80/+354).\"\\t\"1810047J07Rik,2410057H13Rik,3830408E23Rik,AI449204,Mina\"\\tPulmonary Inflammation\\t--\\tD011014\\t--\\t--\\t--\\t\"Sp1,Sp3\"\\t\"1110003E12Rik,AA450830,AI845540-1,Sp1,D130027J01Rik\"\\t\"EMSA,RT-PCR,ChIP,Luciferase Reporter Assay\"\\tEMSA supershift analysis of Sp1/3 association with the four Sp1/3 binding sites comprising the Mina P1 promoter.\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t234', 'E_02_074\\t24368734\\t--\\tmm10\\tchr6\\t115409226\\t115436116\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,qPCR,RNA-seq\"\\t\"To characterize the genomic features of MLL4 binding regions during differentiation, we performed ChIP-Seq of H3K4me1/2/3, H3K27ac and Pol II during adipogenesis. Then, we used histone marks to define four types of gene regulatory elements: active enhancer, silent enhancer, active promoter, and silent promoter. Average profile plots revealed that at day 2 of adipogenesis, MLL4 and adipogenic TFs C/EBP, C/EBP and PPAR were enriched on active enhancers.\"\\tEnhancer\\tPparg\\t--\\tqPCR\\t\"Importantly, deletion of Mll4 markedly decreased H3K4me1 and H3K27ac on over 90% of the 591 recovered C/EBP+ MLL4+ adipogenic Enhancers (Figure 8CCD). MLL4-dependent de novo H3K4me1 and H3K27ac were also observed on the C/EBP+ adipogenic Enhancers located on Pparg gene locus.\"\\t\"Nr1c3,PPAR-gamma,PPAR-gamma2,PPARgamma,PPARgamma2\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Ppargc1a,Cebpa,Cebpb\"\\t\"A830037N07Rik,Gm11133,PGC-1,PPARGC-1-alpha,Pgc-1alpha,Pgc1,Pgco1,Ppargc1,C/EBPbeta,CRP2,IL-6DBP,LAP,LIP,NF-IL6,NF-M,Nfil6,C/ebpalpha,CBF-A,Cebp\"\\tChIP-seq\\t\"To experimentally validate the predicted motifs, we performed ChIP-Seq of C/EBP, C/EBP and PPAR at day 2 of adipogenesis and of MyoD in myocytes. By comparing the genomic localizations of MLL4 with those of C/EBP, C/EBP and PPAR, we found that consistent with the motif analysis, 64% of MLL4 binding regions overlapped with those of C/EBP/ or PPAR at day 2 of adipogenesis.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t61795', 'E_02_075\\t24368734\\t--\\tmm10\\tchr6\\t115507545\\t115533315\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,PCR,RNA-seq\"\\t\"To characterize the genomic features of MLL4 binding regions during differentiation, we performed ChIP-Seq of H3K4me1/2/3, H3K28ac and Pol II during adipogenesis. Then, we used histone marks to define four types of gene regulatory elements: active enhance\"\\tEnhancer\\tPparg\\t--\\tqPCR\\t\"Importantly, deletion of Mll4 markedly decreased H3K4me1 and H3K27ac on over 90% of the 591 recovered C/EBP+ MLL4+ adipogenic Enhancers (Figure 8CCD). MLL4-dependent de novo H3K4me1 and H3K27ac were also observed on the C/EBP+ adipogenic Enhancers located on Pparg gene locus.\"\\t\"Nr1c3,PPAR-gamma,PPAR-gamma2,PPARgamma,PPARgamma2\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Ppargc1a,Cebpa,Cebpb\"\\t\"A830037N07Rik,Gm11133,PGC-1,PPARGC-1-alpha,Pgc-1alpha,Pgc1,Pgco1,Ppargc1,C/EBPbeta,CRP2,IL-6DBP,LAP,LIP,NF-IL6,NF-M,Nfil6,C/ebpalpha,CBF-A,Cebp\"\\tChIP-seq\\t\"To experimentally validate the predicted motifs, we performed ChIP-Seq of C/EBP, C/EBP and PPAR at day 2 of adipogenesis and of MyoD in myocytes. By comparing the genomic localizations of MLL4 with those of C/EBP, C/EBP and PPAR, we found that consistent with the motif analysis, 64% of MLL4 binding regions overlapped with those of C/EBP/ or PPAR at day 2 of adipogenesis.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t159554', 'E_02_076\\t24374176\\t--\\tmm10\\tchr2\\t115794340\\t115800495\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,ChIP,3C\"\\t\"Based on these results, we identified sequence d as a MB Meis2 enhancer, and hereafter designate it as MBE. We went on to look for accumulation of histone H3K27 acetylation (H3K27ac) and H3K4 monomethylation (H3K4me1), which demarcate active and potential enhancers, respectively, at MBE by ChIP-seq.\"\\tEnhancer\\tMeis2\\t3C\\tImmuno-FISH\\t\"Since Meis2 and its upstream region are highly homologous between mouse and human, we used two human BAC clones that contained mainly the upstream or gene body region of Meis2 to test for inclusion of an MB-specific Enhancer.\"\\t\"A430109D20Rik,Mrg1,Stra10\"\\t--\\t--\\t--\\t--\\t--\\t--\\tRnf2\\t\"AI326319,AI450156,AU019207,Ring1B,dinG\"\\t\"ChIP-qPCR,ChIP-seq\"\\t\"RING1B ChIP-seq data (GSE48464) over 10 Mb of the mouse genome (mm9) around Meis2 in FB,MB, and LM at 11.5 dpcBinding of H3K27ac and RING1B in 11.5 dpc MB of WT and Ring1 mutant (Ring1 mut) revealed by ChIP-quantitative PCR (ChIP-qPCR) analysis.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t63845', 'E_02_077\\t24385922\\t--\\tmm10\\tchr6\\t52296135\\t52334522\\tMouse\\tLimb Cells\\tLow+High throughput\\t\"ChIP,PCR,3C,5C,ChIP-seq\"\\t\"Sequences distinct from proximal promoters (RefSeq) that were bound by RNAP2 and at least one other mark, or by both p300 and H3K27Ac were retained as candidate enhancers. Using these criteria, 19 putative enhancers were identified within 850 kb upstream of Hoxa13.\"\\tEnhancer\\tHoxa\\t3C\\tPCR\\t\"Here,we report on the first identification of bona fide transcriptional Enhancers controlling HoxA genes in developing limbs and show that these Enhancers are grouped into distinct topological domains at the sub-megabase scale.\"\\tHox-1\\t--\\t--\\t--\\t--\\t--\\t--\\tCtcf\\tAW108038\\t5C\\t\"Interestingly,loci bound either by CTCF or cohesin in limb buds have been recently identified and comparison with our data shows that almost all loci interacting with 59 HoxA genes overlap with either CTCF o cohesin binding.\"\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_078\\t24385922\\t--\\tmm10\\tchr6\\t52585919\\t52613016\\tMouse\\tLimb Cells\\tLow+High throughput\\t\"ChIP,PCR,3C,5C,ChIP-seq\"\\t\"Sequences distinct from proximal promoters (RefSeq) that were bound by RNAP2 and at least one other mark, or by both p300 and H3K27Ac were retained as candidate enhancers. Using these criteria, 19 putative enhancers were identified within 851 kb upstream true???\"\\tEnhancer\\tHoxa\\t3C\\tPCR\\t\"Here,we report on the first identification of bona fide transcriptional Enhancers controlling HoxA genes in developing limbs and show that these Enhancers are grouped into distinct topological domains at the sub-megabase scale.\"\\tHox-1\\t--\\t--\\t--\\t--\\t--\\t--\\tCtcf\\tAW108038\\t5C\\t\"Interestingly,loci bound either by CTCF or cohesin in limb buds have been recently identified and comparison with our data shows that almost all loci interacting with 59 HoxA genes overlap with either CTCF o cohesin binding.--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_079\\t24385922\\t--\\tmm10\\tchr6\\t52765812\\t52891511\\tMouse\\tLimb Cells\\tLow+High throughput\\t\"ChIP,PCR,3C,5C,ChIP-seq\"\\t\"Sequences distinct from proximal promoters (RefSeq) that were bound by RNAP2 and at least one other mark, or by both p300 and H3K27Ac were retained as candidate enhancers. Using these criteria, 19 putative enhancers were identified within 852 kb upstream true???\"\\tEnhancer\\tHoxa\\t3C\\tPCR\\t\"Here,we report on the first identification of bona fide transcriptional Enhancers controlling HoxA genes in developing limbs and show that these Enhancers are grouped into distinct topological domains at the sub-megabase scale.\"\\tHox-1\\t--\\t--\\t--\\t--\\t--\\t--\\tCtcf\\tAW108038\\t5C\\t\"Interestingly,loci bound either by CTCF or cohesin in limb buds have been recently identified and comparison with our data shows that almost all loci interacting with 59 HoxA genes overlap with either CTCF o cohesin binding.--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_080\\t24385922\\t--\\tmm10\\tchr6\\t52908070\\t53037532\\tMouse\\tLimb Cells\\tLow+High throughput\\t\"ChIP,PCR,3C,5C,ChIP-seq\"\\t\"Sequences distinct from proximal promoters (RefSeq) that were bound by RNAP2 and at least one other mark, or by both p300 and H3K27Ac were retained as candidate enhancers. Using these criteria, 19 putative enhancers were identified within 853 kb upstream true???\"\\tEnhancer\\tHoxa\\t3C\\tPCR\\t\"Here,we report on the first identification of bona fide transcriptional Enhancers controlling HoxA genes in developing limbs and show that these Enhancers are grouped into distinct topological domains at the sub-megabase scale.\"\\tHox-1\\t--\\t--\\t--\\t--\\t--\\t--\\tCtcf\\tAW108038\\t5C\\t\"Interestingly,loci bound either by CTCF or cohesin in limb buds have been recently identified and comparison with our data shows that almost all loci interacting with 59 HoxA genes overlap with either CTCF or cohesin binding.\"\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_081\\t24391132\\t--\\tmm10\\tchr4\\t147962887\\t147964479\\tMouse\\t293T\\tLow+High throughput\\t\"RT-PCR,3C,ChIP,Luciferase Reporter Assay\"\\t\"Among the 11 CRs identified, only CR9 coincided with the binding sites of RNA polymerase II and p300, and overlapped with the gene areas modified by H3K4me1, and filled all criteria for the enhancer.\"\\tEnhancer\\tNppb\\t3C\\t\"RT-PCR,Transgenic mice\"\\t\"The Enhancer is a 650-bp fragment within 50 kb of the Nppa and Nppb loci.To identify the organs in which CR9 functioned as a stress-responsive Enhancer, we examined the luciferase reporter expression in each organ by quantitative PCR.\"\\t\"AA408272,BNF,BNP\"\\tHeart Failure\\t--\\tD006333\\t--\\t--\\t--\\t\"Nkx2-5,Ctcf\"\\t\"CHNG5,CSX,CSX1,HLHS2,NKX2.5,NKX2E,NKX4-1,VSD3,AW108038\"\\tChIP-seq\\t\"ChIP-seq data for H3K4me1, p300, and CTCF were obtained from an open database of the adult mouse heart. Some CRs coincided with the peaks for H3K4me1, RNA polymerase II, and the transcriptional coactivator protein p300.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t22104', 'E_02_082\\t24391766\\t--\\tmm10\\tchr7\\t100474191\\t100474841\\tMouse\\tPlatinum E cells\\tLow+High throughput\\t\"Luciferase Reporter Assay,PCR,EMSA,ChIP-seq\"\\t\"The first intron of the hamster is shorter compared to mouse and rat corresponding to the first half of the introns in these species.Generally, sequence conservation was low across the first intron,except a region of high conservation ranging from IVS(intervening sequence)1+1200 to IVS1+1850 with the IVS1+1505G/A baseexchange in the center of this region.\"\\tEnhancer\\tUcp3\\t--\\t\"Luciferase Reporter Assay,PCR,EMSA\"\\t\"Generally, sequence conservation was low across the first intron,except a region of high conservation ranging from IVS(intervening sequence)1+1200 to IVS1+1850 with the IVS1+1505G/A base exchange in the center of this region.\"\\t\"AI645527,Slc25a9,UCP-3\"\\t--\\t--\\t--\\tThis intronic region is the main Enhancer driving UCP3 expression with SP1/3 and PPARc as the core factors required for expression.\\t\"EMSA,ChIP-seq\"\\t\"While we understand that simple EMSA experiments are not sufficient to validate presence of a complex transcription factor binding module conserved across the whole mammalian class, our data provide good evidence that it an intronic Enhancer in the first intron of the UCP3 gene is of importance in non-rodent species as well.\"\\t\"Sp1,Sp3\"\\t\"1110003E12Rik,AA450830,AI845540-1,Sp1,D130027J01Rik\"\\t\"EMSA,ChIP-seq\"\\t\"We discovered that the transcription factors SP1 and SP3 were binding to the IVS1+1505G element, whereas binding to the mutant allele was strongly diminished. Direct binding of PPAR and RXRa to the IVS1+1505G element could be ruled out.\"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t1526', 'E_02_083\\t24692107\\t--\\tmm10\\tchr17\\t44464986\\t44466986\\tMouse\\tOsteoblast-Specific Cell\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,EMSA\"\\tand found that a 1.3\\\\kb region located about 30 kb upstream of the transcription start site of type II Runx2 was highly conserved among all of the species\\tEnhancer\\tRunx2\\t--\\t\"Transgenic mice,RT-PCR\"\\t\"We searched for regulatory elements of the Runx2 gene,and identified a 343bp Enhancer sequence,which specifically directed the reporter gene expression to osteoblasts.\"\"AML3,Cbf,Cbfa-1,Cbfa1,LS3,Osf2,PEBP2aA,Pebp2a1,Pebpa2a\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Dlx5,Mef2c,Tcf7,Ctnnb1,Sp7,Smad1,Sox6\"\\t\"AI385752,5430401D19Rik,9930028G15Rik,AV011172,Mef2,AI465550,TCF-1,Tcf1,Bfc,Catnb,Mesc,6430578P22Rik,C22,Osx,AI528653,Mad1,Madh1,Madr1,Mlp1,MusMLP,dwf-A,mMad1,AI987981,SOX-LZ\"\\t\"EMSA,ChIP\"\\t\"In EMSA, Mef2c, Dlx5, and Msx2 but not Tcf7, Ctnnb1, Smad1, Sox6, and Sp7 directly bound to the 89\\\\bp core sequence.In ChIP analysis, the bindings of Dlx5, Dlx6, Msx2, Mef2, Tcf7, Ctnnb1, Smad1, Sox5, Sox6, and Sp7 to the 343 enhancer region were detected using each antibody, whereas the binding of Foxa2 was not detected in primary osteoblasts despite the presence of a conserved Foxa2 binding motif in the core 89\\\\bp sequence.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t30000', 'E_02_084\\t24705708\\t--\\tmm10\\tchr6\\t52199038\\t52200503\\tMouse\\t\"Lung,Stomach\"\\tLow+High throughput\\t\"EMSA,ChIP\"\\tA 2.1-kb mesodermal (MES) enhancer important for Hoxa5 paraxial and lateral plate mesoderm expression in the cervico-upper thoracic region of the A-P axis is positioned 39 of the Hoxa5 gene.\\tEnhancer\\tVegfa\\t--\\t\"Transgenic mice,EMSA,qRT-PCR\"\\tA 1.5-kb XbaI-XbaI DNA fragment located in Hoxa4-Hoxa5 intergenic sequences at 3.0-kb upstream the Hoxa4 gene (positions +9351-bp to +10816-bp) was able to target Hoxa5 expression in lung and stomach.\\t\"Vegf,Vpf\"\\t--\\t--\\t--\\t--\\t--\\t--\\tYy1\\t\"NF-E1,YY-1\"\\t\"ChIP,qPCR\"\\t\"In summary, YY1 acts as a positive regulator of Hoxa5 lung, stomach and intestine expression during embryogenesis.\"\\t--\\t--\\t--\\t>2KB\\t6182779', 'E_02_085\\t24734888\\t--\\tmm10\\tchr1\\t133347808\\t133348049\\tMouse\\tKidney\\tLow throughput\\t\"RT-PCR,Southern blot\"\\t\"The mdE, defined by 240 bp DNA sequence, was identified by transient transfection of 50 deletion constructs of the mouse Ren-1c gene into As4.1 cells. This tissue-specific enhancer is located 2.6 kb upstream (_x0002_2866 to _x0002_2625 bp) of the transcriptional start site of the mouse renin gene and its homologous sequence is present about 12 kb upstream of the human renin start site of transcription.\"\\tEnhancer\\tRen1\\t--\\t\"Transgenic mice,RT-PCR\"\\t\"Structural analysis of the transgenic mdE region. In vivo Cre-loxP recombination shown in the left was confirmed by Southern blot analysis of tail DNA from parental,wt,and mut TgM lines,in which the DNA was digested with EcoRI (E),separated on agarosegels,transferred to a nylon membrane,and hybridized to the probe (shaded rectangles in the left panel,from nt 2635 to 2214 relative to transcription start site). The sizes of the expected bands are indicated (in kb) in the right panel and those expected from the endogenous locus are in parentheses.\"\\t\"D19352,Ren,Ren-1,Ren-Ac,Ren1d,Rn-1,Rnr,?Ren1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2744', 'E_02_086\\t24812424\\t--\\tmm10\\tchr16\\t23143036\\t23145036\\tMouse\\tOffspring Adipocytes\\tLow throughput\\t\"ChIP,qRT-PCR,Western blot\"\\t\"Active epigenetic marks were significantly decreased in the promoter and enhancer regions of the AdipoQ gene in 24-week-old offspring from SF-exposed mothers. In contrast,the inactive epigenetic marks in the AdipoQ gene (5-mC and histone H3K9m2) were significantly higher in SF offspring adipocytes\"\\tEnhancer\\tAdipoq\\t--\\t\"qRT-PCR,Western blot\"\\t\"SF exposures during late gestation alter epigenetic markers in the AdipoQ gene promoter and Enhancer regions in adipocytes from VWAT. Active (5-hmC and H3K4me3) and inactive (5-mC and H3K9me2) marks were mapped to the Enhancer (22,500 bp) and promoter (2500 bp) regions and downstream to the transcription start site (+500 bp) region, respectively, using ChIP assays.\"\\t\"30kDa,APN,Acdc,Acrp30,Ad,Adid,GBP28,adipo,apM1\"\\tSleep Fragmentation\\t--\\tD012892\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2499', 'E_02_087\\t24905168\\t--\\tmm10\\tchr4\\t124638991\\t124641160\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tPou3f1\\t--\\t\"qPCR,IP-Western\"\\t\"Upon differentiation, Otx2?/? cells underwent morphological changes indistinguishable from those seen in wt EpiLCs, but showed defects in expression of certain epiblast-associated genes, such as Fgf5 and Oct6. RNA-seq analysis of Otx2?/? EpiLCs identified 260 and 141 genes significantly downregulated or upregulated, respectively, as compared to the genetically-matched wt EpiLCs\"\\t\"Oct-6,Oct6,Otf-6,Otf6,Scip,Test1,Tst-1,Tst1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t17569', 'E_02_088\\t24905168\\t--\\tmm10\\tchr4\\t124645999\\t124647927\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tPou3f1\\t--\\t\"qPCR,IP-Western\"\\t\"Upon differentiation, Otx2?/? cells underwent morphological changes indistinguishable from those seen in wt EpiLCs, but showed defects in expression of certain epiblast-associated genes, such as Fgf5 and Oct6. RNA-seq analysis of Otx2?/? EpiLCs identified 260 and 141 genes significantly downregulated or upregulated, respectively, as compared to the genetically-matched wt EpiLCs\"\\t\"Oct-6,Oct6,Otf-6,Otf6,Scip,Test1,Tst-1,Tst1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t10682', 'E_02_089\\t24905168\\t--\\tmm10\\tchr5\\t119677099\\t119681977\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tTbx3\\t--\\t\"qPCR,IP-Western\"\\t\"However, comparison between ESC and EpiLC showed substantial reorganization of Oct4 occupancy during differentiation with gain of binding near genes associated with post-implantation epiblast (e.g. Fgf5, Oct6, Wnt8a) and loss near genes associated with na?ve pluripotency (e.g. Klf4, Tbx3, Prdm14)\"\\tD5Ertd189e\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t9220', 'E_02_090\\t24905168\\t--\\tmm10\\tchr5\\t119704124\\t119707831\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tTbx3\\t--\\t\"qPCR,IP-Western\"\\t\"However, comparison between ESC and EpiLC showed substantial reorganization of Oct4 occupancy during differentiation with gain of binding near genes associated with post-implantation epiblast (e.g. Fgf5, Oct6, Wnt8a) and loss near genes associated with na?ve pluripotency (e.g. Klf4, Tbx3, Prdm14)\"\\tD5Ertd189e\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t35660', 'E_02_091\\t24905168\\t--\\tmm10\\tchr5\\t119707931\\t119710758\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tTbx3\\t--\\t\"qPCR,IP-Western\"\\t\"However, comparison between ESC and EpiLC showed substantial reorganization of Oct4 occupancy during differentiation with gain of binding near genes associated with post-implantation epiblast (e.g. Fgf5, Oct6, Wnt8a) and loss near genes associated with na?ve pluripotency (e.g. Klf4, Tbx3, Prdm14)\"\\tD5Ertd189e\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t39027', 'E_02_092\\t24905168\\t--\\tmm10\\tchr5\\t119710660\\t119714758\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tTbx3\\t--\\t\"qPCR,IP-Western\"\\t\"However, comparison between ESC and EpiLC showed substantial reorganization of Oct4 occupancy during differentiation with gain of binding near genes associated with post-implantation epiblast (e.g. Fgf5, Oct6, Wnt8a) and loss near genes associated with na?ve pluripotency (e.g. Klf4, Tbx3, Prdm14)\"\\tD5Ertd189e\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t42391', 'E_02_093\\t24905168\\t--\\tmm10\\tchr5\\t98290964\\t98295647\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tFgf5\\t--\\t\"qPCR,IP-Western\"\\t\"Interrogation of our ChIP-seq datasets revealed presence of a proximal Enhancer (PE) at the Fgf5 locus, which exists in a poised state in ESCs, as well as a cluster of four Enhancers (depicted as E1-E4) located within 25C60 kb downstream from the Fgf5 TSS that are unmarked in ESCs, but activated de novo during the transition.\"\\t\"Fgf-5,HBGF-5,angora,go\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t39123', 'E_02_094\\t24905168\\t--\\tmm10\\tchr5\\t98307793\\t98312281\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tFgf5\\t--\\t\"qPCR,IP-Western\"\\t\"Interrogation of our ChIP-seq datasets revealed presence of a proximal Enhancer (PE) at the Fgf5 locus, which exists in a poised state in ESCs, as well as a cluster of four Enhancers (depicted as E1-E4) located within 25C60 kb downstream from the Fgf5 TSS that are unmarked in ESCs, but activated de novo during the transition.\"\\t\"Fgf-5,HBGF-5,angora,go\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t55854', 'E_02_095\\t24905168\\t--\\tmm10\\tchr5\\t98310330\\t98314427\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tFgf5\\t--\\t\"qPCR,IP-Western\"\\t\"Interrogation of our ChIP-seq datasets revealed presence of a proximal Enhancer (PE) at the Fgf5 locus, which exists in a poised state in ESCs, as well as a cluster of four Enhancers (depicted as E1-E4) located within 25C60 kb downstream from the Fgf5 TSS that are unmarked in ESCs, but activated de novo during the transition.\"\\t\"Fgf-5,HBGF-5,angora,go\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t58196', 'E_02_096\\t24905168\\t--\\tmm10\\tchr5\\t98253530\\t98257713\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"To address whether changes in enhancer utilization patterns occur in the majority of cells within the differentiating population, we isolated dual transgenic enhancer reporter lines, in which ESC-specific or EpiLC-specific enhancers of interest were cloned upstream of a minimal promoter and one of two distinct fluorescent reporters and integrated into the ESC genome using a PiggyBac transposon system.\"\\tEnhancer\\tFgf5\\t--\\t\"qPCR,IP-Western\"\\t\"Interrogation of our ChIP-seq datasets revealed presence of a proximal Enhancer (PE) at the Fgf5 locus, which exists in a poised state in ESCs, as well as a cluster of four Enhancers (depicted as E1-E4) located within 25C60 kb downstream from the Fgf5 TSS that are unmarked in ESCs, but activated de novo during the transition.\"\\t\"Fgf-5,HBGF-5,angora,go\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Oct4\"\\t\"E130306E05Rik,2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1\"\\tChIP-seq\\tOur results illuminate regulatory mechanisms underlying pluripotency and suggest that capacity of transcription factors such as Otx2 and Oct4 to pioneer new Enhancer sites is highly context dependent.\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1439', 'E_02_097\\t24915132\\t--\\tmm10\\tchr2\\t25423079\\t25423874\\tMouse\\tTh1\\tLow throughput\\tChIP\\t\"Incontrast, lung fibroblasts, which are of mesenchymal origin andwhich lack Fut7 mRNA and selectin ligand expression, carried adifferential histone modification pattern with an enrichment ofH3K27me3 at the CNS and the Fut7 gene locus and absenceof H3K4me2 marks. This suggests that the pattern of histone mod-ifications might control tissue-specific expression of Fut7, ratherthan the differentiation and activation dependent expression inT cells\"\\tEnhancer\\tFut7\\t--\\t\"qRT-PCR,Luciferase Reporter Assay\"\\tThe CNS was cloned in different orientations in the Enhancer position of the pGL3 vector carrying the +728/+1523 construct in promoter position and transfected into Th1 cells.\\t\"AI853193,FTVII,Fuc-TVII,FucT-VII\"\\t--\\t--\\t--\\t--\\t--\\t--\\tEp300\\t\"A430090G16,A730011L11,KAT3B,p300,p300 HAT\"\\tChIP\\t\"As our histone ChIP data as well as analysis of genome-wide p300 binding data published by Vahedi et al.(2012) revealed distribution ofthe active histone mark as well as enrichment of p300 binding (data not shown) beyond the homology region, i.e. the CNS, we extended the CNS by about 300 bp upstream and 150 bp downstream of the original construct.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1127', 'E_02_098\\t25030696\\t--\\tmm10\\tchr17\\t31172998\\t31173656\\tMouse\\tCOS-1\\tLow throughput\\t\"3C,ChIP,RNA-seq,GRO-seq,Luciferase Reporter Assay\"\\t\"Interestingly, primarily RXR-induced eRNA production could be detected on an enhancer assigned to Vegfa or Tgm2, while an enhancer of Abcg1 also showed robust LXR ligand activation, as expected. These data suggested that the eRNAs can be easily validated and show ligand induction similar to the regulated genes and therefore most likely are linked.\"\\tEnhancer\\tAbcg1\\t3C\\tqRT-PCR\\t\"Box plot representation of the distribution of interaction frequency of Abcg1 B1 Enhancer (chr17: 31,172,998C31,173,656) and Vegfa B1 Enhancer (chr17: 45,890,060C45,890,829) determined by 3C-seq.\"\\t\"AW413978,Abc8,White\"\\t--\\t--\\t--\\t\"A set of angiogenic genes, including Vegfa, has liganded RXR-controlled enhancers and provides the macrophage with a novel inducible program\"\\t\"3C,3C-seq\"\\t\"Using chromosome conformation capture (3C) and 3C combined with sequencing (3C-seq), and we show that selected long-range Enhancers communicate with promoters via stable or RXR-induced loops and that some of the Enhancers interact with each other, forming an interchromosomal network.\"\\tRxra\\t\"9530071D11Rik,Nr2b1,RXRalpha1\"\\t\"RNA-seq,ChIP-seq,GRO-seq,3C-seq\"\\t\"In our studies, we tried to solve these issues by combining RNA-seq, ChIP-seq, GRO-seq (global run-on sequencing), and 3C-seq (chromosome conformation capture [3C] combined with sequencing) in a highly integrated way to unravel the mechanism of RXR-induced transcriptional events in mouse bone marrow-derived macrophages (BMDMs) and, as the result of the process,discovered and validated a novel biological activity promoted by the receptor.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t115634', 'E_02_099\\t25030696\\t--\\tmm10\\tchr17\\t45890060\\t45890829\\tMouse\\tCOS-1\\tLow throughput\\t\"3C,ChIP,RNA-seq,GRO-seq,Luciferase Reporter Assay\"\\t\"Interestingly, primarily RXR-induced eRNA production could be detected on an enhancer assigned to Vegfa or Tgm2, while an enhancer of Abcg1 also showed robust LXR ligand activation, as expected. These data suggested that the eRNAs can be easily validated and show ligand induction similar to the regulated genes and therefore most likely are linked.\"\\tEnhancer\\tVegfa\\t3C\\tqRT-PCR\\t\"Box plot representation of the distribution of interaction frequency of Abcg1 B1 Enhancer (chr17: 31,172,998C31,173,656) and Vegfa B1 Enhancer (chr17: 45,890,060C45,890,829) determined by 3C-seq.\"\\t\"Vegf,Vpf\"\\t--\\t--\\t--\\t\"A set of angiogenic genes, including Vegfa, has liganded RXR-controlled enhancers and provides the macrophage with a novel inducible program\"\\t\"3C,3C-seq\"\\t\"Using chromosome conformation capture (3C) and 3C combined with sequencing (3C-seq), and we show that selected long-range Enhancers communicate with promoters via stable or RXR-induced loops and that some of the Enhancers interact with each other, forming an interchromosomal network.\"\\tRxra\\t\"9530071D11Rik,Nr2b1,RXRalpha1\"\\t\"RNA-seq,ChIP-seq,GRO-seq,3C-seq\"\\t\"In our studies, we tried to solve these issues by combining RNA-seq, ChIP-seq, GRO-seq (global run-on sequencing), and 3C-seq (chromosome conformation capture [3C] combined with sequencing) in a highly integrated way to unravel the mechanism of RXR-induced transcriptional events in mouse bone marrow-derived macrophages (BMDMs) and, as the result of the process,discovered and validated a novel biological activity promoted by the receptor.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t126547', 'E_02_100\\t25153150\\t--\\tmm10\\tchrx\\t140806807\\t140808327\\tMouse\\tF9\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\t\"This suggests that the -65 to +20 bp region is a minimal promoter region for Tex13 gene expression, and various additional repressive and Enhancer regions are present in the -1500 to +20 bp region.\"\\tEnhancer\\tTex13b\\t--\\t\"PCR,Transfection\"\\t\"This suggests that the -65 to +20 bp region is a minimal promoter region for Tex13 gene expression, and various additional repressive and Enhancer regions are present in the -1500 to +20 bp region.\"\\t\"Tex13,4933403J23Rik\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t740', 'E_02_101\\t25185713\\t--\\tmm10\\tchr2\\t91067374\\t91069374\\tMouse\\t\"Bone Marrow Cell,Leukemic Stem Cell,Hematopoietic Stem Cell,416B\"\\tLow throughput\\t\"ChIP,3C,Transgenic mice\"\\t\"Mechanistically, by using a knockin mouse model in which all three Runx binding sites in the214kb enhancer of PU.1 are disrupted, we observed failure to form chromosomal interactions between the PU.1 enhancer and its proximal promoter. Consequently, decreased PU.1 levels resulted in diminished long-term HSC function through HSC exhaustion, which could be rescued by reintroducing a PU.1 transgene.\"\\tEnhancer\\tSpi1\\t3C\\t\"RT-PCR,PCR,Transgenic mice\"\\t\"Mechanistically, by using a knockin mouse model in which all three Runx binding sites in the 214kb Enhancer of PU.1 are disrupted, we observed failure to form chromosomal interactions between the PU.1 Enhancer and its proximal promoter.Runx-dependent PU.1 chromatin interaction and transcription of PU.1 are essential for both normal and leukemia stem cells.\"\\t\"Dis-1,Dis1,PU.1,Sfpi-1,Sfpi1,Spi-1,Tcfpu1,Tfpu.1\"\\tLeukemia\\tDOID:1240\\tD007938\\t--\\t--\\t--\\t\"Runx1,Runx2,Runx3\"\\t\"AML1,CBF-alpha-2,Cbfa2,Pebp2a2,Pebpa2b,AML3,Cbf,Cbfa-1,Cbfa1,LS3,Osf2,PEBP2aA,Pebp2a1,Pebpa2a,AML2,Cbfa3,Pebp2a3,Rx3\"\\tChIP\\t\"ChIP analyses of total bone marrow cells con?rmed the loss of Runx binding to the 214kb URE in PU.1-URE-mRunx mice (supplemental Figure 1C). Importantly, PU.1 mRNA levels in HSCs of PU.1-URE-mRunxmicewere reduced by 72%in comparisonwith controls (WT), greater than the average reduction of 63% observed in Runx1 knockout mice (Figure 1D). These results revealed the major role of the URE Runx sites for PU.1 transcription in HSCs.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t14007', 'E_02_102\\t25223790\\t--\\tmm10\\tchr17\\t35502632\\t35504632\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,ChIP-qPCR\"\\tWe first investigated activation of the Oct4 enhancer by luciferase reporter assay 48 h after transfection of TALE-A and dCas9-A/gRNA in MEFs. These luciferase constructs contain the 2.4 kb region covering all three upstream regulatory elements of the Oct4 locus\\tEnhancer\\tOct4\\tCRISPR/Cas9\\t\"qRT-PCR,Luciferase Reporter Assay\"\\tThe Oct4 luciferase assay reporter constructs carried the genomic DNA 2.4 kb upstream of the Oct4 transcrip_x0002_tion start site (TSS). The region encompasses the 1.7 kb distal and proximal Enhancers and the 0.2 kb promoter.\\t\"NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,?Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Klf4,Oct4,Nanog,Sox2\"\\t\"EZF,Gklf,Zie,NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4,2410002E02Rik,ENK,ecat4,Sox-2,lcc,ysb\"\\tChIP-seq\\t\"To address this possibility, we reviewed the ChIP-seq information of several pluripotency transcription factors, including KLF4,OCT4,NANOG and SOX2 at the Nanog 5kb upstream Enhancer region(42) and found that the Site 2 (targeted by both TALE and dCas9) was surrounded by the predicted KLF4 and NANOG binding sites.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2400', 'E_02_103\\t25223790\\t--\\tmm10\\tchr6\\t122700483\\t122702483\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,ChIP-qPCR\"\\t\"We also designed gRNA constructs and TALE-As to target the Nanog 5 kb upstream enhancer. Similar to the Oct4 locus, dCas9-As could bind their targeted regions and effectively activate the luciferase reporter carrying the 5 kb upstream enhancer \"\\tEnhancer\\tNanog\\tCRISPR/Cas9\\t\"qRT-PCR,Luciferase Reporter Assay\"\\t\"For Nanog luciferase assay reporter, the 1.0 kb DNA frag_x0002_ment of the Nanog 5 kb Enhancer (?5145 to ?4154) was cloned into a mini promoter luciferase vector.\"\\t\"2410002E02Rik,ENK,ecat4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Klf4,Oct4,Nanog,Sox2\"\\t\"EZF,Gklf,Zie,NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4,2410002E02Rik,ENK,ecat4,Sox-2,lcc,ysb\"\\tChIP-seq\\t\"To address this possibility, we reviewed the ChIP-seq information of several pluripotency transcription factors, including KLF4,OCT4,NANOG and SOX2 at the Nanog 5kb upstream Enhancer region(42) and found that the Site 2 (targeted by both TALE and dCas9) was surrounded by the predicted KLF4 and NANOG binding sites.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t6005', 'E_02_104\\t25249570\\t--\\tmm10\\tchr9\\t99131669\\t99132862\\tMouse\\tVentricular Cardiomyocytes (NRVMs)\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"To measure the transcriptional activity of the YAP-bound region of Pik3cb, which we refer to as the Pik3cb enhancer, we cloned a 552 bp genomic DNA fragment containing the conserved element and potential TEAD binding sites into a minimal promoter luciferase reporter construct. Co-transfection with Yap in NRVMs showed that Yap stimulates activity of the enhancer by ~5-fold.\"\\tEnhancer\\tPik3cb\\t--\\t\"qRT-PCR,ChIP-qPCR\"\\t\"To measure the transcriptional activity of the YAP-bound region of Pik3cb, which we refer to as the Pik3cb Enhancer, we cloned a 552 bp genomic DNA fragment containing the conserved element and potential TEAD binding sites into a minimal promoter luciferase reporter construct.\"\\t\"1110001J02Rik,AI447572,p110beta\"\\tHeart Injuries\\t--\\tD006335\\t\"YAP and TEAD occupied a conserved Enhancer within the first intron of Pik3cb, and this Enhancer drove YAP_x0002_dependent reporter gene expression.\"\\t\"ChIP-seq,ChIP-qPCR,Luciferase Reporter Assay\"\\t\"To measure the transcriptional activity of the YAP-bound region of Pik3cb, which we refer to as the Pik3cb Enhancer, we cloned a 552 bp genomic DNA fragment containing the conserved element and potential TEAD binding sites into a minimal promoter luciferase reporter construct. Co-transfection with Yap in NRVMs showed that Yap stimulates activity of the Enhancer by ~5-fold. \"\\tYap1\\t\"AI325207,Yap,Yap65,Yki,Yorkie\"\\t\"ChIP-seq,ChIP-qPCR,Luciferase Reporter Assay\"\\t\"The HL1 ChIP-seq data revealed a YAP-bound sequence residing in the first intron of Pik3cb (Fig. 2A). We validated YAP binding to the identified sequence by ChIP-qPCR,using a pair of primers spanning the YAP bound sequence and a control pair recognizing a sequence 1.3 kb away.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t93864', 'E_02_105\\t25263596\\t--\\tmm10\\tchr1\\t182851627\\t182852853\\tMouse\\tTet2-/-\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"We therefore hypothesized that loss of Tet2 could lead to reduced oxidation and increased 5mC at enhancers. Indeed, hypermethylated DMRs (hyper-DMRs) in Tet2?/? cells exhibit several hallmarks of enhancers including: evolutionary sequence conservation,enrichment for enhancer chromatin marks H3K4me1 and H3K27ac, and significant overlap with co-activator p300 binding sites (o/e = 5.9, p < 1E-200), DNase I hypersensitive sites(o/e = 4.5, p < 1E-200), and predicted enhancers(o/e = 7.1, p < 1E-200)\"\\tEnhancer\\tLefty1\\t--\\t\"PCR,Transgenic mice\"\\t\"Indeed,an Enhancer that physically interacts with the developmental gene Left-Right Determination Factor 1 (Lefty1) is hypermethylated and hypoacetylated in Tet6?/? cells,potentially explaining the significantly decreased expression of this gene.\"\\t\"AI450052,Ebaf,Leftb,Stra3,Tgfb4,lefty,lefty-1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox4,Nanog\"\\t\"EZF,Gklf,Zie,NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4,2410002E02Rik,ENK,ecat4,Sox-2,lcc,ysb\"\\tChIP-seq\\t\"Weaker occupancy of the ES-cell core transcription factors (TFs) OCT4,SOX6,and NANOG than those that do not change DNA methylation status.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1917202', 'E_02_106\\t25263596\\t--\\tmm10\\tchr1\\t182852133\\t182854320\\tMouse\\tTet2-/-\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"We therefore hypothesized that loss of Tet2 could lead to reduced oxidation and increased 5mC at enhancers. Indeed, hypermethylated DMRs (hyper-DMRs) in Tet3?/? cells exhibit several hallmarks of enhancers including: evolutionary sequence conservation,enr\"\\tEnhancer\\tLefty1\\t--\\t\"PCR,Transgenic mice\"\\t\"Indeed,an Enhancer that physically interacts with the developmental gene Left-Right Determination Factor 1 (Lefty1) is hypermethylated and hypoacetylated in Tet7?/? cells,potentially explaining the significantly decreased expression of this gene.\"\\t\"AI450052,Ebaf,Leftb,Stra3,Tgfb4,lefty,lefty-1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox4,Nanog\"\\t\"EZF,Gklf,Zie,NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4,2410002E02Rik,ENK,ecat4,Sox-2,lcc,ysb\"\\tChIP-seq\\t\"Weaker occupancy of the ES-cell core transcription factors (TFs) OCT4,SOX7,and NANOG than those that do not change DNA methylation status.\"\\t--\\t--\\t--\\t>2KB\\t1918189', 'E_02_107\\t25263596\\t \\tmm10\\tchr1\\t182856227\\t182857187\\tMouse\\tTet2-/-\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"We therefore hypothesized that loss of Tet2 could lead to reduced oxidation and increased 5mC at enhancers. Indeed, hypermethylated DMRs (hyper-DMRs) in Tet4?/? cells exhibit several hallmarks of enhancers including: evolutionary sequence conservation,enr\"\\tEnhancer\\tLefty1\\t--\\t\"PCR,Transgenic mice\"\\t\"Indeed,an Enhancer that physically interacts with the developmental gene Left-Right Determination Factor 1 (Lefty1) is hypermethylated and hypoacetylated in Tet8?/? cells,potentially explaining the significantly decreased expression of this gene.\"\\t\"AI450052,Ebaf,Leftb,Stra3,Tgfb4,lefty,lefty-1\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox4,Nanog\"\\t\"EZF,Gklf,Zie,NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4,2410002E02Rik,ENK,ecat4,Sox-2,lcc,ysb\"\\tChIP-seq\\t\"Weaker occupancy of the ES-cell core transcription factors (TFs) OCT4,SOX5,and NANOG than those that do not change DNA methylation status.\"\\t--\\t--\\t--\\t>2KB\\t1921669', 'E_02_108\\t25263596\\t--\\tmm10\\tchr11\\t35178343\\t35180368\\tMouse\\tTet2-/-\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"We therefore hypothesized that loss of Tet2 could lead to reduced oxidation and increased 5mC at enhancers. Indeed, hypermethylated DMRs (hyper-DMRs) in Tet5?/? cells exhibit several hallmarks of enhancers including: evolutionary sequence conservation,enrInstance\"\": 858, \"\"TopActivityInstance\"\": 856, \"\"Success\"\": true}\"\\tEnhancer\\tSlit3\\t--\\t\"PCR,Transgenic mice\"\\t\"we also differentiated a previously reported Tet2 knockdown mESC line and performed quantitative real-time PCR analysis. Consistent with our observations in TET2?/? mESCs, we also find delayed induction of the three marker genes slit3, lmo4, and irx3\"\\t\"Slil2,Slit1,b2b2362.1Clo\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox2,Nanog\"\\t\"2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1,Sox-2,lcc,ysb,2410002E02Rik,ENK,ecat4\"\\tChIP-seq\\t\"Weaker occupancy of the ES-cell core transcription factors (TFs) OCT4,SOX2,and NANOG than those that do not change DNA methylation status.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t57901', 'E_02_109\\t25263596\\t--\\tmm10\\tchr11\\t35813868\\t35814943\\tMouse\\tTet2-/-\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"We therefore hypothesized that loss of Tet2 could lead to reduced oxidation and increased 5mC at enhancers. Indeed, hypermethylated DMRs (hyper-DMRs) in Tet6?/? cells exhibit several hallmarks of enhancers including: evolutionary sequence conservation,enr\"\\tEnhancer\\tSlit3\\t--\\t\"PCR,Transgenic mice\"\\t\"we also differentiated a previously reported Tet2 knockdown mESC line and performed quantitative real-time PCR analysis. Consistent with our observations in TET2?/? mESCs, we also find delayed induction of the three marker genes slit3, lmo4, and irx3\"\\t\"Slil2,Slit1,b2b2362.1Clo\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox4,Nanog\"\\t\"EZF,Gklf,Zie,NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4,2410002E02Rik,ENK,ecat4,Sox-2,lcc,ysb\"\\tChIP-seq\\t\"Weaker occupancy of the ES-cell core transcription factors (TFs) OCT4,SOX4,and NANOG than those that do not change DNA methylation status.\"\\t--\\t--\\t--\\t>2KB\\t\\t692951', 'E_02_110\\t25271055\\t--\\tmm10\\tchr11\\t102006458\\t102008458\\tMouse\\tOsteocyte Cell Lines\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\t\"Three lines of evidence support a direct role for HDAC5 in regulating MEF2C activity at the +45 kB SOST enhancer over the course of osteocyte differentiation. First, HDAC5 shRNA causes increased activity of this element, but not of the proximal SOST promoter, in luciferase assays. MEF2C shRNA reduces the activity of this enhancer. Second, HDAC5 overexpression dose-dependently inhibits the activity of a MEF2-driven reporter from the desmin and the SOST9 (+45kB) enhancer. Third, endogenous HDAC5 association with this region in control (but not HDAC5 shRNA) Ocy454 cells can be detected by ChIP.\"\\tEnhancer\\tSost\\t--\\t\"ChIP,qRT-PCR,Western blot\"\\t\"Using Chromatin immunoprecipitation,we have mapped endogenous MEF2C binding in the SOST gene to a distal intergenic Enhancer 45 downstream from the transcription start site.\"\\t5430411E23Rik\\t--\\t--\\t--\\t--\\t--\\t--\\tMef2c\\t\"5430401D19Rik,9930028G15Rik,AV011172,Mef2\"\\t\"ChIP,PCR\"\\tWe performed Chromatin immunoprecipitation (ChIP) to determine MEF2C occupancy in Ocy454 cell cultured at 37C for 14 days (a time point in which SOST expression is High throughput).\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t45001', 'E_02_111\\t25321476\\t--\\tmm10\\tchr10\\t34385685\\t34385834\\tMouse\\tMCT Cells\\tLow throughput\\t\"Transgenic mice,ChIP,qRT-PCR,EMSA\"\\t\"These results show that multiple copies of the enhancer (150-bp Col10a1 distal promoter) mediates higher level and cellspecific reporter activity, whereas the 10-kb Col10a1 promoter/intronic element contains nonspecific regulatory elements,in addition to the 150-bp enhancer.\"\\tEnhancer\\tCol10a1\\t--\\t\"ChIP,qRT-PCR,EMSA\"\\tTop panel displays Col10a1 gene structure and the 10 -kb promoter and intronic element.Positions of the 150-bp (?4296 to ?4147 bp) Col10a1 cis-Enhancer (purple bar) and its 330-bp (?220 to +110 bp) basal promoter (red bar) are shown.\\t\"Col10,Col10a-1\"\\tSkeletal Diseases\\t--\\t--\\t--\\t--\\t--\\tRunx2\\t\"AML3,Cbf,Cbfa-1,Cbfa1,LS3,Osf2,PEBP2aA,Pebp2a1,Pebpa2a\"\\t\"qRT-PCR,EMSA\"\\t\"The qRT-PCR was performed to examine the mRNA transcript level of marker genes Col10a1 and Runx2 in MCT cells. As illustrated, both Col10a1 and Runx2 are significantly upregulated in hypertrophic MCT cells compared with that in proliferative MCT cells.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t4220', 'E_02_112\\t25369933\\t--\\tmm10\\tchr15\\t63254341\\t63256341\\tMouse\\tT-acute lymphoblastic leukemia (T-ALL)\\tLow+High throughput\\t\"ChIP-seq,EMSA,3C\"\\t\"Notch1 is depleted from the NDME site by short-term gammasecretase inhibitor (GSI) treatment, and loading and unloading of Notch1 is associated with rapid changes in H3K27ac, features that characterize genomic Notch1 binding sites that dynamically regulate gene expression.Notch1 also bound the NDME in primary murine T-ALL cells and in DN3 thymocytes, a stage of T-cell development marked by high levels of Notch1 activation\"\\tEnhancer\\tMyc\\t3C\\t\"ChIP,qRT-PCR,Luciferase Reporter Assay\"\\t\"Schematic of the Luciferasegene constructs including the NDME WT and mutant sequences.To test for chromatin looping between the NDME and the Myc promoter, chromatin conformation capture (3C) assays were carried out in T6E cells.\"\\t\"AU0167572,Niard,Nird,bHLHe39,?Myc\"\\tAcute Lymphoblastic Leukemia\\tDOID:995D054198\\tAltered long-range enhancer activity can mediate resistance to targeted therapies and provide a mechanistic rationale for combined targeting of Notch and Brd4 in leukemia.\\t3C\\t\"To determine whether Notch signaling was required to maintain the interaction between the Myc promoter and the NDME,3C analysis was performed in T6E cells cultured in the presence or absence of GSI.\"\\tNotch\\t\"9930111A19Rik,Mis6,N1,Tan1,lin-12\"\\t\"ChIP-seq,3C\"\\tAnalysis of the ChIP-seq data revealed an RBPJ/Notch1 binding site associated with high levels of H3K4me1 marks located 1.43 Mb 3 of the Myc promoter (Fig. 4A) that shares sequence homology with the murine NDME.\\t--\\t--\\t--\\t>2KB\\t\\t\\t1270001', 'E_02_113\\t25403490\\t--\\tmm10\\tchr7\\t53527011\\t53528648\\tMouse\\tC2C12\\tLow throughput\\t\"ChIP,PCR\"\\t\"To gain insight into the mechanism by which MUNC facilitates muscle differentiation, we examined by ChIP the binding of MyoD and myogenin to the regulatory sites known to bind the two transcription factors, the core enhancer region (CER) at -20kb relative to MyoD TSS and theDRR (at -5kb relative to MyoD TSS), and to the sites in the -0.5kb region of the Myogenin promoter\"\\tEnhancer\\tMyod\\t--\\t\"Western blot,qRT-PCR,Immunofluorescence\"\\t\"RNAs 2 and 3 were of particular interest be_x0002_cause they were located upstream of the TSS of MyoD,a master regulator of skeletal muscle differentiation (12),with ncRNA 2 overlapping a previously known MyoD Enhancer element,known as the distal regulatory region (DRR).\"\"AI503393,MYF3,MyoD,?Myod-1,bHLHc1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tMyod1\\t\"AI503393,MYF3,MyoD,Myod-1,bHLHc1\"\\tChIP\\t\"To gain insight into the mechanism by which MUNC facilitates muscle differentiation,we examined by ChIP the binding of MyoD and myogenin to the regulatory sites known to bind the two transcription factors,the core en_x0002_hancer region (CER) at kb _x0006_20 relative to MyoD TSS and the DRR,(at kb _x0006_5 relative to MyoD TSS),and to the sites in the kb _x0006_0.5 region of the Myogenin promoter.\"\\t--\\t--\\t--\\t>2KB\\t7151357', 'E_02_114\\t25453760\\t--\\tmm10\\tchr2\\t167687771\\t167689503\\tMouse\\tMonocytes\\tLow+High throughput\\t\"ChIP-seq,ATAC-seq\"\\tanalysis of active enhancers (H3K27ac) revealed the same relationship between H3K27ac intensity and cell type specific genes. ChIP-Seq of in vivo splenic pDCs showed similar patterns to our in vitro pDC model with respect to H3K4me1 and H3K27ac in enhancer regions associated to pDC and moDC specific genes\\tEnhancer\\tCebpb\\t--\\t\"qPCR,RNA-seq,Western blot\"\\t\"Taken together,our results suggest that Irf8 and Cebpb form a double negative feedback loop and a positive self-auto-regulatory loop.Such a composite self-reinforcing negative feedback loop confers bi-stability,leading to either expression of one TF or the other;as a consequence, the entire epigenetic landscape is directed towards either a pDC or moDC Enhancer state.\"\\t\"C/EBPbeta,CRP2,IL-6DBP,LAP,LIP,NF-IL6,NF-M,Nfil6\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Tcf4,Spib,Bcl11a\"\\t\"5730422P05Rik,ASP-I2,E2-2,E2.2,ITF-2,ITF-2b,ITF2,ME2,MITF-2A,MITF-2B,SEF-2,SEF2,SEF2-1,TFE,Tcf-4,bHLHb19,Spi-B,2810047E18Rik,BCL-11A,Ctip1,D930021L15Rik,Evi9,Evi9a,Evi9b,Evi9c,mKIAA1809\"\\t\"ATAC-seq,ChIP-seq\"\\t\"To better focus on open chromatin accessible to TF binding within these Enhancers, we narrowed down our search to overlapping peak of ATAC-seq signal.\"\\t--\\t--\\t--\\t<2KB\\t277', 'E_02_115\\t25453760\\t--\\tmm10\\tchr4\\t66837166\\t66852551\\tMouse\\tDendritic Cells\\tLow+High throughput\\t\"ChIP-seq,ATAC-seq\"\\tanalysis of active enhancers (H3K27ac) revealed the same relationship between H3K27ac intensity and cell type specific genes. ChIP-Seq of in vivo splenic pDCs showed similar patterns to our in vitro pDC model with respect to H3K4me1 and H3K28ac in enhance\\tEnhancer\\tTlr4\\t--\\t\"qPCR,RNA-seq,Western blot\"\\t\"Specifically,we profiled genome-wide epigenetic modifications corresponding to candidate Enhancers (monomethylation of histone 3 lysine 4,H3K4me1),active Enhancers (H3K27ac) and promoter regions (H3K4me3) in both moDCs and pDCs.\"\\t\"Lps,Ly87,Ran/M1,Rasl2-8\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Tcf4,Spib,Bcl11a\"\\t\"5730422P05Rik,ASP-I2,E2-2,E2.2,ITF-2,ITF-2b,ITF2,ME2,MITF-2A,MITF-2B,SEF-2,SEF2,SEF2-1,TFE,Tcf-4,bHLHb19,Spi-B,2810047E18Rik,BCL-11A,Ctip1,D930021L15Rik,Evi9,Evi9a,Evi9b,Evi9c,mKIAA1809\"\\tChIP\\t\"Molecularly,pDCs display many features of lymphocytes such as the B cell marker (B220),nucleic acid-sensing TLRs (TLR7,9) and transcription factors (Tcf4,SpiB,Bcl11a),whereas cDCs and moDCs express high levels of receptors that sense bacterial components (TLR 2,4) and inflammatory response genes.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t17309', 'E_02_116\\t25453760\\t--\\tmm10\\tchr8\\t120737580\\t120746913\\tMouse\\tMonocytes\\tLow+High throughput\\t\"ChIP-seq,ATAC-seq\"\\tanalysis of active enhancers (H3K27ac) revealed the same relationship between H3K27ac intensity and cell type specific genes. ChIP-Seq of in vivo splenic pDCs showed similar patterns to our in vitro pDC model with respect to H3K4me1 and H3K29ac in enhance\\tEnhancer\\tIrf8\\t--\\t\"qPCR,RNA-seq,Western blot\"\\t\"Taken together,our results suggest that Irf8 and Cebpb form a double negative feedback loop and a positive self-auto-regulatory loop.Such a composite self-reinforcing negative feedback loop confers bi-stability,leading to either expression of one TF or the other;as a consequence, the entire epigenetic landscape is directed towards either a pDC or moDC Enhancer state.\"\\t\"AI893568,ICSBP,IRF-8,Icsbp1,Myls\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Tcf4,Spib,Bcl11a\"\\t\"5730422P05Rik,ASP-I2,E2-2,E2.2,ITF-2,ITF-2b,ITF2,ME2,MITF-2A,MITF-2B,SEF-2,SEF2,SEF2-1,TFE,Tcf-4,bHLHb19,Spi-B,2810047E18Rik,BCL-11A,Ctip1,D930021L15Rik,Evi9,Evi9a,Evi9b,Evi9c,mKIAA1809\"\\t\"ATAC-seq,ChIP-seq\"\\t\"To better focus on open chromatin accessible to TF binding within these Enhancers, we narrowed down our search to overlapping peak of ATAC-seq signal.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5890', 'E_02_117\\t25486239\\tARC-specific Enhancer\\tmm10\\tchr1\\t133303739\\t133325138\\tMouse\\tKisspeptin-Immunoreactive Cell\\tLow+High throughput\\t\"DNase-seq,Immunofluorescence,Luciferase Reporter Assay,PCR,3C\"\\t\"Using open Chromatin mapping by DNase-seq in HTE cell,the core of DHS-44 was localized to 600 at hg 19,Chr7:117075400-117076000.This region was marked by enrichment of H3K27Ac in 16HBE14o-cell,consistent with DHS-44 encompassing an active Enhancer element.\"\\tEnhancer\\tKiss1\\t3C\\tTransgenic mice\\t\"The present study aimed to determine ARC-specific enhancer(s) of the Kiss1 gene by an in vivo reporter assay.The 3C assay was performed to detect interactions between the Kiss1 promoter region and the 5 -upstream region, the latter of which was identified as an ARCspecific enhancer in the present study\"\\t\"kisspeptin,metastatin\"\\t--\\t--\\t--\\tThe 5\\'-upstream region of Kiss1 locus functions as an enhancer for ARC Kiss1 gene expression in mice.\\t3C\\t\"In addition,the current 3C and ER ChIP assays suggest that the Chromatin loop formation between the ARC-specific Enhancer and the promoter region of Kiss1 gene and unoccupied ER recruitment to the putative Enhancer region are involved in Kiss1 gene expression in the ARC\"\\tEral1\\t\"2610524P08Rik,9130407C09Rik,AU019798,Era,M-ERA,MERA-S,MERA-W\"\\tChIP\\tThe binding of ERa in the 5\\' upstream region (Figure 4) was determined by a ChIP assay with ERa antibody.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t7447', 'E_02_118\\t25486255\\t--\\tmm10\\tchr3\\t34743178\\t34776096\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RNA-seq\"\\t\"We examined ENCODE ChIP-seq data from 23 different mouse tissues or cell types focusing on the Sox2 locus. Using H3K27ac, a histone mark for active enhancers, we observed a distal SE that is only present in mouse ESC lines, which is approximately 100kb downstream from the Sox2 locus. This sequence spans a relatively large 13kb region, and corresponds to a recently defined super-enhancer or stretch enhancer.\"\\tSuper-Enhancer\\tSox2\\tCRISPR/Cas9\\tPCR\\tWe analyzed epigenomic data within the 1.5 Mb gene-desert regions around the Sox2 gene and identified a 13kb-long super-Enhancer (SE) located 100kb downstream of Sox2 in mouse ESCs.\\t\"Sox-2,lcc,ysb\"\\t--\\t--\\t--\\t\"SE is occupied by Oct4, Sox2, Nanog,and the mediator complex, and physically interacts with the Sox2 locus via DNA looping\"\\t\"RT-qPCR,ChIP-seq\"\\t\"To test the in vivo function of Sox2-SEdistal, we sought to delete the entire Enhancer sequence from the endogenous locus. Given that the SE spans a,13kb region, deletion of this region using conventional methods would be very inefficient.Therefore, we explored whether the recently developed CRISPR technology could be utilized to delete this large non-coding sequence in mouse ESCs.\"\\t\"Oct4,Sox2,Nanog\"\\t\"2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1,Sox-2,lcc,ysb,2410002E02Rik,ENK,ecat4\"\\tChIP-seq\\t\"First, we observed strong H3K27ac signal at this SE in two additional mouse ES cell lines (F123 and J1), indicating that usage of this SE is conserved across different mice strains.Previously published ChIP-seq data shows that Sox2-SEdistal is occupied by Oct4,Sox2 and Nanog.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t109643', 'E_02_119\\t25487574\\t--\\tmm10\\tchr11\\t98979280\\t98979880\\tMouse\\tCOS-7\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\t\"The Cx30.2-Luc reporter construct was generated by subcloning a 4.1-kb Cx30.2 enhancer fragment (-4.1 to 0.0 relative to the translation start site) using KpnI-XhoI into pGL2 E1b LUC, which contains a minimal TATA box, followed by the luciferase coding sequence.\"\\tEnhancer\\tGjd3\\t--\\t\"Western blot,ChIP,qRT-PCR\"\\t\"Consistent with our in vitro data, MyoR associated more strongly with the Cx30.2 Enhancer in the AVC relative to the atria and ventricles. Furthermore, sequen_x0002_tial ChIP experiments with MyoR and Gata4 antibodies demon_x0002_strated that both MyoR and Gata4 simultaneously occupy the Cx30.2 Enhancer in vivo.\"\\t\"Gja11,Gjc1,cx30.2\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Msc,Tbx5,Nkx2-5\"\\t\"MyoR,bHLHa22,Tbx5,Csx,Nkx-2.5,Nkx2.5,tinman\"\\t\"Luciferase Reporter Assay,ChIP,qRT-PCR\"\\tHuman and mouse genetic studies have demonstrated that the transcription factors Tbx5 and Nkx2.5 function during development to specify lower AV nodal cells and promote their morphogenesis.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2599', 'E_02_120\\t25487574\\tCx30.2 Enhancer\\tmm10\\tchr14\\t63196014\\t63196614\\tMouse\\tCx30.2+ Cells\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\t\"The Cx30.2-Luc reporter construct was generated by subcloning a 4.1-kb Cx30.2 enhancer fragment (-4.1 to 0.0 relative to the translation start site) using KpnI-XhoI into pGL2 E1b LUC, which contains a minimal TATA box, followed by the luciferase coding sequence.\"\\tEnhancer\\tGata4\\t--\\t\"Western blot,ChIP,qRT-PCR\"\\tSchematic depiction of the Cx30.2-Luciferase construct used in panel B that contains the Cx30.2 Enhancer fused to a minimal TATA box and Luciferase coding sequence. The 4.1- Enhancer fragment encompasses the _x0004_2.9/_x0004_2.3 region that contains the minimal AVC-specific Cx30.2 Enhancer.\\tGata-4\\t--\\t--\\t--\\t--\\t--\\t--\\tMsc\\t\"MyoR,bHLHa22\"\\tChIP\\tCoimmunoprecipitation analysis between MyoR and Gata4. Myc-Gata4 and either Flag vector alone or Flag-MyoR were cotransfected into COS7 cell.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2599', 'E_02_121\\t25505291\\tECR2 Enhancer\\tmm10\\tchr4\\t136148484\\t136149239\\tMouse\\tHep G2\\tLow throughput\\tLuciferase Reporter Assay\\t\"Because enhancers were previously described for Id3 (32), we searched for evolutionally conserved regions that may function as enhancers and enable Smad2/3-dependent upregulation of Id3 in macrophages. Using ECR browser, we found two conserved regions, ECR1 located between -3177 and -2660 bp upstream of transcription start and ECR2 located between +4517 and 4662 bp downstream of the gene . ECR1 overlaps with the enhancer described by Shepherd et al. whereas ECR2 has not been described to date.Luciferase activity of pGL3-ID3prom ECR1 ECR2 plasmid cotransfected with plasmids containing BMP-specific Smad1 and Smad5.\"\\tEnhancer\\tId3\\t--\\t\"siRNA,RT-PCR\"\\t\"To assess the effect of chromatin structure on Id3 gene expression, we used one panC-HDAC inhibitor (TSA) and two selective HDAC inhibitors: MS-275, an inhibitor of HDAC1, and Apicidin, an inhibitor of HDAC2 and HDAC3.\"\\t\"Hlh462,Idb3,bHLHb25\"\\t--\\t--\\t--\\t--\\t--\\t--\\tTgfb1\\t\"TGF-beta1,TGFbeta1,Tgfb,Tgfb-1\"\\tWestern blot\\t\"Western blot analysis of Smad phosphorylation upon stimulation of primary Human monocytederived macrophages with TGF-b1,BMPs,and their combinations.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5041', 'E_02_122\\t25505291\\tECR1 Enhancer\\tmm10\\tchr4\\t136140654\\t136141162\\tMouse\\tHep G2\\tLow throughput\\tLuciferase Reporter Assay\\t\"Because enhancers were previously described for Id3 (32), we searched for evolutionally conserved regions that may function as enhancers and enable Smad2/3-dependent upregulation of Id3 in macrophages. Using ECR browser, we found two conserved regions, ECR1 located between -3177 and -2660 bp upstream of transcription start and ECR2 located between +4517 and 4662 bp downstream of the gene . ECR1 overlaps with the enhancer described by Shepherd et al. whereas ECR2 has not been described to date.Luciferase activity of pGL3-ID3prom ECR1 ECR2 plasmid cotransfected with plasmids containing BMP-specific Smad1 and Smad5.\"\\tEnhancer\\tId3\\t--\\t\"siRNA,RT-PCR\"\\t\"To assess the effect of chromatin structure on Id3 gene expression, we used one panC-HDAC inhibitor (TSA) and two selective HDAC inhibitors: MS-275, an inhibitor of HDAC1, and Apicidin, an inhibitor of HDAC2 and HDAC4.\"\\t\"Hlh462,Idb3,bHLHb25\"\\t--\\t--\\t--\\t--\\t--\\t--\\tTgfb1\\t\"TGF-beta1,TGFbeta1,Tgfb,Tgfb-1\"\\tWestern blot\\t\"Western blot analysis of Smad phosphorylation upon stimulation of primary Human monocytederived macrophages with TGF-b1,BMPs,and their combinations.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2913', 'E_02_123\\t25548254\\tE8VI Enhancer\\tmm10\\tchr6\\t71341692\\t71345212\\tMouse\\tCD4+ T Cell\\tLow throughput\\t\"ChIP,PCR\"\\t\"Strikingly, 6 ECRs (ECR-2, -3, -6, -7, -8, and -10) overlapped with previously identified Cd8 enhancers. This indicated a good correlation between the location of already described Cd8 cis-regulatory elements and the presence of ECRs.\"\\tEnhancer\\t\"Cd8a,Cd8b1\"\\t--\\t\"Transgenic mice,PCR,ChIP\"\\t\"ChIP assays revealed that E8VI was bound by Runx/CBFb complexes and Bcl11b,and the analysis of Runx3-null CD8+T cell showed that E8VI activity was,in part,dependent on Runx3.\"\\t\"BB154331,Ly-2,Ly-35,Ly-B,Lyt-2,Cd8b,Ly-3,Ly-C,Lyt-3\"\\t--\\t--\\t--\\t--\\t--\\t--\\tBcl11b\\t\"9130430L19Rik, AI604821, B630002E05Rik, BCL-11B, Ctip2, Rit1\"\\tChIP\\t\"ChIP assays revealed that E8VI was bound by Runx/CBFb complexes and Bcl11b,and the analysis of Runx3-null CD8+T cell showed that E8VI activity was,in part,dependent on Runx3.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t29974', 'E_02_124\\t25582196\\t--\\tmm10\\tchr10\\t62893206\\t62923461\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"To identify cis-regulatory elements in Tet1, we surveyed its epigenome in mouse ESCs using ChIP-seq data sets.The combinatorial binding profiles of pluripotency factors, Mediator,and cohesin units have previously delineated a superenhancer domain in Tet1, a 15.2-kb region that spans from exon 1c at the distal end to a conserved noncoding sequence (CNS) upstream of the CDS ChIP-seq showed strong binding profiles of Oct4 within the CNS,as previously shown, but also several more strong peaks in the distal region between exons 1b and 1c.\"\\tSuper-Enhancer\\tTet1\\t--\\t\"PCR,EMSA\"\\t\"Here we defined the promoter and Enhancer domains in Tet1 and Tet2.Within a 15- superEnhancer of Tet1,there are two transcription start sites (TSSs) with different activation patterns during development.\"\\t\"2510010B09Rik,AA517754,BB001228,Cxxc6,D10Ertd17e,LCX,mKIAA1676\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox2\"\\t\"2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1,Sox-2,lcc,ysb\"\\tEMSA\\tContribution of Oct and Sox motifs in Tet1 and Tet2 enhancer fragments.\\t--\\t--\\t--\\t>2KB\\t\\t103765', 'E_02_125\\t25582196\\t--\\tmm10\\tchr3\\t133531418\\t133563926\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"To identify cis-regulatory elements in Tet1, we surveyed its epigenome in mouse ESCs using ChIP-seq data sets.The combinatorial binding profiles of pluripotency factors, Mediator,and cohesin units have previously delineated a superenhancer domain in Tet2,?\"\\tSuper-Enhancer\\tTet2\\t--\\t\"PCR,EMSA\"\\t\"A second TSS downstream,associated with a constitutively weak CpG-rich promoter,is activated by a neighboring Enhancer in naive embryonic stem cell (ES cells) and primed epiblast-like cell (EpiLCs).\"\\t\"Ayu17-449,E130014J05Rik,mKIAA1546\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox2\"\\t\"2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1,Sox-2,lcc,ysb\"\\tEMSA\\tContribution of Oct and Sox motifs in Tet1 and Tet2 enhancer fragments.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t83996', 'E_02_126\\t25605944\\t--\\tmm10\\tchr12\\t3944001\\t3947078\\tMouse\\t\"AtT-20,Pituitary Corticotrope Cell\"\\tLow+High throughput\\t\"ChIP-seq,3C,ChIP\"\\t\"Further, we have defined the subset of the enhancers that have active chromatin markers (H3K27ac and p300).Motif enrichment analysis of H3K27ac+ p300+ enhancers revealed enrichment of binding sites for the CCCTC-binding factor (CTCF), neurofibromin 1 (NF1), bHLH, FORKHEAD, X-BOX,and T-BOX families of transcription factors. Although members of the CTCF and NF1 family are general cellular transcription factors, the factors in the bHLH and other families may be critical for lineage development and enhancer programming in corticotropes.\"\\tEnhancerPomc\\t\"3C,CRISPR/Cas9\"\\t\"qPCR,ChIP,GRO-seq\"\\tWe found that enhancer:promoter interactions between LDB1-bound enhancers and the adjacent promoters of POMC and lung carcinoma myc related oncogene 1 (Lmyc1) genes were greatly diminished following knockdown of Ldb1.\\t\"ACTH,BE,Beta-LPH,Clip,Gamma-LPH,Npp-1,Pomc1,alpha-MSH,alphaMSH,beta-MSH,gamma-MSH,?Pomc\"\\t--\\t--\\t--\\tLDB1-mediated enhancer:promoter looping appears to be required for both activation and repression of these target genes\\t\"qPCR,3C,GRO-seq\"\\t\"Together,our data reveal that Enhancer-bound LDB1 has a role in mediating the actions of both activating and repressive Enhancers in a cell type_x0002_specific gene-regulation program.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t9404', 'E_02_127\\t25605944\\t--\\tmm10\\tchr12\\t3956137\\t3961266\\tMouse\\t\"AtT-20,Pituitary Corticotrope Cell\"\\tLow+High throughput\\t\"ChIP-seq,3C,ChIP\"\\t\"Further, we have defined the subset of the enhancers that have active chromatin markers (H3K27ac and p300).Motif enrichment analysis of H3K27ac+ p300+ enhancers revealed enrichment of binding sites for the CCCTC-binding factor (CTCF), neurofibromin 1 (NF1), bHLH, FORKHEAD, X-BOX,and T-BOX families of transcription factors. Although members of the CTCF and NF1 family are general cellular transcription factors, the factors in the bHLH and other families may be critical for lineage development and enhancer programming in corticotropes.\"\\tEnhancerPomc\\t\"3C,CRISPR/Cas9\"\\t\"qPCR,ChIP,GRO-seq\"\\tWe found that enhancer:promoter interactions between LDB1-bound enhancers and the adjacent promoters of POMC and lung carcinoma myc related oncogene 1 (Lmyc1) genes were greatly diminished following knockdown of Ldb2.\\t\"ACTH,BE,Beta-LPH,Clip,Gamma-LPH,Npp-1,Pomc1,alpha-MSH,alphaMSH,beta-MSH,gamma-MSH,?Pomc\"\\t--\\t--\\t--\\tLDB1-mediated enhancer:promoter looping appears to be required for both activation and repression of these target genes\\t\"qPCR,3C,GRO-seq\"\\t\"Together,our data reveal that Enhancer-bound LDB1 has a role in mediating the actions of both activating and repressive Enhancers in a cell type_x0002_specific gene-regulation program.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3758', 'E_02_128\\t25605944\\t--\\tmm10\\tchr12\\t3938522\\t3942093\\tMouse\\tPituitary Corticotrope Cell\\tLow+High throughput\\t\"ChIP-seq,3C,ChIP\"\\t\"Further, we have defined the subset of the enhancers that have active chromatin markers (H3K27ac and p300).Motif enrichment analysis of H3K27ac+ p300+ enhancers revealed enrichment of binding sites for the CCCTC-binding factor (CTCF), neurofibromin 1 (NF1), bHLH, FORKHEAD, X-BOX,and T-BOX families of transcription factors. Although members of the CTCF and NF1 family are general cellular transcription factors, the factors in the bHLH and other families may be critical for lineage development and enhancer programming in corticotropes.\"\\tEnhancer\\tPomc\\t\"3C,CRISPR/Cas9\"\\t\"qPCR,ChIP,GRO-seq\"\\tWe found that enhancer:promoter interactions between LDB1-bound enhancers and the adjacent promoters of POMC and lung carcinoma myc related oncogene 1 (Lmyc1) genes were greatly diminished following knockdown of Ldb3.\\t\"ACTH,BE,Beta-LPH,Clip,Gamma-LPH,Npp-1,Pomc1,alpha-MSH,alphaMSH,beta-MSH,gamma-MSH,?Pomc\"\\t--\\t--\\t--\\tLDB1-mediated enhancer:promoter looping appears to be required for both activation and repression of these target genes\\t\"ChIP,PCR\"\\t\"Together,our data reveal that Enhancer-bound LDB1 has a role in mediating the actions of both activating and repressive Enhancers in a cell type_x0002_specific gene-regulation program.--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t14636', 'E_02_129\\t25644605\\tUcp1 Enhancer\\tmm10\\tchr8\\t83287848\\t83288068\\tMouse\\tBrown Fat Cell\\tLow throughput\\t\"ChIP,RT-PCR\"\\twe hypothesized that PRDM16 might be recruited to the enhancer region of the Ucp1 gene through the interaction with PGC-1a. This possibility was tested in an immobilized template assay with purified proteins and a beadimmobilized DNA fragment spanning the 220-bp Ucp1 enhancer.\\tEnhancer\\tUcp1\\t--\\t\"ChIP,RT-PCR\"\\t\"Overall, these results indicate that the ZF1 domain is required for the recruitment of PRDM16 to the Ucp1 enhancer through MED1 and PGC-1a.\"\\t\"AI385626,Slc25a7,Ucp\"\\t--\\t--\\t--\\t--\\t--\\t--\\tPrdm16\\t\"5730557K01Rik,csp1,mel1ChIP\\t\"To determine whether PRDM16 is recruited to the Enhancer following Ucp1 induction, we treated Med1 +/+MEFs that stably express PPARg and PRDM16 with forskolin and performed a chromatin immunoprecipitation (ChIP) assay with an antibody specific for PRDM16.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2389', 'E_02_130\\t25651906\\tMyoD distal Enhancer\\tmm10\\tchr7\\t46351197\\t46355693\\tMouse\\tC2C12\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\t\"C2C12 myoblasts stably carrying the P6P (?5870 to +95) -driven reporter (MyoD-P6P-luc) or another composite reporter,MyoD-PE-luc, containing P6P plus the DE (?25277 to?20780) were treated with Wnt3a medium. We found that Wnt3a marginally, but significantly, activated MyoDP6P-luc activity; however, Wnt3a strongly activated MyoD-PE-luc, suggesting that one or multiple Wnt-response elements(WREs) are located within the DE.\"\\tEnhancer\\tMyod1\\t--\\t\"Western blot,qRT-PCR\"\\t\"A distal Enhancer(DE) centred at ? 20 kb, when combined with a ? 2.5k proximal pro_x0002_moter, can also recapitulate human MyoD expression in vivo,demonstrating its critical role in determining the spatiotemporal expression of human MyoD.\"\\t\"AI503393,MYF3,MyoD,?Myod-1,bHLHc1\"\\t--\\t--\\t--\\t--\\t--\\t--\\tWnt3a\\t\"Wnt-3a,vt\"\\tLuciferase Reporter Assay\\t\"Serial deletion mutants of DE region were generated later to identify the minimal WRE,and we found that a region of 3 kb (fragment 10, ?24 to ?21 kb)in DE was required to retain the full Wnt3a response.Deletion of sequence at either the 5_x0002_ end (fragment 8) or the 3_x0002_ end(fragment 7) significantly reduced its Wnt3a response, comparedwith fragment.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t23028', 'E_02_131\\t25652130\\tGrhl3 Enhancer\\tmm10\\tchr4\\t135548688\\t135550688\\tMouse\\tNMuMG\\tLow throughput\\t\"Luciferase Reporter Assay,3C\"\\tWe reported previously that intron 2 of Cdh1 is essential for proper E-cad expression.In order to analyze this large DNA sequence(>40 kb) for putative enhancers we focused on evolutionary conserved sequences.\\tEnhancer\\tGrhl3\\t3C\\t\"ChIP,qPCR\"\\t\"Weshow thatCdh1 activity during MET isgovernedby 33 two Enhancers at +7.8 and at +11.5 within intron 2 that are activated by binding of Grhl3 and Hnf4, 34 respectively.\"\\t\"AI561912,Get1,Som,Tfcp2l4,ct\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t7801', 'E_02_132\\t25652130\\t--\\tmm10\\tchr8\\t106613868\\t106615868\\tMouse\\tNMuMG\\tLow throughput\\t\"Luciferase Reporter Assay,3C\"\\tWe reported previously that intron 2 of Cdh1 is essential for proper E-cad expression.In order to analyze this large DNA sequence(>41 kb) for putative enhancers we focused on evolutionary conserved sequences.\\tEnhancer\\tCdh1\\t3C\\t\"ChIP,qPCR\"\\t\"Weshow that Cdh1 activity during MET is governedby two Enhancers at +7.8 and at +11.5 within intron 2 that are activated by binding of Grhl3 and Hnf4,respectively.\"\\t\"AA960649,ARC-1,E-cad,Ecad,L-CAM,UVO,Um\"\\t--\\t--\\t--\\t--\\t--\\t--\\tHnf4a\\t\"HNF-4,Hnf4lpha,MODY1,Nr2a1,TCF-14,Tcf14,Hnf4a\"\\t\"Luciferase Reporter Assay,ChIP\"\\t\"Using luciferase reporter assays, we first tested whether Hnf4 could activate Cl.5 in NMuMG cells.In contrast to Grhl3 enrichment on the Cl.3 intronic sites, Hnf4 recruitment to the chromatin was detectable already in untreated NMuMG cells under steady state conditions, suggesting a distinct function for this Enhancer.\"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t11519', 'E_02_133\\t25652130\\t--\\tmm10\\tchr8\\t109134068\\t109136068\\tMouse\\tNMuMG\\tLow throughput\\t\"Luciferase Reporter Assay,3C\"\\tWe reported previously that intron 2 of Cdh1 is essential for proper E-cad expression.In order to analyze this large DNA sequence(>42 kb) for putative enhancers we focused on evolutionary conserved sequences.\\tEnhancer\\tCdh1\\t3C\\t\"ChIP,qPCR\"\\t\"We show that Cdh1 activity during MET is governedby two Enhancers at +7.8 and at +11.5 within intron 2 that are activated by binding of Grhl3 and Hnf4,respectively.\"\\t\"AA960649,ARC-1,E-cad,Ecad,L-CAM,UVO,Um\"\\t--\\t--\\t--\\tEnhancer cooperativity as a novel mechanism underlying the transcriptional regulation of E-cadherin during mesenchymal to epithelial transition.\\tTransgenic mice\\tEnhancer cooperativity as a novel mechanism underlying the transcriptional regulation of E-cadherin during mesenchymal to epithelial transition.\\tGrhl3\\t\"AI561912,Get1,Som,Tfcp2l4,ct\"\\t\"3C,qPCR,ChIP\"\\tA mutation at site 7.8b alone was sufficient to abrogate Grhl3-dependent luciferase reporter activation and also the lacZ-reporter activity in transgenic embryos.\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t2531719', 'E_02_134\\t25652130\\t--\\tmm10\\tchr8\\t106610150\\t106612150\\tMouse\\tNon-tumorigenic Mouse Mammary Gland Cells\\tLow throughput\\t\"Luciferase Reporter Assay,3C\"\\tWe reported previously that intron 2 of Cdh1 is essential for proper E-cad expression.In order to analyze this large DNA sequence(>43 kb) for putative enhancers we focused on evolutionary conserved sequences.\\tEnhancer\\tCdh1\\t3C\\t\"ChIP,qPCR\"\\t\"We show that Cdh1 activity during MET is governed by two Enhancers at +7.8 kb and at +11.5 kb within intron 2 that are activated by binding of Grhl3 and Hnf4,respectively.\"\\t\"AA960649,ARC-1,E-cad,Ecad,L-CAM,UVO,Um\"\\t--\\t--\\t--\\tEnhancer cooperativity as a novel mechanism underlying the transcriptional regulation of E-cadherin during mesenchymal to epithelial transition.\\tChIP\\tEnhancer cooperativity as a novel mechanism underlying the transcriptional regulation of E-cadherin during mesenchymal to epithelial transition.\\t\"Grhl3,Hnf4a\"\\t\"AI561912,Get1,Som,Tfcp2l4,ct,HNF-4,Hnf4lpha,MODY1,Nr2a1,TCF-14,Tcf14,Hnf4a\"\\t\"Luciferase Reporter Assay,ChIP\"\\t\"Using luciferase reporter assays,we first tested whether Hnf4 could activate Cl.5 in NMuMG cells.We found that indeed Hnf4 was able to exert a robust enhancing effect on the E-cad Cl.5 reporter construct.Two different mutations in the putative Hnf4 binding site decreased the basal activity of this construct and inhibited its Hnf4 dependent activation.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t7801', 'E_02_135\\t25775043\\t--\\tmm10\\tchr17\\t35499359\\t35507923\\tMouse\\tMouse Embryonic Stem Cell\\tLow throughput\\t\"CRISPR/Cas9,3C,Luciferase Reporter Assay,ATAC-seq\"\\t\"We generated mouse embryonic stem cells (mESCs) expressing versions of Neisseria meningitidis (Nm) dCas9 fused with LSD1, a non-effector BirA affinity tag (BAT), or a KRAB repressor and used a viral delivery system for sgRNAs. We first targeted the well-characterized cis-regulatory region of Oct417, a factor critical for the ESC state. Oct4 expression is regulated by a proximal enhancer (OPE) active in epiblast cells, and a distal enhancer (ODE) active in mESCs and cells of the inner cell mass. Targeting of LSD1 to the ODE resulted in loss of Oct4 expression and appearance of OCT4-negative colonies accompanied by phenotypic changes compared to control dCas9-BAT cells targeted to the same enhancer.\"\\tEnhancer\\tOct-4\\t\"3C,CRISPR/Cas9\"\\t\"PCR,ChIP\"\\t\"For follow-up, we focused on the putative Enhancer with the highest differential score,Enh1.Test of Enh1 in a reporter assay confirmed its ability to enhance expression at comparable levels to an Oct4 DE sequence..\"\\t\"NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,?Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4\"\\t--\\t--\\t--\\t\"We conclude that the dCas9-LSD1 fusion protein allows for an effector dependent definition of functional, native enhancer elements that help to maintain a given cellular state. \"\\t3C\\t\"We conclude that the dCas9-LSD1 fusion protein allows for an effector dependent definition of functional,native Enhancer elements that help to maintain a given cellular state.Accordingly,dCas9-LSD1 provides a rapid and powerful approach to understanding distal cis-regulatory regions such as Enhancers without major disruption of the local genomic architecture.\"\\tTbx3\\tD5Ertd189e\\t\"PCR,Luciferase Reporter Assay,ChIP,3C\"\\t\"This previously unannotated ESC_x0002_specific Enhancer is positioned ~10kb upstream of the transcription factor Tbx3, a gene previously implicated in the maintenance of pluripotency20. We therefore hypothesized that Enh1 may function in the ESC network by regulating Tbx3 expression.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2391', 'E_02_136\\t25775043\\t--\\tmm10\\tchr5\\t119654061\\t119663588\\tMouse\\tMouse Embryonic Stem Cell\\tLow throughput\\t\"CRISPR/Cas9,3C,Luciferase Reporter Assay,ATAC-seq\"\\t\"We generated mouse embryonic stem cells (mESCs) expressing versions of Neisseria meningitidis (Nm) dCas9 fused with LSD2, a non-effector BirA affinity tag (BAT), or a KRAB repressor and used a viral delivery system for sgRNAs. We first targeted the well-c\"\\tEnhancer\\tTbx3\\t\"3C,CRISPR/Cas9\"\\t\"PCR,ChIP\"\\t\"This previously unannotated ESC specific Enhancer is positioned ~10kb upstream of the transcription factor Tbx3 (Fig. 1d), a gene previously implicated in the maintenance of pluripotency.We therefore hypothesized that Enh1 may function in the ESC network by regulating Tbx3 expression.\"\\tD5Ertd189e\\t--\\t--\\t--\\t\"We conclude that the dCas9-LSD1 fusion protein allows for an effector dependent definition of functional, native enhancer elements that help to maintain a given cellular state. \"\\t3C\\t\"We conclude that the dCas9-LSD1 fusion protein allows for an effector dependent definition of functional,native Enhancer elements that help to maintain a given cellular state.Accordingly,dCas9-LSD1 provides a rapid and powerful approach to understanding distal cis-regulatory regions such as Enhancers without major disruption of the local genomic architecture.\"\\tTbx3\\tD5Ertd189e\\t\"PCR,Luciferase Reporter Assay,ChIP,3C\"\\t\"This previously unannotated ESC_x0002_specific Enhancer is positioned ~10kb upstream of the transcription factor Tbx3, a gene previously implicated in the maintenance of pluripotency20. We therefore hypothesized that Enh1 may function in the ESC network by regulating Tbx3 expression.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t11493', 'E_02_137\\t25801169\\t--\\tmm10\\tchr1\\t13041306\\t13127367\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\tLuciferase Reporter Assay\\t\"The five super-enhancers show physical interactions with their respective associated genes and are located within insulated neighborhoods in the ESC genome , suggesting that these genes represent the bona fide physiological targets of the five SEs.\"\\tSuper-Enhancer\\tPrdm14\\tCRISPR/Cas9\\t--\\t\"The results showed that deletion of most (12/14) super-enhancer constituents led to reduced expression of the associated gene, while in one case (Prdm14 E5) the deletion caused a small increase in expression of the associated gene\"\\tPrdm14\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox2,Nanog\"\\t\"2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1,Sox-2,lcc,ysb,2410002E02Rik,ENK,ecat4ChIP-seq\\t\"ChIP-Seq binding profiles for OCT4,SOX2 and NANOG (merged) and Mediator (MED1) at the miR-290-295 locus in ES cells.Enhancer activity measured in luciferase reporter assays in wild type cells and the change in enhancer activity after OCT4 shutdown is plotted for each constituent enhancer within the super-enhancer. The super-enhancer is depicted as a black bar above the binding profiles. The difference in values after OCT4 shutdown is statistically significant for all constituents, except from miR-290-295 M1, M3 and M5.\"\\t--\\t--\\t--\\t>2KB\\t\\t29090', 'E_02_138\\t25801169\\t--\\tmm10\\tchr17\\t31794375\\t31829875\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\tLuciferase Reporter Assay\\t\"The five super-enhancers show physical interactions with their respective associated genes and are located within insulated neighborhoods in the ESC genome , suggesting that these genes represent the bona fide physiological targets of the five SEs.\"\\tSuper-Enhancer\\tSik1\\tCRISPR/Cas9\\t--\\t\"These five super-enhancers control transcription of Prdm14, miR-290-295, Sik1, Klf2 and Pou5f1 (Oct5), which play important roles in ESC self-renewal, pluripotency and differentiation.\"\\t\"Hrt-20,Msk,Sik,Sik-1,Snf1lk\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t32124', 'E_02_139\\t25801169\\t--\\tmm10\\tchr17\\t35501272\\t35506872\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\tLuciferase Reporter Assay\\t\"The five super-enhancers show physical interactions with their respective associated genes and are located within insulated neighborhoods in the ESC genome , suggesting that these genes represent the bona fide physiological targets of the five SEs.\"\\tSuper-Enhancer\\tPou5f1\\tCRISPR/Cas9\\t--\\tEnhancer activity measured in Luciferaseassays in wild type cell and the change in Enhancer activity after OCT4 shutdown is plotted for each constituent Enhancer within the Super-enahncer.\\t\"NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t4734', 'E_02_140\\t25801169\\t--\\tmm10\\tchr7\\t3183157\\t3229407\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\tLuciferase Reporter Assa\"The five super-enhancers show physical interactions with their respective associated genes and are located within insulated neighborhoods in the ESC genome , suggesting that these genes represent the bona fide physiological targets of the five SEs.\"\\tSuper-Enhancer\\tMir290-295\\tCRISPR/Cas9\\t--\\t\"These five super-enhancers control transcription of Prdm14, miR-290-295, Sik1, Klf2 and Pou5f1 (Oct5), which play important roles in ESC self-renewal, pluripotency and differentiation.\"\\t\"Mir290, Mirn290, mir-290a, mmu-miR-290, mmu-mir-290a\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_141\\t25804738\\t--\\tmm10\\tchr3\\t127641624\\t127642545\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,RNA-seq\"\\t\"A conserved 447?bp genomic fragment within this?Neurog2?3 Foxa1-bound peak showed enhancer activity specifically in midbrain floor plate and also in the cerebral cortex, when analysed in mouse transient transgenic experiments at E12.5\"\\tEnhancer\\tNeurog2\\t--\\t\"Transgenic mice,qRT-PCR\"\\t\"By contrast, the identification of a Foxa1-bound peak downstream of Neurog2 coding sequences (Neurog2 3) raised the possibility that Foxa2 might directly activate Neurog2 expression through this putative Enhancer region in embryonic mDA progenitors.\"\\t\"Atoh4,Math4A,bHLHa8,ngn-2,ngn2\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Otx2,Foxa1,Foxa2\"\\t\"E130306E05Rik,HNF3-beta,HNF3beta,Hnf-3b,Hnf3b,Tcf-3b,Tcf3b,Hnf-3a,Hnf3a,Tcf-3a,Tcf3a\"\\t\"ChIP-seq,RNA-seq\"\\tOur genome-wide Foxa1-bound cis-regulatory sequences from ChIP-Seq and Foxa1/2 candidate target genes from RNA-Seq analyses of embryonic midbrain dopamine cells also provide an excellent resource for probing mechanistic insights into gene regulatory networks involved in the differentiation of midbrain dopamine neurons.\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t8941', 'E_02_142\\t25810254\\t--\\tmm10\\tchr4\\t34888479\\t34889907\\tMouse\\tT3-1 Cells\\tLow throughput\\t\"qPCR,ChIP,3C\"\\t\"Using quantitative PCR (qPCR) on DNase-treated reverse-transcribed RNA from a gonadotrope cell line, we amplified fragments between ?5300 and ?3872 bp upstream of the Cga TSS, which includes most of the reported enhancer.\"\\tEnhancer\\tCga\\t3C\\t\"qPCR,ChIP\"\\t\"qPCR analysis confirmed the interaction of the distal enhancer regions (?6.7 to ?6.58 and ?4.9 to ?4.85 kbp) with the proximal Cga promoter in both the treated and untreated cells and,with greater frequency, in WT and eRNA-knockdown cells after normalization to the relative levels of the Fsh chimeric fragment\"\\t\"CG-alpha,FSHA,GPHA1,GPHalpha,HCG,LHA,Tsha,aGSU,alpha-GSU,alphaGSU,alphaSU\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t4585', 'E_02_143\\t26110280\\tCcnb1ip1 Enhancer\\tmm10\\tchr14\\t50787449\\t50787634\\tMouse\\tMitotic Cell\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP,RT-PCR\"\\t\"PCR amplification of JF1 genomic DNA using several primer pairs targeted to differentially methylated regions identified a 1.8 kbp genomic segment that was present in the JF1 strain but absent in the B6 strain . Sequence analysis revealed that this 1.8kbp segment consisted of a 1,579 bp interval deleted in the B6 genome, and the remaining 220 bp interval present in the B6 genome but in an inverted orientation.\"\\tEnhancer\\tCcnb1ip1\\t--\\t\"Luciferase Reporter Assay,ChIP,RT-PCR\"\\t\"We hypothesized the existence of an insulator-type element that prevents the spreadof DNA methylation within the1.8 k segment,and actually identified a 242- and a 185- fragments that were located adjacent to eachother and showed insulatorand Enhancer activities,respectively,in reporter assays. We designated these genomicregions as the Ccnb1ip1 insulator and the Ccnb1ip1Enhancer. The Ccnb1ip1 insulator showed Enhancer-blocking activity in the Luciferaseiferase assays andbarrieractivity in the colony formation assays. Further examination of the Ccnb1ip1locus in other mammalian speciesrevealed that the insulator and Enhancerare High throughput lyconserve damonga wide variety of species,and are located immediately upstream of the transcriptional start site of Ccnb1ip1.\"\\t\"Gm288,Hei10,mei4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1706', 'E_02_144\\t26163528\\tId2 Enhancer\\tmm10\\tchr12\\t25090158\\t25090663\\tMouse\\t\"HCT 116,SW480\"\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP-PCR,ChIP\"\\t\"Expression of Id2 was stimulated by Wnt signaling through the enhancer region of the Id2 promoter at the early stage of tumorigenesis in Apc+/716 (Apc716) mice. Genetic depletion of Id2 in Apc716 mice caused ?80% reduction in the number of ileal polyps, but had little effect on tumor size.\"\\tEnhancer\\tId2\\t--\\t\"Transgenic mice,RT-PCR,Western blot\"\\t\"Expression of Id2 was stimulated by Wnt signaling through the enhancer region of the Id2 promoter at the early stage of tumorigenesis in Apc+/716 (Apc716) mice. Genetic depletion of Id2 in Apc716 mice caused ?80% reduction in the number of ileal polyps, but had little effect on tumor size.\"\\t\"AI255428,C78922,Idb2,bHLHb26\"\\tColorectal Carcinoma\\tDOID:0080199\\tD015179\\t--\\t--\\t--\\tTcf4\\t\"5730422P05Rik,ASP-I2,E2-2,E2.2,ITF-2,ITF-2b,ITF2,ME2,MITF-2A,MITF-2B,SEF-2,SEF2,SEF2-1,TFE,Tcf-4,bHLHb19\"\\t\"ChIP-PCR,Luciferase Reporter Assay\"\\t\"To determine if mouse Id2 is also a target gene of Wnt signaling, we performed ChIP-PCR analysis of the binding of Tcf4 to the promoter region of the Id2 gene.Sequencing analysis revealed ten putative Tcf/Lef-binding consensus sequences in the Id2 promoter region spanning from ?4000 to ?1. As expected, Tcf4 bound to the DNA region spanning from ?3300 to ?3146 of the Id2 distal promoter.To test this Tcf4 binding region for enhancer activity, we conducted luciferase reporter assays using the constructs .The putative Id2-enhancer fragment significantly activated Wnt-signaling dependent transcription from the CMV minimal promoter in two colorectal cancer cell lines.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3387', 'E_02_145\\t26303528\\t--\\tmm10\\tchr8\\t84899088\\t84902859\\tMouse\\t\"Murine erythroleukemia (MEL) line 745A,32DEpo1,293T\"\\tLow throughput\\tChIP\\t\"We have shown by cotransfection and in vivo analyses that the small 950-bp region adjacent to the EKLF transcription start site is sufficient for tissue-restricted expression and wished to address whether the epigenetic profile of this region supports its importance. Database perusal of modified histone H3 interactions within the critical 950-bp region of the EKLF promoter/enhancer during hematopoiesis shows that levels of the H3K4me1 active enhancer mark increase exactly in parallel with activation of EKLF expression during hematopoiesis , that is, low within the hematopoietic repopulating cells and in myeloid (CMP and MPP) progenitors but high in MEPs and erythroblasts\"\\tEnhancerKlf1\\t--\\tshRNA\\t\"Collectively, these studies demonstrate that DEK is required for EKLF expression and directly interacts in vitro and in vivo with its cognate site in the EKLF EHS1 Enhancer.\"\\t\"Eklf,Nan\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t953', 'E_02_146\\t26321200\\tAHF Enhancer\\tmm10\\tchr13\\t83661050\\t83662862\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\t\"3C-qPCR,3C-seq\"\\t\"Multiple promoter-interacting elements were detected in the Mef2c lo_x0002_cus, of which some showed a lower signal in Ldb1-deficientEBs (Figure 5B). Importantly, we observed interactions of the', 'E_02_147\\t26321200\\tOFTRV Enhancer\\tmm10\\tchr8\\t59795280\\t59797080\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\t3C-qPCR\\t\"Using 3C-qPCR anal_x0002_ysis with the Hand2 promoter as a viewpoint, we observed a specific close proximity of the promoter with the OFTRV enhancer in d5 EBs from WT ESCs, but not in d5 EBs from Ldb1-deficient cells (Figure 5A).\"\\tEnhancer\\tHand2\\t3C-qPCR\\t--\\t\"Using 3C-qPCR anal_x0002_ysis with the Hand2 promoter as a viewpoint, we observed a specific close proximity of the promoter with the OFTRV enhancer in d5 EBs from WT ESCs, but not in d5 EBs from Ldb1-deficient cells (Figure 5A).\"\\t\"AI225906,AI661148,Ehand2,Hed,Th2,Thing2,bHLHa26,dHAND\"\\tHeart Failure\\t--\\tD006333\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t2475198', 'E_02_148\\t26364592\\tCNS1 Enhancer\\tmm10\\tchr10\\t78046488\\t78046636\\tMouse\\tThymic Medullary Epithelial Cells (mTECsLow throughput\\tqPCR\\t\"To determine the in vivo function of the Aire enhancer region,we generated knockout mice (CNS1-KO) with a deletion of 45 bp within CNS1 including both NF-B sites.We first studied thymic expression of Aire in purified thymic epithelial cell subsets-cTEC, mTEClo, and mTEChi-from homozygous CNS1-KO mice by qPCR targeting exons 7C9. Strik_x0002_ingly, Aire was not expressed in mTEChi from CNS1-KO mice,whereas the gene was highly expressed in mTEChi from WT mice(Fig. 2A).\"\\tEnhancer\\tAire\\t--\\t\"qPCR,Immunofluorescence,RT-PCR\"\\t\"To determine whether the upstream CNS1 region regulates these variants, we analyzed the expression levels of the majority of exons in the Aire coding region by qPCR.Aire-KO mouse results are shown in Supporting Information Fig.2B), indicating that CNS1 is required for the thymic expression of the majority of, if not all, Aire splice variants.Deficiency of Aire', 'E_02_149\\t26443845\\tCD47 Enhancer\\tmm10\\tchr16\\t49740654\\t49742654\\tMouse\\tG1E Cells\\tLow throughput\\t\"RT-qPCR,ChIP\"\\t\"CD47 gene contacts with its distal enhancer at ?80 kb dur_x0002_ing synchronized transcriptional elongation. (A) Map of the CD47 lo_x0002_cus. Blue bars indicate fragments of BglII, and red bars indicate the regions used for ChIP and RT-qPCR.\"\\tEnhancer\\tCd47\\t3C\\t\"RT-qPCR,ChIP\"\\t\"This locus has a well-character- ized Enhancer residing 114 kb upstream of the transcrip- tion start site (TSS),and its coding region encompasses 82 kb,thus lending itself to finely space-and time-re- solved 3C analysis (Jing et al. 2008) No- tably,additional dynamic contacts of the Enhancer with the coding region were observed at the positions of elon- gating RNAPII. Similar dynamic Enhancer gene body con- tacts were identified at the CD47 locus.CD47 gene contacts with its distal enhancer at ?80 kb dur_x0002_ing synchronized transcriptional elongation. (A) Map of the CD47 lo_x0002_cus. Blue bars indicate fragments of BglII, and red bars indicate the regions used for ChIP and RT-qPCR.\"\\t\"9130415E20Rik,AA407862,AI848868,AW108519,B430305P08Rik,IAP,Itgp\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t113598', 'E_02_150\\t26494787\\tNodal proximal epiblast Enhancer (PEE)\\tmm10\\tchr10\\t61403948\\t61408233\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP,ChIP-seq\"\\t\"ChIP (red) and input (blue) wiggle plot overlays showing enrichment of Lhx1 ChIP-seq density at Hesx1,Fzd8,Embigin,Nodal,Otx2,and Foxa3. Purple boxes indicate the positions of previously mapped Nodal Enhancer elements.\"\\tEnhancer\\tNodal\\t--\\t\"ChIP,ChIP-seq\"\\t\"Conditional inactivation of Lhx1 dis_x0002_rupts anterior definitive endoderm development and impedes node and midline morphogenesis in part due to severe disturbances in visceral endoderm displacement. Transcriptional profiling and ChIP-seq (chromatin immunopre_x0002_cipitation [ChIP] followed by high-throughput sequencing) experiments identified Lhx1 target genes, including numerous anterior definitive endoderm markers and components of the Wnt signaling pathway. Interestingly, Lhx1-binding sites were enriched at enhancers, including the Nodal-proximal epiblast enhancer element and enhancer regions controlling Otx2 and Foxa2 expression. \"\\tTg.413d\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Lhx1,Eomes\"\\t\"Lim1,C77258,TBR-2,Tbr2\"\\tChIP\\t\"Several putative Lhx1 target genes were represented in our ChIP data set. For example, Lhx1 binding was detected at two distinct regions at the Hesx1 locus, including a regulatory element in the 5 untranslated region (containing two Lhx1-binding motifs) and a 3 distal enhancer. Lhx1 ChIP peaks were also present upstream of Embigin exon 1.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t11880', 'E_02_151\\t26494787\\tNodal Enhancer (NDE)\\tmm10\\tchr10\\t61402781\\t61405448\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP,ChIP-seq\"\\t\"ChIP (red) and input (blue) wiggle plot overlays showing enrichment of Lhx1 ChIP-seq density at Hesx1,Fzd8,Embigin,Nodal,Otx2,and Foxa3. Purple boxes indicate the positions of previously mapped Nodal Enhancer elements.\"\\tEnhancer\\tNodal\\t--\\t\"ChIP,ChIP-seq\"\\t\"Conditional inactivation of Lhx1 dis_x0002_rupts anterior definitive endoderm development and impedes node and midline morphogenesis in part due to severe disturbances in visceral endoderm displacement. Transcriptional profiling and ChIP-seq (chromatin immunopre_x0002_cipitation [ChIP] followed by high-throughput sequencing) experiments identified Lhx1 target genes, including numerous anterior definitive endoderm markers and components of the Wnt signaling pathway. Interestingly, Lhx1-binding sites were enriched at enhancers, including the Nodal-proximal epiblast enhancer element and enhancer regions controlling Otx2 and Foxa2 expression. \"\\tTg.413d\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Lhx1,Eomes\"\\t\"Lim1,C77258,TBR-2,Tbr2\"\\tChIP\\t\"Several putative Lhx1 target genes were represented in our ChIP data set. For example, Lhx1 binding was detected at two distinct regions at the Hesx1 locus, including a regulatory element in the 5 untranslated region (containing two Lhx1-binding motifs) and a 3 distal enhancer. Lhx1 ChIP peaks were also present upstream of Embigin exon 1.\"\\t--\\t--\\t--\\t>2KB\\t13856', 'E_02_152\\t26537192\\tEphA7 Enhancer\\tmm10\\tchr4\\t29115647\\t29140947\\tMouse\\tEmbryo\\tLow throughput\\t\"PCR,Transgenic mice\"\\t\"Transgenic LacZ reporter analysis, using overlapping EphA7 BACs, was used to narrow down the', 'E_02_153\\t26537192\\tEphA7 Enhancer\\tmm10\\tchr4\\t29119847\\t29120047\\tMouse\\tEmbryo\\tLow throughput\\t\"PCR,Transgenic mice\"\\t\"Transgenic LacZ reporter analysis, using overlapping EphA7 BACs, was used to narrow down the', 'E_02_154\\t26550034\\t--\\tmm10\\tchr4\\t55565908\\t55608014\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"The genome-wide mapping of Tip60 binding in mESCs by chromatin immunoprecipitation (ChIP) coupled with high-throughput sequencing (ChIP-seq) shows that the Tip60 complex is present at promoter regions of predominantly active genes that are bound by RNA polymerase II (Pol II) and contain the H3K4me3 histone mark. The coactivator HAT complexes, Tip60- and Mof (KAT8)-', 'E_02_155\\t26550034\\t--\\tmm10\\tchr6\\t122653828\\t122675804\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"The genome-wide mapping of Tip60 binding in mESCs by chromatin immunoprecipitation (ChIP) coupled with high-throughput sequencing (ChIP-seq) shows that the Tip60 complex is present at promoter regions of predominantly active genes that are bound by RNA polymerase II (Pol II) and contain the H3K4me3 histone mark. The coactivator HAT complexes, Tip60- and Mof (KAT8)-', 'E_02_156\\t26550034\\t--\\tmm10\\tchr6\\t122696853\\t122708211\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"The genome-wide mapping of Tip60 binding in mESCs by chromatin immunoprecipitation (ChIP) coupled with high-throughput sequencing (ChIP-seq) shows that the Tip60 complex is present at promoter regions of predominantly active genes that are bound by RNA polymerase II (Pol II) and contain the H3K4me3 histone mark. The coactivator HAT complexes, Tip60- and Mof (KAT8)-', 'E_02_157\\t26595656\\t--\\tmm10\\tchr12\\t85433906\\t85439068\\tMouse\\tMouse Cortical Neurons\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"These enhancers, which we identified in primary cultures of mouse cortical neurons, appear to be active in the intact brain throughout development, as all of them were tightly aligned with the chroma_x0002_tin immunoprecipitation (ChIP)-seq peaks of the active enhancer marker, acetylated H3K27 (H3K27ac)(Supplementary Fig. 1).\"\\tEnhancer\\tFos\\t3C\\t--\\t\"Characterizing individual c-fos Enhancers by chromosome-conformation capture (3C) analysis and an Enhancer reporter assay, we have corroborated the idea that eRNA is a reliable marker for identifying activated Enhancers.\"\\t\"D12Rfj1,c-fos,cFos\"\\t--\\t--\\t--\\tThese results demonstrate that the combinatorial activation of the c-fosenhancers enables the c-fos gene to be broadly responsive to various signaling pathways.\\t\"CRISPRi,RT-qPCR\"\\t\"We also evaluated the functionality of individual enhancers in their native context by targeted transcription silencing via CRISPR interference (CRISPRi). Using eRNA as a proxy for enhancer activity, we found that distinc_x0002_tive subsets of the c-fos enhancers are activated in the intact brain following chemically induced seizure or light stimulation.\"\\t\"Creb,srf,Mef2a,Mef2d,Mef2c,Npas4\"\\t\"CREB,CREB-1,MCM1,ADCAD1,RSRFC4,RSRFC9,mef2,C5DELq14.3,DEL5q14.3,Le-PAS,NXF,PASD10,bHLHe79\"\\tChIP-seq\\t\"ChIP-_x0002_seq analysis has revealed that the c-fos promoter and enhancers were bound by different combinations of several activity-regulated TFs6 (Supplementary Fig. 1).Knockdown of CREB, MEF2A and NPAS4 significantly impaired c-fos induction by Cl-mediated membrane depolarization (Fig. 4a and Supplementary Fig. 4). Although knockdown of MEF2 family members MEF2D and MEF2C showed a slight decrease in c-fos expression, it was not significant and had a weaker impact than MEF2A knockdown. The decrease in c-fos expression was paralleled with little effect on eRNA transcription (Supplementary Fig. 5).\"\\t--\\t--\\t--\\t>2KB\\t37413', 'E_02_158\\t26595656\\t--\\tmm10\\tchr12\\t85452549\\t85458571\\tMouse\\tMouse Cortical Neurons\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"These enhancers, which we identified in primary cultures of mouse cortical neurons, appear to be active in the intact brain throughout development, as all of them were tightly aligned with the chroma_x0002_tin immunoprecipitation (ChIP)-seq peaks of the active enhancer marker, acetylated H3K27 (H3K27ac)(Supplementary Fig. 1).\"\\tEnhancer\\tFos\\t3C\\t--\\t\"Characterizing individual c-fos Enhancers by chromosome-conformation capture (3C) analysis and an Enhancer reporter assay, we have corroborated the idea that eRNA is a reliable marker for identifying activated Enhancers.\"\\t\"D12Rfj1,c-fos,cFos\"\\t--\\t--\\t--\\tThese results demonstrate that the combinatorial activation of the c-fosenhancers enables the c-fos gene to be broadly responsive to various signaling pathways.\\t\"CRISPRi,RT-qPCR\"\\t\"We also evaluated the functionality of individual enhancers in their native context by targeted transcription silencing via CRISPR interference (CRISPRi). Using eRNA as a proxy for enhancer activity, we found that distinc_x0002_tive subsets of the c-fos enhancers are activated in the intact brain following chemically induced seizure or light stimulation.\"\\t\"Creb,srf,Mef2a,Mef2d,Mef2c,Npas4\"\\t\"CREB,CREB-1,MCM1,ADCAD1,RSRFC4,RSRFC9,mef2,C5DELq14.3,DEL5q14.3,Le-PAS,NXF,PASD10,bHLHe79\"\\tChIP-seq\\t\"ChIP-_x0002_seq analysis has revealed that the c-fos promoter and enhancers were bound by different combinations of several activity-regulated TFs6 (Supplementary Fig. 1).Knockdown of CREB, MEF2A and NPAS4 significantly impaired c-fos induction by Cl-mediated membrane depolarization (Fig. 4a and Supplementary Fig. 4). Although knockdown of MEF2 family members MEF2D and MEF2C showed a slight decrease in c-fos expression, it was not significant and had a weaker impact than MEF2A knockdown. The decrease in c-fos expression was paralleled with little effect on eRNA transcription (Supplementary Fig. 5).\"\\t--\\t--\\t--\\t>2KB\\t18340', 'E_02_159\\t26595656\\t--\\tmm10\\tchr12\\t85466416\\t85469570\\tMouse\\tMouse Cortical Neurons\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"These enhancers, which we identified in primary cultures of mouse cortical neurons, appear to be active in the intact brain throughout development, as all of them were tightly aligned with the chroma_x0002_tin immunoprecipitation (ChIP)-seq peaks of the active enhancer marker, acetylated H3K27 (H3K27ac)(Supplementary Fig. 1).\"\\tEnhancer\\tFos\\t3C\\t--\\t\"Characterizing individual c-fos Enhancers by chromosome-conformation capture (3C) analysis and an Enhancer reporter assay, we have corroborated the idea that eRNA is a reliable marker for identifying activated Enhancers.\"\\t\"D12Rfj1,c-fos,cFos\"\\t--\\t--\\t--\\tThese results demonstrate that the combinatorial activation of the c-fosenhancers enables the c-fos gene to be broadly responsive to various signaling pathways.\\t\"CRISPRi,RT-qPCR\"\\t\"We also evaluated the functionality of individual enhancers in their native context by targeted transcription silencing via CRISPR interference (CRISPRi). Using eRNA as a proxy for enhancer activity, we found that distinc_x0002_tive subsets of the c-fos enhancers are activated in the intact brain following chemically induced seizure or light stimulation.\"\\t\"Creb,srf,Mef2a,Mef2d,Mef2c,Npas4\"\\t\"CREB,CREB-1,MCM1,ADCAD1,RSRFC4,RSRFC9,mef2,C5DELq14.3,DEL5q14.3,Le-PAS,NXF,PASD10,bHLHe79\"\\tChIP-seq\\t\"ChIP-_x0002_seq analysis has revealed that the c-fos promoter and enhancers were bound by different combinations of several activity-regulated TFs6 (Supplementary Fig. 1).Knockdown of CREB, MEF2A and NPAS4 significantly impaired c-fos induction by Cl-mediated membrane depolarization (Fig. 4a and Supplementary Fig. 4). Although knockdown of MEF2 family members MEF2D and MEF2C showed a slight decrease in c-fos expression, it was not significant and had a weaker impact than MEF2A knockdown. The decrease in c-fos expression was paralleled with little effect on eRNA transcription (Supplementary Fig. 5).\"\\t--\\t--\\t--\\t>2KB\\t5907', 'E_02_160\\t26595656\\t--\\tmm10\\tchr12\\t85470646\\t85474086\\tMouse\\tMouse Cortical Neurons\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"These enhancers, which we identified in primary cultures of mouse cortical neurons, appear to be active in the intact brain throughout development, as all of them were tightly aligned with the chroma_x0002_tin immunoprecipitation (ChIP)-seq peaks of the active enhancer marker, acetylated H3K27 (H3K27ac)(Supplementary Fig. 1).\"\\tEnhancer\\tFos\\t3C\\t--\\t\"Characterizing individual c-fos Enhancers by chromosome-conformation capture (3C) analysis and an Enhancer reporter assay, we have corroborated the idea that eRNA is a reliable marker for identifying activated Enhancers.\"\\t\"D12Rfj1,c-fos,cFos\"\\t--\\t--\\t--\\tThese results demonstrate that the combinatorial activation of the c-fosenhancers enables the c-fos gene to be broadly responsive to various signaling pathways.\\t\"CRISPRi,RT-qPCR\"\\t\"We also evaluated the functionality of individual enhancers in their native context by targeted transcription silencing via CRISPR interference (CRISPRi). Using eRNA as a proxy for enhancer activity, we found that distinc_x0002_tive subsets of the c-fos enhancers are activated in the intact brain following chemically induced seizure or light stimulation.\"\\t\"Creb,srf,Mef2a,Mef2d,Mef2c,Npas4\"\\t\"CREB,CREB-1,MCM1,ADCAD1,RSRFC4,RSRFC9,mef2,C5DELq14.3,DEL5q14.3,Le-PAS,NXF,PASD10,bHLHe79\"\\tChIP-seq\\t\"ChIP-_x0002_seq analysis has revealed that the c-fos promoter and enhancers were bound by different combinations of several activity-regulated TFs6 (Supplementary Fig. 1).Knockdown of CREB, MEF2A and NPAS4 significantly impaired c-fos induction by Cl-mediated membrane depolarization (Fig. 4a and Supplementary Fig. 4). Although knockdown of MEF2 family members MEF2D and MEF2C showed a slight decrease in c-fos expression, it was not significant and had a weaker impact than MEF2A knockdown. The decrease in c-fos expression was paralleled with little effect on eRNA transcription (Supplementary Fig. 5).\"\\t--\\t--\\t--\\t<2KB\\t1534', 'E_02_161\\t26595656\\t--\\tmm10\\tchr12\\t85482048\\t85490068\\tMouse\\tMouse Cortical Neurons\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"These enhancers, which we identified in primary cultures of mouse cortical neurons, appear to be active in the intact brain throughout development, as all of them were tightly aligned with the chroma_x0002_tin immunoprecipitation (ChIP)-seq peaks of the active enhancer marker, acetylated H3K27 (H3K27ac)(Supplementary Fig. 1).\"\\tEnhancer\\tFos\\t3C\\t--\\t\"Characterizing individual c-fos Enhancers by chromosome-conformation capture (3C) analysis and an Enhancer reporter assay, we have corroborated the idea that eRNA is a reliable marker for identifying activated Enhancers.\"\\t\"D12Rfj1,c-fos,cFos\"\\t--\\t--\\t--\\tThese results demonstrate that the combinatorial activation of the c-fosenhancers enables the c-fos gene to be broadly responsive to various signaling pathways.\\t\"CRISPRi,RT-qPCR\"\\t\"We also evaluated the functionality of individual enhancers in their native context by targeted transcription silencing via CRISPR interference (CRISPRi). Using eRNA as a proxy for enhancer activity, we found that distinc_x0002_tive subsets of the c-fos enhancers are activated in the intact brain following chemically induced seizure or light stimulation.\"\\t\"Creb,srf,Mef2a,Mef2d,Mef2c,Npas4\"\\t\"CREB,CREB-1,MCM1,ADCAD1,RSRFC4,RSRFC9,mef2,C5DELq14.3,DEL5q14.3,Le-PAS,NXF,PASD10,bHLHe79\"\\tChIP-seq\\t\"ChIP-_x0002_seq analysis has revealed that the c-fos promoter and enhancers were bound by different combinations of several activity-regulated TFs6 (Supplementary Fig. 1).Knockdown of CREB, MEF2A and NPAS4 significantly impaired c-fos induction by Cl-mediated membrane depolarization (Fig. 4a and Supplementary Fig. 4). Although knockdown of MEF2 family members MEF2D and MEF2C showed a slight decrease in c-fos expression, it was not significant and had a weaker impact than MEF2A knockdown. The decrease in c-fos expression was paralleled with little effect on eRNA transcription (Supplementary Fig. 5).\"\\t--\\t--\\t--\\t>2KB\\t12158', 'E_02_162\\t26659182\\t--\\tmm10\\tchr2\\t168744572\\t168752225\\tMouse\\tInduced Pluripotent Stem Cells \\tLow+High throughput\\t\"ChIP-seq,ATAC-seq,CRISPR/Cas9\"\\t\"To validate these observations with an independent, higher resolution method, we performed Assay of Transposase Accessible Chromatin using sequencing (ATAC-seq), which detects integration of the Tn5 transposase in open chromatin regions27 (Extended Data Fig. 7c). Consistent with the SONO-seq data, ATAC-seq analysis of early reprogramming intermediates showed a more accessible chromatin configuration at regulatory regions including ESC-specific enhancers upon suppression of Chaf1a (Fig. 5a; P value <10?15). Moreover, Chaf1a knockdown caused a significant increase in chromatin accessibility across ESC-specific super-enhancers at day 3 of iPSC formation (Supplementary Table 3 and Extended Data Fig. 7d,e; P value <5.310?16). Of note, super-enhancers linked to specialized cell types such as macrophages, lymphocytes and muscle cells were also significantly more accessible in Chaf1a depleted reprogramming intermediates compared to controls (Extended Data Fig. 7f). \"Super-Enhancer\\tSall4\\t--\\t\"ChIP-seq,ATAC-seq\"\\t\"Moreover,Chaf1a knockdown caused a significant increase in Chromatin accessibility across ES-cell-specific Super-Enhancers at day 3 of iPS cell formation.\"\\t\"5730441M18Rik,AA407717,AL022809,AW536104,C330011P20Rik,C78083,C78563,Tex20\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSox2\\t\"Sox-2,lcc,ysb\"\\t\"ChIP-seq,ATAC-seq\"\\t\"Next,we performed ChIP-seq analysis for Sox2 at day 3 of OKSM expression in order to test our hypothesis that increased Chromatin accessibility at Enhancer lements influences reprogramming factor binding. Of the Sox2-bound super-Enhancers unique to CAF-1 knockdown cells, a subset also showed a more accessible chromatin structure by ATAC-seq analysis. \"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t68', 'E_02_163\\t26659182\\t--\\tmm10\\tchr8\\t89041889\\t89090124\\tMouse\\tInduced Pluripotent Stem Cells \\tLow+High throughput\\t\"ChIP-seq,ATAC-seq,CRISPR/Cas9\"\\t\"To validate these observations with an independent, higher resolution method, we performed Assay of Transposase Accessible Chromatin using sequencing (ATAC-seq), which detects integration of the Tn5 transposase in open chromatin regions27 (Extended Data Fig. 7c). Consistent with the SONO-seq data, ATAC-seq analysis of early reprogramming intermediates showed a more accessible chromatin configuration at regulatory regions including ESC-specific enhancers upon suppression of Chaf1a (Fig. 5a; P value <10?15). Moreover, Chaf1a knockdown caused a significant increase in chromatin accessibility across ESC-specific super-enhancers at day 3 of iPSC formation (Supplementary Table 3 and Extended Data Fig. 7d,e; P value <5.310?16). Of note, super-enhancers linked to specialized cell types such as macrophages, lymphocytes and muscle cells were also significantly more accessible in Chaf1a depleted reprogramming intermediates compared to controls (Extended Data Fig. 7f). \"Super-Enhancer\\tSall1\\t--\\t\"ChIP-seq,ATAC-seq\"\\t\"Next,we performed ChIP-seq analysis for Sox2 at day 3 of OKSM expression in order to test our hypothesis that increased Chromatin accessibility at Enhancer lements influences reprogramming factor binding. Of the Sox2-bound super-Enhancers unique to CAF-1 knockdown cells, a subset also showed a more accessible chromatin structure by ATAC-seq analysis (for example, Sall1; Fig. 5c and Supplementary Table 4).\"\\tMsal-3\\t--\\t--\\t--\\t--\\t--\\t--\\tSox2\\t\"Sox-2,lcc,ysb\"\\t\"ChIP-seq,ATAC-seq\"\\t\"Next,we performed ChIP-seq analysis for Sox2 at day 3 of OKSM expression in order to test our hypothesis that increased Chromatin accessibility at Enhancer lements influences reprogramming factor binding. Of the Sox2-bound super-Enhancers unique to CAF-1 knockdown cells, a subset also showed a more accessible chromatin structure by ATAC-seq analysis.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t38766', 'E_02_164\\t26663721\\t--\\tmm10\\tchr12\\t85515101\\t85543901\\tMouse\\tMacrophage\\tLow+High throughput\\tChIP-seq\"To identify the sites involved in the GC response in the two species,Chromatin immunoprecipitation for GR and sequencing (ChIP-seq) was performed 2 h after dexamethasone treatment in both mBMDM and hMDM. Representative UCSC browser tracks for GR binding in mBMDM and hMDM are shown in Fig. 2A and 2C.ChIP-seq data tracks from the UCSC browser for the Fos C Jdp2 region, for GR binding in mBMDM. Data from ChIP with anti-GR antibodies after treatment with 100nM dexamethasone for 2h (Dex GR IP), input material (Dex input) and immunoprecipitated material from a vehicle treated control (Vehicle GR IP) are shown. Enriched motifs found de novo within GR bound sites in mBMDM and hMDM.\"\\tEnhancer\\tFos-Jdp2?\\t--\\tChIP-seq\\t\"To identify the sites involved in the GC response in the two species,Chromatin immunoprecipitation for GR and sequencing (ChIP-seq) was performed 2 h after dexamethasone treatment in both mBMDM and hMDM. Representative UCSC browser tracks for GR binding in mBMDM and hMDM are shown in Fig. 2A and 2C.ChIP-seq data tracks from the UCSC browser for the Fos C Jdp2 region, for GR binding in mBMDM. Data from ChIP with anti-GR antibodies after treatment with 100nM dexamethasone for 2h (Dex GR IP), input material (Dex input) and immunoprecipitated material from a vehicle treated control (Vehicle GR IP) are shown. Enriched motifs found de novo within GR bound sites in mBMDM and hMDM.\"\\t\"D12Rfj1,c-fos,cFos,Jundm2,Jundp2,TIF\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t55601', 'E_02_165\\t26673693\\tTRA Enhancer\\tmm10\\tchr14\\t54227135\\t54228033\\tMouse\\tP5424\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"We found that eRNAs essentially reca_x0002_pitulated Pol II patterns during DN to DP transition as wellas those of Ets1 in DP only, suggesting highest enhancer ac_x0002_tivity in class 4 at the DN stage and classes 4C5 at the DP stage (Figure 3C). Thus, distal Ets1 enhancers broadly re_x0002_cruit Pol II that is able to generate paused transcripts.Significance of gene expression differences by increasing Ets1-fold change. Expression fold changes for all classes, shown as a boxplot. Significance levels for each comparison are indicated above, with 1 star denoting P < 0.05, 2 stars denoting P < 0.01 and 3 stars denoting P < 0.005.\"\\tEnhancer\\tDad1\\t--\\t\"ChIP-seq,Western blot,ChIP-qPCR\"\\t\"We further validated the dy_x0002_namic recruitment of TCF1 and Runx1 by ChIP-qPCR at the Tcra Enhancer, which is bound by similar levels of Ets1 in both DN and DP thymocytes.UCSC genome browser screenshot of the Tcra (left) and Cd2 loci (right)showing increased H3K4me1 signal at the Ets1 binding site following knockdown in the P5424 cell line.\"\\tAI323713\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Ets1,Tcf1,Runx1\"AI196000,AI448617,D230050P06,Ets-1,Tpl1,p54,vs,AI465550,TCF-1,Tcf1,AML1,CBF-alpha-2,Cbfa2,Pebp2a2,Pebpa2b\"\\t\"ChIP-qPCR,Western blot\"\\t\"We further validated the dynamic recruitment of TCF1 and Runx1 by ChIP-qPCR at the Tcra Enhancer, which is bound by similar levels of Ets1 in both DN and DP thymocytes.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t7900', 'E_02_166\\t26673693\\tE8II Enhancer\\tmm10\\tchr6\\t71321750\\t71323696\\tMouse\\tP5424\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"We found that eRNAs essentially reca_x0002_pitulated Pol II patterns during DN to DP transition as wellas those of Ets1 in DP only, suggesting highest enhancer ac_x0002_tivity in class 4 at the DN stage and classes 4C5 at the DP stage (Figure 3C). Thus, distal Ets1 enhancers broadly re_x0002_cruit Pol II that is able to generate paused transcripts.Significance of gene expression differences by increasing Ets1-fold change. Expression fold changes for all classes, shown as a boxplot. Significance levels for each comparison are indicated above, with 1 star denoting P < 0.05, 2 stars denoting P < 0.01 and 3 stars denoting P < 0.005.\"\\tEnhancer\\tCd8b1\\t--\\t\"ChIP-seq,Western blot,ChIP-qPCR\"\\t\"As exemplified at the E8II Enhancer of the Cd8 locus, and at the Tcra and Tcrb Enhancers,Ets1 in fact seemed to recruit Pol II as well as promoting High throughputer H3K4me3 in some cases.\"\\t\"Cd8b,Ly-3,Ly-C,Lyt-3\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Ets1,Tcf1,Runx1\"\\t\"AI196000,AI448617,D230050P06,Ets-1,Tpl1,p54,vs,AI465550,TCF-1,Tcf1,AML1,CBF-alpha-2,Cbfa2,Pebp2a2,Pebpa2b\"\\tChIP-qPCR\\t\"We further validated the dynamic recruitment of TCF1 and Runx1 by ChIP-qPCR at the Tcra Enhancer,which is bound by similar levels of Ets1in both DN and DP thymocytes.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t88', 'E_02_167\\t26748758\\t--\\tmm10\\tchr2\\t172513228\\t172520500\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RT-qPCR\"\\t\"To test if Foxd3 was indeed able to silence na?ve pluripotency enhancers, we evaluated the impact of Foxd3 overexpression on some of these regulatory elements. tetOn Foxd3 mESC were treated with Dox and the levels of various histone modifications (i.e. H3K27ac, H3K27me2, H3K4me1, H3K4me2) at selected enhancers were evaluated by ChIP-qPCR. Compared to control cells, tetOn Foxd3 cells displayed lower levels of H3K27ac and H3K4me2 (active enhancers) , increased H3K27me2 (inactive enhancers) and minor changes in H3K4me1 levels (active and inactive enhancers)at most of the analysed na?ve enhancers (Figure 4B-C, Figure S5A-B).\"\\tEnhancer\\tTfap2C\\t--\\t\"ChIP-seq,RT-qPCR\"\\t\"ChIP-Seq profiles generated in Foxd3-FH mES cells with anti-HA and anti-Flag antibodies around a representative locus (i.e.,Tfap2c).\"\\t\"AA409384,AP2gamma,Ap-2.2,Stra2,Tcfap2c\"\\t--\\t--\\t--\\t--\\t--\\t--\\tFoxd3\\t\"CWH3,Genesis,Hfh2\"\\tChIP-qPCR\\t\"To test if Foxd3 was indeed able to silence naive pluripotency enhancers, we evaluated the impact of Foxd3 overexpression on some of these regulatory elements. tetOn Foxd3 mESCs were treated with Dox, and the levels of various histone modifications (i.e., H3K27ac, H3K27me2, H3K4me1, and H3K4me2) at selected enhancers were evaluated by ChIP-qPCR. Compared to control cells, tetOn Foxd3 cells displayed lower levels of H3K27ac and H3K4me2 (active enhancers) , increased H3K27me2 (inactive enhancers) and minor changes in H3K4me1 levels (active and inactive enhancers)at most of the analysed na?ve enhancers (Figure 4B-C, Figure S5A-B).Using RT-qPCR, we found that Foxd3 overexpression significantly reduced eRNA levels at most of the analysed enhancers (Figure S5C). The role of Foxd3 was similarly evaluated using Foxd3?/? mESC, which displayed increased levels of H3K27ac and H3K4me2 (Figure 4D-E) and minor changes in H3K27me2 (Figure S5D) at most of the analysed enhancers.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t28814', 'E_02_168\\t26748758\\t--\\tmm10\\tchr2\\t172541319\\t172544955\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RT-qPCR\"\\t\"To test if Foxd3 was indeed able to silence na?ve pluripotency enhancers, we evaluated the impact of Foxd3 overexpression on some of these regulatory elements. tetOn Foxd3 mESC were treated with Dox and the levels of various histone modifications (i.e. H3K27ac, H3K27me2, H3K4me1, H3K4me2) at selected enhancers were evaluated by ChIP-qPCR. Compared to control cells, tetOn Foxd3 cells displayed lower levels of H3K27ac and H3K4me2 (active enhancers) , increased H3K27me2 (inactive enhancers) and minor changes in H3K4me1 levels (active and inactive enhancers)at most of the analysed na?ve enhancers (Figure 4B-C, Figure S5A-B).\"\\tEnhancer\\tTfap2C\\t--\\t\"ChIP-seq,RT-qPCR\"\\t\"ChIP-Seq profiles generated in Foxd3-FH mES cells with anti-HA and anti-Flag antibodies around a representative locus (i.e.,Tfap2c).\"\\t\"AA409384,AP2gamma,Ap-2.2,Stra2,Tcfap2c\"\\t--\\t--\\t--\\t--\\t--\\t--\\tFoxd3\\t\"CWH3,Genesis,Hfh2\"\\tChIP-qPCR\\t\"To test if Foxd3 was indeed able to silence naive pluripotency enhancers, we evaluated the impact of Foxd3 overexpression on some of these regulatory elements. tetOn Foxd3 mESCs were treated with Dox, and the levels of various histone modifications (i.e., H3K27ac, H3K27me2, H3K4me1, and H3K4me2) at selected enhancers were evaluated by ChIP-qPCR. Compared to control cells, tetOn Foxd3 cells displayed lower levels of H3K27ac and H3K4me2 (active enhancers) , increased H3K27me2 (inactive enhancers) and minor changes in H3K4me1 levels (active and inactive enhancers)at most of the analysed na?ve enhancers (Figure 4B-C, Figure S5A-B).Using RT-qPCR, we found that Foxd3 overexpression significantly reduced eRNA levels at most of the analysed enhancers (Figure S5C). The role of Foxd3 was similarly evaluated using Foxd3?/? mESC, which displayed increased levels of H3K27ac and H3K4me2 (Figure 4D-E) and minor changes in H3K27me2 (Figure S5D) at most of the analysed enhancers.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t2541', 'E_02_169\\t26748758\\t--\\tmm10\\tchr2\\t172574046\\t172577682\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,RT-qPCR\"\\t\"To test if Foxd3 was indeed able to silence na?ve pluripotency enhancers, we evaluated the impact of Foxd3 overexpression on some of these regulatory elements. tetOn Foxd3 mESC were treated with Dox and the levels of various histone modifications (i.e. H3K27ac, H3K27me2, H3K4me1, H3K4me2) at selected enhancers were evaluated by ChIP-qPCR. Compared to control cells, tetOn Foxd3 cells displayed lower levels of H3K27ac and H3K4me2 (active enhancers) , increased H3K27me2 (inactive enhancers) and minor changes in H3K4me1 levels (active and inactive enhancers)at most of the analysed na?ve enhancers (Figure 4B-C, Figure S5A-B).\"\\tEnhancer\\tTfap2C\\t--\\t\"ChIP-seq,RT-qPCR\"\\t\"ChIP-Seq profiles generated in Foxd3-FH mES cells with anti-HA and anti-Flag antibodies around a representative locus (i.e.,Tfap2c).\"\\t\"AA409384,AP2gamma,Ap-2.2,Stra2,Tcfap2c\"\\t--\\t--\\t--\\t--\\t--\\t--\\tFoxd3\\t\"CWH3,Genesis,Hfh2\"\\tChIP-qPCR\\t\"To test if Foxd3 was indeed able to silence naive pluripotency enhancers, we evaluated the impact of Foxd3 overexpression on some of these regulatory elements. tetOn Foxd3 mESCs were treated with Dox, and the levels of various histone modifications (i.e., H3K27ac, H3K27me2, H3K4me1, and H3K4me2) at selected enhancers were evaluated by ChIP-qPCR. Compared to control cells, tetOn Foxd3 cells displayed lower levels of H3K27ac and H3K4me2 (active enhancers) , increased H3K27me2 (inactive enhancers) and minor changes in H3K4me1 levels (active and inactive enhancers)at most of the analysed na?ve enhancers (Figure 4B-C, Figure S5A-B).Using RT-qPCR, we found that Foxd3 overexpression significantly reduced eRNA levels at most of the analysed enhancers (Figure S5C). The role of Foxd3 was similarly evaluated using Foxd3?/? mESC, which displayed increased levels of H3K27ac and H3K4me2 (Figure 4D-E) and minor changes in H3K27me2 (Figure S5D) at most of the analysed enhancers.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t30186', 'E_02_170\\t26766440\\t--\\tmm10\\tchr14\\t69196226\\t69202226\\tMouse\\tG1ER Erythroid Cell\\tLow+High throughput\\t\"CRISPR/Cas9,ChIP-seq\"\\t\"Specifically, we analyzed active enhancer-associated histone modifications H3K4me1 and H3K27ac by chromatin immunoprecipitation sequencing (ChIP-seq) and transcriptomic profiles in four distinct populations of human HSPCs or ProEs at fetal or adult stage (Figure 1B-E; Figure S1A).Identification of lineage or developmental stage-specific enhancers. Venn diagram shows the overlap between HSPC and ProE, or fetal and adult enhancers. The numbers of lost, shared, or gained enhancers in each comparison are shown.We designed sequence-specific guide RNAs (or sgRNAs) flanking each constituent enhancer (E1, E2 and E3) or the promoter (P) (Figure 3B). \"\\tEnhancer\\tSlc25a37\\t--\\t\"ChIP-seq,ChIP-qPCR\"\\t\"Specifically, while deletion of E1 or E2 individually only modestly or slightly impairs Slc25a37 activation during differentiation, respectively, E3 deletion abolishes its activation resulting in a 15-fold decrease in expression (48h after -estradiol treatment; Figure 3C).The enhancer cluster upstream of the SLC25A37 gene, consisting of three distinct constituent enhancers as measured by H3K4me1 and H3K27ac ChIP-seq, is defined as an erythroid-specific super-enhancer in both human (A5 ProE) and mouse (G1ER) erythroid cells (Figure 3A; Table S4).\"\\t\"Mfrn,Mscp,Mfrn1,AI848481,frascati,mitoferrin,1700020E22Rik,4930513O14Rik,4930526G11Rik,C330015G08Rik\"\\t--\\t--\\t--\\tGATA2-to-GATA1 switch is prevalent at dynamic enhancers and drives erythroid enhancer commissioning.\\tCRISPR/Cas9\\t\"Importantly, by CRISPR/Cas9-mediated genomic editing, we uncover functional hierarchy of constituent enhancers within the SLC25A37 super-enhancer. Despite indistinguishable chromatin features, we reveal through genomic editing the functional diversity of several GATA switch enhancers in which enhancers with opposing functions cooperate to coordinate transcription.\"\\t\"Gata1,Gata2\"\\t\"Gata-1,Gf-1,eryf1,Gata-2\"\\t\"ChIP-seq,CRISPR/Cas9\"\\t\"To further dissect the role of GATA switch in enhancer turnover, we compared GATA2 and GATA1 occupancy within A0 and A5 enhancers by ChIP-seq. Specifically, we enumerated the distribution of enhancers occupied by GATA2-Only, GATA1-Only, or GATA2-to-GATA1 switch.We then employed genomic editing to dissect the requirement of GATA switch enhancers within the paradigmatic Gata2 locus.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t42621', 'E_02_171\\t26766440\\t--\\tmm10\\tchr14\\t119710660\\t119714758\\tMouse\\tG1ER Erythroid Cell\\tLow+High throughput\\t\"CRISPR/Cas9,ChIP-seq\"\\t\"Specifically, we analyzed active enhancer-associated histone modifications H3K4me1 and H3K27ac by chromatin immunoprecipitation sequencing (ChIP-seq) and transcriptomic profiles in four distinct populations of human HSPCs or ProEs at fetal or adult stage (Figure 1B-E; Figure S1A).Identification of lineage or developmental stage-specific enhancers. Venn diagram shows the overlap between HSPC and ProE, or fetal and adult enhancers. The numbers of lost, shared, or gained enhancers in each comparison are shown.\"\\tEnhancer\\tSlc25a37\\t--\\t\"ChIP-seq,ChIP-qPCR\"\\t\"Specifically, while deletion of E1 or E2 individually only modestly or slightly impairs Slc25a37 activation during differentiation, respectively, E3 deletion abolishes its activation resulting in a 15-fold decrease in expression (48h after -estradiol treatment; Figure 3C).\"\\t\"Mfrn,Mscp,Mfrn1,AI848481,frascati,mitoferrin,1700020E22Rik,4930513O14Rik,4930526G11Rik,C330015G08Rik\"\\t--\\t--\\t--\\tGATA2-to-GATA1 switch is prevalent at dynamic enhancers and drives erythroid enhancer commissioning.\\tCRISPR/Cas9\\t\"Importantly, by CRISPR/Cas9-mediated genomic editing, we uncover functional hierarchy of constituent enhancers within the SLC25A37 super-enhancer. Despite indistinguishable chromatin features, we reveal through genomic editing the functional diversity of several GATA switch enhancers in which enhancers with opposing functions cooperate to coordinate transcription.\"\\t\"Gata1,Gata2\"\\t\"Gata-1,Gf-1,eryf1,Gata-2\"\\t\"ChIP-seq,CRISPR/Cas9\"\\t\"To further dissect the role of GATA switch in enhancer turnover, we compared GATA2 and GATA1 occupancy within A0 and A5 enhancers by ChIP-seq. Specifically, we enumerated the distribution of enhancers occupied by GATA2-Only, GATA1-Only, or GATA2-to-GATA1 switch.We then employed genomic editing to dissect the requirement of GATA switch enhancers within the paradigmatic Gata2 locus.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t50470862', 'E_02_172\\t26766440\\t--\\tmm10\\tchr14\\t69221635\\t69222851\\tMouse\\tG1ER Erythroid Cell\\tLow+High throughput\\t\"CRISPR/Cas9,ChIP-seq\"\\t\"Specifically, we analyzed active enhancer-associated histone modifications H3K4me1 and H3K27ac by chromatin immunoprecipitation sequencing (ChIP-seq) and transcriptomic profiles in four distinct populations of human HSPCs or ProEs at fetal or adult stage (Figure 1B-E; Figure S1A).Identification of lineage or developmental stage-specific enhancers. Venn diagram shows the overlap between HSPC and ProE, or fetal and adult enhancers. The numbers of lost, shared, or gained enhancers in each comparison are shown.\"\\tEnhancer\\tSlc25a37\\t--\\t\"ChIP-seq,ChIP-qPCR\"\\t\"Loss of E3 leads to near absence of H3K27ac, GATA1 and TAL1 occupancy at the neighboring E1 and E2 Enhancers, whereas loss of E1 or E2 has minimal impact on H3K27ac or GATA1/TAL1 binding at neighboring Enhancers (Figure 3D-G). These results strongly suggest that, despite the indistinguishable chromatin features and TF occupancy at the Slc25a37 constituent Enhancers, the E3 Enhancer is functionally more potent than its neighboring Enhancers in directing transcriptional activation.\"\\t\"Mfrn,Mscp,Mfrn1,AI848481,frascati,mitoferrin,1700020E22Rik,4930513O14Rik,4930526G11Rik,C330015G08Rik\"\\t--\\t--\\t--\\tGATA2-to-GATA1 switch is prevalent at dynamic enhancers and drives erythroid enhancer commissioning.\\tCRISPR/Cas9\\t\"Importantly, by CRISPR/Cas9-mediated genomic editing, we uncover functional hierarchy of constituent enhancers within the SLC25A37 super-enhancer. Despite indistinguishable chromatin features, we reveal through genomic editing the functional diversity of several GATA switch enhancers in which enhancers with opposing functions cooperate to coordinate transcription. \"\\t\"Gata1,Gata2\"\\t\"Gata-1,Gf-1,eryf1,Gata-2\"\\t\"ChIP-seq,CRISPR/Cas9\"\\t\"To further dissect the role of GATA switch in enhancer turnover, we compared GATA2 and GATA1 occupancy within A0 and A5 enhancers by ChIP-seq. Specifically, we enumerated the distribution of enhancers occupied by GATA2-Only, GATA1-Only, or GATA2-to-GATA1 switch.We then employed genomic editing to dissect the requirement of GATA switch enhancers within the paradigmatic Gata2 locus.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t19604', 'E_02_173\\t26808502\\tTce1 Enhancer\\tmm10\\tchr2\\t10136078\\t10138078\\tMouse\\tT Cell\\tLow throughput\\t\"CRISPR/Cas9,ChIP,RT-PCR\"\\t\"We next asked whether this 1.2-kb Gata3 Tce1 fragment was also necessary for the transcription of a reporter gene in T cells. To answer this question, we prepared another EGFP reporter plasmid in which the 1.2-kb sequence was deleted from full-length Tce1 (Figure 5A) and generated additional F0 Tg mice (Tg1.2 mice). None of the F0 Tg1.2 mice expressed EGFP in peripheral CD4 T cells (Table 1 and Supplemental Figure 5C) or in thymocytes (Figure 5, B and C), except possibly at the DN4 stage (Table 2). We also analyzed Tg1.7, Tg1.5, and Tg2.7 mice and found that all of them expressed EGFP in peripheral CD4 T cells (Tables 1 and ?and2).2). Therefore, the EGFP expression in those mice confirmed that these sequences were not required for Tce1 enhancer activity in peripheral CD4 T cells. These data show that the 1.2-kb T cell enhancer fragment within Tce1 is also necessary for reporter gene transcription in T cells and, taken together with the data shown in Figure 1, that this fragment functions as an enhancer core element in vivo for Gata3 T cellCspecific transcription. \"\\tEnhancer\\tGata3\\tCRISPR/Cas9\\t\"ChIP,PCR\"\\t\"As anticipated, homozygous loss of Tce1 (Tce1C/C) led to a reduction in the number of ETPs (62% of that of heterozygous controls), and Gata3 mRNA in the remaining ETPs was only 52% of that of controls (Figure 1B). In contrast, the number of Tce1C/C DN2 and DN3 stage cells was unaltered compared with that of controls, although Gata3 mRNA levels in DN2 thymocytes were reduced (83% of that of heterozygous controls; Figure 1B). \"\\t\"Gata-3,jal\"\\t--\\t--\\t--\\t\"Together, our data show that Tce1 is both necessary and sufficient for critical aspects of Gata3 T cellCspecific transcriptional activity.\"\\t\"CRISPR/Cas9,ChIP,RT-PCR\"\\t\"Instead, we demonstrated that multiple lineage-affiliated transcription factors bind to Tce1 and that this enhancer confers T lymphocyteCspecific Gata3 activation in vivo, as targeted deletion of Tce1 in a mouse model abrogated critical functions of this T cellCregulatory element.\"\\t\"Tcf7,Tcf12,Rbpj\"\\t\"AI465550,TCF-1,Tcf1,A130037E08Rik,ALF1,HEB,HEBAlt,HTF-4,HTF4,ME1,REB,bHLHb20,AI843960,CBF1,Igkjrb,Igkrsbp,RBP-J,RBP-Jkappa,RBP-Jkappa,RBPjk,Rbpsuh\"\\tChIP\\t\"We therefore examined the association of CSL/RBP-J with predicted binding sites within Tce1. CSL/RBP-J binding to the Cd25 locus, a robust direct Notch target gene, was used as a positive control with an antiCRBP-J antibody in ChIP assays.Taken together, these data show that at least 3 critical T cellCaffiliated transcription factors, TCF-1, HEB, and CSL/RBP-J, occupy binding sites within Tce1 at different developmental stages.Since all 3 have been shown to vitally affect T cell development, it seems likely that these factors and their associated signaling pathways directly modulate Gata3 expression through their binding to multiple consensus sites within Tce1.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t280001', 'E_02_174\\t26937964\\t--\\tmm10\\tchr7\\t35156353\\t35161233\\tMouse\\tMyeloid Cells\\tLow+High throughput\\t\"ChIP-seq,CRISPR/Cas9,Transgenic mice\"\\t\"To determine the effect of deleting the enhancer in vivo, we now characterize C57BL/6 mice in which loxP sites flank a 688 bp DNA segment containing the enhancer. \"\\tEnhancer\\tCebpa\\tCRISPR/Cas9\\tChIP-seq\\t\"These findings demonstrate a critical role for the +37 kb Cebpa Enhancer for hematopoietic-specific Cebpa expression,with Enhancer deletion leading to impaired myelopoiesis and potentially preleukemic progenitor expansion.\"\\t\"C/ebpalpha,CBF-A,Cebp\"\\t--\\t--\\t--\\t\"These findings demonstrate a critical role for the +37 kb Cebpa enhancer for hematopoietic-specific Cebpa expression, with enhancer deletion leading to impaired myelopoiesis and potentially preleukemic progenitor expansion.\"\\t\"CRISPR/Cas9,PCR,Western blot\"\\t\"Together, these data indicate that reduced Cebpa expression consequent to +37 kb Enhancer deletion impairs hematopoietic cell autonomous granulopoiesis in vitro, leading to preservation of immature myeloid progenitors capable of long-term, IL-3-dependent proliferation without complete terminal maturation,a preleukemic phenotype.\"\\tRunx1\\t\"AML1,CBF-alpha-2,Cbfa2,Pebp2a2,Pebpa2b\"\\tChIP-seq\\t\"Runx1,C/EBP,Spi1,Erg,Fli-1,GATA2,Scl,Meis1,and Gfi-1b bind chromatin in the region of this Enhancer in hematopoietic cells as determined by ChIP-Seq.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t39501', 'E_02_175\\t27881681\\t--\\tmm10\\tchr7\\t129554284\\t129564284\\tMouse\\tRat Chondrosarcoma Cells (RCS cells)\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"However, ChIP sequencing revealed that SOX9 does not strongly bind to intron 1, but rather it binds to intron 6 and a site 30 kb upstream of the transcription start site. Here, we aimed to determine the role of the novel SOX9-binding site in intron 6. We prepared reporter constructs that contain a Col2a1 promoter, intron 1 with or without intron 6, and the luciferase gene. \"Enhancer\\tCol2a1\\tCRISPR/Cas9\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"the ChIP-Seq data of SOX9 in the Col2a1 gene using rat chondrosarcoma cells (RCS cells)show that there is a strong SOX9-binding site in intron 6 and a weaker binding site in intron 1.Although the reporter constructs were not activated by SOX9 alone, the construct that contained both introns 1 and 6 was activated 5C10-fold by the SOX9/SOX5 or the SOX9/SOX6 combination in transient-transfection assays in 293T cells. This enhancement was also observed in rat chondrosarcoma cells that stably expressed the construct. CRISPR/Cas9-induced deletion of intron 6 in RCS cells revealed that a 10-bp region of intron 6 is necessary both for Col2a1 expression and SOX9 binding. \"\\t\"Col2,Col2a,Col2a-1,Del1,Dmm,Lpk,M100413,Rgsc413,Rgsc856\"\\tSkeletal Diseases\\t--\\t--\\t--\\t--\\t--\\tSox9\\t\"2010306G03Rik,AV220920,mKIAA4243\"\\t\"ChIP-seq,ChIP-qPCR,EMSA\"\\t\"SOX9 bind to intron 6 and the binding of SOX5/SOX6 to intron 6 may facilitate the binding of SOX9 to the intron 1.To explore this possibility, we tested whether SOX9 and SOX5 bound to the SB2 fragment(600bp)shownabove or an even shorter fragment by means of an EMSA.\"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t31583683', 'E_02_176\\t27895109\\t--\\tmm10\\tchr9\\t78368337\\t78376479\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"Luciferase Reporter Assay,RT-qPCR,ChIP-seq\"\\t\"Reporter assays revealed that each of these regions contained an active enhancer (Fig. 2B). Deletion of the entire EC(DppUp) significantly affects the expression of both Dppa5a and Ooep, although the effect on Ooep is much greater (92% reduction compared to a 36% reduction).Enhancer activity was identified in a reporter assay for the ?1 enhancer and the ?5.8 enhancer within the EC(DppUpA) and EC(DppUpB) regions, respectively.\"\\tEnhancer\\t\"Dppa5a,Ooep\"\\t--\\t\"Luciferase Reporter Assay,RT-qPCR,ChIP-seq\"\\t\"We found that the enhancer cluster EC(DppUp), which lies between the Dppa5a and Ooep genes, regulated both Dppa5a and Ooep in cis. We considered two alternatives for the mechanism underlying this observation: Either this region contained two distinct regulatory modules, one for Dppa5a and one for Ooep, or each of the enhancers within the cluster contributed to the expression of both genes. To differentiate between these two alternatives, we split the EC into two segments and deleted each individually (Fig. 2). The region closest to Ooep, EC(DppUpB), contains one MTL bound by four transcription factors, whereas the region closest to Dppa5a, EC(DppUpA), contains three separate MTLs. Reporter assays revealed that each of these regions contained an active enhancer (Fig. 2B).\"\\t\"AA536857,Dppa5,Esg1,ecat2,2410146L05Rik,Floped,Moep19,Sddr\"\\t--\\t--\\t--\\t\"Robust transcriptional output can be achieved by an isolated enhancer,clusters of enhancers acting on a common target gene act in a partially redundant manner to fine tune transcriptional output of their target genes.\"\\tChIP-seq\\tThis occurs despite the fact that deletion of EC(DppDn) alone has only a modest effect on both Dppa5a (16% reduction) and Ooep (15% reduction) ex_x0002_pression (Table 1). This finding indicates partially redundant control of Dppa5a and Ooep by Enhancers in the upstream and downstream EC.\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5357', 'E_02_177\\t27920250\\t--\\tmm10\\tchr5\\t137465620\\t137479420\\tMouse\\tRenal EPO-producing (REP) Cells\\tLow throughput\\t\"Luciferase Reporter Assay,Transgenic mice\"\\t\"17k-GFP contains the 17-kb upstream and 15-kb downstream regions, whereas 17kXM-GFP contains only the 17-kb upstream region as a regulatory element. As in 22k-GFP mice, GFP expression was observed in the kidney and liver in 17k-GFP mice under anemic conditions (Fig. 1C).\"\\tEnhancer\\tEpo\\t--\\t\"Luciferase Reporter Assay,qRT-PCR\"\\t\"we generated a series of transgenic GFP reporter constructs using 60k-GFP and 22k-GFP as the starting materia GFP expression was observed in the kidney but not in the liver in 17kXM-GFP mice.By means of long genomic PCR assays utilizing a primer set recognizing sequences located 18.6 kb upstream (forward primer) and 3.7 kb downstream (reverse primer) from the Epo gene transcription start site, we confirmed that the bulk of the Epo-BAC DNA containing a 17-kb upstream region and a downstream hepatic enhancer region was integrated into the genome of the transgenic mice in the right order (Fig. 4C).\"\\tEpo\\tRenal Anemia\\tDOID:784\\tD051436\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t10499', 'E_02_178\\t27941798\\t--\\tmm10\\tchr15\\t63833002\\t63863200\\tMouse\\tHCT 116\\tLow+High throughput\\tChIP-seq\\t\"To determine if control of enhancer activity by SWI/SNF complexes is coordinated with TFs, we analyzed sequence motifs at enhancers sensitive to ARID1A loss. The CTCF motif was relatively depleted in regions of H3K27ac loss, implying CTCF-bound insulator regions may resist modulation, while the AP1 (JUND/FOSL1) motif was most enriched (Fig. 5a, Supplementary Table 5).Among factors assessed in HCT116 cells, binding of these TFs correlated most strongly with SWI/SNF occupancy (Supplementary Fig. 6b) and was higher at enhancers that lost activity than at enhancers that were unaffected by ARID1A deficiency (Fig. 5c). \"\\tSuper-Enhancer\\tArid1a\\t--\\t\"ChIP-seq,ChIP-qPCR\"\\t\"To determine if ARID1A loss impairs enhancer activity?in vivo, we examined H3K27ac and gene expression in the colonic epithelium of wildtype and Villin-CreER-T2?Arid1afl/fl?mice. Again, whereas promoter activation states were changed little (Fig. 5d), the effects on H3K27ac at enhancers were significant and were correlated with changes in mRNA levels of the nearest genes (Fig. 5e).?\"\\t\"1110030E03Rik, BAF250a, Osa1, Smarcf1\"\\tColon Cancer\\tDOID:219\\tD003110\\t--\\t--\\t--\\t\"Jun,Cebpb\"\\t\"AP-1c,c-jun,Jun,C/EBPbeta,CRP2,IL-6DBP,LAP,LIP,NF-IL6,NF-M,Nfil6\"\\tChIP-seq\\t\"ChIP-Seq profiles for HCT116 cells, including those generated by ENCODE21 (Encyclopedia of DNA Elements, Supplementary Table 6), revealed further that H3K27ac loss was most strongly associated with sites bound in WT cells by SWI/SNF complexes and/or TFs including AP1, CEBPB, and TEAD4 (Fig. 5b). Among factors assessed in HCT116 cells, binding of these TFs correlated most strongly with SWI/SNF occupancy (Supplementary Fig. 6b) and was higher at enhancers that lost activity than at enhancers that were unaffected by ARID1A deficiency (Fig. 5c)\"\\t--\\t--\\t--\\t>2KB\\t\\t72131', 'E_02_179\\t27941798\\t--\\tmm10\\tchr15\\t63865799\\t63896492\\tMouse\\tHCT 116\\tLow+High throughput\\tChIP-seq\\t\"To determine if control of enhancer activity by SWI/SNF complexes is coordinated with TFs, we analyzed sequence motifs at enhancers sensitive to ARID1A loss. The CTCF motif was relatively depleted in regions of H3K27ac loss, implying CTCF-bound insulator regions may resist modulation, while the AP1 (JUND/FOSL1) motif was most enriched (Fig. 5a, Supplementary Table 5).Among factors assessed in HCT116 cells, binding of these TFs correlated most strongly with SWI/SNF occupancy (Supplementary Fig. 6b) and was higher at enhancers that lost activity than at enhancers that were unaffected by ARID1A deficiency (Fig. 5c). \"\\tSuper-Enhancer\\tArid1a\\t--\\t\"ChIP-seq,ChIP-qPCR\"\\t\"To determine if control of enhancer activity by SWI/SNF complexes is coordinated with TFs, we analyzed sequence motifs at enhancers sensitive to ARID1A loss. The CTCF motif was relatively depleted in regions of H3K27ac loss, implying CTCF-bound insulator regions may resist modulation, while the AP1 (JUND/FOSL1) motif was most enriched (Fig. 5a, Supplementary Table 5).Among factors assessed in HCT116 cells, binding of these TFs correlated most strongly with SWI/SNF occupancy (Supplementary Fig. 6b) and was higher at enhancers that lost activity than at enhancers that were unaffected by ARID1A deficiency (Fig. 5c). \"\\t\"1110030E03Rik, BAF250a, Osa1, Smarcf1\"\\tColon Cancer\\tDOID:219\\tD003110\\t--\\t--\\t--\\t\"Arid1a,Arid1b\"\\t\"1110030E03Rik,BAF250a,Osa1,Smarcf1,8030481M12,9330189K18Rik,AI836955,Ardi1b,B230217J03Rik,BAF250B,mKIAA1235\"\\tChIP-seq\\t\"ChIP-Seq profiles for HCT116 cells, including those generated by ENCODE21?(Encyclopedia of DNA Elements,?Supplementary Table 6), revealed further that H3K27ac loss was most strongly associated with sites bound in WT cells by SWI/SNF complexes and/or TFs including AP1, CEBPB, and TEAD4 (Fig. 5b). Among factors assessed in HCT116 cells, binding of these TFs correlated most strongly with SWI/SNF occupancy (Supplementary Fig. 6b) and was higher at enhancers that lost activity than at enhancers that were unaffected by ARID1A deficiency (Fig. 5c)\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t105176', 'E_02_180\\t27941798\\t--\\tmm10\\tchr15\\t63894512\\t63932136\\tMouse\\tHCT 116\\tLow+High throughput\\tChIP-seq\\t\"To determine if control of enhancer activity by SWI/SNF complexes is coordinated with TFs, we analyzed sequence motifs at enhancers sensitive to ARID1A loss. The CTCF motif was relatively depleted in regions of H3K27ac loss, implying CTCF-bound insulator regions may resist modulation, while the AP1 (JUND/FOSL1) motif was most enriched (Fig. 5a, Supplementary Table 5).Among factors assessed in HCT116 cells, binding of these TFs correlated most strongly with SWI/SNF occupancy (Supplementary Fig. 6b) and was higher at enhancers that lost activity than at enhancers that were unaffected by ARID1A deficiency (Fig. 5c). \"\\tSuper-Enhancer\\tArid1a\\t--\\t\"ChIP-seq,ChIP-qPCR\"\\t\"To determine if control of enhancer activity by SWI/SNF complexes is coordinated with TFs, we analyzed sequence motifs at enhancers sensitive to ARID1A loss. The CTCF motif was relatively depleted in regions of H3K27ac loss, implying CTCF-bound insulator regions may resist modulation, while the AP1 (JUND/FOSL1) motif was most enriched (Fig. 5a, Supplementary Table 5).Among factors assessed in HCT116 cells, binding of these TFs correlated most strongly with SWI/SNF occupancy (Supplementary Fig. 6b) and was higher at enhancers that lost activity than at enhancers that were unaffected by ARID1A deficiency (Fig. 5c). \"\\t\"1110030E03Rik, BAF250a, Osa1, Smarcf1\"\\tColon Cancer\\tDOID:219\\tD003110\\t--\\t--\\t--\\t\"Arid1a,Arid1b\"\\t\"1110030E03Rik,BAF250a,Osa1,Smarcf1,8030481M12,9330189K18Rik,AI836955,Ardi1b,B230217J03Rik,BAF250B,mKIAA1235\"\\tChIP-seq\\t\"ChIP-Seq profiles for HCT116 cells, including those generated by ENCODE21?(Encyclopedia of DNA Elements,?Supplementary Table 6), revealed further that H3K27ac loss was most strongly associated with sites bound in WT cells by SWI/SNF complexes and/or TFs including AP1, CEBPB, and TEAD4 (Fig. 5b). Among factors assessed in HCT116 cells, binding of these TFs correlated most strongly with SWI/SNF occupancy (Supplementary Fig. 6b) and was higher at enhancers that lost activity than at enhancers that were unaffected by ARID1A deficiency (Fig. 5c)\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t137354', 'E_02_181\\t27986456\\t--\\tmm10\\tchr18\\t69656860\\t69683465\\tMouse\\tHoxB8-FL cells\\tLow+High throughput\\tATAC-seq\"ATAC-Seq analysis of Tcf4 locus in primary DCs. Shown are ATAC-Seq peaks across ~1.5 Mb of the Tcf4 locus; indicated are proximal promoters of Tcf4L (green) and Tcf4S (purple), the 3 enhancer (orange) and the cluster of 5 regulatory elements (yellow). In this and subsequent panels, the scale was adjusted between different samples based on signals from housekeeping genes.We analyzed the 3 region by ChIP during the differentiation of HoxB8-FL cells. Histone modifications associated with active enhancers, including histone 3 lysine 4 monomethylation (H3K4me1) and lysine 27 acetylation (H3K27ac), were induced in this region on day 4, i.e. at the onset of pDC differentiation (Fig. 5D).\"\\tSuper-Enhancer\\tTcf4\\tCRISPR/Cas9\\t\"ChIP-seq,ATAC-seq\"\\tChIP analysis of histone modifications (H3K4me1 and H3K27ac) and of TCF4 binding to the 3 Enhancer in differentiating HoxB8-FL cells.\\t\"5730422P05Rik,ASPI2,E2-2,E2.2,ITF-2,ITF-2b,ITF2,ME2,MITF-2A,MITF-2B,SEF-2,SEF2,SEF2-1,TFE,Tcf-4,bHLHb19\"\\t--\\t--\\t--\\t\"A conserved Enhancer downstream of Tcf4 was required for its upregulation during pDC differentiation, revealing a positive feedback loop\"\\t\"qRT-PCR,CRISPR/Cas9\"\\t\"By qRT-PCR, Tcf4S was prominently expressed in pDCs but also present in other cell types including B cells and cDCs\"\\tTcf4\\t\"5730422P05Rik,ASPI2,E2-2,E2.2,ITF-2,ITF-2b,ITF2,ME2,MITF-2A,MITF-2B,SEF-2,SEF2,SEF2-1,TFE,Tcf-4,bHLHb19\"\\tqRT-PCR\\t\"By qRT-PCR, Tcf4S was prominently expressed in pDCs but also present in other cell types including B cells and cDCs.', 'E_02_182\\t28045957\\tTES/TESCO Enhancer\\tmm10\\tchr11\\t112769210\\t112772210\\tMouse\\tTestis\\tLow throughput\\t\"Transgenic mice,CRISPR/Cas9,qRT-PCR\"\\t\"During mouse sex determination, transient expression of the Y-linked gene Sry up-regulates its direct target gene Sox9, via a 3.2 kb testis specific enhancer of Sox9 (TES), which includes a core 1.4 kb element, TESCO. SOX9 activity leads to differentiation of Sertoli cells, rather than granulosa cells from the bipotential supporting cell precursor lineage. Here, we present functional analysis of TES/TESCO, using CRISPR/Cas9 genome editing in mice. Deletion of TESCO or TES reduced Sox9 expression levels in XY fetal gonads to 60 or 45% respectively relative to wild type gonads, and reduced expression of the SOX9 target Amh. \"\\tEnhancer\\tSox9 \\t--\\tqRT-PCR\\t\"These results of qRT-PCR clearly indicate that the TESCO Enhancer is an important regulator of Sox9 expression throughout testis development, accounting for around 40% of its expression levels.\"\\t\"2010306G03Rik,AV220920,mKIAA4243\"\\t--\\t--\\t--\\treduced expression of the SOX9 target Amh.\\tCRISPR/Cas9\\t\"Here, we present functional analysis of TES/TESCO, using CRISPR/Cas9 genome editing in mice. Deletion of TESCO or TES reduced Sox9 expression levels in XY fetal gonads to 60 or 45% respectively relative to wild type gonads, and reduced expression of the SOX9 target Amh.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t11500', 'E_02_183\\t26163528\\tId2 Enhancer\\tmm10\\tchr12\\t25090158\\t25096935\\tMouse\\tApoptotic Cell\\tLow throughput\\tLuciferase Reporter Assay\\t\"To test this Tcf4 binding region for enhancer activity, we conducted luciferase reporter assays using the constructs shown in Fig. 1F. The putative Id2-enhancer fragment significantly activated Wnt-signaling dependent transcription from the CMV minimal promoter in two colorectal cancer cell lines (HCT116 and SW480) (Fig. 1G). Therefore, these results indicate that Id2 expression is stimulated by Wnt signaling through the enhancer region, in tumor epithelial cells of Apc716 mice at an early stage of tumorigenesis.\"\\tEnhancer\\tId2\\t--\\tLuciferase Reporter Assay\\tSchematic representation of theLuciferase (Luciferase) expression constructs containing the Id2 Enhancer with/without a CMV-minimal promoter.\\t\"AI255428,C78922,Idb2,bHLHb26\"\\tColorectal Carcinoma\\tDOID:0080199\\tD015179\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t251', 'E_02_184\\t28087634\\t--\\tmm10\\tchr2\\t105072843\\t105076843\\tMouse\\tSertoli Cells\\tLow+High throughput\\t\"ChIP-seq,DNaseI-seq\"\\t\"H3K27ac ChIP-seq on E13.5 Sertoli cells identified >28,000 H3K27ac peaks, ?70% of which overlapped a DHS in our E13.5 Sertoli cell dataset (Fig. 3A, Fig. S4).H3K27ac-positive DHSs, which are indicative of active enhancers, were significantly enriched specifically in neighboring Sertoli cell-expressed genes (Fig. 3D), as expected.\"\\tEnhancer\\tWt1 \\t--\\t\"ChIP-seq,DNaseI-seq\"\\t\"To demonstrate this, we performed a transient transgenic assay with a putative active enhancer of the Wt1 gene (Fig. 5A,B).We identified a DHS peak located 50?kb upstream of Wt1 that was unique to E13.5 and E15.5 Sertoli cells compared with the other cell types we examined, and marked by H3K27ac, suggesting that it functions as an active enhancer.\"\\t\"D630046I19Rik,Wt-1\"\\tGenitourinary Defects\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t51686', 'E_02_185\\t28104492\\t--\\tmm10\\tchr15\\t97898854\\t97902854\\tMouse\\tYAMC Cell\\tLow throughput\\t\"Luciferase Reporter Assay,Western blot,DNaseI-seq\"\\t\"Zella et al. [22] found that transcription of the mouse VDR gene is regulated at seven promoter and enhancer regions (Fig. 5A) in the mouse osteoblast cell line MC3T3-E1. In addition, the ENCODE project has identified DNAse I hypersensitive sites in the mouse large intestine that overlap with these regions.\"\\tEnhancer\\tVdr\\t--\\t\"Luciferase Reporter Assay,Western blot\"\\t\"However, chromatin accessibility to the VDR gene at the proximal promoter (_x0001_300 bp), an Enhancer region at _x0001_6 kb, and an Enhancer region located in exon 7 was significantly reduced in RAS transformed YAMC cells (YAMC-RAS).\"\\tNr1i1\\tColon Cancer\\tDOID:219\\tD003110\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t46428', 'E_02_186\\t28104492\\t--\\tmm10\\tchr15\\t97911426\\t97915997\\tMouse\\tYAMC Cell\\tLow throughput\\t\"Luciferase Reporter Assay,Western blot,DNaseI-seq\"\\t\"Zella et al. [22] found that transcription of the mouse VDR gene is regulated at seven promoter and enhancer regions (Fig. 5A) in the mouse osteoblast cell line MC3T3-E1. In addition, the ENCODE project has identified DNAse I hypersensitive sites in the mouse large intestine that overlap with these regions.\"\\tEnhancer\\tVdr\\t--\\t\"Luciferase Reporter Assay,Western blot\"\\t\"However, chromatin accessibility to the VDR gene at the proximal promoter (_x0001_300 bp), an Enhancer region at _x0001_6 kb, and an Enhancer region located in exon 8 was significantly reduced in RAS transformed YAMC cells (YAMC-RAS).\"\\tNr1i1\\tColon Cancer\\tDOID:219\\tD003110\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t59286', 'E_02_187\\t28121289\\t--\\tmm10\\tchr6\\t88202011\\t88204330\\tMouse\\tEmbryonic Endothelial Cell\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq,Western blot\"\\t\"Viewing Ep300fb bioChiP-seq signal at genes selectively expressed in heart or brain confirmed robust tissue-specific differences that overlapped enhancers with known tissue-specific activity.To further functionally validate the transcriptional activity of these heterodimer motifs, we measured their enhancer activity using luciferase reporter assays.\"\\tEnhancer\\tGata2\\t--\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"At the skeletal muscle specific gene Myod, Myf5Cre drove strong Ep300fb bioChiP-seq signal at a known distal Enhancer (Goldhamer et al., 1992), as well as a second Ep300 bound region about 12 kb upstream from the transcriptional start site.\"\\tGata-2\\t--\\t--\\t--\\t--\\t--\\t--\\tEp300\\t\"A430090G16,A730011L11,KAT3B,p300,p300 HAT\"\\tChIP-seq\\t\"However, antibodies for Ep300 are marginal for robust ChIP-seq, particularly from tissues, leading to low reproducibility, variation between antibody lots, and inefficient Enhancer identification.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t9735', 'E_02_188\\t28213503\\t--\\tmm10\\tchr11\\t44386118\\t44393090\\tMouse\\tMacrophage\\tLow+High throughput\\tChIP-seq\"However, following treatment with acute IL-10, H3K27ac was significantly lower at peaks associated with genes from Cluster 2 than at peaks associated with genes from Cluster 1 (p-value <0.01 by Mann-Whitney test, Fig. 6A). Examples of Cluster 1 and Cluster 2 enhancers are shown in Fig. 6B and Fig. 6C, respectively. Interestingly this analysis identified a peak inhibited by acute IL-10 that encompasses a DNAse hypersensitivity site 10 kb upstream of Il12b that has previously been demonstrated in a reporter assay to exhibit enhancer activity (Fig. 6C).\"\\tEnhancer\\tCxcl2\\t--\\t\"ChIP-seq,RT-PCR\"\\t\"To evaluate this issue, we compared induction of Cxcl2 transcription in IL-10-deficient macrophages stimulated with LPS alone, both LPS and IL-10, or stimulated with IL-10 for 1 hour prior to stimulation with LPS. As predicted, LPS rapidly induced Cxcl2 pre-mRNA within 1h hour of stimulation. Surprisingly, addition of IL-10 at the time of LPS stimulation or addition 1 hour prior to LPS stimulation had little influence on Cxcl2 pre-mRNA at 1h post LPS stimulation, although significant suppression was observed at 2h post LPS stimulation (Fig. 8A). \"\\t\"CINC-2a,GROb,Gro2,MIP-2,MIP-2a,Mgsa-b,Mip2,Scyb,Scyb2\"\\tInflammatory Disease\\t--\\t--\\t--\\t--\\t--\\tStat3\\t\"1110034C02Rik, AW109958, Aprf\"\\tChIP-seq\\t\"To address this possibility, we performed ChIP-seq with an anti-STAT3 antibody to identify STAT3 binding sites.Interestingly, virtually all identified STAT3 binding sites were located within H3K4me1 peaks, suggesting that STAT3 binds to enhancers. As anticipated, we found strong STAT3 binding near Cluster 3 genes induced by IL-10, such as Socs3 (Fig. 7, bottom left), and IL-10 induced an increase in average mean acetylation at STAT3 binding sites associated with genes in Cluster 3, consistent with enhancer activation (Fig. 7, bottom right).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t46514294', 'E_02_189\\t28262751\\t--\\tmm10\\tchr4\\t97123309\\t97712957\\tMouse\\tEmbryonic Fibroblast\\tLow+High throughput\\tChIP-seq\\t\"Following inactivation of either Smarcb1 or Smarca4, we observed little change in H3K27ac levels at promoters, but a marked reduction at many enhancers (Fig. 1b,c). This reduction at enhancers was robust and consistent across replicate experiments performed on MEFs derived from independent mice (Methods, Supplementary Figs 3-7).\"\\tEnhancer\\tNfia\\t--\\t\"ChIP-seq,Western blot\"\\t\"To investigate whether SWI/SNF is acting directly at active enhancers, we performed ChIP-Seq for the core SWI/SNF subunits SMARCC1 and SMARCA4 in wild-type and Smarcb1-deficient cells. Because the genome-wide binding profiles of these two subunits were very similar (Supplementary Fig. 3), we considered the average of the two experiments to be representative of SWI/SNF binding for each condition. Interestingly, we found that SWI/SNF was bound at the vast majority of enhancers in wild-type cells-over 95% of enhancers showed enrichment (IP>input; Fig. 2a and Supplementary Fig. 9a). Deletion of the Smarcb1 subunit led to a widespread reduction in SWI/SNF binding (Fig. 2b and Supplementary Fig. 9b).Taken together, the targeting of SWI/SNF to enhancers and the association between loss of SWI/SNF binding and the loss of H3K27ac suggests a direct role for the SWI/SNF complex in regulating the enhancer chromatin landscape in MEFs.\"\\t\"1110047K16Rik,9430022M17Rik,CTF,NF1-A,NF1A\"\\tMalignancies\\t--\\t--\\t--\\t--\\t--\\tEp300\\t\"A430090G16,A730011L11,KAT3B,p300,p300 HAT\"\\tChIP-seq\\t\"To further evaluate how the interaction between SWI/SNF and p300 regulates H3K27ac levels, we immunoprecipitated SWI/SNF complexes using an antibody against the core subunit SMARCC1 and measured histone acetylation activity using recombinant histones as a substrate. With immunoprecipitated SWI/SNF complexes from a RT cell line that lacked SMARCB1, the sample lacked acetyltransferase activity. However, re-expression of SMARCB1 increased H3K27-specific acetyltransferase activity, despite acetylation of H3K9ac being not significantly changed, suggesting the presence of H3K27-specific acetylation activity (Fig. 3d). \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t163582', 'E_02_190\\t28335007\\t--\\tmm10\\tchr5\\t9034150\\t9036900\\tMouse\\tLiver\\tLow+High throughput\\tChIP-seq\\t\"On the other hand, the enhancer regions associated with lncRNAs tended to have much higher intensities of H3K27ac marks than the enhancers not associated with lncRNAs (KS test, P-value < 2.2  10?16) (Figure ?(Figure2E).2E). The enhancers associated with circadian lncRNAs have even higher levels of H3K27ac marks than those associated with lncRNAs (KS test, P-value = 0.0017). Taken together, there is a strong association between circadian lncRNAs and super-enhancers.\"\\tSuper-Enhancer\\tGm40264\\t4C\\t\"qPCR,RNA-seq,ChIP-seq\"\\t\"The enhancers defined from mouse liver GRO-seq and 5? CAGE technology were consistent with H3K27ac marks of enhancer at this lnc-Crot region (Supplementary Figure S3C). Using RACE assay, we obtained the full-length sequence of lnc-Crot (Figure ?(Figure4B).4B). Both BMAL1 and REV-ERB binding sites were found on the promoter and gene body of lnc-Crot (Figure ?(Figure4B)4B) but not on the promoters of nearby circadian genes including Crot, Tmem243 and Dmtf1 (Supplementary Figure S3D).By qPCR, we showed that Crot and lnc-Crot were co-expressed (r = 0.8) across adult mouse tissues with elevated expression in liver and kidney (Supplementary Figure S3F).4C signals from three biological replicates at each circadian time were highly reproducible judging from their cross-sample correlations between 0.89 and 0.95. About 44% of 4C signals fell within chromosome 5 where lnc-Crot is situated and the signal profile around lnc-Crot region was displayed in Supplementary Figure S5A. \"\\tlnc-Crot\\t--\\t--\\t--\\tEnhancer-associated circadian lncRNAs mark the genomic loci modulating long-range circadian gene regulation and shed new lights on the evolutionary origin of lncRNAs.\\t\"qPCR,CRISPR/Cas9\"\\tOur result points to a model that the lnc-Crot locus and nearby genes first form a scaffold upon which histone modifications and TF bindings then take place to confer tissue- and time-dependent expression (Figure ?(Figure5G).5G). The circadian lncRNAs that we examined may be a result of circadian transcription at the enhancer regions while the lncRNAs themselves do not influence the circadian gene regulation.\\t\"Arntl,Nr1d2\"\\t\"Arnt3,BMAL1b,Bmal1,MOP3,bHLHe5,bmal1b\\',RVR,Rev-erb\"\\t\"ChIP-seq,RNA-seq,Knockout mice\"\\t\"We defined BMAL1 and REV-ERB binding sites from two sets of BMAL1 ChIP-seq studies and two sets of REV-ERB ChIP-seq data in mouse liver respectively, resulting in 8001 BMAL1and 17545REV-ERB binding sites.Atotal of 259 (43%) circadian lncRNA clusters contain either BMAL1 or REV-ERB binding sites.For each lncRNA cluster, we quantified its expression in the BMAL1 knockout and WT mouse livers at CT0 and CT12 using our RNA-seq data.This suggested that circadian lncRNAs were regulated by these two master circadian TFs in the similar way as circadian protein coding genes.Circadian expression of lnc-Crot was disrupted when BMAL1 or REV-ERB was knocked out, resulting in its downregulation at CT0 in BMAL1 knockout mice and upregulation at CT12 in REV-ERB knockout mice (Figure ?(Figure4B,4B, Supplementary Figure S3E). This demonstrated that lnc-Crot was directly regulated by BMAL1 and REV-ERB.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1075054', 'E_02_191\\t28335007\\t--\\tmm10\\tchr5\\t9037806\\t9041881\\tMouse\\tLiver\\tLow+High throughput\\tChIP-seq\\t\"On the other hand, the enhancer regions associated with lncRNAs tended to have much higher intensities of H3K27ac marks than the enhancers not associated with lncRNAs (KS test, P-value < 2.2  10?16) (Figure ?(Figure2E).2E). The enhancers associated with circadian lncRNAs have even higher levels of H3K27ac marks than those associated with lncRNAs (KS test, P-value = 0.0017). Taken together, there is a strong association between circadian lncRNAs and super-enhancers.\"\\tSuper-Enhancer\\tGm40264\\t4C\\t\"qPCR,RNA-seq,ChIP-seq\"\\t\"The enhancers defined from mouse liver GRO-seq and 5? CAGE technology were consistent with H3K27ac marks of enhancer at this lnc-Crot region (Supplementary Figure S3C). Using RACE assay, we obtained the full-length sequence of lnc-Crot (Figure ?(Figure4B).4B). Both BMAL1 and REV-ERB binding sites were found on the promoter and gene body of lnc-Crot (Figure ?(Figure4B)4B) but not on the promoters of nearby circadian genes including Crot, Tmem243 and Dmtf1 (Supplementary Figure S3D).By qPCR, we showed that Crot and lnc-Crot were co-expressed (r = 0.8) across adult mouse tissues with elevated expression in liver and kidney (Supplementary Figure S3F).4C signals from three biological replicates at each circadian time were highly reproducible judging from their cross-sample correlations between 0.89 and 0.95. About 44% of 4C signals fell within chromosome 5 where lnc-Crot is situated and the signal profile around lnc-Crot region was displayed in Supplementary Figure S5A. \"\\tlnc-Crot\\t--\\t--\\t--\\tEnhancer-associated circadian lncRNAs mark the genomic loci modulating long-range circadian gene regulation and shed new lights on the evolutionary origin of lncRNAs.\\t\"qPCR,CRISPR/Cas9\"\\tOur result points to a model that the lnc-Crot locus and nearby genes first form a scaffold upon which histone modifications and TF bindings then take place to confer tissue- and time-dependent expression (Figure ?(Figure5G).5G). The circadian lncRNAs that we examined may be a result of circadian transcription at the enhancer regions while the lncRNAs themselves do not influence the circadian gene regulation.\\t\"Arntl,Nr1d2\"\\t\"Arnt3,BMAL1b,Bmal1,MOP3,bHLHe5,bmal1b\\',RVR,Rev-erb\"\\t\"ChIP-seq,RNA-seq,Knockout mice\"\\t\"We defined BMAL1 and REV-ERB binding sites from two sets of BMAL1 ChIP-seq studies and two sets of REV-ERB ChIP-seq data in mouse liver respectively, resulting in 8001 BMAL1and 17545REV-ERB binding sites.Atotal of 259 (43%) circadian lncRNA clusters contain either BMAL1 or REV-ERB binding sites.For each lncRNA cluster, we quantified its expression in the BMAL1 knockout and WT mouse livers at CT0 and CT12 using our RNA-seq data.This suggested that circadian lncRNAs were regulated by these two master circadian TFs in the similar way as circadian protein coding genes.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1079373', 'E_02_192\\t28335007\\t--\\tmm10\\tchr5\\t9038118\\t9038593\\tMouse\\tLiver\\tLow+High throughput\\tChIP-seq\\t\"On the other hand, the enhancer regions associated with lncRNAs tended to have much higher intensities of H3K27ac marks than the enhancers not associated with lncRNAs (KS test, P-value < 2.2  10?16) (Figure ?(Figure2E).2E). The enhancers associated with circadian lncRNAs have even higher levels of H3K27ac marks than those associated with lncRNAs (KS test, P-value = 0.0017). Taken together, there is a strong association between circadian lncRNAs and super-enhancers.\"\\tSuper-Enhancer\\tGm40264\\t4C\\t\"qPCR,RNA-seq,ChIP-seq\"\\t\"The enhancers defined from mouse liver GRO-seq and 5? CAGE technology were consistent with H3K27ac marks of enhancer at this lnc-Crot region (Supplementary Figure S3C). Using RACE assay, we obtained the full-length sequence of lnc-Crot (Figure ?(Figure4B).4B). Both BMAL1 and REV-ERB binding sites were found on the promoter and gene body of lnc-Crot (Figure ?(Figure4B)4B) but not on the promoters of nearby circadian genes including Crot, Tmem243 and Dmtf1 (Supplementary Figure S3D).By qPCR, we showed that Crot and lnc-Crot were co-expressed (r = 0.8) across adult mouse tissues with elevated expression in liver and kidney (Supplementary Figure S3F).4C signals from three biological replicates at each circadian time were highly reproducible judging from their cross-sample correlations between 0.89 and 0.95. About 44% of 4C signals fell within chromosome 5 where lnc-Crot is situated and the signal profile around lnc-Crot region was displayed in Supplementary Figure S5A. \"\\tlnc-Crot\\t--\\t--\\t--\\tEnhancer-associated circadian lncRNAs mark the genomic loci modulating long-range circadian gene regulation and shed new lights on the evolutionary origin of lncRNAs.\\t\"qPCR,CRISPR/Cas9\"\\tOur result points to a model that the lnc-Crot locus and nearby genes first form a scaffold upon which histone modifications and TF bindings then take place to confer tissue- and time-dependent expression (Figure ?(Figure5G).5G). The circadian lncRNAs that we examined may be a result of circadian transcription at the enhancer regions while the lncRNAs themselves do not influence the circadian gene regulation.\\t\"Arntl,Nr1d2\"\\t\"Arnt3,BMAL1b,Bmal1,MOP3,bHLHe5,bmal1b\\',RVR,Rev-erb\"\\t\"ChIP-seq,RNA-seq,Knockout mice\"\\t\"We defined BMAL1 and REV-ERB binding sites from two sets of BMAL1 ChIP-seq studies and two sets of REV-ERB ChIP-seq data in mouse liver respectively, resulting in 8001 BMAL1and 17545REV-ERB binding sites.Atotal of 259 (43%) circadian lncRNA clusters contain either BMAL1 or REV-ERB binding sites.For each lncRNA cluster, we quantified its expression in the BMAL1 knockout and WT mouse livers at CT0 and CT12 using our RNA-seq data.This suggested that circadian lncRNAs were regulated by these two master circadian TFs in the similar way as circadian protein coding genes.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1077885', 'E_02_193\\t28575289\\t--\\tmm10\\tchr1\\t136170878\\t136192613\\tMouse\\tC2C12\\tLow+High throughput\\tChIP-seq\\t\"To further monitor the progressive remodeling of SE during the MB differentiation into MTs, we generated a time series of H3K27ac ChIP-seq data covering the differentiating course of ?24,0,6,12,24 and 72 hr (Supplementary Data S2). As shown in Figure??Figure1R,1R, progressive temporal changes in enhancer usage were readily observed during the progression of differentiation.\"\\tSuper-Enhancer\\tMyog\\t3C\\t\"qRT-PCR,ChIP-seq\"\\t\"To further illuminate whether the cooperation is mediated through physical interaction through chromosomal looping, 3C (chromosome confirmation capture) assay was performed followed by PCR to analyze the chromatin loop structure (Figure ?(Figure5L).5L). Detectable looping frequency between H3 and H4 or the promoter but not with the negative control regions upstream of H1 or downstream of Myogenin gene was observed upon early differentiation at 24 h and persisted throughout the differentiation whereas the interaction between H3 and H1 or H2 occurred in a later stage (Figure ?(Figure5L5LCN and Supplementary Figure S6F), confirming H3 is the earliest to be activated then probably helped the establishment of H4 and others through their interactions.\"\\t\"MYF4,bHLHc3,myo\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Foxo3,Myod1\"\\t\"1110048B16Rik,2010203A17Rik,C76856,FKHRL1,Fkhr2a,Foxo3,AI503393,MYF3,MyoD,Myod-1,bHLHc1\"\\t\"CRISPR/Cas9,Western blot\"\\t\"Indeed, the deletion of MyoD by CRISPR-Cas9 in the cells (Supplementary Figure S3ACD) induced extraordinary changes at SE landscape; 584 SEs in WT were lost while 257 new ones were gained (Supplementary Figure S3E and F). The deletion not only sharply attenuated the enhancer activity at the WT SE hotspots (Figure ?(Figure3A),3A), but also resulted in the activation of genes that acquired de novo SEs in KO cells (Supplementary Figure S3G).HA-tagged MyoD was co-transfected with plasmid expressing Flag-tagged full length FoxO3. Lysates were pulled down by HA antibody and examined by western blotting using Flag antibody. For all the above panels, the representative images from two replicates are shown. \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t1891743', 'E_02_194\\t28614915\\tephrin-A5 Enhancer \\tmm10\\tchr17\\t62627957\\t62633457\\tMouse\\tMesencephalon\\tLow throughput\\tLuciferase Reporter Assay\\t\"Luciferase activity assay showing that Ascl1- E47 fusion protein induces transcriptional activation through the ephrin-A5 E-box DNA.Taken together, our findings suggest that the mesencephalon-specific enhancer of ephrin-A5 resided in a genomic region that is +11.6 kb to +57.7 kb from the TSS and is within the first intron (Fig. 1A).\"\\tEnhancer\\tEfna5\\t--\\t\"Luciferase Reporter Assay,PCR\"\\t\"To further restrict the possible locations of the mesencephalon-specific enhancer within the first intron, the 375B7 BAC was modified using a different targeting vector. Importantly, transgenic embryos carrying this recombinant BAC did not show significant LacZ expression in the mesencephalon (Fig. 2C, first panel from the left). This suggested that the deleted 30.4 kb region included the mesencephalon-specific enhancer of ephrin-A5.This suggests that the potential enhancer for ephrin-A5 expression in the mesencephalon was in a 5.5 kb genomic region from +25 kb to +30.5 kb of the TSS.\"\\t\"AL-1,AV158822,EFL-5,Ephrin-A5,Epl7,LERK-7,RAGS\"\\t--\\t--\\t--\\t--\\t--\\t--\\tAscl1\\t\"AI225900,ASH1,Mash1,bHLHa46\"\\t\"EMSA,Luciferase Reporter Assay\"\\t\"Finally, both electrophoretic mobility shift assay and luciferase-based transactivation assay revealed that the transcription factor Ascl1 bound the bHLH consensus binding site in the mesencephalon-specific ephrin-A5 enhancer in vitro. \"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t27749', 'E_02_195\\t28667437\\t--\\tmm10\\tchr9\\t88325101\\t88326201\\tMouse\\tHepatic Myofibroblasts\\tLow throughput\\tLuciferase Reporter Assay\\t\"The measured luciferase activity noticeably decreased between F1- and F2-Luc constructs, suggesting that F1-Luc construct contains a potential enhancer sequence, which spans approximately 1000 bp (?2069 to ?996 bp) and is absent in the shorter F2-Luc construct.\"\\tEnhancer\\tNt5e\\t--\\t\"Luciferase Reporter Assay,PCR\"\\tLuciferase activity as_x0002_says showed that the ENH1074-F5-Luc construct containing the full putative Enhancer (-2069 to -996 bp) sequence has the ability to enhance the luciferase activity of the identified putative mouse Nt5e promoter.\\t\"2210401F01Rik,5\\'-NT,AI447961,CD73,NT,Nt5,eNT\"\\tHepatic Fibrosis\\t--\\tD008103\\t--\\t--\\t--\\tElf2\\t\"2610036A20Rik,A230104O07Rik,AW111824,BB183398,EU32,NERF,NERF-1A,NERF-1B,NERF-2\"\\tLuciferase Reporter Assay\\t\"We found that mutation of Elf2 responsive element alone was sufficient to significantly increase the F5-Luc construct luciferase activity (F5-Luc, 11.79  1.41 vs. mut Elf2-F5-Luc,20.48  2.96, **p < 0.01).These data indicate that repression of mouse Nt5e promoter activity is likely mediated by transcription factor(s) acting at the identified Elf2 site in NIH3T3 fibroblasts.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1957', 'E_02_196\\t28671686\\t--\\tmm10\\tchr8\\t36785613\\t36985613\\tMouse\\tBrain\\tLow+High throughput\\t\"ChIP-seq,Hi-C\"\\t\"We then assessed longer-range chromosomal contacts, spanning >200kb of linear genome. We identified genome-wide 110 long-range loop contacts affected in mutant neurons (DESeq2 P<0.05). Unexpectedly, the large majority, 84/110 or 76%, represented clustered, locus-specific loop aggregates showing massive weakening, or complete loss, after neuronal Setdb1 ablation (Figure 1d, Supplementary Figure 2c). hese deficits were specific, because ChiP-seq profiling for open chromatin-associated acetyl-H3-lysine 27 (H3K27ac) showed that 96.4% of 1,112 differentially tagged sequences were hyperacetylated in Setdb1-deficient neurons (Figure 2a, Supplementary Tables 1,2).\"\\tEnhancer\\tSetdb1\\t--\\tChIP-seq\\t\"Therefore, we predicted that higher order chromatin organization at these positions will be highly conserved in human brain cells, and specifically in neurons, with long-range loopings radiating from ~200kb R1 towards cPCDH promoter and Enhancers primarily anchored in chromatin at and around the Setdb1 peak. To explore, we surveyed with 40kb resolution the cPCDH -bound chromosomal contacts in our in situ Hi-C datasets generated from human neurons and their isogenic neural precursors cells (NPC) and NPC-differentiated astrocytes.\"\\t\"AU022152,ESET,KMT1E,mKIAA0067\"\\tSchizophrenia\\tDOID:5419\\tD012559\\t--\\t--\\t--\\tCtcf\\tAW108038\\tChIP-seq\\tChIP-seq data from stem cells and CD19+ B lymphocytes significant CTCF motif enrichment.CTCF motif (red) enrichment in sequences H3K9me3 hypomethylated in KO. (e) (Left) FACS plots showing separation of crosslinked NeuN+ from NeuN? nuclei (adult cortex) for cell-type specific CTCF ChIP-seq.\\t--\\t--\\t--\\t>2KB\\t\\t\\t58437911', 'E_02_197\\t28749941\\t--\\tmm10\\tchr10\\t67853705\\t67855705\\tMouse\\tEmbryo\\tLow+High throughput\\t\"ChIP-seq,4C-seq\"To investigate the correlation between the activity of the Krox20 cis-regulatory elements and their chromatin modifications and conformation, we first performed ChIP-seq experiments to analyse two histone modifications: H3K4me1 (broad peaks covering active enhancers) and H3K27ac (punctuated peaks covering active enhancers and promoters).In E8.5 wild type embryo heads, a number of H3K4me1 peaks were observed, including those that expectedly correspond to the A, B and C elements and to a previously known neural crest element (NCE; Fig 4A). \"\\tEnhancer\\tEgr2\\t4C-seq\\t--\\t\"To better characterize the Krox20 regulatory neighborhood, we used circular chromosome conformation capture (4C-seq) on multiple viewpoints in the locus. In samples prepared from total embryos at embryonic day (E) 9.5, when Krox20 is no more transcribed, the Krox20 gene and its distant regulatory element A (separated by over 200 kb) show highly similar distributions of 4C-seq signal (Fig 3A and S2B and S2C Fig) preferentially located in the Krox20 TAD. \"\\t\"Egr-2,Krox-20,Krox20,NGF1-B,Zfp-25,Zfp-6\"\\t--\\t--\\t--\\t\"Enhancer element C possesses a dual activity: besides its classical Enhancer function, it is also permanently required in cis to potentiate the autoregulatory activity of element A, by increasing its chromatin accessibility.\"\\tChIP-seq\\t\"Here, using both enhancer knock-outs and investigations of chromatin organisation, we show that element C possesses a dual activity: besides its classical enhancer function, it is also permanently required in cis to potentiate the autoregulatory activity of element A, by increasing its chromatin accessibility. \"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t320253', 'E_02_198\\t28749941\\t--\\tmm10\\tchr10\\t67926705\\t67928705\\tMouse\\tEmbryo\\tLow+High throughput\\t\"ChIP-seq,4C-seq\"To investigate the correlation between the activity of the Krox20 cis-regulatory elements and their chromatin modifications and conformation, we first performed ChIP-seq experiments to analyse two histone modifications: H3K4me1 (broad peaks covering active enhancers) and H3K27ac (punctuated peaks covering active enhancers and promoters).In E8.5 wild type embryo heads, a number of H3K4me1 peaks were observed, including those that expectedly correspond to the A, B and C elements and to a previously known neural crest element (NCE; Fig 4A). \"\\tEnhancer\\tEgr2\\t4C-seq\\t--\\t\"To better characterize the Krox20 regulatory neighborhood, we used circular chromosome conformation capture (4C-seq) on multiple viewpoints in the locus. In samples prepared from total embryos at embryonic day (E) 9.5, when Krox20 is no more transcribed, the Krox20 gene and its distant regulatory element A (separated by over 200 kb) show highly similar distributions of 4C-seq signal (Fig 3A and S2B and S2C Fig) preferentially located in the Krox20 TAD. \"\\t\"Egr-2,Krox-20,Krox20,NGF1-B,Zfp-25,Zfp-6\"\\t--\\t--\\t--\\t\"Enhancer element C possesses a dual activity: besides its classical Enhancer function, it is also permanently required in cis to potentiate the autoregulatory activity of element A, by increasing its chromatin accessibility.\"\\tChIP-seq\\t\"Here, using both enhancer knock-outs and investigations of chromatin organisation, we show that element C possesses a dual activity: besides its classical enhancer function, it is also permanently required in cis to potentiate the autoregulatory activity of element A, by increasing its chromatin accessibility. \"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t393253', 'E_02_199\\t28836200\\tCol1a2 Enhancer\\tmm10\\tchr6\\t4438697\\t4440197\\tMouse\\tMesenchymal-Fibroblast Cells\\tLow+High throughput\\t\"Luciferase Reporter Assay,DNase I Hypersensitivity Assay,ChIP-seq\"\\t\"Look for regions that have high conservation and good his_x0002_tone peaks and DNase1 peaks; you can use other tracks as exclusion criterias, but, generally, these marks are good indica_x0002_tors for enhancers without needing to conduct ChIP-seq or DNaseI hypersensitivity experiments (Fig. 5).\"\\tEnhancer\\tCol1a2\\t--\\t\"Luciferase Reporter Assay,DNase I Hypersensitivity Assay,ChIP-seq\"\\tIn vivo imaging of transgenic mice that harbor a luciferase gene can be used as a noninvasive longitudinal monitoring of the gene expression. Make sure to determine plateau of constant luciferase activity rather than peak of activity (see Notes 13). You may need to shade high-expressing areas to see other low levels (see Notes 14 and 15).\\t\"AA960264,AI325291,Col1a-2,Cola-2,Cola2,oim\"\\tFibrotic Disease\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t66170', 'E_02_200\\t28982762\\t--\\tmm10\\tchr4\\t55449628\\t55461828\\tMouse\\tMouse Embryonic Stem Cell\\tlow throughput\\t\"Luciferase Reporter Assay,3C\"\\t\"DNA fragments containing E1 and E2 specifically exhibited high transcriptional activity in na?ve state mESCs relative to primed EpiSCs as determined by transgene-driven luciferase assays, consistent with the Klf4 expression pattern in these two pluripotent cell types (Fig. 1B,C).To probe whether the putative enhancers E1 and E2 communicate with the Klf4 promoter ?55 kb away, we performed chromosome conformation capture (3C) assays to detect long-range interactions. For both E1 and E2, there was a significantly higher frequency of cross-linking to the Klf4 promoter in mESCs compared with EpiSCs (Fig. 1F), indicating that E1 and E2 spatially juxtapose the promoter in mESCs.\"\\tEnhancerKlf4\\t\"CRISPR/Cas9,3C\"\\t--\\t\"Additionally, biallelic deletion of either E1 or E2 by CRISPR/Cas9-mediated genome editing reduced Klf4 expression by ?70% and 85%, respectively, without significantly affecting the expression of an adjacent gene (Rad23b) or another pluripotency gene (Pou5f1) (Fig. 1E).Nevertheless, for the remainder of this report, we primarily discuss our findings regarding E1 and E2, as these two enhancers clearly represent dominant elements that regulate Klf4 expression in na?ve state mESCs.\"\\t\"EZF,Gklf,Zie\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox2,Stat3 \"\\t\"2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1,Sox-2,lcc,ysb,1110034C02Rik,AW109958,Aprf\"\\t\"ChIP-exo,CRISPR/Cas9,ChIP-qPCR\"\\t\"To validate the direct binding of these candidate TFs to the Klf4 Enhancers, we performed genome-wide high-resolution ChIP-exo experiments for SOX2, STAT3, and ESRRB in mESCs.Here, we probed whether the binding activities of OCT4, SOX2, ESRRB, and STAT3 at E2 are interdependent through ChIP-qPCR (ChIP combined with quantitative PCR) analysis following endogenous deletion of individual TF-binding sites.Peaks identified by ChIP-exo were consistent with peaks from previous ChIP-seq (ChIP combined with high-throughput sequcing) protocols (Supplemental Fig. S2B,C). As expected, we observed significant enrichment of all three TFs at the Klf4 enhancer cluster, particularly at E2 (Fig. 2D).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t71408', 'E_02_201\\t28982762\\t--\\tmm10\\tchr4\\t55453628\\t55465828\\tMouse\\tMouse Embryonic Stem Cell\\tlow throughput\\t\"Luciferase Reporter Assay,3C\"\\t\"DNA fragments containing E1 and E2 specifically exhibited high transcriptional activity in na?ve state mESCs relative to primed EpiSCs as determined by transgene-driven luciferase assays, consistent with the Klf4 expression pattern in these two pluripotent cell types (Fig. 1B,C).To probe whether the putative enhancers E1 and E2 communicate with the Klf4 promoter ?55 kb away, we performed chromosome conformation capture (3C) assays to detect long-range interactions. For both E1 and E2, there was a significantly higher frequency of cross-linking to the Klf4 promoter in mESCs compared with EpiSCs (Fig. 1F), indicating that E1 and E2 spatially juxtapose the promoter in mESCs.\"\\tEnhancerKlf4\\t\"CRISPR/Cas9,3C\"\\t--\\t\"Additionally, biallelic deletion of either E1 or E2 by CRISPR/Cas9-mediated genome editing reduced Klf4 expression by ?70% and 85%, respectively, without significantly affecting the expression of an adjacent gene (Rad23b) or another pluripotency gene (Pou5f1) (Fig. 1E).Nevertheless, for the remainder of this report, we primarily discuss our findings regarding E1 and E2, as these two enhancers clearly represent dominant elements that regulate Klf4 expression in na?ve state mESCs.\"\\t\"EZF,Gklf,Zie\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Oct4,Sox2,Stat3 \"\\t\"2810482H01Rik,NF-A1,Oct-1,Oct1,Otf-1,Otf1,Sox-2,lcc,ysb,1110034C02Rik,AW109958,Aprf\"\\t\"ChIP-exo,CRISPR/Cas9,ChIP-qPCR\"\\t\"To validate the direct binding of these candidate TFs to the Klf4 Enhancers, we performed genome-wide high-resolution ChIP-exo experiments for SOX2, STAT3, and ESRRB in mESCs.Here, we probed whether the binding activities of OCT4, SOX2, ESRRB, and STAT3 at E2 are interdependent through ChIP-qPCR (ChIP combined with quantitative PCR) analysis following endogenous deletion of individual TF-binding sites.Peaks identified by ChIP-exo were consistent with peaks from previous ChIP-seq (ChIP combined with high-throughput sequcing) protocols (Supplemental Fig. S2B,C). As expected, we observed significant enrichment of all three TFs at the Klf4 enhancer cluster, particularly at E2 (Fig. 2D).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t67408', 'E_02_202\\t29042628\\t--\\tmm10\\tchr16\\t92625802\\t92627802\\tMouse\\tHPC-7\\tLow+High throughput\\t\"4C-seq,ChIP\"\\t4C-seq in HPC-7 cells using the P1 and P2 promoters and the +24 enhancer as baits identified genomic interactions at Runx1 (Fig. 1a)\\tEnhancerRunx1\\t4C-seq\\t--\\t\"Strikingly, the +24 enhancer in particular forms many significant interactions outside the Runx1 domain (Fig. 1b,d). Many of these long-range interactions are with other genes or gene promoters on mouse chromosome 16 (Fig. 2). \"\\t\"AML1,CBF-alpha-2,Cbfa2,Pebp2a2,Pebpa2b\"\\tLeukaemia\\t--\\tD007938\\tknown haematopoietic enhancers that are likely to be involved in regulating?Runx1?expression in haematopoietic progenitor cells.\\t\"DNase I Hypersensitivity Assay,4C-seq,Hi-C\"\\t\"These regions not only drive haematopoietic expression, but also physically connect with the Runx1-P1 promoter, lending confidence to the concept that they are bona fide regulators of Runx1 transcription.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t25337', 'E_02_203\\t29078395\\tIl2ra Enhancer \\tmm10\\tchr8\\t11564419\\t11614821\\tMouse\\tT Cell\\tLow+High throughput\\t\"ChIP-seq,ChIA-PET\"\\t\"To clarify how distal enhancer elements within superenhancer loci form enhancer-promoter loops and chromatin interactions, we performed RNA Pol II-based chromatin interaction analysis by paired-end tag (ChIA-PET) sequencing (24, 25) using mouse CD4+ T cells that were preactivated by TCR stimulation (anti-CD3 plus anti-CD28) and then not treated or treated with IL-2 for 4 h.Interestingly, both H3K4me3 (blue peaks in Fig. 3, primarily at the promoter) and H3K27Ac (green peaks in Fig. 3, spanning the Il2ra superenhancer region; also shown in Fig. 1E) were strong after TCR preactivation and only modestly affected by stimulation with IL-2, in marked contrast to the changes in STAT5 binding and chromatin looping. \"\\tSuper-Enhancer\\tIl2ra\\t--\\t\"ChIP-seq,RNA-seq\"\\t\"As noted above, STAT5 and H3K27Ac are highly enriched at a subset of superenhancers, including at the Il2ra superenhancer (Fig. 1 A, B, and E), correlating with IL-2Cinduced gene expression (Fig. 2B and Fig. S1C). We also found extensive H3K27Ac and STAT5 binding at the human IL2RA locus (Fig. S2), analogous to the mouse Il2ra superenhancer, with significant conservation of some of the enhancer elements (Fig. S2).\"\\t\"CD25,Il2r,Ly-43\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Stat5a,Stat3\"\\t\"AA959963,STAT5,1110034C02Rik,AW109958,Aprf\"\\t\"ChIP-seq,CRISPR/Cas9\"\\t\"Based on our genome-wide analysis of chromatin immunoprecipitation coupled to DNA sequencing (ChIP-Seq) profiles, we observed that STAT5-enriched Super-Enhancers spanned large genomic regions (average size of 19 kb) and indeed were associated with high levels of H3K27Ac.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t53171', 'E_02_204\\t29113912\\t--\\tmm10\\tchr14\\t18175266\\t18186392\\tMouse\\tHepatocellular Carcinoma\\tLow+High throughput\\tChIP-seq\\t\"Notably, histone modification markers, including H3K4me1,H3K4me3, H3K27me3 and H3K27ac were presented at the Ang2 gene locus in liver tumor cells and were strongly colocal_x0002_ized with the two Arid1a-binding sites, suggesting an associa_x0002_tion between Arid1a binding and histone modification at the Ang2 gene locus (Fig. 4A). ChIP-qPCR analysis showed Arid1a knockdown increased the H3K27ac at the Ang2 proximal and distal regions.\"\\tEnhancer\\tAng2\\t--\\t\"ChIP-seq,ChIP-qPCR\"\\t\"Notably, histone modification markers, including H3K4me1,H3K4me3, H3K27me3 and H3K27ac were presented at the Ang2 gene locus in liver tumor cells and were strongly colocal_x0002_ized with the two Arid1a-binding sites, suggesting an associa_x0002_tion between Arid1a binding and histone modification at the Ang2 gene locus (Fig. 4A). ChIP-qPCR analysis showed Arid1a knockdown increased the H3K27ac at the Ang2 proximal and distal regions.\"\\t\"Angrp,Raa3,Rnase5b\"\\tHepatocellular CarcinomaDOID:684\\tD006528\\t--\\t--\\t--\\tEp300\\t\"A430090G16,A730011L11,KAT3B,p300,p300 HAT\"\\t\"ChIP-seq,ChIP-PCR\"\\t\"Interestingly,published ChIP-seq data also showed that p300 bound to the Ang2 gene locus, as we validated by ChIP-PCR.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t33014656', 'E_02_205\\t29133016\\t--\\tmm10\\tchr2\\t122651534\\t122668846\\tMouse\\tMacrophage\\tLow+High throughput\\t\"3C-seq,ChIP-seq\"\\t\"Next, we identified an enhancer set associated with LPS-responsive pri-miRNAs based on publicly available H3K4 mono-methylation-specific ChIP-seq and GRO-seq datasets. This enhancer set was further characterized by the combination of publicly available ChIP and ATAC-seq datasets. Finally, direct interactions between the miR-155-coding genomic region and its distal regulatory elements were identified using a 3C-seq approach.\"\\tEnhancer\\tTlr4\\t3C-seq\\t\"RT-qPCR,ChIP-seq,ATAC-seq\"\\t\"This Enhancer set was further characterized by the combination of publicly available ChIP and ATAC-seq datasets.Finally,direct interactions between the miR-155-coding genomic region and its distal regulatory elements were identified using a 3C-seq approach.\"\\t\"Lps,Ly87,Ran/M1,Rasl2-8\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"NFKB1,Junb\"\\t\"CVID12,EBP-1,KBF1,NF-kB,NF-kB1,NF-kappa-B1,NF-kappaB,NFKB-p105,NFKB-p50,NFkappaB,p105,p50\"\\tChIP-seq\\t\"As expected, de novo motif analysis under NFkB-p65 and JunB peaks revealed significant enrichment of NFkB-p65 and Jun-AP1 motifs (Supplementary Figure 3B and C). Targeted motif discovery at the LPS-responsive pri-miRNAs-associated enhancer subsets showed that both NFkB-p65 and Jun-AP1 motifs were found a higher number at the p65high enhancers compared to the p65low and LPS-repressed enhancers (Figure 3D).\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t55832640', 'E_02_206\\t29141987\\t--\\tmm10\\tchr2\\t103887795\\t103888195\\tMouse\\tEmbryo\\tLow+High throughput\\tChIP-seq\\t\"We overlapped TBS with genome\\\\wide H3K4me1 peaks from T WT mesodermal cells and found the same trend of decreased H3K27ac within 1 kb of TBS/H3K4me1 sites (Fig EV4E), suggesting that T functions to coordinate H3K27ac at putative enhancer regions throughout the genome, and may regulate the switch between poised H3K4me1(+) enhancers and active H3K27ac/H3K4me1(+) enhancer regions.\"\\tEnhancer\\tLmo2\\t--\\t\"RT-qPCR,ChIP-qPCR,ChIP-seq\"\\t\"Both the ?70 enhancer and the TSS of Lmo2 display a decrease in H3K27ac in T Y88A mutant mesodermal cells, in the ChIP\\\\Seq data and confirmed by ChIP\\\\qPCR (Figs ?(Figs5A5A and B, and EV5A). We reasoned that T binding to the ?70 enhancer of Lmo2 may recruit permissive chromatin marks to the TSS to regulate expression, and that this regulation is disrupted in T Y88A mutant mesodermal cells.\"\\t\"Rbtn-2,Rbtn2,Rhom-2,Ttg2\"\\t--\\t--\\t--\\t--\\t--\\t--\\tT\\t\"Bra,D17Mit170,Low,Lr1,Tbxt,Tl2,Tl3,cou,me75,T\"\\t\"ChIP-seq,ChIP-qPCR\"\\t\"We performed a co-immunoprecipitation analysis from in vitrodifferentiated mesodermal cells and found an interaction between endogenous T and p300.Strikingly, loss of H3K27ac was found at TBS upstream of the T genomic locus itself by ChIP-Seq . We validated this decrease in H3K27ac at the T genomic locus by ChIP-qPCR in TY88A and TWT mesodermal cells differentiated in vitro.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t69999', 'E_02_207\\t29141987\\t--\\tmm10\\tchr2\\t103886995\\t103888995\\tMouse\\tEmbryo\\tLow+High throughput\\tChIP-seq\\t\"We overlapped TBS with genome\\\\wide H3K4me1 peaks from T WT mesodermal cells and found the same trend of decreased H3K27ac within 1 kb of TBS/H3K4me1 sites (Fig EV4E), suggesting that T functions to coordinate H3K27ac at putative enhancer regions throughout the genome, and may regulate the switch between poised H3K4me1(+) enhancers and active H3K27ac/H3K4me1(+) enhancer regions.\"\\tEnhancer\\tLmo2\\t--\\t\"RT-qPCR,ChIP-qPCR,ChIP-seq\"\\t\"Both the ?70 enhancer and the TSS of Lmo2 display a decrease in H3K27ac in T Y88A mutant mesodermal cells, in the ChIP\\\\Seq data and confirmed by ChIP\\\\qPCR (Figs ?(Figs5A5A and B, and EV5A). We reasoned that T binding to the ?70 enhancer of Lmo2 may recruit permissive chromatin marks to the TSS to regulate expression, and that this regulation is disrupted in T Y88A mutant mesodermal cells.\"\\t\"Rbtn-2,Rbtn2,Rhom-2,Ttg2\"\\t--\\t--\\t--\\t--\\t--\\t--\\tT\\t\"Bra,D17Mit170,Low,Lr1,Tbxt,Tl2,Tl3,cou,me75,T\"\\t\"ChIP-seq,ChIP-qPCR\"\\t\"We performed a co-immunoprecipitation analysis from in vitrodifferentiated mesodermal cells and found an interaction between endogenous T and p300.Strikingly, loss of H3K27ac was found at TBS upstream of the T genomic locus itself by ChIP-Seq . We validated this decrease in H3K27ac at the T genomic locus by ChIP-qPCR in TY88A and TWT mesodermal cells differentiated in vitro.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t69999', 'E_02_208\\t29254942\\t--\\tmm10\\tchr1\\t131878852\\t131942815\\tMouse\\t\"Liver,Kidney\"\\tLow throughput \\t\"4C-seq,Transgenic mice,RNA-seq\"\\tChromosome conformation capture (4C-seq) in liver and kidney identified liver-specific chromatin loops that recruited clock-bound enhancers to promoters to regulate liver-specific transcriptional rhythms.?\\tEnhancer\\tSlc45a3\\t4C-seq\\t\"Transgenic mice,RNA-seq\"\\t\"Targeting the Slc45a3-short promoter with 4C-seq in liver and kidney showed liverspecific chromatin loops at three distal regions (two upstream, one downstream) (Figure 6A).\"\\t\"2210413P12Rik,AU023994,AU043764,IPCA-6,PRST,Pcanap6\"\\t--\\t--\\t--\\t\"Enhancers can contact a rhythmic promoter while looping out nearby nonrhythmic alternative promoters, confining rhythmic enhancer activity to specific promoters\"\\t4C-seq\\tChromosome conformation capture (4C-seq) in liver and kidney identified liver-specific chromatin loops that recruited clock-bound Enhancers to promoters to regulate liver-specific transcriptional rhythms.\\t\"Rore,Onecut1,Foxa\"\\t\"Rore,D9Ertd423e,HNF-6,Hfh12,Hnf6,Hnf6b,OC-1,Oc1,Foxa\"\\tChIP-seq\\t\"These liver-specific DHSs were bound by liver-specific TFs, FOXA2 and ONECUT1, and clock TF, REVERBa, as shown in ChIP-seq.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t52080', 'E_02_209\\t29254942\\t--\\tmm10\\tchr1\\t131920349\\t131962471\\tMouse\\t\"Liver,Kidney\"\\tLow throughput \\t\"4C-seq,Transgenic mice,RNA-seq\"\\tChromosome conformation capture (4C-seq) in liver and kidney identified liver-specific chromatin loops that recruited clock-bound enhancers to promoters to regulate liver-specific transcriptional rhythms.?\\tEnhancer\\tSlc45a3\\t4C-seq\\t\"Transgenic mice,RNA-seq\"\\t\"Targeting the Slc45a3-short promoter with 4C-seq in liver and kidney showed liverspecific chromatin loops at three distal regions (two upstream, one downstream) (Figure 6A).\"\\t\"2210413P12Rik,AU023994,AU043764,IPCA-6,PRST,Pcanap6\"\\t--\\t--\\t--\\t\"Enhancers can contact a rhythmic promoter while looping out nearby nonrhythmic alternative promoters, confining rhythmic enhancer activity to specific promoters\"\\t4C-seq\\tChromosome conformation capture (4C-seq) in liver and kidney identified liver-specific chromatin loops that recruited clock-bound Enhancers to promoters to regulate liver-specific transcriptional rhythms.\\t\"Rore,Onecut1,Foxa\"\\t\"Rore,D9Ertd423e,HNF-6,Hfh12,Hnf6,Hnf6b,OC-1,Oc1,Foxa\"\\tChIP-seq\\t\"These liver-specific DHSs were bound by liver-specific TFs, FOXA2 and ONECUT2, and clock TF, REVERBa, as shown in ChIP-seq.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t21504', 'E_02_210\\t29256171\\t--\\tmm10\\tchr10\\t24357349\\t24375452\\tMouse\\tEmbryonic Limb\\tLow+High throughput\\t\"Transgenic mice,ChIP-seq,DNase I Hypersensitivity Assay\"\\t\"Four functional enhancers were identified, with each driving distinct, tissue-specific patterns of transgene expression. An enhancer located -100 kb from the CCN2 transcription start site facilitated expression within vascular tissue. An enhancer -135 kb upstream of CCN2 drove expression within the articular chondrocytes of synovial joints.The other two enhancers, located at -198 kb and -229 kb, mediated transgene expression within dermal fibroblasts, however the most prevalent activity was found within hypertrophic chondrocytes and periosteal tissue, respectively. \"\\tEnhancer\\tCcn2\\t--\\t\"PCR,Transgenic mice\"\\t\"Four functional enhancers were identified, with each driving distinct, tissue-specific patterns of transgene expression. An enhancer located -100 kb from the CCN2 transcription start site facilitated expression within vascular tissue. An enhancer -135 kb upstream of CCN2 drove expression within the articular chondrocytes of synovial joints.The other two enhancers, located at -198 kb and -229 kb, mediated transgene expression within dermal fibroblasts, however the most prevalent activity was found within hypertrophic chondrocytes and periosteal tissue, respectively. \"\\t\"Ctgf,Fisp12,Hcs24,fisp-12\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t229041', 'E_02_211\\t29256171\\t--\\tmm10\\tchr10\\t24386336\\t24406594\\tMouse\\tEmbryonic Limb\\tLow+High throughput\\t\"Transgenic mice,ChIP-seq,DNase I Hypersensitivity Assay\"\\t\"Four functional enhancers were identified, with each driving distinct, tissue-specific patterns of transgene expression. An enhancer located -100 kb from the CCN2 transcription start site facilitated expression within vascular tissue. An enhancer -135 kb upstream of CCN2 drove expression within the articular chondrocytes of synovial joints.The other two enhancers, located at -198 kb and -229 kb, mediated transgene expression within dermal fibroblasts, however the most prevalent activity was found within hypertrophic chondrocytes and periosteal tissue, respectively. \"\\tEnhancer\\tCcn2\\t--\\t\"PCR,Transgenic mice\"\\t\"Four functional enhancers were identified, with each driving distinct, tissue-specific patterns of transgene expression. An enhancer located -100 kb from the CCN2 transcription start site facilitated expression within vascular tissue. An enhancer -135 kb upstream of CCN2 drove expression within the articular chondrocytes of synovial joints.The other two enhancers, located at -198 kb and -229 kb, mediated transgene expression within dermal fibroblasts, however the most prevalent activity was found within hypertrophic chondrocytes and periosteal tissue, respectively. \"\\t\"Ctgf,Fisp12,Hcs24,fisp-12\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t198977', 'E_02_212\\t29256171\\t--\\tmm10\\tchr10\\t24449590\\t24469418\\tMouse\\tEmbryonic Limb\\tLow+High throughput\\t\"Transgenic mice,ChIP-seq,DNase I Hypersensitivity Assay\"\\t\"Four functional enhancers were identified, with each driving distinct, tissue-specific patterns of transgene expression. An enhancer located -100 kb from the CCN2 transcription start site facilitated expression within vascular tissue. An enhancer -135 kb upstream of CCN2 drove expression within the articular chondrocytes of synovial joints.The other two enhancers, located at -198 kb and -229 kb, mediated transgene expression within dermal fibroblasts, however the most prevalent activity was found within hypertrophic chondrocytes and periosteal tissue, respectively. \"\\tEnhancer\\tCcn2\\t--\\t\"PCR,Transgenic mice\"\\t\"Four functional enhancers were identified, with each driving distinct, tissue-specific patterns of transgene expression. An enhancer located -100 kb from the CCN2 transcription start site facilitated expression within vascular tissue. An enhancer -135 kb upstream of CCN2 drove expression within the articular chondrocytes of synovial joints.The other two enhancers, located at -198 kb and -229 kb, mediated transgene expression within dermal fibroblasts, however the most prevalent activity was found within hypertrophic chondrocytes and periosteal tissue, respectively. \"\\t\"Ctgf,Fisp12,Hcs24,fisp-12\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t135938', 'E_02_213\\t29256171\\t--\\tmm10\\tchr10\\t24483642\\t24503469\\tMouse\\tEmbryonic Limb\\tLow+High throughput\\t\"Transgenic mice,ChIP-seq,DNase I Hypersensitivity Assay\"\\t\"Four functional enhancers were identified, with each driving distinct, tissue-specific patterns of transgene expression. An enhancer located -100 kb from the CCN2 transcription start site facilitated expression within vascular tissue. An enhancer -135 kb upstream of CCN2 drove expression within the articular chondrocytes of synovial joints.The other two enhancers, located at -198 kb and -229 kb, mediated transgene expression within dermal fibroblasts, however the most prevalent activity was found within hypertrophic chondrocytes and periosteal tissue, respectively. \"\\tEnhancer\\tCcn2\\t--\\t\"PCR,Transgenic mice\"\\t\"Four functional enhancers were identified, with each driving distinct, tissue-specific patterns of transgene expression. An enhancer located -100 kb from the CCN2 transcription start site facilitated expression within vascular tissue. An enhancer -135 kb upstream of CCN2 drove expression within the articular chondrocytes of synovial joints.The other two enhancers, located at -198 kb and -229 kb, mediated transgene expression within dermal fibroblasts, however the most prevalent activity was found within hypertrophic chondrocytes and periosteal tissue, respectively. \"\\t\"Ctgf,Fisp12,Hcs24,fisp-12\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t101886', 'E_02_214\\t29272704\\t--\\tmm10\\tchr10\\t21410566\\t21412861\\tMouse\\tPrimary Mouse Embryonic Fibroblasts\\tLow+High throughput\\t\"ChIP-seq,ATAC-seq\"\\t\"We mapped cell identity and LRG enhancers from Spret/EiJ MEFs (ATAC-seq, H3K4me1/2 ChIP-seq, ATAC-seq) to identify strain-specific cell identity and LRG enhancers (Figure 2a,b). We focused on cases in which there is no longer an ATAC-seq peak in the strain in which the enhancer sequence is no longer functional (see methods), reasoning that such examples would be most useful for determining which TF binding motifs are required for enhancer selection. In total, we identified 42 LRG enhancers and 363 cell identity enhancers that were selected in a strain-specific manner (Figure 2b).\"\\tEnhancer\\tLama2\\t--\\t\"ChIP-seq,PCR\"\\t\"Representative LRG Enhancers at the locus of the LRG Spred2. Scale bars indicate normalized read densities for each ChIP-seq and shaded boxes denote LRG Enhancers.To determine whether BAF is involved in inducible nucleosome remodeling at LRG enhancers, we performed ChIP-seq for Smarca4, a core component of BAF to determine if the complex is recruited to AP-1-bound LRG enhancers upon stimulation. This revealed that BAF levels are low at most LRG enhancers prior to stimulation but increase significantly upon stimulation (Figure 7a,b). \"\\tLama2\\t--\\t--\\t--\\t--\\t--\\t--\\tJun\\t\"AP-1c,c-jun,Jun\"\\t\"ChIP-seq,ATAC-seq\"\\t\"To identify fibroblast LDTFs that might be required for targeting AP-1 to fibroblast-specific Enhancers, we first performed ChIP-seq for the AP-1 TFs Fos/Jund in SPRET/EiJ MEFs to identify all strain-specific sites of AP-1 binding (n=1,224; Figure 5f). We next excluded all the strain-specific binding sites at which there was a SNP within an AP-1 motif that could directly explain the observed loss of AP-1 binding in one of the two strains (~47% of strain-specific AP-1 bound sites) (Figure 5g). \"\\t--\\t--\\t--\\t>2KB\\t5569570', 'E_02_215\\t29272704\\t--\\tmm10\\tchr11\\t19833945\\t19851027\\tMouse\\tPrimary Mouse Embryonic Fibroblasts\\tLow+High throughput\\t\"ChIP-seq,ATAC-seq\"\\t\"We mapped cell identity and LRG enhancers from Spret/EiJ MEFs (ATAC-seq, H3K4me1/2 ChIP-seq, ATAC-seq) to identify strain-specific cell identity and LRG enhancers (Figure 2a,b). We focused on cases in which there is no longer an ATAC-seq peak in the strain in which the enhancer sequence is no longer functional (see methods), reasoning that such examples would be most useful for determining which TF binding motifs are required for enhancer selection. In total, we identified 42 LRG enhancers and 363 cell identity enhancers that were selected in a strain-specific manner (Figure 2b).\"\\tEnhancer\\tSpred2\\t--\\t\"ChIP-seq,PCR\"\\t\"Representative LRG Enhancers at the locus of the LRG Spred2. Scale bars indicate normalized read densities for each ChIP-seq and shaded boxes denote LRG Enhancers.To determine whether BAF is involved in inducible nucleosome remodeling at LRG enhancers, we performed ChIP-seq for Smarca4, a core component of BAF to determine if the complex is recruited to AP-1-bound LRG enhancers upon stimulation. This revealed that BAF levels are low at most LRG enhancers prior to stimulation but increase significantly upon stimulation (Figure 7a,b). \"\\tC79158\\t--\\t--\\t--\\t--\\t--\\t--\\tJun\\t\"AP-1c,c-jun,Jun\"\\t\"ChIP-seq,ATAC-seq\"\\t\"To identify fibroblast LDTFs that might be required for targeting AP-1 to fibroblast-specific Enhancers, we first performed ChIP-seq for the AP-1 TFs Fos/Jund in SPRET/EiJ MEFs to identify all strain-specific sites of AP-1 binding (n=1,224; Figure 5f). We next excluded all the strain-specific binding sites at which there was a SNP within an AP-1 motif that could directly explain the observed loss of AP-1 binding in one of the two strains (~47% of strain-specific AP-1 bound sites) (Figure 5g). \"\\t--\\t--\\t--\\t>2KB\\t81955', 'E_02_216\\t29272704\\t--\\tmm10\\tchr17\\t45790442\\t45800636\\tMouse\\tPrimary Mouse Embryonic Fibroblasts\\tLow+High throughput\\t\"ChIP-seq,ATAC-seq\"\\t\"We mapped cell identity and LRG enhancers from Spret/EiJ MEFs (ATAC-seq, H3K4me1/2 ChIP-seq, ATAC-seq) to identify strain-specific cell identity and LRG enhancers (Figure 2a,b). We focused on cases in which there is no longer an ATAC-seq peak in the strain in which the enhancer sequence is no longer functional (see methods), reasoning that such examples would be most useful for determining which TF binding motifs are required for enhancer selection. In total, we identified 42 LRG enhancers and 363 cell identity enhancers that were selected in a strain-specific manner (Figure 2b).\"\\tEnhancer\\tVegfa\\t--\\t\"ChIP-seq,PCR\"\\t\"Representative LRG Enhancers at the locus of the LRG Spred2. Scale bars indicate normalized read densities for each ChIP-seq and shaded boxes denote LRG Enhancers.To determine whether BAF is involved in inducible nucleosome remodeling at LRG enhancers, we performed ChIP-seq for Smarca4, a core component of BAF to determine if the complex is recruited to AP-1-bound LRG enhancers upon stimulation. This revealed that BAF levels are low at most LRG enhancers prior to stimulation but increase significantly upon stimulation (Figure 7a,b). \"\\t\"Vegf,Vpf\"\\t--\\t--\\t--\\t--\\t--\\t--\\tJun\\t\"AP-1c,c-jun,Jun\"\\t\"ChIP-seq,ATAC-seq\"\\t\"To identify fibroblast LDTFs that might be required for targeting AP-1 to fibroblast-specific Enhancers, we first performed ChIP-seq for the AP-1 TFs Fos/Jund in SPRET/EiJ MEFs to identify all strain-specific sites of AP-1 binding (n=1,224; Figure 5f). We next excluded all the strain-specific binding sites at which there was a SNP within an AP-1 motif that could directly explain the observed loss of AP-1 binding in one of the two strains (~47% of strain-specific AP-1 bound sites) (Figure 5g). \"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t221453', 'E_02_217\\t30545847\\t--\\tmm10\\tchr10\\t50623698\\t50625698\\tMouse\\tHypothalamus\\tLow+High throughput\\t\"ChIP-seq,ATAC-seq\"\\t\"CRISPRa targeting of the Sim1 promoter or its hypothalamusspecific enhancer, which is 270 kilobases away from the gene, in Sim1 haploinsufficient mice increased the expression of the normal copy of Sim1. \"\\tEnhancer\\tSim1\\t--\\t\"ChIP-seq,PCR,Transgenic mice\"\\t\"We also carried out ChIP-seq using an antibody against S. pyogenes Cas9 in both CRISPRa-promoter and CRISPRa-Enhancer transfected cells and found on-target binding for the promoter and Enhancer,respectively.\"\\t\"bHLHe14, mSIM1\"\\tHaploinsufficiency\\t--\\tD057895\\t\"Transgenic-based CRISPRa targeting of the Sim1 promoter or its distant hypothalamic enhancer up-regulated its expression from the endogenous functional allele in a tissue-specific manner, rescuing the obesity phenotype in Sim1 heterozygous mice.\"\\t\"CRISPR/Cas9,ChIP-seq\"\\t\"Transgenic-based CRISPRa targeting of the Sim1 promoter or its distant hypothalamic enhancer up-regulated its expression from the endogenous functional allele in a tissue-specific manner, rescuing the obesity phenotype in Sim1 heterozygous mice.\"\\tSim1\\t\"bHLHe14,mSIM1\"\\t\"CRISPR/Cas9,qPCR\"\\t\"CRISPRa targeting of the Sim1 promoter or its hypothalamusspecific enhancer, which is 270 kilobases away from the gene, in Sim1 haploinsufficient mice increased the expression of the normal copy of Sim1. This up-regulation was sufficient to rescue the obesity phenotype of Sim1 heterozygous mice and led to significantly reduced food intake and body fat content in thesemice.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t269976', 'E_02_218\\t29273679\\t--\\tmm10\\tchr2\\t75120051\\t75165771\\tMouse\\tMurine Limb Bud Cells\\tLow+High throughput\\t\"Hi-C,4C,ChIP-seq\"\\t\"In order to gain insights into TAD organization around the HoxD locus during limb bud development, we performed Hi-C (chromosome capture followed by high-throughput sequencing) on microdissected distal and proximal limb bud cells isolated from embryonic day 12.5 (E12.5) embryos. At this stage, T-DOM enhancers regulate Hoxd gene expression in proximal cells and are silent in distal cells, whereas C-DOM enhancers control Hoxd gene targets in future digit cells and are silent in proximal cells. Therefore, the two TADs are either transcriptionally active or inactive in an exclusive manner in the two tissue samples (Fig. 1A,B, top schemes).\"\\tEnhancer\\tHoxd\\t4C\\t\"Hi-C,qPCR\"\\t\"Consequently,under wild-type conditions, Hoxd4 is expressed only in proximal limb cells under the control of the T-DOM,while C-DOM Enhancers control Evx2 transcripts in dis_x0002_tal cells exclusively.\"\\t\"Hox-4,Hox-5,Ul,ulnaless\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t489923', 'E_02_219\\t29273679\\t--\\tmm10\\tchr2\\t75413472\\t75451553\\tMouse\\tMurine Limb Bud Cells\\tLow+High throughput\\t\"Hi-C,4C,ChIP-seq\"\\t\"In order to gain insights into TAD organization around the HoxD locus during limb bud development, we performed Hi-C (chromosome capture followed by high-throughput sequencing) on microdissected distal and proximal limb bud cells isolated from embryonic day 12.5 (E12.5) embryos. At this stage, T-DOM enhancers regulate Hoxd gene expression in proximal cells and are silent in distal cells, whereas C-DOM enhancers control Hoxd gene targets in future digit cells and are silent in proximal cells. Therefore, the two TADs are either transcriptionally active or inactive in an exclusive manner in the two tissue samples (Fig. 1A,B, top schemes).\"\\tEnhancer\\tHoxd\\t4C\\t\"Hi-C,qPCR\"\\t\"Consequently,under wild-type conditions, Hoxd4 is expressed only in proximal limb cells under the control of the T-DOM,while C-DOM Enhancers control Evx2 transcripts in dis_x0002_tal cells exclusively.\"\\t\"Hox-4,Hox-5,Ul,ulnaless\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t779525', 'E_02_220\\t29277702\\t--\\tmm10\\tchr2\\t126601468\\t126603468\\tMouse\\tConnective Tissue Mast Cell\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"Chromatin immunoprecipitation sequencing analysis identified putative Hdc enhancers.Luciferase reporter transcription assay confirmed enhancer activities of putative enhancers in the Hdc gene.MITF bound to an enhancer located 8.8 kb upstream of the transcription start site of the Hdc gene and directed enhancer activity.To determine whether the putative Hdc Enhancers function in Hdc gene transcription, we constructed a luciferase reporter gene in which luciferase expression is under the control of the Hdc minimal promoter (-44 to +1 relative to the TSS) and the -8.8 or +0.3 Hdc Enhancer. We found that the -8.8 Hdc Enhancer dramatically increased the activity of the Hdc minimal promoter (13.4-fold increase) , whereas the +0.3 Hdc Enhancer failed to show significant Enhancer activity.\"\\tEnhancer\\tHdc\\t--\\t\"ChIP-qPCR,Luciferase Reporter Assay\"\\tChromatin immunoprecipitation sequencing analysis identified putative Hdc enhancers.Luciferase reporter transcription assay confirmed enhancer activities of putative enhancers in the Hdc gene.MITF bound to an enhancer located 8.8 kb upstream of the transcription start site of the Hdc gene and directed enhancer activity.\\t\"AW108189-a,?Hdc-c,Hdc-e,?Hdc-s,?Hdc\"\\tIgE/mast cell-mediated anaphylaxis\\t--\\t--\\t--\\t--\\t--\\t\"Gata2,Mitf\"\\t\"Gata-2,BCC2,Bhlhe32,Gsfbcc2,Vitiligo,Wh,bw,mi,vit\"\\t\"ChIP-qPCR,shRNA,Luciferase Reporter Assay\"\\t\"Our ChIP-qPCR analysis of MITF binding in BMMCs revealed that among five predicted MITF-binding sites located within the -8.8 and +0.3 Hdc Enhancers.We found that Hdc mRNA expression was significantly diminished (70% reduction) in TBMMCs with knocked down Mitf mRNA expression.Conversely, overexpression of MITF in a non-mast cell line greatly 383 increased the activity of the -8.8 Hdc Enhancer.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t8810', 'E_02_221\\t29277702\\t--\\tmm10\\tchr2\\t126593905\\t126595430\\tMouse\\tConnective Tissue Mast Cell\\tLow+High throughput\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"Chromatin immunoprecipitation sequencing analysis identified putative Hdc enhancers.Luciferase reporter transcription assay confirmed enhancer activities of putative enhancers in the Hdc gene.MITF bound to an enhancer located 8.8 kb upstream of the transcription start site of the Hdc gene and directed enhancer activity.To determine whether the putative Hdc Enhancers function in Hdc gene transcription, we constructed a luciferase reporter gene in which luciferase expression is under the control of the Hdc minimal promoter (-44 to +1 relative to the TSS) and the -8.8 or +0.3 Hdc Enhancer. We found that the -8.8 Hdc Enhancer dramatically increased the activity of the Hdc minimal promoter (13.4-fold increase) , whereas the +0.3 Hdc Enhancer failed to show significant Enhancer activity.\"\\tEnhancer\\tHdc\\t--\\t\"ChIP-qPCR,Luciferase Reporter Assay\"\\tChromatin immunoprecipitation sequencing analysis identified putative Hdc enhancers.Luciferase reporter transcription assay confirmed enhancer activities of putative enhancers in the Hdc gene.MITF bound to an enhancer located 8.8 kb upstream of the transcription start site of the Hdc gene and directed enhancer activity.\\t\"AW108189-a,?Hdc-c,Hdc-e,?Hdc-s,?Hdc\"\\tIgE/mast cell-mediated anaphylaxis\\t--\\t--\\t--\\t--\\t--\\t\"Gata2,Mitf\"\\t\"Gata-2,BCC2,Bhlhe32,Gsfbcc2,Vitiligo,Wh,bw,mi,vit\"\\t\"ChIP-qPCR,shRNA,Luciferase Reporter Assay\"\\t\"Our ChIP-qPCR analysis of MITF binding in BMMCs revealed that among five predicted MITF-binding sites located within the -8.8 and +0.3 Hdc Enhancers.We found that Hdc mRNA expression was significantly diminished (70% reduction) in TBMMCs with knocked down Mitf mRNA expression.Conversely, overexpression of MITF in a non-mast cell line greatly 383 increased the activity of the -8.8 Hdc Enhancer.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1010', 'E_02_222\\t29284139\\t--\\tmm10\\tchr7\\t143459172\\t143459787\\tMouse\\tMouse Embryonic Fibroblasts\\tLow throughput\\tLuciferase Reporter Assay\\t\"As shown in Fig 5A, enh1 is in the second intron of Cdkn1c while enh2 is located in the 3 UTR. Luciferase reporter constructs were transfected into iMEFs stably overexpressing FOXD1 or a negative control vector.\"\\tEnhancer\\tCdkn1c\\tCRISPR/Cas9\\t\"Luciferase Reporter Assay,RT-PCR\"\\t\"We tested whether FOXD1 directly binds these enhancer sequences using established luciferase reporter constructs. As shown in Fig. 5A, enh1 is in the second intron of Cdkn1c while enh2 is located in the 3 UTR. Luciferase reporter constructs were transfected into iMEFs stably overexpressing FOXD1 or a negative control vector. FOXD1-expressing cells exhibited higher activity of enh1-driven luciferase than negative controls while no difference was observed in luciferase activity driven by enh2(Fig. 5B).\"\\t\"AL024410,CDKI,Kip2,p57(kip2),p57Kip2\"\\tMalignancies\\t--\\t--\\t--\\t--\\t--\\tFoxd1\\t\"AI385632,BF-2,FREAC4,Hfh10,Hfhbf2\"\\tLuciferase Reporter Assa\"Firefly luciferase reporter assay driven by Cdkn1c enhancer elements, enh1 and enh2, in iMEFs overexpressing empty pLenti construct or FOXD1.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1142', 'E_02_223\\t29284139\\t--\\tmm10\\tchr7\\t143462624\\t143463585\\tMouse\\tMouse Embryonic Fibroblasts\\tLow throughput\\tLuciferase Reporter Assay\\t\"As shown in Fig 5A, enh1 is in the second intron of Cdkn1c while enh2 is located in the 3 UTR. Luciferase reporter constructs were transfected into iMEFs stably overexpressing FOXD1 or a negative control vector.\"\\tEnhancer\\tCdkn1c\\tCRISPR/Cas9\\t\"Luciferase Reporter Assay,RT-PCR\"\\t\"We tested whether FOXD1 directly binds these enhancer sequences using established luciferase reporter constructs. As shown in Fig. 5A, enh1 is in the second intron of Cdkn1c while enh2 is located in the 3 UTR. Luciferase reporter constructs were transfected into iMEFs stably overexpressing FOXD1 or a negative control vector. FOXD1-expressing cells exhibited higher activity of enh1-driven luciferase than negative controls while no difference was observed in luciferase activity driven by enh2(Fig. 5B).\"\\t\"AL024410,CDKI,Kip2,p57(kip2),p57Kip2\"\\tMalignancies\\t--\\t--\\t--\\t--\\t--\\tGli1\\t\"AV235269,Zfp-5,Zfp5\"\\t\"RT-PCR,CRISPR/Cas9\"\\t\"SHH-regulation of Ccnd1 and Cdkn1c also appeared dependent upon the GLI transcription factors, as SHH ligand addition in Gli2 iMEFs did not change their expression.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t4767', 'E_02_224\\t29342133\\t--\\tmm10\\tchr15\\t63714919\\t63952099\\tMouse\\tBone Marrow\\tLow+High throughput\\tChIP-seqHere we show that an evolutionarily conserved region located 1.7 megabases downstream of the Myc gene that has previously been labelled as a super_x0002_enhancer2 is essential for the regulation of Myc expression levels in both normal haematopoietic and leukaemic stem cell hierarchies in mice and humans. Deletion of this region in mice leads to a complete loss of Myc expression in haematopoietic stem cells and progenitors.\\tSuper-Enhancer\\tMyc\\t--\\t\"ChIP-seq,ATAC-seq,RT-qPCR\"\\t\"In line with this hypothesis, chromatin conformation capture performedon mouse leukaemic cells2 as well as DNA proximity analysis using fluorescence in situ hybridization (DNA FISH) in LSK cells showed that the Myc promoter and BENC are in close physical proximity to each other, suggesting that cis-regulation is mediated by chromosomal looping in haematopoietic stem and progenitor cells.\"\\t\"AU0167572,Niard,Nird,bHLHe39,?Myc\"\\tLeukaemia\\t--\\tD007938\\t\"Clusters of enhancers, such as BENC, form highly combinatorial systems that allow precise control of gene expression across normal cellular hierarchies and which also can be hijacked in malignancies.\"\"ATAC-seq,RT-qPCR\"\\t\"Analysis of mice carrying deletions of individual Enhancer modules suggests that specific Myc expression levels throughout most of the haematopoietic hierarchy are controlled by the combinatorial and additive activity of individual Enhancer modules, which collectively function as a blood Enhancer cluster(BENC).\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t1848169', 'E_02_225\\t29358217\\tMdE Enhancer\\tmm10\\tchr1\\t133347574\\t133364574\\tMouse\\t\"As4.1,Tsukuba hypertensive mice\"\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\t\"Because the 17-kb sequence exhibited Enhancer activity in vivo and in vitro,we narrowed down the Enhancer to a 2.3-kb core using Luciferase assays in As4.1 cells. When this 2.3-kb sequence was removed from the endogenous mRen gene in the mouse,its basal expression was dramatically reduced,and the hypertension esponsiveness was significantly attenuated.\"\\tEnhancer\\tRen1\\tCRISPR/Cas9\\t\"qRT-PCR,ChIP\"\\t\"While the mRen gene bearing the 3\\' deletion was appropriately downregulated, the one bearing the 5\\'deletion lost this hypertension responsiveness. Because the 17-kb sequence exhib_x0002_ited enhancer activity in vivo and in vitro, we narrowed down the enhancer to a 2.3-kb core using luciferase assays in As4.1 cells. When this 2.3-kb sequence was re_x0002_moved from the endogenous mRen gene in the mouse, its basal expression was dra_x0002_matically reduced, and the hypertension responsiveness was significantly attenuated.\"\\t\"D19352,Ren,Ren-1,Ren-Ac,Ren1d,Rn-1,Rnr,?Ren1\"\\tHypertension\\tDOID:10763\\tD006973\\tThis finding sheds light on the unexplored distal regulatory region of the mRen gene and provides in vivo evidence that a novel 5\\' enhancer plays an essential role in the physiological response to an AII-induced hypertensive environment.\\tqRT-PCR\\t\"We propose that in a hypertensive environment, the activity of this novel enhancer is attenuated, and, as a consequence, mRen gene transcription is suppressed to maintain blood pressure.\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5401', 'E_02_226\\t29514092\\t--\\tmm10\\tchr19\\t61151074\\t61161722\\tMouse\\t\"Monocytes,Dendritic Cells\"\\tLow+High throughput\\tChIP-seq\\t\"Active enhancers shared by monocytes and DCs (but not neutrophils) and those specific', 'E_02_227\\t29514092\\t--\\tmm10\\tchr19\\t61158643\\t61164569\\tMouse\\t\"Monocytes,Dendritic Cells\"\\tLow+High throughput\\tChIP-seq\\t\"Active enhancers shared by monocytes and DCs (but not neutrophils) and those specific', 'E_02_228\\t29514092\\t--\\tmm10\\tchr19\\t61165425\\t61172555\\tMouse\\t\"Monocytes,Dendritic Cells\"\\tLow+High throughput\\tChIP-seq\\t\"Active enhancers shared by monocytes and DCs (but not neutrophils) and those specific', 'E_02_229\\t29514092\\t--\\tmm10\\tchr2\\t144000110\\t144014615\\tMouse\\t\"Monocytes,Dendritic Cells\"\\tLow+High throughput\\tChIP-seq\\t\"Active enhancers shared by monocytes and DCs (but not neutrophils) and those specific', 'E_02_230\\t29514092\\t--\\tmm10\\tchr2\\t144014506\\t144023297\\tMouse\\t\"Monocytes,Dendritic Cells\"\\tLow+High throughput\\tChIP-seq\\t\"Active enhancers shared by monocytes and DCs (but not neutrophils) and those specific', 'E_02_231\\t29514092\\t--\\tmm10\\tchr2\\t103464661\\t103471206\\tMouse\\t\"Monocytes,Dendritic Cells\"\\tLow+High throughput\\tChIP-seq\\t\"Active enhancers shared by monocytes and DCs (but not neutrophils) and those specific', 'E_02_232\\t29527594\\t--\\tmm10\\tchr6\\t127033997\\t127036097\\tMouse\\tOsteocyte Cell Lines\\tLow+High throughput\\t\"ChIP-seq,CRISPR/Cas9\"\\t\"We therefore utilized chromatin immunoprecipitation followed by DNA sequencing (ChIP-seq) data from an osteocyte cell line to identify potential regulatory regions of the Fgf23 gene. Based on ChIP-seq analysis of enhancer_x0002_associated histone modifications, including H3K4 methylation and H3K9 acetylation, we discovered several potential enhancers for', 'E_02_233\\t29903884\\t--\\tmm10\\tchr11\\t112217210\\t112217767\\tMouse\\tTestis\\tLow+High throughput\\t\"DNaseI-seq,ChIP-seq\"\\t\"In vivo high-throughput chromatin accessibility techniques, transgenic assays, and genome editing revealed several novel gonadal regulatory elements in the 2 Mb gene desert upstream of Sox9.Although others are redundant, Enh13, a 557 bp element located 565 kb 5, is essential to initiate mouse testis development.\"\\tEnhancer\\tSox9\\tCRISPR/Cas9\\t\"ChIP-seq,RT-PCR\"\\t\"There was a 30 to 50% decrease in Sox9 mRNA levels in E11.5 XX Enh13-/- gonads compared to the wild type, as reflected by reduced immunofluorescence for SOX9 protein (fig. S11). These data indicate that Enh13 also plays a role in the very early expression of Sox9 in the XX gonad.\"\\t\"2010306G03Rik,AV220920,mKIAA4243\"\\t--\\t--\\t--\\t\"Enh13 is conserved and embedded within a 32.5-kilobase region whose deletion in humans is associated with XY sex reversal, suggesting that it is also critical in humans.\"\\t\"CRISPR/Cas9,RT-qPCR\"\\tThis again supports the conclusion that Enh13 plays a more substantial role than TES during early gonadal development.\\tSry\\t\"Tdf,Tdy\"\\tChIP-qPCR\\t\"We performed ChIP-qPCR on E11.5 gonads dissected from Sry-Myc transgenic embryos by using a specific antibody against the MYC tag.SRY-MYCCpositive gonads had an 11-fold enrichment versus SRY-MYCCnegative gonads with primers spanning the SOX9 consensus site and a sixfold enrichment with primers spanning the SRY site, whereas primers against the strongest SRY binding site in TESCO showed fivefold enrichment.This reveals the strong binding of SRY to Enh13 at E11.5, with a preference for the SOX9 consensus site, possibly due to the adjacent SF1 binding site.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t564720', 'E_02_234\\t29967281\\t--\\tmm10\\tchr17\\t85527960\\t85530660\\tMouse\\tThe Subventricular Zone Of Lateral Ganglionic Eminence??LGE SVZ??\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"We found an SP9 ChIP-Seq peak at the Six3 promoter region (?5 to +411 of the TSS) (Fig. 9C), and several additional peaks upstream of the Six3 gene (regions 1, 2 and 3) (Fig. 9C);Thus, these noncoding domains are strong candidates for the Six3 promoter and three Six3 enhancers.\"\\tEnhancer\\tSix3\\t--\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"Together,the results of ChIP-seq and Dual-Luciferase assay provide evidence that SP9 promotes Six3 expression in the LGE SVZ through its direct binding to the promoter and possibly through binding to regulatory elements within the Six3 locus.\"\\t\"E130112M24Rika,Six3alpha,Six3b,Six3beta,?Six3\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSix3\\tE130112M24RikaSix3alphaSix3bSix3beta\\t\"RNA-seq,ChIP-seq\"\\t\"SP8 and SP9 together drive expression of the transcription factor Six3 in a spatially restricted domain of the LGE subventricular zone. Conditional deletion of Six3 also prevents the formation of most D2 MSNs, phenocopying the Sp8/9 mutants.This implies that there are additional mechanisms that regulate the generation of D2 MSNs in Sp9-KO mice, one of which was suggested by the LGE RNA-Seq analysis. We identified that expression of the TF Six3 was decreased ?5-fold in E16.5 Sp8/9-DCKO mice compared with control mice.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t84297', 'E_02_235\\t29967281\\t--\\tmm10\\tchr17\\t85463916\\t85466386\\tMouse\\tThe Subventricular Zone Of Lateral Ganglionic Eminence??LGE SVZ??\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"We found an SP9 ChIP-Seq peak at the Six3 promoter region (?5 to +411 of the TSS) (Fig. 9C), and several additional peaks upstream of the Six3 gene (regions 1, 2 and 3) (Fig. 9C);Thus, these noncoding domains are strong candidates for the Six3 promoter and three Six3 enhancers.\"\\tEnhancer\\tSix3\\t--\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"Together,the results of ChIP-seq and Dual-Luciferase assay provide evidence that SP9 promotes Six3 expression in the LGE SVZ through its direct binding to the promoter and possibly through binding to regulatory elements within the Six3 locus.\"\\t\"E130112M24Rika,Six3alpha,Six3b,Six3beta,?Six3\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSix3\\tE130112M24RikaSix3alphaSix3bSix3beta\\t\"RNA-seq,ChIP-seq\"\\t\"SP8 and SP9 together drive expression of the transcription factor Six3 in a spatially restricted domain of the LGE subventricular zone. Conditional deletion of Six3 also prevents the formation of most D2 MSNs, phenocopying the Sp8/9 mutants.This implies that there are additional mechanisms that regulate the generation of D2 MSNs in Sp9-KO mice, one of which was suggested by the LGE RNA-Seq analysis. We identified that expression of the TF Six3 was decreased ?5-fold in E16.5 Sp8/9-DCKO mice compared with control mice.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t148456', 'E_02_236\\t29967281\\t--\\tmm10\\tchr17\\t85392908\\t85395234\\tMouse\\tThe Subventricular Zone Of Lateral Ganglionic Eminence??LGE SVZ??\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"We found an SP9 ChIP-Seq peak at the Six3 promoter region (?5 to +411 of the TSS) (Fig. 9C), and several additional peaks upstream of the Six3 gene (regions 1, 2 and 3) (Fig. 9C);Thus, these noncoding domains are strong candidates for the Six3 promoter and three Six3 enhancers.\"\\tEnhancer\\tSix3\\t--\\t\"ChIP-seq,Luciferase Reporter Assay\"\\t\"Together,the results of ChIP-seq and Dual-Luciferase assay provide evidence that SP9 promotes Six3 expression in the LGE SVZ through its direct binding to the promoter and possibly through binding to regulatory elements within the Six3 locus.\"\\t\"E130112M24Rika,Six3alpha,Six3b,Six3beta,?Six3\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSix3\\tE130112M24RikaSix3alphaSix3bSix3beta\\t\"RNA-seq,ChIP-seq\"\\t\"SP8 and SP9 together drive expression of the transcription factor Six3 in a spatially restricted domain of the LGE subventricular zone. Conditional deletion of Six3 also prevents the formation of most D2 MSNs, phenocopying the Sp8/9 mutants.This implies that there are additional mechanisms that regulate the generation of D2 MSNs in Sp9-KO mice, one of which was suggested by the LGE RNA-Seq analysis. We identified that expression of the TF Six3 was decreased ?5-fold in E16.5 Sp8/9-DCKO mice compared with control mice.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t219536', 'E_02_237\\t29979962\\tDRR Enhancer \\tmm10\\tchr7\\t46371356\\t46373481\\tMouse\\tC2C12\\tLow throughput\\tChIRP\\t\"To directly evaluate whether the DRReRNA 2Kb transcript could undergo splicing, we', 'E_02_238\\t30008200\\t--\\tmm10\\tchr13\\t56243248\\t56243833\\tMouse\\tP19\\tLow throughput\\tChIP\\t\"In control cells, the enrichment of H3K27Ac, an active Enhancer marker, was significantly increased at the Neurog1 promoter, as well as other Enhancers, upon RA treatment.\"\\tEnhancer\\tNeurog1\\t--\\tChIP-qPCR\\t\"Genome-wide CTCF occupancy patterns, which have been mapped across more than 100 mammalian cell types, re_x0002_vealed prominent CTCF binding to the -14.4 kb and -7.6 kb transcription start sites of Neurog1.We performed ChIP-qPCR analysis to validate the in vivo binding of CTCF in', 'E_02_239\\t30008200\\t--\\tmm10\\tchr13\\t56244114\\t56244357\\tMouse\\tP19\\tLow throughput\\tChIP\\t\"In control cells, the enrichment of H3K27Ac, an active Enhancer marker, was significantly increased at the Neurog1 promoter, as well as other Enhancers, upon RA treatment.\"\\tEnhancer\\tNeurog1\\t--\\tChIP-qPCR\\t\"Genome-wide CTCF occupancy patterns, which have been mapped across more than 100 mammalian cell types, re_x0002_vealed prominent CTCF binding to the -14.4 kb and -7.6 kb transcription start sites of Neurog1.We performed ChIP-qPCR analysis to validate the in vivo binding of CTCF in', 'E_02_240\\t30008200\\t--\\tmm10\\tchr13\\t56245174\\t56245857\\tMouse\\tP19\\tLow throughput\\tChIP\\t\"In control cells, the enrichment of H3K27Ac, an active Enhancer marker, was significantly increased at the Neurog1 promoter, as well as other Enhancers, upon RA treatment.\"\\tEnhancer\\tNeurog1\\t--\\tChIP-qPCR\\t\"Genome-wide CTCF occupancy patterns, which have been mapped across more than 100 mammalian cell types, re_x0002_vealed prominent CTCF binding to the -14.4 kb and -7.6 kb transcription start sites of Neurog1.We performed ChIP-qPCR analysis to validate the in vivo binding of CTCF in', 'E_02_241\\t30017589\\t--\\tmm10\\tchr13\\t30108722\\t30109111\\tMouse\\t\"Embryonic Stem Cell,Epiblast-like Cells (EpiLCs)\"\\tLow+High throughput\\tChIP-seq\\t\"Our findings were corroborated by H3K4me1 and H3K27ac ChIP-seq data from an alternative na?ve-to-formative differentiation system. As a control, we found little change in these marks at SMC1 sites common to both states. These results indicate that GRHL2 binding correlates with full enhancer activation, suggesting a role for GRHL2 not only in regulation of cohesin binding, but also in regulation of other crucial steps in enhancer activation.Together, these results suggest that our SMC1 ChIP-seq data reflect changing enhancer activity and that cohesin binding at enhancers is highly dynamic during the ESC to EpiLC transition.\"\\tEnhancer\\tDsp\\tCRISPR/Cas9\\t\"ChIP-seq,qRT-PCR\"\\t\"We then assessed the effects of each Enhancer deletion on Dsp expression in ES cells and EpiLCs by qRT-PCR.Indeed, we found that deletion of one or both KLF4-bound enhancers led to a significant reduction in Dsp expression in ESCs but had no effect in EpiLCs(Figure 6B). Conversely, deletion of the GRHL2-bound enhancer resulted in significantly reduced Dsp expression in the EpiLC state but had no effect in ESCs (Figure 6B). We observed similar state-specific reductions in target gene expression with deletions of the KLF4-bound and GRHL2-bound enhancers at the Cdh1 and Jam1 loci (Figure 6C and D, S6A and B).\"\\t\"2300002E22Rik,5730453H04Rik,AA407887,AA407888,AW109828,DP,rul\"\\t--\\t--\\t--\\tGRHL2 therefore control over a subset of the naive network via Enhancer switching to maintain expression of epithelial genes upon exit from naive pluripotency\\tChIP-seq\\t\"Together,these results suggest that our SMC1 ChIP-seq data reflect changing Enhancer activity and that cohesin binding at Enhancers is highly dynamic during the ESC to EpiLC transition.\"\\tGrhl2\\t\"0610015A08Rik,BOM,Tcfcp2l3,clft3\"\\t\"ChIP-seq,ATAC-seq\"\\t\"Given the strong association between GRHL2 binding and various events in enhancer activation, we asked whether GRHL2 binding is necessary for each of these events. To address this question, we assessed levels of each active enhancer mark in wildtype and GHRL2 KO EpiLCs by performing ATAC-seq and ChIP-seq for H3K4me1, H3K27ac, and SMC1. There was a strong reduction in all active enhancer marks (chromatin accessibility, H3K4me1, H3K27ac, and SMC1 levels) at GRHL2 sites in GRHL2 KO EpiLCs, suggesting a near complete block in enhancer activation in the absence of GRHL2. Together, these results indicate an absolute requirement for GRHL2 for enhancer activation specifically at its target sites.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t8042376', 'E_02_242\\t30017589\\t--\\tmm10\\tchr13\\t30123333\\t30124861\\tMouse\\t\"Embryonic Stem Cell,Epiblast-like Cells (EpiLCs)\"\\tLow+High throughput\\tChIP-seq\\t\"Our findings were corroborated by H3K4me1 and H3K27ac ChIP-seq data from an alternative na?ve-to-formative differentiation system. As a control, we found little change in these marks at SMC1 sites common to both states. These results indicate that GRHL2 binding correlates with full enhancer activation, suggesting a role for GRHL2 not only in regulation of cohesin binding, but also in regulation of other crucial steps in enhancer activation.Together, these results suggest that our SMC1 ChIP-seq data reflect changing enhancer activity and that cohesin binding at enhancers is highly dynamic during the ESC to EpiLC transition.\"\\tEnhancer\\tDsp\\tCRISPR/Cas9\\t\"ChIP-seq,qRT-PCR\"\\t\"We then assessed the effects of each Enhancer deletion on Dsp expression in ES cells and EpiLCs by qRT-PCR.Indeed, we found that deletion of one or both KLF4-bound enhancers led to a significant reduction in Dsp expression in ESCs but had no effect in EpiLCs(Figure 6B). Conversely, deletion of the GRHL2-bound enhancer resulted in significantly reduced Dsp expression in the EpiLC state but had no effect in ESCs (Figure 6B). We observed similar state-specific reductions in target gene expression with deletions of the KLF4-bound and GRHL2-bound enhancers at the Cdh1 and Jam1 loci (Figure 6C and D, S6A and B).\"\\t\"2300002E22Rik,5730453H04Rik,AA407887,AA407888,AW109828,DP,rul\"\\t--\\t--\\t--\\tGRHL2 therefore control over a subset of the naive network via Enhancer switching to maintain expression of epithelial genes upon exit from naive pluripotency\\tChIP-seq\\t\"Together,these results suggest that our SMC0 ChIP-seq data reflect changing Enhancer activity and that cohesin binding at Enhancers is highly dynamic during the ESC to EpiLC transition.\"\\tGrhl2\\t\"0610015A08Rik,BOM,Tcfcp2l3,clft3\"\\t\"ChIP-seq,ATAC-seq\"\\t\"Given the strong association between GRHL2 binding and various events in enhancer activation, we asked whether GRHL2 binding is necessary for each of these events. To address this question, we assessed levels of each active enhancer mark in wildtype and GHRL2 KO EpiLCs by performing ATAC-seq and ChIP-seq for H3K4me1, H3K27ac, and SMC1. There was a strong reduction in all active enhancer marks (chromatin accessibility, H3K4me1, H3K27ac, and SMC1 levels) at GRHL2 sites in GRHL2 KO EpiLCs, suggesting a near complete block in enhancer activation in the absence of GRHL2. Together, these results indicate an absolute requirement for GRHL2 for enhancer activation specifically at its target sites.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t8027196', 'E_02_243\\t30017589\\t--\\tmm10\\tchr13\\t30144166\\t30145833\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\tChIP-seq\\t\"Our findings were corroborated by H3K4me1 and H3K27ac ChIP-seq data from an alternative na?ve-to-formative differentiation system. As a control, we found little change in these marks at SMC1 sites common to both states. These results indicate that GRHL2 binding correlates with full enhancer activation, suggesting a role for GRHL2 not only in regulation of cohesin binding, but also in regulation of other crucial steps in enhancer activation.Together, these results suggest that our SMC1 ChIP-seq data reflect changing enhancer activity and that cohesin binding at enhancers is highly dynamic during the ESC to EpiLC transition.\"\\tEnhancer\\tDsp\\tCRISPR/Cas9\\t\"ChIP-seq,qRT-PCR\"\\t\"We then assessed the effects of each Enhancer deletion on Dsp expression in ES cells and EpiLCs by qRT-PCR.Indeed, we found that deletion of one or both KLF4-bound enhancers led to a significant reduction in Dsp expression in ESCs but had no effect in EpiLCs(Figure 6B). Conversely, deletion of the GRHL2-bound enhancer resulted in significantly reduced Dsp expression in the EpiLC state but had no effect in ESCs (Figure 6B). We observed similar state-specific reductions in target gene expression with deletions of the KLF4-bound and GRHL2-bound enhancers at the Cdh1 and Jam1 loci (Figure 6C and D, S6A and B).\"\\t\"2300002E22Rik,5730453H04Rik,AA407887,AA407888,AW109828,DP,rul\"\\t--\\t--\\t--\\tGRHL2 therefore control over a subset of the naive network via Enhancer switching to maintain expression of epithelial genes upon exit from naive pluripotency\\tChIP-seq\\tDeletion of the GRHL2-bound Enhancer resulted in significantly reduced Dsp expression in the EpiLC state but had no effect in ESCs.\\tGrhl2\\t\"0610015A08Rik,BOM,Tcfcp2l3,clft3\"\\t\"ChIP-seq,ATAC-seq\"\\t\"Given the strong association between GRHL2 binding and various events in enhancer activation, we asked whether GRHL2 binding is necessary for each of these events. To address this question, we assessed levels of each active enhancer mark in wildtype and GHRL2 KO EpiLCs by performing ATAC-seq and ChIP-seq for H3K4me1, H3K27ac, and SMC1. There was a strong reduction in all active enhancer marks (chromatin accessibility, H3K4me1, H3K27ac, and SMC1 levels) at GRHL2 sites in GRHL2 KO EpiLCs, suggesting a near complete block in enhancer activation in the absence of GRHL2. Together, these results indicate an absolute requirement for GRHL2 for enhancer activation specifically at its target sites.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t8006293', 'E_02_244\\t30017589\\t--\\tmm10\\tchr8\\t106608533\\t106609800\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\tChIP-seq\\t\"Our findings were corroborated by H3K4me1 and H3K27ac ChIP-seq data from an alternative na?ve-to-formative differentiation system. As a control, we found little change in these marks at SMC1 sites common to both states. These results indicate that GRHL2 binding correlates with full enhancer activation, suggesting a role for GRHL2 not only in regulation of cohesin binding, but also in regulation of other crucial steps in enhancer activation.Together, these results suggest that our SMC1 ChIP-seq data reflect changing enhancer activity and that cohesin binding at enhancers is highly dynamic during the ESC to EpiLC transition.\"\\tEnhancer\\tCdh1\\tCRISPR/Cas9\\t\"ChIP-seq,qRT-PCR\"\\t\"We then assessed the effects of each Enhancer deletion on Dsp expression in ES cells and EpiLCs by qRT-PCR.Indeed, we found that deletion of one or both KLF4-bound enhancers led to a significant reduction in Dsp expression in ESCs but had no effect in EpiLCs(Figure 6B). Conversely, deletion of the GRHL2-bound enhancer resulted in significantly reduced Dsp expression in the EpiLC state but had no effect in ESCs (Figure 6B). We observed similar state-specific reductions in target gene expression with deletions of the KLF4-bound and GRHL2-bound enhancers at the Cdh1 and Jam1 loci (Figure 6C and D, S6A and B).\"\\t\"AA960649,ARC-1,E-cad,Ecad,L-CAM,UVO,Um\"\\t--\\t--\\t--\\tGRHL2 therefore control over a subset of the naive network via Enhancer switching to maintain expression of epithelial genes upon exit from naive pluripotency\\tChIP-seq\\t\"Together,these results suggest that our SMC1 ChIP-seq data reflect changing Enhancer activity and that cohesin binding at Enhancers is highly dynamic during the ESC to EpiLC transition.\"\\tGrhl2\\t\"0610015A08Rik,BOM,Tcfcp2l3,clft3\"\\t\"ChIP-seq,ATAC-seq\"\\t\"Given the strong association between GRHL2 binding and various events in enhancer activation, we asked whether GRHL2 binding is necessary for each of these events. To address this question, we assessed levels of each active enhancer mark in wildtype and GHRL2 KO EpiLCs by performing ATAC-seq and ChIP-seq for H3K4me1, H3K27ac, and SMC1. There was a strong reduction in all active enhancer marks (chromatin accessibility, H3K4me1, H3K27ac, and SMC1 levels) at GRHL2 sites in GRHL2 KO EpiLCs, suggesting a near complete block in enhancer activation in the absence of GRHL2. Together, these results indicate an absolute requirement for GRHL2 for enhancer activation specifically at its target sites.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5818', 'E_02_245\\t30017589\\t--\\tmm10\\tchr8\\t106610533\\t106611666\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\tChIP-seq\\t\"Our findings were corroborated by H3K4me1 and H3K27ac ChIP-seq data from an alternative na?ve-to-formative differentiation system. As a control, we found little change in these marks at SMC1 sites common to both states. These results indicate that GRHL2 binding correlates with full enhancer activation, suggesting a role for GRHL2 not only in regulation of cohesin binding, but also in regulation of other crucial steps in enhancer activation.Together, these results suggest that our SMC1 ChIP-seq data reflect changing enhancer activity and that cohesin binding at enhancers is highly dynamic during the ESC to EpiLC transition.\"\\tEnhancer\\tCdh1\\tCRISPR/Cas9\\t\"ChIP-seq,qRT-PCR\"\\t\"We then assessed the effects of each Enhancer deletion on Dsp expression in ES cells and EpiLCs by qRT-PCR.Indeed, we found that deletion of one or both KLF4-bound enhancers led to a significant reduction in Dsp expression in ESCs but had no effect in EpiLCs(Figure 6B). Conversely, deletion of the GRHL2-bound enhancer resulted in significantly reduced Dsp expression in the EpiLC state but had no effect in ESCs (Figure 6B). We observed similar state-specific reductions in target gene expression with deletions of the KLF4-bound and GRHL2-bound enhancers at the Cdh1 and Jam1 loci (Figure 6C and D, S6A and B).\"\\t\"AA960649,ARC-1,E-cad,Ecad,L-CAM,UVO,Um\"\\t--\\t--\\t--\\tGRHL2 therefore control over a subset of the naive network via Enhancer switching to maintain expression of epithelial genes upon exit from naive pluripotency\\tChIP-seq\\tDeletion of the GRHL2-bound Enhancer resulted in significantly reduced Dsp expression in the EpiLC state but had no effect in ESCs.\\tGrhl2\\t\"0610015A08Rik,BOM,Tcfcp2l3,clft3\"\\t\"ChIP-seq,ATAC-seq\"\\t\"Given the strong association between GRHL2 binding and various events in enhancer activation, we asked whether GRHL2 binding is necessary for each of these events. To address this question, we assessed levels of each active enhancer mark in wildtype and GHRL2 KO EpiLCs by performing ATAC-seq and ChIP-seq for H3K4me1, H3K27ac, and SMC1. There was a strong reduction in all active enhancer marks (chromatin accessibility, H3K4me1, H3K27ac, and SMC1 levels) at GRHL2 sites in GRHL2 KO EpiLCs, suggesting a near complete block in enhancer activation in the absence of GRHL2. Together, these results indicate an absolute requirement for GRHL2 for enhancer activation specifically at its target sites.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t7751', 'E_02_246\\t30285185\\t--\\tmm10\\tchr11\\t6639091\\t6639657\\tMouse\\tMammary\\tLow+High throughput\\tChIP-seq\\t\"Employ_x0002_ing ChIP-seq, we uncover a changing transcription factor occupancy at mammary enhancers, suggest_x0002_ing that their activities evolve with advancing dif_x0002_ferentiation.\"\\tEnhancer\\tWap\\tCRISPR/Cas9\\t\"qRT-PCR,ChIP-seq\"\\t\"Most profoundly, a seed enhancer, which is manda_x0002_tory for the activation of the Wap super-enhancer during pregnancy, is not required during lactation,suggesting compensatory flexibility. Combinatorial deletions of structurally equivalent constituent en_x0002_hancers demonstrated differentiation-specific com_x0002_pensatory activities during lactation.\"\\tWap\\t--\\t--\\t--\\t\"Wap super-enhancer evolve as lactation progresses.', 'E_02_247\\t30285185\\t--\\tmm10\\tchr11\\t6639824\\t6640424\\tMouse\\tMammary\\tLow+High throughput\\tChIP-seq\\t\"Employ_x0002_ing ChIP-seq, we uncover a changing transcription factor occupancy at mammary enhancers, suggest_x0002_ing that their activities evolve with advancing dif_x0002_ferentiation.\"\\tEnhancer\\tWap\\tCRISPR/Cas9\\t\"qRT-PCR,ChIP-seq\"\\t\"Most profoundly, a seed enhancer, which is manda_x0002_tory for the activation of the Wap super-enhancer during pregnancy, is not required during lactation,suggesting compensatory flexibility. Combinatorial deletions of structurally equivalent constituent en_x0002_hancers demonstrated differentiation-specific com_x0002_pensatory activities during lactation.\"\\tWap\\t--\\t--\\t--\\t\"Wap super-enhancer evolve as lactation progresses.', 'E_02_248\\t30285185\\t--\\tmm10\\tchr11\\t6644257\\t6644824\\tMouse\\tMammary\\tLow+High throughput\\tChIP-seq\\t\"Employ_x0002_ing ChIP-seq, we uncover a changing transcription factor occupancy at mammary enhancers, suggest_x0002_ing that their activities evolve with advancing dif_x0002_ferentiation.\"\\tSuper-Enhancer\\tWap\\tCRISPR/Cas9\\t\"qRT-PCR,ChIP-seq\"\\t\"Most profoundly, a seed enhancer, which is manda_x0002_tory for the activation of the Wap super-enhancer during pregnancy, is not required during lactation,suggesting compensatory flexibility. Combinatorial deletions of structurally equivalent constituent en_x0002_hancers demonstrated differentiation-specific com_x0002_pensatory activities during lactation.\"\\tWap\\t--\\t--\\t--\\t\"Wap super-enhancer evolve as lactation progresses.', 'E_02_249\\t30285185\\t--\\tmm10\\tchr11\\t6639029\\t6644397\\tMouse\\tMammary\\tLow+High throughput\\tChIP-seq\\t\"Employ_x0002_ing ChIP-seq, we uncover a changing transcription factor occupancy at mammary enhancers, suggest_x0002_ing that their activities evolve with advancing dif_x0002_ferentiation.\"\\tSuper-Enhancer\\tWap\\tCRISPR/Cas9\\t\"qRT-PCR,ChIP-seq\"\\t\"Most profoundly, a seed enhancer, which is manda_x0002_tory for the activation of the Wap super-enhancer during pregnancy, is not required during lactation,suggesting compensatory flexibility. Combinatorial deletions of structurally equivalent constituent en_x0002_hancers demonstrated differentiation-specific com_x0002_pensatory activities during lactation.\"\\tWap\\t--\\t--\\t--\\t\"Wap super-enhancer evolve as lactation progresses.', 'E_02_250\\t30295986\\tGata3 distal Enhancers\\tmm10\\tchr2\\t9838078\\t9840078\\tMouse\\tMouse Embryos\\tLow throughput\\t\"Luciferase Reporter Assay,qPCR\"\\tWe next employed a transient transfection assay to test putative Gata3 Enhancers in Gata3-expressing 293T cells using a heterologous TATA promoter that drives Luciferasegene expression. Here we describe the identification of a novel bipartite Gata3 lens-specific Enhancer located ~18 kb upstream from its transcriptional start site. We also found that a 5 kb Gata3 promoter possesses low activity in the lens.\\tEnhancerGata3\\t--\\t\"Luciferase Reporter Assay,qPCR\"\\t\"Here we describe the identification of a novel bipartite Gata3 lens-specific enhancer located', 'E_02_251\\t30327417\\tCiita Enhancer\\tmm10\\tchr16\\t10440279\\t10442279\\tMouse\\tMacrophage\\tLow throughput\\tqChIP\\t\"We then analyzed peak A and Ciita promoter I by qChIP in NFAT5-deficient or control macrophages for different histone modifications that are enriched in gene enhancers (H3K4me1)', 'E_02_252\\t30328555\\tSTAT5 Enhancers\\tmm10\\tchr11\\t100854851\\t100856851\\tMouse\\tMammary\\tLow+High throughput\\t\"ChIP-seq,CRISPR/Cas9\"\\t\"Autoregulation of the Stat5 locus in mammary tissue. a ChIP-seq profiles of an auto-regulatory Enhancer at the Stat5 locus. Mammary-specific STAT5 binding coincides with two adjacent GAS motifs,3.5 kb upstream of Stat5a.\"\\tEnhancer\\tStat5a\\t--\\tChIP-seq\\t\"We have further provided biological evidence supporting the in vivo function of a STAT5-driven super_x0002_enhancer with the aid of CRISPR/Cas9 genome editing.Finally, we discuss how the functions of mammary-specific super_x0002_enhancers are confined by the zinc finger protein, CTCF, to allow exclusive activation of mammary-specific genes without affecting common neighboring genes. \"\\t\"AA959963,STAT5\"\\t--\\t--\\t--\\t--\\t--\\t--\\tStat5a\\t\"AA959963,STAT5\"\\t\"ChIP-seq,CRISPR/Cas9\"\\tWe have further provided biological evidence supporting the in vivo function of a STAT5-driven super-Enhancer with the aid of CRISPR/Cas9 genome editing.\\t--\\t--\\t--\\t>2KB\\t\\t3499', 'E_02_253\\t30407507\\tE Enhancer\\tmm10\\tchr12\\t114640000\\t114670000\\tMouse\\tB Cell\\tLow throughput\\tChIP\\t\"Chromatin immunoprecipitation analyses of activated B cells identified significant ER binding to estrogen response elements (ERE) centered within enhancer elements of the immunoglobulin heavy chain locus, including the E enhancer and hypersensitive site', 'E_02_254\\t30407507\\t--\\tmm10\\tchr12\\t114460000\\t114490000\\tMouse\\tB Cell\\tLow throughput\\tChIP\\t\"Chromatin immunoprecipitation analyses of activated B cells identified significant ER binding to estrogen response elements (ERE) centered within enhancer elements of the immunoglobulin heavy chain locus, including the E enhancer and hypersensitive site', 'E_02_255\\t30416088\\t--\\tmm10\\tchr17\\t85795937\\t85796217\\tMouse\\tMetanephric Mesenchymal (MM) cells\\tLow+High throughput\\tChIP-seq\\t\"We conducted comprehensive analyses of the chromatin signature and transcriptome by RNA_x0002_seq and ChIP-seq.We found seventeen lncRNAs that were expressed specifically in MM cells with an active chromatin signature, while remaining silenced in a bivalent chromatin state in non-MM cells.Out of these MM specific lncRNAs, we identified a lncRNA, Gm29418, in a distal enhancer region of Six2,a key regulatory gene of MM cells.\"\\tEnhancer\\tSix2\\t--\\t\"ChIP-seq,RNA-seq,qRT-PCR\"\\t\"We conducted comprehensive analyses of the chromatin signature and transcriptome by RNA_x0002_seq and ChIP-seq.We found seventeen lncRNAs that were expressed specifically in MM cells with an active chromatin signature, while remaining silenced in a bivalent chromatin state in non-MM cells.Out of these MM specific lncRNAs, we identified a lncRNA, Gm29418, in a distal enhancer region of Six2,a key regulatory gene of MM cells.\"\\tSix2\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t133336', 'E_02_256\\t30416088\\t--\\tmm10\\tchr17\\t85797325\\t85797927\\tMouse\\tMetanephric Mesenchymal (MM) cells\\tLow+High throughput\\tChIP-seq\\t\"We conducted comprehensive analyses of the chromatin signature and transcriptome by RNA_x0002_seq and ChIP-seq.We found seventeen lncRNAs that were expressed specifically in MM cells with an active chromatin signature, while remaining silenced in a bivalent chromatin state in non-MM cells.Out of these MM specific lncRNAs, we identified a lncRNA, Gm29418, in a distal enhancer region of Six2,a key regulatory gene of MM cells.\"\\tEnhancer\\tSix2\\t--\\t\"ChIP-seq,RNA-seq,qRT-PCR\"\\t\"We conducted comprehensive analyses of the chromatin signature and transcriptome by RNA_x0002_seq and ChIP-seq.We found seventeen lncRNAs that were expressed specifically in MM cells with an active chromatin signature, while remaining silenced in a bivalent chromatin state in non-MM cells.Out of these MM specific lncRNAs, we identified a lncRNA, Gm29418, in a distal enhancer region of Six2,a key regulatory gene of MM cells.\"\\tSix2\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t134885', 'E_02_257\\t30527662\\tBdR Enhancer\\tmm10\\tchr17\\t30249928\\t30278499\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,ChIA-PET\"\\t\"The data show that the promoters are located within Smc1 ChIA-PET loops,the anchors of which align with strong cohesin and CTCF ChIP-seq peaks lacking H3K27ac ( These insulated neighborhoods encompass the TSS(arrow),promoters (vertical gray bars),and potential Enhancers of Mitf and Sik1 (vertical green bar). View of a genomic region around Mitf integrating Smc1 ChIA-PET,and Smc1,Smc3,Rad21,CTCF,Med1,and H3K27ac ChIP-seq with 4C data and promoter capture Hi-C data.\"\\tEnhancer\\tSik1\\t4C\\t\"Hi-C,ChIP-seq,ChIA-PET\"\\t\"High-resolution 4C was employed using promoter viewpoints to determine whether individual genes fit into the framework described in Figure 1.Figures 2A and 2B compare the Mitf and Sik1 genes expressed at low(RPKM = 2.2) and modest (RPKM = 21) levels, respectively, as', 'E_02_258\\t30527662\\tBdR Enhancer\\tmm10\\tchr6\\t98336842\\t98365566\\tMouse\\tMouse Embryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,ChIA-PET\"\\t\"The data show that the promoters are located within Smc1 ChIA-PET loops,the anchors of which align with strong cohesin and CTCF ChIP-seq peaks lacking H3K27ac ( These insulated neighborhoods encompass the TSS(arrow),promoters (vertical gray bars),and potential Enhancers of Mitf and Sik1 (vertical green bar). View of a genomic region around Mitf integrating Smc1 ChIA-PET,and Smc1,Smc3,Rad21,CTCF,Med1,and H3K27ac ChIP-seq with 4C data and promoter capture Hi-C data.\"\\tEnhancer\\tMitf\\t4C\\t\"Hi-C,ChIP-seq,ChIA-PET\"\\t\"High-resolution 4C was employed using promoter viewpoints to determine whether individual genes fit into the framework described in Figure 1.Figures 2A and 2B compare the Mitf and Sik1 genes expressed at low(RPKM = 2.2) and modest (RPKM = 21) levels, respectively, as', 'E_02_259\\t30544251\\t--\\tmm10\\tchr12\\t35939690\\t35990427\\tMouse\\tST2\\tLow+High throughput\\tChIP-seq\\t\"The four adjacent merged SEs (SE283,SE284,SE285,and SE286) at the Ahr locus together cover a continuous region over 300 kb of active enhancer signal downstream of the Ahr gene in the ST2 cells.Moreover, all four SEs showed a very high correlation (r  0.95) with Ahr mRNA levels as measured by RNA-seq (Figure 5C and D, upper panels) and validated by RT-qPCR.\"\\tSuper-Enhancer\\tAhr\\t--\\t\"RT-qPCR,ChIP,RNA-seq\"\\tAhr is regulated by multiple SEs with lineage-specific dynamics The Ahr mRNA level was measured across the differentiation by RNA-seq and RT-qPCR in both adipocyte and osteoblast differentiation and is indicated astheintactline.\\t\"Ah,Ahhe,In,bHLHe76,Ahr\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Ahr,Glis1\"\\t\"Ah, Ahhe, In, bHLHe76, Ahr,Gli5, Gli6, GliH1\"\\tRT-qPCR\\t\"To confirm the observed differentiation defects in the presence of high AHR and GLIS1 levels, RT-qPCR analysis of the known adipocyte marker gene Lpl was performed. In ST2-TetOn-GFP cells Lpl was upregulated by D5 of differentiation and remained elevated in D9 cells both in presence and absence of doxycycline.\"\\t--\\t--\\t--\\t>2KB\\t\\t467081', 'E_02_260\\t30544251\\t--\\tmm10\\tchr12\\t35841201\\t35921783\\tMouse\\tST2\\tLow+High throughput\\tChIP-seq\\t\"The four adjacent merged SEs (SE283,SE284,SE285,and SE286) at the Ahr locus together cover a continuous region over 300 kb of active enhancer signal downstream of the Ahr gene in the ST2 cells.Moreover, all four SEs showed a very high correlation (r  0.95) with Ahr mRNA levels as measured by RNA-seq (Figure 5C and D, upper panels) and validated by RT-qPCR.\"\\tSuper-Enhancer\\tAhr\\t--\\t\"RT-qPCR,ChIP,RNA-seq\"\\tAhr is regulated by multiple SEs with lineage-specific dynamics The Ahr mRNA level was measured across the differentiation by RNA-seq and RT-qPCR in both adipocyte and osteoblast differentiation and is indicated astheintactline.\\t\"Ah,Ahhe,In,bHLHe76,Ahr\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Ahr,Glis1\"\\t\"Ah, Ahhe, In, bHLHe76, Ahr,Gli5, Gli6, GliH1\"\\tRT-qPCR\\t\"To confirm the observed differentiation defects in the presence of high AHR and GLIS1 levels, RT-qPCR analysis of the known adipocyte marker gene Lpl was performed. In ST2-TetOn-GFP cells Lpl was upregulated by D5 of differentiation and remained elevated in D9 cells both in presence and absence of doxycycline.\"\\t--\\t--\\t--\\t>2KB\\t\\t383514', 'E_02_261\\t30544251\\t--\\tmm10\\tchr12\\t35769572\\t35823293\\tMouse\\tST2\\tLow+High throughput\\tChIP-seq\\t\"The four adjacent merged SEs (SE283,SE284,SE285,and SE286) at the Ahr locus together cover a continuous region over 300 kb of active enhancer signal downstream of the Ahr gene in the ST2 cells.Moreover, all four SEs showed a very high correlation (r  0.95) with Ahr mRNA levels as measured by RNA-seq (Figure 5C and D, upper panels) and validated by RT-qPCR.\"\\tSuper-Enhancer\\tAhr\\t--\\t\"RT-qPCR,ChIP,RNA-seq\"\\tAhr is regulated by multiple SEs with lineage-specific dynamics The Ahr mRNA level was measured across the differentiation by RNA-seqand RT-qPCR in both adipocyte and osteoblastdifferentiation and is indicated astheintactline.\\t\"Ah,Ahhe,In,bHLHe76,Ahr\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Ahr,Glis1\"\\t\"Ah, Ahhe, In, bHLHe76, Ahr,Gli5, Gli6, GliH1\"\\tRT-qPCR\\t\"To confirm the observed differentiation defects in the presence of high AHR and GLIS1 levels, RT-qPCR analysis of the known adipocyte marker gene Lpl was performed. In ST2-TetOn-GFP cells Lpl was upregulated by D5 of differentiation and remained elevated in D9 cells both in presence and absence of doxycycline.\"\\t--\\t--\\t--\\t>2KB\\t\\t298455', 'E_02_262\\t30544251\\t--\\tmm10\\tchr12\\t35690979\\t35768472\\tMouse\\tST2\\tLow+High throughput\\tChIP-seq\\t\"The four adjacent merged SEs (SE283,SE284,SE285,and SE286) at the Ahr locus together cover a continuous region over 300 kb of active enhancer signal downstream of the Ahr gene in the ST2 cells.Moreover, all four SEs showed a very high correlation (r  0.95) with Ahr mRNA levels as measured by RNA-seq (Figure 5C and D, upper panels) and validated by RT-qPCR.\"\\tSuper-Enhancer\\tAhr\\t--\\t\"RT-qPCR,ChIP,RNA-seq\"\\tAhr is regulated by multiple SEs with lineage-specific dynamics The Ahr mRNA level was measuredacross the differentiation by RNA-seq and RT-qPCR in both adipocyte and osteoblast differentiation and is indicatedastheintactline.\\t\"Ah,Ahhe,In,bHLHe76,Ahr\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Ahr,Glis1\"\\t\"Ah, Ahhe, In, bHLHe76, Ahr,Gli5, Gli6, GliH1\"\\tRT-qPCR\\t\"To confirm the observed differentiation defects in the presence of high AHR and GLIS1 levels, RT-qPCR analysis of the known adipocyte marker gene Lpl was performed. In ST2-TetOn-GFP cells Lpl was upregulated by D5 of differentiation and remained elevated in D9 cells both in presence and absence of doxycycline.\"\\t--\\t--\\t--\\t>2KB\\t\\t231748', 'E_02_263\\t19141476\\t--\\tmm10\\tchr5\\t147205131\\t147212149\\tMouse\\tPancreas\\tLow+High throughput\\t\"ChIP,ChIP-seq\"\\t\"Using chromatin immunoprecipitations (ChIP) and ChIP sequencing (ChIP-Seq) of fetal pancreas and islet chromatin, we demonstrate that Foxa1 and Foxa2 predominantly occupy a distal Enhancer at -6.4 kb relative to the transcriptional start site in the Pdx1 gene.\"\\tEnhancer\\tPdx1\\t--\\tChIP\\tWe performed chromatin immunoprecipitation (ChIP) as_x0002_says with antibodies specific to either Foxa1 or Foxa2 on chromatin isolated from primary mouse islets. Both Foxa1 and Foxa2 bound to the Area ICIICIII enhancer of Pdx1 in vivo.\\t\"IDX-1,IPF-1,Ipf1,Mody4,STF-1,pdx-1\"\\tPancreatic Hypoplasia\\t--\\t--\\t--\\t--\\t--\\t\"Foxa2,Foxa1\"\\t\"HNF3-beta,HNF3beta,Hnf-3b,Hnf3b,Tcf-3b,Tcf3b,Hnf-3a,Hnf3a,Tcf-3a,Tcf3a\"\\tChIP\\t\"We performed chromatin immunoprecipitation (ChIP) assays with antibodies specific to either Foxa1 or Foxa2 on chromatin isolated from primary mouse islets. Both Foxa1 and Foxa2 bound to the Area ICIICIII Enhancer of Pdx1 in vivo .Notably, the binding of Foxa2 to Area IV or Area ICIICIII is signifi-cantly enhanced in pancreatic islets compared with fetal pancreas.\"\\t--\\t--\\t--\\t>2KB\\t61490', 'E_02_264\\t19141476\\t--\\tmm10\\tchr5\\t147263601\\t147264086\\tMouse\\tPancreas\\tLow+High throughput\\t\"ChIP,ChIP-seq\"\\t\"Using chromatin immunoprecipitations (ChIP) and ChIP sequencing (ChIP-Seq) of fetal pancreas and islet chromatin, we demonstrate that Foxa1 and Foxa2 predominantly occupy a distal Enhancer at -6.4 kb relative to the transcriptional start site in the Pdx1 gene.\"\\tEnhancer\\tPdx1\\t--\\tChIP\\tWe performed chromatin immunoprecipitation (ChIP) as_x0002_says with antibodies specific to either Foxa1 or Foxa2 on chromatin isolated from primary mouse islets. Both Foxa1 and Foxa2 bound to the Area ICIICIII enhancer of Pdx1 in vivo.\\t\"IDX-1,IPF-1,Ipf1,Mody4,STF-1,pdx-1\"\\tPancreatic Hypoplasia\\t--\\t--\\t--\\t--\\t--\\t\"Foxa2,Foxa1\"\\t\"HNF3-beta,HNF3beta,Hnf-3b,Hnf3b,Tcf-3b,Tcf3b,Hnf-3a,Hnf3a,Tcf-3a,Tcf3a\"\\tChIP\\t\"We performed chromatin immunoprecipitation (ChIP) assays with antibodies specific to either Foxa1 or Foxa2 on chromatin isolated from primary mouse islets. Both Foxa1 and Foxa2 bound to the Area ICIICIII Enhancer of Pdx1 in vivo .Notably, the binding of Foxa2 to Area IV or Area ICIICIII is signifi-cantly enhanced in pancreatic islets compared with fetal pancreas.\"\\t--\\t--\\t--\\t>2KB\\t6286', 'E_02_265\\t21185279\\tEnhancer N2\\tmm10\\tchr3\\t34645597\\t34646134\\tMouse\\tMouse Embryos\\tLow throughput\\tChIP\\t\"ChIP assay demonstrating Pou5f1 binding to the Enhancer N2 region.In transgenic mouse embryos, mouse enhancer N2 showed activity in the entire epiblast at E6.5, and ANP area at E7.75, as indicated by the expression of the N2-hspLacZ transgene, coincident with Sox2 expression in the epiblast and forming ANP.\"\\tEnhancer\\tSox2\\t--\\t\"ChIP,RT-PCR\"\\t\"ChIP assay demonstrating Pou5f1 binding to the Enhancer N2 region. Embryos were injected with pou5f1-FLAG mRNA or left non-injected, and ChIP was performed using anti-FLAG antibody. qPCR analysis shows enrichment of the Enhancer N2 region sequence (N2) in injected embryos without enrichment of the control region (Rpl5b).\"\\t\"Sox-2,lcc,ysb\"\\t--\\t--\\t--\\t--\\t--\\t--\\tPou5f1\\t\"NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4\"\\tEMSA\\t\"These observations indicated that simultaneous binding of POU factors to the core sequence is essential for eliciting enhancer N2 activity in all examined developmental stages of the mouse embryos.these EMSA data in conjunction with the enhancer activity analyses shown in Fig. 3 indicate that the simultaneous binding of POU factors to POU-1 and POU-2/3 sites is required for the activation of the 176-bp enhancer in ES, epiblast/EpiSC and ANP cells.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t4128', 'E_02_266\\t21527504\\t--\\tmm10\\tchr14\\t67745323\\t67746327\\tMouse\\tGT1-1\\tLow throughput\\t\"Luciferase Reporter Assay,ChIP\"\\t\"To examine the enhancer activity of intron A, we constructed reporter vectors containing the full-length mouse intron A sequence (1003 bp) in either the forward or reversed direction followed by the simian vacuolating virus 40 (SV40) minimal promoter (SV40p) driving luciferase expression.These findings strongly suggest that GnRH intron A contains a putative transcriptional enhancer that is likely involved in GnRH gene regulation. In an ensuing experiment, serially deleted and truncated reporter constructs revealed that 315 nucleotides (nt) of GnRH intron A, located proximal to Ex1, were responsible for its transcription enhancing activity; the enhancing activity of this region was comparable with that of the full-length intron A.\"\\tEnhancer\\tGnrh1\\t--\\t\"ChIP,RT-qPCR\"\\t\"GnRH is a pivotal hypothalamic neurohormone governing reproduction and sexual development. Because transcriptional regulation is crucial for the spatial and temporal expression of the GnRH gene, a region approximately 3.0 kb upstream of the mammalian GnRH promoter has been extensive studied. In the present study, we demonstrate a transcription-enhancer located in the first intron (intron A) region of the GnRH gene. \"\\t\"Gnrh,Gnrh2,LHRH,Lhrh1,Lnrh,hpg\"\\t--\\t--\\t--\\t--\\t--\\t--\\tQsox1\\t\"1300003H02Rik,QSOX,Qscn6,SOx,b2b2673Clo\"\\t\"ChIP,EMSA\"\\t\"Chromatin immunoprecipitation (ChIP) assay revealed that SOX11 binds to the Ex1-proximal Enhancer region of intron A, and overexpression of SOX11 in creases this binding in GT1-1 cells.', 'E_02_267\\t25801169\\t--\\tmm10\\tchr17\\t35501113\\t35507015\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"We cloned individual constituent Enhancers of the five super-Enhancers into Enhancer-reporter vectors,and found that the majority (21/24) of super-Enhancer constituents were active in luciferase reporter assays (>1.5 fold over control, P<0.01 Students t-test) in ESCs .\"\\tSuper-Enhancer\\tPou5f1\\t--\\t\"Luciferase Reporter Assay,ChIP-seq\"\\t\"We used the reporter system to investigate how various combinations of the constituents of the Pou5f1 super-enhancer behaved in this assay. The Pou5f1 super-enhancer was selected for this experiment because its SE was small enough to be fully accommodated by the reporter vector. The results revealed that the three constituent enhancers produced slightly less activity than E2 alone, which produced the largest signal.\"\\t\"NF-A3,Oct-3,Oct-3/4,Oct-4,Oct3,Oct3/4,Oct4,Otf-3,Otf-4,Otf3,Otf3-rs7,Otf3g,Otf4\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t<2KB\\t1967', 'E_02_268\\t24385922\\t--\\tmm10\\tchr6\\t52407532\\t52459468\\tMouse\\t\"Limb,Head\"\\tLow+High throughput\\t\"ChIP-seq,3C,5C\"\\t\"Active enhancers are characterized by the binding of several proteins including RNA polymerase II (RNAP2), and subunits of Mediator like Med12.We therefore mapped candidate enhancer sequences by identifying genomic sites enriched in these proteins using chromatin immunoprecipitation combined with deep sequencing(ChIP-seq) in cells isolated from E12.5 distal limb buds.Candidate HoxA enhancers therefore reside amidst other genes including Hibadh, Tax1bp1, and Jazf1, for which expression has been reported in the limb. Consistent with our 3C data, this 5C analysis revealed a similar contact pattern between the 59 HoxA genes and upstream regulatory region in the Shh2/2 mutant and wt limbs.\"\\tEnhancer\\tHoxa\\t\"5C,3C\"\\t--\\t\"We profiled the interaction pattern of the HoxA cluster with the upstream 850 Kb region in', 'E_02_269\\t24385922\\t--\\tmm10\\tchr6\\t52527210\\t52556565\\tMouse\\t\"Limb,Head\"\\tLow+High throughput\\t\"ChIP-seq,3C,5C\"\\t\"Active enhancers are characterized by the binding of several proteins including RNA polymerase II (RNAP2), and subunits of Mediator like Med12.We therefore mapped candidate enhancer sequences by identifying genomic sites enriched in these proteins using chromatin immunoprecipitation combined with deep sequencing(ChIP-seq) in cells isolated from E12.5 distal limb buds.Candidate HoxA enhancers therefore reside amidst other genes including Hibadh, Tax1bp1, and Jazf1, for which expression has been reported in the limb.Consistent with our 3C data, this 5C analysis revealed a similar contact pattern between the 59 HoxA genes and upstream regulatory region in the Shh2/2 mutant and wt limbs.\"\\tEnhancer\\tHoxa\\t\"5C,3C\"\\t--\\t\"We profiled the interaction pattern of the HoxA cluster with the upstream 850 Kb region in', 'E_02_270\\t26855180\\tmiR-290-295 SE\\tmm10\\tchr7\\t3193003\\t3218182\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"We performed a functional dissection of miRNA SEs in vivo by generating cell lines depleted of individual miRNA SE constituents (about 400C700 bp) in multiple cell types through a CRISPR/Cas9-based approach (Table S5). We analyzed three miRNA SEs: miR-290-295 SE in mESCs, miR-1 SE in myotubes, and miR-148a SE in Pro-B cells. Among seven constituents of miR-290-295 SE (E1-E7, Figure 4A), deletion of individual elements except for E6 led to substantial decrease (ranging from 50% to 80%) in de novo production of mature miRNAs in TT-FHAgo2 mESC background without concomitant changes in other miRNA production (Figures 4B and S5A). A 5-fold reduction in expression upon deletion of one constituent of a group of six indicates cooperative interactions.As expected from a decrease in these miRNAs, these deletions also suppressed expression of Ago2-inducible gene Lin28a, a regulator of ES cells (Figure 4B).', 'E_02_271\\t26855180\\tmiR-290-295 SE (E1)\\tmm10\\tchr7\\t3199567\\t3201439\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Among seven constituents of miR-290-295 SE (E1-E7, Figure 4A), deletion of individual elements except for E6 led to substantial decrease (ranging from 50% to 80%) in de novo production of mature miRNAs in TT-FHAgo2 mESC background without concomitant changes in other miRNA production (Figures 4Band S5A). A 5-fold reduction in expression upon deletion of one constituent of a group of six indicates cooperative interactions.As expected from a decrease in these miRNAs, these deletions', 'E_02_272\\t26855180\\tmiR-290-295 SE (E2)\\tmm10\\tchr7\\t3202646\\t3204360\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Among seven constituents of miR-290-295 SE (E1-E7, Figure 4A), deletion of individual elements except for E6 led to substantial decrease (ranging from 50% to 80%) in de novo production of mature miRNAs in TT-FHAgo2 mESC background without concomitant changes in other miRNA production (Figures 4Band S5A). A 5-fold reduction in expression upon deletion of one constituent of a group of six indicates cooperative interactions.As expected from a decrease in these miRNAs, these deletions', 'E_02_273\\t26855180\\tmiR-290-295 SE (E3)\\tmm10\\tchr7\\t3205726\\t3207280\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Among seven constituents of miR-290-295 SE (E1-E7, Figure 4A), deletion of individual elements except for E6 led to substantial decrease (ranging from 50% to 80%) in de novo production of mature miRNAs in TT-FHAgo2 mESC background without concomitant changes in other miRNA production (Figures 4Band S5A). A 5-fold reduction in expression upon deletion of one constituent of a group of six indicates cooperative interactions.As expected from a decrease in these miRNAs, these deletions', 'E_02_274\\t26855180\\tmiR-290-295 SE (E4)\\tmm10\\tchr7\\t3209567\\t3211439\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Among seven constituents of miR-290-295 SE (E1-E7, Figure 4A), deletion of individual elements except for E6 led to substantial decrease (ranging from 50% to 80%) in de novo production of mature miRNAs in TT-FHAgo2 mESC background without concomitant changes in other miRNA production (Figures 4Band S5A). A 5-fold reduction in expression upon deletion of one constituent of a group of six indicates cooperative interactions.As expected from a decrease in these miRNAs, these deletions', 'E_02_275\\t26855180\\tmiR-290-295 SE (E6)\\tmm10\\tchr7\\t3212448\\t3214558\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Among seven constituents of miR-290-295 SE (E1-E7, Figure 4A), deletion of individual elements except for E6 led to substantial decrease (ranging from 50% to 80%) in de novo production of mature miRNAs in TT-FHAgo2 mESC background without concomitant changes in other miRNA production (Figures 4Band S5A). A 5-fold reduction in expression upon deletion of one constituent of a group of six indicates cooperative interactions.As expected from a decrease in these miRNAs, these deletions', 'E_02_276\\t26855180\\tmiR-290-295 SE (E7)\\tmm10\\tchr7\\t3216607\\t3218399\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Among seven constituents of miR-290-295 SE (E1-E7, Figure 4A), deletion of individual elements except for E6 led to substantial decrease (ranging from 50% to 80%) in de novo production of mature miRNAs in TT-FHAgo2 mESC background without concomitant changes in other miRNA production (Figures 4Band S5A). A 5-fold reduction in expression upon deletion of one constituent of a group of six indicates cooperative interactions.As expected from a decrease in these miRNAs, these deletions', 'E_02_277\\t26855180\\tmiR-1 SE\\tmm10\\tchr2\\t180362582\\t180377003\\tMouse\\tmyotubes\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"We performed a functional dissection ofmiRNA SEs in vivo by generating cell lines depleted of individual miRNA SE constituents (about 400C700 bp) in multiple cell types through a CRISPR/Cas9-based approach (Table S5). We analyzed three miRNA SEs: miR-290-295 SE in mESCs, miR-1 SE in myotubes, and miR-148a SE in Pro-B cells.', 'E_02_278\\t26855180\\tmiR-1 SE(M1)\\tmm10\\tchr2\\t180361998\\t180364335\\tMouse\\tmyotubes\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"In the C2C12 cell line, a model for myotube differentiation, SE was only observed for one of the two miR-1 genes (miR-1a-1/133a-2) and myotube differentiation', 'E_02_279\\t26855180\\tmiR-1 SE(M2)\\tmm10\\tchr2\\t180366734\\t180368371\\tMouse\\tmyotubes\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"In the C2C12 cell line, a model for myotube differentiation, SE was only observed for one of the two miR-1 genes (miR-1a-1/133a-2) and myotube differentiation', 'E_02_280\\t26855180\\tmiR-1 SE(M3)\\tmm10\\tchr2\\t180367903\\t180369540\\tMouse\\tmyotubes\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"In the C2C12 cell line, a model for myotube differentiation, SE was only observed for one of the two miR-1 genes (miR-1a-1/133a-2) and myotube differentiation', 'E_02_281\\t26855180\\tmiR-1 SE(M4)\\tmm10\\tchr2\\t180375974\\t180377844\\tMouse\\tmyotubes\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"In the C2C12 cell line, a model for myotube differentiation, SE was only observed for one of the two miR-1 genes (miR-1a-1/133a-2) and myotube differentiation', 'E_02_282\\t26855180\\tmiR-148a SE1\\tmm10\\tchr6\\t51108708\\t51130031\\tMouse\\tPro-B cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"We performed a functional dissection ofmiRNA SEs in vivo by generating cell lines depleted of individual miRNA SE constituents (about 400C700 bp) in multiple cell types through a CRISPR/Cas9-based approach (Table S5). We analyzed three miRNA SEs: miR-290-295 SE in mESCs, miR-1 SE in myotubes, and miR-148a SE in Pro-B cells.Furthermore, in Pro-B cells, deletion of four out of ?ve miR-148a SE constituents dynamically suppressed induction of miR-148a during differenti-', 'E_02_283\\t26855180\\tmiR-148a SE2\\tmm10\\tchr6\\t51168034\\t51174811\\tMouse\\tPro-B cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"We performed a functional dissection ofmiRNA SEs in vivo by generating cell lines depleted of individual miRNA SE constituents (about 400C700 bp) in multiple cell types through a CRISPR/Cas9-based approach (Table S5). We analyzed three miRNA SEs: miR-290-295 SE in mESCs, miR-1 SE in myotubes, and miR-148a SE in Pro-B cells.Furthermore, in Pro-B cells, deletion of four out of ?ve miR-148a SE constituents dynamically suppressed induction of miR-148a during differenti-', 'E_02_284\\t26855180\\tmiR-148a SE(B1)\\tmm10\\tchr6\\t51117747\\t51120656\\tMouse\\tPro-B cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Furthermore, in Pro-B cells, deletion of four out of ?ve miR-148a SE constituents dynamically suppressed induction of miR-148a during differenti-', 'E_02_285\\t26855180\\tmiR-148a SE(B3)\\tmm10\\tchr6\\t51168147\\t51170512\\tMouse\\tPro-B cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Furthermore, in Pro-B cells, deletion of four out of ?ve miR-148a SE constituents dynamically suppressed induction of miR-148a during differenti-', 'E_02_286\\t26855180\\tmiR-148a SE(B4)\\tmm10\\tchr6\\t51171474\\t51173111\\tMouse\\tPro-B cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Furthermore, in Pro-B cells, deletion of four out of ?ve miR-148a SE constituents dynamically suppressed induction of miR-148a during differenti-', 'E_02_287\\t26855180\\tmiR-148a SE(B5)\\tmm10\\tchr6\\t51173363\\t51175544\\tMouse\\tPro-B cell\\tLow+High throughput\\t\"CRISPR/Cas9,RT-qPCR\"\\t\"Furthermore, in Pro-B cells, deletion of four out of ?ve miR-148a SE constituents dynamically suppressed induction of miR-148a during differenti-', 'E_02_288\\t26912657\\t\\tmm10\\tchr2\\t129350852\\t129356319\\tMouse\\tMacrophage\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq\"\\t\"Based on the ENCODE/UCSDbonemarrow-derivedmacrophageH3K27Ac chromatin immunoprecipitation sequencing (ChIP-seq) database, two peaks of DNA sequencing frequency were found in area, -10 kb (peak 1) and -2 kb (peak 2) upstream of the pro-IL-1beta transcription start site (TSS) (Fig. 4A). These peaks also coincided with the genomic regions highly associated with H3K4me1 but not with H3Kme3, indicating active enhancers.\"\\tenhancer\\tIl1b\\t--\\tqPCR\\t\"Therefore, we designed 11 qPCR primer sets encom-passing the genomic area to analyze the production of pro-IL-1 beta -associated eRNAs.These results suggest that the production of peak 2 eRNA was required for the optimal production of pro-IL-1beta mRNA.\"\\t\"IL-1beta, Il-1b\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_289\\t26728555\\t\\tmm10\\tchr12\\t109528035\\t109532509\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"ChIP-seq,ChIP\"\\t\"By performing ChIP-seq in different mESC lines and using two different antibodies, we were able to identify 973 high-confidence AFF3-binding sites.AFF3 is also de_x0002_tected in the intergenic regions within the Dlk1-Dio3 locus, which harbors a paternally methylated gDMR/', 'E_02_290\\t2835657\\t--\\tmm10\\tchr12\\t103852772\\t103852898\\tMouse\\tHepatoma Cells\\tlow throughput\\t\"Transient Assay,EMSA\"\\t\"To test whether the more distant upstream sequences could function as an enhancer, the -523- to -168-bp restriction fragment was attached in either orientation (at -341 bp)to a mouse ,beta-globin promoter.The results of transfection experiments showed a 10- to 15-fold enhancement of beta-globin transcription when this region was present in either orientation .\"\\tenhancer\\tSerpina1a\\t--\\t\"Transient Assay,EMSA\"\\tProteins present in nuclear extracts bind to the functionally defined elements present within the al-AT enhancer. Nuclear extracts from Hep-G2 cells were incubated with end-labeled DNA probes from the enhancer region to detect nuclear protein-binding sites by the gel retardation or gel shift assay. Each protein-binding site correlated with a functionally importantregion necessary for full enhancer activity. These experiments demonstrated a complex arrangement of regulatory elements comprising the al-antitrypsin enhancer.\\t\"Aat-2,Aat2,Dom1,PI1,Spi1-1,Spi1-3\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_291\\t2835657\\t\\tmm10\\tchr12\\t103852898\\t103853032\\tMouse\\tHepatoma Cells\\tlow throughput\\t\"Transient Assay,EMSA\"\\t\"To test whether the more distant upstream sequences could function as an enhancer, the -523- to -168-bp restriction fragment was attached in either orientation (at -341 bp)to a mouse ,beta-globin promoter.The results of transfection experiments showed a 10- to 15-fold enhancement of beta-globin transcription when this region was present in either orientation .\"\\tenhancer\\tSerpina1a\\t--\\t\"Transient Assay,EMSA\"\\tProteins present in nuclear extracts bind to the functionally defined elements present within the al-AT enhancer. Nuclear extracts from Hep-G2 cells were incubated with end-labeled DNA probes from the enhancer region to detect nuclear protein-binding sites by the gel retardation or gel shift assay. Each protein-binding site correlated with a functionally importantregion necessary for full enhancer activity. These experiments demonstrated a complex arrangement of regulatory elements comprising the al-antitrypsin enhancer.\\t\"Aat-2,Aat2,Dom1,PI1,Spi1-1,Spi1-3\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_292\\t2835657\\t\\tmm10\\tchr12\\t103853032\\t103853127\\tMouse\\tHepatoma Cells\\tlow throughput\\t\"Transient Assay,EMSA\"\\t\"To test whether the more distant upstream sequences could function as an enhancer, the -523- to -168-bp restriction fragment was attached in either orientation (at -341 bp)to a mouse ,beta-globin promoter.The results of transfection experiments showed a 10- to 15-fold enhancement of beta-globin transcription when this region was present in either orientation .\"\\tenhancer\\tSerpina1a\\t--\\t\"Transient Assay,EMSA\"\\tProteins present in nuclear extracts bind to the functionally defined elements present within the al-AT enhancer. Nuclear extracts from Hep-G2 cells were incubated with end-labeled DNA probes from the enhancer region to detect nuclear protein-binding sites by the gel retardation or gel shift assay. Each protein-binding site correlated with a functionally importantregion necessary for full enhancer activity. These experiments demonstrated a complex arrangement of regulatory elements comprising the al-antitrypsin enhancer.\\t\"Aat-2,Aat2,Dom1,PI1,Spi1-1,Spi1-3\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_293\\t12861010\\t\\tmm10\\tchr7\\t103771431\\t103772043\\tMouse\\t\"Pleen Cells,MEL cells\"\\tLow throughput \\t\"PCR,DNaseI-seq\"\\t\"We have also found DNase I HSs at kb -85.5 (HS-85.5) and -84.5 in mouse, but there is no corresponding se_x0002_quence or structure in human for these HSs.By comparison,neither the active _x0002_maj-globin gene promoter nor a region located ~1 kb from HS5 shows significant enrichment. HS-85.5 exhibits a modest (twofold) enrichment.\"\\tEnhancer\\tHbb-ar\\t--\\t\"DNaseI-seq,Southern blot,PCR\"\\t\"In ery_x0002_throid cells, the active _x0002_-globin locus contains several DNase I HSs, which have been shown to map to sequences with regu_x0002_latory function.In addition, histone hyperacetylation and dimethylation of histone H3 K4 are not uniform features of the nuclease-sensitive mouse _x0001_-globin domain but rather define distinct subdomains within it. Our results reveal a complex chromatin landscape for the active', 'E_02_294\\t15798207\\tEi\\tmm10\\tchr6\\t67556736\\t67558736\\tMouse\\t\"B Cell,T Cell,MPC-11\"\\tLow throughput \\t\"3C,PCR\"\"The mouse immunoglobulin kappa (Igk) gene contains an intronic enhancer and two enhancers downstream of its transcription unit. Using chromosome conformation capture technology, we demonstrate that rearranged and actively transcribed Igk alleles in MPC-11 plasmacytoma cells exhibit mutual interactions over 22 kb between these three enhancers and Vk gene promoters. We also observe interactions between Ei and E3\\'with 3\\'boundary sequences 24 kb downstream of Ed, adjacent to a neighboring house_x0002_keeping gene.\"\\tenhancer\\tIgk\\t3C\\t\"ChIP,Southern blot\"\\tChromatin segments within the transcription unit exhibit interactions throughout regions downstream of the tran_x0002_scription termination region specifically in active Igk loci.\\tkappa\\t--\\t--\\t--\\t\"We observe interactions between Ei and E3\\' with 3\\' boundary sequences 24 kb downstream of Ed, adjacent to a neighboring house-keeping gene.\"\\t3C\\tThese results fit a looping mechanism for enhancer function like in the beta-globin locus and suggest a dynamic modulation of the spatial organization of the active Igkappa locus.\\t\"Rela,Tcf3,Tfap4 \"\\t\"p65,p65 NF-kappa B,p65 NFkB,A1,AA408400,ALF2,AW209082,E12,E12/E47,E2A,E47,KA1,ME2,Pan1,Pan2, TCF-3,Tcfe2a,VDIR,bHLHb21,AI642933,AP-4,D930048N17Rik,Tcfap4,bHLHc41\"\\t\"ChIP,PCR\"\\t\"Taken together with the hypothesis that proteins that may be recruited by NF-kB and/or E2A might exhibit mutual bridging interactions between these transcription factors on different cis elements, we decided to first assay for the presence of NF-kB and E2A on the Igk gene enhancers and the Vk19-17 gene by performing ChIP experiments.\"\\t--\\t--\\t--\\tIntron\\t\\t\\t\\t2100', 'E_02_295\\t15798207\\tE3\\'\\tmm10\\tchr6\\t67561736\\t67563736\\tMouse\\t\"B Cell,T Cell,MPC-11\"\\tLow throughput \\t\"3C,PCR\"\"The mouse immunoglobulin kappa (Igk) gene contains an intronic enhancer and two enhancers downstream of its transcription unit. Using chromosome conformation capture technology, we demonstrate that rearranged and actively transcribed Igk alleles in MPC-11 plasmacytoma cells exhibit mutual interactions over 22 kb between these three enhancers and Vk gene promoters. We also observe interactions between Ei and E3\\'with 3\\'boundary sequences 24 kb downstream of Ed, adjacent to a neighboring house_x0002_keeping gene.\"\\tenhancer\\tIgk\\t3C\\t\"ChIP,Southern blot\"\\tChromatin segments within the transcription unit exhibit interactions throughout regions downstream of the tran_x0002_scription termination region specifically in active Igk loci.\\tkappa\\t--\\t--\\t--\\t\"We observe interactions between Ei and E3\\' with 3\\' boundary sequences 24 kb downstream of Ed, adjacent to a neighboring house-keeping gene.\"\\t3C\\tThese results fit a looping mechanism for enhancer function like in the beta-globin locus and suggest a dynamic modulation of the spatial organization of the active Igkappa locus.\\t\"Rela,Tcf3,Tfap4 \"\\t\"p65,p65 NF-kappa B,p65 NFkB,A1,AA408400,ALF2,AW209082,E12,E12/E47,E2A,E47,KA1,ME2,Pan1,Pan2, TCF-3,Tcfe2a,VDIR,bHLHb21,AI642933,AP-4,D930048N17Rik,Tcfap4,bHLHc41\"\\t\"ChIP,PCR\"\\t\"Taken together with the hypothesis that proteins that may be recruited by NF-kB and/or E2A might exhibit mutual bridging interactions between these transcription factors on different cis elements, we decided to first assay for the presence of NF-kB and E2A on the Igk gene enhancers and the Vk19-17 gene by performing ChIP experiments.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t7100', 'E_02_296\\t15798207\\tEd\\tmm10\\tchr6\\t67560136\\t67562136\\tMouse\\t\"B Cell,T Cell,MPC-11\"\\tLow throughput \\t\"3C,PCR\"\"The mouse immunoglobulin kappa (Igk) gene contains an intronic enhancer and two enhancers downstream of its transcription unit. Using chromosome conformation capture technology, we demonstrate that rearranged and actively transcribed Igk alleles in MPC-11 plasmacytoma cells exhibit mutual interactions over 22 kb between these three enhancers and Vk gene promoters. We also observe interactions between Ei and E3\\'with 3\\'boundary sequences 24 kb downstream of Ed, adjacent to a neighboring house_x0002_keeping gene.\"\\tenhancer\\tIgk\\t3C\\t\"ChIP,Southern blot\"\\tChromatin segments within the transcription unit exhibit interactions throughout regions downstream of the tran_x0002_scription termination region specifically in active Igk loci.\\tkappa\\t--\\t--\\t--\\t\"We observe interactions between Ei and E3\\' with 3\\' boundary sequences 24 kb downstream of Ed, adjacent to a neighboring house-keeping gene.\"\\t3C\\tThese results fit a looping mechanism for enhancer function like in the beta-globin locus and suggest a dynamic modulation of the spatial organization of the active Igkappa locus.\\t\"Rela,Tcf3,Tfap4 \"\\t\"p65,p65 NF-kappa B,p65 NFkB,A1,AA408400,ALF2,AW209082,E12,E12/E47,E2A,E47,KA1,ME2,Pan1,Pan2, TCF-3,Tcfe2a,VDIR,bHLHb21,AI642933,AP-4,D930048N17Rik,Tcfap4,bHLHc41\"\\t\"ChIP,PCR\"\\t\"Taken together with the hypothesis that proteins that may be recruited by NF-kB and/or E2A might exhibit mutual bridging interactions between these transcription factors on different cis elements, we decided to first assay for the presence of NF-kB and E2A on the Igk gene enhancers and the Vk19-17 gene by performing ChIP experiments.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t5500', 'E_02_297\\t27166834\\t--\\tmm10\\tchr8\\t120876144\\t120876856\\tMouse\\tMouse Embryos\\tLow+High throughput\\t\"3C,ChIP-seq\"\\t\"To identify the presence of physical interaction between the 11 enhancer candidates and the Foxf1 promoter, 3C assays were performed. Among 11 selected regions, three enhancer candidates demonstrated interactions with the Foxf1 promoter.\"\\tEnhancer \\tFoxf1\\t3C\\t\"RT-qPCR,DNaseI-seq,Transgenic mice\"\\t\"To investigate the in vivo activity of the discovered enhanc_x0002_ers and compare the location of activity with Foxf1-express_x0002_ing regions, we generated enhancer-driven lacZ-reporter', 'E_02_298\\t27166834\\t--\\tmm10\\tchr8\\t120882734\\t120883410\\tMouse\\tMouse Embryos\\tLow+High throughput\\t\"3C,ChIP-seq\"\\t\"To identify the presence of physical interaction between the 11 enhancer candidates and the Foxf1 promoter, 3C assays were performed. Among 11 selected regions, three enhancer candidates demonstrated interactions with the Foxf1 promoter.\"\\tEnhancer \\tFoxf1\\t3C\\t\"RT-qPCR,DNaseI-seq,Transgenic mice\"\\t\"To investigate the in vivo activity of the discovered enhanc_x0002_ers and compare the location of activity with Foxf1-express_x0002_ing regions, we generated enhancer-driven lacZ-reporter', 'E_02_299\\t27166834\\t--\\tmm10\\tchr8\\t121088830\\t121089174\\tMouse\\tMouse Embryos\\tLow+High throughput\\t\"3C,ChIP-seq\"\\t\"To identify the presence of physical interaction between the 11 enhancer candidates and the Foxf1 promoter, 3C assays were performed. Among 11 selected regions, three enhancer candidates demonstrated interactions with the Foxf1 promoter.\"\\tEnhancer \\tFoxf1\\t3C\\t\"RT-qPCR,DNaseI-seq,Transgenic mice\"\\t\"To investigate the in vivo activity of the discovered enhanc_x0002_ers and compare the location of activity with Foxf1-express_x0002_ing regions, we generated enhancer-driven lacZ-reporter', 'E_02_300\\t26957309\\t--\\tmm10\\tchr17\\t63168191\\t63175262\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"Luciferase Reporter Assay,PCR,ChIP-seq\"\\t\"To independently test the predictive power of H2BK20ac, we again used luciferase enhancer assaysas above to test 18 genomic regions randomly chosen from among the top 20,000 H2BK20ac ChIP--seq peaks in human embryonic stem cells (H1--ESCs).\"\\tEnhancer \\tFbxl17\\t--\\tChIP-seq\\t\"As above, we noticed that some of the validated forebrain enhancers showed H2BK20ac even in the absence of H3K27ac (Fig. 2C).Notably, the top H3K4me1 peaks were less predictive of forebrain enhancer function, perhaps due to the presence of this mark at in_x0002_active but poised enhancers.Again, H2BK20ac was observed to be the most predictive mark of forebrain enhancers (Fig. 2B). As above, we noticed that some of the validated forebrain enhancers showed H2BK20ac even in the absence of H3K27ac.\"\\t\"6330576B01Rik, AI452053, BB073797, C130023C01Rik, Fbx13, Fbxo13\"\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t>2kb\\t\\t\\t\\t125778', 'E_02_301\\t26957309\\t--\\tmm10\\tchr10\\t87283358\\t87312833\\tMouse\\tEmbryonic Stem Cell\\tLow+High throughput\\t\"Luciferase Reporter Assay,PCR,ChIP-seq\"\\t\"To independently test the predictive power of H2BK20ac, we again used luciferase enhancer assaysas above to test 18 genomic regions randomly chosen from among the top 20,000 H2BK20ac ChIP--seq peaks in human embryonic stem cells (H1--ESCs).\"\\tEnhancer \\tAk157291\\t--\\tChIP-seq\\t\"As above, we noticed that some of the validated forebrain enhancers showed H2BK20ac even in the absence of H3K27ac (Fig. 2C).Notably, the top H3K4me1 peaks were less predictive of forebrain enhancer function, perhaps due to the presence of this mark at in_x0002_active but poised enhancers.Again, H2BK20ac was observed to be the most predictive mark of forebrain enhancers (Fig. 2B). As above, we noticed that some of the validated forebrain enhancers showed H2BK20ac even in the absence of H3K27ac.\"\\tAk157291\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_302\\t26787680\\tMCE\\tmm10\\tchr17\\t26829232\\t26829742\\tMouse\\tCardiac Progenitor Cell\\tLow throughput\\t\"Luciferase Reporter Assay,Transient Transfection Assay\"\\t\"The influence of desmin on nkx2.5 gene expression was monitored first with the luciferase (LUC) reporter plasmid pNKE24 (Searcy et al., 1998), containing the proximal enhancerand promoter region (PEPR) of the nkx2.5 gene and then with the LUC reporter plasmid pMCE, containing the minimal cardiac specific enhancer region(MCE) (Lien et al., 1999) in addition to the PEPR (Fig. 1A; for precise localization along the nkx2.5 gene see Fig. 4A).Co-transfection of 10T1/2 fibroblasts with pNKE24 and a desmin-expressing plasmid (desmin-ect.) resulted in a significant decrease of the LUC activity (Fig. 1B).\"\\tenhancer\\tNkx2-5\\t--\\t\"Luciferase Reporter Assay,PCR,ChIP-seq\"\\t\"The influence of desmin on nkx2.5 gene expression was monitored first with the luciferase (LUC) reporter plasmid pNKE24 (Searcy et al., 1998), containing the proximal enhancerand promoter region (PEPR) of the nkx2.5 gene and then with the LUC reporter plasmid pMCE, containing the minimal cardiac specific enhancer region(MCE) (Lien et al., 1999) in addition to the PEPR (Fig. 1A; for precise localization along the nkx2.5 gene see Fig. 4A).Co-transfection of 10T1/2 fibroblasts with pNKE24 and a desmin-expressing plasmid (desmin-ect.) resulted in a significant decrease of the LUC activity (Fig. 1B).\"\\t\"Csx,Nkx-2.5,Nkx2.5,tinman\"\\t--\\t--\\t--\\t--\\t--\\t--\\tDesmin\\tDesmin\\t\"ChIP,PCR\"\\tDesmin is a component of transcription factor complexes interacting with regulatory regions of the nkx2.5 gene at the beginning of cardiomyogenesis.\\t--\\t--\\t--\\t>2kb\\t\\t\\t\\t9177', 'E_02_303\\t26771354\\t--\\tmm10\\tchr2\\t152735054\\t152735521\\tMouse\\tEmbryonic Stem Cell\\tLow throughput\\t\"qRT-PCR,ChIP\"\\t\"To address the roles of the BMP-SMAD pathway, we performed ChIP-seq analyses (Figures 1C and S1D).SMAD1/5 were enriched in the promoter regions of Pou5f1 (which encodes OCT4) and Nanog in naive mESCs,as well as a positive control region in the Id1 promoter.Consistent with previous findings (Chen et al., 2008), the regions bound by SMAD1/5 were enriched with active enhancer marks (H3K4me1, H3K27ac, and co-activator p300).\"Enhancer\\tId1\\t--\\tChIP-seq\\t\"Impor_x0002_tantly, expression levels of BMP target genes, Id1 and Dusp9, were comparable or even higher in mESD-EpiSCs treated with BMP-4 (24 hr) than in mESCs, suggesting that induction of these genes is not associated with the maintenance or reversion to the naive state.Consistent with previous findings,the regions bound by SMAD1/5 were enriched with active enhancer marks (H3K4me1, H3K27ac, and co-activator p300).\"\\t\"AI323524,D2Wsu140e,Idb1,bHLHb24\"\\t--\\t--\\t--\\t--\\t--\\t--\\tSmad1/5\\t\"AI528653, Mad1, Madh1, Madr1, Mlp1, MusMLP, dwf-A, mMad1,1110051M15Rik, AI451355, Dwf-C, Madh5, MusMLP\"\\tChIP-seq\\t\"In the present study, we have performed both RNA_x0002_sequencing and SMAD1/5 genome-wide chromatin immu_x0002_noprecipitation and sequencing (ChIP-seq) analyses of mESCs in the naive or primed states, and have employed a genome editing method. We show that the BMP_x0002_SMAD pathway is dispensable for maintaining naive pluripotency.\"\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_304\\t26774474\\t--\\tmm10\\tchr13\\t43362622\\t43364444\\tMouse\\t\"C2C12,MEFs\"\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,ChIP,EMSA,Luciferase Reporter Assay,qPCR\"\\t\"Interactions of methylated PGC-1a[K779me] with the Spt-Ada_x0002_Gcn5-acetyltransferase (SAGA) complex, the Medi_x0002_ator members MED1 and MED17, and the NOP2/ Sun RNA methytransferase 7 (NSUN7) reinforce tran_x0002_scription, and are concomitant with the m5C mark on enhancer RNAs (eRNAs).\"\\tEnhancer\\tSirt5\\t--\\tqPCR\\t\"To further evaluate the physiological function of Sirt5 eRNA, we tested the lysine de_x0002_glutarylase activity of SIRT5 upon knockdown of its associated eRNA. We found that carbamoyl phosphate synthase1 (CPS1), a specific substrate of SIRT5, was glutary_x0002_lated upon depletion of Sirt5 eRNA, leading to an overall diminishment in CPS1 activity. Overall, these results suggest that eRNAs are important sensors of the meta_x0002_bolic state.\"\\t\"0610012J09Rik, 1500032M05Rik, AV001953\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Tfiid,Lxr,nrf2,Ppar\"\\t\"GTF2D1, Gtf2d, SCA17, TFIID,AU018371, LXR, RLD1, Unr1,Nrf2,4933429D07Rik, AW742785, Nr1c1, PPAR-alpha, PPARalpha, Ppar\"\\tChIP-seq\\t\"urthermore, DNA motifs overrepresented in PGC-1[K779me] fell into four dominant motif consensus classes, including the TFIID, LXR, NRF2, and the PPAR family of transcription factors (Figure 3F, left and middle), which were localized in four metabolic genes identified as specific targets of Pgc-1 by knockdown experiments (Figure 3F, right; Figure S3A): 6-phosphofructokinase (Pfkl), Sirtuin 5 (Sirt5), isocitrate dehydrogenase 3 (Idh3b), and Heme oxygenase (decycling) 2 (Hmox2).Thus, depletion of Pgc-1 led to a decrease of Pfkl, Sirt5, Idh3b, and Hmox2 expression, which was rescued with wild-type PGC-1, but not mutated PGC-1[K779R], overexpression.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1856', 'E_02_305\\t26774474\\t--\\tmm10\\tchr2\\t130280565\\t130281887\\tMouse\\t\"C2C12,MEFs\"\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,ChIP,EMSA,Luciferase Reporter Assay,qPCR\"\\t\"Interactions of methylated PGC-1a[K779me] with the Spt-Ada_x0002_Gcn5-acetyltransferase (SAGA) complex, the Medi_x0002_ator members MED1 and MED17, and the NOP2/ Sun RNA methytransferase 7 (NSUN7) reinforce tran_x0002_scription, and are concomitant with the m5C mark on enhancer RNAs (eRNAs).\"\\tEnhancer\\tIdh3b\\t--\\tqPCR\\t\"Direct knockdown studies combined with ChIP analysis revealed some bona fide targets as Pfkl,Sirt5,Idh3b,and Hmox2 genes.Although a significant pool of PGC-1a[K779me] resides within the TSS, we were able to localize a significant binding at enhancer regions that allowed the characterization of eRNAs overlapping those PGC-1a[K779me] peaks. Thus, complex ncRNA-protein interactions appear consequential to the fidelity of transcription and to the cellular metabolic fate. \"\\tC78231\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Tfiid,Lxr,nrf2,Ppar\"\\t\"GTF2D1, Gtf2d, SCA17, TFIID,AU018371, LXR, RLD1, Unr1,Nrf2,4933429D07Rik, AW742785, Nr1c1, PPAR-alpha, PPARalpha, Ppar\"\\tChIP-seq\\t\"urthermore, DNA motifs overrepresented in PGC-1[K779me] fell into four dominant motif consensus classes, including the TFIID, LXR, NRF2, and the PPAR family of transcription factors (Figure 3F, left and middle), which were localized in four metabolic genes identified as specific targets of Pgc-1 by knockdown experiments (Figure 3F, right; Figure S3A): 6-phosphofructokinase (Pfkl), Sirtuin 5 (Sirt5), isocitrate dehydrogenase 3 (Idh3b), and Heme oxygenase (decycling) 2 (Hmox2).Thus, depletion of Pgc-1 led to a decrease of Pfkl, Sirt5, Idh3b, and Hmox2 expression, which was rescued with wild-type PGC-1, but not mutated PGC-1[K779R], overexpression.\"\\t--\\t--\\t--\\t<2KB\\t\\t\\t\\t1917', 'E_02_306\\t26774474\\t--\\tmm10\\tchr16\\t4721118\\t4724819\\tMouse\\t\"C2C12,MEFs\"\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,ChIP,EMSA,Luciferase Reporter Assay,qPCR\"\\t\"Interactions of methylated PGC-1a[K779me] with the Spt-Ada_x0002_Gcn5-acetyltransferase (SAGA) complex, the Medi_x0002_ator members MED1 and MED17, and the NOP2/ Sun RNA methytransferase 7 (NSUN7) reinforce tran_x0002_scription, and are concomitant with the m5C mark on enhancer RNAs (eRNAs).\"\\tEnhancer\\tHmox2\\t--\\tqPCR\\t\"Direct knockdown studies combined with ChIP analysis revealed some bona fide targets as Pfkl,Sirt5,Idh3b,and Hmox2 genes.Although a significant pool of PGC-1a[K779me] resides within the TSS, we were able to localize a significant binding at enhancer regions that allowed the characterization of eRNAs overlapping those PGC-1a[K780me] peaks. Thus, complex ncRNA-protein interactions appear consequential to the fidelity of transcription and to the cellular metabolic fate. \"\\t\"HO-2, HO2\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Tfiid,Lxr,nrf2,Ppar\"\\t\"GTF2D1, Gtf2d, SCA17, TFIID,AU018371, LXR, RLD1, Unr1,Nrf2,4933429D07Rik, AW742785, Nr1c1, PPAR-alpha, PPARalpha, Ppar\"\\tChIP-seq\\t\"urthermore, DNA motifs overrepresented in PGC-1[K779me] fell into four dominant motif consensus classes, including the TFIID, LXR, NRF2, and the PPAR family of transcription factors (Figure 3F, left and middle), which were localized in four metabolic genes identified as specific targets of Pgc-1 by knockdown experiments (Figure 3F, right; Figure S3A): 6-phosphofructokinase (Pfkl), Sirtuin 5 (Sirt5), isocitrate dehydrogenase 3 (Idh3b), and Heme oxygenase (decycling) 2 (Hmox2).Thus, depletion of Pgc-1 led to a decrease of Pfkl, Sirt5, Idh3b, and Hmox2 expression, which was rescued with wild-type PGC-1, but not mutated PGC-1[K779R], overexpression.\"\\t--\\t--\\t--\\t>2KB\\t\\t\\t\\t3392', 'E_02_307\\t26774474\\t--\\tmm10\\tchr10\\t77989870\\t77993100\\tMouse\\t\"C2C12,MEFs\"\\tLow+High throughput\\t\"ChIP-qPCR,ChIP-seq,ChIP,EMSA,Luciferase Reporter Assay,qPCR\"\\t\"Interactions of methylated PGC-1a[K779me] with the Spt-Ada_x0002_Gcn5-acetyltransferase (SAGA) complex, the Medi_x0002_ator members MED1 and MED17, and the NOP2/ Sun RNA methytransferase 7 (NSUN7) reinforce tran_x0002_scription, and are concomitant with the m5C mark on enhancer RNAs (eRNAs).\"\\tEnhancer\\tPfkl\\t--\\tqPCR\\t\"qPCR of the Pfkl-associated eRNA and the Dlx2 enhancer-associated Evf-2 (Dlx6os1) transcript used as a negative control, after RIP with PGC-1a[K779me] antibody upon Set7/9 depletion in both Hepa 1-6 and C2C12 cells.Direct knockdown studies combined with ChIP analysis revealed some bona fide targets as Pfkl,Sirt5,Idh3b,and Hmox2 genes.Although a significant pool of PGC-1a[K779me] resides within the TSS, we were able to localize a significant binding at enhancer regions that allowed the characterization of eRNAs overlapping those PGC-1a[K780me] peaks. Thus, complex ncRNA-protein interactions appear consequential to the fidelity of transcription and to the cellular metabolic fate. \"\\t\"AA407869, ATP-PFK, PFK-B, PFK-L\"\\t--\\t--\\t--\\t--\\t--\\t--\\t\"Tfiid,Lxr,nrf2,Ppar\"\\t\"GTF2D1, Gtf2d, SCA17, TFIID,AU018371, LXR, RLD1, Unr1,Nrf2,4933429D07Rik, AW742785, Nr1c1, PPAR-alpha, PPARalpha, Ppar\"\\tChIP-seq\\t\"urthermore, DNA motifs overrepresented in PGC-1[K779me] fell into four dominant motif consensus classes, including the TFIID, LXR, NRF2, and the PPAR family of transcription factors (Figure 3F, left and middle), which were localized in four metabolic genes identified as specific targets of Pgc-1 by knockdown experiments (Figure 3F, right; Figure S3A): 6-phosphofructokinase (Pfkl), Sirtuin 5 (Sirt5), isocitrate dehydrogenase 3 (Idh3b), and Heme oxygenase (decycling) 2 (Hmox2).Thus, depletion of Pgc-1 led to a decrease of Pfkl, Sirt5, Idh3b, and Hmox2 expression, which was rescued with wild-type PGC-1, but not mutated PGC-1[K779R], overexpression.\"\\t--\\t--\\t--\\t--\\t--', 'E_02_308\\t26809507\\t--\\tmm10\\tchr4\\t115040426\\t115042426\\tMouse\\tHPC-7\\tLow+High throughput\\t\"DNaseI-seq,ChIP,ChIP-seq,Hi-C,Transgenic mice\"\\t\"Identification of putative enhancers based on ChIP-Seq data alone cannot assign distal regions to specific genes with confidence because enhancers are known.to have the ability to act over large distances, and may loop over intervening genes.39 To overcome this limitation, we made use of our CHi-C interaction list, and filtered our list of putative enhancers to only retain those that looped to the promoter regions of known regulators of HSPC function.\"\\tEnhancer\\tScl\\t--\\t\"ChIP-seq,Hi-C\"\\t\"Visualization of the interaction files together with our previously published 10 TF ChIP-Seq demonstrated specific interactions of the Scl (also known as Tal1) and Lmo2 promoters with the previously characterized enhancer regions at Scl -15 kb,+19 kb, and +40 kb, as well as Lmo2 -75 kb,-70 kb,-64 kb, and the proximal promoter (pPex).\"\\t\"Hpt,SCL/tal-1,Scl,bHLHa17,tal-1\"\\tMalignant Hematopoiesis\\t--\\t--\\t\"Scl Enhancer elements also interact with the promoter of the neighboring Pdzk1ip1 gene, consistent with previous reports suggesting that Scl and Pdzk1ip4 form a single transcriptional domain.\"\\t\"Hi-C,ChIP-seq\"\\tAnalysis of well_x0002_characterized gene loci encoding key HSPC regulators therefore suggests that the newly generated CHi-C data set represents a valuable resource to advance our understanding of transcriptional control mechanisms in HSPCs.\\t\"Cebpa,Cebpb,Fos,Myc,E2f4,Egr1,Elf1,Cbfa2t3,Jun,Ldb1,Max,Myb,Nfe2,Trp53,Rad21,Stat3\"\\t\"C/ebpalpha, CBF-A, CebpC/EBPbeta, CRP2, IL-6DBP, LAP, LIP, NF-IL6, NF-M, Nfil6D12Rfj1,c-fos,cFos,AU0167572, Niard, Nird, bHLHe39, Myc\"\\t\"DNaseI-seq,ChIP-seq,Hi-C\"\\t\"Given that HPC-7 represents one of the best in vitro models for HSPCs, we wanted to bring genomic information for these cells up to a similar level of completeness, and therefore performed ChIP-Seq experiments for a further 17 TFs (CEBPa,CEBPb, cFOS, cMYC, E2F4, EGR1, ELF1, ETO2,c-JUN,LDB1,MAX, MYB, NFE2, p53, RAD21, pSTAT1, and STAT3) as well as genome-wide DNase I hypersensitive mapping and 3 additional his_x0002_tone marks (H2AK5ac, H3K4me3, and H3K36me3).\"\\t--\\t--\\t--\\t--\\t\\t\\t\\t--', 'E_02_309\\t26883548\\t--\\tmm10\\tchr12\\t113302447\\t113305102\\tMouse\\tBlood\\tLow throughput\\t\"PCR,ChIP\"\\t\"Extracted DNA was amplified by PCR and submitted to high_x0002_throughput sequencing to evaluate SHM. As SHM in light chains is not under the 3RR control9, IgH SHM values along rearranged', 'E_02_310\\t26883548\\t--\\tmm10\\tchr12\\t113305522\\t113308796\\tMouse\\tBlood\\tLow throughput\\t\"PCR,ChIP\"\\t\"Extracted DNA was amplified by PCR and submitted to high_x0002_throughput sequencing to evaluate SHM. As SHM in light chains is not under the 3RR control9, IgH SHM values along rearranged', 'E_02_311\\t26883548\\t--\\tmm10\\tchr12\\t113314217\\t113314925\\tMouse\\tBlood\\tLow throughput\\t\"PCR,ChIP\"\\t\"Extracted DNA was amplified by PCR and submitted to high_x0002_throughput sequencing to evaluate SHM. As SHM in light chains is not under the 3RR control9, IgH SHM values along rearranged', 'E_02_312\\t26446995\\t--\\tmm10\\tchr11\\t100859351\\t100885169\\tMouse\\tMammary Epithelium\\tLow+High throughput\\t\"PCR,ChIP-seq,RT-PCR,DNase-seq\"\\tMED1 binding coincided with STAT5 occupancy. RNA Pol II binding also coincided with these sequences.\\tEnhancerStat5 \\t--\\t\"PCR,ChIP-seq,RT-PCR,DNase-seq\"\\tMED1 binding coincided with STAT5 occupancy. RNA Pol II binding also coincided with these sequences.\\t\"AA959963,STAT5\"\\t--\\t--\\t--\\t\"Transcriptome analysis identified a class of mammary-restricted genes that was particularly']\n"
          ]
        }
      ],
      "source": [
        "import re\n",
        "\n",
        "with open('ENHANCER.txt', 'r') as file:\n",
        "    # Leggi il contenuto del file\n",
        "    my_list = []\n",
        "    # Scorri le linee del file\n",
        "    for line in file:\n",
        "        # Rimuovi eventuali spazi bianchi o caratteri di nuova linea\n",
        "        line = line.strip()\n",
        "\n",
        "        # Controlla se la linea inizia con 'E_'\n",
        "        if line.startswith('E_'):\n",
        "            my_list.append(line)\n",
        "        else:\n",
        "            pass\n",
        "\n",
        "\n",
        "print(my_list)"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "final = []\n",
        "\n",
        "for item in my_list:\n",
        "  index = item.find('hg19')\n",
        "  if index != -1:\n",
        "            # Taglia la parte della stringa fino a 'chr'\n",
        "            substr_chr = item[index:]\n",
        "            parts = substr_chr.split('\\t')\n",
        "            if len(parts) >= 3:\n",
        "                # Costruisci la sottostringa finale\n",
        "                final_substring = parts[1] + ':' + parts[2] + '-' + parts[3]\n",
        "                final.append(final_substring)\n",
        "\n",
        "\n",
        "final"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "QLbVBNEPxj2j",
        "outputId": "f6d32cdf-5339-4f19-bb2e-cb2cd383ef17"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['chr6:86160197-86176777',\n",
              " 'chr11:69329684-69330954',\n",
              " 'chr2:223164458-223164688',\n",
              " 'chr2:223164846-223165065',\n",
              " 'chr6:36631690-36640853',\n",
              " 'chr6:36640690-36643070',\n",
              " 'chr21:36180372-36181392',\n",
              " 'chr8:23448386-23454886',\n",
              " 'chr8:23433011-23439511',\n",
              " 'chr8:23422136-23432636',\n",
              " 'chr8:129166547-129190290',\n",
              " 'chr5:51773619-51857504',\n",
              " 'chr13:73880690-73990596',\n",
              " 'chr6:170855462-170856787',\n",
              " 'chr13:73880413-74042621',\n",
              " 'chr13:27523026-27544353',\n",
              " 'chr20:48997377-49019434',\n",
              " 'chr7:117038917-117040917',\n",
              " 'chr7:117099048-117099450',\n",
              " 'chr7:117228051-117229440',\n",
              " 'chr8:129543949-129554294',\n",
              " 'chr8:98253530-98257713',\n",
              " 'chr8:98310330-98314427',\n",
              " 'chr17:26077792-26078792',\n",
              " 'chr3:112045517-112046117',\n",
              " 'chr6:37772587-37778315',\n",
              " 'chr13:43111372-43112072',\n",
              " 'chr17:26072592-26074592',\n",
              " 'chr3:48598405-48598558',\n",
              " 'chrx:30317939-30318739',\n",
              " 'chr6:160101620-160102231',\n",
              " 'chr6:160101890-160102231',\n",
              " 'chr16:67519201-67519833',\n",
              " 'chr16:67519609-67519833',\n",
              " 'chr16:67520417-67521049',\n",
              " 'chr16:67520417-67520638',\n",
              " 'chr16:67520620-67520851',\n",
              " 'chr6:35574362-35576362',\n",
              " 'chr6:35505138-35511492',\n",
              " 'chr6:35616283-35622638',\n",
              " 'chr6:35636622-35642872',\n",
              " 'chr10:43569296-43572684',\n",
              " 'chr17:32579684-32579693',\n",
              " 'chr19:50850114-50850982',\n",
              " 'chr1:23881789-23881796',\n",
              " 'chr22:24236392-24237411',\n",
              " 'chr18:60786011-60788011',\n",
              " 'chr2:136531405-136531505',\n",
              " 'chr12:100693355-100695355',\n",
              " 'chr12:100731355-100733355',\n",
              " 'chr10:131265598-131265656',\n",
              " 'chr11:32403489-32405489',\n",
              " 'chr6:135459326-135461326',\n",
              " 'chr12:8217947-8219702',\n",
              " 'chr12:54757502-54758950',\n",
              " 'chr17:34411749-34415000',\n",
              " 'chr17:34416207-34419359',\n",
              " 'chr17:34429460-34432613',\n",
              " 'chr2:113595983-113598617',\n",
              " 'chr2:113597789-113600339',\n",
              " 'chr5:158755252-158761634',\n",
              " 'chr5:158783552-158790360',\n",
              " 'chr5:158787331-158794352',\n",
              " 'chr1:203273730-203275395',\n",
              " 'chr1:226065482-226067482',\n",
              " 'chr1:204123719-204123725',\n",
              " 'chr21:35982223-35983306',\n",
              " 'chr21:36141738-36142821',\n",
              " 'chr2:136582836-136583741',\n",
              " 'chr1:101402363-101412724',\n",
              " 'chr1:64083756-64083861',\n",
              " 'chr1:64087305-64087660',\n",
              " 'chr20:8684437-8686556',\n",
              " 'chr20:8687417-8688741',\n",
              " 'chr20:8718344-8719933',\n",
              " 'chr1:33353796-33354196',\n",
              " 'chr8:128747315-128748315',\n",
              " 'chr8:128749415-128750415',\n",
              " 'chr8:128346390-128347994',\n",
              " 'chr8:128327941-128330615',\n",
              " 'chr8:128299198-128301336',\n",
              " 'chr8:128296257-128297326',\n",
              " 'chr21:39823183-39825183',\n",
              " 'chr17:32565596-32566796',\n",
              " 'chr20:44430820-44433428',\n",
              " 'chr20:44441255-44444472',\n",
              " 'chr20:44478646-44484559',\n",
              " 'chr20:44413081-44426646',\n",
              " 'chr20:62593247-62595247',\n",
              " 'chr4:101576220-101578659',\n",
              " 'chr17:42078932-42079032',\n",
              " 'chr12:7940709-7941994',\n",
              " 'chr17:62028378-62028924',\n",
              " 'chr8:117978845-117979466',\n",
              " 'chr5:134350424-134352424',\n",
              " 'chr6:31130679-31130761',\n",
              " 'chr6:31130445-31130606',\n",
              " 'chr6:31129568-31129735',\n",
              " 'chr19:51354804-51355203',\n",
              " 'chr21:42822120-42822800',\n",
              " 'chr20:55417234-55417304',\n",
              " 'chr9:132491985-132492534',\n",
              " 'chr14:90847327-90849327',\n",
              " 'chr5:44385749-44386845',\n",
              " 'chr11:101981958-101982137',\n",
              " 'chr22:35766960-35768960',\n",
              " 'chr17:42078865-42079198',\n",
              " 'chr9:136141944-136147321',\n",
              " 'chr12:8036824-8048824',\n",
              " 'chr12:8035824-8037824',\n",
              " 'chr17:69107686-69108045',\n",
              " 'chr17:69108466-69108765',\n",
              " 'chr8:124323890-124325890',\n",
              " 'chr8:124321890-124323890',\n",
              " 'chr8:124319890-124321890',\n",
              " 'chr17:48258757-48260757',\n",
              " 'chr1:24601612-24626812',\n",
              " 'chr15:90288698-90290698',\n",
              " 'chr1:181104024-181104549',\n",
              " 'chr9 :118706130-118707280',\n",
              " 'chr11:8301851-8303851',\n",
              " 'chr9:21816751-21818751',\n",
              " 'chr2:191834162-191834795',\n",
              " 'chr12:48369059-48369213',\n",
              " 'chr11:72897942-72912333',\n",
              " 'chr5:172648612-172649177',\n",
              " 'chr12:21439733-21439973',\n",
              " 'chr12:68549287-68549545',\n",
              " 'chr12:68549796-68550055',\n",
              " 'chr12:68553503-68553754',\n",
              " 'chr12:68553762-68554012',\n",
              " 'chr21:43770915-43772915',\n",
              " 'chr6:161119825-161121825',\n",
              " 'chr6:161110725-161112725',\n",
              " 'chr12:48368959-48369159',\n",
              " 'chr8:57069398-57069482',\n",
              " 'chr19:57566145-57571006',\n",
              " 'chr1:1083900-1085900',\n",
              " 'chr10:23502416-23510031',\n",
              " 'chr11:111150916-111155431',\n",
              " 'chr8:127047315-127049315',\n",
              " 'chr21:43781591-43782591',\n",
              " 'chr21:43772291-43774291',\n",
              " 'chr21:43771491-43773491',\n",
              " 'chr17:72860066-72860183',\n",
              " 'chr11:110716521-110832306',\n",
              " 'chr3:188892648-189232665',\n",
              " 'chr1:226085596-226087967',\n",
              " 'chr14:77264349-77265491',\n",
              " 'chr15:37116298-37122452',\n",
              " 'chr13:95865319-95866199',\n",
              " 'chr8:128709135-128745673',\n",
              " 'chr8:128392788-128433173',\n",
              " 'chr4:74586223-74591223',\n",
              " 'chr8:128230172-128243000',\n",
              " 'chr8:119677099-119681977',\n",
              " 'chr8:141161900-141175436',\n",
              " 'chr17:60983452-60984775',\n",
              " 'chr17:60986269-60989232',\n",
              " 'chr8:128816178-128817335',\n",
              " 'chr5:139970046-139972237',\n",
              " 'chr5:139974521-139975981',\n",
              " 'chr5:139985479-139987671',\n",
              " 'chr5:139987854-139989680',\n",
              " 'chr5:139990822-139992831',\n",
              " 'chr5:140009726-140011735',\n",
              " 'chr1:9996966-9997243',\n",
              " 'chr3:11249688-11250280',\n",
              " 'chr3:184276287-184278287',\n",
              " 'chr19:51358995-51369041',\n",
              " 'chr12:54357811-54358445',\n",
              " 'chr6:36632237-36634237',\n",
              " 'chr8:143896217-143898217',\n",
              " 'chr16:86212040-86271919',\n",
              " 'chr9:124638991-124641160',\n",
              " 'chr3:32993199-32996133',\n",
              " 'chr3:33000399-33003599',\n",
              " 'chr3:34181595-34184395',\n",
              " 'chr3:34201595-34204662',\n",
              " 'chr12:48368875-48369590',\n",
              " 'chr8:130179643-130181490',\n",
              " 'chr10:6064657-6065397',\n",
              " 'chr2:217918268-217921910',\n",
              " 'chr7:117075514-117075933',\n",
              " 'chr19:2482923-2484923',\n",
              " 'chr10:43568986-43571244',\n",
              " 'chr10:43571244-43572713',\n",
              " 'chr7:94002773-94005073',\n",
              " 'chr22:42411333-42413643',\n",
              " 'chr22:42411544-42415106',\n",
              " 'chr22:42411769-42414470',\n",
              " 'chr1:98509888-98511451',\n",
              " 'chr1:98514701-98515055',\n",
              " 'chr1:98516486-98522430',\n",
              " 'chr1:98517055-98517909',\n",
              " 'chr12:98559923-98563615',\n",
              " 'chr12:98562692-98566384',\n",
              " 'chr14:97410723-97443618',\n",
              " 'chr1:23881244-23881761',\n",
              " 'chr1:23888938-23889083',\n",
              " 'chr20:44637014-44637468',\n",
              " 'chr17:69487560-69491400',\n",
              " 'chr17:61927353-61928827',\n",
              " 'chr17:6285838-6586838',\n",
              " 'chr17:6565038-6567038',\n",
              " 'chr5:125345378-125345495',\n",
              " 'chr5:125995862-125995972',\n",
              " 'chr10:123099588-123100426',\n",
              " 'chr17:54754755-54758235',\n",
              " 'chr17:54775600-54778000',\n",
              " 'chr6:45410618-45417584',\n",
              " 'chr7:155845627-155847652',\n",
              " 'chr11:118767466-118769582',\n",
              " 'chr8:81286250-81293977',\n",
              " 'chr1:214150511-214151337',\n",
              " 'chr10:64737032-64737909',\n",
              " 'chr2:98362566-98364423',\n",
              " 'chr3:186938165-186940165',\n",
              " 'chr3:187088165-187909165',\n",
              " 'chr3:187559165-187589165',\n",
              " 'chr20:42977441-42978040',\n",
              " 'chr4:56895793-56962351',\n",
              " 'chr7:114541370-114542201',\n",
              " 'chr6:1614188-1617051',\n",
              " 'chr6:1615843-1617834',\n",
              " 'chr6:1619348-1621343',\n",
              " 'chr6:1701730-1704441',\n",
              " 'chr12:56728500-56735000',\n",
              " 'chr6:30904342-30904788',\n",
              " 'chr5:148785000-148815000',\n",
              " 'chr22:35686840-35687280',\n",
              " 'chr11:5301600-5302400',\n",
              " 'chr11:8271697-8279492',\n",
              " 'chr4:26075179-26135897',\n",
              " 'chr11:118540532-118575295',\n",
              " 'chr7:117011495-117063767',\n",
              " 'chr7:117282955-117335227',\n",
              " 'chr8:130386715-130591515',\n",
              " 'chr1:181117806-181122451',\n",
              " 'chr6:31139599-31139921',\n",
              " 'chr6:30646843-30647217',\n",
              " 'chr18:60786011-60788011',\n",
              " 'chr6:31126537-31126685',\n",
              " 'chr8:128412386-128415392',\n",
              " 'chr13:40839693-40853195',\n",
              " 'chr9:4600913-4608373',\n",
              " 'chr11:82608774-82612886',\n",
              " 'chr20:54937784-54938182',\n",
              " 'chr1:183149939-183150336',\n",
              " 'chr22:20224442-20228467',\n",
              " 'chr14:77635927-77639784',\n",
              " 'chr11:2219531-2239062',\n",
              " 'chr11:72905735-72906735',\n",
              " 'chr6:1636681-1642781',\n",
              " 'ch3:189109942-189219942',\n",
              " 'ch3:189203942-189313942',\n",
              " 'ch1:24589812-24645812',\n",
              " 'chr17:69541883-69542989',\n",
              " 'chr7:150360500-150366500',\n",
              " 'chr3:46988575-46990242',\n",
              " 'chr12:109986000-109986710',\n",
              " 'chrX:55054584-55054678',\n",
              " 'chrX:55041578-55041679',\n",
              " 'chr1:92921568-92937818',\n",
              " 'chr3:181572712-181574712',\n",
              " 'chr11:8240851-8242851',\n",
              " 'chr9:97396388-97401553',\n",
              " 'chr4:174441057-174442400',\n",
              " 'chr3:107702615-107714185',\n",
              " 'chr3:107842838-107845592',\n",
              " 'chr3:107846969-107854683',\n",
              " 'chr11:17666830-17677080',\n",
              " 'chr11:355447-358949',\n",
              " 'chr1:221049243-221051243',\n",
              " 'chr18:61533339-61545796',\n",
              " 'Chr11:45846943-45846959',\n",
              " 'chr14:60973427-60975430',\n",
              " 'chr3:181432912-181433312',\n",
              " 'chr7:129854050-129895632',\n",
              " 'chr6:12490000-12590000',\n",
              " 'chr4:77511788-77605824',\n",
              " 'chr4:94419116-94437803',\n",
              " 'chr12:56330413-56330909',\n",
              " 'chr6:31092587-31094587',\n",
              " 'chr8:27455903-27456103',\n",
              " 'chr4:23811733-23994829',\n",
              " 'chr17:37879571-37880263',\n",
              " 'chr3:180728712-180730712',\n",
              " 'chr6:91004796-91006944',\n",
              " 'chr7:94002773-94005073',\n",
              " 'chr12:47471429-47487762',\n",
              " 'chr19:34845032-34847032',\n",
              " 'chr13:28545205-28547205',\n",
              " 'chr16:68777928-68779928',\n",
              " 'chr3:184276287-184278287',\n",
              " 'chr4:55859107-55860311',\n",
              " 'chr10:63525013-63527013',\n",
              " 'chr14:56245000-56285000',\n",
              " 'chr2:11633299-11641134',\n",
              " 'chr2:8818311-8820522',\n",
              " 'chr4:174435828-174438057',\n",
              " 'chr3:107662655-107762655',\n",
              " 'chrx:106959618-106962917',\n",
              " 'chr3:69863244-70187654',\n",
              " 'chr14:99770698-99782638',\n",
              " 'chr1:116896120-116904325',\n",
              " 'chr16:74568579-74583481',\n",
              " 'chr6:138226720-138233280',\n",
              " 'chr7:45920458-45923258',\n",
              " 'chr15:66074477-66098025',\n",
              " 'chr17:31209116-31240452',\n",
              " 'chr4:84277627-84281723',\n",
              " 'chr3:150450617-150456921',\n",
              " 'chr6:1679010-1683521',\n",
              " 'chr8:128206162-128314310',\n",
              " 'chr8:129046628-129096067',\n",
              " 'chr8:130569971-130616320',\n",
              " 'chr8:130693569-130739918',\n",
              " 'chr6:31137269-31137697',\n",
              " 'chr5:372500-376000',\n",
              " 'chr21:40003867-40015913',\n",
              " 'chr3:36999364-37001216',\n",
              " 'chr15:96725740-96737960',\n",
              " 'chr12:132285987-132290767',\n",
              " 'chr12:132281970-132287430',\n",
              " 'chr12:132342570-132347690',\n",
              " 'chr1:160536893-160537651',\n",
              " 'chr1:160542651-160543257',\n",
              " 'chr5:149457542-149458807',\n",
              " 'chr11:5279732-5280272',\n",
              " 'chr8:38291071-38304326',\n",
              " 'chr16:86422446-86426834',\n",
              " 'chr20:30244800-30316534',\n",
              " 'chr6:37997692-38001538',\n",
              " 'chr6:38010769-38019538',\n",
              " 'chr6:38032307-38035230',\n",
              " 'chr6:151917241-152028736',\n",
              " 'chr2:234665349-234665739',\n",
              " 'chr14:76419740-76421740',\n",
              " 'chr1:47704791-47705107',\n",
              " 'chr19:15308673-15308842',\n",
              " 'chr10:33492690-33519530',\n",
              " 'chr10:33512793-33539633',\n",
              " 'chr10:33637535-33664375',\n",
              " 'chr14:106327635-106328903',\n",
              " 'chr9:126900000-126900625',\n",
              " 'chr9:126964375-126965000',\n",
              " 'chr9:126980625-126981125',\n",
              " 'chr9:127001250-127001875',\n",
              " 'chr9:127047500-127048375',\n",
              " 'chr6:31139519-31140167',\n",
              " 'chr6:31137056-31138028',\n",
              " 'chr6:31140556-31141204',\n",
              " 'chr14:105051517-105351516',\n",
              " 'chr6:35628362-35641362',\n",
              " 'chr10:43569045-43569456',\n",
              " 'chr12:114463712-114464080',\n",
              " 'chr12:114701207-114704691',\n",
              " 'chr12:114853271-114858238',\n",
              " 'chr11:69468761-69470761',\n",
              " 'chr11:69468873-69470873',\n",
              " 'chr7:116374406-116376406',\n",
              " 'chr11:47363009-47364009',\n",
              " 'chr20:21452815-21462458',\n",
              " 'chr21:42668000-42671000',\n",
              " 'chr22:35736840-35737280',\n",
              " 'chr9:14538685-14540685',\n",
              " 'chr9:14923979-14925979',\n",
              " 'chr9:15066136-15068136',\n",
              " 'chr1:92170642-92172642',\n",
              " 'chr1:92254801-92256801',\n",
              " 'chr1:92331533-92333533',\n",
              " 'chr1:92359751-92361751',\n",
              " 'chr1:92392919-92394919',\n",
              " 'chr1:92567177-92569177',\n",
              " 'chr6:43891400-43898360',\n",
              " 'chr1:67739940-67741075',\n",
              " 'chr5:110175800-110176600',\n",
              " 'chr3:108015219-108015290',\n",
              " 'chr11:109985003-109985222',\n",
              " 'chr7:150700000-150705000',\n",
              " 'chr6:36640937-36642937',\n",
              " 'chr6:36647237-36649237',\n",
              " 'chr11:5288664-5288684',\n",
              " 'chr11:5268575-5268621',\n",
              " 'chr11:5266021-5266708',\n",
              " 'chr16:80617-80837',\n",
              " 'chr11:5248824-5249485',\n",
              " 'chr11:5255062-5258441',\n",
              " 'chr3:108296550-108297750',\n",
              " 'chr11:5250264-5250295',\n",
              " 'chr11:5250535-5250560',\n",
              " 'chr11:5248027-5248047',\n",
              " 'chr11:5250704-5250719',\n",
              " 'chr11:5250751-5250777',\n",
              " 'chr11:5293686-5294250',\n",
              " 'chr11:5211019-5213983',\n",
              " 'chr11:5264845-5266345',\n",
              " 'chr11:5271421-5273421',\n",
              " 'chr11:5286080-5288080',\n",
              " 'chr5:110175120-110177136',\n",
              " 'chr11:95817801-95818824',\n",
              " 'chr1:67739940-67741075',\n",
              " 'chr3:78597238-79767998',\n",
              " 'chr9:91737604-92057238',\n",
              " 'chr19:33299481-33300688',\n",
              " 'chr2:202120390-202125986',\n",
              " 'chr12:131949637-131949801',\n",
              " 'chr8:128310315-128312315',\n",
              " 'chr2:29511793-29765640',\n",
              " 'chr5:893433-1083433',\n",
              " 'chr21:47716490-47892045',\n",
              " 'chr2:221040922-221069549',\n",
              " 'chr5:121568382-121839811',\n",
              " 'chr1:230293530-230305966',\n",
              " 'chr22:18897006-18899006',\n",
              " 'chr5:172196326-172199378',\n",
              " 'chr8:29203575-29214287',\n",
              " 'chr14:24435744-24437744',\n",
              " 'chr14:61121917-61123917',\n",
              " 'chr14:61106417-61108417',\n",
              " 'chr8:140657900-141002079',\n",
              " 'chr7:95210132-95260132',\n",
              " 'chr1:153628434-153632039']"
            ]
          },
          "metadata": {},
          "execution_count": 43
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "\n",
        "with open('LISTA_INDICI_HG19_DEF.txt', 'w') as file:\n",
        "    # Scrivi ogni elemento della lista nel file\n",
        "    for item in final:\n",
        "        file.write(item + '\\n')"
      ],
      "metadata": {
        "id": "_0D-thV6zqBj"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "final2 = []\n",
        "\n",
        "for item in my_list:\n",
        "  index = item.find('mm10')\n",
        "  if index != -1:\n",
        "            # Taglia la parte della stringa fino a 'chr'\n",
        "            substr_chr = item[index:]\n",
        "            parts = substr_chr.split('\\t')\n",
        "            if len(parts) >= 3:\n",
        "                # Costruisci la sottostringa finale\n",
        "                final_substring = parts[0] + '\\t' + parts[1] + '\\t' + parts[2] + '\\t' + parts[3]\n",
        "                final2.append(final_substring)\n",
        "\n",
        "\n",
        "final2"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "9FhLjfhv2QRC",
        "outputId": "34cdc59e-978d-4fa9-d2fe-3f572106861d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['mm10\\tchr10\\t52388022\\t52388224',\n",
              " 'mm10\\tchr7\\t126406352\\t126406616',\n",
              " 'mm10\\tchr3\\t51508734\\t51512187',\n",
              " 'mm10\\tchr15\\t97977739\\t97977942',\n",
              " 'mm10\\tchr8\\t105528423\\t105529045',\n",
              " 'mm10\\tchr4\\t44520195\\t44520234',\n",
              " 'mm10\\tchr3\\t37225520\\t37228865',\n",
              " 'mm10\\tchr6\\t124718594\\t124718929',\n",
              " 'mm10\\tchr13\\t53463266\\t53463541',\n",
              " 'mm10\\tchr17\\t44494486\\t44495486',\n",
              " 'mm10\\tchr17\\t44494598\\t44494816',\n",
              " 'mm10\\tchr15\\t52315678\\t52316298',\n",
              " 'mm10\\tchr4\\t136140539\\t136140546',\n",
              " 'mm10\\tchr3\\t34563695\\t34565695',\n",
              " 'mm10\\tchr14\\t67742375\\t67743103',\n",
              " 'mm10\\tchr7\\t103823354\\t103846754',\n",
              " 'mm10\\tchrx\\t7954360\\t7956360',\n",
              " 'mm10\\tchr5\\t115358083\\t115360083',\n",
              " 'mm10\\tchr14\\t78384835\\t78446843',\n",
              " 'mm10\\tchr14\\t78351421\\t78354313',\n",
              " 'mm10\\tchr14\\t67745496\\t67746000',\n",
              " 'mm10\\tchr18\\t61099404\\t61099740',\n",
              " 'mm10\\tchr11\\t44390342\\t44392251',\n",
              " 'mm10\\tchr5\\t147267614\\t147267639',\n",
              " 'mm10\\tchr5\\t147267670\\t147267721',\n",
              " 'mm10\\tchr5\\t147267701\\t147267751',\n",
              " 'mm10\\tchr14\\t55518040\\t55518674',\n",
              " 'mm10\\tchr7\\t19409838\\t19410044',\n",
              " 'mm10\\tchr15\\t52315678\\t52316298',\n",
              " 'mm10\\tchr15\\t5249678\\t5250298',\n",
              " 'mm10\\tchr3\\t34646162\\t34646551',\n",
              " 'mm10\\tchr7\\t24461573\\t24462573',\n",
              " 'mm10\\tchr1\\t38469166\\t38512500',\n",
              " 'mm10\\tchr11\\t32244715\\t32246895',\n",
              " 'mm10\\tchr11\\t32249976\\t32252156',\n",
              " 'mm10\\tchr11\\t32255189\\t32257369',\n",
              " 'mm10\\tchr11\\t32268128\\t32270308',\n",
              " 'mm10\\tchr11\\t32264052\\t32266232',\n",
              " 'mm10\\tchr6\\t70763764\\t70765090',\n",
              " 'mm10\\tchr6\\t70725576\\t70726090',\n",
              " 'mm10\\tchr6\\t70743710\\t70744950',\n",
              " 'mm10\\tchr6\\t70735256\\t70736064',\n",
              " 'mm10\\tchr10\\t62128342\\t62130342',\n",
              " 'mm10\\tchr4\\t115100426\\t115102426',\n",
              " 'mm10\\tchr4\\t115074426\\t115076426',\n",
              " 'mm10\\tchr14\\t47056226\\t47056692',\n",
              " 'mm10\\tchr9\\t119469800\\t119469811',\n",
              " 'mm10\\tchr9\\t119469879\\t119469890',\n",
              " 'mm10\\tchr9\\t119469902\\t119469913',\n",
              " 'mm10\\tchr14\\t67741735\\t67744661',\n",
              " 'mm10\\tchr14\\t67743318\\t67743610',\n",
              " 'mm10\\tchr14\\t67742046\\t67742550',\n",
              " 'mm10\\tchr14\\t67740982\\t67741286',\n",
              " 'mm10\\tchr1\\t60908875\\t60908895',\n",
              " 'mm10\\tchr11\\t44390063\\t44390123',\n",
              " 'mm10\\tchr17\\t26829229\\t26831311',\n",
              " 'mm10\\tchr6\\t122726952\\t122727236',\n",
              " 'mm10\\tchr6\\t122727606\\t122727632',\n",
              " 'mm10\\tchr4\\t115095426\\t115097426',\n",
              " 'mm10\\tchr7\\t46352720\\t46353957',\n",
              " 'mm10\\tchr7\\t46370375\\t46374302',\n",
              " 'mm10\\tchr17\\t40206839\\t40206900',\n",
              " 'mm10\\tchr5\\t107680655\\t107682655',\n",
              " 'mm10\\tchr7\\t6502892\\t6504892',\n",
              " 'mm10\\tchr5\\t28476034\\t28479024',\n",
              " 'mm10\\tchr5\\t28495262\\t28497421',\n",
              " 'mm10\\tchr5\\t28500370\\t28502363',\n",
              " 'mm10\\tchr5\\t28561915\\t28564905',\n",
              " 'mm10\\tchr19\\t10215891\\t10216703',\n",
              " 'mm10\\tchr2\\t119321593\\t119322529',\n",
              " 'mm10\\tchr2\\t119321593\\t119322529',\n",
              " 'mm10\\tchr11\\t111838889\\t111839334',\n",
              " 'mm10\\tchr16\\t59471850\\t59472124',\n",
              " 'mm10\\tchr6\\t115409226\\t115436116',\n",
              " 'mm10\\tchr6\\t115507545\\t115533315',\n",
              " 'mm10\\tchr2\\t115794340\\t115800495',\n",
              " 'mm10\\tchr6\\t52296135\\t52334522',\n",
              " 'mm10\\tchr6\\t52585919\\t52613016',\n",
              " 'mm10\\tchr6\\t52765812\\t52891511',\n",
              " 'mm10\\tchr6\\t52908070\\t53037532',\n",
              " 'mm10\\tchr4\\t147962887\\t147964479',\n",
              " 'mm10\\tchr7\\t100474191\\t100474841',\n",
              " 'mm10\\tchr17\\t44464986\\t44466986',\n",
              " 'mm10\\tchr6\\t52199038\\t52200503',\n",
              " 'mm10\\tchr1\\t133347808\\t133348049',\n",
              " 'mm10\\tchr16\\t23143036\\t23145036',\n",
              " 'mm10\\tchr4\\t124638991\\t124641160',\n",
              " 'mm10\\tchr4\\t124645999\\t124647927',\n",
              " 'mm10\\tchr5\\t119677099\\t119681977',\n",
              " 'mm10\\tchr5\\t119704124\\t119707831',\n",
              " 'mm10\\tchr5\\t119707931\\t119710758',\n",
              " 'mm10\\tchr5\\t119710660\\t119714758',\n",
              " 'mm10\\tchr5\\t98290964\\t98295647',\n",
              " 'mm10\\tchr5\\t98307793\\t98312281',\n",
              " 'mm10\\tchr5\\t98310330\\t98314427',\n",
              " 'mm10\\tchr5\\t98253530\\t98257713',\n",
              " 'mm10\\tchr2\\t25423079\\t25423874',\n",
              " 'mm10\\tchr17\\t31172998\\t31173656',\n",
              " 'mm10\\tchr17\\t45890060\\t45890829',\n",
              " 'mm10\\tchrx\\t140806807\\t140808327',\n",
              " 'mm10\\tchr2\\t91067374\\t91069374',\n",
              " 'mm10\\tchr17\\t35502632\\t35504632',\n",
              " 'mm10\\tchr6\\t122700483\\t122702483',\n",
              " 'mm10\\tchr9\\t99131669\\t99132862',\n",
              " 'mm10\\tchr1\\t182851627\\t182852853',\n",
              " 'mm10\\tchr1\\t182852133\\t182854320',\n",
              " 'mm10\\tchr1\\t182856227\\t182857187',\n",
              " 'mm10\\tchr11\\t35178343\\t35180368',\n",
              " 'mm10\\tchr11\\t35813868\\t35814943',\n",
              " 'mm10\\tchr11\\t102006458\\t102008458',\n",
              " 'mm10\\tchr10\\t34385685\\t34385834',\n",
              " 'mm10\\tchr15\\t63254341\\t63256341',\n",
              " 'mm10\\tchr7\\t53527011\\t53528648',\n",
              " 'mm10\\tchr2\\t167687771\\t167689503',\n",
              " 'mm10\\tchr4\\t66837166\\t66852551',\n",
              " 'mm10\\tchr8\\t120737580\\t120746913',\n",
              " 'mm10\\tchr1\\t133303739\\t133325138',\n",
              " 'mm10\\tchr3\\t34743178\\t34776096',\n",
              " 'mm10\\tchr11\\t98979280\\t98979880',\n",
              " 'mm10\\tchr14\\t63196014\\t63196614',\n",
              " 'mm10\\tchr4\\t136148484\\t136149239',\n",
              " 'mm10\\tchr4\\t136140654\\t136141162',\n",
              " 'mm10\\tchr6\\t71341692\\t71345212',\n",
              " 'mm10\\tchr10\\t62893206\\t62923461',\n",
              " 'mm10\\tchr3\\t133531418\\t133563926',\n",
              " 'mm10\\tchr12\\t3944001\\t3947078',\n",
              " 'mm10\\tchr12\\t3956137\\t3961266',\n",
              " 'mm10\\tchr12\\t3938522\\t3942093',\n",
              " 'mm10\\tchr8\\t83287848\\t83288068',\n",
              " 'mm10\\tchr7\\t46351197\\t46355693',\n",
              " 'mm10\\tchr4\\t135548688\\t135550688',\n",
              " 'mm10\\tchr8\\t106613868\\t106615868',\n",
              " 'mm10\\tchr8\\t109134068\\t109136068',\n",
              " 'mm10\\tchr8\\t106610150\\t106612150',\n",
              " 'mm10\\tchr17\\t35499359\\t35507923',\n",
              " 'mm10\\tchr5\\t119654061\\t119663588',\n",
              " 'mm10\\tchr1\\t13041306\\t13127367',\n",
              " 'mm10\\tchr17\\t31794375\\t31829875',\n",
              " 'mm10\\tchr17\\t35501272\\t35506872',\n",
              " 'mm10\\tchr7\\t3183157\\t3229407',\n",
              " 'mm10\\tchr3\\t127641624\\t127642545',\n",
              " 'mm10\\tchr4\\t34888479\\t34889907',\n",
              " 'mm10\\tchr14\\t50787449\\t50787634',\n",
              " 'mm10\\tchr12\\t25090158\\t25090663',\n",
              " 'mm10\\tchr8\\t84899088\\t84902859',\n",
              " 'mm10\\tchr13\\t83661050\\t83662862',\n",
              " 'mm10\\tchr8\\t59795280\\t59797080',\n",
              " 'mm10\\tchr10\\t78046488\\t78046636',\n",
              " 'mm10\\tchr16\\t49740654\\t49742654',\n",
              " 'mm10\\tchr10\\t61403948\\t61408233',\n",
              " 'mm10\\tchr10\\t61402781\\t61405448',\n",
              " 'mm10\\tchr4\\t29115647\\t29140947',\n",
              " 'mm10\\tchr4\\t29119847\\t29120047',\n",
              " 'mm10\\tchr4\\t55565908\\t55608014',\n",
              " 'mm10\\tchr6\\t122653828\\t122675804',\n",
              " 'mm10\\tchr6\\t122696853\\t122708211',\n",
              " 'mm10\\tchr12\\t85433906\\t85439068',\n",
              " 'mm10\\tchr12\\t85452549\\t85458571',\n",
              " 'mm10\\tchr12\\t85466416\\t85469570',\n",
              " 'mm10\\tchr12\\t85470646\\t85474086',\n",
              " 'mm10\\tchr12\\t85482048\\t85490068',\n",
              " 'mm10\\tchr2\\t168744572\\t168752225',\n",
              " 'mm10\\tchr8\\t89041889\\t89090124',\n",
              " 'mm10\\tchr12\\t85515101\\t85543901',\n",
              " 'mm10\\tchr14\\t54227135\\t54228033',\n",
              " 'mm10\\tchr6\\t71321750\\t71323696',\n",
              " 'mm10\\tchr2\\t172513228\\t172520500',\n",
              " 'mm10\\tchr2\\t172541319\\t172544955',\n",
              " 'mm10\\tchr2\\t172574046\\t172577682',\n",
              " 'mm10\\tchr14\\t69196226\\t69202226',\n",
              " 'mm10\\tchr14\\t119710660\\t119714758',\n",
              " 'mm10\\tchr14\\t69221635\\t69222851',\n",
              " 'mm10\\tchr2\\t10136078\\t10138078',\n",
              " 'mm10\\tchr7\\t35156353\\t35161233',\n",
              " 'mm10\\tchr7\\t129554284\\t129564284',\n",
              " 'mm10\\tchr9\\t78368337\\t78376479',\n",
              " 'mm10\\tchr5\\t137465620\\t137479420',\n",
              " 'mm10\\tchr15\\t63833002\\t63863200',\n",
              " 'mm10\\tchr15\\t63865799\\t63896492',\n",
              " 'mm10\\tchr15\\t63894512\\t63932136',\n",
              " 'mm10\\tchr18\\t69656860\\t69683465',\n",
              " 'mm10\\tchr11\\t112769210\\t112772210',\n",
              " 'mm10\\tchr12\\t25090158\\t25096935',\n",
              " 'mm10\\tchr2\\t105072843\\t105076843',\n",
              " 'mm10\\tchr15\\t97898854\\t97902854',\n",
              " 'mm10\\tchr15\\t97911426\\t97915997',\n",
              " 'mm10\\tchr6\\t88202011\\t88204330',\n",
              " 'mm10\\tchr11\\t44386118\\t44393090',\n",
              " 'mm10\\tchr4\\t97123309\\t97712957',\n",
              " 'mm10\\tchr5\\t9034150\\t9036900',\n",
              " 'mm10\\tchr5\\t9037806\\t9041881',\n",
              " 'mm10\\tchr5\\t9038118\\t9038593',\n",
              " 'mm10\\tchr1\\t136170878\\t136192613',\n",
              " 'mm10\\tchr17\\t62627957\\t62633457',\n",
              " 'mm10\\tchr9\\t88325101\\t88326201',\n",
              " 'mm10\\tchr8\\t36785613\\t36985613',\n",
              " 'mm10\\tchr10\\t67853705\\t67855705',\n",
              " 'mm10\\tchr10\\t67926705\\t67928705',\n",
              " 'mm10\\tchr6\\t4438697\\t4440197',\n",
              " 'mm10\\tchr4\\t55449628\\t55461828',\n",
              " 'mm10\\tchr4\\t55453628\\t55465828',\n",
              " 'mm10\\tchr16\\t92625802\\t92627802',\n",
              " 'mm10\\tchr8\\t11564419\\t11614821',\n",
              " 'mm10\\tchr14\\t18175266\\t18186392',\n",
              " 'mm10\\tchr2\\t122651534\\t122668846',\n",
              " 'mm10\\tchr2\\t103887795\\t103888195',\n",
              " 'mm10\\tchr2\\t103886995\\t103888995',\n",
              " 'mm10\\tchr1\\t131878852\\t131942815',\n",
              " 'mm10\\tchr1\\t131920349\\t131962471',\n",
              " 'mm10\\tchr10\\t24357349\\t24375452',\n",
              " 'mm10\\tchr10\\t24386336\\t24406594',\n",
              " 'mm10\\tchr10\\t24449590\\t24469418',\n",
              " 'mm10\\tchr10\\t24483642\\t24503469',\n",
              " 'mm10\\tchr10\\t21410566\\t21412861',\n",
              " 'mm10\\tchr11\\t19833945\\t19851027',\n",
              " 'mm10\\tchr17\\t45790442\\t45800636',\n",
              " 'mm10\\tchr10\\t50623698\\t50625698',\n",
              " 'mm10\\tchr2\\t75120051\\t75165771',\n",
              " 'mm10\\tchr2\\t75413472\\t75451553',\n",
              " 'mm10\\tchr2\\t126601468\\t126603468',\n",
              " 'mm10\\tchr2\\t126593905\\t126595430',\n",
              " 'mm10\\tchr7\\t143459172\\t143459787',\n",
              " 'mm10\\tchr7\\t143462624\\t143463585',\n",
              " 'mm10\\tchr15\\t63714919\\t63952099',\n",
              " 'mm10\\tchr1\\t133347574\\t133364574',\n",
              " 'mm10\\tchr19\\t61151074\\t61161722',\n",
              " 'mm10\\tchr19\\t61158643\\t61164569',\n",
              " 'mm10\\tchr19\\t61165425\\t61172555',\n",
              " 'mm10\\tchr2\\t144000110\\t144014615',\n",
              " 'mm10\\tchr2\\t144014506\\t144023297',\n",
              " 'mm10\\tchr2\\t103464661\\t103471206',\n",
              " 'mm10\\tchr6\\t127033997\\t127036097',\n",
              " 'mm10\\tchr11\\t112217210\\t112217767',\n",
              " 'mm10\\tchr17\\t85527960\\t85530660',\n",
              " 'mm10\\tchr17\\t85463916\\t85466386',\n",
              " 'mm10\\tchr17\\t85392908\\t85395234',\n",
              " 'mm10\\tchr7\\t46371356\\t46373481',\n",
              " 'mm10\\tchr13\\t56243248\\t56243833',\n",
              " 'mm10\\tchr13\\t56244114\\t56244357',\n",
              " 'mm10\\tchr13\\t56245174\\t56245857',\n",
              " 'mm10\\tchr13\\t30108722\\t30109111',\n",
              " 'mm10\\tchr13\\t30123333\\t30124861',\n",
              " 'mm10\\tchr13\\t30144166\\t30145833',\n",
              " 'mm10\\tchr8\\t106608533\\t106609800',\n",
              " 'mm10\\tchr8\\t106610533\\t106611666',\n",
              " 'mm10\\tchr11\\t6639091\\t6639657',\n",
              " 'mm10\\tchr11\\t6639824\\t6640424',\n",
              " 'mm10\\tchr11\\t6644257\\t6644824',\n",
              " 'mm10\\tchr11\\t6639029\\t6644397',\n",
              " 'mm10\\tchr2\\t9838078\\t9840078',\n",
              " 'mm10\\tchr16\\t10440279\\t10442279',\n",
              " 'mm10\\tchr11\\t100854851\\t100856851',\n",
              " 'mm10\\tchr12\\t114640000\\t114670000',\n",
              " 'mm10\\tchr12\\t114460000\\t114490000',\n",
              " 'mm10\\tchr17\\t85795937\\t85796217',\n",
              " 'mm10\\tchr17\\t85797325\\t85797927',\n",
              " 'mm10\\tchr17\\t30249928\\t30278499',\n",
              " 'mm10\\tchr6\\t98336842\\t98365566',\n",
              " 'mm10\\tchr12\\t35939690\\t35990427',\n",
              " 'mm10\\tchr12\\t35841201\\t35921783',\n",
              " 'mm10\\tchr12\\t35769572\\t35823293',\n",
              " 'mm10\\tchr12\\t35690979\\t35768472',\n",
              " 'mm10\\tchr5\\t147205131\\t147212149',\n",
              " 'mm10\\tchr5\\t147263601\\t147264086',\n",
              " 'mm10\\tchr3\\t34645597\\t34646134',\n",
              " 'mm10\\tchr14\\t67745323\\t67746327',\n",
              " 'mm10\\tchr17\\t35501113\\t35507015',\n",
              " 'mm10\\tchr6\\t52407532\\t52459468',\n",
              " 'mm10\\tchr6\\t52527210\\t52556565',\n",
              " 'mm10\\tchr7\\t3193003\\t3218182',\n",
              " 'mm10\\tchr7\\t3199567\\t3201439',\n",
              " 'mm10\\tchr7\\t3202646\\t3204360',\n",
              " 'mm10\\tchr7\\t3205726\\t3207280',\n",
              " 'mm10\\tchr7\\t3209567\\t3211439',\n",
              " 'mm10\\tchr7\\t3212448\\t3214558',\n",
              " 'mm10\\tchr7\\t3216607\\t3218399',\n",
              " 'mm10\\tchr2\\t180362582\\t180377003',\n",
              " 'mm10\\tchr2\\t180361998\\t180364335',\n",
              " 'mm10\\tchr2\\t180366734\\t180368371',\n",
              " 'mm10\\tchr2\\t180367903\\t180369540',\n",
              " 'mm10\\tchr2\\t180375974\\t180377844',\n",
              " 'mm10\\tchr6\\t51108708\\t51130031',\n",
              " 'mm10\\tchr6\\t51168034\\t51174811',\n",
              " 'mm10\\tchr6\\t51117747\\t51120656',\n",
              " 'mm10\\tchr6\\t51168147\\t51170512',\n",
              " 'mm10\\tchr6\\t51171474\\t51173111',\n",
              " 'mm10\\tchr6\\t51173363\\t51175544',\n",
              " 'mm10\\tchr2\\t129350852\\t129356319',\n",
              " 'mm10\\tchr12\\t109528035\\t109532509',\n",
              " 'mm10\\tchr12\\t103852772\\t103852898',\n",
              " 'mm10\\tchr12\\t103852898\\t103853032',\n",
              " 'mm10\\tchr12\\t103853032\\t103853127',\n",
              " 'mm10\\tchr7\\t103771431\\t103772043',\n",
              " 'mm10\\tchr6\\t67556736\\t67558736',\n",
              " 'mm10\\tchr6\\t67561736\\t67563736',\n",
              " 'mm10\\tchr6\\t67560136\\t67562136',\n",
              " 'mm10\\tchr8\\t120876144\\t120876856',\n",
              " 'mm10\\tchr8\\t120882734\\t120883410',\n",
              " 'mm10\\tchr8\\t121088830\\t121089174',\n",
              " 'mm10\\tchr17\\t63168191\\t63175262',\n",
              " 'mm10\\tchr10\\t87283358\\t87312833',\n",
              " 'mm10\\tchr17\\t26829232\\t26829742',\n",
              " 'mm10\\tchr2\\t152735054\\t152735521',\n",
              " 'mm10\\tchr13\\t43362622\\t43364444',\n",
              " 'mm10\\tchr2\\t130280565\\t130281887',\n",
              " 'mm10\\tchr16\\t4721118\\t4724819',\n",
              " 'mm10\\tchr10\\t77989870\\t77993100',\n",
              " 'mm10\\tchr4\\t115040426\\t115042426',\n",
              " 'mm10\\tchr12\\t113302447\\t113305102',\n",
              " 'mm10\\tchr12\\t113305522\\t113308796',\n",
              " 'mm10\\tchr12\\t113314217\\t113314925',\n",
              " 'mm10\\tchr11\\t100859351\\t100885169']"
            ]
          },
          "metadata": {},
          "execution_count": 39
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "with open('LISTA_INDICI_MM10.txt', 'w') as file:\n",
        "    # Scrivi ogni elemento della lista nel file\n",
        "    for item in final2:\n",
        "        file.write(item + '\\n')"
      ],
      "metadata": {
        "id": "SaTUklOp2dK-"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}